{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "264c8b21",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/home/pal194/RAG_guided_chat_bot/notebook'"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "4232754a",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "os.chdir(\"../\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "fc151cdd",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/home/pal194/RAG_guided_chat_bot'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%pwd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "ebfa4350",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/pal194/miniconda3/envs/py_rag/lib/python3.10/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
      "  from .autonotebook import tqdm as notebook_tqdm\n"
     ]
    }
   ],
   "source": [
    "from langchain_community.document_loaders import PyPDFLoader\n",
    "from langchain_community.document_loaders import DirectoryLoader\n",
    "from langchain_text_splitters import RecursiveCharacterTextSplitter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "50c3a73e",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Extract text from PDF files\n",
    "def load_pdf_files(data):\n",
    "    loader = DirectoryLoader(\n",
    "        data,\n",
    "        glob=\"*.pdf\",\n",
    "        loader_cls=PyPDFLoader\n",
    "    )\n",
    "\n",
    "    documents = loader.load()\n",
    "    return documents"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "7cacf0a8",
   "metadata": {},
   "outputs": [],
   "source": [
    "extracted_data = load_pdf_files(\"data\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "821f13be",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 0, 'page_label': '1'}, page_content=\"Received: 14 April 2025 | Revised: 15 July 2025 | Accepted: 16 July 2025\\nDOI: 10.1002/imt2.70070\\nREVIEW ARTICLE\\nThe microbiome in cancer\\nAnqi Lin1,2 | Minying Xiong2 | Aimin Jiang3 | Lihaoyun Huang2 |\\nHank Z. H. Wong4 | Suyin Feng1 | Chunyan Zhang2 | Yu Li2 | Li Chen5 |\\nHao Chi6,7 | Pengpeng Zhang8 | Bicheng Ye9 | Hengguo Zhang10 |\\nNan Zhang11 | Lingxuan Zhu2 | Weiming Mou2 | Junyi Shen2 | Kailai Li2 |\\nWentao Xu2 | Haoxuan Ying2 | Cangang Zhang12 | Dongqiang Zeng13,14 |\\nJindong Xie15 | Xinpei Deng16 | Qi Wang17 | Jianying Xu18 |\\nWenjie Shi19 | Chang Qi20 | Chunrun Qu21 | Xufeng Huang22,23 |\\nAndrás Hajdu22 | Chaoqun Li24 | Changmin Peng25 | Xuanye Cao26 |\\nGuangsheng Pei27 | Lin Zhang28 | Yujia Huo28 | Jiabao Xu29 |\\nAntonino Glaviano30 | Attila Gábor Szöllősi31 | Sicheng Bian32 |\\nZhengrui Li33 | Hailin Tang15 | Bufu Tang34 | Zaoqu Liu35 |\\nJian Zhang2 | Kai Miao5,36 | Quan Cheng37,38 | Ting Wei2 |\\nShuofeng Yuan39,40 | Peng Luo1,2,40\\nCorrespondence\\nQuan Cheng, Department of Neurosurgery,\\nXiangya Hospital, Central South University,\\nChangsha 410008, China.\\nEmail: chengquan@csu.edu.cn\\nTing Wei, Department of Oncology,\\nZhujiang Hospital, Southern Medical\\nUniversity, Guangzhou 510282, China.\\nEmail: weitingyouyou@qq.com\\nShuofeng Yuan, Department of Infectious\\nDisease and Microbiology, The University of\\nHong Kong‐Shenzhen Hospital, Shenzhen\\n518053, China.\\nEmail: yuansf@hku.hk\\nPeng Luo, Donghai County People's\\nHospital (Affiliated Kangda College of\\nNanjing Medical University), Lianyungang,\\nChina.\\nEmail: luopeng@smu.edu.cn\\nAbstract\\nThe human microbiome is now recognized as a central regulator of cancer\\nbiology, intricately shaping tumor development, immune dynamics, and thera-\\npeutic response. This comprehensive review delineates the multifaceted roles of\\nb a c t e r i a ,v i r u s e s ,a n df u n g ii nm o d u lating the tumor microenvironment and\\nsystemic immunity across diverse cancer ty p e s .W es y n t h e s i z ec u r r e n te v i d e n c e\\non how microbial dysbiosis promotes carcinogenesis via chronic inflammation,\\nmetabolic reprogramming, genotoxic stress, immune evasion, and epigenetic\\nremodeling. This review emphasizes organ ‐specific microbiome signatures\\nand highlights their potential as non‐invasive biomarkers for early detection,\\ntreatment stratification, and prognosis. Furthermore, we explore the impact of\\nintratumoral microbiota on cancer therapies, uncovering how microbial metab-\\nolites and host–microbe interactions shape therapeutic efficacy and resistance.\\nFinally, advances in microbiome‐targeted strategies, such as probiotics, fecaliMeta. 2025;4:e70070. wileyonlinelibrary.com/journal/imeta | 1 of 104\\nhttps://doi.org/10.1002/imt2.70070\\nThis is an open access article under the terms of theCreative Commons AttributionLicense, which permits use, distribution and reproduction in any medium, provided\\nthe original work is properly cited.\\nFor affiliations refer to page 65.\\nAnqi Lin, Minying Xiong, Aimin Jiang, Lihaoyun Huang, Hank Z. H. Wong, and Suyin Feng contributed equally.\\nZhengrui Li, Hailin Tang, Bufu Tang, Zaoqu Liu, Jian Zhang, Kai Miao were senior authors.\\n© 2025 The Author(s).iMeta published by John Wiley & Sons Australia, Ltd on behalf ofiMeta Science.\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 1, 'page_label': '2'}, page_content=\"Funding information\\nMacao Science and Technology\\nDevelopment Fund,\\nGrant/Award Numbers: 0009/2022/AKP,\\n0073/2021/A2; Hunan Youth Science and\\nTechnology Talent Project,\\nGrant/Award Number: NO.2023RC3074;\\nthe Science and Technology Program of\\nGuangzhou, Grant/Award Number:\\n2023A04J1257; the Guangdong Basic and\\nApplied Basic Research Foundation,\\nGrant/Award Number:\\n2022A1515111212; the National Natural\\nScience Foundation of China,\\nGrant/Award Numbers: 82172750,\\n82172811, 82260546, 82373129; the\\nNatural Science Foundation of\\nGuangdong Province,\\nGrant/Award Number:\\n2021A1515012593; Multiyear research\\ngrant (MYRG), Grant/Award Number:\\nMYRG‐GRG2023‐00150‐FHS‐UMDF to\\nK. M.\\nmicrobiota transplantation, and engineered microbes offer new avenues for\\nadjunctive cancer therapy. This review provides a roadmap for future investiga-\\ntion and underscores the transformative promise of microbiome modulation in\\ncancer prevention and treatment.\\nKEYWORDS\\ncancer, microbiome, precision oncology, treatment, tumor microenvironment\\nHighlights\\n• Microbiome dysbiosis drives carcinogenesis via multiple mechanisms.\\n• Microbiome‐targeted strategies offer novel therapeutic avenues.\\n• Organ‐specific microbiome signatures serve as non‐invasive biomarkers.\\n• Intratumoral microbiota modulate therapy efficacy and resistance.\\n• Microbial metabolites reprogram the tumor microenvironment.\\nINTRODUCTION\\nMicroorganisms comprise diverse microscopic entities,\\nincluding bacteria, fungi, viruses, archaea, and protists.\\nThese microorganisms, including 1013–1014 bacteria col-\\nonizing human skin, intestines, and respiratory tract,\\nestablish complex symbiotic relationships with the host\\nand are collectively referred to as the body's “second\\ngenome” [1–4]. Symbiotic microbes maintain host health\\nthrough complex metabolic reciprocity [5]. In contrast,\\npathogenic microorganisms pose a threat to the host's\\nhealth. Notably, microorganisms can have dual\\n(beneficial and pathogenic) roles. When host immune\\nfunction is compromised, commensal microbes may\\nconvert to conditionally opportunistic pathogens, poten-\\ntially disrupting host health [6]. Clinical studies have\\nconfirmed that the composition of the gut microbiome\\ncan be modulated through fecal microbiota transplanta-\\ntion (FMT) and the administration of probiotic prepara-\\ntions, influencing immune cell infiltration, immune\\nfactor release, and inflammatory responses in the intes-\\ntine and surrounding tumor tissues, thereby improving\\nthe immune status of the tumor microenvironment\\n(TME) [7–9]. Consequently, the microbiome serves not\\nonly as a biomarker for disease diagnosis but also as a\\npromising target for therapeutic intervention [10–12].\\nWhile research on gut microbes dates back to the\\n18th century, comprehensive investigations into the gut\\nmicrobes expanded exponentially in the 21st century. A\\nstudy in 1972 established that gut microbes play a critical\\nrole in drug transformation [13]. This seminal work laid\\nthe groundwork for understanding the microbiome as a\\nkey modulator of pharmacology, a concept fundamental\\nto modern microbiome–drug interaction studies. Since\\n2006, several pivotal studies have begun to emphasize the\\nsignificant impact of diet on gut microbial composition\\nand host metabolism [ 14–19]. In 2007, the National\\nInstitutes of Health (NIH) launched the Human Micro-\\nbiome Project (HMP), the first large‐scale collaborative\\nprogram to study the human gut microbiome [20]. The\\nMetagenomics of the Human Intestinal Tract (MetaHIT)\\nproject, initiated by the European Union in 2008, repre-\\nsents another landmark effort in the comprehensive and\\nsystematic study of gut microbiota. This initiative sys-\\ntematically characterized the genomic architecture of\\nintestinal microbial communities and established foun-\\ndational frameworks for investigating microbiome –\\ndisease associations, significantly advancing the\\ndevelopment of precision medicine approaches [ 21].\\nSubsequently, Yatsunenko et al. conducted a large‐scale\\ncomparative analysis demonstrating that human\\npopulations across distinct geographic regions have sig-\\nnificantly different compositions and functional char-\\nacteristics of gut microbes [ 22]. The American Gut\\nProject (AGP), launched in 2012, was further expanded\\ninto a global microbiome research program to explore the\\ncomposition and diversity of microbes in geographically\\ndiverse populations [ 23]. Notably, the study of gut\\nmicrobes–host interactions has gradually extended to the\\nfield of disease treatment. A groundbreaking study in\\n2018 demonstrated that gut microbial signatures can\\npredict immunotherapeutic response in patients with\\n2 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 2, 'page_label': '3'}, page_content=\"melanoma, non‐small cell lung cancer, and renal cell\\ncarcinoma [24]. This landmark discovery provided the\\nfirst robust clinical evidence establishing a causal rela-\\ntionship between the gut microbiome composition and\\nthe efficacy of cutting ‐edge cancer immunotherapies,\\nthereby creating a paradigm‐shifting theoretical frame-\\nwork for understanding the microbiome's pivotal role in\\ndeveloping therapeutic intervention. These findings es-\\ntablish a crucial theoretical framework for understanding\\nthe pivotal role of tumor‐associated microbes in cancer\\ninitiation, progression, and therapeutic intervention.\\nInvestigations into intratumoral microbes began in\\nthe 19th century, but significant advances remained\\nlimited until recent decades [25]. In 2020, through a\\nlarge‐scale multi‐omic analysis of diverse primary human\\nmalignancies, researchers established the presence of\\nhighly specific microbiome profiles and functional char-\\nacteristics across different tumor types [26]. This study\\nprovides comprehensive evidence for the existence of\\ndistinct tumor‐type‐specific intratumoral microbiomes.\\nIn recent years, the prevalence of microbes within\\ntumors has been further confirmed by increasing\\namounts of reliable evidence obtained through various\\nadvanced technological approaches [27, 28]. Studies have\\ndemonstrated that the species composition and abun-\\ndance distribution of intratumoral microbes in different\\ncancer types show marked heterogeneity [ 29–31]. As\\ninvestigations into tumor‐associated microbiota intensify,\\nthe precise origins and molecular mechanisms governing\\nmicrobial colonization remain fundamental unresolved\\nquestions in the field. Recent investigations have char-\\nacterized three potential routes of microbial access to\\ntumor tissues: disruption of epithelial barrier integrity,\\nmigration from adjacent microenvironmental niches, and\\nsystemic dissemination [ 29, 32]. Substantial evidence\\nindicates that intratumoral microbes are involved in the\\nregulation of tumorigenesis, progression, and therapeutic\\nresponse through multiple molecular mechanisms,\\nincluding DNA damage induction, oncogenic signaling\\npathway activation, and alteration of epigenetic\\nregulation [ 33]. Additionally, intratumoral microbes\\nsignificantly affect the clinical efficacy of immuno-\\ntherapy and chemotherapy by remodeling the tumor\\nimmune microenvironment (TIME). As intratumoral\\nmicrobiome research advances, microbiome ‐targeted\\ntumor precision diagnosis and individualized treatment\\nstrategies have emerged as an international research\\nfrontier. This recognition underscores the immense\\ntranslational potential of the intratumoral microbiome\\nand its associated metabolites, positioning them as criti-\\ncal targets for the next generation of diagnostics and\\npersonalized therapeutic interventions, particularly\\nwithin the TME and immunomodulation approaches.\\nHowever, methodological challenges still exist, including\\nsample acquisition, microbial detection, and functional\\nvalidation [34–37]. The study of intratumoral microbiota\\nholds profound clinical translational potential and is\\npositioned to constitute a significant breakthrough and\\npromising research direction in anti‐tumor therapy.\\nThe microbiome plays a critical role in multiple\\naspects of tumor growth, proliferation, diagnosis, treat-\\nment, and prognosis. Microorganisms and malignant\\ncells engage in complex bidirectional interactions [38].\\nMicroorganisms can promote tumorigenesis and devel-\\nopment through direct genotoxic mechanisms or indirect\\nimmunomodulatory effects [39, 40]. Certain beneficial\\nbacteria are also associated with anti ‐cancer effects.\\nProbiotics maintain intestinal barrier homeostasis, regu-\\nlate anti‐\\ninflammatory cytokine levels with anti‐cancer\\nproperties, and enhance immune surveillance through\\nactivation of phagocytic cells. Germ‐free murine models\\ndemonstrate compromised immunosurveillance and\\nincreased susceptibility to chemical carcinogenesis [41].\\nLactobacillus and Bifidobacterium have also demon-\\nstrated potential to bind and degrade carcinogenic com-\\npounds [ 42]. Importantly, the microbial community\\nserves as a diagnostic and prognostic biomarker [43–45].\\nFurthermore, the microbiome influences therapeutic ef-\\nfects by metabolizing drugs and dynamically modulating\\nthe immune microenvironment [46]. In immunotherapy,\\ndysbiosis of the intestinal microbes (such as reduction of\\nBifidobacteria) exacerbates anti‐CTLA‐4 therapy‐induced\\ncolitis, whereas probiotic supplementation both mitigates\\ntoxicity and enhances therapeutic efficacy [ 47]. As\\nresearch exploring microbiome–tumor interactions in-\\ntensifies, microbe‐targeted therapies are gradually being\\nrecognized for their anti‐tumor effects. For example, pro-\\nbiotics help maintain intestinal homeostasis and inhibit\\nproliferation of malignant cells [48]. FMT therapy con-\\ntributes to immunomodulation of the TME, resulting in a\\ntumor‐preventive effect at an early stage [8, 49]. The\\nmicrobiome, as an emerging biomarker and potential\\ntherapeutic target, has broad applications in tumor diag-\\nnosis and treatment [ 50–53]. Consequently, microbial\\ndysbiosis and intervention emerge as critical factors\\nthroughout carcinogenesis and disease progression. Tar-\\ngeted microbial therapy is emerging as a promising\\napproach for integrative oncological strategies [54–59].\\nThis review comprehensively synthesizes recent ad-\\nvances in understanding the complex interactions\\nbetween the microbiome and cancer biology. Initially, we\\nexamine the intrinsic relationships between the intestinal\\nmicrobiota (encompassing bacteria, viruses, fungi, and\\ntheir metabolites) and the intestinal barrier in carcino-\\ngenesis, characterize microbial signatures within tumor\\ntissues, and assess their organ ‐specific manifestations\\nMICROBIOME IN CANCER | 3 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 3, 'page_label': '4'}, page_content='and diagnostic implications across diverse anatomical\\nsites including the oral cavity, integumentary system,\\nurogenital tract, and respiratory tract. We subsequently\\nfocus on the diagnostic and prognostic implications of\\nmicrobiome profiles, emphasizing the potential applica-\\ntions of site‐specific and kingdom‐specific microbial sig-\\nnatures as clinically actionable biomarkers for early\\ncancer detection, therapeutic response prediction, and\\noutcome assessment. Building upon the above studies,\\nwe further elucidate the synergistic effects of microbiome\\nregulation in chemotherapy, radiotherapy, immuno-\\ntherapy, targeted therapy, and surgery. Additionally,\\nwe comprehensively summarize various innovative\\nmicrobiome‐based therapeutic strategies, including pro-\\nbiotic/prebiotic interventions, FMT, engineered bacteria\\ntherapy, and targeted antimicrobial approaches. Studies\\nhave shown that the microbiome not only influences\\ntumor initiation and progression but also offers promis-\\ning new targets for tumor prevention and treatment.\\nLooking forward, continued technological innovation in\\nmicrobiome research methodologies, elucidation of\\nmolecular mechanisms underlying microbiome –host\\ninteractions, development of precision microbiome ‐\\ntargeted therapeutic strategies, and advancement of\\nrobust clinical translational studies will collectively\\ndeliver more personalized, effective, and safer therapeu-\\ntic interventions for cancer patients.\\nGUT MICROBIOME AND TUMOR\\nDEVELOPMENT\\nAssociation of bacteria with tumorigenesis\\nGut microbes contribute to tumor development through\\na variety of complex mechanisms, predominantly pro-\\nmoting neoplastic progression via induction of chronic\\ninflammatory responses, disruption of epithelial barrier\\nintegrity, and comprehensive remodeling of the TIME\\n[60–63]. The relationship between microbiota composi-\\ntion and carcinogenesis demonstrates pronounced\\nspecies‐specificity, with pro‐tumorigenic effects predom-\\ninantly characterized by depletion of beneficial com-\\nmensal taxa, enrichment of pathogenic bacteria, and\\ndysregulated expansion of opportunistic bacteria [64–66].\\nHowever, some investigations have reported heteroge-\\nneous findings [67, 68]. Table 1 summarizes the asso-\\nciations between different types of bacteria (probiotics\\nand pathogenic bacteria) and various tumors, delineating\\ntheir proposed mechanistic contributions to carcinogen-\\nesis or tumor suppression, thereby providing a compre-\\nhensive framework for understanding the diverse roles of\\nthe microbiome in cancer pathogenesis.\\nProbiotics and tumorigenesis\\nIntestinal probiotics, beneficial bacteria colonizing the\\nhuman gastrointestinal tract, play a crucial role in pro-\\nmoting nutrient absorption and maintaining gastro-\\nintestinal health by orchestrating gut microbial balance\\nand calibrating intestinal mucosa and systemic immune\\nfunction through multiple mechanisms [69–71]. Com-\\nmon intestinal probiotics primarily includeBifidobacter-\\nium, Lactobacillus, and Bacillus [72, 73, 83, 84 ].\\nLactobacillus and Bifidobacterium demonstrate signifi-\\ncant anti‐tumor activity, inhibiting not only digestive\\nmalignancies, such as esophageal, gastric, colon, and li-\\nver cancer, but also bladder cancer (BCa), through\\nmechanisms including regulation of the pH of the TME,\\nmodulation of bile acid metabolism, and degradation of\\npotential carcinogens and their metabolites [66, 85–88].\\nAdditionally, tumor metastasis in patients with gastric\\ncardia adenocarcinoma considerably correlates with\\ndecreased Lactobacillus counts in the stomach [89]. Pa-\\ntients with hepatocellular carcinoma (HCC) exhibit sig-\\nnificantly altered gut microbial composition, with\\nhepatitis B virus (HBV)‐associated HCC displaying sig-\\nnificant enrichment of Bacteroides and Lachnospiraceae\\nincertae sedis, while non ‐HBV‐associated HCC shows\\ndecreased abundance ofFaecalibacterium, Ruminococcus,\\nand Ruminoclostridium [90, 91]. Notably, patients with\\nintrahepatic cholangiocarcinoma had a significantly\\nhigher abundance of Lactobacillus in fecal samples\\ncompared to healthy controls [ 67]. Comprehensive\\nanalyses have established significantly reduced duodenal\\nmicrobial diversity in patients with pancreatic cancer\\n(PC), with enrichment ofBifidobacterium spp. andRothia\\nbacteria [67]. Furthermore, multiple microorganisms,\\nincluding seven Bacteroidales species, significantly en-\\nhanced anti‐tumor immune responses mediated by CD8\\n+\\nT cells in mice through synergistic effects [92]. Although\\nBifidobacterium and Lactobacillus are widely recognized\\nas probiotics within the intestinal ecosystems, their\\nfunctions in a specific TME may diverge substantially\\nfrom their canonical probiotic activities. This effect is\\ninfluenced by multiple factors, including tumor histol-\\nogy, metabolic state, and the dominant bacterial strain.\\nTherefore, “probiotics” cannot be simplistically equated\\nwith “anti‐tumor,” necessitating that future microbiome‐\\ntargeted therapies carefully consider strain ‐specific\\nfunctions within particular TME.\\nPathogenic bacteria and tumorigenesis\\nPathogenic bacteria are disease‐causing microorganisms.\\nThese bacteria can damage host cells either directly or\\n4 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 4, 'page_label': '5'}, page_content=\"TABLE 1 Associations and potential mechanisms between different bacterial types and various tumors.\\nBacterial type Representative strains\\nAssociated\\ncancer types Mechanism Specificity References\\nProbiotics Lactobacillus GC\\nCRC\\nLiver cancer\\nBCa\\n• Regulate the TME pH and bile acid\\nmetabolism;\\n• Degrade carcinogens.\\nParticular case: elevated\\nLactobacillus abundance in\\nintrahepatic\\ncholangiocarcinoma\\n[58, 67, 69–73]\\nBifidobacterium GC\\nCRC\\nLiver cancer\\n• Competitively inhibit pathogen colonization;\\n• Regulate the TME pH and bile acid\\nmetabolism;\\n• Maintain intestinal barrier integrity.\\nEnriched in PC [ 58, 67, 73]\\nPathogens Helicobacter pylori GC\\nCRC\\nPC\\nHCC\\n• Induce DNA damage;\\n• Disrupt DNA repair;\\n• Activate STAT3 signaling;\\n• Decrease Treg cells;\\n• Increase CD3 cells.\\nReduces Barrett's esophagus,\\nEAC and IBD risk\\n[64, 68, 74–78]\\nFusobacterium nucleatum CRC\\nPC\\nOral cancer\\nGliomas\\n• Activate pro‐cancer pathways;\\n• Induce DNA methylation;\\n• Suppress immune cell anti‐tumor function.\\nPromotes tumorigenesis in PC [ 40, 79–81]\\nPseudomonas PC • Pseudoxanthomonas: recruits CD8\\n+ T cells\\n(anti‐tumor).\\nImprove the OS of PC [ 81]\\nEnterotoxigenic Bacteroides fragilis CRC • Highly inducible to inflammatory stimuli. [ 82]\\nMicrobial\\nsynergy\\nHelicobacter pylori + phage CRC • Altering the enterovirus composition. Synergism promotes CRC\\nprogression\\n[75]\\nRuminococcaceae + Bacteroides HCC • Induce hepatic steatosis;\\n• Accumulate toxic metabolites (such as\\nammonia).\\nAlters liver metabolic\\nmicroenvironment\\n[66]\\nAbbreviations: BCa, bladder cancer; CRC, colorectal cancer; EAC, esophageal adenocarcinoma; GC, gastric cancer; HCC, hepatocellular carcinoma;IBD, inflammatory bowel disease; OS, overall survival; PC,\\npancreatic cancer; pH, potential of hydrogen; STAT3, signal transducer and activator of transcription 3; TME, tumor microenvironment.\\nMICROBIOME IN CANCER | 5 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 5, 'page_label': '6'}, page_content=\"indirectly by releasing toxins or inducing immune\\nresponses. Intestinal pathogenic bacteria can induce\\nacute or chronic inflammatory responses in the host,\\nseveral of which directly contribute to tumor develop-\\nment and progression [ 93–98]. Common intestinal\\npathogenic bacteria include Salmonella, Shigella, en-\\nterohemorrhagic E.coli, Staphylococcus aureus(S. aureus),\\nand Clostridium difficile[99–104]. Gastrointestinal tumor\\ndevelopment closely relates to the dynamics of various\\nintestinal microbiota, including enterotoxigenic Bacter-\\noides fragilis, Porphyromonas, Flavonifractor plauti[105],\\nCampylobacter spp.[ 106, 107], and Fusobacterium nu-\\ncleatum (F. nucleatum)[ 66]. Bacteroides and Rumino-\\ncoccaceae can contribute to HCC development through\\npromoting inflammatory responses, facilitating toxic\\nsubstances, and inducing hepatic steatosis [66]. Signifi-\\ncant enrichment ofActinomyces spp. has been identified\\nin extrahepatic cholangiocarcinoma tissues [ 108],\\nwhile chronic Salmonella typhi infection significantly\\ncorrelates with gallbladder cancer development [109]. PC\\ntissues harbor significant Elizabethkingia enrichment\\n[110], and Hungatella hathewayi can promote cancer by\\nreprogramming host DNA methylation patterns [ 79].\\nBeyond digestive system tumors, gut microbiome dysre-\\ngulation exhibits associations with malignancies in\\nother organ systems. Breast cancer (BC) patients' fecal\\nsamples show significantly lower relative abundance\\nof Bacteroidetes, Firmicutes, and Faecalibacterium\\nprausnitzii, with higher levels ofProteobacteria, Actino-\\nbacteria, Verrucomicrobia, and Firmicutes/Bacteroidetes\\nratio [111, 112]. Specific\\nClostridium species and Rumi-\\nnococcaceae family members increase BC risk by altering\\nestrogen metabolic pathways [ 113, 114]. Lung cancer\\npatients show significantly upregulated Ruminococci\\n(R. gnavus ) abundance in fecal microbiome, with\\nsquamous cell carcinoma (SCC) patients having higher\\nProteobacteria, Gammaproteobacteria, Bacteroides,a n d\\nEnterobacteriaceae abundance, while adenocarcinoma\\npatients' feces contain higherFusicatenibacter and Rose-\\nburia abundance [115]. Prostate cancer (PCa) studies\\ndemonstrate a significant correlation between cancer\\ndevelopment and relative abundance of Alphaproteo-\\nbacteria and Bacteroides massiliensis[116].\\nAmong microorganisms closely associated with gas-\\ntric tumors, Helicobacter pylori (H. pylori) is the most\\nextensively documented. H. pylori is the most common\\npathogenic bacterium in gastric tissues, promoting\\ntumorigenesis and progression by inducing DNA dam-\\nage, interfering with DNA repair mechanisms, and acti-\\nvating oncogenic pathways [ 64]. Numerous studies\\nconfirm H. pylori's significant positive association with\\ngastric, colorectal, and PC risks, leading to its classifica-\\ntion as a class I carcinogen by the World Health\\nOrganization [117–119]. Cross‐sectional studies demon-\\nstrate thatH. pyloriinfection significantly correlates with\\ncolorectal cancer (CRC) risk (OR = 1.9), surpassing the\\nassociations observed with established risk factors\\nincluding body mass index (BMI), smoking, and alcohol\\nconsumption [ 120]. Research indicates that both H.\\npylori‐infected CRC patients and mouse models exhibit\\nactivation of pro ‐carcinogenic signal transducer and\\nactivator of transcription 3 (STAT3) signaling, loss of\\nintestinal epithelial goblet cells, reduction in Treg cells,\\nand significant increases in CD3 cells (pro‐inflammatory\\nT cells) [74]. These findings suggest thatH. pylori func-\\ntions not merely as a risk indicator for CRC but also\\nestablishes a pro‐carcinogenic microenvironment within\\nthe colon. Additionally, animal experiments demonstrate\\nthat H. pylori can increase the abundance of CRC ‐\\nassociated intestinal bacteriophages, thereby altering the\\nintestinal virome composition, which is associated with\\nCRC development [75]. Cohort studies reveal that sero-\\nlogical responses to the\\nH. pylori virulence factor VacA\\nexhibit positive correlations with CRC risk [76]. A recent\\ncohort study demonstrated thatH. pylori has significant\\npositive associations with both CRC incidence and\\nmortality. Individuals with untreatedH. pylori infection\\nexhibited significantly higher incidence and mortality\\nrates compared to those who received treatment [121].\\nHelicobacter hepaticus has been identified in hepatitis C\\nvirus (HCV)‐associated HCC tissue specimens [122, 123],\\nalthough subsequent research challenged its pathogenic\\nrole in HBV‐associated HCC [122]. Mouse model studies\\nhave shown that Helicobacter hepaticus can inhibit\\nHCC progression by suppressing intrinsic immunity's\\nrecognition and clearance functions of tumor cells [124].\\nThe detection ofHelicobacter bilis, Helicobacter hepaticus,\\nor H. pylori in biliary tract cancer tissues demonstrates\\nsignificant correlations with increased cancer risk [109,\\n125–127]. Various intestinal microorganisms, including\\nBacteroidetes, Proteobacteria,a n dStreptococcaceae,s h o w\\nsynergistic effects with H. pylori in promoting gastro-\\nintestinal tumorigenesis [128]. Interestingly,H. pylori ex-\\nhibits protective effects against Barrett's esophagus,\\nesophageal adenocarcinoma (EAC), and inflammatory\\nbowel disease (IBD) [69, 77, 78]. Multiple studies have\\ndemonstrated a negative correlation between H. pylori\\ninfection and EAC and Barrett's esophagus [129]. A case‐\\ncontrol study established thatH. pylori infection exhibits\\nstrong negative associations with both erosive esophagitis\\nand Barrett's esophagus [68]. A previous community‐\\nbased study reached the same conclusion, showing thatH.\\npylori infection and CagA\\n+ status are negatively associated\\nwith newly diagnosed Barrett's esophagus [130]. A sub-\\nsequent meta‐analysis including 10 studies revealed that\\namong Asian populations, IBD patients had significantly\\n6 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 6, 'page_label': '7'}, page_content=\"lower H. pylori infection rates compared to non‐IBD pa-\\ntients [78]. The negative association between H. pylori\\ninfection and IBD is independent of ethnicity, age, H.\\npylori detection methods, and previous medication use,\\nwhile antibiotic use influenced this association [131, 132].\\nResearch suggests that the H. pylori genome contains\\nimmunomodulatory elements that may explain its pro-\\ntective effect against IBD through suppression of inflam-\\nmatory responses via downregulation of dendritic\\ncell (DC)‐produced IL‐12 and type I interferon levels\\n[133, 134]. Case‐control studies show that both gastric\\natrophy and H. pylori seropositivity are associated with\\nreduced risk of EAC, Barrett's esophagus, and reflux eso-\\nphagitis, with this negative correlation persisting in pa-\\ntients without gastric atrophy [135, 136]. A recent study\\npostulates that H. pylori eradication may disrupt the\\ninflammatory microenvironment, thereby inhibiting the\\nproliferation of other bacteria and reducing EAC risk\\n[137]. However, some cohort studies have not observed\\nincreased EAC risk followingH. pylorieradication [138].\\nConditional pathogens are bacteria that may cause\\ndisease when there is an imbalance in the host micro-\\nbiota homeostasis [139, 140], and they are associated\\nwith impaired immune systems. These bacteria can ele-\\nvate cancer risk and facilitate tumorigenesis and pro-\\ngression through direct DNA damage or by inducing\\nmicrobiota dysbiosis [6, 141, 142]. Veillonella and Pre-\\nvotella are commensal in the gut most of the time, but\\ncan also cause disease in the context of the dysregulated\\nimmune microenvironment.Veillonella spp. and specific\\nspecies of Prevotella are associated with gastric cancer\\n(GC) risk [143]. Elevated levels of Bacteroides fragilis\\n[144–146] and Streptococcus gallolyticus (S. gallolyticus)\\n[147, 148] were detected in tumor tissues of patients with\\nCRC, and fecal specimens from these patients demon-\\nstrated significantly higher prevalence of Enterococcus\\nfaecalis [149]. Additionally, enterotoxigenic Bacteroides\\nfragili (ETBF) is highly responsive to inflammatory\\nstimuli and considered a risk factor for CRC [82]. En-\\nterococcus faecalis potentially promotes chromosomal\\ninstability associated with both sporadic adenomatous\\npolyps and CRC [150\\n].\\nHepatobiliary tumors have also been associated with\\nvarious conditionally pathogenic bacteria. Enrichment of\\nFusobacterium, Prevotella, and Novosphingobium was\\nobserved in extrahepatic cholangiocarcinoma tissues\\n[108]. Fusobacteria enrichment was observed in duode-\\nnal fluids from PC patients, especially those with\\nshort‐term survival [ 151]. The presence of Pseudox-\\nanthomonas in PC tissues correlated with increased\\ndensity of CD8\\n+ T cells and improved overall survival\\n(OS) [152]. In contrast, animal experiments showedB.\\npseudolongum migrating from intestine to pancreas and\\nenriched in PC tissues, obstructing immunity mediated\\nby T cells, and suppressing immune responses to\\npancreatic tumors [110]. Additionally, large amounts of\\nFusobacterium in PC tumor tissues often indicate poor\\nprognosis [153].\\nF. nucleatum is a common opportunistic pathogen\\nresiding in the gastrointestinal tract and oral cavity,\\nassociated with multiple cancers, including CRC, PC,\\noral cancer, and gliomas [40, 80, 110, 154]. This micro-\\norganism is significantly enriched in neoplastic tissues\\ncompared to healthy mucosa across these diverse cancer\\ntypes. F. nucleatum contributes to tumor development\\nand progression through diverse mechanisms: it repro-\\ngrams host DNA methylation patterns, potentially facil-\\nitating the initiation of PC [79]; enhances CRC metastasis\\nby suppressing anti ‐tumor immunity, promoting\\nimmune evasion, and modulating the E ‐cadherin/β‐\\ncatenin signaling pathway via FadA adhesin [ 81,\\n154–156]; recruits tumor ‐associated macrophages\\n(TAMs) to the TME via CCL20 upregulation, thereby\\naccelerating the progression of CRC [157]; promotes M2\\nmacrophage polarization and activation [ 158]; and\\nstimulates cancer cell invasion in PC through autocrine/\\nparacrine pathways [159]. Within the TME, intratumoral\\nF. nucleatum modulates immune responses by inducing\\nT helper 17 (Th17) cell enrichment and promoting\\nsecretion of IL‐17 family cytokines via the metabolite‐\\nsensing receptor FFAR2, thereby establishing pro ‐\\ninflammatory microenvironmental conditions [ 160].\\nClinically, F. nucleatum serves as a specific diagnostic\\nand prognostic biomarker in CRC, GC, oral cancer, PC,\\nand lung cancer, with its intratumoral abundance dem-\\nonstrating a significant negative correlation with OS in\\nCRC patients [40, 161–164]. Furthermore, F. nucleatum\\ninduces chemoresistance in CRC, particularly to ox-\\naliplatin, by inhibiting caspase ‐mediated cascades via\\nautophagy modulation and BIRC3 upregulation\\n[165–167]. As previously described, F. nucleatum can\\nregulate the expression ofβ‐catenin [156]. Studies have\\nshown that theβ‐\\ncatenin signaling pathway, which can\\nbe activated through multiple pathways, is associated\\nwith the development of resistance to lenvatinib in HCC\\n[168–170]. Therefore, the modulation of β‐catenin ex-\\npression byF. nucleatummay influence the sensitivity of\\ntumor cells to lenvatinib, although the precision molec-\\nular mechanisms and clinical relevance of this associa-\\ntion require further rigorous investigation. Conversely,\\nthe development of targeted therapeutic approaches,\\nsuch as employing butyrate derivatives to inhibit bacte-\\nrial proliferation and adhesion, utilizing engineered\\nbacterial outer membrane vesicles to specifically target\\nF. nucleatum, or leveraging its presence to implement\\nimmunostimulatory strategies, collectively demonstrates\\nMICROBIOME IN CANCER | 7 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 7, 'page_label': '8'}, page_content='promising therapeutic potential, particularly when inte-\\ngrated with established treatment modalities [165, 171].\\nAssociation of viruses with tumorigenesis\\nViruses contribute to tumor development through both\\ndirect oncogenic mechanisms and modulation of the host\\nimmune microenvironment [ 172–174]. The molecular\\nmechanisms span multiple pathways, including onco-\\ngenic virus‐induced genetic material damage, microbiota\\ndysregulation, and phage‐mediated metabolic network\\nregulation [175–177]. Among these, oncogenic viruses\\nfunction as primary drivers of tumorigenesis, exerting\\ncritical regulatory influences on cancer development.\\nTable 2 summarizes the oncogenic mechanisms of major\\nviral types in different tumors and their clinical signifi-\\ncance, providing a systematic reference for under-\\nstanding virus‐mediated tumorigenesis.\\nCarcinogenic viruses and tumorigenesis\\nOncogenic viruses represent a specialized category of\\nviruses that infect host cells and interfere with cellular\\ngrowth regulatory mechanisms, thereby inducing aber-\\nrant cell proliferation and ultimately promoting tumor\\nformation [191–195]. These viruses can promote tumor-\\nigenesis through multiple mechanisms, including\\ninduction of DNA damage, disruption of DNA damage\\nresponse (DDR) system, expression of oncogenic pro-\\nteins, activating cancer‐related signaling pathways, acti-\\nvation of cell cycle control, and dysregulation of\\napoptosis [196, 197]. Within the spectrum of digestive\\nsystem oncogenic viruses, HBV and HCV represent the\\npredominant pathogens responsible for HCC. These\\nviruses could promote HCC development and progres-\\nsion through multiple mechanisms, including inducing\\nepithelial–mesenchymal transition (EMT), regulating the\\ncell cycle, and inducing chronic inflammatory responses\\nand tissue fibrosis [175, 178, 198]. Epstein–Barr virus\\n(EBV) demonstrates strong associations with B ‐cell\\nlymphoproliferative disorders and nasopharyngeal car-\\ncinoma (NPC) [ 179], while additionally influencing\\ntumor development through modulation of intestinal\\nmicrobiota and their metabolites [199], as well as con-\\ntributing to GC progression [180]. Human papilloma-\\nvirus (HPV) may contribute to cervical cancer (CC)\\npathogenesis by suppressing immune surveillance and\\ninhibiting cytotoxic responses in lymphocytes, thereby\\nsignificantly influencing the TIME [181]. Additionally,\\nhuman immunodeficiency virus (HIV) may contribute\\nto the development and progression of oral and\\nanal cancerous lesions through alteration of regional\\nmicrobiota composition via mechanisms linked to\\ncarcinogenesis [182].\\nPhages and tumorigenesis\\nIn recent years, the role of phages in tumor development\\nhas garnered increasing attention, with their regulatory\\nmechanisms exhibiting distinct characteristics across\\ndifferent tumor types [200–206]. Several studies in CRC\\nhave revealed specific alterations in phage communities.\\nPhages of the familiesSiphoviridae and Myoviridae were\\nsignificantly enriched in the feces of CRC patients, while\\nstreptococcal phages and Vibrio‐inhabiting phage popu-\\nlations were also abnormally increased [ 176, 183 ].\\nAlthough the biological significance of temperate phages\\nremains controversial [176], experiments have shown\\nthat they may indirectly influence CRC progression\\nthrough modulation of gut microbial composition and\\nmetabolic functions [184].\\nDysbiosis in phage communities induces lysis of host\\nbacteria and subsequent alterations in gut microbial abun-\\ndance, resulting in the release of antigenic substances\\nincluding proteins, lipids, and nucleic acids from bacteria.\\nThis process can induce host inflammatory responses and\\ntissue damage, promoting CRC [185]. Notably,Caudovirales\\nphages exhibit dual roles inthe intestinal inflammatory\\nenvironment, both enhancing survival in CRC‐prone ani-\\nmals through inhibition of oncogenic bacterial colonization,\\nwhile paradoxically exacerbating IBD pathology through\\ntheir over‐amplification [186]. Certain phages directly inter-\\nact with cancer cells and modulate the expression levels of\\nproteins involved in carcinogenesis and metastasis, particu-\\nlarly integrins [207]. These findings indicate that phages\\nfunction as both potential drivers of tumorigenesis and novel\\nbioregulatory targets, though their specific mechanisms\\nwarrant comprehensive investigation within the context of\\ncancer‐specific microbial‐immune microenvironments.\\nOther viruses and tumorigenesis\\nBeyond established oncogenic viruses, diverse viral\\nagents contribute to tumorigenesis and progression\\nthrough both direct and indirect mechanisms. In patients\\nwith persistent enterovirus 71 (EV71) infection, chronic\\npresence of EV71 viral antigen in intestinal tissue sig-\\nnificantly correlates with advanced CRC, potentially\\npromoting disease progression through recruitment and\\nstimulation of Th17 cells in the TME [177]. In BC studies,\\n8 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 8, 'page_label': '9'}, page_content='TABLE 2 Oncogenic mechanisms of different virus types in different tumors and their clinical significance.\\nVirus type\\nRepresentative\\nviruses\\nAssociated\\ncancer types Primary carcinogenic mechanisms Clinical significance References\\nCarcinogenic viruses HBV\\nHCV\\nHCC • Induce chronic inflammation and fibrosis;\\n• Regulate cell cycle;\\n• Promote EMT.\\nHBV/HCV vaccination and antiviral therapy\\nsignificantly reduce HCC risk.\\n[175, 178]\\nEBV NPC\\nGC\\nB‐cell lymphoma\\n• Encode oncoproteins (such as LMP1 and\\nEBNA2);\\n• Modulate gut microbiota metabolites.\\nEBV DNA detection for early screening of NPC. [179, 180]\\nHPV CC • Suppress immune surveillance;\\n• Inhibit lymphocyte cytotoxicity;\\n• Regulate TIME.\\nThe HPV vaccine can prevent CC and other\\nrelated cancers.\\n[181]\\nHIV Oral cancer\\nAnal cancer\\n• Alter local microbiota composition;\\n• Chronic immunosuppression promotes\\ncarcinogenesis.\\nHIV‐infected individuals require intensified\\ncancer screening.\\n[182]\\nPhages Siphoviridae\\nMyoviridae\\nCRC • Regulate gut microbiota composition;\\n• Release bacterial antigens to induce\\ninflammation;\\n• Indirectly promote carcinogenic bacterial\\nproliferation.\\nPhage community analysis as a potential early\\ndiagnostic marker for CRC.\\n[176, 183, 184]\\nCaudovirales CRC Bidirectional role:\\n• Inhibit carcinogen colonization (anti‐cancer);\\n• Exacerbate inflammation (pro‐cancer) upon\\novergrowth.\\nPhage therapy requires precise regulation to avoid\\ninflammatory aggravation.\\n[185, 186]\\nOther viruses EV71 Advanced CRC • Recruit Th17 cells to promote TME\\ninflammation;\\n• Persistent viral antigen stimulation drives cell\\nproliferation.\\nEV71‐persistent infection requires enhanced CRC\\nmonitoring.\\n[\\n177]\\nMMTV\\nBLV\\nBC • Genomic integration activates host oncogenes;\\n• Evade immune clearance.\\nMMTV/BLV antibody detection as a potential BC\\nrisk prediction tool.\\n[187]\\nHCMV BC\\nGlioma\\n• Encode oncogenes;\\n• Suppress tumor suppressor proteins.\\nHCMV‐targeted therapy may enhance chemo/\\nradiotherapy sensitivity.\\n[188, 189]\\nHERV‐KO C\\nPC\\nHCC\\n• Retrotransposition induces genomic instability;\\n• Activate pro‐cancer signaling pathways.\\nHERV‐K expression correlates with tumor\\nprognosis.\\n[190]\\nAbbreviations: BLV, bovine leukemia virus; CC, cervical cancer; EBNA2, Epstein–Barr virus nuclear antigen‐2; EBV, Epstein–Barr virus; HBV, Hepatitis B virus; HCV, Hepatitis C virus; HIV, human\\nimmunodeficiency virus; HPV, human papilloma virus; EMT, epithelial‐mesenchymal transition; EV71, enterovirus 71; HCMV, human cytomegalovirus; HERV‐K, human endogenous retrovirus‐K; LMP1, latent\\nmembrane protein1; MMTV, mouse mammary tumor virus; NPC, nasopharyngeal carcinoma; OC, ovarian cancer; Th17, T helper 17; TIME, tumor immune microenvironment.\\nMICROBIOME IN CANCER | 9 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 9, 'page_label': '10'}, page_content='both mouse mammary tumor virus (MMTV) and Bovine\\nleukemia virus (BLV) have been associated with BC\\ndevelopment, with studies suggesting a likely causal rela-\\ntionship [ 187]. Additionally, human cytomegalovirus\\n(HCMV) has been associated with various malignant\\ntumors, including BC, possibly related to potent oncogenes\\nin its genome [188, 189]. Human endogenous retrovirus K\\n(HERV‐K) is highly expressed in various reproductive\\ntumors, including ovarian cancer, and has also been asso-\\nciated with the progression of PC and HCC [190].\\nAssociation of fungi with tumorigenesis\\nFungal communities represent a key component of the\\nhuman microbiome, with their homeostatic imbalances\\nstrongly associated with tumordevelopment. Specific fungal\\nspecies induce inflammatory responses and disrupt immune\\nhomeostasis, consequently promoting tumorigenesis and\\ntumor progression [208–211]. This process manifests pri-\\nmarily through aberrant proliferation of commensal fungi or\\nexcessive colonization by pathogenic fungi [ 212, 213].\\nA ss h o w ni nT a b l e3, distinct tumors exhibit characteristic\\nfungal signatures that correlate significantly with tumori-\\ngenesis, progression, and prognosis, providing critical in-\\nsights into fungal ‐mediated tumor pathogenesis and\\npotential therapeutic interventions.\\nSymbiotic fungi and tumorigenesis\\nThe human body harbors numerous fungi that establish\\nsymbiotic relationships with host tissues, predomi-\\nnantly colonizing mucosal surfaces (including the oral\\ncavity, intestines, and vagina) and skin [223, 224]. These\\nsymbiotic fungi serve critical functions in maintaining\\nmicrobial community equilibrium, regulating metabolic\\nprocesses, and preserving immune system homeostasis\\n[225]. However, ecological dysbiosis within the fungal\\nmicrobiome contributes to both initiation and progres-\\nsion of digestive system malignancies through diverse\\nmechanisms, including modulation of the host immune\\nmicroenvironment and fungal –bacterial interactions\\n[212]. Studies demonstrate that fungal community\\nTABLE 3 Characteristic fungal alteration patterns in tumors and their mechanisms.\\nCancer type\\nCharacteristic fungal\\nalterations Mechanism Clinical relevance References\\nGC Symbiotic fungal imbalance:\\nMalassezia globosa ↑\\nSaccharomyces cerevisiae ↓\\n• Fungal dysbiosis induces chronic\\ninflammation;\\n• Disrupts mucosal barrier function.\\nSalivary/tongue coating fungal\\ncommunities as early screening\\nmarkers for GC.\\n[214, 215]\\nCRC Pathogenic fungal\\nenrichment:\\nCandida albicans ↑\\nSchizosaccharomyces\\npombe ↑\\nMalassezia spp. ↑\\n• Activate Wnt/β‐catenin pathway to\\npromote proliferation;\\n• Secrete pro‐cancer proteins;\\n• Induce cell adhesion gene\\ndysregulation.\\n• Candida abundance correlates\\nwith advanced CRC;\\n• Fungal translocation to blood\\nindicates metastasis risk.\\n[216–218]\\nPC 3000 ‐ fold fungal\\nconcentration ↑:\\n• Complement C3 pathway activation\\npromotes pro‐inflammatory\\nmicroenvironment.\\n• Fungal load correlates with poor\\nprognosis;\\n• Potential biomarker for\\ntherapeutic resistance.\\n[219, 220]\\nMalassezia ↑\\nHCC Opportunistic fungal\\ncolonization:\\nCandida albicans ↑\\nMalassezia furfur ↑\\n• Induce oxidative stress;\\n• Synergize with HBV/HCV to\\npromote liver fibrosis.\\nFungal‐viral co‐infection\\naccelerates HCC progression.\\n[217]\\nBC Tumor microenvironment\\ndysbiosis:\\nCandida albicans infection ↑\\n• Increase Treg cells to suppress\\nimmunity;\\n• Promote angiogenesis.\\nAntifungal therapy may enhance\\nICIs efficacy.\\n[213]\\nMelanoma Saccharomycetales ↑ • T cells recognize melanoma\\nantigens.\\nFungal diversity may correlate\\nwith immunotherapy response.\\n[221]\\nPCa Plasma fungal dysbiosis:\\nSordariomycetes ↑\\n• Promote inflammatory\\nmicroenvironment.\\n• Specific fungi may assist in\\nidentifying PCa\\n[222]\\nAbbreviations: ↑, Increased; ↓, Decreased; ICIs, immune checkpoint inhibitors; IL‐6, Interleukin‐6; PCa, prostate cancer; Treg, regulatory T cells.\\n10 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 10, 'page_label': '11'}, page_content='compositions in saliva and tongue samples from GC\\npatients undergo significant alterations, characterized\\nby marked enrichment of Malassezia globosa (M. glo-\\nbosa) and concurrent depletion of the symbiotic fungus\\nSaccharomyces cerevisiae (S. cerevisiae)[ 214]. Patients\\nwith CRC showed not only significant increases in\\nS. cerevisiae and Malassezia spp.a b u n d a n c e[226, 227]\\nbut also elevated levels of proteins secreted bySchizo-\\nsaccharomyces pombe , including four key proteins\\nstrongly implicated in tumor progression [216]. In PC\\nstudies, fungal concentrat ions in tumor tissues were\\napproximately 3000‐fold higher than in normal tissues,\\nwith Malassezia spp. demonstrating significant enrich-\\nment and capacity to promote pancreatic tumor growth\\nthrough the complement pathway. This was particularly\\npronounced in PC, where genes related to immunity\\nand inflammation exhibited significant upregulation,\\nsuggesting a substantial role forMalassezia [219, 220]i n\\ndisease pathogenesis. Additionally, the modulatory role\\nof intestinal fungal microbiota on the TME extends to\\nother cancer types, with melanoma patients demon-\\nstrating significantly elevated fungal abundance com-\\npared to healthy controls and selective enrichment of\\nSaccharomycetales spp.[ 221].\\nPathogenic fungi and tumorigenesis\\nExtensive research has demonstrated that specific path-\\nogenic fungi promote the development of diverse tumor\\ntypes through multiple oncogenic mechanisms. Addi-\\ntionally, certain commensal fungi transition to patho-\\ngenic phenotypes through interactions with other\\nmicroorganisms, influenced by host immune status and\\ngenetic factors [217]. The fungal microbiome of GC pa-\\ntients exhibits significant dysbiosis compared to healthy\\ncontrols. Candida albicans(C. albicans) may promote GC\\nby increasing intestinal inflammation, with significantly\\nelevated proportions ofMalassezia, Cutaneotrichosporon,\\nand Fusicolla acetilerea, alongside significantly reduced\\nabundance of Penicillium arenicola , Aspergillus mon-\\ntevidensis, and C. glabrata [215, 217, 228]. Patients with\\nCRC exhibit enrichment ofBasidiomycota and Ascomy-\\ncota, with significant increases inC. albicans, Malassezia,\\nand Rhodotorula abundance. C. albicans promotes\\nintestinal epithelial cell proliferation through activation\\nof Wnt signaling pathway, thereby accelerating tumor\\nprogression [218, 226, 227]. ElevatedCandida abundance\\nin advanced CRC patients may lead to dysregulated ex-\\npression of cell adhesion genes and facilitate trans-\\nlocation of fungal components into the bloodstream\\n[217]. The observation thatAspergillus rambellii signifi-\\ncantly enhances cancer cell proliferation in vitro and\\naccelerates tumor growth in vivo [229] further substan-\\ntiates the potential causal relationship between fungal\\ndysbiosis and CRC [217].\\nStudies in HCC have established that aberrant colo-\\nnization by C. albicans and M. furfur promotes tumor\\ndevelopment, while Alternaria alternata contributes to\\nPC progression [217]. Beyond digestive system malig-\\nnancies, C. albicans infection enhances BC growth by\\nexpanding regulatory T cell (Treg) populations within\\nsplenocytes and the TME [213]. Patients with head and\\nneck squamous cell carcinoma (HNSCC), GC, CRC, and\\nlung cancer have significant intestinal fungal dysbiosis,\\ncharacterized by enrichment of opportunistic pathogenic\\nfungi (including Cutaneotrichosporon, Malassezia, and\\nTrichosporon)[ 221]. The diversity of circulating fungal\\nmicrobiome in plasma of PCa patients differs signifi-\\ncantly from healthy controls, with marked Sordar-\\niomycetes enrichment observed in PCa cases with high\\npathological grade of tumor [222].\\nMicrobial metabolites and tumorigenesis\\nMicrobial metabolites produced by intestinal micro-\\nbiota can contribute to tumorigenesis and progression\\nthrough multiple mechanisms, including modulation\\nof host immune responses, regulation of cell prolifer-\\nation signaling pathways, and alteration of intestinal\\nbarrier integrity. Short ‐chain fatty acids (SCFAs),\\nsecondary bile acids, and tryptophan metabolites\\nrepresent key microbial products that exert important\\nregulatory effects on the intestinal microenvironment\\nand host health [ 230–236]. Table 4 summarizes the\\nmechanisms through which these key microbial\\nmetabolites influence diverse tumor types and their\\npotential clinical applications, highlighting recent\\nadvances in microbial metabolism within oncological\\nresearch.\\nShort‐chain fatty acids\\nSCFAs constitute essential intestinal microbial metabo-\\nlites, primarily consisting of acetic acid, propionic acid,\\nand butyric acid. These compounds exhibit bidirectional\\nregulatory effects on digestive system tumor development\\nthrough modulation of host immune response, inflam-\\nmatory processes, and epigenetic modifications [252–255].\\nSCFAs may play a significant regulatory role in CRC,\\nparticularly butyric acid, which modulates tumor cell\\nproliferation, apoptosis, and invasive capacity through\\nreceptor‐mediated signaling, thereby inhibiting intestinal\\ninflammation and carcinogenesis [237]. Additionally, oral\\nMICROBIOME IN CANCER | 11 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 11, 'page_label': '12'}, page_content='TABLE 4 Key microbial metabolites: mechanisms of action in different tumor types and their potential clinical significance.\\nMetabolite type Metabolite\\nAssociated\\ncancer types Mechanism Clinical significance References\\nSCFAs Butyrate CRC • Promotes abnormal proliferation and\\ntransformation of colonic epithelial cells;\\n• Regulates HDAC activity to induce apoptosis in\\ncancer cells.\\nProbiotics (such asC. butyricum) are targeted to\\ndeliver butyric acid or as adjunctive therapy.\\n[237, 238]\\nPropionate BC • Inhibits STAT3 pathway;\\n• Promote the accumulation of ROS;\\n• Activates p38;\\n• Reverses microbiota dysbiosis inducedby\\npsychological stress.\\nSodium propionate combined with chemotherapy\\nenhances efficacy.\\n[239, 240]\\nAnti‐inflammatory\\nSCFAs\\nHCC SCFA ‐producing bacteria (such asFaecalibacterium)\\ndecrease, leading to weakened anti‐inflammatory\\neffects and promoting HCC progression.\\nFecal microbiota transplantation to restore SCFA\\nlevels or inhibit HCC progression.\\n[91]\\nPentanoate\\nButyrate\\nMelanoma\\nPancreatic\\ncancer\\nEnhance the anti‐tumor effects of CTLs and CAR‐T\\ncells.\\nSynergistic cancer immunotherapy. [ 241]\\nSecondary bile\\nacids\\nDeoxycholic acid CRC • Induces DNA oxidative damage and chromosomal\\ninstability;\\n• Promotes proliferation of mutagenic bacteria (such\\nas Bilophila).\\nMonitor deoxycholic acid levels in high‐fat diet\\npatients to prevent adenoma‐to‐carcinoma\\ntransition.\\n[242, 243]\\nLithocholic acid BC • Induces oxidative stress to inhibit EMT. Lithocholic acid analogs as potential therapies for\\nmetastatic BC.\\n[244, 245]\\nTryptophan\\nmetabolites\\nIndole derivatives CRC • Akkermansia muciniphila inhibits AhR/β‐\\ncatenin\\npathway to reduce risk.\\nIndole derivatives (such as indole‐3‐carbinol) as\\nchemopreventive agents.\\n[246–249]\\nKynurenine Pan ‐cancer • Suppresses T‐cell function to promote immune\\nevasion;\\n• Activates IDO1 pathway to drive tumor\\nprogression.\\nIDO1 inhibitors combined with ICIs therapy. [ 250, 251]\\nAbbreviations: AhR, aryl hydrocarbon receptor; CTLs, cytotoxic T lymphocytes; HDAC, histone deacetylase; IDO1, indoleamine 2,3‐dioxygenase‐1; ROS, reactive oxygen species; SCFAs, short‐chain fatty acids.\\n12 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 12, 'page_label': '13'}, page_content='probiotic spores (spores‐dex) specifically colonize CRC\\nlesions and generate anti‐tumor SCFAs throughC. butyr-\\nicum‐fermented dextran, achieving targeted intervention\\nwithin the TME [238]. In HCC patients, the abundance of\\nSCFA‐producing commensal bacteria (such as Faecali-\\nbacterium, Ruminococcus,a n dRuminoclostridium)i ss i g -\\nnificantly reduced, and the decrease in SCFA levels may\\npromote HCC progression through impairment of their\\nanti‐inflammatory effects [91]. Disturbances in bacterial\\nmetabolites, including SCFAs, lipopolysaccharides (LPS),\\nand lipoproteins, resulting from microbiota dysbiosis\\nconstitute established pathogenic mechanisms in PC\\ndevelopment [256]. In BC research, sodium propionate\\n(SP) induces apoptosis and suppresses tumor cell prolif-\\neration through inhibiting the STAT3 signaling pathway,\\nenhancement of reactive oxygen species (ROS) accumu-\\nlation, and activating p38 [239]. Concurrently, psycho-\\nlogical stressors including social isolation may diminish\\nSCFA production through disruption of the brain –\\ngut–microbiome axis homeostasis, thereby potentially\\nincreasing BC risk [240]. Furthermore, SCFAs demon-\\nstrate considerable therapeutic potential for neutrophil\\ndysfunction‐related conditions, including IBD and various\\nmalignancies [257, 258]. As critical mediators at the\\nmicrobiota–tumor interface, imbalances in SCFA levels\\nand functions may contribute to the pathogenesis of\\ndigestive and systemic cancers through diverse molecular\\nmechanisms [259–263].\\nMicrobial‐derived SCFAs modulate the activity and\\nfunction of immune cells, including lymphocytes, mac-\\nrophages, DCs, and neutrophils, thereby influencing the\\nTIME and exerting complex bidirectional effects on\\ntumor progression [264]. Consequently, SCFAs represent\\npotential therapeutic targets for cancer therapy. In vitro\\nexperiments demonstrate that sodium butyrate (NaB)\\ninhibits lung cancer development and progression\\nthrough suppression of cancer cell proliferation, induc-\\ntion of tumor cell apoptosis, and modulation of immune\\nresponses [265]. SCFAs generated through dietary fiber\\nfermentation by gut microbiota enhance host antibody\\nresponses, with their capacity to potentiate B cell\\nimmune function documented in both mice and humans\\n[266]. Dietary fiber ‐derived SCFAs regulate immune\\nsystem hematopoiesis, thereby exerting anti‐neoplastic\\neffects [ 267]. In murine tumor models, SCFAs\\n(pentanoate and butyrate) significantly potentiate the\\nanti‐tumor efficacy of cytotoxic T lymphocytes (CTLs)\\nand chimeric antigen receptor T (CAR‐T) cells, demon-\\nstrating potential for synergistic therapy in cancer\\nimmunotherapy [241]. SCFAs are significantly reduced\\nin CRC and supplementation with probiotics that\\nmetabolize SCFAs can inhibit tumor cell growth. In both\\nmouse CRC models and CRC patients, SCFAs have been\\nobserved to enhance host responses to chemotherapy and\\nimmunotherapy [268]. Therefore, SCFAs are regarded as\\nsubstances with potential for adjuvant cancer treatment.\\nSecondary bile acids\\nSecondary bile acids (including deoxycholic acid [DCA] and\\nlithocholic acid [LCA]) are metabolites generated through\\nmicrobial transformation of primary bile acid in the gut and\\nmay play crucial roles in host immunomodulatory and\\nm e t a b o l i cn e t w o r k sw i t hc o m p l e xr e g u l a t o r ye f f e c t so n\\ndigestive system cancers [269–274]. Studies demonstrate\\nsignificantly elevated secondary bile acid concentrations in\\nthe intestines of patients with CRC [ 242], potentially\\ndriving carcinogenesis through induction of cellular dam-\\nage including membrane disruption, mitochondrial dys-\\nfunction, and genomic mutations [ 275]. High‐fat diet\\n(HFD) promotes tumor‐promoting properties of mouse\\ncolonic mesenchymal stromal cells (MSCs) by increasing\\nprimary/secondary bile acid production [276]. Furthermore,\\nsecondary bile acid‐enriched microenvironments selectively\\npromote the proliferation of bile salt‐resistant microorgan-\\nisms (particularly Bilophila and Desulfovibrio), which\\naccelerate adenoma‐to‐CRC transformation through release\\nof pro‐inflammatory or mutagenic metabolites including\\nhydrogen sulfide (H\\n2S) and secondary bile acids [243].\\nHCC‐related studies show that intestinal bacteria suppress\\nhepatic natural killer T (NKT) cell accumulation, thereby\\nattenuating anti‐tumor immune responses and facilitating\\nhepatic metastasis through mediation of primary ‐to‐\\nsecondary bile acid conversion [277]. Animal experiments\\ndemonstrate that in BC, LCA induces oxidative stress,\\npromotes mesenchymal ‐to‐epithelial transformation,\\ninhibits tumor cell proliferation and metastasis, and\\nsuppresses cancer cell proliferation [244, 245]. Secondary\\nbile acids may exert context‐dependent effects, both onco-\\ngenic and tumor‐suppressive, in digestive and systemic\\nmalignancies through mechanisms including direct geno-\\ntoxicity, immune microenvironment modulation, and\\nmicrobial interactions [278–281].\\nTryptophan metabolites\\nTryptophan metabolites (including indoleacetic acid, ky-\\nnurenine, and tryptamine), as key molecules generated\\nthrough microbial tryptophan metabolism, exert complex\\nregulatory effects on digestive cancer development via\\nmultiple pathways encompassing immunomodulation,\\ninflammatory response regulation, and epigenetic modifi-\\ncations [282–288]. Tryptophan catabolism, involving this\\nessential amino acid critical for protein synthesis, modulates\\nMICROBIOME IN CANCER | 13 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 13, 'page_label': '14'}, page_content='cancer‐related immune responses and tumor suppression\\nmechanisms [250, 289]. Studies demonstrate that dysregu-\\nlated microbiota‐mediated tryptophan metabolism strongly\\ncorrelates with impaired intestinal barrier function, while its\\nmetabolites (including indole and indole‐3‐acetic acid)\\nmodulate colon cancer progression through regulation of\\ncell proliferation, metastasis, and anti‐inflammatory activity\\nvia multiple signaling pathways [246, 247]. Notably, mucin‐\\nproducing Akkermansia muciniphilasignificantly reduces\\nCRC development risk in mice through specific inhibition of\\ntryptophan metabolism‐dependent aryl hydrocarbon recep-\\ntor (AhR)/β‐catenin signaling pathway [248]. Additionally,\\nindole derivatives (such as indole sodium analogs) can\\nmodulate IL‐6 expression in colon tumor cell lines through\\nAhR activation [249]. A comprehensive analysis across five\\ncancer types revealed that alterations in plasma tryptophan\\nand its metabolites (such as GABA and melatonin) in can-\\ncer patients may interfere with immune responses and\\ncontribute to malignant progression [251]. Research further\\nestablished that compositionaldifferences in gut microbiota\\nbetween cancer patients and healthy individuals signifi-\\ncantly alter tryptophan metabolite profiles, suggesting these\\nmetabolites may serve as biomarkers for cancer screening\\nand detection [251].\\nIntestinal barrier function alteration and\\ntumorigenesis\\nDisruption of intestinal barrier function constitutes a critical\\ndriver of tumorigenesis and development [290–293]. Studies\\ndemonstrate that altered intestinal permeability represents a\\ncore feature of impaired intestinal barrier function. Addi-\\ntionally, immune dysfunction serves as a pivotal mediator\\nin intestinal barrier disruption processes [294–298]. Notably,\\ninflammatory response activation constitutes a fundamental\\nmolecular mechanism underlying altered intestinal barrier\\nfunction [299, 300]. Importantly, intestinal barrier disrup-\\ntion resulting from microbiota dysbiosis contributes to\\ncancer development and progression through multiple\\npathways. In summary, intestinal barrier function disrup-\\ntion establishes a critical link between microbiome and\\ntumorigenesis through multiple interconnected processes,\\nincluding enhanced intestinal permeability, immune system\\ndysregulation, and inflammatory cascade activation, a\\ncomplex pathophysiological sequence with molecular\\nmechanisms illustrated in Figure1.\\nIntestinal permeability alteration\\nIncreased intestinal permeability represents a cardinal\\npathological feature of microbe –host interactions in\\ndigestive cancer pathogenesis, operating through multiple\\nmechanisms including facilitation of microbial trans-\\nlocation, induction of inflammatory cascades, and en-\\nhancement of carcinogen exposure [301]. Studies show\\nthat tight junctions (TJ) between colonic epithelial cells\\nfunction cooperatively with the epithelial cells to maintain\\nintestinal barrier stability, and increased TJ permeability\\ncorrelates with CRC development and progression [302].\\nRegular probiotic supplementation potentially prevents\\nCRC through the reduction of intestinal permeability and\\ndiminished carcinogen absorption [303]. Hepatic research\\ndemonstrates that obesity‐induced intestinal hyperperme-\\nability facilitates dysbiosis, promotes chronic inflammatory\\nresponses, and consequently accelerates CRC initiation\\nand progression [ 304, 305 ]. Intestinal inflammation\\ninduces altered intestinal permeability, while compro-\\nmised barrier function permits bacterial translocation,\\nthereby exacerbating malignancy through the establish-\\nment of chronic inflammatory states [306]. In contrast,\\nSCFAs serve a crucial function in maintaining intestinal\\nbarrier integrity. Butyrate protects intestinal epithelial\\ncells, alleviates local inflammatory responses, and en-\\nhances barrier function through stabilization of hypoxia‐\\ninducible factor‐1( H I F‐1) and increasing transcription of\\nClaudin‐1 protein (an integral membrane protein present\\nin epithelial and endothelial cells) [307, 308]. Additionally,\\nNaB activates the GPR109A signaling pathway and while\\nconcurrently downregulating NF‐κB p65 and AKT signal-\\ning cascades, thereby suppressing intestinal inflammation\\nand restoring epithelial barrier function [309]. Modulation\\nof intestinal permeability constitutes a critical interface\\nbetween microbiota dysbiosis and tumorigenesis. Conse-\\nquently, targeted interventions aimed at intestinal barrier\\nrestoration or microbiome manipulation represent prom-\\nising therapeutic strategies for prevention and treatment of\\ngastrointestinal malignancies.\\nImmune dysfunction\\nAs a fundamental pathological consequence of dysregulated\\nmicrobiota–host interactions, immune dysfunction exerts\\nmultifaceted influences on digestive cancer development\\nthrough modulation of immune cell polarization, activation\\nof inflammatory signaling pathways, and orchestration of\\ncross‐organ immune responses [310, 311]. Perturbation of\\nthe gastric micro‐ecosystem, encompassing bacterial, fun-\\ngal, and viral community dysbiosis, induces immune dys-\\nfunction within the gastric mucosa, characterized by\\ndiminished commensal microbes and enhanced abundance\\nor virulence of pathogenic microorganisms, consequently\\npromoting carcinogenesis through sustained inflammatory\\nresponses and immune dysregulation [312]. Concurrently,\\n14 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 14, 'page_label': '15'}, page_content=\"pathogenic bacteria such as F. nucleatum can induce\\nintestinal immune dysfunction, thereby accelerating CRC\\nprogression [157]. Additionally, research has established\\nthat the gut microbiome of lung cancer patients indirectly\\npromotes tumor development through mediation of LPS\\ntransport and Tregs recruitment via the gut–lung axis,\\nmodulating IL ‐1β expression in pulmonary tissues\\nand disrupting metabolic ‐immune homeostasis [ 313].\\nThe altered immune microenvironment and dysregulated\\nimmune responses resulting from microbial dysbiosis\\nfacilitate the TME establishment and maintenance in\\ngastrointestinal malignancies and distant anatomical sites.\\nInflammatory response activation\\nThe bidirectional interplay between chronic inflamma-\\ntion and microbiota dysbiosis exerts sustained influence\\non gastrointestinal cancer initiation and progression\\n[314–316]. Studies demonstrate that IBD, comprising\\nCrohn's disease (CD) and ulcerative colitis (UC), func-\\ntions as a significant precursor condition for CRC\\ndevelopment [316–321]. The pathophysiological mecha-\\nnisms underlying progression from IBD to colitis ‐\\nassociated CRC encompass multiple integrated factors.\\nFactors including immune cell dysfunction, intestinal\\nFIGURE 1 Mechanisms and cancer type‐specific characteristics of tumor‐associated microbiota. Multiple mechanisms induce intestinal\\nbarrier function disruption, involving intestinal permeability alteration, immune dysfunction, and inflammatory response activation.\\nIncreased intestinal permeability impairs intestinal barrier function and triggers microbial translocation, potentially exacerbating\\nmalignancy. SCFAs act as a protectors that maintain intestinal barrier integrity through upregulating transcription of Claudin‐1 protein and\\nactivating the GPR109A signaling to downregulate NF‐κB/AKT inflammatory response signaling. Regarding immune dysfunction, an\\nimbalanced Firmicutes and Bacteroidetes ratio, along with the secretion of pro‐inflammatory factors IL‐6 and TNF‐α, promotes EMT and\\ntumor invasion. Pathogenic bacteria likeF. nucleatum recruit TAMs through CCL20 upregulation. Gut microbes of lung cancer patients\\ncould indirectly promote malignancy deterioration by mediating LPS transport and Tregs recruitment via the gut‐lung axis, while\\nmodulating IL‐1β expression in lung tissues. Furthermore, aberrant gene methylation of VIM, GATA4/5, dysregulated histone\\nmodifications, abnormal linolenic acid metabolism, and inflammatory microenvironment formation collectively contribute to inflammatory\\nresponse activation by gut microbes. AKT, protein kinase B; EMT, epithelial‐mesenchymal transition; IL‐1β, interleukin‐1 beta; IL‐6,\\ninterleukin‐6; LPS, lipopolysaccharide; NF‐κB, nuclear factor‐kappa B; SCFAs, short chain fatty acids; TAMs, tumor‐associated\\nmacrophages; TNF‐α, tumor necrosis factor‐alpha; Tregs, regulatory T cells.\\nMICROBIOME IN CANCER | 15 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 15, 'page_label': '16'}, page_content='epithelial cells alterations, and microbial dysbiosis col-\\nlectively potentiate inflammatory responses and drive\\ncarcinogenic processes [322]. Intestinal microbiota dys-\\nregulation in IBD patients manifests through epigenetic\\nmechanisms (including aberrant gene methylation of\\nVIM, GATA4/5, and dysregulated histone modifications)\\nand metabolic pathway perturbations (particularly dis-\\nrupted linolenic acid metabolism), thereby promoting\\ncarcinogenesis [323, 324]. Probiotics, particularlyLacto-\\nbacillus bulgaricus(L. bulgaricus), significantly attenuate\\ntumor progression in CAC models through the reduction\\nof pro‐inflammatory mediators, including IL‐6 and TNF‐\\nα [325]. Dysregulation of the Firmicutes/Bacteroidetes\\nratio, especially significantly increased Bacteroidetes\\nabundance, promotes EMT and enhances tumor growth,\\ninvasion, and metastasis through the induction of pro‐\\ninflammatory cytokine secretion, including IL ‐6 and\\nTNF‐α [157]. Meanwhile, Lactobacillus plantarum and\\nLactococcus lactisshow dual application potential in IBD\\ntreatment and CRC prevention by modulating the\\nintestinal immune microenvironment [ 326]. Concer-\\nningly, even in patients with primary sclerosing cho-\\nlangitis (PSC) without typical IBD symptoms, elevated\\nintestinal inflammatory factors such asIL‐17A and IFNG,\\nalong with microbiota dysbiosis, may still increase\\nCRC risk [327]. Additionally, anti‐inflammatory diets\\n(especially bioactive substances like flavonoids) exert\\nanti‐tumor effects through multiple mechanisms\\nincluding inflammatory response inhibition, intestinal\\nbarrier integrity maintenance, and immune cell function\\nregulation [328, 329]. These findings indicate the central\\nrole of inflammatory microenvironment and microbiota\\ninteractions in digestive system tumorigenesis, providing\\nan important theoretical basis for developing cross ‐\\ncancer prevention and treatment strategies.\\nINTRATUMORAL MICROBIOME\\nCharacterization of bacteria in tumors and\\nmechanisms of tumorigenesis\\nThe characterization of bacteria within tumors and their\\noncogenic mechanisms reveals a complex multilevel\\nbiological regulatory network. Bacterial communities in\\ndifferent tumor types have unique compositional char-\\nacteristics. The complex interactions between bacteria\\nand the TME, and their involvement in tumor develop-\\nment through multiple signaling pathway regulation\\n[330–332], are outlined in Figure 2. It illustrates the\\ncarcinogenesis process from bacterial community com-\\npositional heterogeneity to microenvironmental interac-\\ntions. This framework provides a systematic approach for\\nunderstanding the multidimensional role of the micro-\\nbiome in tumor development.\\nCharacterization of bacterial community\\ncomposition\\nThe intratumoral microbiome exhibits significant com-\\npositional heterogeneity across different cancer types\\n[333]. For instance, theFirmicutes/Bacteroidetes ratio is\\nhighest in precancerous lesions of gastric cancer (PLGC)\\nmouse models, while GC groups show increased abun-\\ndance of Proteobacteria and Actinobacteria [334].\\nResearch indicates that tumor‐associated microbiota in\\nHCC patients is predominantly composed of Actino-\\nbacteria, Proteobacteria, and Firmicutes, while Bacter-\\noidetes shows higher abundance specifically in Child‐\\nPugh grade B HCC [335]. Studies suggest that an elevated\\nFirmicutes/Bacteroidetes ratio is associated with favorable\\nresponses to nutritional interventions [336]. This micro-\\nbial diversity closely relates to tumor biological char-\\nacteristics and patient prognosis. Multiple studies\\ndemonstrate that bacteria exhibit specific spatial distri-\\nbution patterns within tumor tissues. Gram ‐negative\\nbacteria primarily localize in cancer cells and immune\\ncells, whereas Gram‐positive bacteria are mainly detected\\nin melanoma and preferentially appear in macrophages\\n[26]. In HCC, bacterial DNA predominantly enriches in\\nperitumoral hepatic sinusoidal erythrocytes, whereas\\nwithin the tumor it mainly localizes in the hepatocyte\\ncytoplasm [ 337]. This spatial distribution difference\\ndirectly affects biological functions. For example,F. nu-\\ncleatum in PC promotes cancer cell invasion via auto-\\ncrine/paracrine pathways [159], whereas microbes in BC\\nenhance metastatic colonization of tumor cells by regu-\\nlating cytoskeletal reorganization [338]. Bacterial load\\nand abundance in BC tissues are significantly higher\\nthan in adjacent normal tissues and most other cancer\\nsubtypes. This enrichment is particularly evident for\\nviable bacteria ofFirmicutes, Proteobacteria, and Actino-\\nbacteria, withF. nucleatumshowing specific enrichment\\nin BC and PC [158]. However, Jeongshin A et al. found\\nthat the Firmicutes/Bacteroidetes ratio is threefold lower\\nin BC patients compared with healthy individuals and is\\nconsidered a risk factor for BC [339]. Corynebacteriaceae\\nand Micrococcaceae abundance is significantly higher\\nin non ‐gastrointestinal tumors (such as BC and\\nosteosarcoma), whereas Proteobacteria, Bacteroidetes,\\nand Firmicutes dominate the PC microenvironment\\n[26, 110, 158\\n]. In CRC, microbiota community analysis can\\nhelp determine clinical subtypes and prognostic subgroups.\\nAmong stage II/III cases, tumors characterized by\\nFirmicutes/Bacteroidetescommunities demonstrate better OS\\n16 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 16, 'page_label': '17'}, page_content='FIGURE 2 (See caption on next page).\\nMICROBIOME IN CANCER | 17 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 17, 'page_label': '18'}, page_content='for microsatellite stable tumors compared with other\\nmicrobial subtypes (Fusobacterium/oral pathogens; Es-\\ncherichia/Pseudescherichia/Shigella)[ 340]. In CRC, mice\\nsupplemented with kefir exhibited decreasedFirmicutes/\\nBacteroidetes and Ascomycota/Basidiomycota ratios, with\\nimproved CRC outcomes [341]. For patients with locally\\nadvanced non‐small cell lung cancer (NSCLC) receiving\\nconcurrent chemoradiotherapy, early nutritional inter-\\nvention can improve host nutritional status and reduce\\npost‐treatment adverse reactions. In lung cancer, micro-\\nbiota compositional features exhibit significant diagnos-\\ntic and prognostic value. Compared with normal lung\\ntissues, genera such asPrevotella and Streptococcus show\\nsignificantly higher abundance in lung cancer, while\\ngenera such as Acidovorax and Acinetobacter show\\ndecreasing trends. Particularly,Thermus shows cumula-\\ntive enrichment in patients with advanced (stage IIIB/IV)\\ndisease, and Legionella are significantly enriched in\\nmetastatic cases, suggesting that microbiota composi-\\ntional dynamic changes are closely associated with dis-\\nease progression [ 342, 343 ]. Regarding prognosis,\\nVeillonella disparin bronchoalveolar lavage fluid (BALF)\\nof lung cancer patients shows positive correlation with\\nhigh PD‐L1 expression, whileNeisseria is enriched in the\\nlow PD‐L1 expression group [344].\\nNotably, intratumoral microbial diversity demon-\\nstrates a significant negative correlation with immune\\nmicroenvironment parameters (such as tumor‐infiltrating\\nlymphocytes [TILs] and PD‐L1 expression) and exhibits a\\nclear association with shorter overall patient survival\\n[345]. In normal pancreatic tissues, microbial abundance\\nis low and predominantly in extracellular colonization\\nform, whereasElizabethkingia and Pseudomonas spp.a r e\\nsignificantly elevated, and enrichment of specific genera\\n(such asSaccharopolyspora) improves anti‐tumor immune\\nresponses by enhancing CD8\\n+ T cell infiltration in PC\\n[110, 152, 346]. Together, these findings reveal the critical\\nrole of intratumoral microbiota in spatiotemporal hetero-\\ngeneity, immune regulation, and clinical translation,\\nproviding new research directions and ideas for precision\\noncology.\\nBacterial and tumor microenvironment\\ninteractions\\nThe origin of tumor microbiota remains controversial,\\nand its distributional characteristics likely vary\\ndepending on tumor type. Studies have demonstrated\\nthat gut microbes can reach tumors via hematogenous\\ntransfer, and oral microbes that colonize tumors\\nthrough the circulatory system (such as F. nucleatum\\nfrom oral cavity to CRCs) constitute important tumor\\nmicrobiota sources [347, 348]. Microbiota from adjacent\\nhealthy tissues also represents an important source.\\nFor example, one study showed that lung tumor\\nmicrobes originated from the airways [349]. Addition-\\nally, bacteria in PC can migrate retrogradely through\\nthe pancreatic duct [350].\\nIntratumoral microbiota forms a complex interaction\\nnetwork with the TME through immune regulation [32,\\n351–354]. The intratumoral microbiota exhibits a clear\\ndouble‐edged sword effect. Regarding pro‐tumor responses,\\nF. nucleatum is closely associated with M2 macrophage\\nactivation in a specific tumor type;Porphyromonas gingi-\\nvalis (P. gingivalis) attracts tumor‐associated neutrophils by\\nsecreting neutrophil chemokines and stimulates their\\nrelease of pro ‐tumorigenic neutrophil elastase [ 158].\\nAdditionally, Bifidobacterium pseudolongum (B. pseudo-\\nlongum)s u p p r e s s e sT‐cell immunity, whereas microbes\\nsuch as Pseudoxanthomonas show a significant positive\\ncorrelation with CD8\\n+ Tc e l li n f i l t r a t i o n[158].\\nFIGURE 2 Composition and spatial heterogeneity, invasion routes, and immune interactions of bacterial communities in different\\ntumor types. Prevotella and Streptococcus show significantly higher abundance in lung cancer, whileAcidovorax and Acinetobacter show\\ndecreasing trends.Veillonella disparshows positive correlation with high PD‐L1 expression in lung cancer, whileNeisseria predominates in\\npatients with low PD‐L1 expression. In HCC,Actinobacteria, Proteobacteria, and Firmicutes show abundance withBacteroidetes higher in\\nChild‐Pugh B classification liver tissue, and bacterial DNA specifically accumulates within peritumoral hepatic sinusoidal erythrocytes.\\nBreast cancer exhibits significant enrichment ofFirmicutes, Proteobacteria, and Actinobacteria, which regulate cytoskeletal reorganization\\nand subsequent tumor metastasis. In PC,F. nucleatum promotes malignant cell invasion via autocrine/paracrine pathways. Additionally,\\nelevated levels ofElizabethkingia, Pseudomonas spp., and specificSaccharopolyspora enhance anti‐tumor immunity through increased CD8+\\nT cell infiltration. Regarding bacterial colonization routes, colorectal cancer microbiota originates from both oral bacteria via circulation and\\ngut bacteria through hematogenous transfer. Lung tumor microbiota derives from airway microbes, while PC‐associated bacteria utilize\\nretrograde migration through the pancreatic duct to trigger metastasis. The intratumoral microbiota exhibits complex cross‐talk with the\\nimmune microenvironment. Regarding pro‐tumor responses, F. nucleatum associates with M2 macrophage activation, whileP. gingivalis\\ntriggers tumor‐associated neutrophil recruitment through chemokines and elastase secretion. Regarding anti‐tumor immunity,\\nB. pseudolongum impairs CD8+ T cell function, whereas likePseudoxanthomonas positively correlate with CD8+ T cell infiltration.\\nB. pseudolongum, Bifidobacterium pseudolongum; HCC, hepatocellular carcinoma; PD‐L1, programmed death‐ligand 1.\\n18 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 18, 'page_label': '19'}, page_content=\"Mechanisms of viral infection and\\ntumorigenesis in tumors\\nInvestigations of tumor‐associated viral infection pro-\\ncesses and their oncogenic mechanisms provide critical\\ninsights into the complex interactions between micro-\\norganisms and their hosts [ 355–357]. Current studies\\nhave identified numerous viruses with established\\noncogenic potential, including HBV, HCV, EBV, HPV,\\nKaposi's sarcoma‐associated herpesvirus (KSHV), human\\nT‐cell leukemia virus type 1 (HTLV‐1), and Merkel cell\\npolyposis virus (MCPyV), among others. Notably, initial\\noncogenic virus infection in humans typically does not\\ndirectly lead to tumorigenesis. However, under condi-\\ntions of immunosuppression or in synergy with other\\noncogenic factors, these viruses can promote malignant\\ncell transformation through multiple mechanisms.\\nFigure 3 comprehensively illustrates the molecular\\nmechanism network and interactions among major\\noncogenic viruses through three core mechanisms:\\ninfection pattern establishment, genome modification,\\nand immune escape to induce tumorigenesis.\\nVirus infection patterns\\nVirus‐mediated tumorigenesis is intricately linked to the\\nmode of viral infection. The mode involves a systematic\\nbiological process comprising host cell invasion, establish-\\nment of persistent infection, and induction of malignant\\ntransformation. The core mechanisms underlying this pro-\\ncess reside in the molecular interactions between viruses\\nand host cells, as well as the dynamic regulation of viral\\nlatency. For example, high‐risk HPV types (especially types\\n16 and 18) integrate their viral genome into host cell DNA\\nto induce CC transformation [358]. The oncogenic proteins\\nencoded by these viruses, E6/E7, drive the development of\\nepithelial tumors such as CC through the specific degra-\\ndation of tumor suppressors, notably p53 and RB. In the\\nB‐lymphocyte transformation paradigm, viral infection\\npromotes B‐cell immortalization [359], enables evasion of\\napoptotic regulation, and activates proliferative signaling\\npathways, ultimately leading to hematological malignancies\\nlike lymphomas. HBV infectionspecifically targets hepato-\\ncytes, wherein the virus completes DNA replication via\\nreverse transcription and subsequently integrates into the\\nhost genome [360]. Hepatocytes harboring these integrated\\nviral fragments acquire proliferative advantages which, in\\nconcert with a chronic inflammatory microenvironment,\\nsynergistically promote HCC development. In T‐cell leuke-\\nmia, HTLV‐1 induces clonal expansion and genomic\\ninstability through infection of CD4\\n+ T cells and subsequent\\nintegration of its viral genome into host cellular DNA. The\\nviral Tax protein of HTLV‐1 constitutively activates the\\nNF‐κB pathway, ultimately triggering adult T‐cell leukemia/\\nlymphoma [361, 362]. MCPyV initiates infection through\\ncutaneous transmission, followed by viral DNA integration\\ninto the host genome, resulting in expression of the large T\\n(LT) antigen that interacts with tumor suppressor p53\\nthrough its C‐terminal domain. Truncation of this replica-\\ntive domain disrupts anti‐tumor surveillance, resulting in\\ndysregulated cell cycle control and subsequent Merkel cell\\ncarcinoma (MCC) pathogenesis [363]. Although these\\nviruses display tropism for distinct target cell populations,\\nthey employ convergent mechanisms to achieve malignant\\ntransformation, including genomic integration, expression\\nof viral oncoproteins, and subversion of host signaling\\npathways, underscoring the fundamental relationship\\nbetween viral infection dynamics and carcinogenesis.\\nVirus‐mediated genome modifications\\nVirus‐mediated genome modification, as an important\\nmolecular mechanism of tumorigenesis, primarily inter-\\nferes with host cell homeostasis through two pathways:\\nviral genetic material integration and epigenetic regulation,\\nthus promoting abnormal cell proliferation. Among DNA\\nviruses, HPV achieves malignant transformation through\\nintegration into host chromosomal DNA, whereby its viral\\noncoproteins E5, E6, and E7 cooperatively promote carci-\\nnogenesis through enhancing aberrant EGFR signaling,\\ncausing p53 and RB dysfunction, deregulating microRNA\\nexpression, and inhibiting exogenous apoptosis [364]. EBV,\\nwith its unique epigenetic regulation in gastric carcino-\\ngenesis, including hypermethylation features of EBV ‐\\nassociated GC (EBVaGC), induces aberrant methylation\\npatterns in gastric epithelial cells. This effect silences tumor\\nsuppressor genes and cell cycle regulators while concur-\\nrently repressing differentiation‐associated transcriptional\\nprograms, ultimately promoting a poorly differentiated,\\nhighly proliferative precancerous cellular state [365]. The\\noncogenic mechanism of HBV closely relates to its genome\\nintegration. During early clonal amplification stages, HBV\\nDNA integration into host genome triggers insertional\\nmutations and drives genome instability which, synergisti-\\ncally with HBV‐encoded X protein (HBx)‐mediated effects\\n(including impairment of DNA damage repair mechanisms\\nand inactivation of tumor suppressors), collectively accel-\\nerates HCC progression [366–370]. RNA viruses lack direct\\ngenomic integrative capacitybut nevertheless compromise\\nhost genome stability through alternative molecular\\nmechanisms. HCV induces genetic and epigenetic disorders\\nthrough its viral protein repertoire, facilitating both onco-\\ngene activation and tumor suppressor silencing. Mean-\\nwhile, HCV simultaneousl y establishes a chronic\\nMICROBIOME IN CANCER | 19 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 19, 'page_label': '20'}, page_content='inflammatory microenvironment through its persistent\\ninfection that constitutes an important factor for HCC\\ndevelopment [371, 372]. In summary, whether mediated\\nthrough direct genomic integration by DNA viruses or\\nepigenetic reprogramming by RNA viruses, virus‐mediated\\ngenome modification ultimately promotes tumor invasion\\nand metastasis through common pathways such as meta-\\nbolic abnormalities, cell cycle dysregulation, and genetic\\ninstability.\\nVirus‐induced immune escape\\nVirus‐mediated immune escape mechanisms represent\\ncritical determinants of oncogenesis, wherein diverse viral\\npathogens establish persistent infections and induce\\ntumorigenesis by specifically targeting distinct components\\nof immune surveillance networks [373]. HPVs evade host\\nimmune clearance mainly through strategies of spatial\\nimmune escape. Their viral gene expression and viral\\n(B)(A)\\n(C) (D)\\n(E) (F)\\nFIGURE 3 Mechanisms of viral oncogenesis in the TME. The schematic illustration demonstrates how different oncogenic viruses\\nmodulate the TME to promote carcinogenesis. (A) HPV induces oncogenesis through multiple mechanisms, including p53/RB inhibition,\\nenhanced EGFR signaling, suppression of miRNA expression, and regulation of exogenous apoptosis. (B) EBV promotes tumor development\\nvia B cell immortalization, virus‐related miRNA regulation inhibiting antigen presentation, and ECs methylation altering the stability of ECs.\\n(C) HBV integrates its reverse DNA into the host genome, triggers DNA breaks, induces cytokine imbalance (IL‐10, TGF‐β), and leads to NK\\ncell dysfunction. (D) HTLV‐1 induces CD4\\n+ T cell dysfunction and activates NF‐κB signaling, resulting in enhanced HIV amplification.\\n(E) HCV contributes to oncogenesis through chronic inflammation, activation of oncogenes, inhibition of tumor suppressor gene expression,\\nand upregulation of PD‐1 on T cells, leading to exhausted T cell transformation. (F) MCPyV promotes cell cycle dysregulation through\\nC‐terminus‐mediated large T antigen reduction and p53 down‐regulation. EBV, Epstein‐Barr virus; ECs, epithelial cells; EGFR, epidermal\\ngrowth factor receptor; HBV, Hepatitis B virus; HCV, Hepatitis C virus; HIV, Human immunodeficiency virus; HPV, Human papillomavirus;\\nHTLV‐1, Human T‐cell leukemia virus type 1; IL‐10, interleukin‐10; MCPyV, Merkel cell polyomavirus; miRNA, microRNA; NK, natural\\nkilling; PD‐1, programmed cell death protein 1; Tex, exhausted T cell; TGF‐β, transforming growth factor‐beta; TME, tumor microenvironment.\\n20 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 20, 'page_label': '21'}, page_content='particle production are strictly confined to the most\\nsuperficial epithelial tissue layer, which is inherently\\nsubject to limited immune cell surveillance [374, 375].\\nEBV evades immunological clearance through expression\\nof specialized viral microRNAs that modulate host immune\\nresponses and antigen presentation, effectively circum-\\nventing immune surveillance and facilitating lymphoma-\\ngenesis and other related malignancies [376]. Among\\nHCC‐associated viruses, HBV achieves immune evasion\\nthrough remodeling the TME. Its chronic infection can\\ninduce functional exhaustion of intrahepatic immune cells\\n(particularly NK cells) and establish immunosuppressive\\ncytokine networks (characterized by increased IL ‐10\\nand TGF‐β), thereby promoting malignant hepatocyte clo-\\nnal expansion while attenuating antiviral immune\\nresponses [377]. HCV‐induced persistent chronic inflam-\\nmation and tissue fibrosis accelerate cirrhosis, while con-\\ncurrent chronic inflammatory signaling drives T cell\\nexhaustion and aberrant upregulation of PD‐1, ultimately\\nestablishing immune tolerance and promoting HCC pro-\\ngression [371]. These complex molecular mechanisms\\ndemonstrate how viruses circumvent host immune sur-\\nveillance through multidimensional strategies, including\\nimmune microenvironment restructuring, subversion of\\nimmune signaling networks, and induction of immune\\ntolerance.\\nIntratumoral fungi and tumor progression\\nThe mechanism by which intratumoral fungi influence\\ntumor progression can be systematically analyzed across\\nthree interconnected dimensions: specific distribution\\npatterns of fungal communities within the TME, inter-\\naction mechanisms between fungi and the TIME, and\\nfungal‐mediated metabolic reprogramming of tumor\\ncells. These mechanistic dimensions exhibit substantial\\ninterconnectivity and synergism, collectively orchestrat-\\ning the complex processes underlying tumor develop-\\nment and progression.\\nCharacterization of fungal communities\\nFungal community composition and spatial distribution\\nexhibit significant heterogeneity across diverse tumor\\ntypes. Macrogenome sequencing ‐based analyses have\\ndemonstrated that various fungi, includingC. albicans,\\nMalassezia, and S. cerevisiae, are prevalent across multi-\\nple human malignancies, including BC, melanoma, and\\nPC [378]. Fungal abundance is significantly elevated in\\nall examined tumor tissues compared with corresponding\\nnormal tissues, with fungal species predominantly\\nlocalized within intracellular compartments of tumor\\ncells and tumor ‐associated macrophages [ 158]. Each\\ncancer type shows distinct fungal community signatures:\\nSaccharomycetes abundance is higher in colon cancer,\\nmelanoma shows predominant enrichment ofMalasse-\\nziomycetes [158], and PC tissues contain approximately\\n3000‐fold higher fungal concentrations compared to\\nnormal pancreatic tissues, withMalassezia significantly\\nenriched in both human specimens and murine models\\n[219]. Additionally, Candida is detected in GC tissues,\\nwith its increased abundance correlating with early‐stage\\nGC progression, while Blastomyces demonstrates a sig-\\nnificant association with lung cancer pathogenesis [215].\\nInteraction between fungi and the tumor\\nimmune microenvironment\\nIntratumoral fungal commu nities actively modulate\\ntumor–host interactions through direct regulation of\\nthe TIME [379, 380]. Studies have demonstrated that\\nAlternaria alternata and M. globosa induce cancer cell\\nIL‐33 secretion through activation of Src‐Syk‐CARD9‐\\nNF‐κB signaling cascade, which subsequently pro-\\nmotes the migration of Th2 cells and type 2 innate\\nlymphoid cells (ILC2) into the TME. The subsequent\\nsecretion of pro‐tumorigenic cytokines, including IL ‐\\n4, IL‐5, and IL‐13, by these activated immune cells\\nfurther accelerates tumor growth and progression\\n[381, 382]. These findings establish that intratumoral\\nfungi function as critical pro ‐tumorigenic mediators\\nwithin tumor ‐immune interactions through modula-\\ntion of key signaling pathways and orchestration of\\ncytokine networks [158].\\nRegulation of the tumor\\nmicroenvironment by microbial\\nmetabolites\\nMicrobial metabolites function as multifaceted regulators\\nwithin the TME, exerting diverse biological effects across\\nmultiple cellular compartments. These metabolites can\\nmodulate immune cell phenotype and function, mediating\\nimmune responses in the TME that significantly influence\\ntumor progression trajectories [383–387]. Notably, micro-\\nbial metabolites substantially influence tumor growth,\\ninvasion, and metastatic potential through the regulation\\nof angiogenic processes and vascular remodeling. Fur-\\nthermore, microbial metabolites directly interact with\\ntumor cells and modulate cellular metabolic pathways,\\nunderscoring the complexity and diversity of these com-\\npounds within the TME regulatory network. As shown in\\nMICROBIOME IN CANCER | 21 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 21, 'page_label': '22'}, page_content='Figure 4, microbial metabolites establish an intricate reg-\\nulatory network through modulation of immune cell\\nfunction, orchestration of angiogenic processes, and re-\\nprogramming of tumor cell metabolism, collectively\\nshaping the functional characteristics of the TME and\\nultimately influencing tumor development.\\nMetabolites and immune cells\\nThe TIME is a dynamic microenvironment system com-\\nprising immune cells that infiltrate tumor tissues and\\ntheir secreted cytokines, chemokines, and other bioactive\\nmolecules [388–391]. Evidence indicates that microbial\\nmetabolites can regulate immune cell function and\\nphenotype through various pathways, thereby con-\\ntributing to the TIME remodeling processes. SCFAs are\\nproduced when gut microbes ferment dietary fiber.\\nButyrate, a key member of the SCFA family, exhibits\\nsignificant immunomodulatory effects and accumulates\\nin tumor tissues. In the TIME, butyrate enhances anti‐\\ntumor immune responses through inhibiting HDAC\\nactivity, upregulating CCL4 expression, promoting CD8\\n+\\nT cell activation, and enhancing IL‐12R signaling path-\\nway activity. Additionally, it also reduces Tregs and\\npromotes anti‐PD‐L1 effects, thereby augmenting anti‐\\ntumor efficacy [392, 393]. In a CRC mouse model, in-\\ntratumoral F. nucleatum was found to affect the TIME\\nthrough the metabolite‐sensing receptor FFAR2 [381]. A\\nstudy in melanoma demonstrated that afterLactobacillus\\nreuteri (L.reuteri) colonization of tumor cells, its metab-\\nolite indole 3 aldehyde (I3A) promoted CD8 + T cell\\nactivation and recruitment while elevating interferon\\nγ (IFN‐γ) levels through an AhR signaling‐dependent\\npathway, thereby enhancing anti‐tumor immune responses\\nand immune checkpoint inhibitor (ICI) efficacy [394].\\nThese findings elucidate the multidimensional regulatory\\nrole of microbial metabolites in TIME modulation and\\n(A) (B) (C)\\nFIGURE 4 Regulation mechanisms of the TME by microbial metabolites. This schematic illustrates three core mechanisms (reagulation\\nof immune cell function, activation of angigenic process, and metabolic networks) by which microbial metabolites modulate the tumor\\nmicroenvironment. (A) SCFAs inhibit histone deacetylase, leading to increased CCL4 production and subsequent modulation of CD8+ T cell\\nactivation and Tregs decrease via IL‐12/JAK‐STAT4 signaling.F. nucleatumpromotes Th17 cell recruitment and IL17 production, whileL.\\nreuteri also affecting CD8+ T cell recruitment and activation through indole 3 aldehyde and IFN‐γ pathways mediated by AhR signaling.\\n(B) F. nucleatumactivates Th17 cells, leading to increased IL17 and VEGF expression. In addition,Lactobacillus metabolizes tryptophan to\\n5‐HT, ultimately promoting vascular endothelial cell proliferation and tumor angiogenesis. (C)P. aeruginosa enrichment leads to azurin\\nsecretion, which triggers tumor cells to produce aldolase A. This creates a feedback loop that further promotesP. aeruginosacolonization on\\nthe tumor surface, collectively contributing to tumor cell apoptosis. Furthermore, SCFAs generate ROS, which also contribute to tumor cell\\napoptosis through oxidative stress mechanisms. AhR, aryl hydrocarbon receptor; IFN‐γ, interferonγ; IL12, interleukin‐12; IL17, interleukin‐\\n17; L.reuteri, Lactobacillus reuteri; ROS, reactive oxygen species; Th17, T helper 17; VEGF, vascular endothelial growth factor.\\n22 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 22, 'page_label': '23'}, page_content=\"establish a theoretical foundation for developing metabolic\\nintervention‐based tumor treatment strategies.\\nMetabolites and angiogenesis\\nMicrobes and their metabolites significantly shape the\\nTME through modulation of angiogenic pathways [395].\\nIn a CRC mouse model,F. nucleatumwithin tumor cells\\nstimulates IL‐17 secretion via metabolite receptors, pro-\\nmoting the formation of an inflammatory TME. IL‐17 can\\nact on tumor cells to promote angiogenic factor VEGF\\nsecretion, which induces tumor angiogenesis and accel-\\nerates CRC progression [160, 396]. Lactobacillus spp.i n\\ntumor cells have shown a strong association with immu-\\nnosuppression in PC and the immunosuppressive TME in\\nesophageal squamous cell carcinoma (ESCC), significantly\\ncorrelating with poor patient prognosis [397, 398]. Lacto-\\nbacillus converts tryptophan into an intermediate product,\\nwhich promotes 5‐hydroxytryptamine (5‐HT) synthesis, a\\nmetabolite that stimulates vascular endothelial cell pro-\\nliferation and thus promotes tumor angiogenesis [399,\\n400]. These findings reveal that microbial metabolites\\ninfluence tumor evolution by regulating tumor vascular\\nhomeostasis.\\nMetabolites and tumor metabolism\\nMicrobial metabolites regulate TME homeostasis\\nthrough comprehensive remodeling of tumor metabolic\\nnetworks [268, 401, 402]. Microbial metabolites accu-\\nmulate within tumor tissues and modulate tumor cell\\nmetabolic processes through specific interactions with\\ncancer cell surface receptors, thus establishing a distinct\\n“tumor‐microbial microenvironment.” Butyrate directly\\naffects tumor cell metabolic processes independently of\\nits established effects on the immune microenvironment.\\nStudies have demonstrated that butyrate induces lipid\\nperoxidation and promotes ROS generation in HT ‐29\\ncolon cancer cells, consequently promoting tumor cell\\napoptosis [403]. Recent investigations have elucidated\\nthe bidirectional signaling mechanisms between in-\\ntratumoral microbes and cancer cells. Azurin protein\\nsecreted by P. aeruginosa disrupts cancer cell signaling\\npathways through binding to specific receptor molecules\\non cellular surfaces, thereby inducing apoptotic cascades.\\nConversely, cancer cells facilitate P. aeruginosa coloni-\\nzation within tumor tissues through aldolase A‐mediated\\nmetabolic processes [404]. These findings establish that\\nmicrobial metabolites significantly influence tumor\\nprogression and development through direct modulation\\nof tumor cell metabolic pathways.\\nORGAN ‐ SPECIFIC MICROBIOME\\nAND TUMORS\\nOrgan‐specific microbiomes function as critical de-\\nterminants in tumorigenesis and development. Studies\\nshow that distinct microbial communities at different\\nanatomical sites exhibit specific association patterns with\\nneoplastic transformation in corresponding tissues\\n[405–412]. Microbial dysbiosis at multiple anatomical\\nsites, including the oral cavity, skin, genitourinary sys-\\ntem, and respiratory tract, promotes tumor progression\\nthrough multiple pathways, including induction of\\ninflammatory responses, production of oncogenic\\nmetabolites, and activation of immune escape pathways.\\nThe oncogenic role of specific pathogenic microorgan-\\nisms, particularly HPV in cervical and oral cancers, has\\nbeen extensively documented, while the broader micro-\\nbiome significantly influences patient responses to\\nimmunotherapeutic and chemotherapeutic interven-\\ntions. Table 5 summarizes the correlated characteristics\\nbetween major organ‐specific microbiomes and corre-\\nsponding malignancies.\\nOral microbiome and oral cancer\\nCurrent evidence demonstrates a complex and mecha-\\nnistic relationship between the oral microbiome compo-\\nsition and oral cancer pathogenesis. Dysbiosis of the oral\\nmicrobiota represents a significant risk factor for oral\\ncancer development, wherein the aberrant proliferation\\nof specific bacterial communities promotes malignant\\ntransformation and progression through inflammatory\\nresponse activation and dysregulation of critical cellular\\nsignaling pathways. Additionally, HPV infection is sig-\\nnificantly associated with oral squamous cell carcinoma\\n(OSCC) development, while specific patterns of oral\\nfungal colonization strongly predict clinical outcomes in\\noral cancer patients. Metabolites produced by oral mi-\\ncroorganisms significantly reshape the TME, thereby\\nmodulating cancer cell proliferation and metastatic\\npotential and highlighting the microbiome's multifaceted\\nregulatory role in oral carcinogenesis [438, 439].\\nAssociation of oral microbial imbalance with\\noral tumorigenesis\\nNumerous studies confirm the close association between\\noral microbiota dysbiosis and oral cancer development.\\nDysregulation of microbiome is now recognized as an\\nimportant hallmark of carcinogenesis. The oral micro-\\nbiota (including bacteria, fungi, archaea, and viruses)\\nMICROBIOME IN CANCER | 23 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 23, 'page_label': '24'}, page_content='maintains ecological homeostasis through dynamic in-\\nteractions with the host microenvironment, and its dys-\\nregulation can promote carcinogenesis by disrupting\\nthese host –microbiota interactions [ 440]. Pathogenic\\nbacteria such asP. gingivalis, F. nucleatum, C. albicans,\\nStreptococcus anginosus , and Selenomonas spp . show\\nsignificant enrichment in cancerous tissues compared to\\nhealthy oral mucosa [413, 433, 441]. Further studies have\\ndemonstrated that the abundance of Prevotella, Chla-\\nmydia, and Firmicutes in the salivary microbiomes of\\nOSCC patients differs significantly from that of healthy\\ncontrols [442]. Evidence indicates thatC. albicans dem-\\nonstrates significant oncogenic potential in OSCC, con-\\ntributing to tumor progression mainly through multiple\\nmechanisms, including epithelial damage, production of\\noncogenic factors, and regulation of the TME [ 413].\\nNotably, oral microecological dysbiosis induced by betel\\nnut chewing appears to accelerate carcinogenesis, with\\nthe abnormal proliferation ofF. nucleatum, P. gingivalis,\\nPrevotella melaninogenica, and other microbes in the\\nsaliva of cancer patients being mechanistically linked to\\ntheir carcinogenic potential [ 414]. Collectively, these\\nfindings elucidate how oral microbial dysbiosis con-\\ntributes to oral cancer development through diverse\\nmolecular mechanisms.\\nSeveral systematic reviews and meta‐analyses have\\ninvestigated the relationship between the oral micro-\\nbiome composition and oral cancer development.\\nTABLE 5 Characteristics associated with organ‐specific microbiome and corresponding tumor.\\nOrgan system Key microbes\\nAssociated\\ncancer types Core mechanisms References\\nOral Porphyromonas gingivalis\\nFusobacterium nucleatum\\nStreptococcus\\nC. albicans\\nHPV‐16/18\\nOSCC • Chronic inflammation (NF‐κB activation);\\n• Carcinogenic metabolites (such as acetaldehyde)\\nand butyrate;\\n• HPV‐mediated inhibition of DNA repair and\\ninduction of genomic instability.\\n[413–418]\\nSkin Staphylococcus aureus\\nβ‐HPV\\nStaphylococcus epidermidis\\nSCC\\nMelanoma\\n• Release of virulence factors and pro‐proliferative\\ncytokines;\\n• Secretion of 6‐N‐hydroxylaminopurine to inhibit\\ntumor DNA synthesis.\\n[419–422]\\nUrogenital HPV\\nVeillonella\\nBifidobacterium\\nLactobacillus\\nCC\\nBCa\\nPCa\\n• HPV integration causing genomic instability;\\n• Microbiome dysbiosis activating;\\nIGF‐1/TLR pathways;\\n• SCFAs modulating macrophage polarization;\\n• Upregulation of Toll‐like receptor\\nexpression,inducing chronic inflammation.\\n[423–426]\\nRespiratory Streptococcus\\nVeillonella\\nC. Pneumoniae\\nCyanobacteria\\nSCFA‐producing bacteria\\nNSCLC • ERK/PI3K pathway activation promoting\\nproliferation;\\n• Immune‐suppressive microenvironment;\\n• Metabolic metabolites inducing apoptosis;\\n• Decreased CD36;\\n• Increased PARP1;\\n•\\nActivate CD4+ T cells, inhibit CD8+ T cells and\\nNK cells, and induce IFN‐γ‐R1 in NK cells.\\n[265, 427–432]\\nBreast Proteobacteria\\nMethylobacterium\\nradiotolerans\\nHPV\\nBC • TLR pathway dysregulation;\\n• Induced DNA damage.\\n[433–436]\\nBrain Fusobacterium\\nLimosilactobacillus\\nLactobacillus\\nGlioma • Fusobacterium nucleatum promotes tumor\\nproliferation;\\n• Microbial metabolites regulating Neuronal genes.\\n[80]\\nBone Marrow SCFA ‐producing bacteria\\nKlebsiella\\nMultiple\\nmyeloma\\n• SCFAs inhibit NF‐κB‐mediated inflammation;\\n• Nitrogen‐cycling microbiota promoting immune‐\\nsuppressive microenvironment.\\n[437]\\nAbbreviations: CC, cervical cancer; IGF‐1, insulin‐like growth factor 1; NSCLC, non‐small cell lung cancer; OSCC, oral squamous cell carcinoma; PARP1, poly\\n[ADP‐ribose] polymerase 1; PI3K, phosphatidylin‐ositol‐3‐kinase; SCC, squamous cell carcinoma.\\n24 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 24, 'page_label': '25'}, page_content='Han et al. observed significantly increased α and β\\ndiversity in the salivary microbiota of OSCC patients\\ncompared to healthy individuals [443]. The oral micro-\\nbiota in OSCC patients was predominantly characterized\\nby Streptococcus, Lactobacillus, and Prevotella [443]. A\\nrecent meta‐analysis summarized the characteristics of\\noral microbiome across different developmental stages of\\nOSCC, revealing variations in microbial diversity, spe-\\ncies, and abundance among different samples [ 444].\\nAnother meta‐analysis indicated that oral microbiome\\ncomposition is associated with cancer patient prognosis\\n[445]. For instance, patients with lower oral diversity,\\nincreased Fusobacterium abundance, or P. gingivalis col-\\nonization were associated with compromised OS [445].\\nAdditionally, established risk factors, including dietary\\npatterns and tobacco exposure, significantly modulate\\nmicrobiome composition and cancer progression. Cohort\\nstudies have demonstrated that increased carbohydrate\\nintake is associated with increased oral bacterial diver-\\nsity, enrichment ofFusobacterium and Leptotrichia, and\\ndepletion of Actinomyces operational taxonomy units\\n(OTUs) [446]. Conversely, dietary patterns characterized\\nby high sugar, processed foods, and alcohol accelerate\\noral carcinogenesis, whereas diets rich in fruits, vege-\\ntables, and fish demonstrate chemopreventive properties\\nagainst cancer development [447]. In tobacco users, ir-\\nrespective of conventional combustible cigarettes or\\nelectronic nicotine, significant alterations and abundance\\nprofiles in oral microbiome composition have been\\nindicated compared to non‐smoking controls [448]. Pre-\\nvious research has indicated that approximately 74% of\\nOSCC cases are attributed to tobacco exposure and\\nchronic heavy alcohol consumption [449, 450]. Further-\\nmore, in OSCC pathogenesis, tobacco‐derived carcino-\\ngens potentially interact with EBV infection [450].\\nThe composition and functional characteristics of the\\noral microbiome are substantially influenced by modifi-\\nable factors, including dietary patterns and smoking\\nbehaviors, which subsequently modulate susceptibility to\\nmultiple malignancies. Zhang et al. indicate that\\ninadequate oral hygiene practices, including irregular\\ntooth brushing and non‐use of dental floss, facilitate dys-\\nbiotic microbial overgrowth, and significantly elevate oral\\ncarcinoma risk [451]. Another study has established as-\\nsociations between compromised oral hygiene status and\\nelevated risk of NPC, with the oral microbeLeptotrichia\\nwadei as a potential mediator connecting quantitative oral\\nhygiene assessments with NPC pathogenesis [452]. Addi-\\ntionally, a tooth brushing frequency of once daily or less\\n(compared with twice or more daily) is independently\\nassociated with a 1.8‐fold elevated risk of ESCC. In-\\ndividuals with ESCC exhibit significantly reduced oral\\nbacterial diversity compared to healthy individuals [453].\\nMechanisms by specific bacteria promote oral\\ncancer progression\\nOral microbiota contributes to oral cancer development\\nthrough multiple molecular pathways. Substantial evi-\\ndence demonstrates that oral microbes, including C.\\nalbicans and S. aureus, are significantly associated with\\nincreased oral cancer risk. Their oncogenic mechanisms\\ninclude disruption of oral microenvironment homeosta-\\nsis and upregulation of critical oncogenes, such as\\nPI3KCA, hRAS, mTOR, and BRAF, which can accelerate\\nmalignant transformation [40]. Comprehensive micro-\\nbiome analyses have established that multiple oral mi-\\ncroorganisms, including P. gingivalis, F. nucleatum, and\\nStreptococcus spp., are causally linked to oral carcino-\\ngenesis through the induction of chronic inflammatory\\nresponses, regulation of cell proliferation, NF‐κB signal-\\ning pathway activation, and production of carcinogenic\\ncompounds [415, 454]. Microbial components induceβ‐\\ndefensin 2 expression and promote the progression of\\nprecancerous lesions through stimulation of mast cell\\ndegranulation and establishment of persistent inflam-\\nmatory responses, a process directly implicated in the\\npathogenesis of tongue squamous cell carcinoma [455].\\nNotably, specific oral microbes may diminish chemo-\\ntherapeutic agent efficacy and exacerbate treatment ‐\\nrelated complications [456], suggesting the importance of\\nmicrobial modulation in oncology treatment strategies.\\nCollectively, these findings delineate a complex molecu-\\nlar network wherein oral microbiota drives oral carci-\\nnogenesis through interconnected mechanisms\\nencompassing transcriptional regulation, inflammatory\\npathway activation, and comprehensive remodeling of\\nthe immune microenvironment.\\nHPV virus infection and the development of\\noral squamous cell carcinoma\\nHPV functions as a critical driver in OSCC progression.\\nHigh‐risk HPV genotypes (particularly types 16 and 18)\\ndrive oncogenesis by interfering with the DDR system,\\nconsequently suppressing tumor suppressor gene func-\\ntion and inducing genomic instability [457–460]. Addi-\\ntionally, viral components of the oral microbiome,\\nprincipally HPV, contribute to oral mucosal dysbiosis,\\nprecancerous lesions, and OSCC progression through\\nactivation of inflammatory pathways and dysregulation\\nof cancer stem cell‐associated signaling networks [416].\\nHPV interacts synergistically with diverse microorganisms,\\nincluding EBV, herpes simplex viruses, HIV, and multiple\\nbacterial species. This interaction promotes persistent\\nviral infection, accelerating cancer development [457].\\nMICROBIOME IN CANCER | 25 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 25, 'page_label': '26'}, page_content='These mechanisms reveal that HPV both disrupts cellular\\nregulatory processes through genetic material interference\\nand promotes OSCC transformation through remodeling\\nof the local microenvironment, thereby revealing novel\\ntherapeutic targets for improved clinical diagnosis and\\ntreatment strategies.\\nRelationship between fungal colonization and\\nprognosis of oral cancer\\nRecent studies have confirmed significant correlations\\nbetween patterns of fungal colonization within the oral\\necosystem and clinical outcomes in oral cancer patients\\n[461, 462]. Clinical investigations have demonstrated the\\npresence of complex polymicrobial communities, includ-\\ning Chlamydia and diverse fungal taxa in salivary samples\\nfrom OSCC patients [442]. As a representative pathogenic\\nfungus, C. albicans promotes oral carcinogenesis and\\nprogression through multiple molecular mechanisms. Its\\ncell wall components (especiallyβ‐glucan) induce Th17\\ncell‐mediated immune responses and activate the NF‐κB\\nand Wnt signaling pathways, consequently fostering a pro‐\\ncarcinogenic microenvironment formation and enhancing\\ntumor cell resistance to apoptosis [417, 463, 464]. Addi-\\ntionally, C. albicans and yeast upregulate expression of\\npro‐inflammatory cytokines, including IL ‐6a n dI L‐8,\\nthereby accelerating oral cancer progression [418]. C.\\nalbicans also upregulates PD‐L1 expression to facilitate\\ntumor immune escape, while simultaneously accelerating\\nHNSCC progression through modulation of acetaldehyde\\nmetabolism processes [465]. These findings suggest that\\nalterations in fungal community composition may serve as\\nvaluable diagnostic and prognostic biomarkers for oral\\nmalignancies.\\nImpact of oral microbial metabolites on the oral\\ncancer microenvironment\\nMetabolites derived from oral microorganisms orchestrate\\nthe dynamic evolution of the oral cancer microenvironment\\nthrough complex and interconnected signaling networks.\\nDisruption of oral microbial communities induces aberrant\\nmetabolite release, compromisesepithelial barrier integrity,\\nand precipitates immune homeostasis dysregulation. These\\npathological alterations synergistically converge on oral\\nmucosal cells, promoting malignant transformation and\\ndysregulated cellular proliferation [ 466]. Interactions\\nbetween microbial metabolites and the host immune sys-\\ntem constitute a fundamental driver of tumor progression.\\nSpecifically, lactate metabolites attenuate anti ‐tumor\\nimmune responses through recruitment of TAMs, Tregs,\\nand myeloid‐derived suppressor cells [467]. Among micro-\\nbial metabolites, SCFAs are closely associated with OSCC\\ndevelopment. Particularly, NaB produced byP. gingivalis\\npromotes OSCC progression through mechanisms that\\nremain incompletely characterized despite intensive inves-\\ntigation [468, 469]. Microbial–host interactions exhibit\\nmutually reinforcing relationships. Oral microorganisms\\nand their metabolites promote tumor progression, while the\\nTME creates favorable conditions for microbial coloniza-\\ntion. This mutually reinforcing vicious cycle constitutes an\\nimportant pathogenic mechanism in oral cancer progres-\\nsion [470].\\nSkin microbiome and skin cancer\\nThe skin microbiome serves as a critical regulator of\\ncutaneous homeostasis, and disruption of this dynamic\\nbalance is strongly associated with skin carcinogenesis\\nand progression [471]. Accumulating evidence has dem-\\nonstrated that skin microbiome dysregulation promotes\\ncutaneous carcinogenesis through multiple mechanisms,\\nincluding disruption of epidermal barrier integrity,\\ninduction of immune escape, and activation of inflam-\\nmatory cascade responses. UV radiation, a major causative\\nfactor in skin carcinogenesis [472–474], not only directly\\ndamages host cells but also increases the risk of photo-\\ncarcinogenesis by altering the composition and function of\\nskin microbial communities. These multidimensional\\ninteraction mechanisms highlight the central regulatory\\nrole of the skin microbiome in cancer prevention, occur-\\nrence, and treatment, establishing a theoretical foundation\\nfor the development of microbiome‐targeted preventive\\nand therapeutic strategies.\\nRelationship between skin microbial imbalance\\nand skin carcinogenesis\\nDysbiosis of the skin microbial community contributes to\\ncutaneous carcinogenesis through both direct oncogenic\\nor oncostatic effects and indirect immunometabolic reg-\\nulatory mechanisms. Among carcinogenesis ‐associated\\nmicrobes, S. aureus induces carcinogenesis through\\nmultiple virulence factors and cytokine release, promot-\\ning SCC cells proliferation [419]. This pathogen shows a\\nsignificantly elevated relative abundance in SCC patients\\n[475]. Fusobacterium spp., Trueperella spp., and Coryne-\\nbacterium are significantly enriched in melanoma lesions\\n[476, 477]. Notably, Corynebacterium demonstrates a\\nsignificant positive correlation with IL‐17 levels, a critical\\nfactor in melanoma progression [477]. Mendelian ran-\\ndomization analysis has revealed a bidirectional positive\\n26 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 26, 'page_label': '27'}, page_content=\"correlation between class Bacilli and malignant mela-\\nnoma, whereas decreased phylum Bacteroidetes abun-\\ndance may elevate disease risk [ 478]. Pseudomonas\\naeruginosa also promotes skin cancer cell growth [479].\\nAmong protective microorganisms, Staphylococcus epi-\\ndermidis (S. epidermidis ) significantly reduces UV ‐\\ninduced skin cancer incidence by inhibiting tumor cell\\nDNA synthesis through the secretion of 6 ‐N‐\\nhydroxyaminopurine (6 ‐HAP) [ 420]. Similarly, Cuti-\\nbacterium acnes contributes to anti ‐tumor processes\\nthrough anti‐angiogenic effects, and its reduced abun-\\ndance correlates strongly with SCC disease progression\\n[480]. These findings suggest that the skin microbiota\\nbidirectionally regulates skin cancer progression by\\ndirectly affecting cell proliferation and indirectly regu-\\nlating immune metabolism [481], providing new thera-\\npeutic targets for microbial intervention ‐based skin\\ncancer prevention and treatment strategies.\\nRelationship between microbiome alterations\\ndue to ultraviolet exposure and skin cancer\\nUltraviolet exposure promotes both the initiation and\\nprogression of skin cancer. Beyond directly damaging\\nhost cells, ultraviolet radiation can also kill skin micro-\\norganisms and induce dysbiosis of the cutaneous\\nmicrobial community, thereby synergistically promoting\\nskin cancer initiation and progression [ 482–485]. UV\\nradiation (especially the UVC band) exerts broad ‐\\nspectrum bactericidal effects by disrupting microbial\\nnucleic acid structure and enzymatic activity [421, 486],\\nresulting in significant alterations to skin microbiome\\ncomposition. UV irradiation increases the abundance of\\nProteobacteria, impairing anti ‐inflammatory immune\\nresponse functions. Similarly, it decreasesLactobacilla-\\nceae and Pseudomonadaceae, compromising cutaneous\\nbarrier homeostasis. Abnormal Cyanobacteria prolifera-\\ntion may contribute to tumorigenesis, progression, and\\nmetastasis through the release of pro ‐inflammatory\\nmetabolites and upregulation of immunosuppressive\\nfactors [487, 488]. This microbial dysbiosis synergizes\\nwith UV‐induced keratinocyte DNA damage, and the\\ndysregulated microbiota exacerbates oxidative stress and\\nchronic inflammatory states, further destabilizing geno-\\nmic integrity and accelerating non ‐melanoma skin\\ncancer (NMSC) progression [ 489]. Additionally, UV\\nradiation and HPV infection show significant synergistic\\neffects in CSCC, with UV‐induced damage interacting\\nwith HPV oncoproteins through activation of the\\nWnt/β‐catenin signaling pathway, collectively driving\\nmalignant cellular transformation [490]. These mecha-\\nnisms indicate that UV radiation not only directly\\ndamages host cells but also multidimensionally promotes\\ncutaneous carcinogenesis through remodeling the\\nmicrobe–host interaction network.\\nViral infections and the mechanisms of skin\\ncancer development\\nViral infections represent critical etiological factors in the\\npathogenesis and progression of cutaneous malignancies.\\nStudies show β‐HPV exhibits significant tropism for\\ncutaneous tissues and is strongly associated with non‐\\nmelanoma skin cancer development, whereas HPVs of\\nγ, μ, and v genera mainly induce the formation of benign\\ncutaneous lesions [491]. In states of immunosuppression\\n(including verrucous epidermal dysplasia, HIV infection,\\nor long‐term immunosuppressive drug use),β‐HPV sig-\\nnificantly accelerates cutaneous oncogenesis [422, 492].\\nAdditionally, KSHV has been established as a direct\\netiological agent in Kaposi's sarcoma (KS) development,\\nevidenced by the persistence of specific viral DNA\\nsequences within KS lesions and the temporal prece-\\ndence of KSHV infection before tumor formation, further\\ncorroborating its pathogenic mechanism [493]. In studies\\nof MCC, Merkel cell polyomavirus (MCV) DNA is\\ndetectable in approximately 80% of tumor tissues, sug-\\ngesting viral genome integration functions as a potential\\ndriver of clonal tumor expansion [494]. Together, this\\nbody of evidence underscores the pivotal role of onco-\\ngenic viruses in cutaneous carcinogenesis through mul-\\ntiple molecular mechanisms, including viral genome\\nintegration and host immune evasion.\\nMicrobe‐mediated inflammatory response\\npromotes skin cancer progression\\nThe skin microbiota drives the cancer process by acti-\\nvating inflammatory signaling pathways. Pathogenic\\nmicrobes can abnormally activate Toll ‐like receptors\\n(TLRs), triggering chronic inflammatory responses that\\npromote skin cancer development [495, 496]. Animal\\nexperiments show that TLR‐5 signaling pathway activa-\\ntion, through chronic inflammatory responses, plays an\\nimportant role in regulating tumorigenesis in murine\\nskin cancer models [497]. Additionally,S. aureusreleases\\nvarious pro‐inflammatory factors that promote tumor\\ndevelopment‐conducive immune microenvironment\\nformation and induce SCC progression [419]. Conversely,\\nS. epidermidisactivates immune‐inflammatory responses\\nwhile reducing pathogenic bacteria colonization (such as\\nS. aureus) through competitive inhibition, thereby in-\\nhibiting cancer cell proliferation [498]. Aberrations in\\nMICROBIOME IN CANCER | 27 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 27, 'page_label': '28'}, page_content=\"skin microbiota metabolites also influence tumor pro-\\ngression, for example, reduced levels of the tryptophan\\nmetabolite I3A accelerate the malignant transformation\\nof precancerous lesions [499]. These mechanisms suggest\\nthat microbe‐mediated inflammatory networks partici-\\npate in skin tumorigenesis and evolution through\\nmultiple signaling pathways.\\nImpact of the microbiome on melanoma\\nimmunotherapy\\nICI therapy has markedly improved prognosis in melanoma\\npatients; however, substantialheterogeneity exists in indi-\\nvidual treatment responses [500–502]. Several studies have\\nconfirmed that the microbiome plays a pivotal role in\\nmodulating ICI therapeutic efficacy [503, 504]. A clinical\\ninvestigation found that patients receiving antibiotics dur-\\ning cancer treatment, including those with melanoma, ex-\\nhibited significantly reducedresponse rates to ICI therapy\\n[505]. Cutaneous commensal bacteria also show im-\\nmunomodulatory potential. In a murine model, genetically\\nengineered S. epidermidis induced robust T‐cell immune\\nresponses through expression of melanoma‐specific anti-\\ngens and synergistically enhanced ICI efficacy, resulting in\\npotentiated anti‐tumor effects [506]. Notably, synergistic\\neffects between the microbiome and immunotherapy can\\nalso be achieved through alternative strategies, such as the\\ncombination of oncolytic viruses (OVs) and ICIs. OVs\\ninduce both local and systemic immune responses by se-\\nlectively lysing tumor cells and facilitating antigen release,\\nwhile the combination of anti ‐CTLA‐4a n da n t i‐PD‐1\\ntherapy significantly inhibits primary tumor progression\\nand simultaneously promotes regression of metastatic\\nlesions [507]. Collectively, these findings suggest that the\\nskin microbiome not only directly modulates host immune\\nstatus but may potentially remodel the TME through syn-\\nergistic effects with other therapeutic modalities, thereby\\nproviding a multifaceted scientific rationale for optimizing\\nmelanoma immunotherapeutic strategies [508–512].\\nUrinary/genital microbiome and urinary\\ntumors\\nThe genitourinary microbiome exhibits complex inter-\\nactions with urogenital tumorigenesis and progression\\n[513–515]. Studies reveal that dysbiosis of the urogenital\\nmicrobiota represents a significant risk factor for bladder\\ncarcinogenesis, while HPV infection constitutes a key\\ncausative factor in CC development and progression.\\nAdditionally, numerous studies show that specific bac-\\nterial infections are significantly associated with PCa\\nrisk, and the composition and functional profile of the\\nurinary microbiome can markedly influence patients'\\nclinical responses to anti‐tumor therapies. With advanc-\\ning research, microbiome‐derived biomarkers are emer-\\nging as important molecular tools for the early diagnosis\\nand prognostic assessment of urological malignancies.\\nAssociation of urethral microbial imbalance\\nwith bladder cancer\\nRecent studies have progressively elucidated the associ-\\nation between urethral microbial imbalance and BCa\\nthrough microbial characterization. One study showed\\nmicrobial richness (α‐diversity) was significantly lower in\\nBCa patients compared with healthy controls, suggesting\\nthat dysbiosis may contribute to disease progression\\n[516]. Additionally, urinary microbiome β‐diversity in\\nBCa patients differed significantly from that of the\\nrecurrence‐free tumor group, with elevated abundance of\\nVeillonella and Bifidobacterium in the tumor group,\\nwhereas the recurrence‐free group exhibited enrichment\\nof Escherichia‐Shigella and Helococcus [423]. Further\\nstudies identified significant correlations between blad-\\nder carcinogenesis and various microbial taxa, including\\nmembers of the family Rikenellaceae and the genera\\nAllisonella and Senegalimassilia [517]. Microbial com-\\npositional analysis between muscle ‐invasive bladder\\ncancer (MIBC) and non‐muscle‐invasive bladder cancer\\n(NMIBC) revealed that there were no significant changes\\nin the relative abundance of major phyla such asFirmi-\\ncutes and Proteobacteria. However, Flavobacteriales, Eu-\\nbacterium CAG‐581, and Bacteroides sp. 43‐47FAA were\\nenriched in NMIBC and were strongly associated with\\ndecreased patient survival [518]. In summary, structural\\ndysbiosis of the urogenital microbiome may promote BCa\\ndevelopment through regulation of the local micro-\\nenvironment, and these characteristic microbial signa-\\ntures may serve as potential biomarkers, providing new\\navenues for BCa diagnosis and prognostic assessment.\\nAssociation of bacterial infections with prostate\\ncancer risk\\nA substantial body of evidence confirms a significant\\nassociation between bacterial infections and PCa devel-\\nopment. Several studies show that the prostatic micro-\\nbiome of PCa patients displays characteristic alterations,\\nwith significant differences in the relative abundance of\\nspecific bacterial taxa. In a case‐control study of patients\\nwith benign and malignant prostate lesions, researchers\\nfound significant alterations in bacterial communities\\n28 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 28, 'page_label': '29'}, page_content='within high‐grade PCa tissues, with notable enrichment\\nof species including Streptococcus anginosus, Anaero-\\ncoccus lactolyticus, and Actinobaculum schaalii [519].\\nSCFAs produced by intestinal microbial communities\\ncan induce chronic inflammatory responses through\\nmultiple mechanisms: modulation of the IGF‐1 signaling\\npathway, promotion of M2 ‐phenotype macrophage\\npolarization, and upregulation of TLR expression, col-\\nlectively creating a microenvironment conducive to PCa\\nprogression [424]. In summary, urinary tract bacterial\\ninfections promote PCa development through modula-\\ntion of key signaling pathways and induction of chronic\\ninflammatory responses that alter the TME.\\nMechanisms of HPV infection and the\\ndevelopment of cervical cancer\\nThe vast majority of CC cases are attributable to per-\\nsistent HPV infection [520–522]. The mechanisms un-\\nderlying HPV infection and subsequent CC development\\nencompass multifaceted molecular and pathological\\nprocesses, including effects on the cervicovaginal mi-\\ncrobiota, which represents an emerging frontier in\\ngynecologic oncology research. During the progression of\\ncervical lesions from premalignant to malignant states,\\nHPV infection significantly alters the composition and\\nabundance of vaginal microbial communities. Specifi-\\ncally, commensal bacteria such asLactobacillus crispatus\\nand Lactobacillus iners exhibit significant decreases in\\nabundance, whereas potentially pathogenic bacteria\\nincluding Prevotella and Gardnerella demonstrate sig-\\nnificant proliferation [425, 523–526]. Other viruses (such\\nas HIV, EBV, and herpes simplex virus) are also closely\\nassociated with HPV infection, collectively promoting\\nmalignant transformation and tumor progression [527].\\nImportantly, increased cervicovaginal microbiome\\n(CVM) diversity, together with decreased Lactobacillus\\nspp., is significantly associated with elevated risk of\\ntumorigenic intraepithelial lesions resulting from per-\\nsistent HPV infection, ultimately accelerating progres-\\nsion to invasive carcinoma [528]. Notably, HPV infection\\nnot only alters the compositional structure of the vaginal\\nmicrobiota but also collaboratively establishes a tumor‐\\npromoting microenvironment through complex interac-\\ntions with microbial metabolic activities [ 426]. Addi-\\ntionally, HPV maintains its infection status through dual\\nmechanisms: inducing dysbiosis of the CVM and simul-\\ntaneously suppressing the expression of host antimicrobial\\npeptides essential for innate immunity, further exacer-\\nbating vaginal microecological imbalance. HPV‐associated\\nperturbations in the cervicovaginal microenvironment\\nexhibit distinctive metabolomic signatures and\\ninflammatory responses that correlate with the extent of\\nHPV infection, collectively facilitating tumorigenesis and\\ndisease progression [426].\\nImpact of the urinary microbiome on response\\nto treatment\\nDistinct compositions of the urinary tract microbiome\\nexhibit significant heterogeneity in their responses to\\ntherapeutic interventions. A growing body of evidence\\nconfirms that the microbiota plays a pivotal role in renal\\ncell carcinoma (RCC) pathogenesis, progression, and\\ntherapeutic responsiveness, with the underlying mecha-\\nnism mainly realized through immune system function\\nmodulation, host metabolism, and drug response [529].\\nMicrobiome diversity is significantly reduced in RCC\\ntissue specimens compared to adjacent non‐neoplastic\\nrenal parenchyma. The end ‐stage renal disease ‐\\nassociated microbiome demonstrates decreased abun-\\ndance of putatively beneficial bacteria (such as\\nRoseburia) and concurrent enrichment of potentially onco-\\ngenic microbes (such asEscherichia, Fusobacterium,a n d\\nBacteroides) compared to control specimens [530]. Together,\\nthese findings underscore the potential contributory role of\\nurinary microbiome alterations in renal disease patho-\\ngenesis, thereby providing a critical theoretical foundation\\nfor the development of novel microbiome‐targeted thera-\\npeutic strategies in RCC management.\\nMicrobial markers in predicting prognosis of\\nurologic tumors\\nMicrobiome‐derived biomarkers demonstrate significant\\nclinical applications in urologic tumor diagnosis. Numer-\\nous studies confirm dynamic alterations in specific micro-\\nbial communities exhibit significant correlations with PCa\\ndevelopment and prognostic assessment. Theβ‐diversity of\\ncirculating fungal microbiomes in peripheral blood signif-\\nicantly differs between PCa patients and healthy controls,\\nwith particularly pronounceddistinctions in patients with\\nhigh pathological tumor stages (pT3 or pT4). Notably, the\\nabundance of Sordariomycetes is significantly elevated in\\nadvanced PCa and has emerged as a promising biomarker\\nfor identifying high‐grade disease [222]. Additionally, in\\npost‐d i g i t a lr e c t a le x a m i n a t i o nu r i n es a m p l e sa n dp o s t‐\\nprostatectomy prostatic secretions, alterations in the\\nabundance of five strictly anaerobic bacterial genera\\n(including Fenollaria) show significant correlation with\\nPCa aggressiveness and biochemical recurrence risk, sug-\\ngesting their potential utility as non ‐invasive urinary\\ndiagnostic and prognostic biomarkers [531]. These findings\\nMICROBIOME IN CANCER | 29 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 29, 'page_label': '30'}, page_content='suggest that specific microbiome markers may not only\\nparticipate in urologic tumorpathogenesis but also serve as\\npotential diagnostic tools, providing new approaches\\nfor early detection, risk stratification, and therapeutic\\nmonitoring of urologic malignancies.\\nRespiratory microbiome and lung cancer\\nA complex bidirectional relationship exists between the\\nrespiratory microbiome and lung cancer development\\n[532–534]. Emerging evidence demonstrates that respi-\\nratory microbiome dysbiosis is significantly associated\\nwith lung cancer development, involving molecular\\nmechanisms that may contribute to lung cancer pro-\\ngression through interconnected signaling pathways,\\nrevealing the critical role of microbial community\\nalterations in lung cancer pathogenesis. Additionally, the\\nrespiratory microbiome significantly modulates key lung\\ncancer biological behaviors, including proliferation,\\ninvasion, and metastasis, through regulation of the\\nTIME. Notably, respiratory microbiome composition and\\ntemporal dynamics correlate strongly with treatment\\nresponses and clinical outcomes in lung cancer patients,\\noffering promising avenues for the development of\\nmicrobiome‐based precision medicine strategies.\\nAssociation of respiratory microbial imbalance\\nwith lung carcinogenesis\\nSubstantial evidence now establishes that respiratory mi-\\ncrobiota dysbiosis is causally linked to lung cancer patho-\\ngenesis. Clinical investigations confirm that the respiratory\\nmicrobiome composition andcommunity structure in lung\\ncancer patients differ significantly from those of healthy in-\\ndividuals. Lower respiratory microbialα‐diversity is slightly\\nreduced in lung cancer patients [427, 535]. In patients with\\ncentral lung tumors,Streptococcusconstitutes the predomi-\\nnant bronchial microbes at both the lesion site and contra-\\nlateral bronchus, whereasPseudomonaspredominates in the\\nbronchial microbiome of healthy controls [428]. Significant\\ntaxonomic overlap exists between the oral and pulmonary\\nmicrobiomes of lung cancer patients, whose salivary samples\\ndemonstrate significantly altered bacterial abundance\\n(Streptococcus, Lactobacillus, etc.) and fungal composition\\n(Candida, Malassezia, etc.) compared to control individuals\\n[428]. Distinct lung cancer histological subtypes exhibit\\ncharacteristic microbial signatures: lung adenocarcinoma\\n(LUAD) demonstrates predominance of Propionibacter-\\nium, whereas lung squamous cell carcinoma (LUSC)\\nshows enrichment ofEnterobacteriaceae, with significantly\\nhigher microbial diversity observed in LUSC patients\\n[313]. In early ‐stage I/II NSCLC patients, Firmicutes,\\nBacillus,a n d Actinobacteria demonstrate significant\\nenrichment, whereas Actinobacteria predominates in\\npatients with advanced‐stage (III/IV) disease [313]. Res-\\npiratory microbiota dysbiosis drives lung cancer progres-\\nsion through multiple interconnected mechanisms,\\nincluding metabolic reprogramming, inflammatory cas-\\ncade modulation, and establishment of an immuno-\\nsuppressive TME. For example, elevated Veillonella\\nparvula (V. parvula) abundance correlates with reduced\\nCD4\\n+ T cell infiltration and upregulation of PD‐1a n dI L‐\\n17 expression, collectively fostering an immuno-\\nsuppressive TME [429]. These findings suggest respiratory\\nmicrobial dysbiosis functions not only as a critical driver\\nof lung carcinogenesis but also provides novel research\\ndirections for developing innovative diagnostic biomarkers\\nand therapeutic interventions.\\nMolecular mechanisms underlying the\\nassociation between specific bacteria and lung\\ncancer progression\\nCurrent evidence elucidates distinct molecular mecha-\\nnisms connecting specific bacterial taxa to lung cancer\\ninitiation and progression. Specific bacterial genera,\\nincluding Veillonella and Streptococcus, modulate lung\\ncancer cell proliferation and survival through activation\\nof key oncogenic pathways, particularly ERK and PI3K,\\nthereby contributing to lung cancer development and\\nprogression [ 536]. Studies confirm specific bacterial\\ngroups significantly correlate with distinct lung cancer\\nhistopathological subtypes, with Cyanobacteria exhibit-\\ning significant enrichment in LUAD, which correlates\\nwith decreased CD36 expression and elevated poly[ADP‐\\nribose] polymerase 1 (PARP1) levels. These inflammatory\\nmediators are mechanistically linked to carcinogenic\\nprocesses and tumor ‐promoting effects [ 430]. LUSC\\ndemonstrates significant associations with distinct bac-\\nterial families, particularly Enterobacter and Serratia\\n[537, 538]. These specific bacterial taxa contribute to\\ntumorigenesis and malignant progression through mod-\\nulation of multiple cancer‐associated signaling networks.\\nInterestingly, microbial diversity within lung cancer\\nlesions exhibits modest reduction compared to adjacent\\nnormal lung parenchyma, while specific bacterial taxa\\nassociated with SCFA production demonstrate signifi-\\ncant predictive value for lung cancer risk and progression\\n[427, 538]. SCFAs (particularly NaB) modulate immune\\ncell function and immunoregulatory molecules, includ-\\ning activation of CD4\\n+ T cells, Tregs, and Th2 cells;\\ninhibition of CD8+ T cells and NK cells, and induction of\\nIFN‐γ‐R1 in NK cells, thereby suppressing lung cancer\\n30 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 30, 'page_label': '31'}, page_content='development and progression [ 265]. Comprehensive\\nanalyses have identified several bacterial taxa, including\\nActinomycetota, Bacteroidota, andCyanobacteria, that are\\nsignificantly associated with lung cancer progression,\\nwhile respiratory microbiome dysbiosis contributes to\\npulmonary tumorigenesis through multiple inter-\\nconnected mechanisms, including metabolic network\\nperturbation, inflammatory pathway modulation, and\\nimmune response alteration [535].\\nMicrobiome regulation of the immune\\nmicroenvironment in lung cancer\\nThe mechanisms by which the microbiome regulates\\nthe lung cancer immune microenvironment exhibit\\nremarkable complexity and diversity, with profound\\nimplications for tumorigenesis, progression, and ther-\\napeutic response [539]. Existing studies show oral mi-\\ncroorganisms and their metabolites (such as proteins\\nand endotoxins) translocate to pulmonary tissues\\nthrough either direct respiratory tract inhalation or\\nhematogenous disseminat ion. Subsequently, they\\ninfluence lung cancer initiation and progression\\nthrough multiple mechanisms, including chronic\\ninflammatory response induction, host immune system\\nreprogramming, and activation of oncogenic signaling\\npathways [431]. Specifically, oral H. pylori can trans-\\nlocate across lung endothelium, establishing persistent\\ninflammatory microenviron ments that ultimately pro-\\nmote malignant transformation and tumor growth\\nthrough modulation of DC function and suppression of\\nCD8\\n+ T cell cytotoxicity [431]. Additionally, dysregu-\\nlation of oral microbiota can modulate lung cancer cell\\nsurvival through interference with p53 ‐dependent\\napoptotic pathways [540]. Pulmonary microorganisms\\ncontribute to lung cancer development and progression\\nthrough multiple integra ted pathways, including\\ndirectly affecting local immune microenvironment,\\nmodulating tumor ‐associated signaling pathways pro-\\nmoting cell cycle dysregulation and genomic instability,\\nand bacterial metabolite ‐mediated effects involving\\nbacteriocin production, TLR signaling activation, and\\nTNF release [541]. Recent studies identifiedChlamydia\\npneumoniae infection and HPV types 16 and 18 infec-\\ntion as potential lung cancer‐associated risk factors that\\ncontribute to disease progression through profound\\nremodeling of the local immune microenvironment\\n[432]. In summary, the microbiome plays pivotal roles\\nin regulating the TIME regulation of lung cancer, of-\\nfering novel insights and potential therapeutic targets\\nfor comprehensive lung cancer prevention, early\\ndetection, and personalized treatment approaches.\\nRelationship between respiratory microbiome\\nand prognosis of lung cancer treatment\\nStudies show respiratory microbiome composition and\\ndiversity are strongly associated with therapeutic outcomes\\nin patients with lung cancer. A comprehensive study of\\nnever‐smoking female LUAD patients revealed that the\\nrelative abundances ofFaecalibacterium, Fusicatenibacter,\\nand Bacteroides correlate with tumor size, andFusicateni-\\nbacter relative abundances also correlate with tumor stage,\\nsuggesting that specific microbial signatures may play piv-\\notal roles in lung cancer progression and treatment\\nresponse [542]. Additionally, alterations in the abundance\\nof butyrate‐producing bacteria in NSCLC patients have\\ngarnered significant scientific attention [543]. These com-\\nmensal microorganisms may favorably influence lung\\ncancer treatment outcomes through multiple mechanisms,\\nincluding inhibition of pathogenic bacterial growth, en-\\nhancement of nutrient absorption, modulation of immune\\nresponses, and maintenance of intestinal barrier integrity\\n[543]. Recent investigations confirm that physiological\\nconcentrations of SCFAs can significantly inhibit lung\\ncancer cell proliferation by inducing cell cycle arrest and\\npromoting apoptosis, further substantiating the potential\\nmodulatory role of the gut–lung axis in lung cancer treat-\\nment efficacy [544, 545].\\nOther organ microbiomes and tumors\\nMicrobial communities in other organs also demonstrate\\nsignificant associations with tumorigenesis and progres-\\nsion in their respective tissues. Emerging evidence indi-\\ncates that dysbiosis of the breast microbiome\\nsignificantly correlates with BC initiation and progres-\\nsion; similarly, aberrant alterations in the brain micro-\\nbiome increase brain tumor (BT) risk and promote\\ndisease progression, while interactions between the bone\\nmarrow microbiome and hematological malignancies are\\nincreasingly recognized as an important research focus.\\nBreast microbiome and breast cancer\\nRecent studies have demonstrated a significant association\\nbetween breast microbiome and BC development [546, 547].\\nBreast tissue harbors a distinct microbial ecosystem with\\ncharacteristic taxonomic distribution patterns, predomi-\\nnantly dominated by members of theProteobacteriaand\\nFirmicutes [434]. Comparative metagenomic analyses\\nrevealed significant differences in breast tissue microbial\\ncomposition and functional profiles between healthy in-\\ndividuals and BC patients [434]. Quantitative microbiome\\nMICROBIOME IN CANCER | 31 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 31, 'page_label': '32'}, page_content=\"profiling has demonstrated an overall elevation in microbial\\nbiomass within BC tissues, concurrent with a significantly\\nreduced taxonomic diversity index compared to adjacent\\nnormal tissues [548]. In BC patients, total bacterial load is\\nsignificantly elevated, with characteristic enrichment of\\nspecific microbial species such asMethylobacterium radio-\\ntolerans, while the relative abundance ofSphingomonas\\nyanoikuyae is significantly depleted compared to non‐\\nmalignant tissues [435]. Additionally, the TLR signaling\\npathway is significantly dysregulated in BC tissues, a phe-\\nnomenon closely associated with microbial pattern\\nrecognition and immune response modulation [433]. At the\\nmolecular mechanism level, breast microbiome may con-\\ntribute to mammary carcinogenesis. Key biological path-\\nways implicated in this process include DNA damage\\nmediated by toxins fromEscherichia coli(E. coli)a n do t h e r\\nbacteria, and cancer susceptibility mediated by HPV [434].\\nFurthermore, the composition and functional status of the\\nbreast microbiome significantly influence BC treatment\\noutcomes. For example,Lactobacillus inersis significantly\\nassociated with poor patient prognosis and may lead to\\ndecreased chemotherapy and radiotherapy sensitivity [436].\\nIn summary, breast microbiome plays complex and impor-\\ntant regulatory roles in BC occurrence, development, and\\ntherapeutic response. However, the precise molecular\\nmechanisms underlying these interactions and their poten-\\ntial translational applications in clinical oncology warrant\\nfurther comprehensive investigation [549–552].\\nBrain microbiome and brain tumor\\nResearch exploring the association between BTs and the\\nmicrobiome has faced considerable challenges due to the\\nblood–brain barrier's (BBB) intrinsic function in preventing\\ndirect microbial invasion intothe central nervous system.\\nHowever, recent studies have revealed the critical role of\\nspecific microbiomes in modulating the brain TME. For\\nexample, comprehensive metagenomic analyses have\\ndemonstrated significant differences in microbial composi-\\ntion between glioma tissues and adjacent non‐neoplastic\\nbrain parenchyma, with the phylaFirmicutes and Fuso-\\nbacteria exhibiting significantly higher abundance in glio-\\nmas. In‐depth analysis showed that genera including\\nFusobacterium, Limosilactobacillus,a n dPasteurellaare sig-\\nnificantly enriched within glioma tissues [80, 553]. Notably,\\nintegrated multi‐omics analyses have indicated that the\\nintratumoral microbiome may regulate neuron‐related gene\\nexpression networks through bacterial‐derived metabolites.\\nBoth in vivo and in vitro experimental models have con-\\nfirmed that key bacterial species enriched in gliomas\\ninfluence tumor growth. F. nucleatum significantly\\npromotes tumor cell proliferation [80]. These findings\\nsuggest that although microorganisms rarely directly pen-\\netrate the intact BBB, resident microbial communities\\nwithin the brain TME may significantly influence tumor\\nprogression through metabolite production or immune\\nregulation, thereby providing novel direction for exploring\\nthe complex“microbiome–tumor” interaction mechanisms.\\nThe gut–brain axis represents a bidirectional commu-\\nnication network through which the gut microbiota and\\ncentral nervous system reciprocally regulate each other,\\ninfluencing the progression of neurological malignancies\\nand gastrointestinal neoplasms [554]. Research using a\\nneurofibromatosis type 1 ‐associated low‐grade glioma\\nmouse model has elucidated the molecular mechanisms by\\nwhich intestinalBacteroidesspecies regulate optic pathway\\nglioma progression through the TGF‐β signaling pathway\\nwithin the gut–brain–tumor axis, suggesting that targeted\\nmanipulation of gut microbiota could represent a novel\\ntherapeutic strategy for neurological malignancies [555].\\nThrough integrated multi‐omics analysis and experimental\\nvalidation in mouse models, Lin et al. have identified a gut–\\nbrain–tumor axis regulatory network involving enrichment\\nof Clostridium\\nspecies and consequent purine metabolism\\ndysregulation, which collectively mediate target organ\\nredox imbalance. These molecular alterations synergisti-\\ncally contribute to disease pathogenesis, potentially med-\\niating the initiation and progression of craniopharyngioma‐\\nassociated hypothalamic complications [556]. The loss of\\nintestinal epithelial interleukin‐17 receptor A (IL17RA)\\nsignaling induced by aberrant expansion of Th17 cells can\\ninfluence brain tumor growth by regulating the gut\\nmicrobial community ecology [557]. Additionally, the mi-\\ncrobiome may participate in regulating immune responses,\\nmodulating metabolite secretion, and influencing thera-\\npeutic efficacy to affect tumor development [558].\\nBone marrow microbiome and hematologic\\nneoplasms\\nThe bone marrow microbiome has emerged as a critical\\nfactor influencing the initiation, development, and pro-\\ngression of hematologic malignancies [559–561]. Numerous\\ninvestigations have confirmed that the microbiota signifi-\\ncantly influences the bone marrow microenvironment\\nthrough the synthesis of various bioactive metabolites.\\nSCFAs exert bidirectional i mmunomodulatory effects\\nwithin the bone marrow niche by inhibiting the expression\\nof pro‐inflammatory factors, including NF‐κB, IL‐6, and\\nTNF‐α, while concurrently promoting anti‐inflammatory\\nfactor IL‐10 production and enhancing Th17 and Th1 cell\\nactivities [562–564]. Clinical studies have revealed a\\n32 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 32, 'page_label': '33'}, page_content='significant decrease in the abundance of SCFA‐producing\\nbacterial taxa in multiple myeloma (MM) patients, con-\\ncurrent with significant proliferation of bacterial groups\\ninvolved in the nitrogen cycle, such asKlebsiella [437].\\nThese findings reveal that microbiome affects hematologic\\nmalignancies by modulating immune microenvironments\\nand altering metabolic profiles, thereby providing potential\\ntargets for related disease treatment.\\nMICROBIOME IN TUMOR\\nDIAGNOSIS\\nMicrobiome markers\\nMicrobiome markers have demonstrated significant clinical\\nutility in tumor diagnostics. Comprehensive studies have\\nestablished that microbial signatures can serve as robust\\ndiagnostic bioindicators, encompassing (including specific\\nmicrobes, diversity indicators, and metabolites), viral\\nmarkers (including infection status, viral load quantifica-\\ntion, and viral oncoproteins), and fungal markers (including\\nmycobiome community structures and fungal ‐derived\\nmetabolites) [565–569]. Among these, bacterial marker\\nresearch has been more extensively characterized, with\\ncompelling examples includingF. nucleatumserving as a\\nspecific bacterial marker in CRC, GC, and oral cancer; gut\\nmicrobiota diversity indicesand SCFA metabolic profiles\\nshowing value for early detection, risk stratification, and\\nprognostic assessment in CRC. Specific viral biomarkers\\ndemonstrate exceptionally high diagnostic value for partic-\\nular malignancies, exemplified by high‐risk HPV for CC,\\nHBV for liver cancer, and EBV for NPC. Additionally, liquid\\nbiopsy techniques show advantages for non‐invasive diag-\\nnosis in liver and PCs, particularly suitable for early\\nscreening and therapeutic monitoring. These microbiome‐\\nderived biomarkers offer multifaceted clinical applications,\\nincluding population‐based early screening, minimal\\nresidual disease detection, enhancement of conventional\\ndiagnostic accuracy, and provision of novel strategies for\\nprecision oncology diagnostics. Table6 summarizes recent\\nprogress and the clinical application value of microbiome\\nbiomarker research in different tumor types.\\nBacterial markers\\nSpecific microbes: The human microbiome has demon-\\nstrated significant value in cancer diagnostics, with the\\nbiomarker potential of specific microbial taxa attracting\\nsubstantial research interest [582–587]. Numerous stud-\\nies have confirmed that specific microbiota, when uti-\\nlized as tumor diagnostic biomarkers, has significant\\nclinical utility. In colorectal neoplasia studies, compara-\\ntive analyses between patients with colorectal polyps and\\nhealthy controls have revealed significant differences in\\nboth salivary and fecal microbiota composition and\\ndiversity, characterized predominantly by increased\\nabundance of potentially pathogenic bacteria and con-\\ncurrent decreased representation of beneficial microbial\\ntaxa [588]. Additionally, intestinal microorganisms like\\nVeillonella, Bifidobacterium dentium, and Lactobacillus\\nsalivarius have demonstrated good diagnostic value for\\nGC [143]. F. nucleatum has emerged as a promising\\nbiomarker for CRC [ 161]. In familial adenomatous\\npolyposis studies, researchers found intratumoralE. coli,\\nas a precancerous lesion, could serve as an early micro-\\nbial biomarker for CRC risk detection [589]. Addition-\\nally, peripheral blood CRC‐associated microorganisms,\\nsuch as B. fragilis and S. gallolyticus, have shown bacte-\\nrial marker potential in CRC prediction [570]. In GC\\ndiagnostics, specific microbial signatures, including\\nFusobacterium, Streptococcus, and Pseudomonas, have\\nbeen identified as important discriminatory indicators\\ndifferentiating malignant from non‐malignant conditions\\n[575]. In BC research, investigators have observed a sig-\\nnificant inverse correlation between tumor tissue total\\nbacterial load and tumor stage, providing novel insights\\ninto microbiome ‐based approaches for BC diagnosis\\n[580]. These collective findings confirm the significant\\npotential of microbiome ‐based biomarkers in cancer\\nscreening and diagnosis. In conclusion, the human mi-\\ncrobiome demonstrates extensive clinical potential for\\noncological diagnostics, with microbial‐based biomarkers\\nrepresenting a promising frontier that warrants further\\nmechanistic investigation.\\nIndicators of bacterial diversity: As microbiome\\nresearch has advanced, bacterial diversity metrics have\\nemerged as particularly promising diagnostic indicators\\nwith significant clinical potential across multiple cancer\\ntypes. Extensive investigations have confirmed that\\nquantitative assessments of microbial abundance and\\ncommunity compositional characteristics provide inno-\\nvative approaches for cancer detection and classification\\n[588]. Different cancer types present unique bacterial\\ncompositional features. Tissue ‐associated microbiome\\nacross various cancer types has demonstrated prognostic\\nvalue for predicting disease recurrence, with a specific\\ncombination of nine bacterial genera significantly en-\\nhancing the accuracy of patient survival prediction [590].\\nGut microbiota characterization has evolved as an\\neffective non‐invasive assessment methodology for eval-\\nuating HCC risk. Through microbial sequencing tech-\\nnology, investigators have successfully identified optimal\\ncombinations of microbial markers demonstrating sig-\\nnificant diagnostic potential for both early\\n‐stage and\\nMICROBIOME IN CANCER | 33 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 33, 'page_label': '34'}, page_content='TABLE 6 Advances and clinical applications of microbiomebiomarkers in tumor diagnosis.\\nTumor\\ntype\\nBiomarker\\ncategory Representative biomarkers Related research Clinical utility References\\nCRC Bacteria &\\nMetabolites\\nFusobacterium nucleatum\\nBacteroides fragilis\\nγ‐aminobutyric acid\\nL‐aspartic acid\\nphenylacetic acid\\nEnrichment of Fusobacterium\\nnucleatum in tumor tissue correlates\\npositively with tumor stage.\\n• Non‐invasive fecal screening\\n(sensitivity 85%);\\n• Recurrence monitoring.\\n[161, 570, 571]\\nViruses Bacteroides phage\\nStreptococcus phage\\nBiomarker group containing\\n14 viruses\\nViral biomarker panel distinguishes\\nCRC from healthy controls (AUC 0.89).\\nDifferentiation of adenoma vs. carcinoma\\n(specificity 79%).\\n[572, 573]\\nFungi Rhodotorula dairenensis\\nCutaneotrichosporon curvatus\\nReduced fungal diversity in CRC,\\nenrichment of specific taxa.\\nTME profiling: predicting immunotherapy\\nresponse.\\n[574]\\nGC Bacteria Fusobacterium\\nStreptococcus\\nStreptococcus ↑\\nLactobacillus ↑\\nNon‐invasive salivary screening (AUC 0.76). [ 575, 576]\\nFungi Candida albicans Malassezia\\nglobosa\\nC. albicans promotes nitrosamine\\nsynthesis, correlates with GC risk.\\nTissue biopsy support: risk stratification. [ 43, 214]\\nHCC Viruses HBV viral load\\nmiR‐122\\nThe viral load of HBV is negatively\\ncorrelated with mi‐R122.\\nStratified monitoring in hepatitis patients:\\nantiviral efficacy assessment.\\n[577]\\nCC Viruses HPV ‐16/18 ctDNA HPV ‐ctDNA positivity predicts\\nrecurrence risk.\\nMinimal residual disease\\nmonitoring:survival prognosis.\\n[578]\\nPC Bacterial\\nmetabolites\\nPolyamine metabolites\\n(spermine, spermidine)\\nElevated serum polyamine levels\\nprecede imaging abnormalities.\\nHigh\\n‐risk population screening: improved\\ndiagnosis with CA19‐9.\\n[579]\\nBC Bacteria Total bacterial load in\\ntumor tissue\\nLoad inversely correlates with tumor\\nstage.\\nPrognostic assessment (low load indicates\\nadvanced risk).\\n[580]\\nNPC Viruses EBV antibodies Combined detection AUC 0.93 (vs.\\nhealthy controls).\\nCreening in high‐risk regions. [ 581]\\nPCa Fungi Sordariomycetes Increased in plasma\\nabundance (p < 0.01).\\nNon‐invasive diagnosis of advanced PCa;\\nBiochemical recurrence prediction.\\n[222]\\nAbbreviations: AUC, area under curve; CA19‐9, carbohydrate antigen 19‐9; ctDNA, circulating tumor DNA; LPS, lipopolysaccharide.\\n34 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 34, 'page_label': '35'}, page_content='advanced HCC. Concurrently, blood microbiome\\nprofiling‐based diagnostic models have achieved high\\naccuracy in distinguishing HCC from healthy controls\\n[591, 592]. In a comparative study, bacterial diversity was\\ndetermined through OTUs analysis and represented by\\nShannon index, Simpson index, and Invsimpson index\\n[591]. This study demonstrated that all three indices of\\nfecal microbial diversity were significantly reduced in\\ncirrhosis patients compared to the control group. Con-\\nversely, compared to cirrhosis, all three indices were\\nsignificantly elevated, suggesting a characteristic micro-\\nbial diversity pattern associated with hepatocarcinogen-\\nesis. The study concluded that indices based on OTU\\nmarkers possess good diagnostic value [591]. A cross‐\\nsectional study indicated that α‐diversity of circulating\\nmicrobiota was significantly reduced in HCC patients. At\\nthe genus level, seven bacterial taxa demonstrated sig-\\nnificantly differential abundance between HCC and\\ncontrol subjects. Among these, a diagnostic index com-\\nposed of five genus ‐level microbial signatures could\\ncharacteristically distinguish HCC [592]. Additionally,\\ngut microbiota‐based prediction models incorporating 37\\nspecific strains have accurately identified advanced\\nfibrosis status in non‐alcoholic steatohepatitis (NASH)\\npatients, further substantiating the important clinical\\napplication of microbial signatures in early disease\\ndetection [593]. The diagnostic potential of microbial\\ndiversity metrics has also been extensively demonstrated\\nin oral microbiome studies. Studies confirm characteris-\\ntic alterations in tongue microbiota are significantly\\nassociated with GC development, potentially becoming\\nnovel non‐invasive biomarkers [576]. This study repre-\\nsented microbial α‐ and β‐diversity using multiple indi-\\nces, finding that GC patients exhibited significantly\\nincreased tongue coating bacterial richness metrics con-\\ncurrent with decreased overall bacterial diversity indices\\ncompared to non‐cancer controls. Specifically, GC pa-\\ntients demonstrated significantly reduced relative abun-\\ndance of Bacteroidetes, Fusobacteria, Proteobacteria, and\\nActinobacteria [576]. In summary, microbial taxonomic\\nsignatures and bacterial diversity metrics demonstrate\\nsubstantial clinical potential and research value in tumor\\ndiagnosis.\\nBacterial metabolites: Bacterial metabolite markers\\nplay an increasingly important role in tumor diagnosis.\\nStudies confirm intestinal microbial metabolites can\\naccurately reflect the homeostatic state of the gut mi-\\ncroecosystem and serve as key biomarkers for cancer\\ndiagnosis [571]. In non‐invasive diagnostic approaches,\\nthe combination of specific metabolite markers (such as\\nγ‐aminobutyric acid, L‐aspartic acid, and phenylacetic\\nacid) and bacterial markers (such asF. nucleatumand P.\\nanaerobius) has significantly improved discriminatory\\naccuracy between CRC, pre‐existing lesions (CRA), and\\nhealthy controls [ 571]. Metabolomic analysis has\\nrevealed primary microbial metabolites associated with\\ntumor progression, particularly polyamine metabolites.\\nThese metabolites demonstrate significant elevation in\\nthe serum of PC patients, with notably preceding\\ndetectable histologic changes [579]. This suggests intes-\\ntinal microbes analysis and microbial metabolite detec-\\ntion (such as polyamines) could serve as potential non‐\\ninvasive PC detection biomarkers [579]. In HCC studies,\\nbacterial population compositional alterations and their\\nmetabolites demonstrate strong mechanistic correlations\\nwith tumorigenesis. Specifically, decreases in beneficial\\nbutyrate‐producing bacterial populations and increases\\nin LPS‐producing bacteria are strongly associated with\\nearly HCC development. Consequently, researchers have\\ndeveloped and validated relevant microbial profiles that\\ndemonstrate excellent diagnostic performance in inde-\\npendent case‐control validation cohorts, further con-\\nfirming the significant promise of microbiome in tumor\\ndiagnosis [591].\\nViral markers\\nIndicators of viral infection: Viral infection‐related bio-\\nmarkers represent a critical category of microbial signa-\\ntures with important application value in cancer\\ndiagnosis [594–598]. Various oncogenic viruses, includ-\\ning HBV, high‐risk HPV, and EBV, can infect host cells\\nand integrate their genomic material into the cellular\\ngenome, thereby serving as biomarkers for assessing\\nminimal residual diseases (MRDs) through detection of\\nviral circulating tumor DNA (ctDNA) or viral oncopro-\\nteins in liquid biopsies. For example, persistent detection\\nof HPV ctDNA in CC patients following completion of\\nradiotherapy strongly correlates with inferior\\nprogression‐free survival (PFS), and MRDs can be\\nquantified through HPV‐ctDNA detection [578]. In CRC\\ndiagnosis, viral markers have demonstrated significant\\nclinical applications. Studies have shown specific gut\\nvirome signatures, including Phage FAKO27_000271F,\\nFaecalibacterium virus Toutatis, and Faecalibacterium\\nvirus Lugh markers, can effectively differentiate CRC\\npatients from healthy individuals. Furthermore, distinct\\npatterns of Faecalibacterium virus Brigit, Streptococcus\\nphage Javan191, and Streptococcus phage YMC‐2011\\nabundance can discriminate between CRC and CRA\\nwith clinically relevant sensitivity and specificity [572].\\nAdditionally, a multi‐component viral signature com-\\nprising 14 novel viruses has been shown to significantly\\ndifferentiate CRC patients from healthy controls, with\\nseveral viral species enriched in CRC patients [ 573].\\nMICROBIOME IN CANCER | 35 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 35, 'page_label': '36'}, page_content='Moreover, phages not only participate in CRC patho-\\ngenesis but also show potential in diagnosis. Studies have\\nconfirmed specific phage prevalence in early, intermedi-\\nate, and advanced CRC patients, potentially serving as\\nCRC biomarkers [599]. In summary, viral components of\\nthe human microbiome and relative infection indicators\\ndemonstrate potential as biomarkers for cancer diagnosis.\\nViral loads: Viral load and related molecular markers\\nhold valuable diagnostic and prognostic indicators in\\ntumor diagnosis. HBV and HCV, as major hepatotropic\\nviral pathogens, not only significantly disrupt normal\\nliver physiological functions but also demonstrate clear\\netiological associations with HCC initiation, develop-\\nment, and progression. MicroRNA ‐122 (miR ‐122)\\npromotes HCV replication, while HBV viral load dem-\\nonstrates a significant inverse correlation with miR‐122\\nexpression. Dysregulated miRNAs mediate complex in-\\nteractions at the host–virus interface and can promote\\nviral persistence in HCC, thereby potentially serving as\\nnovel detection biomarkers [577]. Other viruses like HPV\\ndemonstrate well ‐established etiological associations\\nwith malignancies. In cervical pre‐cancer (CIN) and CC,\\nHPV viral load serves as an important risk assessment\\nparameter [600]. Therefore, viral components of the mi-\\ncrobiome and viral load represent important molecular\\ntools in cancer diagnosis. In‐depth investigation of virus‐\\nrelated molecular signatures and their mechanistic con-\\nnections to carcinogenesis will provide critical theoretical\\nfoundations and innovative research directions for early\\ncancer diagnosis, prognosis assessment, and personalized\\ntherapeutic strategy development.\\nViral antigens: Viral antigens and their corresponding\\nhost‐derived antibody responses represent critical viral\\nbiomarkers in cancer diagnosis. Current studies explore\\ndiverse predictive biomarkers incorporating viral anti-\\ngens, with the objective of enhancing diagnostic sensi-\\ntivity and specificity [601]. Viral markers demonstrate\\nparticular significance in NPC diagnosis and monitoring.\\nSerological analyses have demonstrated that five EBV‐\\nspecific antibodies, including BLRF2 ‐IgA, BLRF2‐IgG,\\nand BDLF1‐IgG5, exhibit significantly elevated levels in\\nNPC patients compared to healthy controls, highlighting\\ntheir potential as biomarkers for early disease detection\\n[581]. Follow‐up studies have confirmed that integrating\\nthese five EBV‐associated antibodies with EBNA1‐IgA\\nsignificantly enhances the diagnostic accuracy for NPC\\n[581]. Viral antigens exhibit significant clinical utility\\nbeyond NPC, contributing to diagnostic approaches for\\nvarious other malignancies. In China, chronic hepatitis B\\nand hepatitis B carriers (HBsAg positive) are correlated\\nwith significantly elevated PC risk [602], suggesting that\\nHBsAg screening may facilitate early risk assessment and\\ninform preventive interventions. In summary, viral\\ncomponents of the microbiome, particularly viral anti-\\ngens and their corresponding antibody responses, play an\\nincreasingly important role in tumor diagnosis.\\nFungal markers\\nCharacterization of fungal communities: Studies have\\nshown that alterations in fungal abundance and com-\\nmunity composition within tumor tissues and associated\\nsamples correlate significantly with cancer development\\n[603]. For example, C. albicans exhibits significant en-\\nrichment in GC tissues and demonstrates potential utility\\nas a fungal biomarker [43]. Additionally, oralM. globosa\\nhas emerged as a candidate fungal biomarker species for\\nGC diagnosis [ 214]. Another study revealed that 14\\nfungal biomarkers effectively distinguished CRC patients\\nfrom healthy individuals across different racial popula-\\ntions, highlighting the significant diagnostic potential of\\nfungal signatures in CRC [ 227]. In a tongue micro-\\norganism study, 14 fungal taxa, includingAmpelomyces\\nsp IRAN 1and related species, demonstrated significant\\nabundance increases in GC patients and exhibited\\npotential as diagnostic biomarkers, further expanding the\\ntaxonomic scope of fungi in cancer diagnosis [ 604].\\nLinear discriminant analysis effect size (LEfSe) analysis\\nfurther revealed significant differences in fungal\\ntaxonomic profiles among patients with CRC, colorectal\\npolyps, and healthy controls. Several fungal taxa,\\nincluding Rhodotorula dairenensis and Cutaneo-\\ntrichosporon curvatus, exhibited significant enrichment\\nin CRC patients and may contribute to the formation of a\\ntumor‐permissive microenvironment [574]. Additionally,\\nother studies have reported the significant correlation\\nbetween specific fecal fungal signatures and GC, pro-\\nviding rationale for the application of fecal mycobiome\\nprofiling as a non‐invasive cancer screening approach\\n[143]. Notably, disease progression stages demonstrated\\nsignificant correlation with characteristic patterns of\\nfungal microbiota [605].\\nFungal metabolites: Fungi play important roles in\\ntumorigenesis and progression through metabolite regu-\\nlation, with both fungal taxa and their secreted metabo-\\nlites demonstrating significant potential as clinical\\nbiomarkers. Fungal biomarkers and metabolites poten-\\ntially exhibit diagnostic potential across multiple cancer\\ntypes. For example,Candida spp., identified as potential\\nfungal biomarkers for OSCC, synthesize carcinogenic\\nmetabolites including nitrosamines that participate in\\noncogenic transformation and progression. This mecha-\\nnistic relationship provides a scientific foundation for the\\nutilization of fungal‐derived metabolomic signatures as\\ndiagnostic indicators [606]. Recent studies have identified\\n36 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 36, 'page_label': '37'}, page_content='fecal fungi, including Sordaria pseudoproxies, Gibellu-\\nlopsis,a n dCercophora, significantly correlate with ele-\\nvated GC risk. They also demonstrate significant\\ncorrelations with altered serum amino acid profiles, par-\\nticularly methionine, L‐alanine, and L‐threonine levels,\\nsuggesting these metabolic signatures may serve as com-\\nplementary biomarkers for enhanced GC detection [607].\\nMicrobial metabolite markers\\nAs the field of cancer metabolomics advances, microbial‐\\nderived metabolite signatures demonstrate significant\\ndiagnostic potential across multiple cancer types. Evi-\\ndence indicates that intestinal microbial communities\\nand their associated metabolites (including bile acid)\\nundergo significant alterations during PC development,\\naffecting chemotherapeutic efficacy and clinical out-\\ncomes while potentially serving as discriminatory bio-\\nmarkers for PC [608]. Additionally, metabolites derived\\nfrom gastrointestinal microorganisms, particularly\\nSCFAs, demonstrate significant associations with diverse\\nmalignancies and exhibit potential as reliable biomarkers\\nfor cancer type differentiation [609]. In CRC investiga-\\ntions, researchers have documented significant differ-\\nences in gut microbial composition and metabolic\\nfunctions between healthy individuals and patients\\nacross the colorectal neoplasia continuum, from adeno-\\nmas to invasive adenocarcinomas, providing novel\\nmetabolic biomarker candidates for early ‐stage CRC\\ndetection [ 610]. Quantitative analysis of eight gut\\nmicrobiota‐derived serum metabolites has validated their\\nhigh diagnostic accuracy for distinguishing CRC and\\nprecancerous adenomas [611]. In lung cancer studies,\\nspecific metabolites including cysteinyl valine,\\n3‐chlorobenzoic acid, and 3,4‐dihydroxyphenyl ethanol\\neffectively discriminated lung cancer patients from con-\\ntrols and exhibited enhanced diagnostic performance\\nwhen analyzed as a composite signature, suggesting\\npotential utility as clinically relevant biomarkers for lung\\ncancer diagnosis [612]. Various intestinal microorganism\\nmetabolites (such as secondary bile acids) demonstrate\\nsignificant associations with gastrointestinal carcino-\\ngenesis, most notably in CRC, and represent promising\\nnon‐invasive biomarkers [ 613]. Of particular clinical\\nsignificance, early detection of PC may be achieved\\nthrough analysis of intestinal microbial community al-\\nterations and metabolites (such as polyamines). These\\naberrant patterns exist prior to histologically detectable\\ndisease [579]. In conclusion, microbial‐derived metabo-\\nlites demonstrate considerable potential as cancer bio-\\nmarkers, offering novel avenues for early detection and\\ntherapeutic strategy development.\\nInflammation‐related markers\\nIn recent years, inflammation ‐related markers have\\nemerged as promising biomarkers for cancer detection.\\nMultiple investigations have demonstrated significant\\nenrichment of Parvimonas micra (P. micra)i nb o t ht h e\\nintestinal mucosal tissues and fecal samples of CRC pa-\\ntients compared to healthy controls [ 614]. Notably,\\nP. micracolonization significantly upregulates the expres-\\nsion of several pro‐inflammatory mediators, including IL‐5,\\nIL‐8, CCL20, and CSF2, which collectively contribute to the\\nestablishment and maintenance of a pro ‐tumorigenic\\ninflammatory microenvironment in CRC [614]. Therefore,\\nP. micra abundance and its associated inflammatory sig-\\nnature may serve as clinicallyrelevant biomarkers for CRC,\\nproviding new strategies for early detection and treatment\\nof CRC. Additionally,C. albicanspromotes cytokine pro-\\nduction, particularly through Th17 cell response induction,\\nand enhances adhesion molecule expression, thereby\\nmediating pro‐inflammatory responses that contribute to\\ncancer progression [417]. This mechanistic relationship\\nsuggests thatC. albicanscolonization patterns and associ-\\nated inflammatory signaturesmay represent valuable bio-\\nmarkers for cancer.\\nOrgan‐specific microbial markers\\nApplications of microbiome analysis in cancer diag-\\nnostics demonstrate distinct organ ‐specific signatures,\\nwith site‐specific microbial markers offering novel ap-\\nproaches for early cancer detection and characterization.\\nExisting studies have explored microbial markers in the\\noral cavity, skin, urinary system, and respiratory tract,\\ncollectively establishing a comprehensive framework for\\nmicrobiome‐based cancer diagnosis. Table7 summarizes\\ndifferent organ system key microbial markers and their\\nclinical applications and research progress in corre-\\nsponding tumor types, providing an important reference\\nfor advancing precision oncology and individualized\\ntreatment strategies.\\nOral microbial markers\\nOral microbiome signatures represent a critical subset of\\norgan‐specific microbial markers with demonstrated utility\\nin disease diagnosis, prognostic stratification, and thera-\\npeutic response prediction across multiple cancer types.\\nRecent studies have revealed that enrichment of peri-\\nodontal pathogens, includingF. nucleatum, P. gingivalis,\\nand T. denticola, correlates with oral potentially malignant\\ndiseases and oral cancer [40, 162]. The magnitude of\\nMICROBIOME IN CANCER | 37 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 37, 'page_label': '38'}, page_content='pathogenic bacterial enrichment is strongly associated with\\noncogenic genes and coincides with a significant reduction\\nin commensal bacteria, particularly Streptococcus,s u g -\\ngesting these distinctive microbial signatures may serve as\\nnovel biomarkers for oral cancer [162]. Meta‐analytical\\nstudies have established significant correlations between\\noral microbiome profiles and multiple cancer types. Spe-\\ncifically, taxonomically diverse microorganisms, including\\nAlloprevotella, Streptococcus,a n dM. globosa, have been\\ncharacterized as potential oral biomarkers for GC\\n[214, 618]. Notably, reduced α‐diversity within the oral\\nmicrobiome has emerged as a predictive marker for lung\\ncancer risk. Although the direct causal relationship\\nbetween periodontal pathogens and lung cancer requires\\nfurther mechanistic elucidation, specific microorganisms,\\nparticularly F. nucleatum, demonstrate potential utility as\\nnon‐invasive biomarkers for lung cancer [163]. Alterations\\nin oral microbial composition, particularly changes in the\\nabundance of five genera includingBacillus spp.a n dEn-\\nterococcus spp., effectively discriminate between patients\\nwith epithelial precursor lesions and those with invasive\\ncarcinoma, potentially serving as clinically relevant bio-\\nmarkers for monitoring oral carcinogenesis progression\\n[619]. In summary, oral microbial signatures represent\\nessential components of the organ‐specific microbiome\\nlandscape with substantial clinical applications in cancer\\ndiagnosis, prognostic assessment, and longitudinal disease\\nmonitoring across diverse malignancies.\\nSkin microbial markers\\nEmerging evidence indicates that cutaneous microbiome\\nsignatures demonstrate significant diagnostic, prognostic,\\nand risk stratification potential across various dermato-\\nlogical malignancies. Multiple studies have confirmed\\nsignificant differences in microbiome composition and\\nabundance (especiallyS. aureus) between melanoma and\\nnon‐melanoma skin carcinomas compared to site ‐\\nmatched healthy controls [615]. Researchers have iden-\\ntified novel microbial markers associated with actinic\\nkeratosis (AK) or SCC, including AK ‐associated Ral-\\nstonia and Diaphorobacter, as well as SCC ‐associated\\nRalstonia and Streptococcus [480]. Host factors including\\nimmunosuppression, chronic inflammatory conditions,\\nand oncogenic viral infections significantly increase\\ncutaneous susceptibility to malignant transformation,\\nwhile their complex interactions with the resident\\nTABLE 7 Organ‐specific microbial biomarkers and their association with tumor characteristics.\\nOrgan system\\nKey microbial\\nbiomarkers\\nAssociated\\ncancer types Related research Clinical utility References\\nOral cavity Fusobacterium\\nnucleatum\\nPorphyromonas\\ngingivalis\\nAlloprevotella\\nStreptococcus\\nMalassezia globosa\\nOral cancer\\nGC\\nLung cancer\\n• Fusobacterium nucleatum\\ncorrelates with oral cancer gene\\nexpression;\\n• Reduced Streptococcus\\nabundance indicates GC risk;\\n• Decreased oral α‐diversity\\npredicts LC risk.\\n• Non‐invasive oral swab\\nscreening;\\n• Pre‐cancerous lesion\\nmonitoring;\\n• Prognostic stratification.\\n[162, 214]\\nSkin Staphylococcus\\naureus\\nRalstonia\\nDiaphorobacter\\nStreptococcus\\nMelanoma\\nSCC\\nStaphylococcus aureusvirulence\\nfactors promote carcinogenesis.\\n• Skin swab diagnostic\\nbiomarkers;\\n• High‐risk lesion.\\n[480, 615]\\nUrogenital Bacteroides\\nPorphyrobacter\\nHerbaspirillum\\nAcinetobacter\\nFusobacterium\\nFenollaria/Ezakiella\\nBCa\\nPCa\\n• High prevalence of\\nAcinetobacter in BCa;\\n• Fusobacterium positively\\ncorrelates with PCa\\ninvasiveness.\\n• Urine microbiome non‐\\ninvasive diagnosis;\\n• Recurrence risk\\nprediction;\\n• Personalized therapy\\nresponse assessment.\\n[531, 616]\\nRespiratory Streptococcus\\nPrevotella\\nVeillonella\\nAkkermansia\\nmuciniphila\\nLung cancer • Streptococcus enrichment\\ncorrelates with poor prognosis;\\n• Akkermansia muciniphila\\npredicts immunotherapy\\nresponse.\\n•\\nSputum/\\nbronchoalveolar\\nlavage fluid testing;\\n• Immunotherapy efficacy\\nprediction;\\n• Survival period\\nassessment.\\n[429, 617]\\n38 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 38, 'page_label': '39'}, page_content='microbiome further modulate disease initiation and\\nprogression. Despite the significant potential of cutane-\\nous microbial signatures as biomarkers, in ‐depth\\nexploration in this field remains relatively limited. With\\nincreasing research investment, cutaneous microbiome\\nmarkers are anticipated to provide innovative strategies\\nfor early diagnosis, personalized treatment, and\\nprognostic stratification of skin cancer.\\nMicrobial markers of the urinary system\\nUrinary microbiome signatures represent an emerging\\nclass of non‐invasive biomarkers with significant utility\\nin the diagnosis, molecular classification, risk stratifi-\\ncation, and prognostic assessment of genitourinary\\nmalignancies. Recent comprehensive studies targeting\\nurinary microbiota have systematically identified sev-\\neral bacterial genera with statistically significant asso-\\nciations with urinary tumors. The relative abundance of\\nBacteroides , Porphyrobacter ,a n dHerbaspirillum shows\\nsignificant increases in BCa patients characterized by\\nhigh recurrence and progression risk, suggesting their\\npotential value as risk stratification biomarkers [616].\\nTo thoroughly elucidate the core microbiome char-\\nacteristics of BCa, researchers integrated multiple data\\nsets and identified 31 characteristic bacterial genera,\\nincluding Acinetobacter [620]. The detection frequency\\nof Acinetobacter was significantly higher in tumor pa-\\ntients, highlighting its po tential value as a microbial\\nmarker for BCa.\\nAdditionally, urinary microbial signatures have dem-\\nonstrated significant clinical potential in the non‐invasive\\ndiagnosis, molecular classification, staging, and prognostic\\nassessment of PCa. Comprehensive metagenomic analysis\\nof post ‐prostate massage urine specimens and post ‐\\nprostatectomy prostate secretions revealed that alterations\\nin five strictly anaerobic bacterial genera, includingFuso-\\nbacterium and Fenollaria/Ezakiella, significantly correlate\\nwith PCa aggressiveness and recurrence risk. This suggests\\ntheir potential utility as urinary cancer biomarkers [531].\\nThis emerging evidence not only emphasizes the important\\nrole of the urinary microbiome in the clinical management\\nof PCa but also provides new research directions for\\ndeveloping novel liquid biopsy approaches and precision\\nmedicine strategies.\\nRespiratory microbial markers\\nThe respiratory microbiome represents a complex eco-\\nlogical niche with emerging diagnostic, prognostic, and\\ntherapeutic implications in pulmonary malignancies.\\nComprehensive metagenomic analyses have demonstrated\\nthat elevated abundance of specific microbial communi-\\nties within the lower respiratory tract, particularly oral\\ncommensals includingStreptococcus, Prevotella,a n dVeil-\\nlonella, is significantly associated with poor prognosis in\\nlung cancer patients [429]. Furthermore, a clinical study\\nencompassing patients with colorectal, breast, and lung\\nmalignancies demonstrated that intratumoral microbial\\ndiversity exhibits a significant negative correlation with\\nimmunohistopathological markers, including TILs and\\nPD‐L1 expression. These immunological relationships are\\nstrongly associated with unfavorable clinical outcomes\\n[345]. Additionally, the abundance of the intestinal mu-\\ncinophilic bacterium Akkermansia muciniphila has\\nemerged as a significant predictive biomarker for immu-\\nnotherapeutic response in lung cancer patients [617]. In\\nsummary, respiratory microbiome signatures demonstrate\\nsubstantial clinical utility in lung cancer detection and\\nprognostic assessment, advancing our mechanistic un-\\nderstanding of lung cancer biology while establishing\\nnovel biomarker platforms and potential therapeutic tar-\\ngets for precision oncology approaches in pulmonary\\nmalignancies.\\nCombined application of liquid biopsy and\\nmicrobiome markers\\nThe integration of microbiome analysis with liquid biopsy\\ntechnologies represents a rapidly evolving frontier in\\ncancer diagnosis, with this multimodal approach demon-\\nstrating exceptional potential. The detection of circulating\\nmicrobial DNA offers the possibility of non ‐invasive\\nmonitoring of tumor microbiome changes, while\\nmicrobial‐derived exosomes function as biological couri-\\ners, conveying specific markers of the tumor microbiome.\\nThis combinatorial approach establishes novel paradigms\\nfor early cancer diagnosis, treatment efficacy monitoring,\\nand recurrence surveillance across tumor types.\\nCirculating microbial DNA testing\\nThe integration of liquid biopsy methodologies with mi-\\ncrobiome analysis, particularly through the detection and\\ncharacterization of circulating microbial DNA (cmDNA),\\ndemonstrates remarkable diagnostic and prognostic\\npotential across diverse malignancies [621–623]. Circu-\\nlating microbial DNA encompasses free DNA fragments\\nof microbial origin detectable in peripheral blood and has\\ndemonstrated significant diagnostic value across various\\nmalignancies, particularly CRC [624]. In HCC research,\\nsignificant changes in the abundance of seven bacterial\\nMICROBIOME IN CANCER | 39 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 39, 'page_label': '40'}, page_content='species were detected in the serum of HCC patients\\ncompared to patients with cirrhosis and healthy controls,\\nwith particularly notable enrichment ofStaphylococcus.\\nInvestigators subsequently developed a multivariate\\nscoring model based on the abundance profiles of five\\nbacterial species that successfully discriminated HCC\\npatients, strongly suggesting the potential application of\\ncmDNA in HCC diagnosis [592].\\nBeyond HCC, cmDNA has demonstrated significant\\ndiagnostic and prognostic value in other cancer types.\\nFor instance, consistently detected HPV‐ctDNA follow-\\ning chemoradiotherapy (CRT) for CC correlates with\\ninferior PFS. Assessment of MRD through HPV‐ctDNA\\nhas demonstrated substantial clinical utility [ 578]. In\\noropharyngeal cancer (OPC) studies, detection of HPV‐\\n16 DNA in plasma and saliva serves as a highly sensitive\\npredictor of disease recurrence [625]. These collective\\nfindings not only underscore the critical role of HPV in\\ncancer diagnosis and prognostic assessment but also es-\\ntablish scientific foundations for the clinical application\\nof cmDNA detection technology across a diverse spec-\\ntrum of malignancies.\\nExosomes of microbial origin\\nExtracellular Vesicles (EVs) of microbial origin have\\ndemonstrated significant value as novel biomarkers in\\ndisease diagnosis. A systematic analysis of bacterial\\nEVs isolated from both peripheral blood and tissues of\\npatients with BTs revealed significant differences in\\nbacterial EV distributio n between BT patients and\\nhealthy controls. Specifically, Saccharibacteria , Pre-\\nvotellaceae ,a n d Dialister demonstrated significant\\ndepletion in both circulation and tissue microenviron-\\nments of BT patients, andErysipelotrichia was signifi-\\ncantly increased in both circulation and tissue\\nmicroenvironments of BT patients. Lachnospiraceae\\nNK4A136 exhibited divergent compartment ‐specific\\nalterations, showing significant enrichment in periph-\\neral blood yet concurrent depletion in tumor tissues of\\nBT patients [626]. This finding elucidates the potential\\nvalue of bacterial EVs for BT diagnosis, particularly for\\ndetecting early‐stage lesions that conventional imaging\\nmodalities fail to identify [626]. Furthermore, bacterial\\nEVs in serum have important applications in HCC\\ndiagnosis [592]. A study developed a diagnostic method\\nbased on detecting glypican ‐1( G P C 1 ) ,am e m b r a n e‐\\nanchored proteoglycan enriched on the surface of EV.\\nThis approach effectively dif ferentiates patients with\\nbenign pancreatic disorders from those with pre ‐\\nmalignant pancreatic lesions and demonstrates supe-\\nrior sensitivity for identifying patients with advanced\\nPC, offering significant advantages over conventional\\nPC marker carbohydrate antigen 19‐9( C A 1 9‐9) [627].\\nIn cancer biology research, EV‐based microRNA char-\\nacterization provides novel concepts for developing\\nclinical diagnostic algorithms applicable to early\\ndetection of various gastrointestinal cancers, including\\nCRC [628]. In summary, the application of microbial‐\\nderived EVs, representing a combination of liquid\\nbiopsy and microbiome markers, has demonstrated\\nsignificant diagnostic advantages across various malig-\\nnancies, including biliary tract tumors, HCC, and gas-\\ntrointestinal cancers. This multidisciplinary approach\\nhas established innovative research directions for early\\ndisease diagnosis and treatment plan development.\\nStandardization of microbiome diagnostic\\nmethods\\nThe analytical validity and reproducibility of\\nmicrobiome‐based diagnostic applications depend criti-\\ncally on rigorous standardization of sample collection\\nprotocols. Consequently, implementation of standardized\\noperating procedures for specimen acquisition represents\\na fundamental prerequisite for generating reproducible,\\nclinically actionable microbiome data.\\nSpecifications for sample collection and\\nhandling\\nImplementation of standardized microbial sample col-\\nlection and processing protocols represents a critical\\ndeterminant of analytical accuracy and reliability.\\nAcknowledging the spatial heterogeneity of microbial\\ncommunity distribution across anatomical niches [629],\\nstandardization efforts must ensure consistent sampling\\nsite selection with precise anatomical localization and\\ndocumentation. Specimens must be collected in statis-\\ntically sufficient quantitie st oe n s u r ea d e q u a t er e p r e -\\nsentation of microbial diversity and biomass, with\\noptimal sampling preferably performed during the acute\\nphase of disease and before initiating antimicrobial\\ntherapy. Sampling procedures must follow standardized\\naseptic protocols to minimize environmental and cross‐\\nsite contamination from commensal microbiota.\\nAcquired specimens should avoid contact with disin-\\nfectants or preservatives and be promptly submitted for\\ntesting [630]. Samples should be collected in sterile,\\nsealed, specialized containers to maintain microbial\\nviability during transportation [ 630]. Additionally,\\nsample‐specific processing workflows must be estab-\\nlished based on the unique biological characteristics and\\n40 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 40, 'page_label': '41'}, page_content='stability profiles of distinct specimen types. Specimens\\nfrom sterile sites, including cerebrospinal fluid and\\nbody fluids, require expedi ted processing protocols.\\nFecal and urinary specimens, however, demonstrate\\nmicrobiome stability and can be maintained under\\nvalidated preservation conditions for extended periods.\\nFor sensitive organisms such as Shigella spp .a n d\\nNeisseria gonorrhoeae ,i m m e d i a t es a m p l ep r o c e s s i n g\\nis necessary to ensure testing accuracy [ 631]. A\\nstandardized system for microbiome research requires\\nestablishing evidence ‐based standard operating proce-\\ndures, encompassing critical elements including sam-\\npling site consistency, aseptic protocol implementation,\\nappropriate collection method selection, suitable trans-\\nport media, and differentiated sample processing\\nstrategies.\\nStandardization of detection methods\\nThe standardization of microbiome characterization meth-\\nodologies represents a fundamental challenge in translating\\nmicrobiome research. Currently, microbial taxonomic\\nclassification primarily employs two complementary ana-\\nlytical approaches, targeted amplicon sequencing of phy-\\nlogenetic marker genes and metagenomic shotgun\\nsequencing analysis, which collectively provide robust\\ntechnical frameworks for systematic microbiome investi-\\ngation [632]. Establishing standardized processes is equally\\nessential for implementing rapid detection techniques.\\nThese techniques include immunofluorescence, agglutina-\\ntion tests, immunochromatography (ICT), enzyme im-\\nmunoassays (EIA), and molecular microbiology techniques,\\nall widely implemented in clinical microbiome analysis due\\nto their rapid turnaround time, operational simplicity, and\\ndiagnostic performance characteristics [631]. However,\\nadvanced assays alone cannot guarantee diagnostic per-\\nformance; systematic optimization of the entire analytical\\nworkflow remains indispensable. Laboratories typically\\nadopt selective testing strategies, including taking a cau-\\ntious approach to culture results of suspected contaminants\\nand avoiding unnecessary antimicrobial susceptibility tests.\\nThese practices collectively represent critical strategies for\\nenhancing diagnostic specificity and optimizing anti-\\nmicrobial stewardship [633]. The standardization of mi-\\ncrobiome diagnostic methods requires not only\\nharmonization of testing techniques but also optimization\\nof testing processes and standardized application of rapid\\ntesting detection methods. Collectively, these measures\\nconstitute vital components of a standardized microbiome\\ndiagnostic framework, establishing a solid foundation for\\nimproving diagnostic accuracy, optimizing antimicrobial\\nuse, and advancing microbiome research.\\nMICROBIOME IN TUMOR\\nPROGNOSIS AND EFFICACY\\nPREDICTION\\nThe microbiome has emerged as a crucial bioindicator\\nfor tumor prognosis assessment and treatment efficacy\\nprediction. Evidence indicates that alterations in micro-\\nbial diversity, taxonomic composition, and metabolite\\nprofiles correlate significantly with clinical outcomes in\\ncancer patients. The microbiome influences therapeutic\\noutcomes through multiple mechanisms, including\\nmodulation of tumor response to chemotherapy, radio-\\ntherapy, and immunotherapy, while also contributing to\\ntreatment‐related adverse events. Table 8 summarizes\\ncurrent evidence regarding microbiome ‐based bio-\\nmarkers for tumor prognosis prediction and treatment\\nresponse assessment, along with their potential clinical\\napplications.\\nMicrobiome and tumor prognosis\\nMicrobial diversity and prognosis\\nMicrobial diversity and its influence on tumor prognosis\\nhave emerged as a critical focus in contemporary onco-\\nlogical research [646]. Evidence suggests that microbiota\\nfrom both oral cavity and intestinal tract are associated\\nwith PC prognosis, with Streptococcus spp., Prevotella\\nspp., or Veillonella spp. showing significant negative\\ncorrelations with survival outcomes [429, 634]. These\\nfindings indicate that microorganisms from multiple\\nanatomical sites have potential as prognostic biomarkers\\nfor PC. Previous studies have revealed significant al-\\nterations in the microbiome composition of patients with\\nHNSCC before and after surgery. Microbialα‐diversity\\nsignificantly decreases following surgery but subse-\\nquently increases in patients experiencing tumor recur-\\nrence [635], indicating that microbiota diversity patterns\\nmay serve as predictive indicators for HNSCC prognosis.\\nAdditionally, higher gut microbiome diversity con-\\nsistently correlates with favorable clinical responses in\\nNSCLC patients undergoing immunotherapy [636].\\nSpecific microbial species and prognosis\\nAccumulating evidence highlights the prognostic signif-\\nicance of specific microbial species in predicting tumor\\nprogression and treatment outcomes. Multiple investi-\\ngations have confirmed that oral mucositis (OM), a\\ncommon adverse effect of cancer treatment. It exhibits\\nvariations in microbial composition that correlate with\\nMICROBIOME IN CANCER | 41 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 41, 'page_label': '42'}, page_content='TABLE 8 Key roles and clinical applications of the microbiome in tumor prognosis and treatment response prediction.\\nApplication\\ndirection Tumor type Key microbes/metabolites Key roles Clinical application strategies References\\nPrognostic prediction\\nMicrobial diversity PC Streptococcus\\nPrevotella\\nVeillonella\\nReduced microbial diversity correlates with\\nshorter survival (HR = 1.8).\\nMulti‐site microbial detection (oral& gut) to\\nguide postoperative management.\\n[429, 634]\\nHNSCC Postoperative microbial\\ndynamics\\nIncreased α‐diversity in recurrent patients\\n(p = 0.006).\\nPostoperative microbiome monitoring to\\npredict recurrence risk.\\n[635]\\nNSCLC Gut microbial diversity High diversity linked to better immunotherapy\\nresponse.\\nStratification of immunotherapy patients. [ 636]\\nSpecific microbial\\nbiomarkers\\nCRC Fusobacterium nucleatum Tumor‐enriched F. nucleatum shortens OS and\\npromotes pro‐inflammatory microenvironment.\\nFecal testing to identify high‐risk patients\\nguiding adjuvant chemotherapy intensity.\\n[164]\\nBC\\nGC\\nLUAD\\nTissierella Higher abundance correlates with prolonged OS\\n(p < 0.01) and inhibits metastasis.\\nDevelopment of probiotic formulations to\\nenhance anti‐tumor efficacy.\\n[44]\\nHCC Lachnoclostridium ↑\\nPrevotella 9 ↓\\nCombined signature predicts high response\\nto ICI.\\nFMT to optimize immunotherapy. [ 637]\\nLung cancer Prevotella\\nStreptococcus\\nVeillonella\\nEnrichment correlates with poor survival\\nprognosis (p < 0.05).\\nSputum testing to guide prognosis\\nmanagement.\\n[429]\\nCC Microbacterium\\nStreptococcaceae\\nHigher abundance correlates with reduced OS/\\nRFS (p < 0.001).\\nIdentification of high‐\\nrisk patients;\\nRecurrence monitoring.\\n[638]\\nNSCLC Bacteroides dorei\\nParabacteroides distasonis\\nEnrichment linked to prolonged OS (>6 mo). Gut microbiome testing to identify patients\\nlikely to benefit from immunotherapy.\\n[639]\\nMetabolite biomarkers HCC Galanthaminone (bacterial\\nmetabolite)\\nHigh levels correlate with prolonged OS in ICI\\ntherapy, Inhibits PD‐1/PD‐L1 pathway.\\nDynamic serum metabolite monitoring to\\nguide ICI treatment timing.\\n[640]\\nHCC Bile acids (fecal metabolites) Specific bile acid profiles correlate positively\\nwith ICI efficacy.\\nPersonalized immunotherapy strategy\\ndevelopment.\\n[637]\\nPan‐cancer Butyrate Activates T cell anti ‐tumor activity; Enhances\\nPD‐1 efficacy.\\nOral butyrate supplementation combined\\nwith immunotherapy.\\n[641]\\nEfficacy prediction\\nDrug resistance PC Escherichia coli Enzyme‐mediated gemcitabine inactivation. Antibiotics (such as ciprofloxacin)\\ncombined with chemotherapy to reverse\\nresistance.\\n[642]\\n42 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 42, 'page_label': '43'}, page_content=\"disease severity, specifically decreased abundance of\\nPrevotella, Leptotrichia, and Actinomyces alongside\\nincreased prevalence of Treponema [647]. In CRC, the\\nabundance of F. nucleatum in tumor tissues may have\\nthe potential to serve as a prognostic biomarker [164]. In\\nvivo experiments have established that E. coli signifi-\\ncantly reduces gemcitabine's anti‐tumor effect, resulting\\nin increased tumor burden and compromised patient\\nsurvival [46]. Conversely, co‐administration of Lactoba-\\ncillus with cisplatin significantly suppresses tumor\\ngrowth and improves survival in mouse models of lung\\ncancer [648].\\nNotably, a potentially anti‐cancer microbiome genus,\\nTissierella, has been significantly associated with im-\\nproved prognosis across various tumors, including breast,\\nlung, and gastric cancers, further confirming the critical\\nrole of specific microbial species in tumor prognosis [44].\\nAdditionally, the salivary microbiota of PC patients ex-\\nhibits unique prognosis ‐related features. Poor oral\\nhygiene and periodontal disease caused by microbial\\ndysbiosis represent independent risk factors for PC, un-\\nderscoring the oral microbiome's significant role in PC\\nprognosis [429, 649]. Moreover, differential abundances\\nof Neisseria elongata and Streptococcus mitis in saliva\\nsamples from PC patients compared to healthy controls\\ndemonstrate potential as non‐invasive biomarkers for PC\\nprediction [650].\\nIn lung cancer studies, enrichment ofPrevotella spp.,\\nStreptococcus spp., and Veillonella spp. consistently cor-\\nrelates with inferior survival outcomes [429]. Specifically,\\nbacterial abundance in normal tissues adjacent to lung\\ncancer correlates with recurrence ‐free survival (RFS).\\nSignificant positive correlations exist between Kor-\\nibacteraceae family enrichment and RFS, while inverse\\ncorrelations are observed between Bacteroidaceae,\\nRuminococcaceae, and Lachnospiraceae family enrich-\\nment and RFS [651].\\nViral–microbial interactions play equally important\\nroles in tumor prognosis. HPV has been demonstrated to\\nbe a potent prognostic biomarker, with HPV ‐positive\\npatients typically experiencing better prognosis, while\\nthe presence of HPV DNA in plasma and saliva serves as\\nan effective predictor of OPC recurrence [601, 652]. In\\nHPV‐independent cervical adenocarcinoma, the abun-\\ndance of Microbacterium and Streptococcaceae family\\nmicroorganisms significantly correlates with diminished\\nOS and RFS, further validating the utility of specific\\nmicrobial signatures as prognostic indicators [638].\\nIn patients with HCC undergoing immunotherapy,\\nsubstantial evidence demonstrates significant associa-\\ntions between fecal microbiome composition and pa-\\ntients' OS and PFS [637]. Specifically,Lachnoclostridium\\nenrichment coupled withPrevotella 9depletion correlates\\nTABLE 8 (Continued)\\nApplication\\ndirection Tumor type Key microbes/metabolites Key roles Clinical application strategies References\\nPan‐Cancer High butyrate/propionate levels Suppresses CTLA ‐4 efficacy (p = 0.01), promotes\\nTreg expansion.\\nLow‐fiber diet to reduce SCFAs and\\nenhance immunotherapy response.\\n[643]\\nAdverse effects CRC Escherichia coli β‐glucuronidase Converts irinotecan to toxic metabolites. β‐glucuronidase inhibitors (such as nicasyn)\\nto reduce intestinal toxicity.\\n[644]\\nMelanoma Bifidobacterium Supplementation reduces colitis incidence. Probiotic formulations to support\\nimmunotherapy and reduce toxicity.\\n[47]\\nRadiation\\nDermatitis\\nReduced skin microbiome\\ndiversity\\nPost‐radiation dysbiosis exacerbates\\ninflammation.\\nTopical probiotic ointments to restore skin\\nbarrier.\\n[645]\\nAbbreviations: CTLA‐4, cytotoxic T lymphocyte associate protein‐4; FMT, fecal microbiota transplantation; HNSCC, head and neck squamous cell carcinoma; IL‐17, Interleukin‐17; LUAD, lung adenocarcinoma; OS,\\noverall survival; PD‐1, programmed death 1; PD‐L1, programmed cell death ligand 1; RFS, recurrence‐free survival.\\nMICROBIOME IN CANCER | 43 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 43, 'page_label': '44'}, page_content=\"with improved OS, while patients harboring favorable\\nmicrobial signatures exhibit extended PFS [ 637]. En-\\nrichment of Bacteroides dorei and Parabacteroides dis-\\ntasonis in NSCLC patients treated with ICIs significantly\\ncorrelates with prolonged OS, whereas abundance of\\nChaetosphaeriales, Cortinarius davemallochii, and Helo-\\ntiales associates with shorter OS. These findings illumi-\\nnate the complex interplay between the microbiome and\\ntherapeutic efficacy in cancer patients [ 639]. Conse-\\nquently, the broad application of microbiome analysis in\\ntumor prognosis prediction, while elucidating the re-\\nlationships between specific microbial taxa and clinical\\noutcomes, provides novel avenues for developing\\nprecision oncology approaches for cancer patients.\\nMicrobial metabolites and prognosis\\nMicrobial metabolites play crucial roles in modulating\\ntumor progression and treatment outcomes. Investiga-\\ntions have identified the bacterial metabolite galantha-\\nminone as a biomarker for predicting outcomes in HCC\\npatients treated with ICIs. It provides novel insights for\\nclinical applications of microbial metabolites in tumor\\nprognosis assessment [640]. Further studies have dem-\\nonstrated that fecal microbiota and their metabolites,\\nespecially bile acids, are closely associated with clinical\\nefficacy and prognosis in HCC patients undergoing\\nimmunotherapy [637]. These observations substantiate\\nthe prognostic and predictive value of gut microbial\\ncommunities and their metabolic derivatives in predict-\\ning ICI therapeutic responses in HCC patients. A study\\nhas revealed that the gut microbial metabolite butyrate\\nplays a pivotal role in modulating CD8\\n+ T cell func-\\ntionality and phenotype, significantly potentiating anti‐\\nPD‐1 immunotherapy efficacy. This important finding\\nestablishes butyrate as a potential prognostic biomarker\\nfor enhancing anti‐tumor immune responses [641]. Col-\\nlectively, microbial metabolites, as functional mediators\\nof the microbiome, demonstrate potential for integration\\ninto tumor prognostication frameworks and therapeutic\\nresponse prediction.\\nMicrobiome and tumor treatment efficacy\\nThe relationship between the microbiome and tumor\\ntreatment efficacy has been comprehensively investi-\\ngated. The composition and abundance of the micro-\\nbiome closely correlate with treatment outcomes while\\nsimultaneously exerting significant impacts on tumor\\ntherapy resistance. Furthermore, perturbations in mi-\\ncrobiome structure and function significantly contribute\\nto treatment‐related toxicities observed during oncologic\\ninterventions.\\nMicrobiome and drug resistance\\nThe contribution of tumor‐associated microbiota to che-\\nmoresistance mechanisms has emerged as a critical focus\\nin translational oncology research. Studies have con-\\nfirmed that colonizing bacteria present in the PC\\nmicroenvironment can induce gemcitabine resistance\\nand attenuate its cytotoxic efficacy through expression of\\nspecialized cytidine deaminase enzymes [642]. Further\\nresearch has found that antibiotic therapy can effectively\\ninhibit this drug inactivation pathway, consequently\\nrestoring gemcitabine's therapeutic efficacy [ 642].\\nBeyond gemcitabine, the intestinal microbiota demon-\\nstrates the capacity to mediate resistance to diverse\\nchemotherapeutic agents, including cyclophosphamide,\\n5‐fluorouracil (5‐FU), and oxaliplatin [653]. However,\\nhigh concentrations of SCFAs (butyrate and propionate)\\nin the microenvironment have demonstrated significant\\nassociations with resistance to CTLA‐4 blockers [643].\\nMicrobiome and adverse effects\\nInvestigating the interplay between the microbiome and\\ntumor therapy outcomes (including efficacy and toxicity)\\nhas emerged as a central paradigm in cancer research.\\nExtensive research has confirmed that gut microbiota\\ndirectly modulates the efficacy of chemotherapeutic agents\\nvia metabolic regulation. Specifically,E. coli converts ir-\\ninotecan into toxic metabolites throughβ‐glucuronidase\\nexpression, resulting in severe diarrhea. A clinical investi-\\ngation has found that co‐administration ofβ‐glucuronidase\\ninhibitors significantly attenuates intestinal epithelial\\ndamage [644]. Additionally,E. colipossesses nitroreductase\\nactivity that bioactivates the prodrug CB 1954, enhancing\\nits cytotoxic effects while concurrently compromising\\ngemcitabine's anti‐tumor activity, resulting in increased\\ntumor burden and diminished survival outcomes [46]. In\\ntumor immunotherapy, perturbations in intestinal micro-\\nbial homeostasis significantly exacerbate immune‐mediated\\nadverse events. For instance, antibiotic therapy worsens\\nanti‐CTLA‐4t h e r a p y‐induced colitis, whereasBifidobacter-\\nium supplementation mitigates intestinal inflammatory\\npathology through suppression of pro‐inflammatory cyto-\\nkine cascades [47, 654]. Cutaneous microbiome perturba-\\ntions strongly correlate with treatment‐related toxicity.\\nSignificant reductions in cutaneous microbial diversity and\\nalterations in taxonomic proportions during radiotherapy\\nor EGFR inhibitor treatment combined with chemotherapy\\n44 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 44, 'page_label': '45'}, page_content='potentiate localized inflammatory response. This condition\\nmay precipitate adverse effects including radiation derma-\\ntitis, palmar–plantar erythrodysesthesia syndrome, and\\npapulopustular eruptions [645, 655]. Notably, specific\\nmicrobial characteristics demonstrate utility as predictive\\nbiomarkers for treatment‐associated toxicities. For example,\\nin metastatic RCC patients treated with vascular en-\\ndothelial growth factor tyrosine kinase inhibitors (VEGF‐\\nTKI), the elevated abundance ofBacteroides spp.s h o w sa\\nsignificant positive correlation with diarrhea incidence,\\nwhile Prevotella spp. demonstrates a protective effect [656].\\nThese insights suggest that precision microbiome modula-\\ntion strategies, including microbial transplantation or\\ncombination with specific enzyme inhibitors, may consti-\\ntute innovative approaches to optimize the safety window\\nof oncologic interventions.\\nMICROBIOME AND TUMOR\\nTHERAPY\\nMicrobiome and chemotherapy\\nThe relationship between microbiome and tumor therapy,\\nwith particular emphasis on chemotherapeutic modalities,\\nhas emerged as a rapidly expanding research domain.\\nBacteria, viruses, and fungi all significantly impact\\nchemotherapy efficacy, while chemotherapeutic agents\\nsimultaneously affect microbiome composition and func-\\ntion. Furthermore, microbiome alterations may mediate\\ntoxic effects during chemotherapy, impacting therapeutic\\ntolerability and health ‐related quality of life. Conse-\\nquently, microbiome modulation strategies to enhance\\nchemotherapy efficacy while attenuating treatment ‐\\nlimiting toxicities represent an emerging direction in\\ncancer therapy. As illustrated in Figure5,t h i sf r a m e w o r k\\nencompasses the bidirectional regulatory networks\\nbetween microbial ecosystems and chemotherapeutic\\nagents, including influences exerted by bacterial, viral, and\\nfungal constituents, chemotherapy‐induced alterations in\\nthe microbiome, microbe‐mediated toxicity mechanisms,\\nand microbiome‐based intervention strategies.\\nImpact of the microbiome on chemotherapy\\nRole of bacteria: The regulatory influence of microbiota\\non antineoplastic agent pharmacodynamics, particularly\\nintestinal microbial communities, has emerged as a\\ncentral focus in translational oncology research. Certain\\nbacteria, such as Akkermansia muciniphila , enhance\\nchemotherapy efficacy [657]. In CRC, the intestinal mi-\\ncrobiota closely relates to chemotherapeutic efficacy and\\nresistance. F. Nucleatum can induce chemotherapy\\nresistance and reduce the therapeutic effect [165–167],\\nwhile butyric acid and its derivatives can alleviate the\\nchemotherapy resistance induced byF. Nucleatum[165].\\nThe microbial ‐derived butyrate, beyond glucose\\nmetabolism inhibition, targets the G protein ‐coupled\\nreceptor 109a ‐AKT signaling pathway and enhances\\nchemotherapeutic anti‐tumor effects [658]. In PC,Gam-\\nmaproteobacteria enhance chemotherapeutic agent\\ninactivation, including gemcitabine, substantially\\ndiminishing therapeutic efficacy [642, 659]. However,\\ncertain probiotics or microbiota metabolites may benefit\\nCRC treatment. For example, Lactobacillus plantarum‐\\nderived metabolites augment butyrate‐mediated tumor\\nsuppression and reverse multiple drug resistance [660].\\nAdditionally, specific oral bacterial species diminish\\nchemotherapeutic efficacy while simultaneously exacer-\\nbating treatment‐induced oral mucositis [456]. In sum-\\nmary, bacterial effects on chemotherapeutic drug\\nmetabolism exhibit remarkable complexity, playing cru-\\ncial roles in tumor chemotherapy.\\nRole of viruses: Viruses, particularly bacteriophages,\\nhave emerged as remarkable mediators of chemo-\\ntherapeutic efficacy. Experimental evidence demon-\\nstrates that engineered bacteriophages conjugated with\\nchemotherapeutic‐loaded nanoparticles significantly\\npotentiate cytotoxic activity against malignant cells [661].\\nAdditionally, several studies have hypothesized that\\nHPV‐positive HNSCC exhibits better responses to radio-\\ntherapy [662, 663]. Therefore, bacteriophages represent\\npromising biological vectors for modulating microbiome‐\\nmediated chemotherapeutic metabolism, offering\\nremarkable application prospects for enhancing anti-\\nneoplastic efficacy.\\nRole of fungi: Fungal communities have emerged as\\ncritical mediators of chemotherapeutic pharmaco-\\nkinetics and treatment response through diverse meta-\\nbolic and immunological mechanisms. Studies show\\nthat Candida tropicalis (C.tropicalis) is closely associ-\\nated with CRC and promotes chemotherapy resistance\\nthrough dual mechanisms, including lactate production\\nand suppression of MLH1 expression. In this process,\\nlactate, as a critical signaling metabolite, regulates\\nMMR protein expression levels through activation of the\\nGPR81‐cAMP‐\\nPKA‐CREB signaling pathway, enhan-\\ncing tumor cell chemoresistance [664]. To further elu-\\ncidate fungal mechanisms in chemotherapeutic\\nresponses, investigators employed amphotericin B to\\nselectively deplete the mycobiome, demonstrating that\\nfungal ablation significantly enhanced gemcitabine\\ncytotoxicity [ 219]. These findings reveal the critical\\ncontribution of fungi to chemotherapeutic agent\\nmetabolism and treatment efficacy, identifying potential\\nMICROBIOME IN CANCER | 45 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 45, 'page_label': '46'}, page_content=\"interventional targets for novel adjunctive strategies to\\noptimize chemotherapeutic outcomes.\\nEffect of chemotherapy on the microbiome\\nChemotherapeutic interventions induce profound and\\nheterogeneous alterations in microbial communities, ex-\\nhibiting notable variabilityacross malignancy types and\\nchemotherapeutic agents. In HNSCC patients, chemo-\\ntherapy and radiotherapy significantly alter the salivary\\nmicrobiome, evidenced by substantially increasedCandida\\nand reduction in overall bacterial and fungal diversity\\n[665]. In CRC patients, different chemotherapy regimens\\n(including FOLFIRI[fluorouracil, leucovorin, and\\nirinotecan] and XELOX[capecitabine and oxaliplatin])\\ninduce significant taxonomic restructuring of intestinal\\nmicrobial communities, with characteristic alterations in\\nbacterial and fungal population dynamics [666]. PC pa-\\ntients treated with different chemotherapeutic agents\\n(gemcitabine and paclitaxel) exhibit differentiable patterns\\nof intestinal microbiome perturbation [667, 668]. Addi-\\ntional chemotherapeutic agents, including pyrimidine\\nanalogs (5 ‐fluorouracil) and platinum compounds\\n(oxaliplatin), similarly induce characteristic microbial\\ncompositional shifts [669, 670]. 5‐FU treatment induces\\nsignificant restructuring of intestinal microbial communi-\\nties, decreasing abundance of Streptococcus spp.a n d\\nFIGURE 5 The role of microbiome in chemotherapy. This figure illustrates four key aspects of microbiome‐chemotherapy interactions.\\nRegulation of chemoresistance (upper left): Various microorganisms, includingF. nucleatum, Gammaproteobacteria, Lactobacillus, Phage,\\nand C. tropicalis, influence drug resistance through different mechanisms. Mediation of chemotoxicity (upper right): Specific\\nmicroorganisms (Fusobacteria, Proteobacteria, Bifidobacterium, and Lactobacillus) influence chemotherapy side effects, including gut\\ninflammatory responses, diarrhea, and weight loss. Additionally, decreases in specific microbial populations may contribute to\\nchemotherapy‐induced peripheral neuropathy. Alteration of species diversity (lower left): This section demonstrates different chemotherapy\\ntreatments' effects on microbial populations, including FOLFIRI/XELOX in CRC, radiation with platinum‐based chemotherapy in head and\\nneck squamous cell carcinoma, and combinations of gemcitabine and paclitaxel in PC. Additional chemotherapeutic agents contributing to\\nmicrobial community alterations include 5‐Fluorouracil, capecitabine, methotrexate, bleomycin, and paclitaxel. Improvement of\\nchemotherapy efficacy (lower right):Akkermansia muciniphila influences tumor response through UDAC and butyrate production,\\nsubsequently inhibiting tumor glycolysis. Furthermore, urolithin A enhances chemosensitivity, whileLactobacillus species exhibit tumor\\ngrowth inhibition. (Red text for anti‐chemoresistance; Black text for pro‐chemoresistance). CRC, colorectal cancer;C. tropicalis, Candida\\ntropicalis; FOLFIRI, folinic acid (leucovorin), fluorouracil (5‐FU), and irinotecan; UDAC, ursodeoxycholic acid; XELOX, capecitabine\\n(Xeloda) and oxaliplatin.\\n46 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 46, 'page_label': '47'}, page_content=\"Bacteroides spp. while increasing enrichment ofClostrid-\\nium hathewayi and Lachnospiraceae [671]. Gemcitabine\\nsignificantly decreases the proportion of Firmicutes and\\nBacteroidetes [667]. Additionally, capecitabine, methotrex-\\nate, bleomycin, and paclitaxel modulate the composition\\nand abundance of gut microbiome [671]. Furthermore,\\nchemotherapeutic interventions exert modulatory effects\\nbeyond intestinal microbialcommunities, significantly in-\\nfluencing intratumoral microbiome composition and eco-\\nlogical diversity. Comprehensive analyses reveal significant\\nreductions in intratumoral bacterial diversity and charac-\\nteristic shifts within the post‐treatment TME [548, 672].\\nThis microbial community restructuring demonstrates\\nsubstantial correlations with therapeutic response param-\\neters and treatment‐associated adverse effects profiles.\\nMicrobiome‐mediated toxicities of\\nchemotherapy\\nAccumulating evidence establishes the microbiome as a\\ncritical mediator of chemotherapy ‐induced toxicities.\\nDuring irinotecan administration, gut microbial struc-\\nture experiences profound alteration, characterized by\\ndecreased microbial diversity with elevated proportions\\nof Fusobacteria and Proteobacteria phyla. This dysbiosis\\npositively correlates with intestinal inflammatory\\nresponses, potentially exacerbating chemotherapy ‐\\ninduced toxicity [673]. Studies demonstrate that specific\\nbacterial strains (including Bifidobacterium and Lacto-\\nbacillus) significantly alleviate irinotecan‐induced weight\\nloss and diarrhea, further confirming the microbiome's\\ncritical role in modulating chemotherapy ‐induced\\nadverse effects [674]. Additionally, Shen et al. demon-\\nstrated that transient intestinal microbial depletion\\nthrough targeted antibiotic intervention significantly at-\\ntenuates oxaliplatin‐induced peripheral neuropathy in\\npreclinical models, providing mechanistic evidence for\\nunderstanding gut microbiome's role in oxaliplatin tox-\\nicity [675]. These findings elucidate the microbiome's\\nrole in modulating chemotherapeutic agent efficacy and\\nreveal its important mechanisms in mediating chemo-\\ntherapeutic toxicity, providing new research directions\\nfor developing precision microbiome‐targeted interven-\\ntions to minimize therapeutic adverse effects [676].\\nMicrobiome modulation strategies to improve\\nchemotherapy outcomes\\nMicrobiome‐based therapeutic interventions demon-\\nstrate considerable potential for enhancing chemo-\\ntherapeutic efficacy. Accumulating evidence indicates\\nthat intestinal microbial communities significantly\\ninfluence chemotherapeutic outcomes through both\\ndirect effects and metabolite modulation. Specifically,\\ncommensal bacteria with immunomodulatory properties\\nlike Akkermansia muciniphila significantly potentiate\\nchemotherapeutic efficacy [ 657]. Additionally, gut\\nmicrobiota‐produced metabolites such as butyrate and\\nursodeoxycholic acid (UDCA) enhance chemo-\\ntherapeutic efficacy through multiple mechanisms,\\nincluding glucose metabolism inhibition, specific sig-\\nnaling pathway modulation, and intestinal microbial\\ncommunity reconfiguration [677]. Metabolites like ur-\\nolithin A can enhance cancer cell chemosensitivity and\\nameliorate chemotherapy ‐induced systemic toxicities\\n[678]. Furthermore, microbiome ‐targeted modulation\\ninfluences gemcitabine metabolism, generating syner-\\ngistic therapeutic effects in specific cancer types [642].\\nExperimental studies demonstrate that cisplatin com-\\nbined with Lactobacillus supplementation yields en-\\nhanced therapeutic responses in murine pulmonary\\ncarcinoma models, manifesting as significant tumor\\ngrowth inhibition and prolonged survival intervals. The\\noutcome closely relates to alterations in oncogene ex-\\npression profiles and notable augmentation of anti‐tumor\\nimmune responses [ 648]. In summary, microbiome\\nmodulation strategies provide promising adjunctive\\ntherapeutic approaches to enhance chemotherapeutic\\nefficacy. Through optimizing microbial communities and\\ntheir metabolic outputs, these strategies may offer\\npotential for significantly improved therapeutic out-\\ncomes while concurrently mitigating treatment ‐\\nassociated adverse events, ultimately enhancing quality\\nof life metrics for oncology patients.\\nMicrobiome and radiotherapy\\nHost‐associated microbial communities demonstrate signif-\\nicant bidirectional interactions with radiotherapeutic out-\\ncomes across diverse malignancies. Microbial ecosystems\\ninfluence tumor radiosensitivity while concurrently, ioniz-\\ning radiation significantly perturbs microbial community\\nstructures. Additionally, the microbiome may mediate\\nradiotherapy's toxicities, potentially exacerbating treatment‐\\nrelated burdens in patients.\\nWithin the interaction between radiotherapy and the\\nmicrobiome, distinct hierarchical patterns emerge re-\\ngarding the clinical relevance of various bacteria, viruses,\\nand fungi. Bacteria occupy a predominant position due to\\ntheir extensive involvement in regulating radiotherapy\\nsensitivity and mediating radiation ‐induced toxicities.\\nFor example, Lactobacillus iners modulates CC and\\nBacteroides vulgatus influences rectal cancer response to\\nMICROBIOME IN CANCER | 47 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 47, 'page_label': '48'}, page_content=\"radiotherapy [436, 679], with targeted probiotic inter-\\nventions already demonstrating clinical efficacy for mi-\\ntigating radiation ‐induced enteropathy [ 680]. Among\\nviruses, HPV demonstrates a well ‐established role in\\nHNSCC radiotherapy sensitivity. It induces im-\\nmunomodulation significantly enhancing control rates,\\nthough its effects remain restricted to virus‐associated\\nmalignancies [662]. Fungal research primarily focuses on\\nthe bidirectional regulation between intestinal com-\\nmensal fungi and the post‐radiotherapy immune micro-\\nenvironment, though mechanistic understanding and\\nclinical translation remain insufficient. In summary,\\nbacterial research is the most systematic and has the\\ngreatest value in clinical translation in radiotherapy.\\nViral factors provide established clinical guidance in\\nspecific malignancy subtypes, while fungal contributions\\nto radiotherapeutic outcomes warrant further mecha-\\nnistic investigation.\\nConsequently, the development of targeted micro-\\nbiome modulation strategies to potentiate radio-\\ntherapeutic efficacy while attenuating radiation‐induced\\nnormal tissue complications represents an emerging\\nparadigm in precision radiation oncology. As illustrated\\nin Figure 6, the microbiome ‐radiotherapy interaction\\nmanifests in four key aspects: microbial community\\ncontributions to tumor radiosensitivity modulation,\\nradiotherapeutic effects on microbial composition,\\nmicrobiome‐mediated radiotherapy ‐induced toxicities,\\nand microbiome modulation intervention to improve\\nradiotherapeutic outcomes.\\nEffect of microbiome on sensitivity to\\nradiotherapy\\nThe regulation of tumor radiosensitivity by the micro-\\nbiome has emerged as a critical research area. Preclinical\\ninvestigations utilizing murine models demonstrate that\\nradiotherapeutic efficacy is determined by an intricate\\ninterplay between intrinsic tumor characteristics and\\nintestinal microbial community composition, encom-\\npassing bacterial and fungal constituents [681]. Across\\ndiverse cancer types, gut microbes significantly modulate\\nradiotherapy efficacy and influence radiotherapy ‐\\ninduced adverse effects [681]. For instance,Lactobacillus\\niners produces L‐lactic acid, which is significantly asso-\\nciated with unfavorable clinical outcomes and induces\\nradiotherapy resistance in CC cells [436]. Additionally,\\nintestinal microbial dysbiosis may constitute an impor-\\ntant mechanism of primary radiotherapeutic resistance\\nthrough impairing anti ‐tumor immune responses via\\ninhibition of antigen presentation and effector T cell\\nfunction [682].\\nThe microbiome's clinical significance is particularly\\nprominent in rectal cancer treatment. Specific microbes\\nand metabolic pathway indicators serve as biomarkers\\nfor predicting efficacy and adverse effects in rectal cancer\\npatients undergoing neoCRT [683]. A prospective longi-\\ntudinal investigation confirmed that there are remark-\\nable differences in the microbiome between responders\\nand non‐responders to treatment. For example, butyrate‐\\nproducing bacteria dominated in radiotherapy‐sensitive\\nindividuals, whereas Fusobacterium and Coriobacter-\\niaceae predominated in non‐responders [684]. Recent\\ninvestigations have revealed that Bacteroides vulgatus\\ninhibits rectal cancer radiosensitivity. Through mod-\\nulating the nucleotide biosynthesis pathway,Bacteroides\\nvulgatus substantiates the critical regulatory role of spe-\\ncific bacterial taxa in regulating radiotherapeutic\\nresponses [ 679]. Collectively, diverse bacterial taxa\\ndemonstrate remarkable modulatory effects on tumor\\nradiosensitivity. Elucidation of these regulatory pathways\\nwill provide a basis for developing novel microbiome‐\\ntargeted strategies to enhance tumor radiotherapy\\nefficacy.\\nViral components of the microbiome have garnered\\nincreasing attention in tumor radiosensitivity regulation.\\nStudies show that HPV‐positive patients with HNSCC\\nexhibit higher radiotherapeutic sensitivity [662]. Clinical\\ndata confirms that HPV‐positive HNSCC patients dem-\\nonstrate significantly higher local control rates following\\nradiotherapy, representing a principal factor of improved\\nOS [685]. Further mechanistic studies have revealed that\\nHPV‐positive HNSCC patients exhibit significantly pro-\\nlonged OS following radiotherapy compared to HPV ‐\\nnegative cohorts [685]. These favorable prognostic dis-\\ntinctions appear related to HPV‐mediated alterations in\\nHNSCC.\\nRecent comprehensive investigations have examined\\nmycobiome‐\\nmediated regulatory mechanisms influen-\\ncing tumor response to radiotherapy. Intestinal fungal\\ncommunities significantly modulate post ‐radiotherapy\\nanti‐tumor immune responses in murine models of BC\\nand melanoma, demonstrating regulatory patterns con-\\ntrast with those observed in bacterial communities [686].\\nSpecifically, antifungal interventions enhance tumor\\nradiotherapy efficacy by eliminating commensal fungi,\\nwhile triggering a decrease in commensal bacterial\\nabundance is associated with poor efficacy. Furthermore,\\nelevated expression of Dectin‐1 in BC is associated with\\nunfavorable survival outcomes. In murine radiotherapy\\nmodels, Dectin‐1 demonstrated essential functionality for\\nmediating commensal fungi‐induced biological effects.\\nIt provides that molecular mechanistic insights into\\nthe mycobiome contribute to radiotherapeutic response\\nmodulation [ 686], further establishing fungal\\n48 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 48, 'page_label': '49'}, page_content='communities as critical factors in tumor radiotherapy. In\\nsummary, fungal components significantly influence\\nradiotherapeutic response.\\nEffects of radiotherapy on the microbiome\\nRadiotherapy exerts complex and multifaceted effects on\\nhost‐associated microbial ecosystems, with differential\\nimpacts observed across diverse anatomical niches\\nand microbial kingdoms. In HNSCC patients, salivary\\nmicrobial communities undergo substantial compositional\\nrestructuring following radiotherapy, characterized by\\nsubstantial reductions in taxonomic diversity and concur-\\nrent enrichment ofCandida [665]. Similarly, oral micro-\\nbiome composition undergoes reconfiguration during\\nradiotherapy in OSCC patients. Whileα‐diversity remains\\nstatistically unchanged between pre‐treatment and imme-\\ndiate post‐treatment timepoints,β‐diversity reveals signifi-\\ncant divergence at 6 months post ‐radiotherapy [441].\\nSpecific species includingCorynebacterium, Haemophilus,\\nVeillonella,a n dActinomyces decrease in OSCC patients,\\nwhile concurrent enrichment ofSelenomonas and Myco-\\nplasma is observed [ 441]. Investigations confirm\\nFIGURE 6 The role of microbiome in radiotherapy. This figure illustrates four key aspects of microbiome‐radiotherapy interactions.\\nRegulation of radioresistance (upper left): Key mechanisms related to bacteria include L‐lactic acid production, affecting antigen presentation\\nand effector T cell function, elevated abundance ofFusobacterium and Coriobacteriaceae, and nucleotide biosynthesis modulation by\\nBacteroides vulgatus. HPV enrichment or commensal fungi elimination improves radiotherapy sensitivity, whereas high Dectin‐1e x p r e s s i o no f\\nfungi contributes to resistance mechanisms. Mediation of radiotoxicity (upper right): Microbes includingProteobacteria, pathogenic\\nStaphylococcus aureustrigger skin inflammation.Lactobacillus acidophilusplus Bifidobacterium supplementation correlates with reduced\\ndiarrhea and abdominal pain. Radiation‐induced decreases in microbial populations are associated with reduced homeostatic cytokine levels.\\nAlteration of species diversity (lower right): Distinct microbial community changes occur across different anatomical sites. Gynecological\\ncancer patients after radiotherapy show intestinalBifidobacterium and Clostridium reduction concurrent withBacteroides and Enterococcus\\nincreases. In OSCC,β‐diversity shows differences, particularly inCorynebacterium, Haemophilus, Veillonella, Prevotella melaninogenica,\\nActinomyces,a n dMycoplasma populations. Additionally,Candida and C. albicansincreased following radiotherapy. Only gut microbiome\\nβ‐diversity alterations were found in prostate cancer patients during radiotherapy. Likewise, gut microbiota diversity in CRC patients\\nundergoing radiotherapy shows elevation with significant enrichment ofClostridium IV, Roseburia,a n dPhascolarctobacterium.I m p r o v e m e n t\\nof radiotherapy efficacy (lower left): Antibiotic pre‐treatment suppresses inflammatory responses, TLR4/MyD88/NF‐κB pathway signaling, and\\nSmad‐3/pSMAD pathway activation to improve radiotherapy efficacy. Furthermore,Lactobacillus rhamnosus GGpromotes intestinal crypt\\nsurvival while inhibiting epithelial cell apoptosis. (Red text for anti‐radioresistance; Black text for pro‐radioresistance). OPC, oropharyngeal\\ncancer; OSCC, oral squamous cell carcinoma; pSMAD, phosphorylated SMAD; TLR4, Toll‐like receptor 4.\\nMICROBIOME IN CANCER | 49 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 49, 'page_label': '50'}, page_content='significant dynamics in oral microbial communities\\nduring and following OSCC radiotherapy, with notable\\ndepletion of Prevotella melaninogenica abundance in\\npost‐radiotherapy samples [687]. Conversely, C. albicans\\ndemonstrates notable expansion following radiotherapy in\\nHNSCC [688].\\nIn gastrointestinal and pelvic malignancies, radio-\\ntherapy induces profound perturbations in intestinal\\nmicrobial ecosystems. Pelvic radiotherapy for gynecologi-\\ncal malignancies induces substantial intestinal dysbiosis,\\ncharacterized by selective depletion ofBifidobacteriumand\\nClostridium, and remarkable enrichment in Bacteroides\\nand Enterococcus [660]. A prospective study of PCa pa-\\ntients has found substantial alterations in gut microbiome\\nβ‐diversity without corresponding changes inα‐diversity\\nduring radiotherapy [ 689]. Ferreira et al. similarly\\nobserved significantly reduced gut microbiota diversity in\\npatients undergoing radiotherapy, characterized by\\ndepletion ofClostridium perfringensalongside enrichment\\nof Clostridium IV, Roseburia,a n dPhascolarctobacterium,\\nwhich may contribute to the pathogenesis of radiation‐\\ninduced enteropathy [690].\\nNotably, radiotherapeutic interventions modulate not\\nonly microbial community structure and ecological diver-\\nsity but also alter the bacterial functional capacity and\\nvirulence expression. For example, neoadjuvant radio-\\ntherapy can convert inoculatedPseudomonas aeruginosa\\ninto a highly pathogenic phenotype that destroys anasto-\\nmotic sites [ 691]. During HPV ‐associated OPC radio-\\ntherapy, substantial reductions in microbial α‐diversity\\nwith concurrent alterations in specific taxonomic abun-\\ndances have been demonstrated [692]. These radiation‐\\ninduced microbial perturbations reflect both direct effects\\non host physiology and clinical prognosis of patients.\\nMicrobiome‐mediated radiotherapy toxicities\\nThe microbiome represents a critical role in radiation‐\\ninduced normal tissue complication pathogenesis and pro-\\ngression. Clinical investigations demonstrate that radio-\\ntherapeutic intervention induces significant taxonomic\\nshifts within microbial communities. These alterations are\\nstrongly, negatively correlated with homeostatic cytokine\\nexpression in the intestinal mucosa, potentially exacerbat-\\ning radiotherapy‐induced mucosal injury [693]. In cutane-\\nous tissues, radiation‐induced dermatitis demonstrates a\\nmarked association with diminished microbial diversity and\\ncharacteristic taxonomic alterations, particularly elevated\\nProteobacteria/Firmicutesratios and pathogenic S. aureus\\nproliferation [645]. However, probiotic interventions offer\\nnovel therapeutic strategies to alleviate radiotherapy‐related\\ntissue complications. Studies confirm that probiotics effec-\\ntively attenuate multiple radiation ‐induced toxicities\\nthrough downregulating pro‐inflammatory cytokines [680].\\nSpecifically, clinical studies have demonstrated that the\\nprophylactic and therapeutic administration ofLactobacil-\\nlus acidophilusplus Bifidobacteriumsupplements can alle-\\nviate radiation‐induced gastrointestinal toxicity symptoms,\\nincluding diarrhea and abdominal pain. That concurrently\\nimproves health‐related quality of life during radiotherapy\\n[680]. In summary, microbial communities represent criti-\\ncal mediators of radiotherapy‐induced toxicities, establish-\\ning a crucial theoretical basis for developing novel\\ntherapeutic strategies.\\nMicrobiome modulation strategies to improve\\nradiotherapy outcomes\\nEmerging evidence demonstrates that targeted micro-\\nbiome manipulation strategies hold significant clinical\\npotential in improving radiotherapy outcomes.Lactoba-\\ncillus rhamnosus GGeffectively attenuates radiotherapy‐\\ninduced epithelial damage by enhancing crypt survival\\nand reducing epithelial cell apoptosis [ 694]. Further\\nstudies show that altering microbiota composition\\nthrough antibiotic interventions significantly modulates\\nradiotherapy outcomes. For example, preclinical studies\\nshowed that antibiotic pretreatment before radiotherapy\\npartially reversed microbiota dysbiosis and accelerated\\nrecovery from radiation‐induced intestinal damage [690].\\nThis process is associated with multiple factors, including\\nthe TLR4/MyD88/NF‐κB signaling pathway, macrophage\\npolarization, and inflammatory mediator regulation,\\ndemonstrating novel mechanisms of microbiota modu-\\nlation in improving radiotherapy efficacy [ 695, 696].\\nAdditionally, prophylactic antifungal interventions\\nappear more beneficial, as commensal fungal species\\nmay influence the TIME through pattern recognition\\nreceptor‐mediated interactions with macrophages and\\nT cells, potentially enhancing radiotherapy efficacy [686].\\nMicrobiome and immunotherapy\\nHost‐associated microbial communities represent critical\\ndeterminants of immunotherapeutic efficacy. They are\\nstrongly associated with ICI efficacy and significantly\\ninfluence CAR‐T cell therapeutic outcomes and cancer\\nvaccine efficacy. Modulation of the microbiome shows\\ntherapeutic potential for enhancing immunotherapy effi-\\ncacy and strengthening anti‐tumor immune responses.\\nFigure 7 summarizes the principal mechanistic pathways\\n50 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 50, 'page_label': '51'}, page_content='through which the microbiome modulates tumor immu-\\nnotherapeutic outcomes, including bacterial and fungal\\neffects on ICI efficacy, interactions between the micro-\\nbiome and CAR‐T cell therapy, and microbiome‐mediated\\neffects in vaccine therapy. Collectively, these mechanisms\\nprovide a theoretical foundation and practical direction for\\nmicrobiome modulation strategies to optimize tumor\\nimmunotherapy.\\nMicrobiome and immune checkpoint inhibitors\\nImmunomodulatory effects of bacteria: The gut micro-\\nbiome exerts a significant influence on ICI responses\\nthrough diverse immunomodulatory pathways,\\nrepresenting a critical focus in contemporary immuno‐\\noncology research. Studies demonstrate that specific\\nbacterial species, includingBacteroides thetaiotaomicron,\\nB. fragilis, and Bifidobacterium, promote CD8\\n+ T cell\\nimmune responses through activation of TLR‐4 signaling\\npathway or adenosine 2A (A2A) receptor signaling\\npathway, consequently enhancing anti ‐tumor im-\\nmunosurveillance and regulating anti ‐tumor immune\\nresponses [66]. Intestinal Lactobacillus johnsonii abun-\\ndance demonstrates remarkable positive correlations\\nwith ICI efficacy. Specifically, indole ‐3‐propionic acid\\n(IPA), synergistically produced through intestinal pro-\\nbiotics Clostridium sporogenes and Lactobacillus johnso-\\nnii, promotes the generation of precursor ‐exhausted\\nCD8\\n+ T cells, thereby enhancing ICI responses in\\nFIGURE 7 The role of microbiome in immunotherapy. This figure illustrates four key aspects of microbiome‐immunotherapy\\ninteractions. Upregulation of immune checkpoint inhibitors (upper left): Specific bacterial species (Faecalibacterium prausnitzii, Bacteroides\\ncaccae, Holdemania filiformis, Dorea formicogenerans, Bifidobacterium longum, andAkkermansia muciniphila) modulate immune responses.\\nB. fragilis and Bacteroides thetaiotaomicron activate TLR4 and A2A receptor signaling pathways.Candida albicans influences PD‐1\\nexpression during pDC activation, whileClostridium sporogenes and Lactobacillus johnsonii enhance precursor‐exhausted CD8+ T cell\\nresponses via increased IPA production. Mediation of CAR‐T therapy (upper right): Decreased microbial diversity correlates with reduced\\nCAR‐T therapy efficacy. Specifically,Bifidobacterium, Ruminococcus, andAkkermansia populations significantly influence CAR‐T treatment\\noutcomes. Improvement of immunotherapy efficacy (lower right):Bifidobacterium species combined with ICIs enhance immune responses\\nthrough DC activation and increased CD8+ T cell infiltration. Additionally, vancomycin treatment enhances CTLA‐4 blockade therapy and\\nT cell activation to show anti‐tumor effects by inducingBacteroidales overexpression with subsequent inosine secretion. Influence in\\nvaccination (lower left): In HPV vaccine‐naïve individuals, specific microbiota (Caldithrix, Nitrospirae, andPrevotella) exhibited enrichment.\\nAdditionally, immunotgerapy including ICIs and CAR‐T shows enhanced efficacy when combined with oncolytic virus vaccines, through\\npromoting T cell activation, proliferation, and cytotoxic efficiency. A2A, adenosine 2A; CTLA‐4, cytotoxic T‐lymphocyte‐associated protein\\n4; DC, dendritic cell; ICIs, immune checkpoint inhibitors; IPA, indole‐3‐propionic acid; pDC, plasmacytoid dendritic cell.\\nMICROBIOME IN CANCER | 51 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 51, 'page_label': '52'}, page_content=\"diverse malignancies including CRC, BC, and mela-\\nnoma [697].\\nClinical investigations su ggest intestinal microbial\\ncommunity is strongly associated with anti ‐tumor ef-\\nf e c t sa c r o s sI C Ic l a s s e s[698–700]. Cohort studies indi-\\ncate that in advanced melanoma, specific intestinal\\nmicrobial signatures, characterized by enrichment of\\nBifidobacterium pseudocatenulatum, Roseburia spp., and\\nAkkermansia muciniphila , demonstrate notable asso-\\nciations with favorable immunotherapeutic response\\nprofiles [701]. In patients with metastatic melanoma,\\nintestinal microbiota enriched with Faecalibacterium\\nand otherFirmicutes tend to show enhanced response to\\nipilimumab treatment [702]. Meanwhile, elevated Bac-\\nteroidetes phylum demonstrates remarkable negative\\ncorrelations with ICI ‐induced colitis incidence and\\nseverity [703]. Additional investigations have confirmed\\nthat metastatic melanoma patients exhibiting favorable\\nr e s p o n s e st oc o m b i n e dI C I s( n i v o l u m a bp l u si p i l i m u -\\nmab therapy) show fecal enrichment of Faecalibacter-\\nium prausnitzii , Bacteroides caccae ,a n d Holdemania\\nfiliformis strains, while patie nts responding to\\npembrolizumab exhibit selective enrichment of Dorea\\nformicogenerans [704].\\nPreclinical investigations suggest that gut microbiome\\ndiversity and composition significantly impact PD ‐1\\nblockade therapy efficacy. Specific strains includingBifi-\\ndobacterium longum, Akkermansia muciniphila,a n dFae-\\ncalibacterium spp. are strongly associated with enhanced\\nanti‐PD‐1 therapeutic efficacy [681]. In summary, the gut\\nmicrobiome exerts a significant immunomodulatory\\ninfluence on ICI therapeutic outcomes through modula-\\ntion of immune cell functionality and regulation of drug\\nmetabolism and effects.\\nImmunomodulatory effects of fungi: Complex\\nimmunoregulatory networks exist between mycobiome\\ncomponents and ICIs, with fungus ‐mediated im-\\nmunomodulation demonstrating particular research\\nvalue and clinical significance. A study exploring the\\nrelationship between the mycobiome and immuno-\\ntherapy response demonstrated that predictive algo-\\nrithms based on fungal taxonomic signatures exhibited\\nsuperior predictive performance (mean AUC of 0.87),\\noutperforming models relying solely on bacterial bio-\\nmarkers. Further studies found that integration of fungal\\nand bacterial taxonomic signatures into predictive mod-\\nels substantially enhanced discriminatory accuracy\\n(AUC = 0.89), strongly establishing fungi's important\\nregulatory role in ICI responses [705]. Additionally, C.\\nalbicans\\ncolonization induces upregulation of PD ‐1\\npathways in monocytes and plasmacytoid DCs (pDCs),\\nand also CTLA‐4 pathways in both CD4\\n+ and CD8+ T\\ncells [706]. These findings provide a scientific foundation\\nfor understanding the contribution of the mycobiome to\\nimmune checkpoint regulation and ICI efficacy.\\nMicrobiome and CAR‐T cell therapy\\nEmerging clinical evidence establishes the microbiome\\nas a critical factor in CAR ‐T cell therapy efficacy\\n[707–712]. Studies show diminished intestinal microbial\\ndiversity indices are strongly and negatively correlated\\nwith CAR‐T cell therapeutic response [ 713]. Specific\\nbacterial taxa abundance, including Bifidobacterium\\nspp., Ruminococcus spp., and Akkermansia spp., dem-\\nonstrates a remarkable positive association with favor-\\nable CAR ‐T cell therapeutic outcomes [ 681]. These\\nfindings provide new research trajectories for elucidat-\\ning microbiome ‐mediated regulation of CAR ‐Tc e l l\\ntherapeutic efficacy. Additionally, OVs as innovative\\ntherapeutic vectors demonstrate notable synergistic\\nanti‐tumor activity when strategically combined with\\nICIs, including CAR ‐T cells and anti ‐PD‐1/PD‐L1 or\\nanti‐CTLA‐4[ 714]. In conclusion, microbial communi-\\nties significantly influence CAR‐T cell efficacy through\\nmultiple immunomodulatory mechanisms, establishing\\nrationale for developing precision microbiome modu-\\nl a t i o nm e t h o d sa sa d j u n c t i v es t r a t e g i e st oc e l l u l a r\\nimmunotherapy.\\nMicrobiome modulation strategies to improve\\nimmunotherapy outcomes\\nAs research advances, microbiome modulation strategies\\nexhibit considerable promise for enhancing the efficacy\\nof cancer immunotherapy. Intestinal commensal bacteria\\nexert particularly significant roles in immunotherapy.\\nStudies show Bifidobacterium combined with PD ‐L1\\ntherapy exhibits significant therapeutic effects on solid\\ntumors through enhancement of DC activity and aug-\\nmentation of CD8\\n+ T cell infiltration [715]. Vancomycin\\nenhances the anti ‐tumor effects of CTLA ‐4 blockade\\ntherapy by inducing the expansion ofBacteroidales [716].\\nAdditionally, the metabolite inosine promotes T cell\\nactivation in the gut, leading to enhanced efficacy of ICIs\\n[717]. Notably, therapeutic strategies specifically target-\\ning distinct microbial populations may provide new\\nresearch directions for improving immunotherapy effi-\\ncacy [171]. In summary, microbiome modulation strate-\\ngies, including microbiome remodeling through\\nantibiotic treatment, probiotic supplementation, or spe-\\ncific microbe ‐targeted therapies, present promising\\nresearch avenues for improving immunotherapy efficacy\\nand mitigating treatment‐related adverse effects.\\n52 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 52, 'page_label': '53'}, page_content='Microbiome and vaccine therapy\\nThe complex interplay between the host microbiome and\\ncancer vaccine efficacy represents an emerging focus in\\ntumor immunology research. Recent studies investigat-\\ning HPV therapeutic vaccines reveal associations\\nbetween cervical microbiome and vaccine ‐induced\\nimmune responses. Research systematically has ana-\\nlyzed cervical microbiomes of patients with high‐grade\\nsquamous intraepithelial lesions (HSIL) before and after\\nadministration of HPV therapeutic vaccination. Results\\nshowed that five bacterial groups, includingCaldithrix,\\nNitrospirae, and Prevotella, exhibited enrichment in vac-\\ncine non‐responders, suggesting these bacterial commu-\\nnities may suppress vaccine‐induced immune responses\\n[718]. These findings provide new insights into the role\\nof the local microbiome in immune responses in patients\\nwith HSIL. Notably, enrichment ofPrevotella may pro-\\nmote persistent HPV infection, thereby potentially con-\\nferring resistance to therapeutic vaccination [ 718].\\nFurthermore, OVs can be combined with anti ‐tumor\\nvaccines to synergistically enhance therapeutic efficacy\\n[714]. In summary, the regulatory mechanisms by which\\nthe microbiome influences vaccine immunotherapy and\\ntheir potential clinical application value require further\\nin‐depth study.\\nMicrobiome and targeted therapies\\nThe microbiome engages in complex bidirectional inter-\\nactions with targeted anti‐tumor therapeutics. It influ-\\nences the efficacy of targeted agents through various\\nmechanisms. Concurrently, alterations in microbiome\\ncomposition are associated with the development of\\nresistance to targeted therapies, with specific microbial\\nspecies potentially contributing to resistance. Integration\\nof microbiome‐modulating interventions with targeted\\ntherapeutic strategies represents innovative approaches\\nfor cancer treatment.\\nMechanisms in microbiome influence the\\nefficacy of targeted drugs\\nThe microbiome influences the efficacy of targeted drug\\ntherapeutic through multiple pathways, with bacterial‐\\nmediated molecular mechanisms emerging as particu-\\nlarly significant mediators of treatment response. Studies\\nindicate that in CRC cells, Leuconostoc mesenteroides\\nregulates the NF‐κB/AKT/PTEN/MAPK signaling path-\\nway to induce apoptosis [719], while P. gingivalis acti-\\nvates the MAPK signaling pathway to promote tumor cell\\nproliferation [720]. BRAF mutations activate the MAPK\\npathway and trigger oncogenic effects, which can be\\neffectively suppressed by BRAF and MEK inhibitors\\n[721]. These findings demonstrate that microorganisms\\ncan affect tumor proliferation and survival via directly\\nmodulating cancer cell signaling networks, potentially\\naltering the efficacy of therapeutic agents that target\\nthese signaling pathways.\\nMicrobiome and targeted therapy resistance\\nThe microbiome appears to play a significant role in the\\ndevelopment of resistance to targeted anti ‐tumor\\ntherapies. Accumulating evidence demonstrates F. nu-\\ncleatum can activate the E‐cadherin/ β‐catenin signaling\\npathway [156], and theβ‐catenin signaling pathway is\\nrelated to the development of resistance to lenvatinib in\\nHCC [168–170]. Therefore, the modulation ofβ‐catenin\\nexpression by F. nucleatum may affect the sensitivity of\\ntumor cells to lenvatinib. Additionally, NaB, a metab-\\nolite produced by specific gut microbes including Ro-\\nseburia cecical and Roseburia intestinalis ,i n h i b i t s\\nexpression of sorafenib ‐targeted miR ‐7641 and miR ‐\\n199. A combination approach using NaB with anti‐miR‐\\n7641 or anti‐miR‐199 enhances apoptotic signaling and\\nreduces the survival of drug‐resistant cells [722]. These\\nfindings suggest modulating the composition and\\nmetabolites of gut microbiota may provide novel\\nintervention strategies to overcome resistance to tar-\\ngeted anti‐tumor agents.\\nTargeted therapeutic strategies for combined\\nmicrobiome interventions\\nResearch advances in integrating microbiome ‐\\nmodulating interventions with targeted therapeutic\\nstrategies have established promising new avenues for\\ncancer treatment. Studies show microbial metabolite\\nbutyrate significantly enhances the therapeutic efficacy\\nof sorafenib in HCC, though its lack of target specificity\\nlimits clinical translation. To overcome this limitation,\\nresearchers developed nanoparticles (NPs) encapsulating\\nboth butyrate and sorafenib, significantly enhancing\\nHCC therapeutic efficacy [723]. Researchers are inten-\\nsively exploring strategies to optimize targeted therapy\\nefficacy through gut microbiota modulation [724]. Hahn\\net al. found antibiotics targetingBacteroides spp. modu-\\nlation improved PFS in patients treated with VEGF‐TKI\\n[725]. Conversely, FMT significantly alleviates diarrhea\\nin metastatic RCC patients treated with TKIs, while\\nsimultaneously promoting favorable alterations in gut\\nMICROBIOME IN CANCER | 53 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 53, 'page_label': '54'}, page_content='microbiome composition [726]. This suggests while an-\\ntibiotics and microbiome modulation offer promising\\nstrategies for improving targeted therapeutic response,\\nspecific microbiota intervention strategies require careful\\nselection to minimize potential adverse effects.\\nMicrobiome and surgery\\nThe microbiome influences the outcome and prognosis\\nof surgical treatment of tumors. Strategic preoperative\\nmicrobiome modulation through targeted antimicrobial\\ntherapy or evidence‐based probiotic administration may\\nenhance therapeutic efficacy and substantially reduce the\\nincidence of postoperative complications. Additionally,\\nsurgical stress affects microbial homeostasis, while post-\\noperative complications are associated with microbiome\\ncomposition and metabolism. Therefore, microbiome\\nmodulation strategies may accelerate postoperative recov-\\nery, mitigate perioperative complications, and optimize\\noutcomes following surgery.\\nPreoperative microbiome preparation\\nTargeted microbiome modulation approaches in colo-\\nrectal surgery, particularly preoperative microbiome\\noptimization protocols, significantly influence long‐term\\nsurgical prognosis. Numerous studies demonstrate colo-\\nrectal anastomotic leak (AL) occurrence is associated\\nwith reduction in intestinal microbial diversity, whereas\\noral antibiotic preparations in elective colorectal surgery\\nsignificantly reduce the incidence of postoperative com-\\nplications, particularly AL risk [727, 728]. These findings\\nunderscore the critical importance of maintaining gut\\nmicrobiota homeostasis in preventing postoperative\\ncomplications. Probiotic supplementation demonstrates\\npromise as a prophylactic measure in preoperative mi-\\ncrobiome modulation. Studies demonstrate probiotic\\ninterventions effectively reduce inflammatory factor lev-\\nels, attenuate chemotherapeutic side effects, prevent\\nsevere diarrhea, reduce postoperative infectious compli-\\ncation incidence, and shorten antibiotic treatment cycles\\n[729]. Accumulating evidence suggests that preoperative\\nprobiotic or synbiotic interventions effectively improve\\nsurgical outcomes in patients with CRC. Additionally,\\npreoperative mechanical bowel preparation (MBP) com-\\nbined with antibiotics has garnered substantial clinical\\ninterest. Large‐scale retrospective analyses have demon-\\nstrated that MBP with oral antibiotics significantly\\nreduces the incidence of postoperative complications,\\nincluding surgical site infection (SSI), bowel obstruction,\\nand AL [ 730]. These findings further substantiate\\npreoperative microbiome optimization strategies, partic-\\nularly evidence ‐based probiotic interventions, in im-\\nproving clinical prognosis following colorectal surgery.\\nEffect of surgical stress on the microbiome\\nSurgical stress effects on body microbiome represent\\nkey research issues in microbiome and surgical therapy.\\nStudies show significantly reduced diversity in fecal\\nmicrobiota of postoperative CRC patients, with signifi-\\ncant compositional differences compared to healthy\\ncontrols and preoperative CRC patients [ 731]. Pre-\\nclinical animal models have confirmed significantly\\naltered gut microbial composition following small\\nbowel surgery, with marked reductions in the relative\\nabundance of Bacteroidetes and Proteobacteria phyla\\n[732]. Further studies systematically evaluating the ef-\\nfects of surgical resection and chemotherapy on intes-\\ntinal microbiota in CRC patients have demonstrated\\nsignificant decreases in obligate anaerobes, tumor ‐\\nassociated microbial signatures, and butyric acid ‐\\nproducing bacteria, with significant increases in\\nopportunistic pathogens when comparing postoperative\\nversus preoperative fecal samples [ 733]. Additionally,\\nhepatectomy can induce gut microbial dysbiosis in\\nHBV‐associated HCC patients, with elevated abundance\\nof Klebsiella emerging as a potential predictive bio-\\nmarker for postoperative liver failure (PHLF) in this\\npatient population [734]. In conclusion, surgical stress\\nmodulating effects on microbiome deserve an in‐depth\\nstudy and attention.\\nPostoperative complications and the\\nmicrobiome\\nCurrent evidence reveals that the complex associations\\nbetween postoperative complications following oncologic\\nsurgery and the patient microbiome are receiving\\nincreasing investigative attention. AL represents one of\\nthe most common life‐threatening postoperative com-\\nplications in patients treated with gastrointestinal resec-\\ntion with primary anastomosis, with its pathogenesis\\nstrongly linked to gut microbiome regulation. Studies\\nfind gut microbiome influences AL development through\\ntwo distinct molecular mechanisms. First, specific gut\\nbacteria, notably Enterococcus faecalis and Pseudomonas\\naeruginosa, dissolve collagen and activate matrix me-\\ntalloproteinases, potentially contributing to AL initiation\\nand progression. Second, bacterial communities demon-\\nstrate protective effects on maintaining anastomotic\\nintegrity and promoting tissue healing [ 653].\\n54 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 54, 'page_label': '55'}, page_content='Experimental studies have confirmedPseudomonas aer-\\nuginosa inoculation significantly increased AL incidence\\nfollowing radiotherapy combined with low colonic\\nanastomosis construction [ 691]. Additionally, patients\\ndeveloping postoperative pulmonary infections following\\nGC surgical treatment exhibited significant alterations in\\nthe functional gene profiles of gut microbes, with en-\\nrichment of potentially pathogenic genera including\\nKlebsiella and Enterobacter [735]. In summary, complex\\nregulatory networks exist between the microbiome\\nand postoperative complications, and comprehensive\\nelucidation of these mechanistic interactions holds\\nsignificance for improving patient prognosis.\\nMicrobiome modulation for postoperative\\nrecovery\\nMicrobiome modulation may play a fundamental role\\nin optimizing postoperative recovery. Clinical studies\\nconfirm that high ‐fat, high ‐cholesterol (HFHC) die-\\ntary patterns significantly exacerbate gut microbiota\\ndysbiosis and subsequently induce intestinal inflam-\\nmatory responses in post ‐cholecystectomy patients,\\nproviding a mechanistic rationale for targeted pro-\\nbiotic intervention strategies to restore microbiome\\nhomeostasis and accelerate postoperative recovery\\n[736]. Specifically, gut microbiota significantly regu-\\nlates surgical incision healing. Studies show gut mi-\\ncrobiota stimulates vagus nerve ‐mediated oxytocin\\nrelease through lactic acid fermentation, subsequently\\npromoting T cell recruitment and accelerating\\nincision ‐healing processes [ 737]. Microbiome modu-\\nlation represents an important therapeutic avenue for\\nenhancing postoperative recovery, and patients may\\nbenefit from more optimized post ‐surgical care pro-\\ntocols through evidence ‐based nutritional interven-\\ntion, strain ‐specific probiotic supplementation, and\\nprecision microbiome ‐targeted therapies.\\nMicrobiome‐based strategies for tumor\\ntherapy\\nMicrobiome‐targeted therapeutic approaches in oncology\\nare being systematically investigated across multiple ex-\\nperimental and clinical platforms with accelerating\\nmomentum. Current microbial intervention approaches\\ninclude probiotic, prebiotic and synbiotic interventions,\\nFMT, precision antibiotic modulation, dietary modifica-\\ntion, microbiome ‐targeted drugs, OVs therapy, en-\\ngineered bacterial therapy, and fungal therapeutic\\nstrategies. These options significantly expand the\\ntherapeutic landscape and offer promising avenues for\\nimproving therapeutic outcomes. Figure 8 provides a\\ncomprehensive mechanistic framework illustrating dual\\napproaches for optimization of host microbiome and\\ntargeted methods with microbes.\\nProbiotic, prebiotic, and synbiotic interventions\\nProbiotics, prebiotics, and synbiotics, as established modu-\\nlators of gut microbial ecology, play fundamental roles in\\nmaintaining intestinal microecological homeostasis. Spe-\\ncifically, probiotics are defined as live microorganisms,\\nprimarily Bifidobacteriumand Lactobacillus.P r e b i o t i c sa r e\\nnon‐digestible substrates that selectively promote the\\ngrowth and metabolic activity of beneficial microbial\\ncommunities. Synbiotics represent strategic combinations\\nof probiotics and prebiotics. With advancing mechanistic\\nunderstanding of host–microbiome interactions, probiotic\\ninterventions demonstrate significant therapeutic potential\\nin cancer treatment paradigms, particularly in enhancing\\nanti‐tumor efficacy and mitigating treatment‐associated\\nadverse effects. Preclinical investigations have confirmed\\nthat oral multi‐strain probiotic preparations significantly\\nimprove chemotherapy ‐related symptoms, including\\nweight loss, adipose tissue depletion, and increased intes-\\ntinal permeability, while effectively reducing pro ‐\\ninflammatory cytokine levels. These implications further\\ndemonstrate the therapeutic potential of targeted probiotic\\ninterventions in alleviating chemotherapy‐related adverse\\neffects [738]. Diverse probiotic strains, predominantly\\nwithin theBifidobacteriumand Lactobacillusgenera, exhibit\\nanti‐tumor effects through multiple mechanisms, including\\nenhancement of anti‐tumor immune responses and inhi-\\nbition of tumor invasion [715, 739]. In the context of\\nchemotherapeutic potentiation, probiotics enhance\\nchemotherapeutic efficacy and reduce toxicity through\\nS C F A sp r o d u c t i o n ,w h i c hi sf u r t h e re n h a n c e dt h r o u g ht h e\\nsynergistic application of prebiotics [677]. Specific Lacto-\\nbacillus strains modulate intestinal microbiome homeosta-\\nsis and metabolite profiles through dual mechanisms,\\nfavorably remodeling the TME, collectively demonstrating\\nsignificant potential for suppressing CRC progression [740].\\nIn PC models,Lactobacillusspecifically regulates intestinal\\nmicrobial homeostasis through inhibition of TLR4 signaling\\npathway, potentially effectively inhibiting PC progression\\n[48]. In IBD patients, probiotics likeLactobacillus plantar-\\num and Lactococcus lactisare proposed as potential thera-\\npeutic agents for both IBD management and associated\\nCRC prevention through modulating intestinal microbial\\ncomposition and rebalancing dysregulated mucosal\\nimmune responses [326]. In CRC prevention and treat-\\nment, probiotic/synbiotic interventions significantly reduce\\nMICROBIOME IN CANCER | 55 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 55, 'page_label': '56'}, page_content='FIGURE 8 Therapeutic strategies for microbiome modulation in cancer treatment. On the one hand, optimize the host microbial\\necosystem (Left). (I). Probiotic and prebiotic interventions show the beneficial effects on intestinal permeability, pro‐inflammatory cytokine\\nexpression, gut microbiota homeostasis, weight loss, adipose tissue exhaustion, and gastrointestinal adverse effects. (II). FMT shows\\noutcomes including increasedβ‐diversity, decreased regulatory T cells (Tregs), enhanced ICI responses, improved intestinal epithelial\\nintegrity, and reduced treatment‐related toxicity. (III). Precision antibiotic interventions (vancomycin, ciprofloxacin) in HCC and CRC\\nmouse models improve prognosis, while antibiotics for unresectable/metastatic HCC patients receiving immunotherapy/chemotherapy\\nshow the converse affects. (IV). Dietary options represent the key modifier in microbiome‐mediated tumor treatment. Dietary fiber and\\nprobiotics serve as protectors, whereas the role of saturated fatty acid‐rich foods is strongly associated with tumorigenesis and microbial\\ndysbiosis. Alternatively, targeting tumors directly with microbes (Right). (I). Microbiome‐targeted drug development provides new\\ntherapeutic directions. S. epidermidis produces 6‐HAP with demonstrated efficacy against skin cancer, while engineered nanoparticles\\ncontaining sorafenib/rapamycin facilitate delivery of microbial metabolites (butyrate) for liver cancer therapy. (II). Oncolytic virus therapy,\\nincluding RNA oncolytic viruses (coxsackieviruses, measles viruses, polioviruses), DNA oncolytic viruses (adenoviruses, cowpox viruses,\\nherpes simplex viruses), and particularly clinical applications (T‐VEC, H101, ECHO‐7, and Teserpaturev), shows potential for improvement\\nin anti‐tumor therapeutic strategies. (III). Engineered bacteria therapy involves bacterial‐derived outer membrane vesicles, which can target\\nF. nucleatumand attack triple‐negative breast cancer cells.S. epidermidispromotes tumor‐specific T cell proliferation. Engineered bacteria,\\nincluding Salmonella typhimurium expressing vascular endothelial growth factor receptor 2 (VEGFR2)\\n+, and Listeria monocytogenes\\nattenuated live vaccine (ANZ‐100, CRS‐207) have progressed to Phase I trials. Additional bacterial candidates under investigation, including\\nfacultative anaerobic Salmonella typhimurium (VNP)conjuncted with CaCO3 andE. coli expressing anti‐TREM2 single‐chain antibody\\nfragments (scFv), exert anti‐cancer activity and promote tumor radioimmunotherapy. (IV). Fungi‐based therapeutic approaches, when\\ncombined with amphotericin B or specific fungal‐derived metabolites, can induce cancer cell death. 1scFv, single‐chain antibody fragments;\\n6‐HAP, 6‐N‐hydroxylaminopurine; FMT, Fecal microbiota transplantation; VEGFR2, vascular endothelial growth factor receptor 2.\\n56 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 56, 'page_label': '57'}, page_content='aberrant crypt foci formation while promoting the restora-\\ntion of beneficial microbes, enhancing SCFA production,\\nand modulating expression of key inflammatory mediators\\n[741]. In patients with advanced lung cancer receiving\\nplatinum‐based combination chemotherapy regimens,\\nclinical studies show probiotic complexes significantly\\nattenuate chemotherapy‐related gastrointestinal adverse\\neffects [742]. However, context‐dependent effects have been\\nobserved wherein Lactobacillus may exert potential pro‐\\ntumorigenic activities by increasing lactic acid concentra-\\ntions of the TME and subsequently suppressing anti‐tumor\\nimmune responses. This suggests that it is critical to con-\\nsider the TME complexity when developing therapeutic\\nprobiotic interventions [743]. In RCC patients receiving\\nICIs, probiotic preparation of CBM588 containingClos-\\ntridium butyricum has demonstrated strong correlations\\nwith enhanced therapeutic responses through promoting\\nproliferation ofBifidobacterium spp. and facilitating acetate\\nand butyrate metabolism [ 744]. Additionally,\\nprobiotic spores (spores ‐dex) specifically demonstrate\\nselective accumulation within colon cancer tissues, exerting\\nanti‐tumor effects through local production of SCFA and\\nmodulation of gut microbiota composition, collectively,\\nsignificantly inhibiting tumor growth [238]. In conclusion,\\nthe strategically designed probiotic intervention represents\\na cornerstone of microbiome ‐targeted therapeutic ap-\\nproaches, showing considerable promise across multiple\\nclinical applications. However, the mechanistic complexity\\nand context‐dependent efficacy of these interventions un-\\nderscore the importance of rigorous investigation.\\nFecal microbiota transplantation\\nFMT exhibits significant therapeutic potential across\\nd i v e r s em a l i g n a n c i e s[745–749]. In a primary HCC murine\\nmodel, FMT significantly altered intestinal microbialβ‐\\ndiversity while concurrently reducing Treg density, indi-\\ncating FMT efficacy in modulating gut microbiota and\\nimmune microenvironment [750]. Subsequent investiga-\\ntions revealed FMT potentially enhances clinical out-\\ncomes following ICI administration in advanced or\\nmetastatic CRC treatment, particularly among CRC\\npatients with poor immunotherapy response [ 751].\\nMechanistically, FMT combined with lactic acid ‐\\nproducing bacteria and their metabolic products (post‐\\nprobiotics) synergistically restructures intestinal microbial\\necosystems and modulates the TIME, conferring both\\nprophylactic and therapeutic benefits against CRC [7].\\nInterestingly, FMT from ICI‐responsive cancer patients\\nsignificantly enhanced anti‐PD‐1 efficacy when adminis-\\ntered to germ‐free or antibiotic‐treated murine models\\n[24]. Additionally, FMT efficacy in the treatment of\\nrecurrent Clostridium difficile infections (CDI), coupled\\nwith its capacity to enhance survival, restore gastro-\\nintestinal function, and preserve intestinal epithelial bar-\\nrier integrity in radiation ‐exposed animal models,\\nsupports its therapeutic potential to improve clinical out-\\ncomes in patients with cancer [752]. In summary, FMT\\nrepresents a central component of microbiome‐targeted\\ntherapeutic strategies with the potential to modulate the\\ngut microbiota, enhance immunotherapeutic responses,\\nand mitigate treatment‐associated toxicities.\\nMultiple clinical investigations are systematically eval-\\nuating FMT efficacy across diverse oncological contexts\\n[753]. Randomized controlled trials show significantly\\nhigher rates of TKI‐induced diarrhea symptom relief in\\nhealthy donors treated with FMT compared to vehicle‐only\\ncontrols [726]. A recent study suggests this process corre-\\nlates with enhanced mucosa‐associated invariant T cell\\nfunctionality [754]. A retrospective case series demon-\\nstrated that for cancer patients receiving chemotherapy\\nwho developed CDI, the majority of them experienced\\nsignificant improvement without recurrence following\\nFMT administration, supporting FMT as an effective and\\nwell‐tolerated intervention for chemotherapy‐associated\\ncomplications [755]. Similarly, a recent study analyzing\\npatients with ICI‐induced refractory immune‐mediated\\ncolitis found 83% achieved remission following FMT\\ntreatment [756]. A clinical trial exploring FMT to overcome\\nICI resistance in melanoma utilized fecal samples from\\nmelanoma patients exhibiting partial or complete ICI\\nresponses as donors. They have observed a 30% objective\\nresponse rate in 10 melanoma patients receiving FMT\\ncombined with ICI therapy, a substantial improvement\\ncompared to ICI rechallenge response rates (<10%) [757].\\nAnother study conducted a similar trial combining FMT\\nwith pembrolizumab in 15 immunotherapy ‐refractory\\nmelanoma patients, documenting clinical benefit in three\\npatients and disease control exceeding 1 year in three pa-\\ntients [758]. Administration of FMT derived from healthy\\ndonors to 20 melanoma patients concurrently receiving\\nanti‐PD‐1 therapy yielded a 65% objective response rate\\n[759]. For urothelial and prostate cancer patients develop-\\ning colitis after ICI treatment, FMT from healthy donors\\neffectively restructured intestinal microbial communities,\\nresulting in rapid and significant improvement of refractory\\nimmune‐related colitis [760]. These clinical observations\\nsuggest that FMT holds promise for improving remission\\nrates and reducing treatment‐associated adverse events.\\nAntibiotic regulation\\nAntibiotic‐mediated microbiome modulation represents a\\ncritical focus of microbiome‐directed tumor intervention\\nMICROBIOME IN CANCER | 57 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 57, 'page_label': '58'}, page_content=\"strategies [761–763]. Vancomycin treatment significantly\\ndepletes intestinalClostridium cluster XIVapopulations in\\nHCC murine models, consequently reducing circulating\\nsecondary bile acid concentrations and attenuating hepa-\\ntocarcinogenesis. This finding elucidates a mechanistic\\npathway through which antibiotics influence oncogenesis\\nvia alterations of microbial metabolite profiles [ 764].\\nNotably, combining antimicrobial agents with chemo-\\ntherapy demonstrates significant synergistic effects. Pre-\\nclinical investigations utilizing colon cancer murine\\nmodels demonstrate administration of gemcitabine and\\nciprofloxacin elicits significantly enhanced therapeutic\\nresponses compared to gemcitabine monotherapy. That\\nsuggests antibiotic‐mediated microbiome modulation may\\nenhance chemotherapeutic efficacy [642]. However, anti-\\nmicrobial interventions necessitate judicious implemen-\\ntation, as perturbations in intestinal microbial ecosystems\\nmay significantly influence immunotherapeutic outcomes.\\nA retrospective analysis of clinical trial data of un-\\nresectable/metastatic HCC patients treated with ICIs\\ndemonstrated that early antimicrobial exposure signifi-\\ncantly compromised ICIs' efficacy. The phenomenon is\\npotentially due to disruption of microbiota communities\\n[765]. Administration of anti ‐anaerobic antimicrobial\\nagents correlates with poor prognosis in HCC patients\\nundergoing chemotherapy, whereas intestinal enrichment\\nwith anaerobicBlautia demonstrates a significant associ-\\nation with favorable outcomes. Consequently, rigorous\\nantimicrobial stewardship protocols require implementa-\\ntion for agents targeting anaerobic microbial communi-\\nties [766].\\nDietary regulation\\nDietary modification represents a key modifier in\\nmicrobiome‐mediated oncological therapeutic ap-\\nproaches [767–770]. In advanced melanoma patients\\nreceiving ICI therapy, adherence to the Mediterranean\\ndiet is positively correlated with immunotherapy efficacy\\n[761]. Comprehensive analyses utilizing 3‐day, 24‐h die-\\ntary recall methodologies in lung cancer patients have\\nestablished significant associations among dietary intake\\npatterns, intestinal microbial diversity indices, and mul-\\ntiple clinical health parameters [772]. Dietary fiber, a\\ncritical nutritional component, exerts multiple biological\\nfunctions in maintaining host homeostasis. It serves as a\\nsubstrate for intestinal microbial fermentation, subse-\\nquently generating bioactive metabolites, including\\nSCFAs and bile acids, and significantly influencing car-\\ncinogenesis [ 773]. Substantial evidence demonstrates\\ndietary fiber has significant chemopreventive and thera-\\npeutic effects in preventing and intervening in CRC,\\nincluding restoration of microbial community diversity,\\nelevation of SCFA levels, and suppression of EMT [774].\\nFurthermore, strategic dietary interventions synergize\\nwith circadian regulatory mechanisms to maintain\\nphysiological homeostasis. Time ‐restricted feeding\\n(TRF), a chronobiologically aligned nutritional inter-\\nvention, preserves circadian rhythm synchronization,\\nameliorates metabolic disorders, and significantly delays\\nlung tumor growth. TRF exerts anti ‐tumor effects\\nthrough restructuring intestinal microbial communities,\\nparticularly increasing Lactobacillus and Bacillus abun-\\ndance, and influencing immune and inflammatory pro-\\ncesses [775]. Notably, different dietary components\\nvariably affect the gut microbiome. Dietary regimens\\nincorporating sorghum, allium species, and cruciferous\\nvegetables demonstrate inverse associations with risk\\nof BT, which is mediated through circulating extra-\\ncellular vesicle (EV) associated microbial signatures.\\nHowever, consumption of specific plant‐derived foods,\\nincluding tumbleweeds and py rus fruits, potentially\\nenhances oncogenic risk [626]. Furthermore, compar-\\native analysis of dietary lipid models demonstrates that\\nsaturated fatty acids and cocoa butter exhibit signifi-\\ncantly greater pro ‐tumorigenic effects compared to\\nmonounsaturated (olive oil) or polyunsaturated (corn\\noil) fatty acid sources [776].\\nMicrobiome‐targeted drugs\\nWithin the expanding domain of microbiome‐directed\\ntherapeutic approaches, the development of agents spe-\\ncifically targeting microbial communities provides new\\ntherapeutic ideas and directions for cancer treatment.\\nStudies show 6‐HAP secreted by S. epidermidis specifi-\\ncally inhibits cancer cells, establishing novel therapeutic\\nstrategies for cutaneous malignancies [420]. NPs func-\\ntionalized with butyrate and co‐encapsulating sorafenib\\nand rapamycin demonstrate significantly enhanced\\ntherapeutic efficacy in HCC [777]. Researchers increas-\\ningly emphasize the critical contributions of intestinal\\nmycobiome components to hepatocarcinogenesis. Spe-\\ncific fungi demonstrate capacity for translocation from\\nintestinal compartments to liver, closely related to HCC\\ndevelopment, suggesting fungal genomes represent\\npromising therapeutic targets [ 723]. Additionally,\\nmicrobiome‐targeted therapies demonstrate significant\\nsynergistic interactions with chemotherapeutic agents,\\npotentiating chemotherapeutic efficacy through modu-\\nlation of drug metabolism [ 642]. Notably, precision\\ntherapeutic strategies against specific microorganisms\\nlike F. nucleatumshow promising results for BC patients,\\nespecially when combined with immunotherapy and\\n58 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 58, 'page_label': '59'}, page_content='autophagy inhibitors [171]. In summary, microbiome‐\\ntargeted drug development may represent a significant\\npotential to revolutionize contemporary oncological\\nmanagement.\\nOncolytic virus therapy\\nAmong microbiome‐mediated tumor therapy strategies,\\nOV therapy (OVT) exhibits substantial anti‐tumor poten-\\ntial as an emerging therapeutic modality [778–783]. OVs\\nderive predominantly from attenuated human pathogens,\\nvaccine vector platforms, and genetically modified\\nreplication‐selective viral constructs. RNA virus‐derived\\noncolytic agents include coxsackieviruses, measles viruses,\\nand polioviruses represent typical OVs, while DNA virus‐\\nderived therapeutic vectors encompass adenoviruses,\\nvaccinia viruses, and herpes simplex viruses [784]. Each\\npossesses a distinct clinical application. OVs mediate\\ntherapeutic effects through the induction of immunogenic\\ncell death and modulation of host anti‐tumor immunity,\\nachieving direct oncolysis while preserving non‐malignant\\ntissues [785, 786]. Currently, four OV types have received\\nregulatory approval for clinical oncological applications,\\nincluding talimogene laherparepvec (T ‐VEC), H101,\\nECHO‐7 (Echovirus), and Teserpaturev [785]. OVT dem-\\nonstrates promising efficacy across diverse malignancies,\\nyet faces challenges including virus leakage, safety hazards\\nof accidental transmission, strict transportation and stor-\\nage requirements, and necessity for specialized delivery\\nsystems [786, 787]. Therefore, developing safer and more\\neffective OVT strategies represents a critical priority\\nfor advancing clinical implementation of its clinical\\ntranslation.\\nEngineered bacteria therapy\\nWithin the expanding arsenal of microbiome ‐directed\\ntherapeutic modalities, engineered bacterial platforms\\nrepresent a transformative approach with exceptional\\ntherapeutic potential. Engineered bacteria can be pre-\\ncisely modified to selectively colonize neoplastic tissues\\nand functionalized as delivery vehicles for chemo-\\ntherapeutic payloads, significantly enhancing chemo-\\ntherapy efficacy [788]. These engineered microorganisms\\napply not only independently but also synergistically\\nwith other anti‐tumor therapies, serving as diagnostic\\nsignals and preventive vaccines, demonstrating remark-\\nable multifunctionality [789]. Various bacteria have been\\nengineered as therapeutic agents, with Listeria mono-\\ncytogenes and Salmonella enterica representing the most\\nextensively characterized. Currently, multiple engineered\\nbacteria have progressed to Phase I and II trials, dem-\\nonstrating favorable results [790]. In a Phase I clinical\\ntrial for advanced PC, oral administration of attenuated\\nSalmonella typhimurium expressing VEGFR2 increased\\nVEGFR2‐reactive CD4\\n+ and CD8+ T cell responses and\\npromoted production of IFN γ, TNF, and IL ‐2, while\\nsignificantly reducing tumor invasion [ 791]. Simulta-\\nneously, Listeria monocytogenes attenuated live vaccine\\n(ANZ‐100) and Listeria monocytogenes attenuated live\\nvaccine expressing tumor differentiation antigen me-\\nsothelin (CRS‐207) have completed Phase I clinical trials,\\ndemonstrating significant efficacy and good tolerability\\nin patients with mesothelioma, lung cancer, PC, and\\novarian cancer [792]. Comprehensive analyses of en-\\ngineered bacterial projects show their capacity to regulate\\ntumor cell and reprogram immune cell metabolism in\\nthe TME [793]. Additionally, engineered S. epidermidis\\ninduces tumor‐specific T cell production, resulting in\\nsignificant inhibition of both primary and metastatic\\nmelanoma progression [506]. These studies indicate that\\nengineered bacteria are associated with the TIME [794].\\nFurthermore, engineered bacteria can assist with\\nimmunotherapy for tumors [795–797]. For the specific\\nTME, engineered bacterial therapy functions as a potent\\nimmunotherapeutic adjuvant. A recent study shows that\\nengineered E.coli precisely expressing anti ‐TREM2\\nsingle‐chain antibody fragments (scFv) can regulate\\nmacrophages to promote tumor radioimmunotherapy\\n[798]. Bioconjugates of engineered bacteria (facultative\\nanaerobic Salmonella typhimurium VNP20009, VNP)\\nwith CaCO\\n3 can precisely target and colonize tumor\\ncells, exerting anti‐cancer activity and helping improve\\nimmunotherapeutic efficacy [799]. Targeting the unique\\nphysiological and immunological characteristics of\\nthe TME, engineered microbial therapeutics facilitate\\nprecise and multifaceted immunomodulation. In triple‐\\nnegative BC research, a bacterial‐derived outer mem-\\nbrane vesicle (OMV)‐coated nanoplatform demonstrates\\ndual‐targeting capacity against pathogenicF. nucleatum\\nand cancer cells, effectively repurposing intratumoral\\nmicrobial communities as an immunostimulatory adju-\\nvant, providing an innovative therapeutic approach for\\ntriple‐negative BC [800]. Engineered bacterial therapies\\nfurther demonstrate the capacity to maintain oxygen\\nsupply, activate chemotherapeutic agents, and enhance\\ntumor radiosensitivity [794].\\nFungal treatment strategies\\nWithin the expanding landscape of microbiome‐directed\\noncological interventions, mycobiome ‐targeted thera-\\npeutic approach emerge as an innovative anti ‐tumor\\nMICROBIOME IN CANCER | 59 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 59, 'page_label': '60'}, page_content='paradigm. Emerging evidence demonstrates that modu-\\nlation of mycobiome composition in PC significantly\\ninfluences treatment response, positioning antifungal\\ninterventions as promising therapeutic strategies for\\nspecific PC patients [221]. Additionally, targeted modu-\\nlation of fungal ‐associated immunological mediators,\\nincluding MBL and IL ‐33, provides novel immuno-\\ntherapeutic strategies in cancer treatment [ 801]. Pre-\\nclinical studies demonstrate that combination regimens\\nincorporating antifungal agents with radiotherapy en-\\nhance tumor cell apoptosis and prolong survival com-\\npared to radiotherapy alone. This emphasizes the\\npotential of mycobiome ‐targeted interventions as\\nradiotherapy‐sensitizing strategies [686]. However, the\\nimpact of fungal species on oncological outcomes dem-\\nonstrates significant strain ‐specific heterogeneity. For\\nexample, in BC murine models,C. albicans supplemen-\\ntation combined with radiotherapy accelerated tumor\\nprogression and decreased survival intervals, illustrating\\nthe critical importance of species ‐specific mycobiome\\neffects on therapeutic response [ 686]. Furthermore,\\nfungal‐derived bioactive metabolites show cancer therapy\\npotential, including impairment of DNA damage repair\\npathways of malignant cells and direct cytotoxic and\\ngenotoxic activities, collectively contributing to their anti‐\\ntumor effects [802]. In conclusion, mycobiome‐directed\\ntherapeutic strategies represent an innovative component\\nof the microbiome‐targeted oncological armamentarium,\\ndemonstrating notable promise through diverse mecha-\\nnistic pathways.\\nFUTURE PERSPECTIVES AND\\nCONCLUSION\\nThe evolving landscape of microbiome science presents\\nunprecedented opportunities coupled with formidable\\nmethodological and translational challenges that demand\\ninnovative solutions. Microbiome research technologies\\nrequire continuous refinement and transformative tech-\\nnological innovations. Elucidation of mechanistic path-\\nways mediating microbial influences in oncogenesis,\\ndevelopment of precision microbiome ‐targeted thera-\\npeutic interventions, and acceleration of robust clinical\\ntranslation collectively advance our comprehensive un-\\nderstanding of host–microbe interactions in malignancy\\nwhile establishing novel paradigms for cancer manage-\\nment. Optimizing microbiome combination therapy\\nstrategies will enhance therapeutic efficacy and expand\\ntherapeutic paradigms. As scientific understanding ad-\\nvances, microbiome‐directed therapeutic agents are pro-\\ngressively entering clinical development pipelines,\\nthrough necessitating rigorous regulatory frameworks\\nthat address concerns regarding the safety and efficacy.\\nIntegration of emerging research directions, exploration\\nof understudied microbial communities, and characteri-\\nzation of their functional interactions promise notable\\nclinical significance. Figure9 comprehensively illustrates\\nthe major challenges and future directions of microbiome\\nand oncology research, including technological innova-\\ntions, mechanistic explorations, therapeutic strategy\\ndevelopment, clinical translation approaches, and mul-\\ntivariate microbiome research frontiers, providing read-\\ners with an integrated framework and strategic\\nperspective on the future development of microbiome\\nand tumor research.\\nDevelopments and innovations in\\nmicrobiome research technologies\\nRecent technological innovations in microbiome research\\nare driving unprecedented analytical depth and expan-\\nding investigative breadth across oncology. Single ‐cell\\nFIGURE 9 Challenges and future directions of microbiome in\\nthe field of precision oncology. This circular diagram illustrates\\neight key aspects defining the frontier of microbiome‐oncology\\nresearch: (1) quality control, (2) clinical translation pathways,\\n(3) combination strategies with microbes‐based cancer treatments,\\n(4) personalized therapies based on individual mirobiome profiles,\\n(5) cutting‐edge research developments, (6) new microbial species\\nand their potential application, (7) emerging technological\\nplatforms and methodological advances of technologies, and\\n(8) mechanistic insights into biological processes governing\\nmicrobe‐tumor microenvironment interactions.\\n60 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 60, 'page_label': '61'}, page_content='sequencing technology, as a representative emerging\\nmethodology, shows unique advantages and broad appli-\\ncation prospects in microbiome research. Microbial split‐\\npool ligation transcriptomics (microSPLiT) enables high‐\\nthroughput transcriptional analysis of Gram‐negative and\\nGram‐positive bacteria, revealing gene expression hetero-\\ngeneity in Bacillus subtilis [801]. Additionally, the\\ninnovative Barcoding Bacteria for Identification and\\nQuantification (BarBIQ) method significantly improves\\n16S rRNA identification accuracy through single‐bacterial\\nbarcode labeling [803]. In the spatial dimension, emerging\\ngenomic technologies reveal complex microbe –host\\nrelationships within tissue microenvironments, with re-\\nsearchers successfully achieving in situ resolution of host–\\nmicrobe interactions in OSCC and CRC using GeoMx\\ndigital spatial profiling (DSP) [ 804]. At the systemic\\nresearch level, multi‐omics integration strategies establish\\nnew paradigms for studying microbial functions within\\nlung cancer and other tumors by simultaneously capturing\\nthe microbiome and host transcriptional responses [349].\\nWith the advent of the big‐data era, artificial intelligence\\n(AI) and machine‐learning applications in microbiome\\ndata analysis have become increasingly critical for inter-\\npreting complex microbiome data sets. AI demonstrates\\nremarkable potential in predicting host–microbe interac-\\ntions, resolving uncharacterized metagenomic sequences,\\nand optimizing experimental design [ 805]. Machine ‐\\nlearning algorithms excel in pattern recognition for cancer\\nrisk prediction and therapeutic response forecasting [806].\\nIn summary, multidimensional innovative developments\\nin microbiome research technologies synergistically drive\\nmicrobiome research in oncology toward greater precision\\nand comprehensive mechanistic understanding.\\nDespite significant advances in microbiome research\\ntechnologies, critical challenges remain to be addressed.\\nThe low biomass of intratumoral microbiota presents\\nfundamental challenges in detection sensitivity and\\nspecificity, with heightened susceptibility to host DNA\\ncontamination and background noise that may generate\\nfalse‐positive or false‐negative results. Furthermore, ex-\\nisting functional validation methods (such as in vitro co‐\\nculture or murine models) inadequately recapitulate the\\ncomplex microbe–host interaction networks in humans,\\nwhile most studies rely on correlation analyses that\\nprovide insufficient evidence for causal relationships. For\\nexample, the carcinogenic effects of intratumoral\\nmicrobes are often indirectly verified through gene\\nknockout or antibiotic intervention approaches, but it\\nremains difficult to exclude the influence of off‐target\\neffects or ecological niche disturbances. Future efforts\\nrequire the development of high ‐resolution in situ\\ndetection technologies, particularly single ‐cell spatial\\nmulti‐omics, combined with organoid‐microbe co‐culture\\nsystems to enhance both analytical precision and clinical\\ntranslational value.\\nIn‐depth exploration of microbiome\\nmechanisms of tumors\\nIntensified mechanistic investigations have progres-\\nsively elucidated the complex bidirectional interactions\\nbetween the microbiome and cancer initiation, pro-\\ngression, and therapeutic response. Microbiome ‐\\nmediated metabolic reprogramming has emerged as a\\nfocal area of investigation, with recent research deeply\\nelucidating molecular mechanisms underlying the\\nmodulation of tumor proliferation and invasion. The\\nimmune–tumor–microbiota (IOM) axis framework,\\nrecently conceptualized, illustrates the complex regu-\\nlatory network among the immune system, cancer\\ncells, and microbiota. Systematic resolution of the gut\\nIOM axis has emerged as a research priority, offering\\nnovel perspectives on mechanisms by which micro-\\nbiota influences the TME through immune regulation.\\nIn the realm of epigenetic regulation, significant ad-\\nvances have characterized microbiome ‐mediated\\nmechanisms, with evidence supporting a bidirectional\\n“epigenome –microbiome axis ” [807]. This evidence\\nreveals the critical role of microbiota‐mediated epige-\\nnetic modifications in tumorigenesis and development.\\nAdditionally, SAHMI technology has successfully\\nidentified microbes associated with single cells,\\nrevealing critical relationships between the in-\\ntratumoral microbiome and lung cancer progression\\nthat may yield novel diagnostic biomarkers and ther-\\napeutic targets [808]. In summary, microbiome mech-\\nanism studies involve multiple dimensions, including\\nmetabolic reprogramming, immune regulation, epige-\\nnetic modifications, and cellular interaction networks\\nwithin the TME. These mechanistic insights substan-\\ntially advance our understanding of how diverse\\nmicrobial communities influence tumorigenesis and\\ntumor progression.\\nDevelopment of precision microbiome\\ntherapeutic strategies\\nThe development of precision microbiome ‐targeted\\ntherapeutic strategies has emerged as a significant fron-\\ntier in addressing diverse malignancies. As an integral\\ncomponent of precision oncology, microbiome analyses\\nin human diseases continue to evolve, providing a reli-\\nable scientific foundation for individualized therapeutic\\nstrategies [809]. The development of engineered bacterial\\nMICROBIOME IN CANCER | 61 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 61, 'page_label': '62'}, page_content=\"platforms represents a pivotal technological approach for\\nmodulating microbiome‐associated diseases, with thera-\\npeutic potential systematically demonstrated in several\\nstudies [810, 811]. Evidence indicates that genetically\\nengineered bacteria can function as sophisticated deliv-\\nery vectors, precisely transporting therapeutic payloads\\nto tumor sites, and significantly enhancing local drug\\nconcentrations [812]. Additionally, synthetic biology ap-\\nproaches applied to cancer‐associated microbiomes pro-\\nvide innovative strategies for overcoming treatment\\nresistance, with CRISPR‐based and genome‐engineering\\ntechnologies demonstrating significant potential for\\ndeveloping novel therapeutic modalities [813]. Therefore,\\nthe advancement of precision microbiome‐based cancer\\ntherapeutics hinges on complementary approaches,\\nincluding individualized microbiome intervention pro-\\ntocol development, engineered bacterial development\\nand optimization, microbiome‐targeted delivery system\\ndesign, and innovative applications of synthetic biology\\nin microbiome therapy. These integrated approaches\\ncollectively advance microbiome ‐based cancer ther-\\napeutics toward improved target specificity and truly\\npersonalized interventions.\\nAmong these, bacteriophage‐based therapeutic strat-\\negies are garnering significant attention in oncology due\\nto their exceptional target selectivity and molecular\\naffinity. Phages interact with immune cell cytokine net-\\nworks, participating in the regulation of inflammation\\nand immune tolerance [ 814], while simultaneously\\nmodulating the TME and effectively inhibiting tumor cell\\ngrowth and metastasis. Based on the tumor‐suppressive\\nfunctions of phages, numerous studies have screened\\nhighly selective and high ‐affinity phage vectors com-\\nbined with bioengineering techniques for cancer treat-\\nment. Lei et al. developed a CD40‐targeting engineered\\nM13 phage (H ‐GM‐M13CD40) that functions as an\\nin situ vaccine by selectively targeting and activating\\nDCs, inducing DC infiltration within tumor tissues to\\nreverse the immunosuppressive TME [815]. Hou et al.\\ndeveloped a phage vector T4‐Lox‐DNA‐Fe (TLDF) that\\nintegrates T4 phage and biointelligent plasmids to dis-\\nrupt redox homeostasis within the TME, achieving\\nremarkable tumor growth inhibition of up to 78% and\\nestablishing a novel approach for precise micro-\\nenvironmental modulation [ 816]. Additionally, re-\\nsearchers developed a tumor ‐targeting transforming\\nphage/AAV (RGD4C.TPA) vector engineered to deliver\\nTNF‐α genes, which induced selective tumor cell apo-\\nptosis and vascular disruption. This approach enhances\\nefficacy when combined with cisplatin, establishing a\\npromising approach against medulloblastoma [ 817].\\nM2 macrophage ‐specific targeting peptides identified\\nthrough phage display technology have been engineered\\nto bind with high affinity to both M1 and M2 phenotypes\\nof tumor‐associated macrophages. When combined with\\ncomplementary genetic modifications, these integrative\\nproducts enable synergistic elimination of malignant\\ncells while providing a dual‐targeting, light‐controlled\\ntherapeutic strategy for precise TME modulation [817].\\nPhage display technology was recognized with the Nobel\\nPrize in Chemistry in 2018, with increasing research\\nbeing devoted to phage‐based therapeutic exploration.\\nMechanistic models centered on phage‐based interven-\\ntions will be crucial for future research, as systematic\\nexploration of this direction will significantly advance\\nthe translational impact of microbial approaches in\\nprecision oncology.\\nAdvancement of clinical translational\\nresearch\\nTranslational research bridging the microbiome and\\ntumor therapy is rapidly evolving as a critical research\\ndirection. Large‐scale multicenter clinical trials not only\\nestablish solid data foundations for clinical translational\\nresearch but also provide strong support for the reliability\\nof research outcomes. For example, a multicenter clinical\\ntrial revealed microbiota's potential as a novel thera-\\npeutic target in urological malignancies, with implica-\\ntions for developing personalized treatment algorithms\\n[818]. Meanwhile, the development of standardized\\ntreatment protocols represents a critical prerequisite for\\nadvancing rigorous clinical investigation. Currently,\\ntumor treatment modalities are diverse, including\\nchemotherapy, radiotherapy, immunotherapy, targeted\\ntherapy, surgery, and microorganism‐based combination\\ntherapy [712, 819–826]. While this therapeutic diversity\\nexpands clinical options, it simultaneously presents\\nsubstantial methodological challenges for comparative\\nresearch. In particular, the lack of uniform research\\nstandards impedes rigorous cross ‐study analysis and\\nmeta‐analytical interpretation. The development of\\nstandardized treatment protocols will harmonize trans-\\nlational research methodologies and enhance clinical\\ndecision‐making efficiency, representing a critical prior-\\nity for advancing evidence‐based oncological practice.\\nAdditionally, existing clinical biomarkers demonstrate\\ninherent limitations despite their established utility.\\nNovel biomarker development and validation have sig-\\nnificant value in clinical translation of microbiome\\nresearch. Particularly, microbiome ‐based assessment\\nprotocols may provide precise cancer patient stratifica-\\ntion by severity, potentially facilitating optimized inter-\\nvention strategies and improved survival outcomes.\\nFor example, identification of conserved microbial\\n62 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 62, 'page_label': '63'}, page_content='biomarkers across ethnically heterogeneous patients en-\\nables applicability of optimal diagnostic and prognostic\\nmodels across diverse populations [50]. Establishment of\\ncomprehensive systems for monitoring and preventing\\nadverse effects constitutes an essential component of\\ntranslational research. Despite continuous refinement of\\ntherapeutic modalities, adverse events remain key factors\\naffecting the efficacy and quality of life of patients.\\nConsequently, implementing robust pharmacovigilance\\nsystems for adverse effect detection and prevention is\\nparamount for optimizing therapeutic outcomes and\\nsafeguarding patient well‐being. In summary, advance-\\nment in microbiome ‐oncology clinical translational\\nresearch requires integration of multiple components,\\nincluding large‐scale multicenter clinical trials, stan-\\ndardized treatment protocol development, biomarker\\ndevelopment and validation, and adverse events\\nmonitoring and prevention systems. These integrated\\napproaches collectively advance oncological therapeutics\\ntoward enhanced precision, efficiency, and safety.\\nRigorous clinical validation through well ‐designed\\nprospective multicenter trials represents a critical prior-\\nity, exemplified by protocols that integrate longitudinal\\nmicrobiome profiling with treatment response assess-\\nment in ICI therapy. Such protocols may randomize\\neligible patients to receive either monotherapy with ICIs\\nor combination therapy with microbiome ‐modulating\\nagents, employing standardized techniques for sampling\\nand monitoring changes in microbial marker dynamics.\\nThese protocols would incorporate systematic assessment\\nof the TIME parameters, with subsequent analysis of\\ncorrelations with PFS. Simultaneously, quantitative\\nanalysis of microbial metabolites could facilitate the\\ndevelopment of integrated diagnostic models incorpo-\\nrating microbial taxa with metabolite signatures, poten-\\ntially enhancing diagnostic precision and predictive\\naccuracy. In the future, Mendelian randomization\\nmethods could further validate causal relationships\\nbetween the microbiome and clinical outcomes, while\\nelucidating the immunomodulatory mechanisms under-\\nlying microbiota‐targeted therapeutic interventions.\\nOptimization of microbiome combination\\ntherapy strategies\\nOptimization of microbiome ‐targeted combinatorial\\ntherapeutic strategies represents a frontier area in con-\\ntemporary oncological research. In this field, elucidation\\nof synergistic mechanisms between the microbiome and\\nimmunotherapy provides a key theoretical foundation for\\ntherapeutic development. Numerous studies demonstrate\\ncritical associations between the microbiome and\\nimmunotherapy, with in ‐depth investigation of\\nmicrobiome‐mediated enhancement of anti ‐tumor\\nimmunity providing valuable insights for optimizing\\nclinical protocols and developing precisely targeted\\ncombination therapies [827, 828]. Design and evaluation\\nof multimodal therapeutic regimens aim to integrate\\nmultiple therapeutic modalities, leveraging mechanism‐\\nspecific advantages to generate synergistic therapeutic\\neffects. This multimodal treatment strategy integrates\\nvarious therapeutic approaches, including immuno-\\ntherapy, chemotherapy, and radiotherapy, while com-\\nprehensively accounting for microbiome ‐mediated\\ninfluences on treatment efficacy. Through systematic\\ncharacterization of patient‐specific microbiome profiles,\\nprecisely tailored multimodal therapeutic regimens can\\nbe developed, potentially enhancing treatment efficacy\\nwhile mitigating adverse events. Additionally, elucida-\\ntion of microbiome ‐mediated drug resistance mecha-\\nnisms represents a critical research priority for\\novercoming therapeutic resistance and improving treat-\\nment efficacy. Studies demonstrate that gut microbes can\\nmediate immunotherapy resistance and accelerate CRC\\nprogression [ 829]. Research has identified intestinal\\nmicrobe‐mediated drug catabolic pathways and key en-\\nzymes, providing novel perspectives into microbiome‐\\nmediated treatment resistance mechanisms [830]. Nota-\\nbly, the development of patient‐specific combinatorial\\ntherapeutic approaches constitutes a primary objective in\\noptimizing microbiome‐informed treatment strategies.\\nIntegrating microbiome profiles with other biomarkers\\nsignificantly improves diagnostic precision and facilitates\\nprediction of therapeutic responsiveness, enabling the\\ndevelopment of precisely tailored interventional ap-\\nproaches. The gut microbiota shows expansive potential\\napplications in precision medicine and individualized\\ntherapy, establishing a solid theoretical foundation for\\ndeveloping personalized combinatorial intervention\\nstrategies [809, 831–834]. In summary, optimization of\\nmicrobiome‐targeted combinatorial therapeutic strate-\\ngies requires comprehensive consideration across multi-\\nple dimensions, including mechanistic synergies with\\nimmunotherapies, multimodal treatment regimen design\\nand evaluation, microbiome‐mediated resistance mech-\\nanisms, and individualized combination therapy strategy\\ndevelopment, collectively advancing oncology toward\\nmaximized treatment efficacy.\\nQuality control in microbiome products\\nAs a cutting‐edge biomedical innovation, microbiome‐\\nbased therapeutic product development and regulation\\naim to advance microbiome drug research, development,\\nMICROBIOME IN CANCER | 63 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 63, 'page_label': '64'}, page_content=\"and clinical application through establishing scientifi-\\ncally standardized processes, stringent quality control\\nstandards, comprehensive regulatory frameworks, and\\nsystematic ethical guidelines, ultimately facilitating\\nnovel precision medicine approaches. Standardization of\\nmicrobiome‐based therapeutic development processes\\nconstitutes the foundational infrastructure for successful\\nproduct advancement. Integrating gut microbiomics into\\nnew drug development systems provides new ideas\\nfor resolving challenges of drug development while\\npotentially enhancing therapeutic candidate success\\nrates. Throughout therapeutic development pipelines,\\nenhanced interdisciplinary collaboration between the\\nmicrobiome and complementary research is essential,\\nnecessitating a comprehensive evaluation of candidate\\ncompounds' effects on microbiome composition and\\nfunction while maintaining scientifically rigorous devel-\\nopment protocols. Establishment of a quality control\\nsystem critically determines the safety profile and ther-\\napeutic efficacy of microbiome ‐based interventions.\\nPharmaceutical quality management systems should\\naddress all critical parameters influencing product\\nquality to ensure consistent compliance with established\\nquality standards and regulatory requirements. For\\nmicrobiome‐based therapeutic products, quality control\\nstrategies must emphasize maintenance of microbial\\nviability, stability, and thorough characterization of in-\\nteractions with other formulation elements. Robust reg-\\nulatory frameworks are fundamental to microbiome\\ntherapeutic development, encompassing scientifically\\nsound product classification criteria, rigorous approval\\npathways, and comprehensive post ‐marketing surveil-\\nlance requirements to ensure thorough evaluation of\\nsafety and efficacy throughout product lifecycles. Mean-\\nwhile, ethical guidelines play a critical role in governing\\nmicrobiome therapeutic advancement, necessitating\\nthorough consideration of potential risks including\\nmicrobial ecological dysbiosis and drug resistance. In\\nsummary, advancement of microbiome‐based therapeu-\\ntic products requires coordinated multidimensional ap-\\nproaches, including R&D process standardization,\\nquality control system construction, regulatory frame-\\nwork improvement, and ethical guideline formulation, to\\nensure product safety, efficacy, and regulatory compli-\\nance while facilitating innovation in precision medicine\\napplications.\\nProspects for research direction\\nMicrobiome research represents a frontier area in con-\\ntemporary biomedical science with profound scientific\\nimplications and extensive applications in oncology. Ex-\\nisting studies have confirmed that the microbiome exerts\\ncritical regulatory influences in tumorigenesis and tumor\\ndevelopment. Investigators propose that identifying\\nmicrobial markers with cancer‐suppressive properties and\\ndeveloping microbiome‐targeted preventive agents will\\nprovide a scientific basis forpreventive interventions in\\nhigh‐risk populations. Intensive studies show that the mi-\\ncrobiome has important indicative roles in tumor pro-\\ngression prediction. Specific microbial signatures and\\ncompositional alterations correlate significantly with tumor\\nstage, histological grade, and patient outcomes, providing a\\ntheoretical basis for developing more refined microbiome‐\\nderived biomarkers. In studies examining microbiome‐\\nmediated remodeling of the TIME, researchers focus on\\noptimizing anti‐tumor immune responses through targeted\\nmodulation of gut microbial communities, thereby en-\\nhancing immunotherapeutic efficacy. Strategic manipula-\\ntion of microbial community composition and function can\\nreshape the TIME to enhance immune cell activation and\\ntumor clearance, offering novel approaches to augment\\ncancer immunotherapy. Additionally, rigorous assessment\\nof the long‐term efficacy and durability of microbiome‐\\ntargeted interventions represents a critical scientific prior-\\nity. Considering the complexity and dynamic character-\\nistics of microbiome therapy, establishing comprehensive\\nevaluation frameworks and longitudinal monitoring of\\nmicrobial community dynamics and host responses is\\nessential for ensuring therapeutic safety and efficacy.\\nMicrobiome research shows significant research value and\\napplication potential across many oncological contexts.\\nBy delineating interactions between the microbiome\\nand tumorigenesis, progression, and treatment response,\\nresearchers may achieve more precise approaches to\\ncancer prevention, prediction, therapeutic selection, and\\noutcome assessment, ultimately providing novel strategies\\nto improve patient prognosis.\\nNovel research directions for microbiome\\nMicrobiome research, at the forefront of biomedical\\ninvestigation particularly within oncology, has illumi-\\nnated complex host–microbe interaction networks and\\nopened expansive research horizons. In studies examin-\\ning interactions between viral communities and the\\nTME, researchers have thoroughly investigated viral\\ncontributions to tumorigenesis, development, and\\nmicroenvironmental regulation. Future efforts resolving\\nhow viral communities interact with tumor cells,\\nimmune components, and other microorganisms to\\nshape the TME may provide novel possibilities for\\n64 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 64, 'page_label': '65'}, page_content=\"exploring virus‐targeted therapeutic opportunities. Stud-\\nies investigating fungal communities and tumor immune\\nresponses focus on how fungal communities regulate\\nimmune responses, affecting tumor therapeutic efficacy\\nand clinical prognosis. Research in this area has eluci-\\ndated associations between fungal diversity and immune\\nevasion, immune cell functionality, and ICI efficacy, pro-\\nviding a mechanistic foundation for mycobiome‐targeted\\nimmunotherapeutic approaches. Multidimensional mi-\\ncrobiome analyses are emerging as a critical research\\nparadigm that aims to decipher complex ecological net-\\nworks among diverse microbial communities and host\\nsystems by integrating multi‐omic data across bacterial,\\nviral, and fungal domains. This comprehensive cross‐\\ndimensional analysis enables more sophisticated under-\\nstanding of microbiome contributions to tumorigenesis,\\ntumor progression, and treatment mechanisms, offering\\nnovel insights for precision medicine and personalized\\ntherapy approaches. Development of new microbiome‐\\nbased therapies represents the translational culmination of\\nmicrobial oncology research. Informed by enhanced\\nmechanistic understanding of microbiome–tumor inter-\\nactions, interventions targeting specific microbial com-\\nmunities or functions, including probiotics, prebiotics, and\\nbacteriophage therapy, are being developed to regulate\\nanti‐tumor immunity, enhance therapeutic efficacy, miti-\\ngate adverse effects, and potentially achieve durable tumor\\ncontrol.\\nIn summary, the human microbiome has emerged as a\\npivotal regulator in cancer pathogenesis, progression, and\\ntherapeutic outcomes. This comprehensive review has\\nsynthesized substantial evidence demonstrating that\\ndiverse microbial communities influence tumorigenesis\\nthrough multiple mechanisms, including inflammatory\\npathways, metabolite production, immune regulatory\\nnetworks, and epigenetic alterations. Beyond their signif-\\nicant diagnostic potential as biomarkers, microorganisms\\nfundamentally influence therapeutic efficacy and modu-\\nlate treatment‐associated toxicities. Innovative therapeutic\\nstrategies targeting the microbiome, including probiotics,\\nFMT, engineered bacteria, and precision antimicrobial\\napproaches, have shown considerable promise in enhan-\\ncing the efficacy of conventional cancer treatments and\\novercoming resistance. Future advances in this rapidly\\nevolving field will require continued technological inno-\\nvation, deeper mechanistic insights, standardized meth-\\nodologies, and rigorous clinical validation to fully harness\\nthe microbiome's transformative potential in precision\\noncology and personalized medicine.\\nAUTHOR CONTRIBUTIONS\\nAnqi Lin:Writing—original draft; writing—review and\\nediting. Minying Xiong: Writing—original draft;\\nwriting—review and editing. Aimin Jiang: Writing—\\nreview and editing.Lihaoyun Huang:Writing—review\\nand editing; visualization.Hank Z. H. Wong:Writing—\\nreview and editing. Suyin Feng: Writing—review and\\nediting. Chunyan Zhang: Visualization. Yu Li: Inves-\\ntigation. Li Chen:Investigation. Hao Chi:Investigation.\\nPengpeng Zhang: Investigation. Bicheng Ye: Investi-\\ngation. Hengguo Zhang: Investigation. Nan Zhang:\\nInvestigation. Lingxuan Zhu: Investigation. Weiming\\nMou: Investigation. Junyi Shen: Investigation. Kailai\\nLi: Investigation. Wentao Xu: Investigation. Haoxuan\\nYing: Investigation. Cangang Zhang: Investigation.\\nDongqiang Zeng: Investigation. Jindong Xie: Investi-\\ngation. Xinpei Deng: Investigation. Qi Wang: Investi-\\ngation. Jianying Xu: Investigation. Wenjie Shi:\\nInvestigation. Chang Qi: Investigation. Chunrun Qu:\\nInvestigation. Xufeng Huang: Investigation. András\\nHajdu: Investigation. Chaoqun Li: Investigation.\\nChangmin Peng: Investigation. Xuanye Cao: Investi-\\ngation. Guangsheng Pei: Investigation. Lin Zhang:\\nInvestigation. Yujia Huo: Investigation. Jiabao Xu:\\nInvestigation. Antonino Glaviano:Investigation. Attila\\nGábor Szöllősi: Investigation. Sicheng Bian: Investi-\\ngation. Zhengrui Li: Validation. Hailin Tang: Valida-\\ntion. Bufu Tang: Validation. Zaoqu Liu: Validation.\\nJian Zhang: Validation. Kai Miao: Validation. Quan\\nCheng: Conceptualization. Ting Wei: Conceptualiza-\\ntion. Shuofeng Yuan: Conceptualization. Peng Luo:\\nConceptualization.\\nAFFILIATIONS\\n1Donghai County People's Hospital (Affiliated Kangda College of\\nNanjing Medical University), Lianyungang, China\\n2Department of Oncology, Zhujiang Hospital, Southern Medical\\nUniversity, Guangzhou, China\\n3Department of Urology, Changhai hospital, Naval Medical University\\n(Second Military Medical University), Shanghai, China\\n4Li Ka Shing Faculty of Medicine, The University of Hong Kong, Hong\\nKong, China\\n5Cancer Centre and Institute of Translational Medicine, Faculty of\\nHealth Sciences, University of Macau, Macau, China\\n6Western (Chongqing) Institut for Digital‐Intelligent Medicine,\\nChongqing, China\\n7Clinical Medical College, Southwest Medical University, Luzhou,\\nChina\\n8Department of Lung Cancer, Tianjin Lung Cancer Center, National\\nClinical Research Center for Cancer, Key Laboratory of Cancer\\nPrevention and Therapy, Tianjin's Clinical Research Center for Cancer,\\nTianjin Medical University Cancer Institute and Hospital, Tianjin, China\\n9Liver Disease Center of Integrated Traditional Chinese and Western\\nMedicine, Department of Radiology, Zhongda Hospital, Medical School,\\nSoutheast University, Nurturing Center of Jiangsu Province for State\\nLaboratory of AI Imaging & Interventional Radiology (Southeast\\nUniversity), Nanjing, China\\nMICROBIOME IN CANCER | 65 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 65, 'page_label': '66'}, page_content=\"10Key Laboratory of Oral Diseases Research of Anhui Province,\\nCollege & Hospital of Stomatology, Anhui Medical University, Hefei,\\nChina\\n11College of Life Science and Technology, Huazhong University of\\nScience and Technology, Hubei, China\\n12Department of Pathogenic Microbiology and Immunology,\\nSchool of Basic Medical Sciences, Xi'an Jiaotong University, Xi'an,\\nChina\\n13Department of Oncology, Nanfang Hospital, Southern Medical\\nUniversity, Guangzhou, China\\n14Cancer Center, the Sixth Affiliated Hospital, School of Medicine,\\nSouth China University of Technology, Foshan, China\\n15State Key Laboratory of Oncology in South China, Guangdong\\nProvincial Clinical Research Center for Cancer, Sun Yat‐sen University\\nCancer Center, Guangzhou, China\\n16Department of Urology, State Key Laboratory of Oncology in\\nSouthern China, Sun Yat‐sen University Cancer Center, Guangdong\\nProvincial Clinical Research Center for Cancer, Guangzhou, China\\n17Department of Oncology, Ruijin Hospital, Shanghai Jiao Tong\\nUniversity School of Medicine, Shanghai, China\\n18Department of Medicine II, LMU University Hospital, Munich,\\nGermany\\n19Molecular and Experimental Surgery, Clinic for General‐, Visceral‐,\\nVascular‐ and Transplantation Surgery, Medical Faculty and University\\nHospital Magdeburg, Otto‐von‐Guericke University, Magdeburg,\\nGermany\\n20Institute of Logic and Computation, Vienna University of Technology,\\nVienna, Austria\\n21Ludwig Institute for Cancer Research, Nuffield Department of\\nMedicine, University of Oxford, Oxford, UK\\n22Department of Data Visualization, Faculty of Informatics, University\\nof Debrecen, Debrecen, Hungary\\n23Faculty of Dentistry, University of Debrecen, Debrecen, Hungary\\n24Hillman Cancer Center, UPMC, Pittsburgh, Pennsylvania, USA\\n25Department of Biochemistry & Molecular Medicine, GW Cancer\\nCenter, George Washington University, Washington, DC, USA\\n26Department of Molecular and Cellular Biology, Baylor College of\\nMedicine, Houston, Texas, USA\\n27Center for Precision Health, McWilliams School of Biomedical\\nInformatics, The University of Texas Health Science Center at Houston,\\nHouston, Texas, USA\\n28The School of Public Health and Preventive Medicine, Monash\\nUniversity, Melbourne, Victoria, Australia\\n29Division of Biomedical Engineering, James Watt School of\\nEngineering, University of Glasgow, Glasgow, UK\\n30Department of Biological, Chemical and Pharmaceutical Sciences and\\nTechnologies, University of Palermo, Palermo, Italy\\n31Department of Immunology, Faculty of Medicine, University of\\nDebrecen, Debrecen, Hungary\\n32Department of Medicine, The MetroHealth System, Case Western\\nReserve University, Cleveland, Ohio, USA\\n33Department of Oral and Cranio‐Maxillofacial Su rgery, Shanghai\\nNinth People's Hospital, College of Stomatology, Shanghai Jiao\\nTong University School of Medicine, National Clinical Research\\nCenter for Oral Diseases, Shanghai Key Laboratory of Stomatology\\nand Shanghai Research Institute of Stomatology, Shanghai, China\\n34Department of Radiation Oncology, Zhongshan Hospital Affiliated to\\nFudan University, Shanghai, China\\n35Institute of Basic Medical Sciences, Chinese Academy of Medical\\nSciences and Peking Union Medical College, Beijing, China\\n36MoE Frontiers Science Center for Precision Oncology, University of\\nMacau, Macau, China\\n37Department of Neurosurgery, Xiangya Hospital, Central South\\nUniversity, Changsha, China\\n38National Clinical Research Center for Geriatric Disorders, Xiangya\\nHospital, Central South University, Changsha, China\\n39Department of Infectious Disease and Microbiology, The University of\\nHong Kong‐Shenzhen Hospital, Shenzhen, China\\n40Department of Microbiology, State Key Laboratory of Emerging\\nInfectious Diseases, Carol Yu Centre for Infection, School of Clinical\\nMedicine, Li Ka Shing Faculty of Medicine, The University of Hong\\nKong, Hong Kong, China\\nACKNOWLEDGMENTS\\nThis work was supported by Macao Science and Technology\\nDevelopment Fund (FDCT) grants (No. 0073/2021/A2, and\\n0009/2022/AKP); Multiyear research grant (MYRG) (No.\\nMYRG‐GRG2023‐00150‐FHS‐UMDF). This work was sup-\\nported by grants from the Natural Science Foundation of\\nGuangdong Province (No. 2021A1515012593), the National\\nNatural Science Foundation of China (No. 82373129,\\n82172750, 82172811 and 82260546), the Guangdong Basic\\nand Applied Basic Research Foundation (Guangdong‐\\nGuangzhou Joint Funds) (No. 2022A1515111212), the\\nScience and Technology Program of Guangzhou (No.\\n2023A04J1257), and Hunan YouthScience and Technology\\nTalent Project (No. 2023RC3074).\\nCONFLICT OF INTEREST STATEMENT\\nThe authors declare no conflict of interest.\\nDATA AVAILABILITY STATEMENT\\nNo new data and scripts were used for this review.\\nSupplementary information (graphical abstract, slides,\\nvideos, Chinese translated version, and update materials)\\nis available online DOI orhttp://www.imeta.science/.\\nETHICS STATEMENT\\nNo animals or humans were involved in this study.\\nORCID\\nPengpeng Zhang\\nhttps://orcid.org/0009-0004-\\n6951-2634\\nHengguo Zhang https://orcid.org/0000-0002-4438-8348\\nJindong Xie https://orcid.org/0000-0002-2482-7358\\nXinpei Deng https://orcid.org/0000-0001-9973-2559\\nZhengrui Li https://orcid.org/0000-0003-4923-0088\\nHailin Tang https://orcid.org/0000-0002-3206-782X\\nBufu Tang https://orcid.org/0000-0003-3525-4433\\n66 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 66, 'page_label': '67'}, page_content='Zaoqu Liu https://orcid.org/0000-0002-0452-742X\\nQuan Cheng https://orcid.org/0000-0003-2401-5349\\nPeng Luo https://orcid.org/0000-0002-8215-2045\\nREFERENCES\\n1. Human Microbiome Project Consortium. 2012. “Structure,\\nFunction and Diversity of the Healthy Human Microbiome.”\\nNature 486: 207–14. https://doi.org/10.1038/nature11234\\n2. Kuziel, Gavin A., Seth Rakoff‐Nahoum. 2022. “The Gut Mi-\\ncrobiome.” Current Biology 32: R257–64. https://doi.org/10.\\n1016/j.cub.2022.02.023\\n3. Virtanen, Seppo, Schahzad Saqib, Tinja Kanerva,\\nRebecka Ventin‐Holmberg, Pekka Nieminen, Tiina Holster,\\nIlkka Kalliala, Anne Salonen. 2024.“Metagenome‐Validated\\nCombined Amplicon Sequencing and Text Mining ‐Based\\nAnnotations for Simultaneous Profiling of Bacteria and\\nFungi: Vaginal Microbiota and Mycobiota in Healthy\\nWomen. ” Microbiome 12: 273. https://doi.org/10.1186/\\ns40168‐024‐01993‐9\\n4. Avershina, Ekaterina, Arfa Irej Qureshi, Hanne C. Winther‐\\nLarsen, Trine B. Rounge. 2025.“Challenges in Capturing the\\nMycobiome From Shotgun Metagenome Data: Lack of Soft-\\nware and Databases.” Microbiome 13: 66. https://doi.org/10.\\n1186/s40168‐025‐02048‐3\\n5. Rooks, Michelle G., Wendy S. Garrett. 2016.“Gut Microbiota,\\nMetabolites and Host Immunity. ” Nature Reviews\\nImmunology 16: 341–52. https://doi.org/10.1038/nri.2016.42\\n6. Shea, Allyson E., Valerie S. Forsyth, Jolie A. Stocki,\\nTaylor J. Mitchell, Arwen E. Frick‐Cheng, Sara N. Smith,\\nSicily L. Hardy, Harry L. T. Mobley. 2024.“Emerging Roles\\nfor ABC Transporters as Virulence Factors in Uropathogenic\\nEscherichia coli.” Proceedings of the National Academy of\\nSciences of the United States of America121: e2310693121.\\nhttps://doi.org/10.1073/pnas.2310693121\\n7. Xu, Yi, Xiahui Wu, Yan Li, Xuejie Liu, Lijian Fang,\\nZiyu Jiang. 2024. “Probiotics and the Role of Dietary Sub-\\nstrates in Maintaining the Gut Health: Use of Live Microbes\\nand Their Products for Anticancer Effects Against Colorectal\\nCancer.” Journal of Microbiology and Biotechnology 34:\\n1933–46.\\nhttps://doi.org/10.4014/jmb.2403.03056\\n8. Montassier, Emmanuel, Rafael Valdés ‐Mas, Eric Batard,\\nNiv Zmora, Mally Dori‐Bachash, Jotham Suez, Eran Elinav.\\n2021. “Probiotics Impact the Antibiotic Resistance Gene\\nReservoir Along the Human GI Tract in a Person‐Specific\\nand Antibiotic‐Dependent Manner.” Nature Microbiology 6:\\n1043–54. https://doi.org/10.1038/s41564‐021‐00920‐0\\n9. Bajaj, Jasmohan S., Siew C. Ng, Bernd Schnabl. 2022.\\n“Promises of Microbiome ‐Based Therapies. ” Journal of\\nHepatology 76: 1379–91. https://doi.org/10.1016/j.jhep.2021.\\n12.003\\n10. Serrano ‐Del Valle, Alfonso, Javier Naval, Alberto Anel,\\nIsabel Marzo. 2020.“Novel Forms of Immunomodulation for\\nCancer Therapy.” Trends in Cancer 6: 518–32. https://doi.\\norg/10.1016/j.trecan.2020.02.015\\n11. Negrón ‐Figueroa, Dalissa, Lauren E. Colbert. 2024.“Mech-\\nanisms by Which the Intratumoral Microbiome May Poten-\\ntiate Immunotherapy Response. ” Journal of Clinical\\nOncology 42: 3350–2. https://doi.org/10.1200/JCO.24.00908\\n12. Dzutsev, Amiran K., Romina S. Goldszmid. 2024.“Towards\\nEnhancing the Predictive Value of the Microbiota for Cancer\\nImmunotherapy.” Trends in Cancer 10: 771–3. https://doi.\\norg/10.1016/j.trecan.2024.07.007\\n13. Peppercorn, MARK A., PETER Goldman. 1972.“The Role of\\nIntestinal Bacteria in the Metabolism of Salicylazosulfapyr-\\nidine.” The Journal of Pharmacology and Experimental\\nTherapeutics 181: 555 –62, https://doi.org/10.1016/S0022 ‐\\n3565(25)29238‐2\\n14. Ley, Ruth E., Peter J. Turnbaugh, Samuel Klein,\\nJeffrey I. Gordon. 2006.“Human Gut Microbes Associated With\\nObesity.” Nature 444: 1022–3. https://doi.org/10.1038/4441022a\\n15. Qin, Youwen, Aki S. Havulinna, Yang Liu, Pekka Jousilahti,\\nScott C. Ritchie, Alex Tokolyi, Jon G. Sanders, et al.2022.\\n“Combined Effects of Host Genetics and Diet on Human Gut\\nMicrobiota and Incident Disease in a Single Population\\nCohort.” Nature Genetics54: 134–42. https://doi.org/10.1038/\\ns41588‐021‐00991‐z\\n16. Nakatsu, Geicho, Natalia Andreeva, Meghan H. MacDonald,\\nWendy S. Garrett. 2024.“Interactions Between Diet and Gut\\nMicrobiota in Cancer. ” Nature Microbiology 9: 1644 –54.\\nhttps://doi.org/10.1038/s41564‐024‐01736‐4\\n17. Lei, Chao, Cong Liu, Yuling Peng, Yu Zhan,\\nXiaoming Zhang, Ting Liu, Zhihua Liu. 2023.“A High‐Salt\\nDiet Induces Synaptic Loss and Memory Impairment via Gut\\nMicrobiota and Butyrate in Mice.” iMeta 2: e97. https://doi.\\norg/10.1002/imt2.97\\n18. Huang, Yina, Jinxin Liu, Hein Min Tun, Catherine Stanton,\\nTingtao Chen, Hani El‐Nezami, Hua Wei, Mingfu Wang,\\nQinglong Wu. 2022. “Gut Microbiota Insights Into Human\\nAdaption to High‐Plateau Diet.” iMeta 1: e6.https://doi.org/\\n10.1002/imt2.6\\n19. Dapa, Tanja, Ricardo Serotte Ramiro, Miguel Filipe Pedro,\\nIsabel Gordo, Karina Bivar Xavier. 2022. “Diet Leaves a\\nGenetic Signature in a Keystone Member of the Gut Micro-\\nbiota.” Cell Host & Microbe30: 183–99.e10. https://doi.org/10.\\n1016/j.chom.2022.01.002\\n20. NIH HMP Working Group, Jane Peterson, Garges, Susan,\\nMaria Giovanni, Pamela McInnes, Lu Wang,\\nSchloss, Jeffery A., Vivien Bonazzi, McEwen Jean E., et al.\\n2009. “The NIH Human Microbiome Project. ” Genome\\nResearch 19: 2317–23. https://doi.org/10.1101/gr.096651.109\\n21. Qin, Junjie, Ruiqiang Li, Jeroen Raes, Manimozhiyan\\nArumugam, Kristoffer Solvsten Burgdorf, Chaysavanh\\nManichanh, Trine Nielsen, et al. 2010. “A Human Gut\\nMicrobial Gene Catalogue Established by Metagenomic\\nSequencing.” Nature 464: 59\\n–65. https://doi.org/10.1038/\\nnature08821\\n22. Yatsunenko, Tanya, Federico E. Rey, Mark J. Manary,\\nIndi Trehan, Maria Gloria Dominguez ‐Bello, Monica\\nContreras, Magda Magris, et al. 2012.“Human Gut Micro-\\nbiome Viewed Across Age and Geography. ” Nature 486:\\n222–7. https://doi.org/10.1038/nature11053\\n23. McDonald, Daniel, Embriette Hyde, Justine W. Debelius,\\nJames T. Morton, Antonio Gonzalez, Gail Ackermann,\\nAlexander A. Aksenov, et al. 2018.“American Gut: An Open\\nPlatform for Citizen Science Microbiome Research. ”\\nmSystems 3: e00031 ‐18. https://doi.org/10.1128/mSystems.\\n00031‐18\\nMICROBIOME IN CANCER | 67 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 67, 'page_label': '68'}, page_content='24. Routy, Bertrand, Emmanuelle Le Chatelier, Lisa Derosa,\\nConnie P. M. Duong, Maryam Tidjani Alou, Romain Daillère,\\nAurélie Fluckiger, et al. 2018.“Gut Microbiome Influences\\nEfficacy of PD‐1‐Based Immunotherapy Against Epithelial\\nTumors.” Science 359: 91–7. https://doi.org/10.1126/science.\\naan3706\\n25. Gagliani, Nicola, Bo Hu, Samuel Huber, Eran Elinav,\\nRichard A. Flavell. 2014.“The Fire Within: Microbes Inflame\\nTumors.” Cell 157: 776–83. https://doi.org/10.1016/j.cell.2014.\\n03.006\\n26. Nejman, Deborah, Ilana Livyatan, Garold Fuks, Nancy Gavert,\\nYaara Zwang, Leore T. Geller, Aviva Rotter‐Maskowitz, et al.\\n2020. “The Human Tumor Microbiome Is Composed of\\nTumor Type‐Specific Intracellular Bacteria.” Science 368:\\n973–80. https://doi.org/10.1126/science.aay9189\\n27. Cogdill, Alexandria P., Pierre Olivier Gaudreau, Reetakshi Arora,\\nVancheswaran Gopalakrishnan, Jennifer A. Wargo. 2018.“The\\nImpact of Intratumoral and Gastrointestinal Microbiota on Sys-\\ntemic Cancer Therapy.” Trends in Immunology 39: 900–20.\\nhttps://doi.org/10.1016/j.it.2018.09.007\\n28. Wong ‐Rolle, Abigail, Haohan Karen Wei, Chen Zhao,\\nChengcheng Jin. 2021. “Unexpected Guests in the Tumor\\nMicroenvironment: Microbiome in Cancer.” Protein & Cell\\n12: 426–35. https://doi.org/10.1007/s13238‐020‐00813‐8\\n29. Xie, Yifan, Feng Xie, Xiaoxue Zhou, Lei Zhang, Bing Yang,\\nJun Huang, Fangwei Wang, et al. 2022. “Microbiota in\\nTumors: From Understanding to Application. ” Advanced\\nScience (Weinheim, Baden ‐Wurttemberg, Germany) 9:\\ne2200470. https://doi.org/10.1002/advs.202200470\\n30. Schwabe, Robert F., Christian Jobin. 2013.“The Microbiome\\nand Cancer.” Nature Reviews Cancer13: 800–12. https://doi.\\norg/10.1038/nrc3610\\n31. Castanheira, Cristina Paula, Mayara Luciana Sallas,\\nRafaella Almeida Lima Nunes, Noely Paula Cristina Lorenzi,\\nLara Termini. 2021. “Microbiome and Cervical Cancer. ”\\nPathobiology 88: 187–97. https://doi.org/10.1159/000511477\\n32. Cao, Yaqi, Hui Xia, Xueyun Tan, Chunwei Shi, Yanling Ma,\\nDaquan Meng, Mengmeng Zhou, et al. 2024.“Intratumoural\\nMicrobiota: A New Frontier in Cancer Development and\\nTherapy.” Signal Transduction and Targeted Therapy9: 15.\\nhttps://doi.org/10.1038/s41392‐023‐01693‐0\\n33. Scott, Alasdair J., James L. Alexander, Claire A. Merrifield,\\nDavid Cunningham, Christian Jobin, Robert Brown,\\nJohn Alverdy, et al. 2019.“International Cancer Microbiome\\nConsortium Consensus Statement on the Role of the Human\\nMicrobiome in Carcinogenesis.” Gut 68: 1624–32. https://doi.\\norg/10.1136/gutjnl‐2019‐318556\\n34. Zhao, Lin‐Yong, Jia‐Xin Mei, Gang Yu, Lei Lei, Wei‐Han Zhang,\\nKai Liu, Xiao‐Long Chen, et al. 2023.“Role of the Gut Micro-\\nbiota in Anticancer Therapy: From Molecular Mechanisms to\\nClinical Applications.” Signal Transduction and Targeted\\nTherapy 8: 201.https://doi.org/10.1038/s41392‐023‐01406‐7\\n35. Yang, Li, Aitian Li, Ying Wang, Yi Zhang. 2023. “In-\\ntratumoral Microbiota: Roles in Cancer Initiation, Develop-\\nment and Therapeutic Efficacy.” Signal Transduction and\\nTargeted Therapy 8: 35. https://doi.org/10.1038/s41392‐022‐\\n01304‐4\\n36. Mou, Weiming, Zhixing Deng, Lingxuan Zhu, Aimin Jiang,\\nAnqi Lin, Liling Xu, Gengwen Deng, et al. 2025\\n“Intratumoral Mycobiome Heterogeneity Influences the\\nTumor Microenvironment and Immunotherapy Outcomes in\\nRenal Cell Carcinoma. ” Science Advances 11:eadu1727.\\nhttps://doi.org/10.1126/sciadv.adu1727\\n37. Li, Lei, Shouhua He, Boyi Liao, Manchun Wang,\\nHuimin Lin, Ben Hu, Xinyue Lan, et al. 2024. “Orally\\nAdministrated Hydrogel Harnessing Intratumoral Micro-\\nbiome and Microbiota ‐\\nRelated Immune Responses for\\nPotentiated Colorectal Cancer Treatment. ” Research\\n(Washington, D.C.) 7: 0364. https://doi.org/10.34133/\\nresearch.0364\\n38. Liu, Weici, Zheshun Pi, Ning‐Ning Liu, Wenjun Mao. 2024.\\n“Into the Era of Mycobiome ‐Driven Cancer Research. ”\\nTrends in Cancer10: 389–92. https://doi.org/10.1016/j.trecan.\\n2024.02.009\\n39. Soto, David, Christine Song, Margaret E. McLaughlin‐Drubin.\\n2017. “Epigenetic Alterations in Human Papillomavirus ‐\\nAssociated Cancers.” Viruses 9: 248. https://doi.org/10.3390/\\nv9090248\\n40. Amaya Arbeláez, María Isabel, Ana Carolina Alves\\nDe Paula E. Silva, Geovana Navegante, Valeria Valente,\\nPaula Aboud Barbugli, Carlos Eduardo Vergani. 2021.“Proto‐\\nOncogenes and Cell Cycle Gene Expression in Normal and\\nNeoplastic Oral Epithelial Cells Stimulated With Soluble Factors\\nFrom Single and Dual Biofilms ofCandida albicansand Staph-\\nylococcus aureus.” Frontiers in Cellular and Infection Microbiology\\n11: 627043.https://doi.org/10.3389/fcimb.2021.627043\\n41. Górska, Agata, Dawid Przystupski, Magdalena J. Niemczura,\\nJulita Kulbacka. 2019.“Probiotic Bacteria: A Promising Tool\\nin Cancer Prevention and Therapy.” Current Microbiology76:\\n939–49. https://doi.org/10.1007/s00284‐019‐01679‐8\\n42. Lili, Zhao, Wei Junyan, Zhao Hongfei, Zhu Baoqing,\\nZhang Bolin. 2018. “Detoxification of Cancerogenic Com-\\npounds by Lactic Acid Bacteria Strains.” Critical Reviews in\\nFood Science and Nutrition 58: 2727–42. https://doi.org/10.\\n1080/10408398.2017.1339665\\n43. Zhong, Mengya, Yubo Xiong, Jiabao Zhao, Zhi Gao,\\nJingsong Ma, Zhengxin Wu, Yongxi Song, Xuehui Hong.\\n2021. “Candida albicansDisorder Is Associated With Gastric\\nCarcinogenesis.” Theranostics 11: 4945–56. https://doi.org/\\n10.7150/thno.55209\\n44. Guan, Shi ‐Wei, Quan Lin, Xi‐Dong Wu, Hai\\n‐Bo Yu. 2023.\\n“Weighted Gene Coexpression Network Analysis and\\nMachine Learning Reveal Oncogenome Associated Micro-\\nbiome Plays an Important Role in Tumor Immunity and\\nPrognosis in Pan‐Cancer.” Journal of Translational Medicine\\n21: 537. https://doi.org/10.1186/s12967‐023‐04411‐0\\n45. Liu, Le, Liping Liang, YingJie Luo, Jimin Han, Di Lu,\\nRuiJun Cai, Gautam Sethi, Shijie Mai. 2024.“Unveiling the\\nPower of Gut Microbiome in Predicting Neoadjuvant Im-\\nmunochemotherapy Responses in Esophageal Squamous\\nCell Carcinoma. ” Research (Washington, D.C.) 7: 0529.\\nhttps://doi.org/10.34133/research.0529\\n46. Lehouritis, Panos, Joanne Cummins, Michael Stanton,\\nCarola T. Murphy, Florence O. McCarthy, Gregor Reid,\\nCamilla Urbaniak, William L. Byrne, Mark Tangney. 2015.\\n“Local Bacteria Affect the Efficacy of Chemotherapeutic\\nDrugs.” Scientific Reports 5: 14554. https://doi.org/10.1038/\\nsrep14554\\n68 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 68, 'page_label': '69'}, page_content=\"47. Badgeley, Aja, Hina Anwar, Karan Modi, Paige Murphy,\\nAshakumary Lakshmikuttyamma. 2021.“Effect of Probiotics\\nand Gut Microbiota on Anti ‐Cancer Drugs: Mechanistic\\nPerspectives.” Biochimica et Biophysica Acta (BBA)—Reviews\\non Cancer 1875: 188494. https://doi.org/10.1016/j.bbcan.\\n2020.188494\\n48. Zhu, Zemin, Bo Yi, Zikai Tang, Xun Chen, Ming Li, Tao Xu,\\nZhijian Zhao, Caixi Tang. 2023. “Lactobacillus casei Com-\\nbined With Lactobacillus Reuteri Alleviate Pancreatic Cancer\\nby Inhibiting TLR4 to Promote Macrophage M1 Polarization\\nand Regulate Gut Microbial Homeostasis.” BMC Cancer 23:\\n1044. https://doi.org/10.1186/s12885‐023‐11557‐z\\n49. Khoruts, Alexander, Christopher Staley, Michael J.\\nSadowsky. 2021. “Faecal Microbiota Transplantation for\\nClostridioides difficile : Mechanisms and Pharmacology. ”\\nNature Reviews Gastroenterology & Hepatology 18: 67–80.\\nhttps://doi.org/10.1038/s41575‐020‐0350‐4\\n50. Dai, Jia ‐Hao, Xi ‐Rong Tan, Han Qiao, Na Liu. 2024.\\n“Emerging Clinical Relevance of Microbiome in Cancer:\\nPromising Biomarkers and Therapeutic Targets.” Protein &\\nCell 15: 239–60. https://doi.org/10.1093/procel/pwad052\\n51. Ting, Nick Lung‐Ngai, Harry Cheuk‐Hay Lau, Jun Yu. 2022.\\n“Cancer Pharmacomicrobiomics: Targeting Microbiota to\\nOptimise Cancer Therapy Outcomes. ” Gut 71: 1412 –25.\\nhttps://doi.org/10.1136/gutjnl‐2021‐326264\\n52. Matson, Vyara, Carolina Soto Chervin, Thomas F. Gajewski.\\n2021. “Cancer and the Microbiome ‐Influence of the\\nCommensal Microbiota on Cancer, Immune Responses, and\\nImmunotherapy.” Gastroenterology 160: 600–13. https://doi.\\norg/10.1053/j.gastro.2020.11.041\\n53. Lee, Mong\\n‐Hong. 2021. “Harness the Functions of Gut\\nMicrobiome in Tumorigenesis for Cancer Treatment. ”\\nCancer Communications41: 937–67. https://doi.org/10.1002/\\ncac2.12200\\n54. Queen, Jessica, Fyza Shaikh, Cynthia L. Sears. 2023.\\n“Understanding the Mechanisms and Translational Impli-\\ncations of the Microbiome for Cancer Therapy Innovation.”\\nNature Cancer 4: 1083–94. https://doi.org/10.1038/s43018‐\\n023‐00602‐2\\n55. Park, Elizabeth M., Manoj Chelvanambi, Neal Bhutiani,\\nGuido Kroemer, Laurence Zitvogel, Jennifer A. Wargo. 2022.\\n“Targeting the Gut and Tumor Microbiota in Cancer. ”\\nNature Medicine28: 690–703. https://doi.org/10.1038/s41591‐\\n022‐01779‐2\\n56. Lou, Xinyu, Zhichao Chen, Zhonggui He, Mengchi Sun,\\nJin Sun. 2021. “Bacteria‐Mediated Synergistic Cancer Ther-\\napy: Small Microbiome Has a Big Hope.” Nano‐Micro Letters\\n13: 37.https://doi.org/10.1007/s40820‐020‐00560‐9\\n57. Glitza, Isabella C., Yongwoo David Seo, Christine N. Spencer,\\nJennifer R. Wortman, Elizabeth M. Burton, Farah A. Alayli,\\nChristopher P. Loo, et al. 2024. “Randomized Placebo‐\\nControlled, Biomarker‐Stratified Phase Ib Microbiome Modu-\\nlation in Melanoma: Impact of Antibiotic Preconditioning on\\nMicrobiome and Immunity.” Cancer Discovery 14: 1161–75.\\nhttps://doi.org/10.1158/2159‐8290.CD‐24‐0066\\n58. Fernandes, Miriam R., Poonam Aggarwal, Raquel G. F. Costa,\\nAlicia M. Cole, Giorgio Trinchieri. 2022.“Targeting the Gut\\nMicrobiota for Cancer Therapy.” Nature Reviews Cancer22:\\n703–22. https://doi.org/10.1038/s41568‐022‐00513\\n‐x\\n59. Bhatt, Aadra P., Matthew R. Redinbo, Scott J. Bultman. 2017.\\n“The Role of the Microbiome in Cancer Development and\\nTherapy.” CA: A Cancer Journal for Clinicians67: 326–44.\\nhttps://doi.org/10.3322/caac.21398\\n60. Zhang, Xiang, Olabisi Oluwabukola Coker, Eagle Sh Chu,\\nKaili Fu, Harry C. H. Lau, Yi‐Xiang Wang, Anthony W. H.\\nChan, et al. 2021. “Dietary Cholesterol Drives Fatty Liver‐\\nAssociated Liver Cancer by Modulating Gut Microbiota and\\nMetabolites.” Gut 70: 761–74. https://doi.org/10.1136/gutjnl‐\\n2019‐319664\\n61. Wong, Sunny H., Jun Yu. 2019. “Gut Microbiota in Colo-\\nrectal Cancer: Mechanisms of Action and Clinical Applica-\\ntions.” Nature Reviews Gastroenterology & Hepatology 16:\\n690–704. https://doi.org/10.1038/s41575‐019‐0209‐8\\n62. Ohtani, Naoko, Eiji Hara. 2021. “Gut‐Liver Axis‐Mediated\\nMechanism of Liver Cancer: A Special Focus on the Role of\\nGut Microbiota.” Cancer Science 112: 4433–43. https://doi.\\norg/10.1111/cas.15142\\n63. Cheng, Wing Yin Wu, Chun‐Ying Yu, Jun. 2020.“The Role of\\nGut Microbiota in Cancer Treatment: Friend or Foe?” Gut 69:\\n1867–76. https://doi.org/10.1136/gutjnl‐2020‐321153\\n64. Cullin, Nyssa, Camila Azevedo Antunes, Ravid Straussman,\\nChristoph?K. Stein‐Thoeringer, Eran Elinav. 2021. “Micro-\\nbiome and Cancer.” Cancer Cell39: 1317–41. https://doi.org/\\n10.1016/j.ccell.2021.08.006\\n65. Zaidi, Ali H., Lori A. Kelly, Rachael E. Kreft, Mark Barlek,\\nAshten N. Omstead, Daisuke Matsui, Natalie H. Boyd, et al.\\n2016. “Associations of Microbiota and Toll‐\\nLike Receptor\\nSignaling Pathway in Esophageal Adenocarcinoma.” BMC\\nCancer 16: 52.https://doi.org/10.1186/s12885‐016‐2093‐8\\n66. Xue, Xiaoyu, Rui Li, Zhenni Chen, Guiyu Li, Bisheng Liu,\\nShanshan Guo, Qianhua Yue, et al. 2023.“The Role of the\\nSymbiotic Microecosystem in Cancer: Gut Microbiota, Me-\\ntabolome, and Host Immunome.” Frontiers in Immunology\\n14: 1235827. https://doi.org/10.3389/fimmu.2023.1235827\\n67. Jia, Xiaodong, Shanshan Lu, Zhen Zeng, Qingyan Liu,\\nZheng Dong, Chen, Yan Zhu, Zhenyu, et al. 2020.“Charac-\\nterization of Gut Microbiota, Bile Acid Metabolism, and\\nCytokines in Intrahepatic Cholangiocarcinoma.” Hepatology\\n71: 893–906. https://doi.org/10.1002/hep.30852\\n68. Rubenstein, Joel H., John M. Inadomi, James Scheiman,\\nPhilip Schoenfeld, Henry Appelman, Min Zhang, Val Metko,\\nJohn Y. Kao. 2014.“Association Between Helicobacter Pylori and\\nBarrett's Esophagus, Erosive Esophagitis, and Gastroesophageal\\nReflux Symptoms.” Clinical Gastroenterology and Hepatology12:\\n239–45. https://doi.org/10.1016/j.cgh.2013.08.029\\n69. Pei, Jiaxin, Chaoxu Zhang, Qian Zhang, Hao Yu,\\nHuiya Yuan, Yufu Guo, Hui Shen, et al. 2024.“Probiotics\\nAlleviate Chronic Ethanol Exposure‐Induced Anxiety‐Like\\nBehavior and Hippocampal Neuroinflammation in Male\\nMice Through Gut Microbiota‐Derived Extracellular Vesi-\\ncles.” Journal of Nanobiotechnology22: 730. https://doi.org/\\n10.1186/s12951‐024‐03017‐y\\n70. Lai, Hao, Yunfeng Li, Yafang He, Fangyao Chen, Baibing Mi,\\nJunqi Li, Jiawen Xie, et al. 2023.“Effects of Dietary Fibers or\\nProbiotics on Functional Constipation Symptoms and Roles\\nof Gut Microbiota: A Double‐Blinded Randomized Placebo\\nTrial.” Gut Microbes 15: 2197837. https://doi.org/10.1080/\\n19490976.2023.2197837\\nMICROBIOME IN CANCER | 69 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 69, 'page_label': '70'}, page_content=\"71. Khoruts, Alexander, Diane E. Hoffmann, Robert A. Britton.\\n2020. “Probiotics: Promise, Evidence, and Hope. ”\\nGastroenterology 159: 409 –13. https://doi.org/10.1053/j.\\ngastro.2020.05.058\\n72. Van Rossum, Thea, Annette Haiß, Rebecca L. Knoll, Janina\\nMarißen, Daniel Podlesny, Julia Pagel, Marina Bleskina,\\net al. 2024. “Bifidobacterium and Lactobacillus Probiotics\\nand Gut Dysbiosis in Preterm Infants: The PRIMAL Ran-\\ndomized Clinical Trial. ” JAMA Pediatrics 178: 985 –95.\\nhttps://doi.org/10.1001/jamapediatrics.2024.2626\\n73. Castro ‐Mejía, Josué L., Sinéad O'Ferrall, Łukasz Krych,\\nElaine O'Mahony, Hanifa Namusoke, Betty Lanyero,\\nWitold Kot, et al. 2020. “Restitution of Gut Microbiota in\\nUgandan Children Administered With Probiotics (Lactoba-\\ncillus rhamnosus GG and Bifidobacterium animalis subsp.\\nlactis BB ‐12) During Treatment for Severe Acute Mal-\\nnutrition.” Gut Microbes11: 855–67. https://doi.org/10.1080/\\n19490976.2020.1712982\\n74. Ralser, Anna, Alisa Dietl, Sebastian Jarosch, Veronika\\nEngelsberger, Andreas Wanisch, Klaus Peter Janssen, Moritz\\nMiddelhoff, et al. 2023.“Helicobacter pyloriPromotes Colorectal\\nCarcinogenesis by Deregulating Intestinal Immunity and\\nInducing a Mucus‐Degrading Microbiota Signature.” Gut 72:\\n1258–70. https://doi.org/10.1136/gutjnl‐2022‐328075\\n75. Luo, Shiqi, Jinlong Ru, Mohammadali Khan Mirzaei,\\nJinling Xue, Xue Peng, Anna Ralser, Joshua Lemuel Hadi,\\net al. 2024.“Helicobacter pylori Infection Alters Gut Virome\\nby Expanding Temperate Phages Linked to Increased Risk of\\nColorectal Cancer.” Gut 73: 1592–5. https://doi.org/10.1136/\\ngutjnl‐2023‐330362\\n76. Butt, Julia, Matthew G. Varga, William J. Blot, Lauren Teras,\\nKala Visvanathan, Loïc Le Marchand, Christopher Haiman,\\net al. 2019. “Serologic Response to Helicobacter pylori Pro-\\nteins Associated With Risk of Colorectal Cancer Among\\nDiverse Populations in the United States.” Gastroenterology\\n156: 175–86.e2. https://doi.org/10.1053/j.gastro.2018.09.054\\n77. Hansen S., K. K. Melby, S. Aase, E. Jellum, S.E. Vollset 1999.\\n“Helicobacter pyloriInfection and Risk of Cardia Cancer and\\nNon‐Cardia Gastric Cancer. A Nested Case‐Control Study.”\\nScandinavian Journal of Gastroenterology34: 353–60. https://\\ndoi.org/10.1080/003655299750026353\\n78. Wu, Xiao‐Wei, Hong‐Zan Ji, Miao‐Fang Yang, Lin Wu, Fang‐\\nYu Wang. 2015. “Helicobacter pylori Infection and Inflam-\\nmatory Bowel Disease in Asians: A Meta‐Analysis.” World\\nJournal of Gastroenterology 21: 4750–6. https://doi.org/10.\\n3748/wjg.v21.i15.4750\\n79. Gutierrez ‐Angulo, Melva, Maria de la Luz Ayala‐Madrigal,\\nJose Miguel Moreno ‐Ortiz, Jorge Peregrina ‐Sandoval,\\nFernando Daniel Garcia‐Ayala. 2023. “Microbiota Composi-\\ntion and Its Impact on DNA Methylation in Colorectal\\nCancer.” Frontiers in Genetics14: 1037406.https://doi.org/10.\\n3389/fgene.2023.1037406\\n80. Li, Ting, Zhanyi Zhao, Meichang Peng, Lu Zhang,\\nCheng Wang, Feiyang Luo, Meiqin Zeng, et al. 2025.“Multi‐\\nOmics Analysis Reveals the Interplay Between Intratumoral\\nBacteria and Glioma.” mSystems 10: e0045724. https://doi.\\norg/10.1128/msystems.00457‐24\\n81. Gur, Chamutal, Yara Ibrahim, Batya Isaacson,\\nRachel Yamin, Jawad Abed, Moriya Gamliel, Jonatan Enk,\\net al. 2015. “Binding of the Fap2 Protein ofFusobacterium\\nnucleatum to Human Inhibitory Receptor TIGIT Protects\\nTumors From Immune Cell Attack.” Immunity 42: 344–55.\\nhttps://doi.org/10.1016/j.immuni.2015.01.010\\n82. Goodwin, Andrew C., Christina E. Destefano Shields,\\nShaoguang Wu, David L. Huso, XinQun Wu, Tracy R.\\nMurray‐Stewart, Amy Hacker‐Prietz, et al. 2011.“Polyamine\\nCatabolism Contributes to Enterotoxigenic Bacteroides fra-\\ngilis‐Induced Colon Tumorigenesis. ”\\nProceedings of the\\nNational Academy of Sciences108: 15354–9. https://doi.org/\\n10.1073/pnas.1010203108\\n83. Cao, Fangfang, Lulu Jin, Yong Gao, Yuan Ding, Hongyang\\nWen, Zhefeng Qian, Chenyin Zhang, et al. 2023.“Artificial‐\\nEnzymes‐Armed Bifidobacterium longum Probiotics for\\nAlleviating Intestinal Inflammation and Microbiota Dysbio-\\nsis.” Nature Nanotechnology 18: 617–27. https://doi.org/10.\\n1038/s41565‐023‐01346‐x\\n84. Spencer, Christine N., Jennifer L. McQuade, Vancheswaran\\nGopalakrishnan, John A. McCulloch, Marie Vetizou,\\nAlexandria P. Cogdill, Md A. Wadud Khan, et al. 2021.\\n“Dietary Fiber and Probiotics Influence the Gut Microbiome\\nand Melanoma Immunotherapy Response. ” Science 374:\\n1632–40. https://doi.org/10.1126/science.aaz7015\\n85. Zitvogel, Laurence, Lisa Derosa, Guido Kroemer. 2022.\\n“Modulation of Cancer Immunotherapy by Dietary Fibers\\nand Over‐the‐Counter Probiotics.” Cell Metabolism34: 350–2.\\nhttps://doi.org/10.1016/j.cmet.2022.02.004\\n86. Sugimura, Naoki, Qing Li, Eagle Siu Hong Chu,\\nHarry Cheuk Hay Lau, Winnie Fong, Weixin Liu,\\nCong Liang, et al. 2021.“Lactobacillus gallinarumModulates\\nthe Gut Microbiota and Produces Anti‐Cancer Metabolites to\\nProtect Against Colorectal Tumourigenesis. ” Gut 71:\\n2011–21. https://doi.org/10.1136/gutjnl‐2020‐323951\\n87. Nan, Ke, Ziwen Zhong, Ying Yue, Yang Shen, Hao Zhang,\\nZhiqiang Wang, Kameina Zhuma, et al. 2025. “Fasting‐\\nMimicking Diet ‐Enriched Bifidobacterium pseudolongum\\nSuppresses Colorectal Cancer by Inducing Memory CD8+ T\\nCells.” Gut 74:775–86. https://doi.org/10.1136/gutjnl‐2024‐\\n333020\\n88. Mitra, Anita, Murat Gultekin, Laura Burney Ellis,\\nNicolò Bizzarri, Sarah Bowden, Nadja Taumberger,\\nTaja Bracic, et al. 2024.\\n“Genital Tract Microbiota Composi-\\ntion Profiles and Use of Prebiotics and Probiotics in Gynae-\\ncological Cancer Prevention: Review of the Current\\nEvidence, the European Society of Gynaecological Oncology\\nPrevention Committee Statement.” The Lancet. Microbe 5:\\ne291–300. https://doi.org/10.1016/S2666‐5247(23)00257‐4\\n89. Yu, Guoqin, Nan Hu, Lemin Wang, Chaoyu Wang, Xiao‐\\nYou Han, Mike Humphry, Jacques Ravel, et al. 2017.\\n“Gastric Microbiota Features Associated With Cancer Risk\\nFactors and Clinical Outcomes: A Pilot Study in Gastric\\nCardia Cancer Patients From Shanxi, China.” International\\nJournal of Cancer 141: 45–51. https://doi.org/10.1002/ijc.\\n30700\\n90. Huang, Hechen, Zhigang Ren, Xingxing Gao, Xiaoyi Hu,\\nYuan Zhou, Jianwen Jiang, Haifeng Lu, et al. 2020.“Inte-\\ngrated Analysis of Microbiome and Host Transcriptome\\nReveals Correlations Between Gut Microbiota and Clinical\\nOutcomes in HBV ‐Related Hepatocellular Carcinoma. ”\\n70 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 70, 'page_label': '71'}, page_content='Genome Medicine 12: 102. https://doi.org/10.1186/s13073‐\\n020‐00796‐5\\n91. Liu, Qisha, Fan Li, Yaoyao Zhuang, Jian Xu, Jianwei Wang,\\nXuhua Mao, Yewei Zhang, Xingyin Liu. 2019.“Alteration in\\nGut Microbiota Associated With Hepatitis B and Non ‐\\nHepatitis Virus Related Hepatocellular Carcinoma. ” Gut\\nPathogens 11, 1: 1.https://doi.org/10.1186/s13099‐018‐0281‐6\\n92. Tanoue, Takeshi, Satoru Morita, Damian R. Plichta, Ashwin\\nN. Skelly, Wataru Suda, Yuki Sugiura, Seiko Narushima, et al.\\n2019. “A Defined Commensal Consortium Elicits CD8 T Cells\\nand Anti‐Cancer Immunity.” Nature 565: 600–5. https://doi.\\norg/10.1038/s41586‐019‐0878‐z\\n93. Bäumler, Andreas J., Vanessa Sperandio. 2016.“Interactions\\nBetween the Microbiota and Pathogenic Bacteria in the Gut.”\\nNature 535: 85–93. https://doi.org/10.1038/nature18849\\n94. Chen, Yongyan, Wenwen Cui, Xiao Li, Huan Yang. 2021.\\n“Interaction Between Commensal Bacteria, Immune\\nResponse and the Intestinal Barrier in Inflammatory Bowel\\nDisease.” Frontiers in Immunology 12: 761981. https://doi.\\norg/10.3389/fimmu.2021.761981\\n95. Zheng, Yajuan, Zhaoyuan Fang, Yun Xue, Jian Zhang,\\nJunjie Zhu, Renyuan Gao, Shun Yao, et al. 2020.“Specific\\nGut Microbiome Signature Predicts the Early ‐Stage Lung\\nCancer.” Gut Microbes 11: 1030–42. https://doi.org/10.1080/\\n19490976.2020.1737487\\n96. Hu, Shixian, Arno R. Bourgonje, Ranko Gacesa, Bernadien\\nH. Jansen, Johannes R. Björk, Amber Bangma, Iwan J. Hidding,\\net al. 2024.“Mucosal Host‐Microbe Interactions Associate With\\nClinical Phenotypes in Inflammatory Bowel Disease.” Nature\\nCommunications 15: 1470. https://doi.org/10.1038/s41467‐024‐\\n45855‐2\\n97. Grasberger, Helmut, Andrew T. Magis, Elisa Sheng,\\nMatthew P. Conomos, Min Zhang, Lea S. Garzotto,\\nGuoqing Hou, et al. 2021.“DUOX2 Variants Associate With\\nPreclinical Disturbances in Microbiota‐Immune Homeostasis\\nand Increased Inflammatory Bowel Disease Risk. ” The\\nJournal of Clinical Investigation 131: e141676. https://doi.\\norg/10.1172/JCI141676\\n98. Bishehsari, Faraz, Robin M. Voigt, Ali Keshavarzian. 2020.\\n“Circadian Rhythms and the Gut Microbiota: From the Meta-\\nbolic Syndrome to Cancer.” Nature Reviews Endocrinology16:\\n731–9. https://doi.org/10.1038/s41574‐020‐00427‐4\\n99. Smith, Alexander B., Matthew L., Jenior L. Jenior, Orlaith\\nKeenan, Jessica L. Hart, Jonathan Specker, Arwa Abbas,\\nPaula C. Rangel, et al. 2022. “Enterococci Enhance Clos-\\ntridioides difficile Pathogenesis.” Nature 611: 780–6. https://\\ndoi.org/10.1038/s41586‐022‐05438‐x\\n100. Leimbach, Andreas, Jörg Hacker, Ulrich Dobrindt. 2013.“E.?\\ncoli as an All ‐Rounder: The Thin Line Between\\nCommensalism and Pathogenicity. ” Current Topics in\\nMicrobiology and Immunology 358: 3–32. https://doi.org/10.\\n1007/82_2012_303\\n101. Vogt, Stefanie L., Antonio Serapio ‐Palacios, Sarah E.\\nWoodward, Andrew S. Santos, Stefan P. W. De Vries,\\nMichelle C. Daigneault, Lisa V. Brandmeier, et al. 2023.“En-\\nterohemorrhagic Escherichia coliResponds to Gut Microbiota\\nMetabolites by Altering Metabolism and Activating Stress\\nResponses.” Gut Microbes 15: 2190303. https://doi.org/10.\\n1080/19490976.2023.2190303\\n102. Scott, Timothy A., Kate S. Baker, Caroline Trotter, Claire\\nJenkins, Serge Mostowy, Jane Hawkey, Hayden Schmidt, et al.\\n2024. “Shigella sonnei: Epidemiology, Evolution, Pathogenesis,\\nResistance and Host Interactions.” Nature Reviews Microbiology\\n23: 303–317 https://doi.org/10.1038/s41579‐024‐01126‐x\\n103. Rao, Krishna, Preeti N. Malani. 2020.“Diagnosis and Treat-\\nment of Clostridioides (Clostridium) difficile Infection in\\nAdults in 2020.” JAMA 323: 1403–4. https://doi.org/10.1001/\\njama.2019.3849\\n104. Lin, Liyuan, Yahui Du, Jia Song, Wei Wang, Chaoyong Yang.\\n2021. “Imaging Commensal Microbiota and Pathogenic\\nBacteria in the Gut.” Accounts of Chemical Research 54:\\n2076–87. https://doi.org/10.1021/acs.accounts.1c00068\\n105. Kong, Cheng, Lei Liang, Guang Liu, Lutao Du,\\nYongzhi Yang, Jianqiang Liu, Debing Shi, Xinxiang Li,\\nYanlei Ma. 2023.“Integrated Metagenomic and Metabolomic\\nAnalysis Reveals Distinct Gut‐Microbiome‐Derived Pheno-\\ntypes in Early‐Onset Colorectal Cancer.” Gut 72: 1129–42.\\nhttps://doi.org/10.1136/gutjnl‐2022‐327156\\n106. Warren, René L., Douglas J. Freeman, Stephen Pleasance,\\nPeter Watson, Richard A. Moore, Kyla Cochrane,\\nEmma Allen‐Vercoe, Robert A. Holt. 2013.“Co‐Occurrence\\nof Anaerobic Bacteria in Colorectal Carcinomas. ”\\nMicrobiome 1: 16.https://doi.org/10.1186/2049‐2618‐1‐16\\n107. Wu, Na, Xi Yang, Ruifen Zhang, Jun Li, Xue Xiao, Yongfei Hu,\\nYanfei Chen, et al. 2013.“Dysbiosis Signature of Fecal Mi-\\ncrobiota in Colorectal Cancer Patients.” Microbial Ecology66:\\n462–70. https://doi.org/10.1007/s00248‐013‐0245‐9\\n108. Avilés ‐Jiménez, F., A. Guitron, F. Segura‐López, A. Méndez‐\\nTenorio, S. Iwai, A. Hernández‐Guerrero, J. Torres. 2016.\\n“Microbiota Studies in the Bile Duct Strongly Suggest a Role\\nfor Helicobacter pyloriin Extrahepatic Cholangiocarcinoma.”\\nClinical Microbiology and Infection 22: 178.e11 –178.e22.\\nhttps://doi.org/10.1016/j.cmi.2015.10.008\\n109. Nagaraja, V., G. D. Eslick. 2014. “Systematic Review With\\nMeta‐Analysis: The Relationship Between Chronic Salmo-\\nnella Typhi Carrier Status and Gall ‐Bladder Cancer. ”\\nAlimentary Pharmacology & Therapeutics39: 745–50. https://\\ndoi.org/10.1111/apt.12655\\n110. Pushalkar, Smruti, Mautin Hundeyin, Donnele Daley,\\nConstantinos P. Zambirinis, Emma Kurz, Ankita Mishra,\\nNavyatha Mohan, et al. 2018.“The Pancreatic Cancer Mi-\\ncrobiome Promotes Oncogenesis by Induction of Innate and\\nAdaptive Immune Suppression.” Cancer Discovery8: 403–16.\\nhttps://doi.org/10.1158/2159‐8290.CD‐17‐1134\\n111. Ma, Ji, Lingqi Sun, Ying Liu, Hui Ren, Yali Shen, Feng Bi,\\nTao Zhang, Xin Wang. 2020.“Alter Between Gut Bacteria and\\nBlood Metabolites and the Anti‐Tumor Effects of Faecali-\\nbacterium prausnitziiin Breast Cancer.” BMC Microbiology20:\\n82. https://doi.org/10.1186/s12866‐020‐01739‐1\\n112. He, Chuan, Yue Liu, Shandong Ye, Shiwu Yin, Junfei Gu.\\n2021. “Changes of Intestinal Microflora of Breast Cancer in\\nPremenopausal Women. ” European Journal of Clinical\\nMicrobiology & Infectious Diseases: Official Publication of the\\nEuropean Society of Clinical Microbiology40: 503–13. https://\\ndoi.org/10.1007/s10096‐020‐04036‐x\\n113. Plottel, Claudia S., Martin J. Blaser. 2011.“Microbiome and\\nMalignancy.” Cell Host & Microbe10: 324–35. https://doi.org/\\n10.1016/j.chom.2011.10.003\\nMICROBIOME IN CANCER | 71 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 71, 'page_label': '72'}, page_content=\"114. Rea, Domenica, Giovanni Coppola, Giuseppe Palma,\\nAntonio Barbieri, Antonio Luciano, Paola Del Prete,\\nSabrina Rossetti, et al. 2018.“Microbiota Effects on Cancer:\\nFrom Risks to Therapies.” Oncotarget 9: 17915–27. https://\\ndoi.org/10.18632/oncotarget.24681\\n115. Lu, Xingbing, Li Xiong, Xi Zheng, Qiuju Yu, Yuling Xiao,\\nYi Xie. 2023. “Structure of Gut Microbiota and Character-\\nistics of Fecal Metabolites in Patients With Lung Cancer.”\\nFrontiers in Cellular and Infection Microbiology13: 1170326.\\nhttps://doi.org/10.3389/fcimb.2023.1170326\\n116. Wei, Zixin, Biying Yang, Tiantian Tang, Zijing Xiao,\\nFengzhan Ye, Xiaoyu Li, Shangbin Wu, Jin‐gang Huang,\\nShanping Jiang. 2023. “Gut Microbiota and Risk of Five\\nCommon Cancers: A Univariable and Multivariable Mende-\\nlian Randomization Study.” Cancer Medicine 12: 10393–405.\\nhttps://doi.org/10.1002/cam4.5772\\n117. Ajani, Jaffer A., Jeeyun Lee, Takeshi Sano, Yelena Y. Janjigian,\\nDaiming Fan, Shumei Song. 2017.“Gastric Adenocarcinoma.”\\nNature Reviews Disease Primers3: 17036. https://doi.org/10.\\n1038/nrdp.2017.36\\n118. Zhao, Ya ‐Shuang, Fan Wang, Dong Chang, Bing Han, Ding‐\\nYun You. 2008.“Meta‐analysis of Different Test Indicators:\\nHelicobacter pylori Infection and the Risk of Colorectal\\nCancer.” International Journal of Colorectal Disease23: 875–\\n82. https://doi.org/10.1007/s00384‐008‐0479‐z\\n119. Ertz ‐Archambault, Natalie, Paul Keim, Daniel Von Hoff. 2017.\\n“Microbiome and Pancreatic Cancer: A Comprehensive Topic\\nReview of Literature.” World Journal of Gastroenterology23:\\n1899–908. https://doi.org/10.3748/wjg.v23.i10.1899\\n120. Kim, Tae Jun Kim, Eun Ran Chang, Dong Kyung Kim, Young\\n‐Ho Baek, Sun‐Young Kim, Kyunga Hong, Sung Noh. 2017.\\n“Helicobacter pyloriInfection Is an Independent Risk Factor of\\nEarly and Advanced Colorectal Neoplasm.” Helicobacter 22:\\nelocator. https://doi.org/10.1111/hel.12377\\n121. Shah, Shailja C., M. Constanza Camargo, Mark Lamm,\\nRanier Bustamante, Christianne L. Roumie, Otis Wilson,\\nAlese E. Halvorson, et al. 2024.“Impact ofHelicobacter pylori\\nInfection and Treatment on Colorectal Cancer in a Large,\\nNationwide Cohort.” Journal of Clinical Oncology42: 1881–9.\\nhttps://doi.org/10.1200/JCO.23.00703\\n122. Krüttgen, Alexander, Hans ‐Peter Horz, Josefine Weber ‐\\nHeynemann, Mihael Vucur, Christian Trautwein, Gerhard\\nHaase, Tom Luedde, Christoph Roderburg. 2012.“Study on\\nthe Association of Helicobacter Species With Viral Hepatitis‐\\nInduced Hepatocellular Carcinoma.” Gut Microbes3: 228–33.\\nhttps://doi.org/10.4161/gmic.19922\\n123. Rocha, M. 2005. “Association of Helicobacter Species With\\nHepatitis C Cirrhosis With or Without Hepatocellular Car-\\ncinoma.” Gut 54: 396–401. https://doi.org/10.1136/gut.2004.\\n042168\\n124. Fox, J. G., Y. Feng, E. J. Theve, A. R. Raczynski, J. L. A. Fiala,\\nA. L. Doernte, M. Williams, et al. 2010.“Gut Microbes Define\\nLiver Cancer Risk in Mice Exposed to Chemical and Viral\\nTransgenic Hepatocarcinogens.” Gut 59: 88–97. https://doi.\\norg/10.1136/gut.2009.183749\\n125. Murphy, Gwen, Angelika Michel, Philip R. Taylor,\\nDemetrius Albanes, Stephanie J. Weinstein, Jarmo Virtamo,\\nDominick Parisi, et al. 2014.“Association of Seropositivity to\\nHelicobacter Species and Biliary Tract Cancer in the ATBC\\nStudy.” Hepatology 60: 1963–71. https://doi.org/10.1002/hep.\\n27193\\n126. Segura ‐López, Fany K., Francisco Avilés ‐Jiménez, Alfredo\\nGüitrón‐Cantú, Hilda A. Valdéz ‐Salazar, Samuel León ‐\\nCarballo, Leoncio Guerrero‐Pérez, James G. Fox, Javier Torres.\\n2015. “Infection With Helicobacter bilis but not Helicobacter\\nhepaticus Was Associated With Extrahepatic Cholangiocarci-\\nnoma.” Helicobacter 20: 223–30. https://doi.org/10.1111/hel.\\n12195\\n127. Shimoyama, Tadashi, Ryoki Takahashi, Daijiro Abe,\\nIchiro Mizuki, Tetsu Endo, Shinsaku Fukuda. 2010.“Sero-\\nlogical Analysis of Helicobacter hepaticus Infection in Pa-\\ntients With Biliary and Pancreatic Diseases.\\n” Journal of\\nGastroenterology and Hepatology25 Suppl 1: S86–9. https://\\ndoi.org/10.1111/j.1440‐1746.2010.06224.x\\n128. Sharma, Prateek, Shravani M. Phatak, Prisha Warikoo,\\nAkshita Mathur, Shweta Mahant, Kunal Das, Rajashree Das.\\n2023. “Crosstalk Between Helicobacter pylori and Gastro-\\nintestinal Microbiota in Various Gastroduodenal Diseases—\\nA Systematic Review.” 3 Biotech 13: 303. https://doi.org/10.\\n1007/s13205‐023‐03734‐5\\n129. Rokkas, Theodoros, Dimitrios Pistiolas, Panos Sechopoulos,\\nIoannis Robotis, Georgios Margantinis. 2007.“Relationship\\nBetween Helicobacter pylori Infection and Esophageal Neo-\\nplasia: A Meta ‐Analysis.” Clinical Gastroenterology and\\nHepatology 5: 1413–7.e2.e2.e14172. https://doi.org/10.1016/j.\\ncgh.2007.08.010\\n130. Corley, D. A., A. Kubo, T. R. Levin, G. Block, L. Habel,\\nW. Zhao, P. Leighton, et al. 2008.“Helicobacter pylori Infec-\\ntion and the Risk of Barrett's Oesophagus: A Community‐\\nBased Study.” Gut 57: 727–33. https://doi.org/10.1136/gut.\\n2007.132068\\n131. Castaño ‐Rodríguez, Natalia, Nadeem O. Kaakoush,\\nWay Seah Lee, Hazel M. Mitchell. 2017.“Dual Role of He-\\nlicobacter and Campylobacter Species in IBD: A Systematic\\nReview and Meta‐Analysis.” Gut 66: 235–49. https://doi.org/\\n10.1136/gutjnl‐2015‐310545\\n132. Whiteman, David C., Priya Parmar, Paul Fahey,\\nSuzanne P. Moore, Mitchell Stark, Zhen Zhen Zhao,\\nGrant W. Montgomery, et al. 2010.“Association of Helico-\\nbacter pylori Infection With Reduced Risk for Esophageal\\nCancer Is Independent of Environmental and Genetic Mod-\\nifiers.” Gastroenterology 139: 73–83; quiz e11‐12. https://doi.\\norg/10.1053/j.gastro.2010.04.009\\n133. Owyang, Stephanie Y., Jay Luther, Christopher C. Owyang,\\nMin Zhang, John Y. Kao. 2012.“Helicobacter pylori DNA's\\nAnti‐Inflammatory Effect on Experimental Colitis. ” Gut\\nMicrobes 3: 168–71. https://doi.org/10.4161/gmic.19181\\n134. Luther, J., S. Y. Owyang, T. Takeuchi, T. S. Cole, M. Zhang,\\nM. Liu, J. Erb‐Downward, et al. 2011. “Helicobacter pylori\\nDNA Decreases Pro‐Inflammatory Cytokine Production by\\nDendritic Cells and Attenuates Dextran Sodium Sulphate‐\\nInduced Colitis.” Gut 60: 1479–86. https://doi.org/10.1136/\\ngut.2010.220087\\n135. McColl, K. E. L., H. Watabe, M. H. Derakhshan. 2008.“Role of\\nGastric Atrophy in Mediating Negative Association Between\\nHelicobacter pyloriInfection and Reflux Oesophagitis, Barrett's\\nOesophagus and Oesophageal Adenocarcinoma.” Gut 57: 721–3.\\nhttps://doi.org/10.1136/gut.2007.144774\\n72 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 72, 'page_label': '73'}, page_content=\"136. Anderson, L. A., S. J. Murphy, B. T. Johnston, R. G. P. Watson,\\nH. R. Ferguson, K. B. Bamford, A. Ghazy, et al. 2008.“Rela-\\ntionship Between Helicobacter pylori Infection and Gastric\\nAtrophy and the Stages of the Oesophageal Inflammation,\\nMetaplasia, Adenocarcinoma Sequence: Results From the\\nFINBAR Case‐Control Study.” Gut 57: 734–9. https://doi.org/\\n10.1136/gut.2007.132662\\n137. Chen, Hui, Chuanli Ren. 2024.“Helicobacter pylori Eradica-\\ntion Treatment Might Help Reduce the Risk of Esophageal\\nAdenocarcinoma.” Gastroenterology 167: 1507–8. https://doi.\\norg/10.1053/j.gastro.2024.08.043\\n138. Wiklund, Anna ‐Klara, Giola Santoni, Jane Yan, Cecilia\\nRadkiewicz, Shaohua Xie, Helgi Birgisson, Eivind Ness‐Jensen,\\net al. 2024.“Risk of Esophageal Adenocarcinoma AfterHelico-\\nbacter pylori Eradication Treatment in a Population‐Based\\nMultinational Cohort Study.” Gastroenterology167: 485–92.e3.\\nhttps://doi.org/10.1053/j.gastro.2024.03.016\\n139. Zhu, Lei, Xingxing Jian, Bingjing Zhou, Runqiu Liu, Melba\\nMuñoz, Wan Sun, Lu Xie, et al. 2024.“Gut Microbiota Facilitate\\nChronic Spontaneous Urticaria.” Nature Communications15:\\n112. https://doi.org/10.1038/s41467‐023‐44373‐x\\n140. Agaronyan, Karen, Lokesh Sh arma, Bharat Vaidyanathan,\\nKeith Glenn, Shuang Yu, Charles Annicelli, Talia D. Wiggen,\\net al. 2022.“Tissue Remodeling by an Opportunistic Pathogen\\nTriggers Allergic Inflammation.” Immunity 55: 895–911.e10.\\nhttps://doi.org/10.1016/j.immuni.2022.04.001\\n141. Yu, Linda Chia‐Hui. 2018.“Microbiota Dysbiosis and Barrier\\nDysfunction in Inflammatory Bowel Disease and Colorectal\\nCancers: Exploring a Common Ground Hypothesis.” Journal\\nof Biomedical Science25: 79. https://doi.org/10.1186/s12929‐\\n018‐0483‐8\\n142. Schwabe, Robert F., Tim F. Greten. 2020.“\\nGut Microbiome in\\nHCC—Mechanisms, Diagnosis and Therapy. ” Journal of\\nHepatology72: 230–8. https://doi.org/10.1016/j.jhep.2019.08.016\\n143. Wu, Juan, Cong Zhang, Shuo Xu, Chunjie Xiang, Ruiping Wang,\\nDongqing Yang, Lu, Bin, et al. 2020.“Fecal Microbiome Altera-\\ntion May Be a Potential Marker for Gastric Cancer.” Disease\\nMarkers 2020: 3461315.https://doi.org/10.1155/2020/3461315\\n144. Boleij, Annemarie, Elizabeth M. Hechenbleikner,\\nAndrew C. Goodwin, Ruchi Badani, Ellen M. Stein,\\nMark G. Lazarev, Brandon Ellis, et al. 2015.“The Bacteroides\\nFragilis Toxin Gene Is Prevalent in the Colon Mucosa of\\nColorectal Cancer Patients.” Clinical Infectious Diseases: An\\nOfficial Publication of the Infectious Diseases Society of\\nAmerica 60: 208–15. https://doi.org/10.1093/cid/ciu787\\n145. Ulger Toprak, N. Yagci, A. Gulluoglu, B. M. Akin, M. L.\\nDemirkalem, P. Celenk, T. Soyletir, G. 2006.“A Possible Role\\nof Bacteroides Fragilis Enterotoxin in the Aetiology of\\nColorectal Cancer.” Clinical Microbiology and Infection 12:\\n782–6. https://doi.org/10.1111/j.1469‐0691.2006.01494.x\\n146. Wei, Zhiliang, Shougen Cao, Shanglong Liu, Zengwu Yao,\\nTeng Sun, Yi Li, Jiante Li, Dongfeng Zhang, Yanbing Zhou.\\n2016. “Could Gut Microbiota Serve as Prognostic Biomarker\\nAssociated With Colorectal Cancer Patients' Survival? A Pilot\\nStudy on Relevant Mechanism. ” Oncotarget 7: 46158–72.\\nhttps://doi.org/10.18632/oncotarget.10064\\n147. Abdulamir, Ahmed S., Rand R. Hafidh, Fatimah Bakar. 2010.\\n“Molecular Detection, Quantification, and Isolation of\\nStreptococcus gallolyticus Bacteria Colonizing Colorectal\\nTumors: Inflammation‐Driven Potential of Carcinogenesis\\nvia IL‐1, COX‐2, and IL‐8.” Molecular Cancer9: 249.https://\\ndoi.org/10.1186/1476‐4598‐9‐249\\n148. Boleij, Annemarie, Harold Tjalsma. 2013. “The Itinerary of\\nStreptococcus gallolyticus Infection in Patients With Colonic\\nMalignant Disease.” The Lancet. Infectious Diseases13: 719–24.\\nhttps://doi.org/10.1016/S1473‐\\n3099(13)70107‐5\\n149. Balamurugan, Ramadass, Ethendhar Rajendiran, Sarah George,\\nG. Vijay Samuel, Balakrishnan S. Ramakrishna. 2008.“Real‐\\nTime Polymerase Chain Reaction Quantification of Specific\\nButyrate‐Producing Bacteria, Desulfovibrio andEnterococcus\\nfaecalis in the Feces of Patients With Colorectal Cancer.”\\nJournal of Gastroenterology and Hepatology 23: 1298–303.\\nhttps://doi.org/10.1111/j.1440‐1746.2008.05490.x\\n150. Huycke, Mark M., Victoria Abrams, Danny R. Moore. 2002.\\n“Enterococcus faecalisProduces Extracellular Superoxide and\\nHydrogen Peroxide That Damages Colonic Epithelial Cell DNA.”\\nCarcinogenesis 23: 529–36. https://doi.org/10.1093/carcin/23.\\n3.529\\n151. Kohi, Shiro, Anne Macgregor‐Das, Mohamad Dbouk, Takeichi\\nYoshida, Miguel Chuidian, Toshiya Abe, Michael Borges, et al.\\n2022. “Alterations in the Duodenal?Fluid Microbiome of\\nPatients With Pancreatic Cancer.” Clinical Gastroenterology and\\nHepatology 20: e196–227. https://doi.org/10.1016/j.cgh.2020.\\n11.006\\n152. Riquelme, Erick, Yu Zhang, Liangliang Zhang, Maria Montiel,\\nMichelle Zoltan, Wenli Dong, Pompeyo Quesada, et al. 2019.\\n“Tumor Microbiome Diversity and Composition Influence\\nPancreatic Cancer Outcomes.” Cell 178: 795–806.e12. https://\\ndoi.org/10.1016/j.cell.2019.07.008\\n153. Mitsuhashi, Kei, Katsuhiko Nosho, Yasutaka Sukawa,\\nYasutaka Matsunaga, Miki Ito, Hiroyoshi Kurihara,\\nShinichi Kanno, et al. 2015.“Association of Fusobacterium\\nSpecies in Pancreatic Cancer Tissues With Molecular Fea-\\ntures and Prognosis.” Oncotarget 6: 7209–20. https://doi.org/\\n10.18632/oncotarget.3109\\n154. Bullman, Susan, Chandra S. Pedamallu, Ewa Sicinska,\\nThomas E. Clancy, Xiaoyang Zhang, Diana Cai,\\nDonna Neuberg, et al. 2017. “Analysis of Fusobacterium\\nPersistence and Antibiotic Response in Colorectal Cancer.”\\nScience 358: 1443–8. https://doi.org/10.1126/science.aal5240\\n155. Yu, Jiahui, Yongyu Chen, Xiangsheng Fu, Xian Zhou,\\nYan Peng, Lei Shi, Ting Chen, Yaxin Wu. 2016.“Invasive\\nFusobacterium nucleatum May Play a Role in the Carcino-\\ngenesis of Proximal Colon Cancer Through the Serrated\\nNeoplasia Pathway.” International Journal of Cancer 139:\\n1318–26. https://doi.org/10.1002/ijc.30168\\n156. Rubinstein, Mara Roxana, Xiaowei Wang, Wendy Liu, Yujun\\nHao, Guifang Cai, Yiping W. Han. 2013.\\n“Fusobacterium\\nnucleatum Promotes Colorectal Carcinogenesis by Modulat-\\ning E‐Cadherin/β‐Catenin Signaling via Its FadA Adhesin.”\\nCell Host & Microbe 14: 195–206. https://doi.org/10.1016/j.\\nchom.2013.07.012\\n157. Zhou, Dongqin, Yongsheng Li. 2023. “Gut Microbiota and\\nTumor‐Associated Macrophages: Potential in Tumor Diag-\\nnosis and Treatment.” Gut Microbes15: 2276314.https://doi.\\norg/10.1080/19490976.2023.2276314\\n158. Wang, Man, Fei Yu, Peifeng Li. 2023.“Intratumor Microbiota in\\nCancer Pathogenesis and Immunity: From Mechanisms of\\nMICROBIOME IN CANCER | 73 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 73, 'page_label': '74'}, page_content=\"Action to TherapeuticO p p o r t u n i t i e s .” Frontiers in Immunology\\n14: 1269054.https://doi.org/10.3389/fimmu.2023.1269054\\n159. Udayasuryan, Barath, Raffae N. Ahmad, Tam T. D. Nguyen,\\nAriana Umaña, LaDeidra Monét Roberts, Polina Sobol,\\nStephen D. Jones, et al. 2022. “Fusobacterium nucleatum\\nInduces Proliferation and Migration in Pancreatic Cancer\\nCells Through Host Autocrine and Paracrine Signaling. ”\\nScience Signaling 15: eabn4948. https://doi.org/10.1126/\\nscisignal.abn4948\\n160. Brennan, Caitlin A., Slater L. Clay, Sydney L. Lavoie,\\nSena Bae, Jessica K. Lang, Diogo Fonseca‐Pereira, Kathryn\\nG. Rosinski, et al. 2021.“Fusobacterium nucleatum Drives a\\nPro‐Inflammatory Intestinal Microenvironment Through\\nMetabolite Receptor‐Dependent Modulation of IL ‐17 Ex-\\npression.” Gut Microbes13: 1987780.https://doi.org/10.1080/\\n19490976.2021.1987780\\n161. Wong, Sunny H., Thomas N. Y. Kwong, Tai‐Cheong Chow,\\nArthur K. C. Luk, Rudin Z. W. Dai, Geicho Nakatsu,\\nThomas Y. T. Lam, et al. 2017.“Quantitation of Faecal Fu-\\nsobacterium Improves Faecal Immunochemical Test in\\nDetecting Advanced Colorectal Neoplasia.” Gut 66: 1441–8.\\nhttps://doi.org/10.1136/gutjnl‐2016‐312766\\n162. Radaic, Allan, Eliah R. Shamir, Kyle Jones, Alessandro Villa,\\nNandita R. Garud, Aaron D. Tward, Pachiyappan Kamarajan,\\nYvonne L. Kapila. 2023.“Specific Oral Microbial Differences\\nin Proteobacteria and Bacteroidetes Are Associated With\\nDistinct Sites When Moving From Healthy Mucosa to Oral\\nDysplasia—A Microbiome and Gene Profiling Study and\\nFocused Review.” Microorganisms 11: 2250. https://doi.org/\\n10.3390/microorganisms11092250\\n163. Zhang, Kai, Cheng He, Yuan Qiu, Xiuyang Li, Jian Hu,\\nBaiping Fu. 2023.“Association of Oral Microbiota and Peri-\\nodontal Disease With Lung Cancer: A Systematic Review and\\nMeta‐Analysis.” Journal of Evidence‐Based Dental Practice23:\\n101897. https://doi.org/10.1016/j.jebdp.2023.101897\\n164. Pignatelli, Pamela, Federi ca Nuccio, Adriano Piattelli,\\nMaria Cristina Curia. 2023.“T h eR o l eo fFusobacterium nucle-\\natum in Oral and Colorectal Carcinogenesis.” Microorganisms\\n11: 2358.https://doi.org/10.3390/microorganisms11092358\\n165. Chen, Linfu, Rui Zhao, Zheyu Kang, Zhiqin Cao, Nanhui Liu,\\nJingjing Shen, Cheng Wang, et al. 2023.“Delivery of Short\\nChain Fatty Acid Butyrate to OvercomeFusobacterium nu-\\ncleatum\\n‐Induced Chemoresistance.” Journal of Controlled\\nRelease 363: 43 –56. https://doi.org/10.1016/j.jconrel.2023.\\n09.028\\n166. Zhang, Sheng, Yongzhi Yang, Wenhao Weng, Bomin Guo,\\nGuoxiang Cai, Yanlei Ma, Sanjun Cai. 2019.“Fusobacterium\\nnucleatum Promotes Chemoresistance to 5‐Fluorouracil by\\nUpregulation of BIRC3 Expression in Colorectal Cancer.”\\nJournal of Experimental & Clinical Cancer Research: CR38:\\n14. https://doi.org/10.1186/s13046‐018‐0985‐y\\n167. Yu, TaChung, Fangfang Guo, Yanan Yu, Tiantian Sun,\\nMa, Dan, Jixuan Han, Yun Qian, et al. 2017.“Fusobacterium\\nnucleatum Promotes Chemoresistance to Colorectal Cancer\\nby Modulating Autophagy.” Cell 170: 548–63.e16. https://doi.\\norg/10.1016/j.cell.2017.07.008\\n168. Yang, Hui, Jinghui Cheng, Hao Zhuang, Hongchuang Xu,\\nYinuo Wang, Tingting Zhang, Yinmo Yang, et al. 2024.\\n“Pharmacogenomic Profiling of Intra‐Tumor Heterogeneity\\nUsing a Large Organoid Biobank of Liver Cancer.” Cancer\\nCell 42: 535–51.e8. https://doi.org/10.1016/j.ccell.2024.03.004\\n169. Ma, Xiaolu, Kaixia Zhou, Tianqing Yan, Ling Hu,\\nSuhong Xie, Hui Zheng, Ying Tong, et al. 2024.“Calpain 2\\nPromotes Lenvatinib Resistance and Cancer Stem Cell Traits\\nvia Both Proteolysis‐Dependent and Independent Approach\\nin Hepatocellular Carcinoma.” Molecular Biomedicine 5: 74.\\nhttps://doi.org/10.1186/s43556‐024‐00242‐7\\n170. Li, Xinxiu, Hongmeng Su, Wenqing Tang, Shihui Shu,\\nLuyu Zhao, Jinghan Sun, Hong Fan. 2024.“Targeting LEF1‐\\nMediated Epithelial‐Mesenchymal Transition Reverses Len-\\nvatinib Resistance in Hepatocellular Carcinoma. ”\\nInvestigational New Drugs 42: 185–95. https://doi.org/10.\\n1007/s10637‐024‐01426‐2\\n171. Van der Merwe, Michelle, Gustav Van Niekerk, Alf Botha,\\nAnna‐Mart Engelbrecht. 2021.\\n“The Onco‐Immunological\\nImplications ofFusobacterium nucleatum in Breast Cancer.”\\nImmunology Letters 232: 60 –6. https://doi.org/10.1016/j.\\nimlet.2021.02.007\\n172. Allen, D. W., P. Cole. 1973.“Oncogenic RNA Viruses.” CA:\\nA Cancer Journal for Clinicians23: 193–200. https://doi.org/\\n10.3322/canjclin.23.3.193\\n173. MacLennan, Signe A., Marco A. Marra. 2023. “Oncogenic\\nViruses and the Epigenome: How Viruses Hijack Epigenetic\\nMechanisms to Drive Cancer. ” International Journal of\\nMolecular Sciences 24: 9543. https://doi.org/10.3390/\\nijms24119543\\n174. Zur Hausen, Harald 2001. “Oncogenic DNA Viruses. ”\\nOncogene 20: 7820–3. https://doi.org/10.1038/sj.onc.1204958\\n175. Akkari, Leila, Damien Grégoire, Nicolas Floc'h, Marie\\nMoreau, Céline Hernandez, Yannick Simonin, Arielle R.\\nRosenberg, Patrice Lassus, Urszula Hibner. 2012.“Hepatitis\\nC Viral Protein NS5A Induces EMT and Participates in\\nOncogenic Transformation of Primary Hepatocyte Precur-\\nsors.” Journal of Hepatology 57: 1021–8. https://doi.org/10.\\n1016/j.jhep.2012.06.027\\n176. Hannigan, Geoffrey D., Melissa B. Duhaime, Mack T. Ruffin,\\nCharlie C. Koumpouras, Patrick D. Schloss. 2018. “Diag-\\nnostic Potential and Interactive Dynamics of the Colorectal\\nCancer Virome.” mBio 9: e02248‐18. https://doi.org/10.1128/\\nmBio.02248‐18\\n177. Li, Yazhen, Qiancheng Qiu, Zhiqiang Fan, Ping He,\\nHuanzhu Chen, Xiaoyang Jiao. 2018.“Th17 Cytokine Pro-\\nfiling of Colorectal Cancer Patients With or Without En-\\nterovirus 71 Antigen Expression. ” Cytokine 107: 35 –42.\\nhttps://doi.org/10.1016/j.cyto.2017.11.012\\n178. Arzumanyan, A., T. Friedman, E. Kotei, I. O. L. Ng, Z. Lian,\\nM. A. Feitelson. 2012.“Epigenetic Repression of E‐Cadherin\\nExpression by Hepatitis B Virus x Antigen in Liver Cancer.”\\nOncogene 31: 563–72. https://doi.org/10.1038/onc.2011.255\\n179. Pagano, Joseph?S., Martin Blaser, Marie ‐Annick Buendia,\\nBlossom Damania, Kamel Khalili, Nancy Raab ‐Traub,\\nBernard Roizman. 2004. “Infectious Agents and Cancer:\\nCriteria for a Causal Relation.” Seminars in Cancer Biology\\n14: 453 –71. https://doi.org/10.1016/j.semcancer.2004.\\n06.009\\n180. Shin, Wing Sum, Fuda Xie, Bonan Chen, Jun Yu,\\nKwok Wai Lo, Gary M. K. Tse, Ka Fai To, Wei Kang. 2023.\\n“Exploring the Microbiome in Gastric Cancer: Assessing\\n74 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 74, 'page_label': '75'}, page_content='Potential Implications and Contextualizing Microorganisms\\nbeyond H. pyloriand Epstein–Barr Virus.” Cancers 15: 4993.\\nhttps://doi.org/10.3390/cancers15204993\\n181. Liu, Liying, Yanan Xie, Hong Yang, Anqi Lin, Minjun Dong,\\nHaitao Wang, Cangang Zhang, et al. 2023.“HPVTIMER: A\\nShiny Web Application for Tumor Immune Estimation in\\nHuman Papillomavirus‐Associated Cancers.” iMeta 2: e130.\\nhttps://doi.org/10.1002/imt2.130\\n182. Lacunza, Ezequiel, Valeria Fink, María E. Salas, Romina\\nCanzoneri, Julián Naipauer, Sion Williams, Omar Coso, et al.\\n2023. “Oral and Anal Microbiome From HIV‐Exposed In-\\ndividuals: Role of Host‐Associated Factors in Taxa Composition\\nand Metabolic Pathways.” NPJ Biofilms and Microbiomes9: 48.\\nhttps://doi.org/10.1038/s41522‐023‐00413‐4\\n183. Nakatsu, Geicho, Haokui Zhou, William Ka Kei Wu,\\nSunny Hei Wong, Olabisi Oluwabukola Coker, Zhenwei Dai,\\nXiangchun Li, et al. 2018.“Alterations in Enteric Virome Are\\nAssociated With Colorectal Cancer and Survival Outcomes.”\\nGastroenterology 155: 529 –41.e5. https://doi.org/10.1053/j.\\ngastro.2018.04.018\\n184. Luo, Shiqi, Jinlong Ru, Mohammadali Khan Mirzaei,\\nJinling Xue, Xue Peng, Anna Ralser, Raquel Mejías‐Luque,\\nMarkus Gerhard, Li Deng. 2023. “Gut Virome Profiling\\nIdentifies an Association Between Temperate Phages and\\nColorectal Cancer Promoted by Helicobacter pylori Infec-\\ntion.” Gut Microbes 15: 2257291. https://doi.org/10.1080/\\n19490976.2023.2257291\\n185. Wang, Zhihan, Kai Guo, Yingying Liu, Canhua Huang,\\nMin Wu. 2022.“Dynamic Impact of Virome on Colitis and\\nColorectal Cancer: Immunity, Inflammation, Prevention and\\nTreatment.” Seminars in Cancer Biology86: 943–54. https://\\ndoi.org/10.1016/j.semcancer.2021.10.004\\n186. Gogokhia, Lasha, Kate Buhrke, Rickesha Bell, Brenden\\nHoffman, D. Garrett Brown, Christin Hanke ‐Gogokhia,\\nNadim J. Ajami, et al. 2019.“Expansion of Bacteriophages Is\\nLinked to Aggravated Intestinal Inflammation and Colitis.”\\nCell Host & Microbe25: 285–99.e8. https://doi.org/10.1016/j.\\nchom.2019.01.008\\n187. Lawson, James S., Wendy K. Glenn. 2021. “Catching Viral\\nBreast Cancer.” Infectious Agents and Cancer16: 37.https://\\ndoi.org/10.1186/s13027‐021‐00366‐3\\n188. Herbein, Georges, Amit Kumar. 2014. “The Oncogenic\\nPotential of Human Cytomegalovirus and Breast Cancer.”\\nFrontiers in Oncology 4: 230. https://doi.org/10.3389/fonc.\\n2014.00230\\n189. Richardson, A. K., L. C. Walker, B. Cox, H. Rollag,\\nB. A. Robinson, H. Morrin, J. F. Pearson, et al. 2020.“Breast\\nCancer and Cytomegalovirus. ” Clinical & Translational\\nOncology: Official Publication of the Federation of Spanish\\nOncology Societies and of the National Cancer Institute of\\nMexico 22: 585 –602. https://doi.org/10.1007/s12094 ‐019‐\\n02164‐1\\n190. Curty, Gislaine, Jez L. Marston, Miguel De Mulder Rougvie,\\nFabio E. Leal, Douglas F. Nixon, Marcelo A. Soares. 2020.\\n“Human Endogenous Retrovirus K in Cancer: A Potential\\nBiomarker and Immunotherapeutic Target.” Viruses 12: 726.\\nhttps://doi.org/10.3390/v12070726\\n191. Gao, Peng, Jie Zheng. 2011.“Oncogenic Virus‐Mediated Cell\\nFusion: New Insights Into In itiation and Progression of\\nOncogenic Viruses—Related Cancers.” Cancer Letters303: 1–8.\\nhttps://doi.org/10.1016/j.canlet.2010.12.021\\n192. Purdy, John G., Micah A. Luftig. 2019.“Reprogramming of\\nCellular Metabolic Pathways by Human Oncogenic Viruses.”\\nCurrent Opinion in Virology 39: 60–9. https://doi.org/10.\\n1016/j.coviro.2019.11.002\\n193. Huang, Binhao, Qin Li, Qirong Geng, Jiawen Lao, Jing Guo,\\nShenglin Huang, Dazhi Xu. 2022.“ASTE1 Frameshift Muta-\\ntion Triggers the Immune Response in Epstein–Barr Virus‐\\nAssociated Gastric Cancer.” Signal Transduction and Targeted\\nTherapy 7: 4.https://doi.org/10.1038/s41392‐021‐00771‐5\\n194. Deng, Chu‐Xia. 2024. “Revealing the Secret Behind Epstein–\\nBarr Virus ‐Specific Tumor Immune Contexture. ” Cancer\\nCommunications44: 491–4. https://doi.org/10.1002/cac2.12529\\n195. Armstrong, Heather, Mandana Rahbari, Heekuk Park,\\nDavid Sharon, Aducio Thiesen, Naomi Hotte, Ning Sun, et al.\\n2023. “Mouse Mammary Tumor Virus Is Implicated in\\nSeverity of Colitis and Dysbiosis in the IL‐10−/− Mouse\\nModel of Inflammatory Bowel Disease.” Microbiome 11: 39.\\nhttps://doi.org/10.1186/s40168‐023‐01483‐4\\n196. McBride, Alison A., Elizabeth A. White. 2023. “HPV Inte-\\ngration Can Drive the Formation of Virus –Host Extra-\\nchromosomal DNA in Tumors.” Cancer Discovery13: 814–6.\\nhttps://doi.org/10.1158/2159‐8290.CD‐23‐0097\\n197. He, Jiang, Liyu Liu, Feiyu Tang, You Zhou, Huan Liu,\\nCan Lu, Deyun Feng, et al. 2021.“Paradoxical Effects of DNA\\nTumor Virus Oncogenes on Epithelium‐Derived Tumor Cell\\nFate During Tumor Progression and Chemotherapy\\nResponse.” Signal Transduction and Targeted Therapy6: 408.\\nhttps://doi.org/10.1038/s41392‐021‐00787‐x\\n198. Sekiba, Kazuma, Motoyuki Otsuka, Kazuyoshi Funato,\\nYu Miyakawa, Eri Tanaka, Takahiro Seimiya, Mari Yamagami,\\net al. 2022. “HBx‐Induced Degradation of Smc5/6 Complex\\nImpairs Homologous Recombination ‐Mediated Repair of\\nDamaged DNA.” Journal of Hepatology76: 53–62. https://doi.\\norg/10.1016/j.jhep.2021.08.010\\n199. Xia, Wei, Lei Liu, Nan Shi, Chaoyin Zhang, Anzhou Tang,\\nGuangyao He. 2023. “Epstein Barr Virus Infection in Tree\\nShrews Alters the Composition of Gut Microbiota and\\nMetabolome Profile.” Virology Journal 20: 177. https://doi.\\norg/10.1186/s12985‐023\\n‐02147‐3\\n200. Easwaran, Maheswaran, Fatma Abdelrahman,\\nSivagnanavelmurugan Madasamy, Baskar Venkidasamy.\\n2024. “Phage‐Delivered Melittin for Oral Squamous Cell\\nCarcinoma: A Potential Therapeutic Agent.” International\\nImmunopharmacology 134: 112163. https://doi.org/10.\\n1016/j.intimp.2024.112163\\n201. York, Ashley. 2025. “Predicting Phage ‐Host Specificity. ”\\nNature Reviews Microbiology 23: 5. https://doi.org/10.1038/\\ns41579‐024‐01132‐z\\n202. York, Ashley. 2020. “Prophages Are Gut Virome Pioneers.”\\nNature Reviews Microbiology18: 317.https://doi.org/10.1038/\\ns41579‐020‐0374‐3\\n203. Wahida, Adam, Fang Tang, Jeremy J. Barr. 2021.“Rethink-\\ning Phage –Bacteria–Eukaryotic Relationships and Their\\nInfluence on Human Health.” Cell Host & Microbe29: 681–8.\\nhttps://doi.org/10.1016/j.chom.2021.02.007\\n204. Tisza, Michael J., Richard E. Lloyd, Kristi Hoffman,\\nDaniel P. Smith, Marian Rewers, Sara J. Javornik Cregeen,\\nMICROBIOME IN CANCER | 75 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 75, 'page_label': '76'}, page_content=\"Joseph?F. Petrosino. 2025. “Longitudinal Phage –Bacteria\\nDynamics in the Early Life Gut Microbiome. ” Nature\\nMicrobiology 10: 420–30. https://doi.org/10.1038/s41564‐024‐\\n01906‐4\\n205. Shkoporov, Andrey N., Colin Hill. 2019.“Bacteriophages of\\nthe Human Gut: The‘Known Unknown’ of the Microbiome.”\\nCell Host & Microbe 25: 195–209. https://doi.org/10.1016/j.\\nchom.2019.01.017\\n206. Kotsiliti, Eleni. 2024. “Bacteriophages and Host Inflamma-\\ntion in IBD.” Nature Reviews Gastroenterology & Hepatology\\n21: 300.https://doi.org/10.1038/s41575‐024‐00925‐0\\n207. Putra, Ramendra Dirgantara, Diana Lyrawati. 2020. “Inter-\\nactions Between Bacteriophages and Eukaryotic Cells. ”\\nScientifica 2020: 3589316. https://doi.org/10.1155/2020/\\n3589316\\n208. Magnaye, Kevin M., Susan V. Lynch. 2021. “Fungus Fuels\\nMucosal Wounds in Crohn's Disease.” Immunity 54: 856–8.\\nhttps://doi.org/10.1016/j.immuni.2021.04.013\\n209. Huo, Xiaokui, Dawei Li, Fan Wu, Shenghui Li, Yanling Qiao,\\nChao Wang, Wang, Yan, et al. 2022. “Cultivated Human\\nIntestinal Fungus Candida metapsilosis M2006B Attenuates\\nColitis by Secreting Acyclic Sesquiterpenoids as FXR Ago-\\nnists.” Gut 71: 2205–17. https://doi.org/10.1136/gutjnl‐2021‐\\n325413\\n210. Cho, Daniel H., Gloria B. Choi. 2022.“Fungus Packs a Punch\\nin the Gut.” Immunity 55: 586–8. https://doi.org/10.1016/j.\\nimmuni.2022.03.012\\n211. Bordon, Yvonne. 2022. “Fungus Hijacks TLR4 to Build a\\nType 2 Immune Niche.” Nature Reviews Immunology 22:\\n532–3. https://doi.org/10.1038/s41577‐022‐00773‐6\\n212. Ost, Kyla S., June L. Round. 2023. “Commensal Fungi in\\nIntestinal Health and Disease. ” Nature Reviews\\nGastroenterology & Hepatology 20: 723–34. https://doi.org/\\n10.1038/s41575‐023‐00816‐w\\n213. Ahmadi, N., A. Ahmadi, E. Kheirali, M. Hossein Yadegari,\\nM. Bayat, A. Shajiei, A. Ali Amini, et al. 2019.“Systemic\\nInfection With Candida albicans in Breast Tumor Bearing\\nMice: Cytokines Dysregulation and Induction of Regulatory\\nT Cells.” Journal de Mycologie Médicale 29: 49–55. https://\\ndoi.org/10.1016/j.mycmed.2018.10.006\\n214. He, Shengfu, Yating Sun, Weijie Sun, Mingyang Tang,\\nBao Meng, Yanyan Liu, Qinxiang Kong, et al. 2023.“Oral\\nMicrobiota Disorder in GC Patients Revealed by 2b‐RAD‐M.”\\nJournal of Translational Medicine21: 831.https://doi.org/10.\\n1186/s12967‐023‐04599‐1\\n215. Dohlman, Anders B., Jared Klug, Marissa Mesko, Iris H. Gao,\\nSteven M. Lipkin, Xiling Shen, Iliyan D. Iliev. 2022.“A Pan‐\\nCancer Mycobiome Analysis Reveals Fungal Involvement in\\nGastrointestinal and Lung Tumors.” Cell 185: 3807–22.e12.\\nhttps://doi.org/10.1016/j.cell.2022.09.015\\n216. Chin, Siok ‐Fong, Putri Intan Hafizah Megat Mohd Azlan,\\nLuqman Mazlan, Hui‐Min Neoh. 2018. “Identification of\\nSchizosaccharomyces pombe in the Guts of Healthy In-\\ndividuals and Patients With Colorectal Cancer: Preliminary\\nEvidence From a Gut Microbiome Secretome Study.” Gut\\nPathogens 10: 29.https://doi.org/10.1186/s13099‐018‐0258‐\\n5\\n217. Li, Fan, Yunhuan Gao, Wenyue Cheng, Xiaomin Su,\\nRongcun Yang. 2023.“Gut Fungal Mycobiome: A Significant\\nFactor of Tumor Occurrence and Development. ” Cancer\\nLetters 569: 216302. https://doi.org/10.1016/j.canlet.2023.\\n216302\\n218. Wang, Yu, Yahui Ren, Yongming Huang, Xiangnan Yu,\\nYiming Yang, Di Wang, Liang Shi, et al. 2021. “Fungal\\nDysbiosis of the Gut Microbiota Is Associated With Colo-\\nrectal Cancer in Chinese Patients. ” American Journal of\\nTranslational Research 13: 11287–301.\\n219. Aykut, Berk, Smruti Pushalkar, Ruonan Chen, Qianhao Li,\\nRaquel Abengozar, Jacqueline I. Kim, Sorin A. Shadaloey,\\net al. 2019. “The Fungal Mycobiome Promotes Pancreatic\\nOncogenesis via Activation of MBL. ” Nature 574: 264–7.\\nhttps://doi.org/10.1038/s41586‐019‐1608‐2\\n220. Brayer, Kathryn J., Joshua A. Hanson, Shashank Cingam,\\nCathleen Martinez, Scott A. Ness, Ian Rabinowitz. 2023.“The\\nInflammatory Response of Human Pancreatic Cancer Sam-\\nples Compared to Normal Controls.” PloS One18: e0284232.\\nhttps://doi.org/10.1371/journal.pone.0284232\\n221. Zhang, Lilong, Dongqi Chai, Chen Chen, Chunlei Li,\\nZhendong Qiu, Tianrui Kuang, Mungur Parveena, et al. 2022.\\n“Mycobiota and C‐Type Lectin Receptors in Cancers: Know\\nThy Neighbors.” Frontiers in Microbiology13: 946995.https://\\ndoi.org/10.3389/fmicb.2022.946995\\n222. Wang, Xu, Zejun Zhou, David Turner, Michael Lilly,\\nTongwen Ou, Wei Jiang. 2022. “Differential Circulating\\nFungal Microbiome in Prostate Cancer Patients Compared to\\nHealthy Control Individuals. ” Journal of Immunology\\nResearch 2022: 2574964. https://doi.org/10.1155/2022/\\n2574964\\n223. Cheng, Wenyue, Fan Li, Yunhuan Gao, Rongcun Yang. 2024.\\n“Fungi and Tumors: The Role of Fungi in Tumorigenesis\\n(Review).” International Journal of Oncology64: 52. https://\\ndoi.org/10.3892/ijo.2024.5640\\n224. Galloway ‐Peña, Jessica, Iliyan D. Iliev, Florencia McAllister.\\n2024. “Fungi in Cancer.” Nature Reviews Cancer 24: 295–8.\\nhttps://doi.org/10.1038/s41568‐024‐00665‐y\\n225. Xu, Bin, Zan Luo, Xing Niu, Zhi Li, Yeping Lu, Junyu Li.\\n2025. “Fungi, Immunosenescence and Cancer.” Seminars in\\nCancer Biology 109: 67 –82. https://doi.org/10.1016/j.\\nsemcancer.2025.01.002\\n226. Li, Xiaopeng, Jiahui Feng, Zhanggui Wang, Gang Liu,\\nFan Wang. 2023. “Features of Combined Gut Bacteria and\\nFungi From a Chinese Cohort of Colorectal Cancer, Colo-\\nrectal Adenoma, and Post‐Operative Patients.” Frontiers in\\nMicrobiology 14: 1236583. https://doi.org/10.3389/fmicb.\\n2023.1236583\\n227. Coker, Olabisi Oluwabukola, Geicho Nakatsu, Rudin\\nZhenwei Dai, William Ka Kei Wu, Sunny Hei Wong,\\nSiew Chien Ng, Francis Ka Leung Chan, Joseph?Jao Yiu Sung,\\nJun Yu. 2019. “Enteric Fungal Microbiota Dysbiosis and\\nEcological Alterations in Colorectal Cancer.” Gut 68: 654–62.\\nhttps://doi.org/10.1136/gutjnl‐2018‐317178\\n228. Yang, Ping, Xiaoshan Zhang, Rui Xu, Khan Adeel,\\nXiaofeng Lu, Min Chen, Han Shen, Zhiyang Li, Zhipeng Xu.\\n2022. “Fungal Microbiota Dysbiosis and Ecological Altera-\\ntions in Gastric Cancer.” Frontiers in Microbiology13: 889694.\\nhttps://doi.org/10.3389/fmicb.2022.889694\\n229. Huang, Hui, Qiurong Wang, Ying Yang, Wei Zhong,\\nFeng He, Jun Li. 2024.“The Mycobiome as Integral Part of\\nthe Gut Microbiome: Crucial Role of Symbiotic Fungi in\\n76 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 76, 'page_label': '77'}, page_content='Health and Disease.” Gut Microbes 16: 2440111. https://doi.\\norg/10.1080/19490976.2024.2440111\\n230. Wang, Rui, Bo Li, Bingyuan Huang, Yikang Li, Qiaoyan Liu,\\nZhuwan Lyu, Ruiling Chen, et al. 2024. “Gut Microbiota‐\\nDerived Butyrate Induces Epigenetic and Metabolic Repro-\\ngramming in Myeloid‐Derived Suppressor Cells to Alleviate\\nPrimary Biliary Cholangitis. ” Gastroenterology 167:\\n733–49.e3. https://doi.org/10.1053/j.gastro.2024.05.014\\n231. Sinha, Anurag K., Martin F. Laursen, Julius E. Brinck,\\nMorten L. Rybtke, Anna Pii Hjørne, Nicola Procházková,\\nMikael Pedersen, Henrik M. Roager, Tine R. Licht. 2024.\\n“Dietary Fibre Directs Microbial Tryptophan Metabolism via\\nMetabolic Interactions in the Gut Microbiota. ” Nature\\nMicrobiology 9: 1964–78. https://doi.org/10.1038/s41564‐024‐\\n01737‐3\\n232. Lu, Yijing, Wenlong Yang, Zhiyong Qi, Rifeng Gao,\\nJing Tong, Tingwen Gao, Yin Zhang, et al. 2023. “Gut\\nMicrobe‐Derived Metabolite Indole‐3‐Carboxaldehyde Alle-\\nviates Atherosclerosis.” Signal Transduction and Targeted\\nTherapy 8: 378.https://doi.org/10.1038/s41392‐023‐01613‐2\\n233. Kwon, Byungsuk. 2023.“A Metabolite of the Gut Microbiota:\\nA Facilitator of Chemotherapy Efficacy in Cancer.” Signal\\nTransduction and Targeted Therapy 8: 238. https://doi.org/\\n10.1038/s41392‐023‐01506‐4\\n234. Huang, Kan, Zilun Li, Xi He, Jun Dai, Bingding Huang,\\nYongxia Shi, Dongxiao Fan, et al. 2024.“Gut Microbial Co‐\\nMetabolite 2‐Methylbutyrylcarnitine Exacerbates Thrombo-\\nsis via Binding to and Activating Integrin α2β1.” Cell\\nMetabolism 36: 598–616.e9. https://doi.org/10.1016/j.cmet.\\n2024.01.014\\n235. Collins, Stephanie L., Jonathan G. Stine, Jordan E. Bisanz,\\nC. Denise Okafor, Andrew D. Patterson. 2023.\\n“Bile Acids\\nand the Gut Microbiota: Metabolic Interactions and Impacts\\non Disease.” Nature Reviews Microbiology21: 236–47. https://\\ndoi.org/10.1038/s41579‐022‐00805‐x\\n236. Agus, Allison, Karine Clément, Harry Sokol. 2021. “Gut\\nMicrobiota‐Derived Metabolites as Central Regulators in\\nMetabolic Disorders.” Gut 70: 1174–82. https://doi.org/10.\\n1136/gutjnl‐2020‐323071\\n237. Belcheva, Antoaneta, Thergiory Irrazabal, Susan J. Robertson,\\nCatherine Streutker, Heather Maughan, Stephen Rubino,\\nEduardo H. Moriyama, et al. 2014.“Gut Microbial Metabolism\\nDrives Transformation of MSH2‐Deficient Colon Epithelial\\nCells.” Cell 158: 288–99. https://doi.org/10.1016/j.cell.2014.\\n04.051\\n238. Zheng, Di‐Wei, Run‐Qing Li, Jia‐Xin An, Tian‐Qiu Xie, Zi‐Yi\\nHan, Rui Xu, Yu Fang, Xian‐Zheng Zhang. 2020.“Prebiotics‐\\nEncapsulated Probiotic?Spores Regulate Gut Microbiota and\\nSuppress Colon Cancer.” Advanced Materials 32: e2004529.\\nhttps://doi.org/10.1002/adma.202004529\\n239. Park, Hyun‐Soo, Joo‐Hui Han, Jeong Won Park, Do‐Hyung Lee,\\nKeun‐Woo Jang, Miji Lee, Kyung ‐Sun Heo, Chang ‐Seon\\nMyung. 2021.“Sodium Propionate Exerts Anticancer Effect in\\nMice Bearing Breast Cancer Cell Xenograft by Regulating JAK2/\\nSTAT3/ROS/p38 MAPK Signaling.” Acta Pharmacologica Sinica\\n42: 1311–23. https://doi.org/10.1038/s41401‐020‐\\n00522‐2\\n240. Hilakivi ‐Clarke, Leena, Fabia De Oliveira Andrade. 2023.\\n“Social Isolation and Breast Cancer. ” Endocrinology 164:\\nbqad126. https://doi.org/10.1210/endocr/bqad126\\n241. Luu, Maik, Zeno Riester, Adrian Baldrich, Nicole Reichardt,\\nSamantha Yuille, Alessandro Busetti, Matthias Klein, et al.\\n2021. “Microbial Short‐Chain Fatty Acids Modulate CD8+ T\\nCell Responses and Improve Adoptive Immunotherapy for\\nCancer.” Nature Communications 12: 4077. https://doi.org/\\n10.1038/s41467‐021‐24331‐1\\n242. Kim, Minsuk, Emily Vogtmann, David A. Ahlquist,\\nMary E. Devens, John B. Kisiel, William R. Taylor,\\nBryan A. White, et al. 2020.“Fecal Metabolomic Signatures in\\nColorectal Adenoma Patients Are Associated With Gut Mi-\\ncrobiota and Early Events of Colorectal Cancer Pathogenesis.”\\nmBio 11: e03186‐19. https://doi.org/10.1128/mBio.03186‐19\\n243. Hale, Vanessa L., Jun Chen, Stephen Johnson, Sean C.\\nHarrington, Tracy C. Yab, Thomas C. Smyrk, Heidi Nelson,\\net al. 2017. “Shifts in the Fecal Microbiota Associated With\\nAdenomatous Polyps.” Cancer Epidemiology, Biomarkers &\\nPrevention: A Publication of the American Association for\\nCancer Research, Cosponsored by the American Society of\\nPreventive Oncology 26: 85–94. https://doi.org/10.1158/1055‐\\n9965.EPI‐16‐0337\\n244. Mikó, Edit, András Vida, Tünde Kovács, Gyula Ujlaki,\\nGyörgy Trencsényi, Judit Márton, Zsanett Sári, et al. 2018.\\n“Lithocholic Acid, a Bacterial Metabolite Reduces Breast\\nCancer Cell Proliferation and Aggressiveness.” Biochimica et\\nBiophysica Acta (BBA)—Bioenergetics 1859: 958–74. https://\\ndoi.org/10.1016/j.bbabio.2018.04.002\\n245. Kovács, Patrik, Tamás Csonka, Tünde Kovács, Zsanett Sári,\\nGyula Ujlaki, Adrien Sipos, Zsolt Karányi, et al. 2019.“Li-\\nthocholic Acid, a Metabolite of the Microbiome, Increases\\nOxidative Stress in Breast Cancer.” Cancers 11: 1255.https://\\ndoi.org/10.3390/cancers11091255\\n246. Sun, Xi‐Zhen, Dong‐Yan Zhao, Yuan‐Chen Zhou, Qian‐Qian\\nWang, Geng Qin, Shu‐\\nKun Yao. 2020. “Alteration of Fecal\\nTryptophan Metabolism Correlates With Shifted Microbiota\\nand May Be Involved in Pathogenesis of Colorectal Cancer.”\\nWorld Journal of Gastroenterology 26: 7173–90. https://doi.\\norg/10.3748/wjg.v26.i45.7173\\n247. Díaz ‐Díaz, Carol J., Sean M. Ronnekleiv ‐Kelly,\\nManabu Nukaya, Peter G. Geiger, Silvia Balbo, Romel Dator,\\nBryant W. Megna, et al. 2016. “The Aryl Hydrocarbon\\nReceptor Is a Repressor of Inflammation ‐Associated\\nColorectal Tumorigenesis in Mouse.” Annals of Surgery264:\\n429–36. https://doi.org/10.1097/SLA.0000000000001874\\n248. Zhang, Lu, Qing Ji, Qian Chen, Zhenzhen Wei, Shuochuan\\nLiu, Long Zhang, Yuli Zhang, et al. 2023. “Akkermansia\\nmuciniphila Inhibits Tryptophan Metabolism via the\\nAhR/β‐Catenin Signaling Pathway to Counter the Progres-\\nsion of Colorectal Cancer.” International Journal of Biological\\nSciences 19: 4393–410. https://doi.org/10.7150/ijbs.85712\\n249. Patel, Dhwani, Iain A. Murray, Fangcong Dong,\\nAndrew J. Annalora, Krishne Gowda, Denise M. Coslo,\\nJacek Krzeminski, et al. 2023.“Induction of AHR Signaling\\nin Response to the Indolimine Class of Microbial Stress\\nMetabolites.” Metabolites 13: 985. https://doi.org/10.3390/\\nmetabo13090985\\n250. Seo, Su ‐Kil, Byungsuk Kwon. 2023. “Immune Regulation\\nThrough Tryptophan Metabolism.” Experimental & Molecular\\nMedicine 55: 1371 –9. https://doi.org/10.1038/s12276 ‐023‐\\n01028‐7\\nMICROBIOME IN CANCER | 77 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 77, 'page_label': '78'}, page_content='251. Hussain, Ahad, Li Xie, Guozhe Deng, Xuejun Kang. 2023.\\n“Common Alterations in Plasma Free Amino Acid Profiles\\nand Gut Microbiota‐Derived Tryptophan Metabolites of Five\\nTypes of Cancer Patients. ” Amino Acids 55: 1189 –200.\\nhttps://doi.org/10.1007/s00726‐023‐03308‐y\\n252. He, Jin, Peiwen Zhang, Linyuan Shen, Lili Niu, Ya Tan,\\nLei Chen, Ye Zhao, et al. 2020.“Short‐Chain Fatty Acids and\\nTheir Association With Signalling Pathways in Inflamma-\\ntion, Glucose and Lipid Metabolism.” International Journal\\nof Molecular Sciences 21: 6356. https://doi.org/10.3390/\\nijms21176356\\n253. Mann, Elizabeth R., Ying Ka Lam, Holm H. Uhlig. 2024.\\n“Short‐Chain Fatty Acids: Linking Diet, the Microbiome and\\nImmunity.” Nature Reviews Immunology24: 577–95. https://\\ndoi.org/10.1038/s41577‐024‐01014‐8\\n254. Vinolo, Marco A. R., Hosana G. Rodrigues, Renato T. Nachbar,\\nRui Curi. 2011.“Regulation of Inflammation by Short Chain\\nFatty Acids.” Nutrients 3: 858–76. https://doi.org/10.3390/\\nnu3100858\\n255. Van der Hee, Bart, Jerry M. Wells. 2021.“Microbial Regu-\\nlation of Host Physiology by Short‐chain Fatty Acids.” Trends\\nin Microbiology 29: 700 –12. https://doi.org/10.1016/j.tim.\\n2021.02.001\\n256. Tsunedomi, Ryouichi, Yoshitaro Shindo, Masao Nakajima,\\nKiyoshi Yoshimura, Hiroaki Nagano. 2023. “The Tumor\\nImmune Microenvironment in Pancreatic Cancer and\\nIts Potential in the Identification of Immunotherapy\\nBiomarkers.” Expert Review of Molecular Diagnostics 23:\\n1121–34. https://doi.org/10.1080/14737159.2023.2281482\\n257. Yan, Qingzhu, Shengnan Jia, Dongfu Li, Junling Yang. 2023.\\n“The Role and Mechanism of Action of Microbiota‐Derived\\nShort‐Chain Fatty Acids in Neutrophils: From the Activation\\nto Becoming Potential Biomarkers. ” Biomedicine &\\nPharmacotherapy = Biomedecine & Pharmacotherapie 169:\\n115821. https://doi.org/10.1016/j.biopha.2023.115821\\n258. Sun, Mingming, Wei Wu, Zhanju Liu, Yingzi Cong. 2017.\\n“Microbiota Metabolite Short Chain Fatty Acids, GPCR, and\\nInflammatory Bowel Diseases.” Journal of Gastroenterology\\n52: 1–8. https://doi.org/10.1007/s00535‐016‐1242‐9\\n259. Tsukuda, Naoki, Kana Yahagi, Taeko Hara, Yohei Watanabe,\\nHoshitaka Matsumoto, Hiroshi Mori, Koichi Higashi, et al.\\n2021. “Key Bacterial Taxa and Metabolic Pathways Affecting\\nGut Short‐Chain Fatty Acid Profiles in Early Life.” The ISME\\nJournal 15: 2574 –90. https://doi.org/10.1038/s41396 ‐021‐\\n00937‐7\\n260. Quinn‐Bohmann, Nick, Tomasz W ilmanski, Katherine\\nRamos Sarmiento, Lisa Levy, Johanna W. Lampe, Thomas\\nGurry, Noa Rappaport, et al. 2024.“Microbial Community‐Scale\\nMetabolic Modelling Predicts Personalized Short‐Chain Fatty\\nAcid Production Profiles in the Human Gut. ” Nature\\nMicrobiology 9: 1700–12. https://doi.org/10.1038/s41564‐024‐\\n01728‐4\\n261. Nogal, Ana, Francesco Asnicar, Amrita Vijay,\\nAfroditi Kouraki, Alessia Visconti, Panayiotis Louca,\\nKari Wong, et al. 2023.“Genetic and Gut Microbiome De-\\nterminants of SCFA Circulating and Fecal Levels, Post-\\nprandial Responses and Links to Chronic and Acute\\nInflammation.” Gut Microbes15: 2240050.https://doi.org/10.\\n1080/19490976.2023.2240050\\n262. Ikeda, Takako, Akari Nishida, Mayu Yamano, Ikuo Kimura.\\n2022. “Short‐Chain Fatty Acid Receptors and Gut Microbiota\\nas Therapeutic Targets in Metabolic, Immune, and Neuro-\\nlogical Diseases.” Pharmacology & Therapeutics239: 108273.\\nhttps://doi.org/10.1016/j.pharmthera.2022.108273\\n263. Hu, Tongtong, Qingqing Wu, Qi Yao, Kebing Jiang,\\nJiabin Yu, Qizhu Tang. 2022. “Short‐Chain Fatty Acid\\nMetabolism and Multiple Effects on Cardiovascular Dis-\\neases.” Ageing Research Reviews\\n81: 101706. https://doi.org/\\n10.1016/j.arr.2022.101706\\n264. Gomes, Sara, Ana Catarina Rodrigues, Valerio Pazienza,\\nAna Preto. 2023. “Modulation of the Tumor Micro-\\nenvironment by Microbiota ‐Derived Short ‐Chain Fatty\\nAcids: Impact in Colorectal Cancer Therapy.” International\\nJournal of Molecular Sciences 24: 5069. https://doi.org/10.\\n3390/ijms24065069\\n265. Thome, Carolin D., Patrick Tausche, Katja Hohenberger,\\nZuqin Yang, Susanne Krammer, Denis I. Trufa, Horia Sirbu,\\nJoachim Schmidt, Susetta Finotto. 2024.“Short‐Chain Fatty\\nAcids Induced Lung Tumor Cell Death and Increased\\nPeripheral Blood CD4+ T Cells in NSCLC and Control Pa-\\ntients Ex Vivo. ” Frontiers in Immunology 15: 1328263.\\nhttps://doi.org/10.3389/fimmu.2024.1328263\\n266. Kim, Myunghoo, Yaqing Qie, Jeongho Park, Chang H. Kim.\\n2016. “Gut Microbial Metabolites Fuel Host Antibody\\nResponses.” Cell Host & Microbe20: 202–14. https://doi.org/\\n10.1016/j.chom.2016.07.001\\n267. Thiruvengadam, Muthu, Umadevi Subramanian, Baskar\\nVenkidasamy, Prabhu Thirupathi, Ramkumar Samynathan,\\nMohammad Ali Shariati, Maksim Rebezov, Ill‐Min Chung,\\nKannan R. R. Rengasamy. 2023.“Emerging Role of Nutri-\\ntional Short‐Chain Fatty Acids (SCFAs) Against Cancer via\\nModulation of Hematopoiesis. ” Critical Reviews in Food\\nScience and Nutrition 63: 827–44. https://doi.org/10.1080/\\n10408398.2021.1954874\\n268. Hou, Huiqin, Danfeng Chen, Kexin Zhang, Wanru Zhang,\\nTianyu Liu, Sinan Wang, Xin Dai, et al. 2022. “Gut\\nMicrobiota‐Derived Short‐Chain Fatty Acids and Colorectal\\nCancer: Ready for Clinical Translation?” Cancer Letters 526:\\n225–35. https://doi.org/10.1016/j.canlet.2021.11.027\\n269. Winston, Jenessa A., Casey M. Theriot. 2020.“Diversification\\nof Host Bile Acids by Members of the Gut Microbiota.” Gut\\nMicrobes 11: 158–71. https://doi.org/10.1080/19490976.2019.\\n1674124\\n270. Su, Xiaomin, Yunhuan Gao, Rongcun Yang. 2023.“Gut Mi-\\ncrobiota Derived Bile Acid Metabolites Maintain the\\nHomeostasis of Gut and Systemic Immunity.” Frontiers in\\nImmunology 14: 1127743. https://doi.org/10.3389/fimmu.\\n2023.1127743\\n271. Sinha, Sidhartha R., Yeneneh Haileselassie, Linh P. Nguyen,\\nC a r o l i n aT r o p i n i ,M i nW a n g ,L a r e nS .B e c k e r ,D a v i sS i m ,e ta l .\\n2020. “Dysbiosis‐Induced Secondary Bile Acid Deficiency Pro-\\nmotes Intestinal Inflammation.” Cell Host & Microbe 27:\\n659–70.e5. https://doi.org/10.1016/j.chom.2020.01.021\\n272. Re žen, Tadeja, Damjana Rozman, Tünde Kovács,\\nPatrik Kovács, Adrienn Sipos, Péter Bai, Edit Mikó. 2022.\\n“The Role of Bile Acids in Carcinogenesis.” Cellular and\\nMolecular Life Sciences 79: 243. https://doi.org/10.1007/\\ns00018‐022‐04278‐2\\n78 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 78, 'page_label': '79'}, page_content=\"273. Lenci, Ilaria, Martina Milana, Alessandro Signorello,\\nGiuseppe Grassi, Leonardo Baiocchi. 2023.“Secondary Bile\\nAcids and the Biliary Epithelia: The Good and the Bad.”\\nWorld Journal of Gastroenterology29: 357–36. https://doi.org/\\n10.3748/wjg.v29.i2.357\\n274. Situ, Yingheng, Pengpeng Zhang, Cangang Zhang,\\nAimin Jiang, Nan Zhang, Lingxuan Zhu, Weiming Mou, et al.\\n2025. “The Metabolic Dialogue Between Intratumoural\\nMicrobes and Cancer: Implications for Immunotherapy. ”\\nEBioMedicine 115: 105708. https://doi.org/10.1016/j.ebiom.\\n2025.105708\\n275. Krumz, L. M., R. B. Gudkova, L. Kh. Indejkina,\\nE. A. Sabelnikova, A. I. Parfenov. 2020.“[Bile Acids Are a\\nRisk Factor for Colorectal Cancer].” Terapevticheskii Arkhiv\\n92: 93–6. https://doi.org/10.26442/00403660.2020.02.000457\\n276. Kim, Tae ‐Young, Seungil Kim, Yeji Kim, Yong ‐Soo Lee,\\nSohyeon Lee, Su‐Hyun Lee, Mi‐Na Kweon. 2022.“A High‐Fat\\nDiet Activates the BAs ‐FXR Axis and Triggers Cancer ‐\\nAssociated Fibroblast Properties in the Colon.” Cellular and\\nMolecular Gastroenterology and Hepatology 13: 1141 –59.\\nhttps://doi.org/10.1016/j.jcmgh.2021.12.015\\n277. Ma, Chi, Miaojun Han, Bernd Heinrich, Qiong Fu,\\nQianfei Zhang, Milan Sandhu, David Agdashian, et al. 2018.\\n“Gut Microbiome‐Mediated Bile Acid Metabolism Regulates\\nLiver Cancer via NKT Cells.” Science 360: eaan5931. https://\\ndoi.org/10.1126/science.aan5931\\n278. Zha, Andong, Ming Qi, Yuankun Deng, Hao Li, Nan Wang,\\nChengming Wang, Simeng Liao, et al. 2024. “Gut Bifido-\\nbacterium pseudocatenulatumProtects Against Fat Deposition\\nby Enhancing Secondary Bile Acid Biosynthesis.” iMeta 3:\\ne261. https://doi.org/10.1002/imt2.261\\n279. Tobón ‐Cornejo, Sandra, Monica Sanchez‐Tapia, Rocio Guizar‐\\nHeredia, Laura Velázquez Villegas, Lilia G. Noriega,\\nJanette Furuzawa‐Carballeda, Rogelio Hernández‐Pando, et al.\\n2025. “Increased Dietary Protein Stimulates Amino Acid\\nCatabolism via the Gut Microbiota and Secondary Bile Acid\\nProduction.” Gut Microbes 17: 2465896.\\nhttps://doi.org/10.\\n1080/19490976.2025.2465896\\n280. Louca, Panayiotis, Abraham S. Meijnikman, Ana Nogal,\\nFrancesco Asnicar, Ilias Attaye, Amrita Vijay,\\nAfroditi Kouraki, et al. 2023.“The Secondary Bile Acid Iso-\\nursodeoxycholate Correlates With Post ‐Prandial Lipemia,\\nInflammation, and Appetite and Changes Post‐Bariatric Sur-\\ngery.” Cell Reports. Medicine 4: 100993. https://doi.org/10.\\n1016/j.xcrm.2023.100993\\n281. Lamichhane, Santosh, Partho Sen, Alex M. Dickens,\\nMarina Amaral Alves, Taina Härkönen, Jarno Honkanen,\\nTommi Vatanen, et al. 2022.“Dysregulation of Secondary Bile\\nAcid Metabolism Precedes Islet Autoimmunity and Type 1\\nDiabetes.” Cell Reports. Medicine3: 100762.https://doi.org/10.\\n1016/j.xcrm.2022.100762\\n282. Wei, Yanxin, Chen Lu, Shengsheng Jiang, Yanyan Zhang,\\nQiuchun Li, Wen ‐Ju Bai, Xiqing Wang. 2020. “Directed\\nEvolution of a Tryptophan 2,3‐Dioxygenase for the Diaster-\\neoselective Monooxygenation of Tryptophans.” Angewandte\\nChemie International Edition 59: 3043–7. https://doi.org/10.\\n1002/anie.201911825\\n283. Cervenka, Igor, Leandro Z. Agudelo, Jorge L. Ruas. 2017.\\n“Kynurenines: Tryptophan's Metabolites in Exercise,\\nInflammation, and Mental Health.” Science 357: eaaf9794.\\nhttps://doi.org/10.1126/science.aaf9794\\n284. Zhu, Yueting, Aolei Hu, Meilin Chen, Yunting Zhang,\\nTao Gong, Zhirong Zhang, Ruilian Yu, Yao Fu. 2025.\\n“3‐Indoleacetic Acid‐Modified Chondroitin Sulfate‐Mediated\\nPaclitaxel Nanocrystal Assembly for the Treatment of Pan-\\ncreatic Cancer. ” ACS Applied Materials & Interfaces 17:\\n9035–46. https://doi.org/10.1021/acsami.4c19450\\n285. Li, Zhuangzhuang, Baoyan Ding, Mustafa R. K. Ali,\\nLizhen Zhao, Xiaoling Zang, Zhihua Lv. 2022.“Dual Effect of\\nTryptamine on Prostate Cancer Cell Growth Regulation: A\\nPilot Study.” International Journal of Molecular Sciences23:\\n11087. https://doi.org/10.3390/ijms231911087\\n286. Jing, Wanghui, Sijing Dong, Yinyue Xu, Jingjing Liu,\\nJiawei Ren, Xue Liu, Min Zhu, et al. 2025.“Gut Microbiota‐\\nDerived Tryptophan Metabolites Regulated by Wuji Wan to\\nAttenuate Colitis Through AhR Signaling Activation.” Acta\\nPharmaceutica Sinica. B15: 205–23. https://doi.org/10.1016/j.\\napsb.2024.11.009\\n287. Chen, Chuan, Zheng Cao, Hehua Lei, Cui Zhang,\\nMengjing Wu, Shaohua Huang, Xinzhi Li, et al. 2024.\\n“Microbial Tryptophan Metabolites Ameliorate Ovariectomy\\n‐\\nInduced Bone Loss by Repairing Intestinal AhR‐Mediated\\nGut‐Bone Signaling Pathway.” Advanced Science (Weinheim,\\nBaden‐Wurttemberg, Germany) 11: e2404545. https://doi.org/\\n10.1002/advs.202404545\\n288. Arinze, Nkiruka V., Wenqing Yin, Saran Lotfollahzadeh,\\nMarc Arthur Napoleon, Sean Richards, Joshua A. Walker,\\nMostafa Belghasem, et al. 2022. “Tryptophan Metabolites\\nSuppress the Wnt Pathway and Promote Adverse Limb\\nEvents in Chronic Kidney Disease.” The Journal of Clinical\\nInvestigation 132: e142260.https://doi.org/10.1172/JCI142260\\n289. Platten, Michael, Nikolaus Von Knebel Doeberitz, Iris Oezen,\\nWolfgang Wick, Katharina Ochs. 2014. “Cancer Immuno-\\ntherapy by Targeting IDO1/TDO and Their Downstream Ef-\\nfectors.” Frontiers in Immunology 5: 673. https://doi.org/10.\\n3389/fimmu.2014.00673\\n290. Ghosh, Sweta, Caleb Samuel Whitley, Bodduluri Haribabu,\\nVenkatakrishna Rao Jala. 2021. “Regulation of Intestinal\\nBarrier Function by Microbial Metabolites. ” Cellular and\\nMolecular Gastroenterology and Hepatology 11: 1463 –82.\\nhttps://doi.org/10.1016/j.jcmgh.2021.02.007\\n291. Turpin, Williams, Sun‐Ho Lee, Juan Antonio Raygoza Garay,\\nKaren L. Madsen, Jonathan B. Meddings, Larbi Bedrani,\\nNamita Power, et al. 2020.“Increased Intestinal Permeability\\nIs Associated With Later Development of Crohn's Disease.”\\nGastroenterology 159: 2092–100.e5. https://doi.org/10.1053/j.\\ngastro.2020.08.005\\n292. Tozzi, Michela, Alessia Fiore, Sara Travaglione, Francesca\\nMarcon, Gabriella Rainaldi, Elena Angela Pia Germinario,\\nIlenia Laterza, et al. 2025. “E. Coli Cytotoxic Necrotizing\\nFactor‐1 Promotes Colorectal Carcinogenesis by Causing\\nOxidative Stress, DNA Damage and Intestinal Permeability\\nAlteration.” Journal of Experimental & Clinical Cancer\\nResearch: CR 44: 29. https://doi.org/10.1186/s13046 ‐024‐\\n03271‐w\\n293. Leibovitzh, Haim, Sun ‐Ho Lee, Mingyue Xue,\\nJuan Antonio Raygoza Garay, Cristian Hernandez ‐Rocha,\\nKaren L. Madsen, Jonathan B. Meddings, et al. 2022.“Altered\\nMICROBIOME IN CANCER | 79 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 79, 'page_label': '80'}, page_content=\"Gut Microbiome Composition and Function Are Associated\\nWith Gut Barrier Dysfunction in Healthy Relatives of Patients\\nWith Crohn's Disease.” Gastroenterology 163: 1364–76.e10.\\nhttps://doi.org/10.1053/j.gastro.2022.07.004\\n294. Santos, Javier, Maria Rescigno. 2024.“Gut Barrier Leakiness:\\nTime to Take It Seriously?” Gastroenterology 167: 1080–2.\\nhttps://doi.org/10.1053/j.gastro.2024.08.011\\n295. Magnus, Yorick, Joelle BouSaba, Wassel Sannaa,\\nSanna McKinzie, Irene Busciglio, Michael Camilleri. 2022.\\n“Bile Acid Diarrhea Is Associated With Increased Intestinal\\nPermeability Compared With Irritable Bowel Syndrome ‐\\nDiarrhea.” Gastroenterology 162: 1343–5.e1. https://doi.org/\\n10.1053/j.gastro.2021.12.243\\n296. Lee, Sun‐Ho, Maham Bushra, Lanhui Qiu, Anne M. Griffiths,\\nWilliams Turpin, Kenneth Croitoru, Crohn's and Colitis\\nCanada‐Genetic, Environmental, Microbial (CCC ‐GEM)\\nProject Research Consortium, Kenneth Croitoru, Sun ‐Ho\\nLee, et al. 2025. “Early Life Exposure to Parental Crohn's\\nDisease Is Associated With Offspring's Gut Microbiome, Gut\\nPermeability, and Increased Risk of Future Crohn's Disease.”\\nGastroenterology 168: 385 –8.e3. https://doi.org/10.1053/j.\\ngastro.2024.09.033\\n297. Horowitz, Arie, Sandra D. Chanez ‐Paredes, Xenia Haest,\\nJerrold R. Turner. 2023.“Paracellular Permeability and Tight\\nJunction Regulation in Gut Health and Disease. ” Nature\\nReviews Gastroenterology & Hepatology20: 417–32. https://doi.\\norg/10.1038/s41575‐023‐00766‐3\\n298. Chopyk, Daniel M., Arash Grakoui. 2020. “Contribution of\\nthe Intestinal Microbiome and Gut Barrier to Hepatic Dis-\\norders.” Gastroenterology 159: 849 –63. https://doi.org/10.\\n1053/j.gastro.2020.04.077\\n299. Rashidah, Nur Hannah, Siong Meng Lim, Chin Fen Neoh,\\nMajeed, Abu Bakar Abdul Tan, Maw Pin, Hui Min Khor,\\nAi Huey Tan, Siti Hajar Rehiman, Kalavathy Ramasamy,.\\n2022. “Differential Gut Microbiota and Intestinal Permeabil-\\nity Between Frail and Healthy Older Adults: A Systematic\\nReview.” Ageing Research Reviews82: 101744.https://doi.org/\\n10.1016/j.arr.2022.101744\\n300. Martel, Jan, Shih\\n‐Hsin Chang, Yun ‐Fei Ko, Tsong ‐Long\\nHwang, John D. Young, David M. Ojcius. 2022.“Gut Bar-\\nrier Disruption and Chronic Disease. ” Trends in\\nEndocrinology & Metabolism 33: 247–65. https://doi.org/10.\\n1016/j.tem.2022.01.002\\n301. Giambra, Vincenzo, Danilo Pagliari, Pierluigi Rio, Beatrice Totti,\\nChiara Di Nunzio, Annalisa Bosi, Cristina Giaroni, et al. 2023.\\n“Gut Microbiota, Inflammatory Bowel Disease, and Cancer: The\\nRole of Guardians of Innate Immunity.” Cells 12: 2654.https://\\ndoi.org/10.3390/cells12222654\\n302. Soler, A. P. 1999.“Increased Tight Junctional Permeability Is\\nAssociated With the Development of Colon Cancer. ”\\nCarcinogenesis 20: 1425–32. https://doi.org/10.1093/carcin/20.\\n8.1425\\n303. Karczewski, Jurgen, Freddy J. Troost, Irene Konings,\\nJan Dekker, Kleerebezem, Michiel Brummer, Robert‐Jan M.\\nWells, Jerry M. 2010.“Regulation of Human Epithelial Tight\\nJunction Proteins by Lactobacillus plantarum In Vivo and\\nProtective Effects on the Epithelial Barrier. ” American\\nJournal of Physiology‐Gastrointestinal and Liver Physiology\\n298: G851–9. https://doi.org/10.1152/ajpgi.00327.2009\\n304. Méndez ‐Sánchez, Nahum, Alejandro Valencia ‐Rodriguez,\\nAlfonso Vera ‐Barajas, Ludovico Abenavoli, Emidio\\nScarpellini, Guadalupe Ponciano ‐Rodriguez. 2020. “The\\nMechanism of Dysbiosis in Alcoholic Liver Disease Leading\\nto Liver Cancer.” Hepatoma Research6: 5.https://doi.org/10.\\n20517/2394‐5079.2019.29\\n305. Rychter, Anna Maria, Liliana Łykowska‐Szuber, Agnieszka\\nZawada, Aleksandra Szymczak‐Tomczak, Alicja Ewa Ratajczak,\\nKinga Skoracka, Michalina Kolan, Agnieszka Dobrowolska,\\nIwona Krela‐Kaźmierczak. 2023. “Why Does Obesity as an\\nInflammatory Condition Predispose to Colorectal Cancer?”\\nJournal of Clinical Medicine12: 2451.\\nhttps://doi.org/10.3390/\\njcm12072451\\n306. Klein, Gerald L., Bryon W. Petschow, Audrey L. Shaw,\\nEric Weaver. 2013. “Gut Barrier Dysfunction and Microbial\\nTranslocation in Cancer Cachexia: A New Therapeutic Tar-\\nget.” Current Opinion in Supportive and Palliative Care 7:\\n361–7. https://doi.org/10.1097/SPC.0000000000000017\\n307. Wang, Hong ‐Bo, Peng‐Yuan Wang, Xin Wang, Yuan‐Lian\\nWan, Yu‐Cun Liu. 2012.“Butyrate Enhances Intestinal Epi-\\nthelial Barrier Function via Up‐Regulation of Tight Junction\\nProtein Claudin ‐1 Transcription. ” Digestive Diseases and\\nSciences 57: 3126 –35. https://doi.org/10.1007/s10620 ‐012‐\\n2259‐4\\n308. Fachi, José Luís, Jaqueline de Souza Felipe, Laís\\nPassariello Pral, Bruna Karadi Da Silva, Renan Oliveira Corrêa,\\nMirella Cristiny Pereira De Andrade, Denise Morais\\nDa Fonseca, et al. 2019.“Butyrate Protects Mice From Clos-\\ntridium Difficile‐Induced Colitis Through an HIF‐1‐Dependent\\nMechanism.” Cell Reports 27: 750–61.e7. https://doi.org/10.\\n1016/j.celrep.2019.03.054\\n309. Chen, Guangxin, Xin Ran, Bai Li, Yuhang Li, Dewei He,\\nBingxu Huang, Shoupeng Fu, Juxiong Liu, Wei Wang. 2018.\\n“Sodium Butyrate Inhibits Inflammation and Maintains Epi-\\nthelium Barrier Integrity in a TNBS‐Induced Inflammatory\\nBowel Disease Mice Model. ” EBioMedicine 30: 317 –25.\\nhttps://doi.org/10.1016/j.ebiom.2018.03.030\\n310. Tong, Yao, Huiru Gao, Qiuchen Qi, Xiaoyan Liu, Juan Li,\\nJie Gao, Peilong Li, et al. 2021. “High Fat Diet, Gut\\nMicrobiome and Gastrointestinal Cancer.” Theranostics 11:\\n5889–910. https://doi.org/10.7150/thno.56157\\n311. Muszy ński, Damian, Anna Kudra, Bartosz Kamil Sobocki,\\nMarcin Folwarski, Ermanno Vitale, Veronica Filetti,\\nWojciech Dudzic, Karolina Ka źmierczak‐Siedlecka,\\nKarol Połom. 2022.\\n“Esophageal Cancer and Bacterial Part of\\nGut Microbiota —A Multidisciplinary Point of View. ”\\nFrontiers in Cellular and Infection Microbiology12: 1057668.\\nhttps://doi.org/10.3389/fcimb.2022.1057668\\n312. Huang, Hui, Wei Zhong, Xiaojiao Wang, Ying Yang,\\nTianmu Wu, Runyang Chen, Yanling Liu, Feng He, Jun Li.\\n2023. “The Role of Gastric Microecological Dysbiosis in\\nGastric Carcinogenesis. ” Frontiers in Microbiology 14:\\n1218395. https://doi.org/10.3389/fmicb.2023.1218395\\n313. Miao, Sainan, Huan Qiu. 2024. “The Microbiome in the\\nPathogenesis of Lung Cancer: The Role of Microbiome in\\nLung Cancer Pathogenesis.” APMIS 132: 68–80. https://doi.\\norg/10.1111/apm.13359\\n314. Lo, Bobby, Mirabella Zhao, Ida Vind, Johan Burisch. 2021.\\n“The Risk of Extraintestinal Cancer in Inflammatory Bowel\\n80 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 80, 'page_label': '81'}, page_content=\"Disease: A Systematic Review and Meta ‐analysis of\\nPopulation‐Based Cohort Studies.” Clinical Gastroenterology\\nand Hepatology 19: 1117–38.e19. https://doi.org/10.1016/j.\\ncgh.2020.08.015\\n315. Lavelle, Aonghus, Stéphane Nancey, Jean‐Marie Reimund,\\nDavid Laharie, Philippe Marteau, Xavier Treton, Matthieu\\nAllez, et al. 2022.“Fecal Microbiota and Bile Acids in IBD\\nPatients Undergoing Screening for Colorectal Cancer.” Gut\\nMicrobes 14: 2078620.https://doi.org/10.1080/19490976.2022.\\n2078620\\n316. Jin, Xuanhong, Liangkun You, Jincheng Qiao, Weidong Han,\\nHongming Pan. 2024. “Autophagy in Colitis ‐Associated\\nColon Cancer: Exploring Its Potential Role in Reducing Ini-\\ntiation and Preventing IBD ‐Related CAC Development. ”\\nAutophagy 20: 242 –58. https://doi.org/10.1080/15548627.\\n2023.2259214\\n317. Zhao, Rui, Qian ‐Yi Wan, Yutao Wu, Yong Wang, Ya‐Ping\\nCui, Xiaoding Shen, Xiao Ting Wu. 2021.“Crohn's Disease\\nInstead of UC Might Increase the Risk of Small Bowel Can-\\ncer.” Gut 70: 809 –10. https://doi.org/10.1136/gutjnl‐2020‐\\n322201\\n318. Wetwittayakhlang, Panu, Petra A. Golovics, Lorant Gonczi,\\nLaszlo Lakatos, Peter L. Lakatos, Gyula David, Zsuzsanna\\nErdelyi, et al. VESZPREM EPIDEMIOLOGY STUDY\\nGROUP. 2024. “Stable Incidence and Risk Factors of Colo-\\nrectal Cancer in Ulcerative Colitis: A Population ‐Based\\nCohort Between 1977–2020.” Clinical Gastroenterology and\\nHepatology 22: 191–3.e3. https://doi.org/10.1016/j.cgh.2023.\\n03.022\\n319. Shah, Shailja C., Steven H. Itzkowitz. 2022. “Colorectal\\nCancer in Inflammatory Bowel Disease: Mechanisms and\\nManagement.” Gastroenterology 162: 715–30.e3. https://doi.\\norg/10.1053/j.gastro.2021.10.035\\n320. Axelrad, Jordan E., Ola Olén, Michael C. Sachs, Rune\\nErichsen, Lars Pedersen, Jonas Halfvarson, Johan Askling,\\net al. 2021.“Inflammatory Bowel Disease and Risk of Small\\nBowel Cancer: A Binational Population‐Based Cohort Study\\nFrom Denmark and Sweden.”\\nGut 70: 297–308. https://doi.\\norg/10.1136/gutjnl‐2020‐320945\\n321. Elmahdi, Rahma, Camilla E. Lemser, Sandra B. Thomsen,\\nKristine H. Allin, Manasi Agrawal, Tine Jess. 2022.“Devel-\\nopment of Cancer Among Patients With Pediatric ‐Onset\\nInflammatory Bowel Disease: A Meta ‐Analysis of\\nPopulation‐Based Studies.” JAMA Network Open5: e220595.\\nhttps://doi.org/10.1001/jamanetworkopen.2022.0595\\n322. Li, Junshu, Yanhong Ji, Na Chen, Lei Dai, Hongxin Deng.\\n2023. “Colitis‐Associated Carcinogenesis: Crosstalk Between\\nTumors, Immune Cells and Gut Microbiota. ” Cell &\\nBioscience 13: 194. https://doi.org/10.1186/s13578 ‐023‐\\n01139‐8\\n323. Deris Zayeri, Zeinab, Abazar Parsi, Saeid Shahrabi,\\nMasoud Kargar, Nader Davari, Najmaldin Saki. 2023.“Epi-\\ngenetic and Metabolic Reprogramming in Inflammatory\\nBowel Diseases: Diagnostic and Prognostic Biomarkers in\\nColorectal Cancer.” Cancer Cell International23: 264.https://\\ndoi.org/10.1186/s12935‐023‐03117‐z\\n324. Liang, Bing, Yanhong Wang, Jiazhen Xu, Yingchun Shao,\\nDongming Xing. 2023.“Unlocking the Potential of Targeting\\nHistone‐Modifying Enzymes for Treating IBD and CRC. ”\\nClinical Epigenetics15: 146.https://doi.org/10.1186/s13148‐023‐\\n01562‐1\\n325. Silveira, Denise Sayuri Calheiros, Luciana Chain Veronez,\\nLuís Carlos Lopes ‐Júnior, Elen Anatriello, Mariângela\\nOttoboni Brunaldi, Gabriela Pereira‐da‐Silva. 2020. “Lacto-\\nbacillus bulgaricus Inhibits Colitis‐Associated Cancer via a\\nNegative Regulation of Intestinal Inflammation in Azox-\\nymethane/Dextran Sodium Sulfate Model.” World Journal of\\nGastroenterology 26: 6782\\n–94. https://doi.org/10.3748/wjg.\\nv26.i43.6782\\n326. Javid, Hossein, Mahsa Akbari Oryani, Sanaz Akbari,\\nTaghi Amiriani, Samaneh Ravanbakhsh, Nastaran\\nRezagholinejad, Amir ‐R. Afshari, Mehdi Karimi ‐Shahri.\\n2023. “L. plantarum and L. lactis as a Promising Agent in\\nTreatment of Inflammatory Bowel Disease and Colorectal\\nCancer.” Future Microbiology 18: 1197–209. https://doi.org/\\n10.2217/fmb‐2023‐0076\\n327. Wittek, Agnes, Babett Steglich, Christian Casar, Oliver Seiz,\\nPhilipp Huber, Hanno Ehlken, Dominik Reher, et al. 2024.\\n“A Gradient of Intestinal Inflammation in Primary Sclerosing\\nCholangitis.” Inflammatory Bowel Diseases 30: 900 –10.\\nhttps://doi.org/10.1093/ibd/izad137\\n328. Zhang, Henan, Junrui Wu, Na Li, Rina Wu, Wei Chen. 2023.\\n“Microbial Influence on Triggering and Treatment of Host\\nCancer: An Intestinal Barrier Perspective. ” Biochimica et\\nBiophysica Acta (BBA)—Reviews on Cancer 1878: 188989.\\nhttps://doi.org/10.1016/j.bbcan.2023.188989\\n329. Wang, Lei, Mengfan Li, Yu Gu, Junli Shi, Jing Yan,\\nXin Wang, Bingqing Li, et al. 2024. “Dietary Flavonoids‐\\nMicrobiota Crosstalk in Intestinal Inflammation and Carci-\\nnogenesis.” The Journal of Nutritional Biochemistry 125:\\n109494. https://doi.org/10.1016/j.jnutbio.2023.109494\\n330. Intratumor Microbes Promote Murine Breast Cancer Cell\\nInvasion. 2022.Cancer Discovery12: 1407.https://doi.org/10.\\n1158/2159‐8290.CD‐RW2022‐067\\n331. Liu, Ning ‐Ning, Cheng‐Xiang Yi, Lu‐Qi Wei, Jin‐An Zhou,\\nTong Jiang, Cong‐Cong Hu, Lu Wang, et al. 2024.“The In-\\ntratumor Mycobiome Promotes Lung Cancer Progression via\\nMyeloid‐Derived Suppressor Cells.” Cancer Cell 42: 318–22.\\nhttps://doi.org/10.1016/j.ccell.2024.01.005\\n332. Liu, Ning ‐Ning, Cheng‐Xiang Yi, Lu\\n‐Qi Wei, Jin‐An Zhou,\\nTong Jiang, Cong‐Cong Hu, Lu Wang, et al. 2023.“The In-\\ntratumor Mycobiome Promotes Lung Cancer Progression via\\nMyeloid‐Derived Suppressor Cells. ” Cancer Cell 41:\\n1927–44.e9. https://doi.org/10.1016/j.ccell.2023.08.012\\n333. Gao, Zhifei, A imin Jiang, Zizhuo Li, Lingxuan Zhu,\\nWeiming Mou, Weitao Shen, Peng Luo, et al. 2025.“Heteroge-\\nneity of Intratumoral Microbiota Within the Tumor Micro-\\nenvironment and Relationship to Tumor Development.” Med\\nResearch 1(1): 32–61. https://doi.org/10.1002/mdr2.70006\\n334. Yu, Chunyue, Zeqi Su, Yicong Li, Yadong Li, Kaige Liu,\\nFuhao Chu, Ting Liu, Runhua Chen, Xia Ding. 2020.“Dysbiosis\\nof Gut Microbiota Is Associated With Gastric Carcinogenesis in\\nRats.” Biomedicine & Pharmacotherapy = Biomedecine &\\nPharmacotherapie 126: 110036. https://doi.org/10.1016/j.\\nbiopha.2020.110036\\n335. Sun, Lejia, Xindi Ke, Ai Guan, Bao Jin, Jiangming Qu,\\nYinhan Wang, Xiang Xu, et al. 2023.“Intratumoural Micro-\\nbiome Can Predict the Prognosis of Hepatocellular\\nMICROBIOME IN CANCER | 81 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 81, 'page_label': '82'}, page_content=\"Carcinoma After Surgery. ” Clinical and Translational\\nMedicine 13: e1331.https://doi.org/10.1002/ctm2.1331\\n336. Liu, Fangjie, Qiaoting Luo, Yu Xi, Pengxin Zhang,\\nYingjia Wu, Suping Guo, Yaoling Dong, et al. 2025.“Early\\nNutritional Intervention in Patients With Non‐Small Cell\\nLung Cancer Receiving Concurrent Chemoradiotherapy: A\\nPhase II Prospective Study.” Nutrients 17: 1389. https://doi.\\norg/10.3390/nu17081389\\n337. Huang, Jian‐Hang, Jie Wang, Xiao‐Qiang Chai, Zhong‐Chen Li,\\nYing‐Hua Jiang, Jun Li, Xing Liu, et al. 2022.“The Intratumoral\\nBacterial Metataxonomic Signature of Hepatocellular Carci-\\nnoma.” Microbiology Spectrum10: e0098322.https://doi.org/10.\\n1128/spectrum.00983‐22\\n338. Fu, Aikun, Bingqing Yao, Tingting Dong, Yongyi Chen,\\nJia Yao, Yu Liu, Hang Li, et al. 2022. “Tumor‐Resident\\nIntracellular Microbiota Promotes Metastatic Colonization in\\nBreast Cancer.” Cell 185: 1356–72.e26. https://doi.org/10.\\n1016/j.cell.2022.02.027\\n339. An, Jeongshin, Hyungju Kwon, Young Ju Kim. 2023.“The\\nFirmicutes/Bacteroidetes Ratio as a Risk Factor of Breast\\nCancer.” Journal of Clinical Medicine 12: 2216. https://doi.\\norg/10.3390/jcm12062216\\n340. Mouradov, Dmitri, Paul Greenfield, Shan Li, Eun‐Jung In,\\nClaire Storey, Anuratha Sakthianandeswaren, Peter\\nGeorgeson, et al. 2023.“Oncomicrobial Community Profiling\\nIdentifies Clinicomolecular and Prognostic Subtypes of\\nColorectal Cancer.” Gastroenterology 165: 104–20. https://doi.\\norg/10.1053/j.gastro.2023.03.205\\n341. Zeng, Xuejun, Hang Jia, Xiao Zhang, Xin Wang,\\nZhouli Wang, Zhenpeng Gao, Yahong Yuan, Tianli Yue.\\n2021. “Supplementation of Kefir Ameliorates Azox-\\nymethane/Dextran Sulfate Sodium Induced Colorectal Can-\\ncer by Modulating the Gut Microbiota.” Food & Function12:\\n11641–55. https://doi.org/10.1039/d1fo01729b\\n342. Liu, Yanhong, Jacqueline L. O'Brien, Nadim J. Ajami,\\nMichael E. Scheurer, E. Susan Amirian, Georgina Armstrong,\\nSpiridon Tsavachidis, et al. 2018. “Lung Tissue Microbial\\nProfile in Lung Cancer Is Distinct From Emphysema. ”\\nAmerican Journal of Cancer Research8: 1775–87.\\n343. Yu, Guoqin, Mitchell H. Gail, Dario Consonni, Michele\\nCarugno, Michael Humphrys, Angela C. Pesatori, Neil E.\\nCaporaso, et al. 2016.“Characterizing Human Lung Tissue\\nMicrobiota and Its Relationship to Epidemiological and\\nClinical Features.” Genome Biology 17: 163. https://doi.org/\\n10.1186/s13059‐016‐1021‐1\\n344. Jang, Hye Jin, Ji Yeon Choi, Kangjoon Kim, Seung\\nHyun Yong, Yeon Wook Kim, Song Yee Kim, Eun\\nYoung Kim, et al. 2021.“Relationship of the Lung Micro-\\nbiome With PD ‐L1 Expression and Immunotherapy\\nResponse in Lung Cancer.” Respiratory Research 22: 322.\\nhttps://doi.org/10.1186/s12931‐021‐01919‐1\\n345. Hilmi, Marc, Maud Kamal, Sophie Vacher, Célia Dupain,\\nSabrina Ibadioune, Maral Halladjian, Marie Paule Sablin,\\net al. 2023. “Intratumoral Microbiome Is Driven by Meta-\\nstatic Site and Associated With Immune Histopathological\\nParameters: An Ancillary Study of the SHIVA Clinical Trial.”\\nEuropean Journal of Cancer183: 152–61. https://doi.org/10.\\n1016/j.ejca.2023.01.024\\n346. Peng, Shengkun, Anqi Lin, Aimin Jiang, Cangang Zhang,\\nJian Zhang, Quan Cheng, Peng Luo, Yifeng Bai. 2024.“CTLs\\nHeterogeneity and Plasticity: Implications for Cancer\\nImmunotherapy.” Molecular Cancer 23: 58. https://doi.org/\\n10.1186/s12943‐024‐01972‐6\\n347. Zhu, Zhuxian, Jixu Cai, Weiwei Hou, Ke Xu, Xuxiao Wu,\\nYuanlin Song, Chunxue Bai, Yin‐Yuan Mo, Ziqiang Zhang.\\n2023. “Microbiome and Spatially Resolved Metabolomics\\nAnalysis Reveal the Anticancer Role of Gut Akkermansia\\nmuciniphila by Crosstalk With Intratumoral Microbiota and\\nReprogramming Tumoral Metabolism in Mice.” Gut Microbes\\n15: 2166700. https://doi.org/10.1080/19490976.2023.2166700\\n348. Abed, Jawad, Johanna E. M. Emgård, Gideon Zamir,\\nMouhammad Faroja, Gideon Almogy, Amalie Grenov,\\nAsaf Sol, et al. 2016.“Fap2 Mediates Fusobacterium nucle-\\natum Colorectal Adenocarcinoma Enrichment by Binding to\\nTumor‐Expressed Gal‐GalNAc.” Cell Host & Microbe 20:\\n215–25. https://doi.org/10.1016/j.chom.2016.07.006\\n349. Wong ‐\\nRolle, Abigail, Qiang Dong, Yunhua Zhu, Prajan\\nDivakar, Jyh Liang Hor, Noemi Kedei, Madeline Wong, et al.\\n2022. “Spatial Meta‐Transcriptomics Reveal Associations of\\nIntratumor Bacteria Burden With Lung Cancer Cells Showing\\na Distinct Oncogenic Signature.” Journal for Immunotherapy\\nof Cancer 10: e004698. https://doi.org/10.1136/jitc ‐2022‐\\n004698\\n350. Shirai, Hiroaki, Cocoro Ito, Kosuke Tsukada. 2022. “pH‐\\nTaxis Drives Aerobic Bacteria in Duodenum to Migrate Into\\nthe Pancreas With Tumors. ” Scientific Reports 12: 1783.\\nhttps://doi.org/10.1038/s41598‐022‐05554‐8\\n351. Zhang, Zhilin, Yiqun Liao, Dong Tang. 2022.“Intratumoral\\nMicrobiota: New Frontiers in Tumor Immunity.” Carcinogenesis\\n43: 719–27. https://doi.org/10.1093/carcin/bgac063\\n352. Peng, Fei, Mengyuan Hu, Zhiyue Su, Lin Hu, Lingchuan Guo,\\nKai Yang. 2024. “Intratumoral Microbiota as a Target for\\nAdvanced Cancer Therapeutics. ” Advanced Materials 36:\\ne2405331. https://doi.org/10.1002/adma.202405331\\n353. Lu, Ying ‐Qi, Han Qiao, Xi‐Rong Tan, Na Liu. 2024.“Broad-\\nening Oncological Boundaries: The Intratumoral Micro-\\nbiota.” Trends in Microbiology32: 807–22. https://doi.org/10.\\n1016/j.tim.2024.01.007\\n354. Ferrari, Valentina, Maria Rescigno. 2023.“The Intratumoral\\nMicrobiota: Friend or Foe? ” Trends in Cancer 9: 472–9.\\nhttps://doi.org/10.1016/j.trecan.2023.03.005\\n355. Zhang, Baochun, Il‐Kyu Choi. 2022.“Facts and Hopes in the\\nRelationship of EBV With Cancer Immunity and Immuno-\\ntherapy.” Clinical Cancer Research: An Official Journal of the\\nAmerican Association for Cancer Research28: 4363–9. https://\\ndoi.org/10.1158/1078‐0432.CCR‐21‐3408\\n356. Yang, Jing, Zhifeng Liu, Bin Zeng, Hu, Guangsheng\\nGan, Runliang. 2020.“Epstein\\n–Barr Virus‐Associated Gastric\\nCancer: A Distinct Subtype. ” Cancer Letters 495: 191–9.\\nhttps://doi.org/10.1016/j.canlet.2020.09.019\\n357. Berti, Fernanda Costa Brandão, Ana Paula Lombardi Pereira,\\nGuilherme Cesar Martelossi Cebinelli, Kleber Paiva Trugilo,\\nKaren Brajão de Oliveira. 2017.“The Role of Interleukin 10\\nin Human Papilloma Virus Infection and Progression to\\nCervical Carcinoma.” Cytokine & Growth Factor Reviews34:\\n1–13. https://doi.org/10.1016/j.cytogfr.2017.03.002\\n82 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 82, 'page_label': '83'}, page_content=\"358. Zur Hausen, Harald. 2002. “Papillomaviruses and Cancer:\\nFrom Basic Studies to Clinical Application.” Nature Reviews\\nCancer 2: 342–50. https://doi.org/10.1038/nrc798\\n359. Zheng, Zhi‐Ming. 2010.“Viral Oncogenes, Noncoding RNAs,\\nand RNA Splicing in Human Tumor Viruses.” International\\nJournal of Biological Sciences6: 730–755. https://doi.org/10.\\n7150/ijbs.6.730\\n360. Tu, Thomas, Magdalena Budzinska, Nicholas Shackel,\\nStephan Urban. 2017. “HBV DNA Integration: Molecular\\nMechanisms and Clinical Implications. ” Viruses 9: 75.\\nhttps://doi.org/10.3390/v9040075\\n361. Tan, Benjy J. Y., Kenji Sugata, Omnia Reda, Misaki Matsuo,\\nKyosuke Uchiyama, Paola Miyazato, Vincent Hahaut, et al.\\n2021. “HTLV‐1 Infection Promotes Excessive T Cell Activa-\\ntion and Transformation Into Adult T Cell Leukemia/Lym-\\nphoma.” The Journal of Clinical Investigation131: e150472.\\nhttps://doi.org/10.1172/JCI150472\\n362. He, Yunlong, Nagesh Pasupala, Huijun Zhi, Batsuhk Dorjbal,\\nImran Hussain, Hsiu‐Ming Shih, Sharmistha Bhattacharyya,\\net al. 2021.“NF‐κB‐Induced R‐Loop Accumulation and DNA\\nDamage Select for Nucleotide Excision Repair Deficiencies in\\nAdult T Cell Leukemia. ” Proceedings of the National\\nAcademy of Sciences of the United States of America 118:\\ne2005568118. https://doi.org/10.1073/pnas.2005568118\\n363. Ameya, Gemechu, Dagim Jirata Birri. 2023.“The Molecular\\nMechanisms of Virus‐Induced Human Cancers.” Microbial\\nPathogenesis 183: 106292. https://doi.org/10.1016/j.micpath.\\n2023.106292\\n364. Chen, Jiezhong. 2015. “Signaling Pathways in HPV ‐\\nAssociated Cancers and Therapeutic Implications.” Reviews\\nin Medical Virology 25 Suppl 1: 24–53. https://doi.org/10.\\n1002/rmv.1823\\n365. Stanland, Lyla J., Micah A. Luftig. 2020.“The Role of EBV‐\\nInduced Hypermethylation in Gastric Cancer Tumorigenesis.”\\nViruses 12: 1222.\\nhttps://doi.org/10.3390/v12111222\\n366. Lamontagne, R. Jason, Sumedha Bagga, Michael J.\\nBouchard. 2016. “Hepatitis B Virus Molecular Biology and\\nPathogenesis.” Hepatoma Research2: 163–86. https://doi.org/\\n10.20517/2394‐5079.2016.05\\n367. Yeh, Chau ‐Ting 2000. “Hepatitis B Virus X Protein: Search-\\ning for a Role in Hepatocarcinogenesis. ” Journal of\\nGastroenterology and Hepatology15: 339–41. https://doi.org/\\n10.1046/j.1440‐1746.2000.02166.x\\n368. Levrero, Massimo, Jessica Zucman ‐Rossi. 2016. “Mecha-\\nnisms of HBV‐Induced Hepatocellular Carcinoma.” Journal\\nof Hepatology 64: S84–S101. https://doi.org/10.1016/j.jhep.\\n2016.02.021\\n369. Wu, Qiong, Lu Zhang, Xiazhen Xu, Yi Zhang, Jiajian Shi,\\nXu Lin, Wannan Chen. 2022.“Hepatitis B Virus X Protein Is\\nStabilized by the Deubiquitinating Enzyme VCPIP1 in a\\nUbiquitin‐Independent Manner by Recruiting the 26S Pro-\\nteasome Subunit PSMC3.” Journal of Virology96: e0061122.\\nhttps://doi.org/10.1128/jvi.00611‐22\\n370. Kim, Chang ‐Min, Kazuhiko Koike, Izumu Saito,\\nTatsuo Miyamura, Gilbert Jay. 1991.“HBx Gene of Hepatitis\\nB Virus Induces Liver Cancer in Transgenic Mice.” Nature\\n351: 317–20. https://doi.org/10.1038/351317a0\\n371. Pol, Stanislas, Anaïs Vallet‐Pichard, Olivier Hermine. 2018.\\n“Extrahepatic Cancers and Chronic HCV Infection.” Nature\\nReviews Gastroenterology & Hepatology 15: 283–90. https://\\ndoi.org/10.1038/nrgastro.2017.172\\n372. Liang, T. Jake Heller, Theo. 2004.“Pathogenesis of Hepatitis\\nC‐Associated Hepatocellular Carcinoma. ” Gastroenterology\\n127: S62–71. https://doi.org/10.1053/j.gastro.2004.09.017\\n373. Cinatl, Jindrich, Martin Scholz, Hans Wilhelm Doerr. 2005.\\n“\\nRole of Tumor Cell Immune Escape Mechanisms in\\nCytomegalovirus‐Mediated Oncomodulation. ” Medicinal\\nResearch Reviews 25: 167–85. https://doi.org/10.1002/med.\\n20018\\n374. Doorbar, John, Nagayasu Egawa, Heather Griffin,\\nChristian Kranjec, Isao Murakami. 2015. “Human Papillo-\\nmavirus Molecular Biology and Disease Association. ”\\nReviews in Medical Virology25 Suppl 1: 2–23. https://doi.org/\\n10.1002/rmv.1822\\n375. Tian, Tian, Cai Aijie, Huang Bingxue, Dai Jianghong. 2016.\\n“Research Progress of Human Papilloma Virus Immune\\nEvasion Mechanisms.” Chinese Journal of Disease Control\\nand Prevention 20: 1172–5. https://doi.org/10.16462/j.cnki.\\nzhjbkz.2016.11.024\\n376. Židovec Lepej, Snje žana, Maja Matuli ć,P a u l aG ršković,\\nMirjana Pavlica, Leona Radmanić, Petra Korać. 2020.“miRNAs:\\nE B VM e c h a n i s mf o rE s c a p i n gH o s t ' sI m m u n eR e s p o n s ea n d\\nSupporting Tumorigenesis.” Pathogens (Basel, Switzerland)9:\\n353. https://doi.org/10.3390/pathogens9050353\\n377. Yang, Pengyuan, Geoffrey J. Markowitz, Xiao ‐Fan Wang.\\n2014. “The Hepatitis B Virus ‐Associated Tumor Micro-\\nenvironment in Hepatocellular Carcinoma.” National Science\\nReview 1: 396–412. https://doi.org/10.1093/nsr/nwu038\\n378. Narunsky ‐Haziza, Lian, Gregory D. Sepich ‐Poore, Ilana\\nLivyatan, Omer Asraf, Cameron Martino, Deborah Nejman,\\nNancy Gavert, et al. 2022. “Pan‐Cancer Analyses Reveal\\nCancer‐Type‐Specific Fungal Ecologies and Bacteriome Inter-\\nactions.” Cell 185: 3789–806.e17. https://doi.org/10.1016/j.cell.\\n2022.09.005\\n379. Rahal, Zahraa, Humam Kadara. 2023. “\\nBeyond Bacteria:\\nHow the Intratumor Mycobiome Modulates Lung Adeno-\\ncarcinoma Progression.” Cancer Cell 41: 1846–8. https://doi.\\norg/10.1016/j.ccell.2023.09.002\\n380. Li, Xin, Deepak Saxena. 2022. “The Mycobiome‐Immune\\nAxis: The Next Frontier in Pancreatic Cancer.” Cancer Cell\\n40: 120–2. https://doi.org/10.1016/j.ccell.2022.01.009\\n381. Alam, Aftab, Eric Levanduski, Parker Denz, Helena\\nSolleiro Villavicencio, Maulasri Bhatta, Lamees Alhorebi,\\nYali Zhang, et al. 2022. “Fungal Mycobiome Drives IL‐33\\nSecretion and Type 2 Immunity in Pancreatic Cancer. ”\\nCancer Cell 40: 153–67.e11. https://doi.org/10.1016/j.ccell.\\n2022.01.003\\n382. Chevalier, Mathieu F., Sara Trabanelli, Julien Racle,\\nBérengère Salomé, Valérie Cesson, Dalila Gharbi,\\nPerrine Bohner, et al. 2017. “ILC2‐Modulated T Cell ‐to‐\\nMDSC Balance Is Associated With Bladder Cancer Recur-\\nrence.” The Journal of Clinical Investigation 127: 2916–29.\\nhttps://doi.org/10.1172/JCI89717\\n383. Liu, Lanxiang, Haiyang Wang, Xueyi Chen, Yangdong\\nZhang, Hanping Zhang, Peng Xie. 2023.“Gut Microbiota and\\nIts Metabolites in Depression: From Pathogenesis to Treat-\\nment.” EBioMedicine 90: 104527. https://doi.org/10.1016/j.\\nebiom.2023.104527\\nMICROBIOME IN CANCER | 83 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 83, 'page_label': '84'}, page_content='384. Li, Xinpei, Shijie Shang, Meng Wu, Qian Song, Dawei Chen.\\n2024. “Gut Microbial Metabolites in Lung Cancer Develop-\\nment and Immunotherapy: Novel Insights Into Gut‐Lung\\nAxis.” Cancer Letters 598: 217096. https://doi.org/10.1016/j.\\ncanlet.2024.217096\\n385. Koh, Ara, Filipe De Vadder, Petia Kovatcheva ‐Datchary,\\nFredrik Bäckhed. 2016.“From Dietary Fiber to Host Physi-\\nology: Short‐Chain Fatty Acids as Key Bacterial Metabolites.”\\nCell 165: 1332–45. https://doi.org/10.1016/j.cell.2016.05.041\\n386. Jia, Dingjiacheng, Zheng Kuang, Liangjing Wang. 2024.“The\\nRole of Microbial Indole Metabolites in Tumor. ” Gut\\nMicrobes 16: 2409209. https://doi.org/10.1080/19490976.\\n2024.2409209\\n387. Descamps, Hélène C., Beatrice Herrmann, Daphne Wiredu,\\nChristoph?A. Thaiss. 2019.“The Path Toward Using Micro-\\nbial Metabolites as Therapies. ” EBioMedicine 44:\\n747–54. https://doi.org/10.1016/j.ebiom.2019.05.063\\n388. Qiu, Zilin, Zhengrui Li, Cangang Zhang, Qun Zhao,\\nZaoqu Liu, Quan Cheng, Jian Zhang, Anqi Lin, Peng Luo.\\n2025. “NK Cell Senescence in Cancer: From Molecular\\nMechanisms to Therapeutic Opportunities. ” Aging and\\nDisease. https://doi.org/10.14336/AD.2025.0053\\n389. Wang, Zeyu, Ziyu Dai, Hao Zhang, Nan Zhang, Xisong Liang,\\nLuo Peng, Jian Zhang, et al. 2023.“Comprehensive Analysis\\nof Pyroptosis ‐Related Gene Signatures for Glioblastoma\\nImmune Microenvironment and Target Therapy. ” Cell\\nProliferation 56: e13376.https://doi.org/10.1111/cpr.13376\\n390. Huang, Lihaoyun, Cangang Zhang, Aimin Jiang, Anqi Lin,\\nLingxuan Zhu, Weiming Mou, Dongqiang Zeng, et al. 2025.\\n“T‐Cell Senescence in the Tumor Microenvironment. ”\\nCancer Immunology Research13: 618–32. https://doi.org/10.\\n1158/2326‐6066.CIR‐24‐0894\\n391. Lin, Anqi, Ting Wei, Hui Meng, Peng Luo, Jian Zhang. 2019.\\n“Role of the Dynamic Tumor Microenvironment in Contro-\\nversies Regarding Immune Checkpoint Inhibitors for the\\nTreatment of Non‐Small Cell Lung Cancer (NSCLC) With\\nEGFR Mutations.”\\nMolecular Cancer18: 139.https://doi.org/\\n10.1186/s12943‐019‐1062‐7\\n392. Ma, Jiayao, Lingjuan Huang, Die Hu, Shan Zeng, Ying Han,\\nHong Shen. 2021.“The Role of the Tumor Microbe Micro-\\nenvironment in the Tumor Immune Microenvironment:\\nBystander, Activator, or Inhibitor?” Journal of Experimental\\n& Clinical Cancer Research: CR40: 327. https://doi.org/10.\\n1186/s13046‐021‐02128‐w\\n393. Li, Yu, Cangang Zhang, Aimin Jiang, Anqi Lin, Zaoqu Liu,\\nXiangshu Cheng, Wanting Wang, et al. 2024.“Potential Anti‐\\nTumor Effects of Regulatory T Cells in the Tumor Micro-\\nenvironment: A Review.” Journal of Translational Medicine\\n22: 293.https://doi.org/10.1186/s12967‐024‐05104‐y\\n394. Bender, Mackenzie J., Alex C. McPherson, Catherine M. Phelps,\\nSurya P. Pandey, Colin R. Laughlin, Jake H. Shapira,\\nLuzmariel Medina Sanchez, et al. 2023.“Dietary Tryptophan\\nMetabolite Released by Intratumoral Lactobacillus reuteri\\nFacilitates Immune Checkpoint Inhibitor Treatment.” Cell 186:\\n1846–62.e26. https://doi.org/10.1016/j.cell.2023.03.011\\n395. Wang, Yan, Mingshuai Bai, Qifan Peng, Leping Li,\\nFeng Tian, Ying Guo, Changqing Jing. 2024.“Angiogenesis, a\\nKey Point in the Association of Gut Microbiota and Its\\nMetabolites With Disease. ” European Journal of Medical\\nResearch 29: 614.https://doi.org/10.1186/s40001‐024‐02224‐5\\n396. Liu, Jiankun, Yuzhong Duan, Xiaoming Cheng, Xi Chen,\\nWei Xie, Haixia Long, Zhihua Lin, Bo Zhu. 2011.“IL‐17 Is\\nAssociated With Poor Prognosis and Promotes Angiogenesis\\nvia Stimulating VEGF Production of Cancer Cells in Colo-\\nrectal Carcinoma.” Biochemical and Biophysical Research\\nCommunications 407: 348–54. https://doi.org/10.1016/j.bbrc.\\n2011.03.021\\n397. Abe, Shohei, Atsuhiro Masuda, Tomonori Matsumoto,\\nJun Inoue, Hirochika Toyama, Arata Sakai, Takashi Kobayashi,\\net al. 2024. “Impact of Intratumoral Microbiome on Tumor\\nImmunity and Prognosis in Human Pancreatic Ductal Adeno-\\ncarcinoma.” Journal of Gastroenterology59: 250\\n–62. https://doi.\\norg/10.1007/s00535‐023‐02069‐5\\n398. Zhang, Shuyue, Shuishen Zhang, Xiaofan Ma, Jing Zhan,\\nChuqing Pan, Huizhong Zhang, Xiuying Xie, Jing Wen,\\nXuan Xie. 2023.“Intratumoral Microbiome Impacts Immune\\nInfiltrates in Tumor Microenvironment and Predicts Prog-\\nnosis in Esophageal Squamous Cell Carcinoma Patients.”\\nFrontiers in Cellular and Infection Microbiology13: 1165790.\\nhttps://doi.org/10.3389/fcimb.2023.1165790\\n399. Gao, Kan, Chun‐Long Mu, Aitak Farzi, Wei‐Yun Zhu. 2020.\\n“Tryptophan Metabolism: A Link Between the Gut Micro-\\nbiota and Brain.” Advances in Nutrition11: 709–23. https://\\ndoi.org/10.1093/advances/nmz127\\n400. Peters, Marloes A. M., Annemiek M. E. Walenkamp,\\nIdo P. Kema, Coby Meijer, Elisabeth G. E. De Vries,\\nSjoukje F. Oosting. 2014.“Dopamine and Serotonin Regulate\\nT u m o rB e h a v i o rb yA f f e c t i n gA n g i o g e n e s i s .” Drug Resistance\\nUpdates 17: 96–104. https://doi.org/10.1016/j.drup.2014.09.001\\n401. Yang, Qiqing, Bin Wang, Qinghui Zheng, Heyu Li,\\nXuli Meng, Fangfang Zhou, Long Zhang. 2023.“A Review of\\nGut Microbiota‐Derived Metabolites in Tumor Progression\\nand Cancer Therapy.” Advanced Science (Weinheim, Baden‐\\nWurttemberg, Germany) 10: e2207366. https://doi.org/10.\\n1002/advs.202207366\\n402. Liu, Yali, Harry Cheuk‐Hay Lau, Jun Yu. 2023.“Microbial\\nMetabolites in Colorectal Tumorigenesis and Cancer Ther-\\napy.” Gut Microbes 15: 2203968. https://doi.org/10.1080/\\n19490976.2023.2203968\\n403. Hofmanová, J., A. Vaculová, A. Lojek, A. Kozubík. 2005.\\n“Interaction of Polyunsaturated Fatty Acids and Sodium\\nButyrate During Apoptosis in HT‐29 Human Colon Adeno-\\ncarcinoma Cells.” European Journal of Nutrition44: 40–51.\\nhttps://doi.org/10.1007/s00394‐004‐0490‐2\\n404. Choi, Juliana K., Samer A. Naffouje, Masahide Goto,\\nJing Wang, Konstantin Christov, David J. Rademacher,\\nAlbert Green, et al. 2023.“Cross‐Talk Between Cancer and\\nPseudomonas aeruginosa Mediates Tumor Suppression.\\n”\\nCommunications Biology 6: 16. https://doi.org/10.1038/\\ns42003‐022‐04395‐5\\n405. Li, Ruomeng, Jing Li, Xikun Zhou. 2024.“Lung Microbiome:\\nNew Insights Into the Pathogenesis of Respiratory Diseases.”\\nSignal Transduction and Targeted Therapy9: 19.https://doi.\\norg/10.1038/s41392‐023‐01722‐y\\n406. Dickson, Robert P., John R. Erb ‐Downward, Fernando J.\\nMartinez, Gary B. Huffnagle. 2016.“T h eM i c r o b i o m ea n dt h e\\n84 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 84, 'page_label': '85'}, page_content=\"Respiratory Tract.” Annual Review of Physiology78: 481–504.\\nhttps://doi.org/10.1146/annurev‐physiol‐021115‐105238\\n407. Łaniewski, Paweł, Zehra Esra Ilhan, Melissa M. Herbst ‐\\nKralovetz. 2020.“The Microbiome and Gynaecological Cancer\\nDevelopment, Prevention and Therapy.” Nature Reviews Urology\\n17: 232–50. https://doi.org/10.1038/s41585‐020‐0286‐z\\n408. Markowski, Mark C., Stephen A. Boorjian, Jeremy P. Burton,\\nNoah M. Hahn, Molly A. Ingersoll, Saman Maleki Vareki,\\nSumanta K. Pal, Karen S. Sfanos. 2019.“T h eM i c r o b i o m ea n d\\nGenitourinary Cancer: A Collaborative Review.” European\\nUrology 75: 637–46. https://doi.org/10.1016/j.eururo.2018.12.043\\n409. Scharschmidt, Tiffany C., Julia A. Segre. 2025.“Skin Micro-\\nbiome and Dermatologic Disorders.” The Journal of Clinical\\nInvestigation 135: e184315.https://doi.org/10.1172/JCI184315\\n410. Boxberger, Manon, Valérie Cenizo, Nadim Cassir,\\nBernard La Scola. 2021.“Challenges in Exploring and Ma-\\nnipulating the Human Skin Microbiome.” Microbiome 9: 125.\\nhttps://doi.org/10.1186/s40168‐021‐01062‐5\\n411. Sedghi, Lea, Vincent DiMassa, Anthony Harrington,\\nSusan V. Lynch, Yvonne L. Kapila. 2021.“The Oral Micro-\\nbiome: Role of Key Organisms and Complex Networks in\\nOral Health and Disease.” Periodontology 2000 87: 107–31.\\nhttps://doi.org/10.1111/prd.12393\\n412. Lim, Yenkai, Makrina Totsika, Mark Morrison,\\nChamindie Punyadeera. 2017. “Oral Microbiome: A New\\nBiomarker Reservoir for Oral and Oropharyngeal Cancers.”\\nTheranostics 7: 4313–21. https://doi.org/10.7150/thno.21804\\n413. Yang, Zhixin, Shiyu Zhang, Ning Ji, Jing Li, Qianming Chen.\\n2024. “The Evil Companion of OSCC: Candida albicans.”\\nOral Diseases 30: 1873–86. https://doi.org/10.1111/odi.14700\\n414. Diwan, Prerna, Mohit Nirwan, Mayank Bahuguna,\\nShashi Prabha Kumari, James Wahlang, Rakesh Kumar Gupta.\\n2023. “Evaluating Alterations of the Oral Microbiome and Its\\nLink to Oral Cancer among Betel Quid Chewers: Prospecting\\nReversal Through Probiotic Intervention.” Pathogens (Basel,\\nSwitzerland)12: 996.https://doi.org/10.3390/pathogens12080996\\n415. Karpi ński, Tomasz M. 2019. “Role of Oral Microbiota in\\nCancer Development.” Microorganisms 7: 20.https://doi.org/\\n10.3390/microorganisms7010020\\n416. Saikia, Partha Jyoti, Lekhika Pathak, Shirsajit Mitra,\\nBikul Das 2023.“The Emerging Role of Oral Microbiota in\\nOral Cancer Initiation, Progression and Stemness.” Frontiers\\nin Immunology 14: 1198269. https://doi.org/10.3389/fimmu.\\n2023.1198269\\n417. Ramirez ‐Garcia, Andoni, Aitor Rementeria, Jose Manuel\\nAguirre‐Urizar, Maria Dolores Moragues, Aitziber Antoran,\\nAize Pellon, Ana Abad ‐Diaz‐de‐Cerio, Fernando Luis\\nHernando. 2016. “Candida albicans and Cancer: Can This\\nYeast Induce Cancer Development or Progression?” Critical\\nReviews in Microbiology 42: 181–93. https://doi.org/10.3109/\\n1040841X.2014.913004\\n418. Arzmi, Mohd Hafiz, Stuart Dashper, Michael McCullough.\\n2019. “Polymicrobial Interactions ofCandida albicansand Its\\nRole in Oral Carcinogenesis.” Journal of Oral Pathology &\\nMedicine: Official Publication of the International Association\\nof Oral Pathologists and the American Academy of Oral\\nPathology 48: 546–51. https://doi.org/10.1111/jop.12905\\n419. Madhusudhan, Nandhitha, Manuela R. Pausan, Bettina\\nHalwachs, Marija Durdević, Markus Windisch, Jan Kehrmann,\\nVijayKumar Patra, et al. 2020.“Molecular Profiling of Kerati-\\nnocyte Skin Tumors Links Staphylococcus aureus Over-\\nabundance and Increased Humanβ‐Defensin‐2 Expression to\\nGrowth Promotion of Squamous Cell Carcinoma.” Cancers 12:\\n541. https://doi.org/10.3390/cancers12030541\\n420. Nakatsuji, Teruaki, Tiffany H. Chen, Anna M. Butcher,\\nLynnie L. Trzoss, Sang ‐Jip Nam, Karina T. Shirakawa,\\nWei Zhou, et al. 2018.“A Commensal Strain ofStaphylococcus\\nepidermidisProtects Against Skin Neoplasia.” Science Advances\\n4: eaao4502.https://doi.org/10.1126/sciadv.aao4502\\n421. Bhardwaj, Sanjeev K., Harpreet Singh, Akash Deep,\\nMadhu Khatri, Jayeeta Bhaumik, Ki ‐Hyun Kim,\\nNeha Bhardwaj. 2021.“UVC‐Based Photoinactivation as an\\nEfficient Tool to Control the Transmission of Cor-\\nonaviruses.” Science of The Total Environment792: 148548.\\nhttps://doi.org/10.1016/j.scitotenv.2021.148548\\n422. Patel, Tejas, L. Katie Morrison, Peter Rady, Stephen Tyring.\\n2010. “Epidermodysplasia Verruciformis and Susceptibility\\nto HPV.” Disease Markers 29: 199–206. https://doi.org/10.\\n1155/2010/345436\\n423. Hussein, Ahmed A., Tariq A. Bhat, Zhe Jing, Eduardo\\nCortes Gomez, Mahmood Abdul Wasay, Prashant K. Singh,\\nSong Liu, Gary Smith, Khurshid A. Guru. 2023.“Does the\\nUrinary Microbiome Profile Change After Treatment of Bladder\\nCancer?” World Journal of Urology41: 3593–8. https://doi.org/\\n10.1007/s00345‐023‐04627‐1\\n424. Prakash, Pranav, Shiv Verma, Sanjay Gupta. 2024.“Influence\\nof Microbiome in Intraprostatic Inflammation and Prostate\\nCancer.” The Prostate 84: 1179–88. https://doi.org/10.1002/\\npros.24756\\n425. Li, Xiaoge, Jin Wu, Yutong Wu, Zhaoning Duan, Ming Luo,\\nLing Li, Sijing Li, Ying Jia. 2023. “Imbalance of Vaginal\\nMicrobiota and Immunity: Two Main Accomplices of Cer-\\nvical Cancer in Chinese Women.” International Journal of\\nWomen's Health 15: 987 –1002. https://doi.org/10.2147/\\nIJWH.S406596\\n426. Kombe, Arnaud John Kombe Zoa ‐Assoumou, Samira\\nBounda, Guy ‐Armel Nsole ‐Biteghe, Fleury ‐Augustin\\nJin, Tengchuan Zouré, Abdou Azaque. 2023.“Advances in\\nEtiopathological Role and Control of HPV in Cervical Cancer\\nOncogenesis.”\\nFrontiers in Bioscience ‐Landmark 28: 245.\\nhttps://doi.org/10.31083/j.fbl2810245\\n427. Zhang, Yong, Xiangxiang Chen, Yuan Wang, Ling Li,\\nQing Ju, Yan Zhang, Hangtian Xi, et al. 2023.“Alterations of\\nLower Respiratory Tract Microbiome and Short‐Chain Fatty\\nAcids in Different Segments in Lung Cancer: A Multiomics\\nAnalysis.” Frontiers in Cellular and Infection Microbiology13:\\n1261284. https://doi.org/10.3389/fcimb.2023.1261284\\n428. Bello, Salvador, José J. Vengoechea, Manuel Ponce‐Alonso,\\nAna L. Figueredo, Elisa Mincholé, Antonio Rezusta,\\nPaula Gambó, et al. 2021.“Core Microbiota in Central Lung\\nCancer With Streptococcal Enrichment as a Possible Diag-\\nnostic Marker.” Archivos De Bronconeumologia57: 681–689.\\nhttps://doi.org/10.1016/j.arbres.2020.05.034\\n429. Tsay, Jun ‐Chieh J., Benjamin G. Wu, Imran Sulaiman,\\nKatherine Gershner, Rosemary Schluger, Yonghua Li, Ting‐\\nAn Yie, et al. 2021.“Lower Airway Dysbiosis Affects Lung\\nCancer Progression.” Cancer Discovery 11: 293–307. https://\\ndoi.org/10.1158/2159‐8290.CD‐20‐0263\\nMICROBIOME IN CANCER | 85 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 85, 'page_label': '86'}, page_content=\"430. Apopa, Patrick L., Lisa Alley, Rosalind B. Penney,\\nKonstantinos Arnaoutakis, Mathew A. Steliga, Susan Jeffus,\\nEmine Bircan, et al. 2018.“PARP1 Is Up‐Regulated in Non‐\\nSmall Cell Lung Cancer Tissues in the Presence of the Cya-\\nnobacterial Toxin Microcystin.” Frontiers in Microbiology 9:\\n1757. https://doi.org/10.3389/fmicb.2018.01757\\n431. Lan, Zhou, Wei‐Jia Liu, Hao Cui, Ke‐Long Zou, Hao Chen,\\nYu‐Yue Zhao, Guang‐Tao Yu. 2023. “The Role of Oral Mi-\\ncrobiota in Cancer.” Frontiers in Microbiology 14: 1253025.\\nhttps://doi.org/10.3389/fmicb.2023.1253025\\n432. Drokow, Emmanuel Kwateng, Clement Yaw Effah, Clement\\nAgboyibor, Jemima Twumwaah Budu, Francisca Arboh,\\nPriscilla Akyaa Kyei ‐Baffour, Yao Xiao, Fan Zhang,\\nIrene Xy Wu. 2023. “Microbial Infections as Potential Risk\\nFactors for Lung Cancer: Investigating the Role of Human Pa-\\npillomavirus andChlamydia pneumoniae.” AIMS Public Health\\n10: 627–46. https://doi.org/10.3934/publichealth.2023044\\n433. Aitmanait ė, Lina, KarolisŠirmonaitis, Giancarlo Russo. 2023.\\n“Microbiomes, Their Function, and Cancer: How Metatran-\\nscriptomics Can Close the Knowledge Gap.” International\\nJournal of Molecular Sciences 24: 13786. https://doi.org/10.\\n3390/ijms241813786\\n434. Actis, Silvia, Massimiliano C azzaniga, Valentina Elisabetta\\nBounous, Marta D'Alonzo, Roberta Rosso, Francesca\\nAccomasso, Carola Minella, Nicoletta Biglia. 2023.“Emerging\\nEvidence on the Role of Breast Microbiota on the Development\\nof Breast Cancer in High‐Risk Patients.” Carcinogenesis 44:\\n718–25. https://doi.org/10.1093/carcin/bgad071\\n435. Xuan, Caiyun, Jaime M. Shamonki, Alice Chung,\\nMaggie L. Dinome, Maureen Chung, Peter A. Sieling,\\nDelphine J. Lee. 2014. “Microbial Dysbiosis Is Associated\\nWith Human Breast Cancer.” PloS One9: e83744.https://doi.\\norg/10.1371/journal.pone.0083744\\n436. Colbert, Lauren E., Molly B. El Alam, Rui Wang,\\nTatiana Karpinets, David Lo, Erica J. Lynn, Timothy A. Harris,\\net al. 2023.“Tumor‐Resident Lactobacillus inersConfer Che-\\nmoradiation Resistance Through Lactate‐Induced Metabolic\\nRewiring.”\\nCancer Cell 41: 1945–62.e11. https://doi.org/10.\\n1016/j.ccell.2023.09.012\\n437. Jian, Xingxing, Yinghong Zhu, Jian Ouyang, Yihui Wang,\\nQian Lei, Jiliang Xia, Yongjun Guan, et al. 2020.“Alterations\\nof Gut Microbiome Accelerate Multiple Myeloma Progres-\\nsion by Increasing the Relative Abundances of Nitrogen‐\\nRecycling Bacteria.” Microbiome 8: 74. https://doi.org/10.\\n1186/s40168‐020‐00854‐5\\n438. Zheng, Xiao ‐Hui, Xi‐Zhao Li, Cao‐Li Tang, Yu‐Meng Zhang,\\nTing Zhou, Xiao‐Jing Yang, Ying Liao, et al. 2024.“Detection\\nof Epstein‒Barr Virus DNA Methylation as Tumor Markers\\nof Nasopharyngeal Carcinoma Patients in Saliva, Oro-\\npharyngeal Swab, Oral Swab, and Mouthwash.” MedComm\\n5: e673.https://doi.org/10.1002/mco2.673\\n439. Muñoz‐Grez, Camila Paz, Mabel Angélica Vidal, Tamara\\nBeatriz Rojas, Luciano Esteban Ferrada, Felipe Andrés Zuñiga,\\nAgustin Andrés Vera, Sergio Andrés Sanhueza, et al. 2025.“Host‐\\nMicrobe Computational Proteomic Landscape in Oral Cancer\\nRevealed Key Functional and Metabolic Pathways BetweenFu-\\nsobacterium nucleatumand Cancer Progression.” International\\nJournal of Oral Science17, 1: 1.https://doi.org/10.1038/s41368‐\\n024‐00326‐8\\n440. Wang, Xiu ‐Li, Hua‐Wen Xu, Ning‐Ning Liu. 2023. “Oral\\nMicrobiota: A New Insight into Cancer Progression, Diag-\\nnosis and Treatment.” Phenomics 3: 535–47. https://doi.org/\\n10.1007/s43657‐023‐00124‐y\\n441. Anjali, Kanadan, Muhammed Manzoor, Mangesh\\nVasant Suryavanshi, Parthiban Rudrapathy, Punchappady\\nDevasya Rekha, Ranajit Das, Asif Hameed,\\nAnanthapadmanabha Bhagwath Arun. 2023.“Dysbiosis of the\\nOral Microbiota Composition Is Associated With Oral Squa-\\nmous Cell Carcinoma and the Impact of Radiotherapy: A Pilot\\nStudy.” FEMS Microbiology Letters370: fnad111.https://doi.org/\\n10.1093/femsle/fnad111\\n442. Mauceri, Rodolfo, Martina Coppini, Davide Vacca, Giorgio\\nBertolazzi, Valeria Cancila, Claudio Tripodo, Giuseppina\\nCampisi. 2023.“No Clear Clustering Dysbiosis From Salivary\\nMicrobiota Analysis by Long Sequencing Reads in Patients\\nAffected by Oral Squamous CellCarcinoma: A Single Center\\nStudy. ” Cancers 15: 4211. https://doi.org/10.3390/\\ncancers15174211\\n443. Han, Zewen, Yichen Hu, Xin Lin, Hongyu Cheng,\\nBiao Dong, Xuan Liu, Buling Wu, Zhenjiang Zech Xu. 2025.\\n“Systematic Analyses Uncover Robust Salivary Microbial\\nSignatures and Host –Microbiome Perturbations in Oral\\nSquamous Cell Carcinoma.” mSystems 10: e0124724.https://\\ndoi.org/10.1128/msystems.01247‐24\\n444. Wang, Zizheng, Yilong Chen, Haoning Li, Yuan Yue,\\nHaopeng Yu. 2025. “Exploring Oral Microbiome in Oral\\nSquamous Cell Carcinoma Across Environment‐Associated\\nSample Types.” Microbiology Spectrum13: e0085224.https://\\ndoi.org/10.1128/spectrum.00852‐24\\n445. Li, Shuluan, Tianyu Wang, Ya Ren, Zhou Liu, Jidong Gao,\\nZhi Guo. 2024. “Prognostic Impact of Oral Microbiome on\\nSurvival of Malignancies: A Systematic Review and Meta‐\\nAnalysis.” Systematic Reviews13: 41.https://doi.org/10.1186/\\ns13643‐023‐02419‐7\\n446. Monson, Kelsey R., Brandilyn A. Peters, Mykhaylo Usyk,\\nCaroline Y. Um, Paul E. Oberstein, Marjorie L. McCullough,\\nMark P. Purdue, et al. 2022.“Elevated Dietary Carbohydrate\\nand Glycemic Intake Associate With an Altered Oral\\nMicrobial Ecosystem in Two Large U.S. Cohorts.” Cancer\\nResearch Communications 2: 1558 –68. https://doi.org/10.\\n1158/2767‐9764.crc‐22‐0323\\n447. Menon, Anil, Vimi S. Mutalik, Yongqiang Chen, Spd Ponamgi,\\nSujatha Peela, Robert J. Schroth, Saeid Ghavami, Prashen\\nChelikani. 2025.“Beyond Genetics: Exploring Lifestyle, Micro-\\nbiome, and Social Determinants in Oral Cancer Development.”\\nCancers 17: 1094.https://doi.org/10.3390/cancers17071094\\n448. Wang, Xue, Qili Mi, Ji Yang, Ying Guan, Wanli Zeng,\\nHaiying Xiang, Xin Liu, et al. 2022. “Effect of Electronic\\nCigarette and Tobacco Smoking on the Human Saliva\\nMicrobial Community.”\\nBrazilian Journal of Microbiology53:\\n991–1000. https://doi.org/10.1007/s42770‐022‐00721‐5\\n449. Petersen, Poul Erik. 2009. “Oral Cancer Prevention and\\nControl—The Approach of the World Health Organization.”\\nOral Oncology 45: 454 –60. https://doi.org/10.1016/j.\\noraloncology.2008.05.023\\n450. Al ‐Hebshi, Nezar N., Akram T. Nasher, David J. Speicher,\\nMushfiq H. Shaikh, Newell W. Johnson. 2016. “Possible\\nInteraction Between Tobacco Use and EBV in Oral\\n86 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 86, 'page_label': '87'}, page_content='Squamous Cell Carcinoma.” Oral Oncology59: e4–5. https://\\ndoi.org/10.1016/j.oraloncology.2016.06.005\\n451. Zhang, Yaobang, Pimporn Thongmuang, Sarisak Soontornchai.\\n2025. “Analysis on Risk Factors of Oral Cancer in Chengdu\\nCommunity.” Proceeding National & International Conference\\n18: 204. http://www.journalgrad .ssru.ac.th/index.php/\\n8thconference/article/view/5078\\n452. Liu, Qiao ‐Yun, Ying Liao, Yan‐Xia Wu, Hua Diao, Yan Du,\\nYi‐Wei Chen, Jin‐Ru Xie, et al. 2023.“The Oral Microbiome\\nas Mediator Between Oral Hygiene and Its Impact on\\nNasopharyngeal Carcinoma. ” Microorganisms 11: 719.\\nhttps://doi.org/10.3390/microorganisms11030719\\n453. Conway, Erica, Haisheng Wu, Linwei Tian. 2023.“Overview\\nof Risk Factors for Esophageal Squamous Cell Carcinoma in\\nChina. ” Cancers 15: 5604. https://doi.org/10.3390/\\ncancers15235604\\n454. La Rosa, Giusy Rita Maria, Giuseppe Gattuso, Eugenio Pedullà,\\nErnesto Rapisarda, Daria Nicolosi, Mario Salmeri. 2020.\\n“Association of Oral Dysbiosis With Oral Cancer Develop-\\nment.” Oncology Letters19: 3045–58. https://doi.org/10.3892/ol.\\n2020.11441\\n455. Salem, Abdelhakim, Rabeia Almahmoudi, Jaana Hagström,\\nHolger Stark, Dan Nordström, Tuula Salo, Kari K. Eklund.\\n2019. “Human β‐Defensin 2 Expression in Oral Epithelium:\\nPotential Therapeutic Targets in Oral Lichen Planus. ”\\nInternational Journal of Molecular Sciences20: 1780.https://\\ndoi.org/10.3390/ijms20071780\\n456. Triarico, Silvia, Pierpaolo Agresti, Emanuele Rinninella,\\nMaria Cristina Mele, Alberto Romano, Giorgio Attinà,\\nPalma Maurizi, Stefano Mastrangelo, Antonio Ruggiero.\\n2022. “Oral Microbiota During Childhood and Its Role in\\nChemotherapy‐Induced Oral Mucositis in Children With\\nCancer.” Pathogens (Basel, Switzerland)11: 448. https://doi.\\norg/10.3390/pathogens11040448\\n457. Akbari, Elahe, Alireza Milani, Masoud Seyedinkhorasani,\\nAzam Bolhassani. 2023. “HPV Co‐Infections With Other\\nPathogens in Cancer Development: A Comprehensive\\nReview.” Journal of Medical Virology95: e29236.https://doi.\\norg/10.1002/jmv.29236\\n458. Wallace, Nicholas A., Sujita Khanal, Kristin L. Robinson,\\nSebastian O. Wendel, Joshua J. Messer, Denise A. Galloway.\\n2017. “High\\n‐Risk Alphapapillomavirus Oncogenes Impair\\nthe Homologous Recombination Pathway. ” Journal of\\nVirology 91: e01084‐17. https://doi.org/10.1128/JVI.01084‐17\\n459. Zhou, Chaozheng, Anqi Lin, Manming Cao, Weimin Ding,\\nWeiming Mou, Ningyi Guo, Zhenyu Chen, Jian Zhang,\\nPeng Luo. 2021. “Activation of the DDR Pathway Leads to\\nthe Down‐Regulation of the TGFβ Pathway and a Better\\nResponse to ICIs in Patients With Metastatic Urothelial\\nCarcinoma.” Frontiers in Immunology12: 634741.https://doi.\\norg/10.3389/fimmu.2021.634741\\n460. Shi, Congqi, Kaiyu Qin, Anqi Lin, Aimin Jiang, Quan Cheng,\\nZaoqu Liu, Jian Zhang, Peng Luo. 2022.“The Role of DNA\\nDamage Repair (DDR) System in Response to Immune\\nCheckpoint Inhibitor (ICI) Therapy. ” Journal of\\nExperimental & Clinical Cancer Research: CR 41: 268.\\nhttps://doi.org/10.1186/s13046‐022‐02469‐0\\n461. Raudenská, Martina, Maria Bugajová, David Kalfe řt,\\nJan Plzák, Šubrt, Adam Tesařová, Petra Masařík, Michal.\\n2024. “The Interplay Between Microbiome and Host Factors\\nin Pathogenesis and Therapy of Head and Neck Cancer.”\\nBiochimica et Biophysica Acta (BBA) —Reviews on Cancer\\n1879: 189216. https://doi.org/10.1016/j.bbcan.2024.189216\\n462. Stasiewicz, Mark, Tomasz M. Karpi ński. 2022. “The Oral\\nMicrobiota and Its Role in Carcinogenesis. ” Seminars in\\nCancer Biology 86: 633 –42. https://doi.org/10.1016/j.\\nsemcancer.2021.11.002\\n463. Dai, Zhujiang, Jingqiu Zhang, Qi Wu, Juan Chen, Jun Liu,\\nLu Wang, Chaowu Chen, et al. 2019.“The ROLE of Microbiota\\nin the development of colorectal cancer.” International Journal\\nof Cancer145: 2032–41. https://doi.org/10.1002/ijc.32017\\n464. Karati, Dipanjan, Dileep Kumar. 2024.“Molecular Insight Into\\nthe Apoptotic Mechanism of Cancer Cells: An Explicative\\nReview.” Current Molecular Pharmacology17: e18761429273223.\\nhttps://doi.org/10.2174/0118761429273223231124072223\\n465. Kumai, Takumi, Hirotaka Shinomiya, Hirofumi Shibata,\\nHideaki Takahashi, Toshihiro Kishikawa, Ryuhei Okada,\\nShigeharu Fujieda, Masafumi Sakashita. 2024.“\\nTranslational\\nResearch in Head and Neck Cancer: Molecular and Immu-\\nnological Updates. ” Auris, Nasus, Larynx 51: 391 –400.\\nhttps://doi.org/10.1016/j.anl.2023.08.006\\n466. Zhou, Jiaying, Zixuan Hu, Lei Wang, Qinchao Hu,\\nZixu Chen, Tao Lin, Rui Zhou, et al. 2024.“Tumor‐Colonized\\nStreptococcus mutans Metabolically Reprograms Tumor\\nMicroenvironment and Promotes Oral Squamous Cell Car-\\ncinoma. ” Microbiome 12: 193. https://doi.org/10.1186/\\ns40168‐024‐01907‐9\\n467. Hayes, Conall, Claire L. Donohoe, Maria Davern,\\nNoel E. Donlon. 2021.“The Oncogenic and Clinical Impli-\\ncations of Lactate Induced Immunosuppression in the Tu-\\nmour Microenvironment.” Cancer Letters500: 75–86. https://\\ndoi.org/10.1016/j.canlet.2020.12.021\\n468. Zang, Wenli, Junchao Liu, Fengxue Geng, Dongjuan Liu,\\nShuwei Zhang, Yuchao Li, Yaping Pan. 2022. “Butyrate\\nPromotes Oral Squamous Cell Carcinoma Cells Migration,\\nInvasion and Epithelial‐Mesenchymal Transition.” PeerJ 10:\\ne12991. https://doi.org/10.7717/peerj.12991\\n469. Barbour, Abdelahhad, Omnia Elebyary, Noah Fine,\\nMorvarid Oveisi, Michael Glogauer. 2022.“Metabolites of the\\nOral Microbiome: Important Mediators of Multikingdom\\nInteractions.” FEMS Microbiology Reviews 46: fuab039.\\nhttps://doi.org/10.1093/femsre/fuab039\\n470. Chattopadhyay, Indranil, Mukesh Verma, Madhusmita Panda.\\n2019. “Role of Oral Microbiome Signatures in Diagnosis and\\nPrognosis of Oral Cancer.” Technology in Cancer Research &\\nTreatment 18: 1533033819867354. https://doi.org/10.1177/\\n1533033819867354\\n471. Chen, Yiyin Erin Tsao, Hensin. 2013.“The Skin Microbiome:\\nCurrent Perspectives and Future Challenges.” Journal of the\\nAmerican Academy of Dermatology69: 143–55.e3. https://doi.\\norg/10.1016/j.jaad.2013.01.016\\n472. Skin Cancer and Ultraviolet‐Radiation Exposure in Different\\nJobs. 2023. The British Journal of Dermatology 188: e22.\\nhttps://doi.org/10.1093/bjd/ljad020\\n473. Wittlich, Marc, Stephan Westerhausen, Benjamin Strehl,\\nHelmut Versteeg, Wiho Stöppelmann. 2023.“The GENESIS‐\\nUV Study on Ultraviolet Radiation Exposure Levels in 250\\nOccupations to Foster Epidemiological and Legislative\\nMICROBIOME IN CANCER | 87 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 87, 'page_label': '88'}, page_content=\"Efforts to Combat Nonmelanoma Skin Cancer.” The British\\nJournal of Dermatology188: 350–60. https://doi.org/10.1093/\\nbjd/ljac093\\n474. Pega, Frank, Natalie C. Momen, Kai N. Streicher, Maria Leon\\n‐Roux, Subas Neupane, Mary K. Schubauer ‐Berigan,\\nJoachim Schüz, et al. 2023.“Global, Regional and National\\nBurdens of Non ‐Melanoma Skin Cancer Attributable to\\nOccupational Exposure to Solar Ultraviolet Radiation for 183\\nCountries, 2000 –2019: A Systematic Analysis From the\\nWHO/ILO Joint Estimates of the Work‐Related Burden of\\nDisease and Injury.” Environment International181: 108226.\\nhttps://doi.org/10.1016/j.envint.2023.108226\\n475. Kullander, Johanna, Ola Forslund, Joakim Dillner. 2009.\\n“Staphylococcus aureusand Squamous Cell Carcinoma of the\\nSkin.” Cancer Epidemiology, Biomarkers & Prevention:\\nA Publication of the American Association for Cancer Research,\\nCosponsored by the American Society of Preventive Oncology18:\\n472–8. https://doi.org/10.1158/1055‐9965.EPI‐08‐0905\\n476. Mrázek, Jakub, Chahrazed Mekadim, Petra Ku čerová,\\nRoman Švejstil, Hana Salmonová, Jitka Vlasáková,\\nRenata Tarasová, Jana Čížková, Monika Červinková. 2019.\\n“Melanoma‐Related Changes in Skin Microbiome. ” Folia\\nMicrobiologica 64: 435 –42. https://doi.org/10.1007/s12223‐\\n018‐00670‐3\\n477. Mizuhashi, Satoru, Ikko Kajihara, Soichiro Sawamura,\\nHisashi Kanemaru, Katsunari Makino, Jun Aoi,\\nTakamitsu Makino, et al. 2021.“Skin Microbiome in Acral\\nMelanoma: Corynebacterium Is Associated With Advanced\\nMelanoma.” The Journal of Dermatology48: e15–6. https://\\ndoi.org/10.1111/1346‐8138.15633\\n478. Zhu, Yuhang, Wanguo Liu, Mei Wang, Xu Wang, Sibo Wang.\\n2024.\\n“Causal Roles of Skin Microbiota in Skin Cancers\\nSuggested by Genetic Study.” Frontiers in Microbiology 15:\\n1426807. https://doi.org/10.3389/fmicb.2024.1426807\\n479. Takei, Itsuki, Soichiro Sawamura, Tselmeg Mijiddorj Myangat,\\nIkko Kajihara, Hisashi Kanemaru, Kayo Kashiwada ‐\\nNakamura, Katsunari Makino, et al. 2021.“Clinical Signifi-\\ncance of Skin Colonization of Pseudomonas aeruginosa in\\nCutaneous Squamous Cell Carcinoma. ” The Journal of\\nDermatology 48: e581–2. https://doi.org/10.1111/1346‐8138.\\n16148\\n480. Voigt, Anita Y., Akintunde Emiola, Jethro S. Johnson,\\nElizabeth S. Fleming, Hoan Nguyen, Wei Zhou,\\nKenneth Y. Tsai, Christine Fink, Julia Oh. 2022.“Skin Mi-\\ncrobiome Variation With Cancer Progression in Human\\nCutaneous Squamous Cell Carcinoma. ” Journal of\\nInvestigative Dermatology 142: 2773–82.e16. https://doi.org/\\n10.1016/j.jid.2022.03.017\\n481. Xia, Chenglai, Jiyan Su, Can Liu, Zhikai Mai,\\nShuanghong Yin, Chuansheng Yang, Liwu Fu. 2023.\\n“Human Microbiomes in Cancer Development and Ther-\\napy.” MedComm 4: e221.https://doi.org/10.1002/mco2.221\\n482. Rangwala, S., K. Y. Tsai. 2011.“Roles of the Immune System\\nin Skin Cancer.” The British Journal of Dermatology 165:\\n953–65. https://doi.org/10.1111/j.1365‐2133.2011.10507.x\\n483. Yu, Zhen ‐Wei, Min Zheng, Hua‐Yang Fan, Xin‐Hua Liang,\\nYa‐Ling Tang. 2024.“Ultraviolet (UV) Radiation: A Double‐\\nEdged Sword in Cancer Development and Therapy. ”\\nMolecular Biomedicine 5: 49. https://doi.org/10.1186/s43556‐\\n024‐00209‐8\\n484. Yamada, Miko, Tarl W. Prow. 2020.“Physical Drug Delivery\\nEnhancement for Aged Skin, UV Damaged Skin and Skin\\nCancer: Translation and Commercialization. ” Advanced\\nDrug Delivery Reviews 153: 2–17. https://doi.org/10.1016/j.\\naddr.2020.04.008\\n485. Lopes, Fabiana C. P. S., Marc G. Sleiman, Kate Sebastian,\\nRoxanne Bogucka, Elizabeth A. Jacobs, Adewole S.\\nAdamson. 2021.\\n“UV Exposure and the Risk of Cutaneous\\nMelanoma in Skin of Color: A Systematic Review.” JAMA\\nDermatology 157: 213 –9. https://doi.org/10.1001/\\njamadermatol.2020.4616\\n486. Coohill, Thomas P., Jose‐Luis Sagripanti. 2008.“Overview of\\nthe Inactivation by 254?nm Ultraviolet Radiation of Bacteria\\nWith Particular Relevance to Biodefense. ” Photochemistry\\nand Photobiology84: 1084–90. https://doi.org/10.1111/j.1751‐\\n1097.2008.00387.x\\n487. Burns, Erin M., Hana Ahmed, Prescilia N. Isedeh,\\nIndermeet Kohli, William Van Der Pol, Abdullah Shaheen,\\nAnum F. Muzaffar, et al. 2019.“Ultraviolet Radiation, Both\\nUVA and UVB, Influences the Composition of the Skin Mi-\\ncrobiome.” Experimental Dermatology28: 136–41. https://doi.\\norg/10.1111/exd.13854\\n488. Willmott, Thomas, Paul M. Campbell, Christopher\\nE. M. Griffiths, Clare O'Connor, Michael Bell, Rachel\\nE. B. Watson, Andrew J. McBain, Abigail K. Langton. 2023.\\n“Behaviour and Sun Exposure in Holidaymakers Alters Skin\\nMicrobiota Composition and Diversity.” Frontiers in Aging4:\\n1217635. https://doi.org/10.3389/fragi.2023.1217635\\n489. Azzimonti, Barbara, Chiara Ballacchino, Paola Zanetta,\\nMarie Angele Cucci, Chiara Monge, Margherita Grattarola,\\nChiara Dianzani, Giuseppina Barrera, Stefania Pizzimenti.\\n2023. “Microbiota, Oxidative Stress, and Skin Cancer: An\\nUnexpected Triangle.” Antioxidants (Basel, Switzerland) 12:\\n546. https://doi.org/10.3390/antiox12030546\\n490. Tang Hongbo, Ma Qingyu, Sang Yingbing, Mao Lidan,\\nLiang Junqin, Kang Xiaojing. 2022.“Mechanisms Underlying\\nSynergistic Induction and Promotion of Cutaneous Squamous\\nCell Carcinoma in Nude Mice by Ultraviolet Light and Human\\nP a p i l l o m a v i r u s1 6E 6 .” Chinese Journal of Dermatology55: 982‐\\n989. https://doi.org/10.35541/cjd.20220084\\n491. Harden, Mallory E., Karl Munger. 2017. “Human Papillo-\\nmavirus Molecular Biology.” Mutation Research/Reviews in\\nMutation Research772: 3–12. https://doi.org/10.1016/j.mrrev.\\n2016.07.002\\n492. Rogers, Heather D., Jennifer L. Macgregor, Kristin M. Nord,\\nStephen Tyring, Peter Rady, Danielle E. Engler,\\nMarc E. Grossman. 2009.“Acquired Epidermodysplasia Ver-\\nruciformis.” Journal of the American Academy of Dermatology\\n60: 315–20. https://doi.org/10.1016/j.jaad.2008.08.035\\n493. Dupin, Nicolas. 2020. “Update on Oncogenesis and Therapy\\nfor Kaposi Sarcoma.” Current Opinion in Oncology32: 122\\n–8.\\nhttps://doi.org/10.1097/CCO.0000000000000601\\n494. Feng, Huichen, Masahiro Shuda, Yuan Chang,\\nPatrick S. Moore. 2008. “Clonal Integration of a Poly-\\nomavirus in Human Merkel Cell Carcinoma.” Science 319:\\n1096–100. https://doi.org/10.1126/science.1152586\\n88 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 88, 'page_label': '89'}, page_content='495. Zhao, Qihang, Qiang Wang, Tengjiao Wang, Junfang Xu,\\nTingting Li, Qiuyan Liu, Qinghua Yao, Pin Wang. 2021.\\n“Pattern Recognition Receptors (PRRs) in Macrophages\\nPossess Prognosis and Immunotherapy Potential for Mela-\\nnoma.” Frontiers in Immunology12: 765615. https://doi.org/\\n10.3389/fimmu.2021.765615\\n496. Vijay, Kumar. 2018. “Toll‐Like Receptors in Immunity and\\nInflammatory Diseases: Past, Present, and Future. ”\\nInternational Immunopharmacology59: 391–412. https://doi.\\norg/10.1016/j.intimp.2018.03.002\\n497. Hoste, Esther, Esther N. Arwert, Rohit Lal, Andrew P. South,\\nJulio C. Salas ‐Alanis, Dedee F. Murrell, Giacomo\\nDonati, Fiona M. Watt. 2015.“Innate Sensing of Microbial\\nProducts Promotes Wound‐Induced Skin Cancer.” Nature\\nCommunications 6: 5932. https://doi.org/10.1038/ncomms\\n6932\\n498. Severn, Morgan M., Alexander R. Horswill. 2023. “Staphy-\\nlococcus epidermidisand Its Dual Lifestyle in Skin Health and\\nInfection.” Nature Reviews Microbiology 21: 97–111. https://\\ndoi.org/10.1038/s41579‐022‐00780‐3\\n499. Yu, Jinlei, Yang Luo, Zhenlai Zhu, Yufeng Zhou,\\nLicheng Sun, Jixin Gao, Jinlv Sun, et al. 2019.“A Tryptophan\\nMetabolite of the skin Microbiota Attenuates Inflammation\\nin Patients With Atopic Dermatitis Through the Aryl\\nHydrocarbon Receptor. ” Journal of Allergy and Clinical\\nImmunology 143: 2108–19.e12. https://doi.org/10.1016/j.jaci.\\n2018.11.036\\n500. Axelrod, Margaret L., Douglas B. Johnson, Justin M. Balko.\\n2018. “Emerging Biomarkers for Cancer Immunotherapy Iin\\nMelanoma.” Seminars in Cancer Biology52: 207–15. https://\\ndoi.org/10.1016/j.semcancer.2017.09.004\\n501. Hoeijmakers, Lotte L., Irene L. M. Reijers, Christian U. Blank.\\n2023. “Biomarker‐Driven Personalization of Neoadjuvant\\nImmunotherapy in Melanoma.” Cancer Discovery13: 2319–38.\\nhttps://doi.org/10.1158/2159‐8290.CD‐23‐\\n0352\\n502. Long, Georgina V., Alexander M. Menzies, Richard A. Scolyer.\\n2023. “Neoadjuvant Checkpoint Immunotherapy and Mela-\\nnoma: The Time Is Now.” Journal of Clinical Oncology41:\\n3236–48. https://doi.org/10.1200/JCO.22.02575\\n503. Ianiro, Gianluca. 2025. “Longitudinal Gut Microbiome Shifts\\nCorrelate With Clinical Outcomes of Immune Checkpoint\\nInhibitors‐Treated Melanoma: Time for Microbial Bio-\\nmarkers in Oncology?” Gastroenterology 168: 177–8. https://\\ndoi.org/10.1053/j.gastro.2024.07.034\\n504. Ballerini, Mattia, Serena Galiè, Punit Tyagi, Carlotta Catozzi,\\nHariam Raji, Amir Nabinejad, Angeli D. G. Macandog, et al.\\n2025, 9, 967–84. “A Gut‐on‐a‐Chip Incorporating Human\\nFaecal Samples and Peristalsis Predicts Responses to\\nImmune Checkpoint Inhibitors for Melanoma. ” Nature\\nBiomedical Engineering. https://doi.org/10.1038/s41551‐024‐\\n01318‐z\\n505. Pinato, David J., Daria Gramenitskaya, Daniel M. Altmann,\\nRosemary J. Boyton, Benjamin H. Mullish, Julian R. Marchesi,\\nMark Bower. 2019.“Antibiotic Therapy and Outcome From\\nImmune‐Checkpoint Inhibitors.” Journal for Immunotherapy\\nof Cancer7: 287.https://doi.org/10.1186/s40425‐019‐0775‐x\\n506. Chen, Y. Erin, Djenet Bousbaine, Alessandra Veinbachs,\\nKatayoon Atabakhsh, Alex Dimas, Victor K. Yu,\\nAishan Zhao, et al. 2023.“Engineered Skin Bacteria Induce\\nAntitumor T Cell Responses Against Melanoma.” Science\\n380: 203–10. https://doi.org/10.1126/science.abp9563\\n507. He, Mengying, Tao Yang, Yuhan Wang, Mengyuan Wang,\\nXingye Chen, Dawei Ding, Yiran Zheng, Huabing Chen.\\n2021. “Immune Checkpoint Inhibitor‐Based Strategies for\\nSynergistic Cancer Therapy.” Advanced Healthcare Materials\\n10: e2002104.https://doi.org/10.1002/adhm.202002104\\n508. Plaçais, Léo, Stéphane Dalle, Olivier Dereure, Sabiha Trabelsi,\\nSophie Dalac, Delphine Legoupil, Henri Montaudié, et al.\\n2022. “Risk of irAEs in Patients With Autoimmune Diseases\\nTreated by Immune Checkpoint Inhibitors for Stage III or IV\\nMelanoma: Results From a Matched Case‐\\nControl Study.”\\nAnnals of the Rheumatic Diseases81: 1445–52. https://doi.org/\\n10.1136/ard‐2022‐222186\\n509. Pan, Catherina X., Mofei Liu, Charles B. Lau,\\nWilliam C. Lau, Daniel Y. Kim, Shahin A. Saberi,\\nRachael Rowley, et al. 2024.“Histopathological Predictors of\\nImmune‐Related Adverse Events Among Patients With\\nMelanoma Treated With Immune Checkpoint Inhibitors.”\\nJournal of the American Academy of Dermatology90: 826–9.\\nhttps://doi.org/10.1016/j.jaad.2023.11.037\\n510. Otto, Grant. 2023. “Diverse Routes to Melanoma Metastasis\\nand ICI Resistance.” Nature Cancer 4: 1643. https://doi.org/\\n10.1038/s43018‐023‐00679‐9\\n511. Killock, David. 2020. “ICI for resected Stage IV Melanoma.”\\nNature Reviews Clinical Oncology17: 450. https://doi.org/10.\\n1038/s41571‐020‐0397‐8\\n512. Carlino, Matteo S., James Larkin, Georgina V. Long. 2021.\\n“Immune Checkpoint Inhibitors in Melanoma.” The Lancet\\n398: 1002 –14. https://doi.org/10.1016/S0140 ‐6736(21)\\n01206‐X\\n513. Yacouba, Abdourahamane, Maryam Tidjani Alou, Jean ‐\\nChristophe Lagier, Grégory Dubourg, Didier Raoult. 2022.\\n“Urinary Microbiota and Bladder Cancer: A Systematic\\nReview and a Focus on Uropathogens.” Seminars in Cancer\\nBiology 86: 875–84. https://doi.org/10.1016/j.semcancer.2021.\\n12.010\\n514. Wahid, Mohd, Sajad A. Dar, Arshad Jawed, Raju Kumar Mandal,\\nNaseem Akhter, Saif Khan, Farah Khan, et al. 2022.“Microbes in\\nGynecologic Cancers: Causes or Consequences and Therapeutic\\nPotential.” Seminars in Cancer Biology86: 1179–89. https://doi.\\norg/10.1016/j.semcancer.2021.07.013\\n515. Lee, Seoho, Karen Sfanos, Nirmish Singla. 2025.“The Role of the\\nUrinary Microbiome in Genitourinary Cancers.”\\nNature Reviews\\nUrology22: 544–561. https://doi.org/10.1038/s41585‐025‐01011‐z\\n516. Mansour, Bassel, Ádám Monyók, Márió Gajdács,\\nBalázs Stercz, Nóra Makra, Kinga Pénzes, István Vadnay,\\nDóra Szabó, Eszter Ostorházi. 2022.“Bladder Tissue Micro-\\nbiome Composition in Patients of Bladder Cancer or Benign\\nProstatic Hyperplasia and Related Human Beta Defensin\\nLevels.” Biomedicines 10: 1758. https://doi.org/10.3390/\\nbiomedicines10071758\\n517. Yin, Zhaofa, Bohan Liu, Shijian Feng, Yushi He, Cai Tang,\\nPengan Chen, Xinyi Wang, Kunjie Wang. 2023. “AL a r g e\\nGenetic Causal Analysis of the Gut Microbiota and Urological\\nCancers: A Bidirectional Mendelian Randomization Study.”\\nNutrients 15: 4086.https://doi.org/10.3390/nu15184086\\n518. Porto, Joao G., Maria Camila Suarez Arbelaez, Brandon Pena,\\nArchan Khandekar, Ankur Malpani, Bruno Nahar,\\nMICROBIOME IN CANCER | 89 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 89, 'page_label': '90'}, page_content=\"Sanoj Punnen, et al. 2023.“The Influence of the Microbiome\\non Urological Malignancies: A Systematic Review.” Cancers\\n15: 4984.https://doi.org/10.3390/cancers15204984\\n519. Shrestha, Eva, James R. White, Shu‐Han Yu, Ibrahim Kulac,\\nOnur Ertunc, Angelo M. De Marzo, Srinivasan\\nYegnasubramanian, et al. 2018.“Profiling the Urinary Micro-\\nbiome in Men With Positive Versus Negative Biopsies for Pros-\\ntate Cancer.” Journal of Urology199: 161–71. https://doi.org/10.\\n1016/j.juro.2017.08.001\\n520. Qi, Xiaoyu, Jing Zhou, Xinyue Wang, Yan Shen, Yuxun Cao,\\nLiangzi Jiang, Miaomiao Shen, et al. 2025. “HPV E6/E7‐\\nInduced Acetylation of a Peptide Encoded by a Long Non‐\\nCoding RNA Inhibits Ferroptosis to Promote the Malignancy\\nof Cervical Cancer.” Advanced Science (Weinheim, Baden‐\\nWurttemberg, Germany) 12: e2414018. https://doi.org/10.\\n1002/advs.202414018\\n521. Molina, Mariano A., Renske D. M. Steenbergen, Anna\\nPumpe, Angelique N. Kenyon, Willem J. G. Melchers. 2024.\\n“HPV Integration and Cervical Cancer: A Failed Evolu-\\ntionary Viral Trait. ” Trends in Molecular Medicine 30:\\n890–902. https://doi.org/10.1016/j.molmed.2024.05.009\\n522. Dias Gonçalves Lima, Fernando, Mariano A. Molina. 2024.\\n“Uncovering the HPV Types Causing Cervical Cancer. ”\\nNature Microbiology 9: 2795 –6. https://doi.org/10.1038/\\ns41564‐024‐01835‐2\\n523. Mitra, Anita, David A. MacIntyre, Julian R. Marchesi,\\nYun S. Lee, Phillip R. Bennett, Maria Kyrgiou. 2016.“The\\nVaginal Microbiota, Human Papillomavirus Infection and\\nCervical Intraepithelial Neoplasia: What Do We Know and\\nWhere Are We Going Next?” Microbiome 4: 58. https://doi.\\norg/10.1186/s40168‐016‐0203‐0\\n524. Wei, Zhen ‐Tong, Hong‐Liang Chen, Chun‐Feng Wang, Gui‐\\nLian Yang, Shu‐Mei Han, Song‐Ling Zhang. 2020.“Depiction\\nof Vaginal Microbiota in Women With High‐Risk Human\\nPapillomavirus Infection.\\n” Frontiers in Public Health 8:\\n587298. https://doi.org/10.3389/fpubh.2020.587298\\n525. Gillet, Evy, Joris Fa Meys, Hans Verstraelen, Carolyne\\nBosire, Philippe De Sutter, Marleen Temmerman,\\nDavy Vanden Broeck. 2011. “Bacterial vaginosis Is Associ-\\nated With Uterine Cervical Human Papillomavirus Infection:\\nA Meta‐Analysis.” BMC Infectious Diseases11: 10.https://doi.\\norg/10.1186/1471‐2334‐11‐10\\n526. Chen, Yulian, Xingdi Qiu, Wenjing Wang, Dong Li,\\nAnyue Wu, Zubei Hong, Wen Di, Lihua Qiu. 2020.“Human\\nPapillomavirus Infection and Cervical Intraepithelial Neo-\\nplasia Progression Are Associated With Increased Vaginal\\nMicrobiome Diversity in a Chinese Cohort.” BMC Infectious\\nDiseases 20: 629. https://doi.org/10.1186/s12879 ‐020 ‐\\n05324‐9\\n527. Gong, Xiangjin, Hao Chi, Zhijia Xia, Guanhu Yang,\\nGang Tian. 2023. “Advances in HPV ‐Associated Tumor\\nManagement: Therapeutic Strategies and Emerging In-\\nsights.” Journal of Medical Virology95: e28950. https://doi.\\norg/10.1002/jmv.28950\\n528. Usyk, Mykhaylo, Christine P. Zolnik, Philip E. Castle, Carolina\\nPorras, Rolando Herrero, Ana Gradissimo, Paula Gonzalez, et al.\\n2020. “Cervicovaginal Microbiome and Natural History of HPV\\nin a Longitudinal Study.” PLoS Pathogens16: e1008376.https://\\ndoi.org/10.1371/journal.ppat.1008376\\n529. Wu, Ke, Yaorong Li, Kangli Ma, Weiguang Zhao, Zhixian Yao,\\nZhong Zheng, Feng Sun, et al. 2024.“The Microbiota and Renal\\nCell Carcinoma.” Cellular Oncology (Dordrecht, Netherlands)47:\\n397–413. https://doi.org/10.1007/s13402‐023‐00876‐9\\n530. Jiang, Shuanghong, Shan Xie, Dan Lv, Pu Wang,\\nHanchang He, Ting Zhang, Youlian Zhou, et al. 2017.\\n“Alteration of the Gut Microbiota in Chinese Population\\nWith Chronic Kidney Disease.” Scientific Reports 7: 2870.\\nhttps://doi.org/10.1038/s41598‐017‐02989‐2\\n531. Hurst, Rachel, Emma Meader, Abraham Gihawi,\\nGhanasyam Rallapalli, Jeremy Clark, Gemma L. Kay,\\nMartyn Webb, et al. 2022.“Microbiomes of Urine and the\\nProstate Are Linked to Human Prostate Cancer Risk\\nGroups.”\\nEuropean Urology Oncology 5: 412–9. https://doi.\\norg/10.1016/j.euo.2022.03.006\\n532. Marshall, Erin A., Fernando S. L. Filho, Don D. Sin,\\nStephen Lam, Janice M. Leung, Wan L. Lam. 2022.“Distinct\\nBronchial Microbiome Precedes Clinical Diagnosis of Lung\\nCancer.” Molecular Cancer 21: 68. https://doi.org/10.1186/\\ns12943‐022‐01544‐6\\n533. Li, Bowen, Daoyun Wang, Chengye Zhang, Yadong Wang,\\nZhicheng Huang, Libing Yang, Huaxia Yang, et al. 2024.\\n“Role of Respiratory System Microbiota in Development of\\nLung Cancer and Clinical Application. ” iMeta 3: e232.\\nhttps://doi.org/10.1002/imt2.232\\n534. Goto, Taichiro. 2022.“Microbiota and Lung Cancer.” Seminars\\nin Cancer Biology86: 1–10. https://doi.org/10.1016/j.semcancer.\\n2022.07.006\\n535. O'Shaughnessy, Megan, Orla Sheils, Anne‐Marie Baird. 2023.\\n“The Lung Microbiome in COPD and Lung Cancer: Explor-\\ning the Potential of Metal ‐Based Drugs. ” International\\nJournal of Molecular Sciences 24: 12296. https://doi.org/10.\\n3390/ijms241512296\\n536. Tsay, Jun ‐Chieh J., Benjamin G. Wu, Michelle H. Badri,\\nJose C. Clemente, Nan Shen, Peter Meyn, Yonghua Li, et al.\\n2018. “Airway Microbiota Is Associated With Upregulation of\\nthe PI3K Pathway in Lung Cancer.” American Journal of\\nRespiratory and Critical Care Medicine198: 1188–98. https://\\ndoi.org/10.1164/rccm.201710‐2118OC\\n537. Gomes, Sílvia, Bruno Cavadas,Joana Catarina Ferreira, Patrícia\\nIsabel Marques, Catarina Monteiro, Maria Sucena, Catarina\\nSousa, et al. 2019.“Profiling of Lung Microbiota Discloses Dif-\\nferences in Adenocarcinoma and Squamous Cell Carcinoma.”\\nScientific Reports 9: 12838. https://doi.org/10.1038/s41598‐019‐\\n49195‐w\\n538. Ashique, Sumel, Gabriele De Rubis, Ekta Sirohi, Neeraj Mishra,\\nMohd Rihan, Ashish Garg, Ruby‐Jean Reyes, et al. 2022.“Short\\nChain Fatty Acids: Fundamental Mediators of the Gut‐Lung Axis\\nand Their Involvement in Pulmonary Diseases.” Chemico‐\\nBiological Interactions368: 110231.https://doi.org/10.1016/j.cbi.\\n2022.110231\\n539. Liu, Weici, Jingtong Xu, Zheshun Pi, Yundi Chen,\\nGuanyu Jiang, Yuan Wan, Wenjun Mao. 2023.“Untangling\\nthe Web of Intratumor Microbiota in Lung Cancer. ”\\nBiochimica et Biophysica Acta (BBA) —Reviews on Cancer\\n1878: 189025. https://doi.org/10.1016/j.bbcan.2023.189025\\n540. Yang, Junjie, Xiaofeng Mu, Ye Wang, Dequan Zhu,\\nJiaming Zhang, Cheng Liang, Chen, Bin, et al. 2018.“Dys-\\nbiosis of the Salivary Microbiome Is Associated With Non‐\\n90 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 90, 'page_label': '91'}, page_content='smoking Female Lung Cancer and Correlated With Immu-\\nnocytochemistry Markers.” Frontiers in Oncology 8: 520.\\nhttps://doi.org/10.3389/fonc.2018.00520\\n541. Czarnecka‐Chrebelska, Karolina H. , Jacek Kordiak, Ewa\\nBrzeziańska‐Lasota, Dorota Pastuszak‐Lewandoska. 2023.“Res-\\npiratory Tract Oncobiome in Lung Carcinogenesis: Where Are\\nWe Now? ” Cancers 15: 4935. https://doi.org/10.3390/\\ncancers15204935\\n542. Otoshi, Takehiro, Tatsuya Nagano, Jonguk Park, Koji Hosomi,\\nTomoya Yamashita, Motoko Tachihara, Tokiko Tabata, et al.\\n2022. “The Gut Microbiome as a Biomarker of Cancer Progres-\\nsion Among Female Never‐Smokers With Lung Adenocarci-\\nnoma.” Anticancer Research 42: 1589–98. https://doi.org/10.\\n21873/anticanres.15633\\n543. Gui, Qifeng, Hanyu Li, Ange Wang, Xinxiu Zhao,\\nZhongju Tan, Lufang Chen, Keying Xu, Chi Xiao. 2020.“The\\nAssociation Between Gut Butyrate‐Producing Bacteria and\\nNon‐Small‐Cell Lung Cancer.” Journal of Clinical Laboratory\\nAnalysis 34: e23318. https://doi.org/10.1002/jcla.23318\\n544. Upreti, Deepa, Susumu Ishig uro, Morgan Phillips, Ayaka\\nNakashima, Kengo Suzuki, Jeffrey Comer, Masaaki Tamura.\\n2023. “Euglena gracilisExtract Protects From Tobacco Smoke\\nCarcinogen‐Induced Lung Cancer by Altering Gut Microbiota\\nMetabolome.” Integrative Cancer Therapies 22: 1534735423\\n1195323.https://doi.org/10.1177/15347354231195323\\n545. Shin, Yoonhwa, Sunhee Han, Juhui Kwon, Songhyun Ju,\\nTae Choi, Insug Kang, Sung Kim. 2023. “Roles of Short‐\\nChain Fatty Acids in Inflammatory Bowel Disease.” Nutrients\\n15: 4466.https://doi.org/10.3390/nu15204466\\n546. Xiong, Xin, Le‐Wei Zheng, Yu Ding, Yu‐Fei Chen, Yu‐Wen\\nCai, Lei‐Ping Wang, Liang Huang, et al. 2025.“Breast Can-\\ncer: Pathogenesis and Treatments.” Signal Transduction and\\nTargeted Therapy10: 49.https://doi.org/10.1038/s41392\\n‐024‐\\n02108‐4\\n547. Chadha, Jatin, Deeptashree Nandi, Yama Atri, Alo Nag.\\n2021. “Significance of Human Microbiome in Breast Cancer:\\nTale of an Invisible and an Invincible.” Seminars in Cancer\\nBiology 70: 112–27. https://doi.org/10.1016/j.semcancer.2020.\\n07.010\\n548. Thu, May Soe, Korn Chotirosniramit, Tanawin Nopsopon,\\nNattiya Hirankarn, Krit Pongpirul. 2023. “Human Gut,\\nBreast, and Oral Microbiome in Breast Cancer: A Systematic\\nReview and Meta ‐Analysis.” Frontiers in Oncology 13:\\n1144021. https://doi.org/10.3389/fonc.2023.1144021\\n549. Tzeng, Alice, Naseer Sangwan, Margaret Jia, Chin‐Chih Liu,\\nKaren S. Keslar, Erinn Downs‐Kelly, Robert L. Fairchild,\\net al. 2021. “Human Breast Microbiome Correlates With\\nPrognostic Features and Immunological Signatures in Breast\\nCancer.” Genome Medicine 13: 60. https://doi.org/10.1186/\\ns13073‐021‐00874‐2\\n550. Parida, Sheetal, Julia L. Drewes. 2022. “Unwanted Passen-\\ngers: Microbes Hitchiking in Breast Cancer Metastases.” Cell\\nHost & Microbe 30: 875–7. https://doi.org/10.1016/j.chom.\\n2022.05.010\\n551. Hou, Ting, Xiaoling Huang, Jiahui Lai, Dongfang Zhou.\\n2025. “Intra‐Tumoral Bacteria in Breast Cancer and Inter-\\nvention Strategies.” Advanced Drug Delivery Reviews 217:\\n115516. https://doi.org/10.1016/j.addr.2025.115516\\n552. Arnone, Alana A., Yu ‐Ting Tsai, J. Mark Cline,\\nAdam S. Wilson, Brian Westwood, Meghan E. Seger,\\nAkiko Chiba, et al. 2025. “Endocrine‐Targeting Therapies\\nShift the Breast Microbiome to Reduce Estrogen Receptor‐α\\nBreast Cancer Risk.” Cell Reports. Medicine6: 101880.https://\\ndoi.org/10.1016/j.xcrm.2024.101880\\n553. Wang, Wenhui, Zihao Ou, Xixin Huang, Jingyu Wang,\\nQianbei Li, Minghui Wen, Lei Zheng. 2024.“Microbiota and\\nGlioma: A New Perspective From Association to Clinical\\nTranslation.” Gut Microbes 16: 2394166. https://doi.org/10.\\n1080/19490976.2024.2394166\\n554. Ge, Yanshan, Xinhui Wang, Yali Guo, Junting Yan,\\nAliya Abuduwaili, Kasimujiang Aximujiang, Jie Yan,\\nMinghua Wu. 2021. “Gut Microbiota Influence Tumor\\nDevelopment and Alter Interactions With the Human\\nImmune System.” Journal of Experimental & Clinical Cancer\\nResearch: CR 40: 42. https://doi.org/10.1186/s13046 ‐021‐\\n01845‐6\\n555. Chatterjee, Jit, Xuanhe Qi, Rui Mu, Xuanwei Li,\\nTalia Eligator, Megan Ouyang, Stephanie L. Bozeman, et al.\\n2025. “Intestinal Bacteroides Drives Glioma Progression by\\nRegulating CD8+ T Cell Tumor Infiltration.” Neuro‐Oncology\\n27(6): 1579–1593. https://doi.org/10.1093/neuonc/noaf024\\n556. Lin, Ben, Zhen Ye, Zhan Cao, Zhao Ye, Yifei Yu,\\nWeiliang Jiang, Sichen Guo, et al. 2024.“Integrated Micro-\\nbiome and Metabolome Analysis Reveals Hypothalamic ‐\\nComorbidities Related Signatures in Craniopharyngioma.”\\nAdvanced Science (Weinheim, Baden‐Wurttemberg, Germany)\\n11: e2400684.https://doi.org/10.1002/advs.202400684\\n557. Chandra, Vidhi, Le, Li, Olivereen Le Roux, Yu Zhang,\\nRian M. Howell, Dhwani N. Rupani, Seyda Baydogan, et al.\\n2024. “Gut Epithelial Interleukin‐17 Receptor A Signaling\\nCan Modulate Distant Tumors Growth Through Microbial\\nRegulation.” Cancer Cell 42: 85–100.e6. https://doi.org/10.\\n1016/j.ccell.2023.12.006\\n558. Mehrian ‐Shai, Ruty, Juergen K. V. Reichardt,\\nCurtis C. Harris, Amos Toren. 2019.“The Gut–Brain Axis,\\nPaving the Way to Brain Cancer.” Trends in Cancer5: 200–7.\\nhttps://doi.org/10.1016/j.trecan.2019.02.008\\n559. Zhu, Gengjun, Lifang Jin, Weizhang Shen, Meng Zhao,\\nNing Liu. 2023.“Intratumor Microbiota: Occult Participants\\nin the Microenvironment of Multiple Myeloma.” Biochimica\\net Biophysica Acta (BBA)—Reviews on Cancer1878: 188959.\\nhttps://doi.org/10.1016/j.bbcan.2023.188959\\n560. Shah, Urvi A., Richa Parikh, Francesca Castro,\\nMatteo Bellone, Alexander M. Lesokhin. 2023.“Dietary and\\nMicrobiome Evidence in Multiple Myeloma and Other\\nPlasma Cell Disorders.” Leukemia 37: 964–80.\\nhttps://doi.\\norg/10.1038/s41375‐023‐01874‐4\\n561. Fernandez Sanchez, Josaura, Arushana A. Maknojia,\\nKatherine Y. King. 2024.“Blood and Guts: How the Intesti-\\nnal Microbiome Shapes Hematopoiesis and Treatment of\\nHematologic Disease.” Blood 143: 1689–701. https://doi.org/\\n10.1182/blood.2023021174\\n562. Manzo, Veronica E., Ami S. Bhatt. 2015.“The Human Mi-\\ncrobiome in Hematopoiesis and Hematologic Disorders. ”\\nBlood 126: 311 –8. https://doi.org/10.1182/blood ‐2015‐04‐\\n574392\\nMICROBIOME IN CANCER | 91 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 91, 'page_label': '92'}, page_content='563. Manohar, Sonal M. 2024.“At the Crossroads of TNFα Signaling\\nand Cancer.” Current Molecular Pharmacology17: e0609232\\n20758. https://doi.org/10.2174/1874467217666230908111754\\n564. Li, Guoqing, Yuxuan Wan, Alan Jiao, Ke Jiang, Gaoyuan Cui,\\nJinxin Tang, Simin Yu, et al.“Breaking Boundaries: Chronic\\nDiseases and the Frontiers of Immune Microenvironments.”\\nMed Research 1: 62–102. https://doi.org/10.1002/mdr2.70007\\n565. Zhang, Sheng, Cheng Kong, Yongzhi Yang, Sanjun Cai,\\nXinxiang Li, Guoxiang Cai, Yanlei Ma. 2020.“Human Oral\\nMicrobiome Dysbiosis as a Novel Non‐Invasive Biomarker in\\nDetection of Colorectal Cancer.” Theranostics 10: 11595–606.\\nhttps://doi.org/10.7150/thno.49515\\n566. Xian, Rena R., Richard F. Ambinder. 2023.“Cell‐Free Cir-\\nculating Tumor DNA and Epstein–Barr Virus DNA for Early\\nDiagnosis of Epstein –Barr Virus ‐Associated Cancers. ”\\nJournal of Clinical Oncology 41: 4290–2. https://doi.org/10.\\n1200/JCO.23.00687\\n567. Wang, Ni, Jing‐Yuan Fang. 2023.“Fusobacterium nucleatum,\\na Key Pathogenic Factor and Microbial Biomarker for Colo-\\nrectal Cancer.” Trends in Microbiology 31: 159–72. https://\\ndoi.org/10.1016/j.tim.2022.08.010\\n568. Li, Chiao ‐Ling, Ming ‐Chih Ho, You ‐Yu Lin, Sheng ‐Tai\\nTzeng, Yun‐Ju Chen, Hsin‐Yung Pai, Ya‐Chun Wang, et al.\\n2020. “Cell‐Free Virus‐Host Chimera DNA From Hepatitis B\\nVirus Integration Sites as a Circulating Biomarker of Hepa-\\ntocellular Cancer.” Hepatology 72: 2063–76. https://doi.org/\\n10.1002/hep.31230\\n569. Hakozaki, Taiki, Kentaro Tanaka, Yoshimasa Shiraishi,\\nYuta Sekino, Noriko Mitome, Yusuke Okuma, Tomoiki Aiba,\\net al. 2025, 20, 912–27. “Gut Microbiota in Advanced NSCLC\\nReceiving Chemoimmunotherapy: An Ancillary Biomarker\\nStudy From the Phase III Trial JCOG2007 (NIPPON). ”\\nJournal of Thoracic Oncology. https://doi.org/10.1016/j.jtho.\\n2025.02.026\\n570. Kwong, Thomas N. Y., Xiansong Wang, Geicho Nakatsu,\\nTai Cheong Chow, Timothy Tipoe, Rudin Z. W. Dai,\\nKelvin K. K. Tsoi, et al. 2018.“Association Between Bacte-\\nremia From Specific Microbes and Subsequent Diagnosis of\\nColorectal Cancer.” Gastroenterology 155: 383–90.e8. https://\\ndoi.org/10.1053/j.gastro.2018.04.028\\n571. Coker, Olabisi Oluwabukola, Changan Liu, William Ka Kei Wu,\\nSunny Hei Wong, Wei Jia, Joseph?J. Y. Sung, Jun Yu. 2022.\\n“Altered Gut Metabolites and Microbiota Interactions Are Im-\\nplicated in Colorectal Carcinogenesis and Can Be Non‐Invasive\\nDiagnostic Biomarkers.” Microbiome 10: 35. https://doi.org/10.\\n1186/s40168‐021‐01208‐5\\n572. Zhang, Jia, Yangting He, Lu Xia, Jing Yi, Zhen Wang,\\nYingying Zhao, Xuemei Song, et al. 2022. “Expansion of\\nColorectal Cancer Biomarkers Based on Gut Bacteria and\\nViruses. ” Cancers 14: 4662. https://doi.org/10.3390/\\ncancers14194662\\n573. Zhao, Liuyang, Yu, Shi, Harry Cheuk‐Hay Lau, Weixin Liu,\\nGuangwen Luo, Guoping Wang, Changan Liu, et al. 2022.\\n“Uncovering 1058 Novel Human Enteric DNA Viruses\\nThrough Deep Long‐Read Third‐Generation Sequencing and\\nTheir Clinical Impact. ” Gastroenterology 163: 699 –711.\\nhttps://doi.org/10.1053/j.gastro.2022.05.048\\n574. Yunus, Aisyah, Norfilza Mohd Mokhtar, Raja Affendi Ali,\\nSiti Maryam Ahmad Kendong, Hajar Fauzan Ahmad. 2024.\\n“Methods for Identification of the Opportunistic Gut Myco-\\nbiome From Colorectal Adenocarcinoma Biopsy Tissues.”\\nMethodsX 12: 102623. https://doi.org/10.1016/j.mex.2024.\\n102623\\n575. Chen, Jijun, Siru Nie, Xunan Qiu, Shuwen Zheng,\\nChuxuan Ni, Yuan Yuan, Yuehua Gong. 2023.“Leveraging\\nExisting 16S rRNA Microbial Data to Identify Diagnostic\\nBiomarker in Chinese Patients With Gastric Cancer: A Sys-\\ntematic Meta‐Analysis.” mSystems 8: e0074723. https://doi.\\norg/10.1128/msystems.00747‐23\\n576. Xu, Shuo, Chunjie Xiang, Juan Wu, Yuhao Teng,\\nZhenfeng Wu, Ruiping Wang, Bin Lu, et al. 2021.“\\nTongue\\nCoating Bacteria as a Potential Stable Biomarker for Gastric\\nCancer Independent of Lifestyle. ” Digestive Diseases and\\nSciences 66: 2964 –80. https://doi.org/10.1007/s10620 ‐020‐\\n06637‐0\\n577. Schinzari, V., V. Barnaba, S. Piconese. 2015.“Chronic Hep-\\natitis B Virus and Hepatitis C Virus Infections and Cancer:\\nSynergy Between Viral and Host Factors. ” Clinical\\nMicrobiology and Infection 21: 969–74. https://doi.org/10.\\n1016/j.cmi.2015.06.026\\n578. Han, Kathy, Jinfeng Zou, Zhen Zhao, Zeynep Baskurt,\\nYangqiao Zheng, Elizabeth Barnes, Jennifer Croke, et al.\\n2024. “Clinical Validation of Human Papilloma Virus Cir-\\nculating Tumor DNA for Early Detection of Residual Disease\\nAfter Chemoradiation in Cervical Cancer. ” Journal of\\nClinical Oncology 42: 431–40. https://doi.org/10.1200/JCO.\\n23.00954\\n579. Mendez, Roberto, Kousik Kesh, Nivedita Arora,\\nLeá Di Martino, Florencia McAllister, Nipun Merchant,\\nSulagna Banerjee, Santanu Banerjee. 2020.“Microbial Dys-\\nbiosis and Polyamine Metabolism as Predictive Markers for\\nEarly Detection of Pancreatic Cancer.” Carcinogenesis 41:\\n561–70. https://doi.org/10.1093/carcin/bgz116\\n580. Yazdi, Hamid Reza, Abolfazl Movafagh, Fateme Fallah,\\nShohreh Alizadeh Shargh, Neda Mansouri, Atefeh Heidary\\nPour, Mehrdad Hashemi. 2016. “Evaluation of Methylo-\\nbacterium Radiotolerance andSphyngomonas yanoikoaie in\\nSentinel Lymph?Nodes of Breast Cancer Cases. ” Asian\\nPacific Journal of Cancer Prevention17: 279–85. https://doi.\\norg/10.7314/apjcp.2016.17.s3.279\\n581. Liang, Te, Hao Chen, Lei Liu, Yongqiang Zheng, Zhaoen Ma,\\nLing Min, Jiahui Zhang, et al. 2024.“Antibody Profiling of\\nPan‐Cancer Viral Proteome Reveals Biomarkers for Naso-\\npharyngeal Carcinoma Diagnosis and Prognosis.” Molecular\\n& Cellular Proteomics: MCP 23: 100729. https://doi.org/10.\\n1016/j.mcpro.2024.100729\\n582. Wang, Na, Si Wu, Lanxiang Huang, Yue Hu, Xin He,\\nJourong He, Ben Hu, et al. 2025.“Intratumoral Microbiome:\\nImplications for Immune Modulation and Innovative Ther-\\napeutic Strategies in Cancer.” Journal of Biomedical Science\\n32: 23. https://doi.org/10.1186/s12929‐025‐01117‐x\\n583. Jiang, Zhengting, Wenjie Zhang, Zhilin Zhang, Gengyu Sha,\\nDaorong Wang, Dong Tang. 2023.“Intratumoral Microbiota:\\nA New Force in Diagnosing and Treating Pancreatic Cancer.”\\nCancer Letters 554: 216031. https://doi.org/10.1016/j.canlet.\\n2022.216031\\n584. Liu, Lanxiang, Haiyang Wang, Hanping Zhang, Xueyi Chen,\\nYangdong Zhang, Ji Wu, Libo Zhao, et al. 2022.“Toward a\\n92 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 92, 'page_label': '93'}, page_content='Deeper Understanding of Gut Microbiome in Depression:\\nThe Promise of Clinical Applicability. ” Advanced Science\\n(Weinheim, Baden ‐Wurttemberg, Germany) 9: e2203707.\\nhttps://doi.org/10.1002/advs.202203707\\n585. Kang, Xing, Harry Cheuk ‐Hay Lau, Jun Yu. 2024. “Mod-\\nulating Gut Microbiome in Cancer Immunotherapy: Har-\\nnessing Microbes to Enhance Treatment Efficacy. ” Cell\\nReports. Medicine 5: 101478. https://doi.org/10.1016/j.xcrm.\\n2024.101478\\n586. Han, Eui ‐Jeong, Ji‐Seon Ahn, Yu‐Jin Choi, Da‐Hye Kim,\\nJong‐Soon Choi, Hea‐Jong Chung. 2025.“Exploring the Gut\\nMicrobiome: A Potential Biomarker for Cancer Diagnosis,\\nPrognosis, and Therapy. ” Biochimica et Biophysica Acta\\n(BBA)—Reviews on Cancer 1880: 189251. https://doi.org/10.\\n1016/j.bbcan.2024.189251\\n587. Liu, Yali, Harry Cheuk‐Hay Lau, Wing Yin Cheng, Jun Yu.\\n2023. “Gut Microbiome in Colorectal Cancer: Clinical Diag-\\nnosis and Treatment.” Genomics, Proteomics & Bioinformatics\\n21: 84–96. https://doi.org/10.1016/j.gpb.2022.07.002\\n588. Zhang, Limin, Ziying Feng, Yinghua Li, Cuiting Lv,\\nChunchun Li, Yue Hu, Mingsheng Fu, Liang Song. 2023.\\n“Salivary and Fecal Microbiota: Potential New Biomarkers\\nfor Early Screening of Colorectal Polyps. ” Frontiers in\\nMicrobiology 14: 1182346. https://doi.org/10.3389/fmicb.\\n2023.1182346\\n589. Che, Shusheng, Zhiyong Yan, Yugong Feng, Hai Zhao. 2024.\\n“Unveiling the Intratumoral Microbiota Within Cancer\\nLandscapes.” iScience 27: 109893. https://doi.org/10.1016/j.\\nisci.2024.109893\\n590. Li, Xuebo, Xuelian Yuan, Xiumin Zhu, Changjun Li, Lei Ji,\\nKebo Lv, Geng Tian, Kang Ning, Jialiang Yang. 2023.“A\\nMeta‐Analysis Of Tissue Microbial Biomarkers for Recur-\\nrence and Metastasis in Multiple Cancer Types.” Journal of\\nMedical Microbiology 72. https://doi.org /10.1099/jmm.0.\\n001744\\n591. Ren, Zhigang, Ang Li, Jianwen Jiang, Lin Zhou, Zujiang Yu,\\nHaifeng Lu, Haiyang Xie, et al. 2019. “Gut Microbiome\\nAnalysis as a Tool Towards Targeted Non ‐Invasive Bio-\\nmarkers for Early Hepatocellular Carcinoma. ” Gut 68:\\n1014–23. https://doi.org/10.1136/gutjnl‐2017‐315084\\n592. Cho, Eun Ju, Sangseob Leem, Sun Ah Kim, Jinho Yang,\\nYun Bin Lee, Soon Sun Kim, Jae Youn Cheong, et al. 2019.\\n“Circulating Microbiota‐Based Metagenomic Signature for\\nDetection of Hepatocellular Carcinoma.” Scientific Reports9:\\n7536. https://doi.org/10.1038/s41598‐019‐44012‐w\\n593. Loomba, Rohit, Victor Seguritan, Weizhong Li, Tao Long,\\nNiels Klitgord, Archana Bhatt, Parambir Singh Dulai, et al.\\n2017. “Gut Microbiome‐Based Metagenomic Signature for\\nNon‐invasive Detection of Advanced Fibrosis in Human\\nNonalcoholic Fatty Liver Disease. ” Cell Metabolism 25:\\n1054–62.e5. https://doi.org/10.1016/j.cmet.2017.04.001\\n594. Liu, Ying, Hao Ai. 2024. “Comprehensive Insights Into\\nHuman Papillomavirus and Cervical Cancer: Patho-\\nphysiology, Screening, and Vaccination Strategies. ”\\nBiochimica et Biophysica Acta (BBA) —Reviews on Cancer\\n1879: 189192.https://doi.org/10.1016/j.bbcan.2024.189192\\n595. Malpica, Luis, Mario L. Marques‐Piubelli, Brady E. Beltran,\\nJulio C. Chavez, Roberto N. Miranda, Jorge J. Castillo. 2024.\\n“EBV‐Positive Diffuse Large B ‐Cell Lymphoma, Not\\nOtherwise Specified: 2024 Update on the Diagnosis, Risk‐\\nStratification, and Management. ” American Journal of\\nHematology 99: 2002–15. https://doi.org/10.1002/ajh.27430\\n596. Zoulim, Fabien, Pei ‐Jer Chen, Maura Dandri,\\nPatrick T. Kennedy, Christoph Seeger. 2024. “Hepatitis B\\nVirus DNA Integration: Implications for Diagnostics, Ther-\\napy, and Outcome. ” Journal of Hepatology 81: 1087–99.\\nhttps://doi.org/10.1016/j.jhep.2024.06.037\\n597. Arbyn, Marc, Marie Simon, Eliana Peeters, Lan Xu,\\nChris J. L. M. Meijer, Johannes Berkhof, Kate Cuschieri,\\net al. 2021. “2020 List of Human Papillomavirus Assays\\nSuitable for Primary Cervical Cancer Screening.” Clinical\\nMicrobiology and Infection 27: 1083–\\n95. https://doi.org/10.\\n1016/j.cmi.2021.04.031\\n598. Wolf, Jonas, Lucas Felipe Kist, Samanta Brangel Pereira,\\nMarilze Alves Quessada, Helena Petek, Arthur Pille,\\nJuçara Gasparetto Maccari, Mohamed Parrini Mutlaq,\\nLuiz Antonio Nasi. 2024. “Human Papillomavirus Infec-\\ntion: Epidemiolog y, Biology, Host Interactions, Cancer\\nDevelopment, Prevention, and Therapeutics. ” Reviews in\\nMedical Virology 34: e2537. https://doi.org/10.1002/\\nrmv.2537\\n599. Emlet, Cade, Mack Ruffin, Regina Lamendella. 2020. “En-\\nteric Virome and Carcinogenesis in the Gut. ” Digestive\\nDiseases and Sciences 65: 852–64. https://doi.org/10.1007/\\ns10620‐020‐06126‐4\\n600. Zhang, Zhemei, Yongqing Yang, Lei Zhang, Yang Wu,\\nPengxia Jia, Qingmei Ma, Danni Wang. 2023.“Relationship\\nBetween Cervicovaginal Microecological Changes and\\nHPV16/18 Infection and Cervical Cancer in Women of\\nChildbearing Age.” Annals of Clinical and Laboratory Science\\n53: 825–34.\\n601. Elbehi, Attia M., R. I. Anu, Bene Ekine ‐Afolabi,\\nElizabeth Cash. 2020. “Emerging Role of Immune Check-\\npoint Inhibitors and Predictive Biomarkers in Head and Neck\\nCancers.” Oral Oncology 109: 104977. https://doi.org/10.\\n1016/j.oraloncology.2020.104977\\n602. Xu, Jian ‐Hua. 2013. “Hepatitis B or C Viral Infection and\\nRisk of Pancreatic Cancer: A Meta‐Analysis of Observational\\nStudies.” World Journal of Gastroenterology 19: 4234–41.\\nhttps://doi.org/10.3748/wjg.v19.i26.4234\\n603. Saftien, Aurelia, Jens Puschhof, Eran Elinav. 2023. “Fungi\\nand Cancer.” Gut 72: 1410–25. https://doi.org/10.1136/gutjnl‐\\n2022‐327952\\n604. Xu, Jing, Chunjie Xiang, Cong Zhang, Boqi Xu, Juan Wu,\\nRuiping Wang, Yaping Yang, et al. 2019. “Microbial Bio-\\nmarkers of Common Tongue Coatings in Patients With\\nGastric Cancer.” Microbial Pathogenesis127: 97–105. https://\\ndoi.org/10.1016/j.micpath.2018.11.051\\n605. Ka źmierczak‐Siedlecka, Karolina, Ale š Dvořák,\\nMarcin Folwarski, Agnieszka Daca, Katarzyna Przewłócka,\\nWojciech Makarewicz. 2020. “Fungal Gut Microbiota Dys-\\nbiosis and Its Role in Colorectal, Oral, and Pancreatic Car-\\ncinogenesis. ” Cancers 12: 1326. https://doi.org/10.3390/\\ncancers12051326\\n606. Chung, Li‐Min, Ji‐An Liang, Cheng‐Li Lin, Li‐Min Sun, Chia\\n‐Hung Kao. 2017.“Cancer Risk in Patients With Candidiasis:\\nA Nationwide Population‐Based Cohort Study.” Oncotarget 8:\\n63562–73. https://doi.org/10.18632/oncotarget.18855\\nMICROBIOME IN CANCER | 93 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 93, 'page_label': '94'}, page_content='607. Kashyap, Sheetal, Soumya Pal, Gourav Chandan, Vipin Saini,\\nSasanka Chakrabarti, Neeraj K. Saini, Amit Mittal, et al.\\n2022. “Understanding the Cross‐Talk Between Human Mi-\\ncrobiota and Gastrointestinal Cancer for Developing Poten-\\ntial Diagnostic and Prognostic Biomarkers. ” Seminars in\\nCancer Biology 86: 643 –51. https://doi.org/10.1016/j.\\nsemcancer.2021.04.020\\n608. Malhotra, Pratibha, Ranjith Palanisamy, Jose A. Caparros‐\\nMartin, Marco Falasca. 2023. “Bile Acids and Microbiota\\nInterplay in Pancreatic Cancer.” Cancers 15: 3573. https://\\ndoi.org/10.3390/cancers15143573\\n609. Chalova, Petra, Anton Tazky, Ludovit Skultety,\\nLenka Minichova, Michal Chovanec, Sona Ciernikova,\\nPeter Mikus, Juraj Piestansky. 2023.“Determination of Short‐\\nChain Fatty Acids as Putative Biomarkers of Cancer Diseases\\nby Modern Analytical Strategies and Tools: A Review. ”\\nFrontiers in Oncology 13: 1110235. https://doi.org/10.3389/\\nfonc.2023.1110235\\n610. Fang, Chuan ‐Yin, Jung ‐Sheng Chen, Bing ‐Mu Hsu,\\nBashir Hussain, Jagat Rathod, Kuo‐Hsin Lee. 2021. “Colo-\\nrectal Cancer Stage‐Specific Fecal Bacterial Community Finger-\\nprinting of the Taiwanese Population and Underpinning of\\nPotential Taxonomic Biomarkers.” Microorganisms 9: 1548.\\nhttps://doi.org/10.3390/microorganisms9081548\\n611. Chen, Feng, Xudong Dai, Chang‐Chun Zhou, Ke‐Xin Li, Yu‐\\nJuan Zhang, Xiao‐Ying Lou, Yuan‐Min Zhu, et al. 2022.\\n“Integrated Analysis of the Faecal Metagenome and Serum\\nMetabolome Reveals the Role of Gut Microbiome‐Associated\\nMetabolites in the Detection of Colorectal Cancer and Ade-\\nnoma.” Gut 71: 1315–25. https://doi.org/10.1136/gutjnl‐2020‐\\n323476\\n612. Liu, Bo, Yige Li, Lijun Suo, Wei Zhang, Hongyun Cao,\\nRuicai Wang, Jiahui Luan, et al. 2022.“Characterizing Mi-\\ncrobiota and Metabolomics Analysis to Identify Candidate\\nBiomarkers in Lung Cancer. ” Frontiers in Oncology 12:\\n1058436. https://doi.org/10.3389/fonc.2022.1058436\\n613. Louis, Petra, Georgina L. Hold, Harry J. Flint. 2014.“\\nThe Gut\\nMicrobiota, Bacterial Metabolites and Colorectal Cancer.”\\nNature Reviews Microbiology 12: 661–72. https://doi.org/10.\\n1038/nrmicro3344\\n614. Hatta, Muhammad Nur Adam, Ezanee Azlina Mohamad Hanif,\\nSiok‐Fong Chin, Teck Yew Low, Hui‐Min Neoh. 2023.“Parvi-\\nmonas micraInfection Enhances Proliferation, Wound Healing,\\nand Inflammation of a Colorectal Cancer Cell Line.” Bioscience\\nReports 43: BSR20230609.https://doi.org/10.1042/BSR20230609\\n615. Ding, Ruojun, ShengYang Bertrand Lian, Yew Chong Tam,\\nChoon Chiat Oh. 2024.“The Cutaneous Microbiome in Skin\\nCancer—A Systematic Review. ” Journal Der Deutschen\\nDermatologischen Gesellschaft = Journal of the German\\nSociety of Dermatology: JDDG22: 177–84. https://doi.org/10.\\n1111/ddg.15294\\n616. Wu, Peng, Guihao Zhang, Jie Zhao, Jiawei Chen, Yang Chen,\\nWeina Huang, Jialei Zhong, Jiarong Zeng. 2018.“Profiling\\nthe Urinary Microbiota in Male Patients With Bladder Can-\\ncer in China.” Frontiers in Cellular and Infection Microbiology\\n8: 167.https://doi.org/10.3389/fcimb.2018.00167\\n617. Derosa, Lisa, Bertrand Routy, Andrew Maltez Thomas,\\nValerio Iebba, Gerard Zalcman, Sylvie Friard, Julien Mazieres,\\net al. 2022.“IntestinalAkkermansia muciniphilaPredicts Clinical\\nResponse to PD‐1 Blockade in Patients With Advanced Non‐\\nSmall‐Cell Lung Cancer.” Nature Medicine28: 315–24. https://\\ndoi.org/10.1038/s41591‐021‐01655‐5\\n618. Liu, Yuting, Huiwen Wang, Haiyang Jiang, Zhumei Sun,\\nAnyang Sun. 2023. “Alloprevotella Can be Considered as a\\nPotential Oral Biomarker in Intestinal Metaphase of Gastric\\nPatients.” Studies in Health Technology and Informatics308:\\n155–67. https://doi.org/10.3233/SHTI230836\\n619. Lee, Wei‐Hsiang, Hui‐Mei Chen, Shun‐Fa Yang, Chao Liang,\\nChih‐Yu Peng, Feng ‐Mao Lin, Lo\\n‐Lin Tsai, et al. 2017.\\n“Bacterial Alterations in Salivary Microbiota and Their\\nAssociation in Oral Cancer. ” Scientific Reports 7: 16540.\\nhttps://doi.org/10.1038/s41598‐017‐16418‐x\\n620. Mai, Guoqin, Limei Chen, Ran Li, Quan Liu, Haoran Zhang,\\nYingfei Ma. 2019. “Common Core Bacterial Biomarkers of\\nBladder Cancer Based on Multiple Datasets. ” BioMed\\nResearch International 2019: 4824909. https://doi.org/10.\\n1155/2019/4824909\\n621. Law, Helen Ka Wai, Howard Chi Ho Yim. 2024. “Early\\nDiagnosis of Cancer Using Circulating Microbial DNA.” Cell\\nReports. Medicine 5: 101502. https://doi.org/10.1016/j.xcrm.\\n2024.101502\\n622. Kataria, Radhika, Saeed Shoaie, Anita Grigoriadis,\\nJonathan C. M. Wan. 2023.“Leveraging Circulating Micro-\\nbial DNA for Early Cancer Detection.” Trends in Cancer9:\\n879–82. https://doi.org/10.1016/j.trecan.2023.08.001\\n623. Dzutsev, Amiran, Giorgio Trinchieri. 2020.“Microbial DNA\\nSignature in Plasma Enables Cancer Diagnosis. ” Nature\\nReviews Clinical Oncology17: 453–4. https://doi.org/10.1038/\\ns41571‐020‐0391‐1\\n624. Xiao, Qian, Wei Lu, Xiangxing Kong, Yang W. Shao,\\nYeting Hu, Ao Wang, Hua Bao, et al. 2021.“Alterations of\\nCirculating Bacterial DNA in Colorectal Cancer and Ade-\\nnoma: A Proof‐of‐Concept Study.” Cancer Letters499: 201–8.\\nhttps://doi.org/10.1016/j.canlet.2020.11.030\\n625. Ahn, Sun M., Jason Y. K. Chan, Zhe Zhang, Hao Wang,\\nZubair Khan, Justin A. Bishop, William Westra, Wayne M. Koch,\\nJoseph?A. Califano. 2014. “Saliva and Plasma Quantitative\\nPolymerase Chain Reaction‐Based Detection and Surveillance of\\nHuman Papillomavirus‐Related Head and Neck Cancer.” JAMA\\nOtolaryngology—Head & Neck Surgery140: 846–54. https://doi.\\norg/10.1001/jamaoto.2014.1338\\n626. Yang, Jinho, Hyo Eun Moon, Hyung Woo Park, Andrea\\nMcDowell, Tae‐Seop Shin, Young‐Koo Jee, Sungmin Kym,\\nSun Ha Paek, Yoon‐Keun Kim. 2020. “Brain Tumor Diag-\\nnostic Model and Dietary Effect Based on Extracellular\\nVesicle Microbiome Data in Serum. ” Experimental &\\nMolecular Medicine 52: 1602 –13. https://doi.org/10.1038/\\ns12276‐020‐00501‐x\\n627. Melo, Sonia A., Linda B. Luecke, Christoph Kahlert,\\nAgustin F. Fernandez, Seth T. Gammon, Judith Kaye,\\nValerie S. LeBleu, et al. 2015.“Glypican‐1 Identifies Cancer\\nExosomes and Detects Early Pancreatic Cancer.” Nature 523:\\n177–82. https://doi.org/10.1038/nature14581\\n628. Barteneva, Natasha S., Yeldar Baiken, Elizaveta Fasler‐Kan,\\nKenneth Alibek, Sheng Wang, Natalia Maltsev,\\nEugene D. Ponomarev, et al. 2017.“Extracellular Vesicles in\\nGastrointestinal Cancer in Conjunction With Microbiota: On\\nthe Border of Kingdoms. ” Biochimica et Biophysica Acta\\n94 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 94, 'page_label': '95'}, page_content='(BBA)—Reviews on Cancer 1868: 372–93. https://doi.org/10.\\n1016/j.bbcan.2017.06.005\\n629. Gilbert, Jack A., Martin J. Blaser, J. Gregory Caporaso,\\nJanet K. Jansson, Susan V. Lynch, Rob Knight. 2018.“Cur-\\nrent Understanding of the Human Microbiome. ” Nature\\nMedicine 24: 392–400. https://doi.org/10.1038/nm.4517\\n630. Baron, Ellen Jo, J. Michael Miller, Melvin P. Weinstein,\\nSandra S. Richter, Peter H. Gilligan, Richard B. Thomson,\\nPaul Bourbeau, et al. 2013.“A Guide to Utilization of the\\nMicrobiology Laboratory for Diagnosis of Infectious Diseases:\\n2013 Recommendations by the Infectious Diseases Society of\\nAmerica (IDSA) and the American Society for Microbiology\\n(ASM)(a).” Clinical Infectious Diseases: An Official\\nPublication of the Infectious Diseases Society of America57:\\ne22–e121. https://doi.org/10.1093/cid/cit278\\n631. Sánchez ‐Romero, M. Isabel, Juan Manuel García ‐Lechuz\\nMoya, Juan José González López, Nieves Orta Mira. 2019.\\n“Collection, Transport and General Processing of Clinical\\nSpecimens in Microbiology Laboratory. ” Enfermedades\\nInfecciosas Y Microbiologia Clinica (English Ed.) 37: 127–34.\\nhttps://doi.org/10.1016/j.eimc.2017.12.002\\n632. Bharti, Richa, Dominik G. Grimm. 2021. “Current Chal-\\nlenges and Best‐Practice Protocols for Microbiome Analysis.”\\nBriefings in Bioinformatics 22: 178–93. https://doi.org/10.\\n1093/bib/bbz155\\n633. Fabre, Valeria, Angelina Davis, Daniel J. Diekema,\\nBruno Granwehr, Mary K. Hayden, Christopher F. Lowe,\\nChristopher D. Pfeiffer, et al. 2023.“Principles of Diagnostic\\nStewardship: A Practical Guide From the Society for\\nHealthcare Epidemiology of America Diagnostic Stewardship\\nTask Force.” Infection Control & Hospital Epidemiology44:\\n178–85. https://doi.org/10.1017/ice.2023.5\\n634. Derosa, Lisa, Bertrand Routy, Antoine Desilets, Romain Daillère,\\nSafae Terrisse, Guido Kroemer, Laurence Zitvogel. 2021.\\n“Microbiota‐Centered Interventions: The Next Breakthrough in\\nImmuno‐Oncology?” Cancer Discovery11: 2396–412. https://doi.\\norg/10.1158/2159‐8290.CD‐21‐0236\\n635. Guerrero‐Preston, Rafael, Filipa Godoy‐\\nVitorino, Anne Jedlicka,\\nArnold Rodríguez‐Hilario, Herminio González, Jessica Bondy,\\nFahcina Lawson, et al. 2016.“16S rRNA Amplicon Sequencing\\nIdentifies Microbiota Associated With Oral Cancer, Human\\nPapilloma Virus Infection and Surgical Treatment.” Oncotarget7:\\n51320–34. https://doi.org/10.18632/oncotarget.9710\\n636. Ren, Shengnan, Lingxin Feng, Haoran Liu, Yuke Mao,\\nZhuang Yu. 2024.“Gut Microbiome Affects the Response to\\nImmunotherapy in Non‐Small Cell Lung Cancer.” Thoracic\\nCancer 15: 1149–63. https://doi.org/10.1111/1759‐7714.15303\\n637. Lee, Pei ‐Chang, Wu, Chi‐Jung Hung, Ya‐Wen Lee, Chieh Ju\\nChi, Chen‐Ta Lee, I‐Cheng.‐Kuo Yu‐Lun, et al. 2022.“Gut\\nMicrobiota and Metabolites Associate With Outcomes of\\nImmune Checkpoint Inhibitor‐Treated Unresectable Hepa-\\ntocellular Carcinoma.” Journal for Immunotherapy of Cancer\\n10: e004779.https://doi.org/10.1136/jitc‐2022‐004779\\n638. Zhou, Xin, Lili Chen, Wanrun Lin, Wenxin Zheng,\\nHuijuan Zhang, Feng Zhou. 2024. “Diagnostic and Prog-\\nnostic Potential of the Intra‐Tumoral Microbiota Profile in\\nHPV‐Independent Endocervical Adenocarcinoma.” Frontiers\\nin Cellular and Infection Microbiology14: 1440017. https://\\ndoi.org/10.3389/fcimb.2024.1440017\\n639. Dora, David, Glen J. Weiss, Zsolt Megyesfalvi, Gabriella Gállfy,\\nEdit Dulka, Anna Kerpel‐Fronius, Judit Berta, et al. 2023.\\n“Computed Tomography‐Based Quantitative Texture Analysis\\nand Gut Microbial Community Signatures Predict Survival in\\nNon‐Small Cell Lung Cancer.” Cancers 15: 5091. https://doi.\\norg/10.3390/cancers15205091\\n640. Zhu, Chengpei, Chenchen Zhang, Shanshan Wang,\\nZiyu Xun, Dongya Zhang, Zhou Lan, Longhao Zhang,\\net al. 2024. “Characterizations of Multi ‐\\nKingdom Gut\\nMicrobiota in Immune Checkpoint Inhibitor ‐Treated\\nHepatocellular Carcinoma. ” Journal for Immunotherapy\\nof Cancer 12: e008686. https://doi.org/10.1136/jitc ‐2023‐\\n008686\\n641. Zhu, Xinhai, Ke Li, Guichao Liu, Ruan Wu, Yan Zhang,\\nSiying Wang, Meng Xu, Ligong Lu, Peng Li. 2023.“Microbial\\nMetabolite Butyrate Promotes Anti‐PD‐1 Antitumor Efficacy\\nby Modulating T Cell Receptor Signaling of Cytotoxic CD8 T\\nCell.” Gut Microbes 15: 2249143. https://doi.org/10.1080/\\n19490976.2023.2249143\\n642. Geller, Leore T., Michal Barzily ‐Rokni, Tal Danino,\\nOliver H. Jonas, Noam Shental, Deborah Nejman,\\nNancy Gavert, et al. 2017. “Potential Role of Intratumor\\nBacteria in Mediating Tumor Resistance to the Chemo-\\ntherapeutic Drug Gemcitabine. ” Science 357: 1156 –60.\\nhttps://doi.org/10.1126/science.aah5043\\n643. Coutzac, Clélia, Jean ‐Mehdi Jouniaux, Angelo Paci,\\nJulien Schmidt, Domenico Mallardo, Atmane Seck,\\nVahe Asvatourian, et al. 2020.“Systemic Short Chain Fatty\\nAcids Limit Antitumor Effect of CTLA‐4 Blockade in Hosts\\nWith Cancer.” Nature Communications11: 2168.https://doi.\\norg/10.1038/s41467‐020‐16079‐x\\n644. Wallace, Bret D., Hongwei Wang, Kimberly T. Lane, John\\nE. Scott, Jillian Orans, Ja Seol Koo, Madhukumar Venkatesh,\\net al. 2010.“Alleviating Cancer Drug Toxicity by Inhibiting a\\nBacterial Enzyme.” Science 330: 831–5. https://doi.org/10.1126/\\nscience.1191175\\n645. Richardson, Brianna N., Jolinta Lin, Zachary S. Buchwald,\\nJinbing Bai. 2022.“Skin Microbiome and Treatment‐Related\\nSkin Toxicities in Patients With Cancer: A Mini‐Review.”\\nFrontiers in Oncology 12: 924849. https://doi.org/10.3389/\\nfonc.2022.924849\\n646. Porcari, Serena, Nicolas Benech, Mireia Valles ‐Colomer,\\nNicola Segata, Antonio Gasbarrini, Giovanni Cammarota,\\nHarry Sokol, Gianluca Ianiro. 2023. “Key Determinants of\\nSuccess in Fecal Microbiota Transplantation: From Micro-\\nbiome to Clinic.” Cell Host & Microbe\\n31: 712–33. https://doi.\\norg/10.1016/j.chom.2023.03.020\\n647. Frey ‐Furtado, Leonor, Inês Magalhães, Benedita Sampaio‐\\nMaia, Maria João Azevedo. 2023.“Oral Microbiome Char-\\nacterization in Oral Mucositis Patients —A Systematic\\nReview.” Journal of Oral Pathology & Medicine: Official\\nPublication of the International Association of Oral\\nPathologists and the American Academy of Oral Pathology52:\\n911–8. https://doi.org/10.1111/jop.13492\\n648. Gui, Q.‐F., H.‐F. Lu, C.‐X. Zhang, Z.‐R. Xu, Y.‐H. Yang. 2015.\\n“Well‐Balanced Commensal Microbiota Contributes to Anti‐\\nCancer Response in a Lung Cancer Mouse Model.” Genetics\\nand Molecular Research14: 5642–51. https://doi.org/10.4238/\\n2015.May.25.16\\nMICROBIOME IN CANCER | 95 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 95, 'page_label': '96'}, page_content='649. Stolzenberg ‐Solomon, Rachael Z., Kevin W. Dodd,\\nMartin J. Blaser, Jarmo Virtamo, Philip R. Taylor,\\nDemetrius Albanes. 2003. “Tooth Loss, Pancreatic Cancer,\\nand Helicobacter pylori.” The American Journal of Clinical\\nNutrition 78: 176–81. https://doi.org/10.1093/ajcn/78.1.176\\n650. Farrell, James J., Lei Zhang, Hui Zhou, David Chia,\\nDavid Elashoff, David Akin, Bruce J. Paster,\\nKaumudi Joshipura, David T. W. Wong. 2012.“Variations of\\nOral Microbiota Are Associated With Pancreatic Diseases\\nIncluding Pancreatic Cancer.” Gut 61: 582–8. https://doi.org/\\n10.1136/gutjnl‐2011‐300784\\n651. Peters, Brandilyn A., Richard B. Hayes, Chandra Goparaju,\\nChristopher Reid, Harvey I. Pass, Jiyoung Ahn. 2019.“The\\nMicrobiome in Lung Cancer Tissue and Recurrence ‐Free\\nSurvival.” Cancer Epidemiology, Biomarkers & Prevention:\\nA Publication of the American Association for Cancer\\nResearch, Cosponsored by the American Society of Preventive\\nOncology 28: 731–40. https://doi.org/10.1158/1055‐9965.EPI‐\\n18‐0966\\n652. Hu, Min, Samuel Coleman, Muhammad Zaki\\nHidayatullah Fadlullah, DanielSpakowicz, Christine H. Chung,\\nAik Choon Tan. 2023. “Deciphering the Tumor‐Immune‐\\nMicrobe Interactions in HPV‐Negative Head and Neck Cancer.”\\nGenes 14: 1599.https://doi.org/10.3390/genes14081599\\n653. Garajová, Ingrid, Rita Balsano, Heling Wang,\\nFrancesco Leonardi, Elisa Giovannetti, Dongmei Deng,\\nGodefridus J. Peters. 2021.“The Role of the Microbiome in\\nDrug Resistance in Gastrointestinal Cancers.” Expert Review\\nof Anticancer Therapy 21: 165–76. https://doi.org/10.1080/\\n14737140.2021.1844007\\n654. Wang, Feng, Qian Yin, Liang Chen, Mark M. Davis. 2018.\\n“Bifidobacterium Can Mitigate Intestinal Immunopathology\\nin the Context Of CTLA‐4 Blockade.” Proceedings of the\\nNational Academy of Sciences115: 157–61. https://doi.org/10.\\n1073/pnas.1712901115\\n655. Klufa, Jörg, Thomas Bauer, Buck Hanson, Craig Herbold,\\nPhilipp Starkl, Beate Lichtenberger, Dagmar Srutkova, et al.\\n2019. “Hair Eruption Initiates and Commensal Skin Micro-\\nbiota Aggravate Adverse Events of Anti ‐EGFR Therapy.”\\nScience Translational Medicine11: eaax2693.https://doi.org/\\n10.1126/scitranslmed.aax2693\\n656. Pal, Sumanta K., Sierra M. Li, Xiwei Wu, Hanjun Qin,\\nMarcin Kortylewski, JoAnn Hsu, Courtney Carmichael,\\nPaul Frankel. 2015.“Stool Bacteriomic Profiling in Patients\\nWith Metastatic Renal Cell Carcinoma Receiving Vascular\\nEndothelial Growth Factor ‐Tyrosine Kinase Inhibitors. ”\\nClinical Cancer Research: An Official Journal of the American\\nAssociation for Cancer Research21: 5286–93. https://doi.org/\\n10.1158/1078‐0432.CCR‐15‐0724\\n657. Bawaneh, Alaa, Adam S. Wilson, Nicole Levi,\\nMarissa M. Howard‐McNatt, Akiko Chiba, David R. Soto‐\\nPantoja, Katherine L. Cook. 2022. “Intestinal Microbiota\\nInfluence Doxorubicin Responsiveness in Triple ‐Negative\\nBreast Cancer.” Cancers 14: 4849. https://doi.org/10.3390/\\ncancers14194849\\n658. Geng, Hong ‐Wei, Feng‐Yi Yin, Zhi‐Fa Zhang, Xu Gong,\\nYun Yang. 2021. “Butyrate Suppresses Glucose Metabolism\\nof Colorectal Cancer Cells via GPR109a ‐AKT Signaling\\nPathway and Enhances Chemotherapy. ” Frontiers in\\nMolecular Biosciences 8: 634874. https://doi.org/10.3389/\\nfmolb.2021.634874\\n659. Geller, Leore T., Ravid Straussman. 2018. “Intratumoral\\nBacteria May Elicit Chemoresistance by Metabolizing An-\\nticancer Agents.” Molecular & Cellular Oncology5: e1405139.\\nhttps://doi.org/10.1080/23723556.2017.1405139\\n660. Wang, Xinyi, Xicai Sun, Jinjin Chu, Wenchang Sun,\\nShushan Yan, Yaowen Wang. 2023. “Gut Microbiota and\\nMicrobiota ‐Derived Metabolites in Colorectal Cancer:\\nEnemy or Friend. ” World Journal of Microbiology &\\nBiotechnology 39: 291. https://doi.org/10.1007/s11274 ‐\\n023‐03742 ‐w\\n661. Zheng, Di ‐Wei, Xue Dong, Pei Pan, Ke‐\\nWei Chen, Jin‐Xuan\\nFan, Si‐Xue Cheng, Xian‐Zheng Zhang. 2019.“Phage‐Guided\\nModulation of the Gut Microbiota of Mouse Models of\\nColorectal Cancer Augments Their Responses to Chemo-\\ntherapy.” Nature Biomedical Engineering 3: 717–28. https://\\ndoi.org/10.1038/s41551‐019‐0423‐2\\n662. Lindel, Katja, Karl T. Beer, Jean Laissue, Richard H. Greiner,\\nDaniel M. Aebersold. 2001.“Human Papillomavirus Positive\\nSquamous Cell Carcinoma of the Oropharynx: A Radio-\\nsensitive Subgroup of Head and Neck Carcinoma.” Cancer\\n92: 805 –13. https://doi.org/10.1002/1097 ‐0142(20010815)\\n92:4<805::AID‐CNCR1386>3.0.CO;2‐9\\n663. Chen, Allen M., Carol Felix, Pin‐Chieh Wang, Sophia Hsu,\\nVincent Basehart, Jordan Garst, Phillip Beron, et al. 2017.\\n“Reduced‐Dose Radiotherapy for Human Papillomavirus ‐\\nAssociated Squamous‐Cell Carcinoma of the Oropharynx: A\\nSingle‐Arm, Phase 2 Study.” The Lancet. Oncology18: 803–11.\\nhttps://doi.org/10.1016/S1470‐2045(17)30246‐2\\n664. Qu, Junxing, Zhiheng Sun, Chen Peng, Daoqian Li,\\nWenyue Yan, Zhen Xu, Yayi Hou, et al. 2021.“C. tropicalis\\nPromotes Chemotherapy Resistance in Colon Cancer\\nThrough Increasing Lactate Production to Regulate the\\nMismatch Repair System.” International Journal of Biological\\nSciences 17: 2756–69. https://doi.org/10.7150/ijbs.59262\\n665. Kumpitsch, Christina, Christine Moissl ‐Eichinger,\\nJakob Pock, Dietmar Thurnher, Axel Wolf. 2020.“Prelimi-\\nnary Insights Into the Impact of Primary Radiochemotherapy\\non the Salivary Microbiome in Head and Neck Squamous\\nCell Carcinoma.” Scientific Reports10: 16582.https://doi.org/\\n10.1038/s41598‐020‐73515‐\\n0\\n666. Shuwen, Han, Yang Xi, Pan Yuefen, Xu Jiamin, Qi Quan,\\nLiao Haihong, Jiang Yizhen, Wu Wei. 2020. “Effects of\\nPostoperative Adjuvant Chemotherapy and Palliative\\nChemotherapy on the Gut Microbiome in Colorectal Can-\\ncer.” Microbial Pathogenesis 149: 104343. https://doi.org/10.\\n1016/j.micpath.2020.104343\\n667. Panebianco, Concetta, Kaarel Adamberg, Madis Jaagura,\\nMassimiliano Copetti, Andrea Fontana, Signe Adamberg,\\nKaia Kolk, et al. 2018.“Influence of Gemcitabine Chemo-\\ntherapy on the Microbiota of Pancreatic Cancer Xenografted\\nMice.” Cancer Chemotherapy and Pharmacology81: 773–82.\\nhttps://doi.org/10.1007/s00280‐018‐3549‐0\\n668. Loman, B. R., K. R. Jordan, B. Haynes, M. T. Bailey,\\nL. M. Pyter. 2019. “Chemotherapy‐Induced Neuroin-\\nflammation Is Associated With Disrupted Colonic and Bac-\\nterial Homeostasis in Female Mice. ” Scientific Reports 9:\\n16490. https://doi.org/10.1038/s41598‐019‐52893‐0\\n96 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 96, 'page_label': '97'}, page_content='669. Stringer, Andrea M., Rachel J. Gibson, Richard M. Logan,\\nJoanne M. Bowen, Ann S. J. Yeoh, Juliette Hamilton,\\nDorothy M. K. Keefe. 2009.“Gastrointestinal Microflora and\\nMucins May Play a Critical Role in the Development of\\n5‐Fluorouracil‐Induced Gastrointestinal Mucositis.” Exp Biol\\nMed (Maywood) 234: 430–41. https://doi.org/10.3181/0810‐\\nRM‐301\\n670. Stojanovska, Vanesa, Rachel M. McQuade, Sarah Fraser,\\nMonica Prakash, Shakuntla Gondalia, Rhian Stavely,\\nEnzo Palombo, et al. 2018.“Oxaliplatin‐Induced Changes in\\nMicrobiota, TLR4+ Cells and Enhanced HMGB1 Expression\\nin the Murine Colon.” PloS One 13: e0198359. https://doi.\\norg/10.1371/journal.pone.0198359\\n671. Ka źmierczak‐Siedlecka, Karolina, Nikola Bulman,\\nPaweł Ulasiński, Bartosz Kamil Sobocki, Karol Po łom,\\nLuigi Marano, Leszek Kalinowski, Karolina Skonieczna ‐\\nŻydecka. 2023. “Pharmacomicrobiomics of Cell‐Cycle Spe-\\ncific Anti‐Cancer Drugs—Is it a New Perspective for Per-\\nsonalized Treatment of Cancer Patients?” Gut Microbes 15:\\n2281017. https://doi.org/10.1080/19490976.2023.2281017\\n672. Chiba, Akiko, Alaa Bawaneh, Christine Velazquez,\\nKenysha Y. J. Clear, Adam S. Wilson, Marissa Howard ‐\\nMcNatt, Edward A. Levine, et al. 2020. “Neoadjuvant\\nChemotherapy Shifts Breast Tumor Microbiota Popula-\\ntions to Regulate Drug Responsiveness and the Develop-\\nment of Metastasis.” Molecular Cancer Research: MCR 18:\\n130–9. https://doi.org/10.1158/1541 ‐7786.MCR ‐19‐0451\\n673. Forsgård, Richard A., Vannina G. Marrachelli, Katri Korpela,\\nRafael Frias, Maria Carmen Collado, Riitta Korpela,\\nDaniel Monleon, Thomas Spillmann, Pia Österlund. 2017.\\n“Chemotherapy‐Induced Gastrointestinal Toxicity Is Associ-\\nated With Changes in Serum and Urine Metabolome and\\nFecal Microbiota in Male Sprague ‐Dawley Rats.” Cancer\\nChemotherapy and Pharmacology80: 317–32. https://doi.org/\\n10.1007/s00280‐017‐3364‐z\\n674. Bowen, Joanne M., Andrea M. Stringer, Rachel J. Gibson,\\nAnn S. J. Yeoh, Sarah Hannam, Dorothy M. K. Keefe. 2007.\\n“VSL#3 Probiotic Treatment Reduces Chemotherapy ‐\\nInduced Diarrhea and Weight Loss. ” Cancer Biology &\\nTherapy 6: 1449–54. https://doi.org/10.4161/cbt.6.9.4622\\n675. Shen, Shiqian, Grewo Lim, Zerong You, Weihua Ding,\\nPeigen Huang, Chongzhao Ran, Jason Doheny, et al. 2017.“Gut\\nMicrobiota Is Critical for the Induction of Chemotherapy‐\\nInduced Pain.” Nature Neuroscience20: 1213–6. https://doi.org/\\n10.1038/nn.4606\\n676. Ramakrishna, Chandran, Jose Corleto, Paul M. Ruegger,\\nGeoffrey D. Logan, Beth B. Peacock, Stacee Mendonca,\\nShanni Yamaki, et al. 2019.“Dominant Role of the Gut Mi-\\ncrobiota in Chemotherapy Induced Neuropathic Pain. ”\\nScientific Reports 9: 20324. https://doi.org/10.1038/s41598‐\\n019‐56832‐x\\n677. Li, Shiyu, Shuangli Zhu, Jun Yu. 2024. “The Role of Gut\\nMicrobiota and Metabolites in Cancer Chemotherapy. ”\\nJournal of Advanced Research64: 223–35. https://doi.org/10.\\n1016/j.jare.2023.11.027\\n678. Mirzaei, Sara, Milad Iranshahy, Hamid Gholamhosseinian,\\nMaryam M. Matin, Fatemeh B. Rassouli. 2022.“Urolithins\\nIncreased Anticancer Effects of Chemical Drugs, Ionizing\\nRadiation and Hyperthermia on Human Esophageal\\nCarcinoma Cells In Vitro.” Tissue & Cell77: 101846.https://\\ndoi.org/10.1016/j.tice.2022.101846\\n679. Teng, Huajing, Yan Wang, Xin Sui, Jiawen Fan, Shuai Li,\\nXiao Lei, Chen Shi, et al. 2023.“Gut Microbiota‐Mediated\\nNucleotide Synthesis Attenuates the Response to Neoadju-\\nvant Chemoradiotherapy in Rectal Cancer.” Cancer Cell 41:\\n124–38.e6. https://doi.org/10.1016/j.ccell.2022.11.013\\n680. Linn, Ye Htut, K. Khine Thu, Nang Hla Hla Win. 2019.\\n“Effect of Probiotics for the Prevention of Acute Radiation‐\\nInduced Diarrhoea Among Cervical Cancer Patients: A\\nRandomized Double ‐Blind Placebo ‐Controlled Study. ”\\nProbiotics and Antimicrobial Proteins11: 638–47. https://doi.\\norg/10.1007/s12602‐018‐9408‐9\\n681. Blake, Stephen J., Yochai Wolf, Ben Boursi, Lynn, David J.\\n2024. “Role of the Microbiota in Response to and Recovery\\nFrom Cancer Therapy. ” Nature Reviews Immunology 24:\\n308–25. https://doi.org/10.1038/s41577‐023‐00951‐0\\n682. Li, Zongjuan, Yang Zhang, Weifeng Hong, Biao Wang,\\nYixing Chen, Ping Yang, Jian Zhou, et al. 2022.“Gut Mi-\\ncrobiota Modulate Radiotherapy ‐Associated Antitumor\\nImmune Responses Against Hepatocellular Carcinoma via\\nSTING Signaling.” Gut Microbes14: 2119055.https://doi.org/\\n10.1080/19490976.2022.2119055\\n683. Shi, Wei, Lijun Shen, Wei Zou, Jingwen Wang, Jianing Yang,\\nYuezhu Wang, Bingdong Liu, et al. 2020.“The Gut Micro-\\nbiome Is Associated With Therapeutic Responses and Toxi-\\ncities of Neoadjuvant Chemoradiotherapy in Rectal Cancer\\nPatients—A Pilot Study.” Frontiers in Cellular and Infection\\nMicrobiology 10: 562463. https://doi.org/10.3389/fcimb.2020.\\n562463\\n684. Yi, Yuxi, Lijun Shen, Wei Shi, Fan Xia, Hui Zhang,\\nYan Wang, Jing Zhang, et al. 2021.“Gut Microbiome Com-\\nponents Predict Response to Neoadjuvant Chemor-\\nadiotherapy in Patients With Locally Advanced Rectal\\nCancer: A Prospective, Longitudinal Study.” Clinical Cancer\\nResearch: An Official Journal of the American Association for\\nCancer Research 27: 1329–40. https://doi.org/10.1158/1078‐\\n0432.CCR‐20‐3445\\n685. Chen, Linda, Nadeem Riaz, Nancy Lee, Sean McBride. 2021.\\n“Current Considerations for Radiotherapy in HPV ‐\\nAssociated Head and Neck Cancer. ” Journal of Surgical\\nOncology 124: 945–51. https://doi.org/10.1002/jso.26689\\n686. Shiao, Stephen L., Kathleen M. Kershaw, Jose J. Limon,\\nSungyong You, Junhee Yoon, Emily Y. Ko, Jlenia Guarnerio,\\net al. 2021. “Commensal Bacteria and Fungi Differentially\\nRegulate Tumor Responses to Radiation Therapy.” Cancer\\nCell 39: 1202 –13.e6. https://doi.org/10.1016/j.ccell.2021.\\n07.002\\n687. Mougeot, Jean ‐Luc C., Craig B. Stevens, Kathryn G. Almon,\\nBruce J. Paster, Rajesh V. Lalla, Michael T. Brennan,\\nFarah Bahrani Mougeot. 2019.“\\nCaries‐Associated Oral Mi-\\ncrobiome in Head and Neck Cancer Radiation Patients: A\\nLongitudinal Study. ” Journal of Oral Microbiology 11:\\n1586421. https://doi.org/10.1080/20002297.2019.1586421\\n688. Arrifin, Azirrawani, Ellie Heidari, Mary Burke,\\nMichael R. Fenlon, Avijit Banerjee. 2018. “The Effect of\\nRadiotherapy for Treatment of Head and Neck Cancer on\\nOral Flora and Saliva.” Oral Health & Preventive Dentistry16:\\n425–9. https://doi.org/10.3290/j.ohpd.a41364\\nMICROBIOME IN CANCER | 97 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 97, 'page_label': '98'}, page_content='689. Danckaert, Willeke, Mathieu Spaas, Nora Sundahl,\\nAurélie De Bruycker, Valérie Fonteyne, Ellen De Paepe,\\nCarlos De Wagter, Lynn Vanhaecke, Piet Ost. 2023.“Micro-\\nbiome and Metabolome Dynamics During Radiotherapy for\\nProstate Cancer.” Radiotherapy and Oncology 189: 109950.\\nhttps://doi.org/10.1016/j.radonc.2023.109950\\n690. Reis Ferreira, Miguel, H. Jervoise N. Andreyev, Kabir\\nMohammed, Lesley Truelove, Sharon M. Gowan, Jia Li,\\nSarah L. Gulliford, Julian R. Marchesi, David P. Dearnaley.\\n2019. “Microbiota‐ and Radiotherapy‐Induced Gastrointestinal\\nSide‐Effects (MARS) Study: A Large Pilot Study of the Micro-\\nbiome in Acute and Late‐Radiation Enteropathy.” Clinical\\nCancer Research: An Official Journal of the American Association\\nf o rC a n c e rR e s e a r c h25: 6487–500. https://doi.org/10.1158/1078‐\\n0432.CCR‐19‐0960\\n691. Olivas, Andrea D., Benjamin D. Shogan, Vesta Valuckaite,\\nAlexander Zaborin, Natalya Belogortseva, Mark Musch,\\nFolker Meyer, et al. 2012.“Intestinal Tissues Induce an SNP\\nMutation in Pseudomonas aeruginosa That Enhances Its\\nVirulence: Possible Role in Anastomotic Leak.” PloS One 7:\\ne44326. https://doi.org/10.1371/journal.pone.0044326\\n692. Bahig, Houda, Clifton D. Fuller, Aparna Mitra,\\nKyoko Yoshida ‐Court, Travis Solley, Sweet Ping Ng,\\nIbrahim Abu‐Gheida, et al. 2021.“Longitudinal Characteri-\\nzation of the Tumoral Microbiome During Radiotherapy in\\nHPV‐Associated Oropharynx Cancer. ” Clinical and\\nTranslational Radiation Oncology 26: 98–103. https://doi.\\norg/10.1016/j.ctro.2020.11.007\\n693. Reis Ferreira, Miguel, H. Jervoise N. Andreyev,\\nKabir Mohammed, Lesley Truelove, Sharon M. Gowan,\\nJia Li, Sarah L. Gulliford, Julian R. Marchesi, David P.\\nDearnaley. 2019. “Microbiota‐ and Radiotherapy‐Induced\\nGastrointestinal Side‐Effects (MARS) Study: A Large Pilot\\nStudy of the Microbiome in Acute and Late‐Radiation En-\\nteropathy.” Clinical Cancer Research: An Official Journal of\\nthe American Association for Cancer Research25: 6487–500.\\nhttps://doi.org/10.1158/1078‐0432.CCR‐19\\n‐0960\\n694. Ciorba, Matthew A., Terrence E. Riehl, M. Suprada Rao,\\nClara Moon, Xueping Ee, Gerardo M. Nava, Monica R.\\nWalker, et al. 2012.“Lactobacillus Probiotic Protects Intes-\\ntinal Epithelium From Radiation Injury in a TLR‐2/Cyclo‐\\nOxygenase‐2‐Dependent Manner.” Gut 61: 829–38. https://\\ndoi.org/10.1136/gutjnl‐2011‐300367\\n695. Zhao, Zhenguo, Wei Cheng, Wei Qu, Guoyi Shao,\\nShuanghai Liu. 2020. “Antibiotic Alleviates Radiation ‐\\nInduced Intestinal Injury by Remodeling Microbiota,\\nReducing Inflammation, and Inhibiting Fibrosis. ” ACS\\nOmega 5: 2967 –77. https://doi.org/10.1021/acsomega.\\n9b03906\\n696. Abdollahi, Elham, Thomas P. Johnston, Zahra Ghaneifar,\\nP a r v i zV a h e d i ,P o u y aG o l e i j ,S a r aA z h d a r i ,A b b a s\\nShapouri Moghaddam. 2023. “Immunomodulatory Ther-\\napeutic Effects of Curcumin on M1/M2 Macrophage\\nPolarization in Inflammatory Diseases. ” Current\\nMolecular Pharmacology 16: 2 –14. https://doi.org/10.\\n2174/1874467215666220324114624\\n697. Jia, Dingjiacheng, Qiwen Wang, Yadong Qi, Yao Jiang,\\nJiamin He, YIFENG Lin, Yong Sun, et al. 2024.“Microbial\\nMetabolite Enhances Immunotherapy Efficacy by\\nModulating T Cell Stemness in Pan ‐Cancer.” Cell 187:\\n1651–65.e21. https://doi.org/10.1016/j.cell.2024.02.022\\n698. McCulloch, John A., Diwakar Davar, Richard R. Rodrigues,\\nJonathan H. Badger, Jennifer R. Fang, Alicia M. Cole,\\nAscharya K. Balaji, et al. 2022.“Intestinal Microbiota Sig-\\nnatures of Clinical Response and Immune‐Related Adverse\\nEvents in Melanoma Patients Treated With Anti ‐PD‐1.”\\nNature Medicine 28: 545–56. https://doi.org/10.1038/s41591‐\\n022‐01698‐2\\n699. Gunjur, Ashray, Yan Shao, Timothy Rozday, Oliver Klein,\\nAndre Mu, Bastiaan W. Haak, Ben Markman, et al. 2024.“A\\nGut Microbial Signature For Combination Immune Check-\\npoint Blockade Across Cancer Types.” Nature Medicine 30:\\n797–809. https://doi.org/10.1038/s41591‐024‐02823‐z\\n700. Björk, Johannes R., Laura A. Bolte, Andrew Maltez Thomas,\\nKarla A. Lee, Niccolo Rossi, Thijs T. Wind, Lotte M. Smit,\\net al. 2024. “Longitudinal Gut Microbiome Changes in\\nImmune Checkpoint Blockade ‐Treated Advanced Mela-\\nnoma.” Nature Medicine30: 785–96. https://doi.org/10.1038/\\ns41591‐024‐02803‐3\\n701. Lee, Karla A., Andrew Maltez Thomas, Laura A. Bolte,\\nJohannes R. Björk, Laura Kist De Ruijter, Federica Armanini,\\nFrancesco Asnicar, et al. 2022. “Cross‐Cohort Gut Micro-\\nbiome Associations With immune Checkpoint Inhibitor\\nResponse in Advanced Melanoma.” Nature Medicine28: 535–\\n44. https://doi.org/10.1038/s41591‐022‐01695‐5\\n702. Chaput, N., P. Lepage, C. Coutzac, E. Soularue, K. Le Roux,\\nC. Monot, L. Boselli, et al. 2017.“Baseline Gut Microbiota\\nPredicts Clinical Response and Colitis in Metastatic Melanoma\\nPatients Treated With Ipilimumab.” Annals of Oncology 28:\\n1368–79. https://doi.org/10.1093/annonc/mdx108\\n703. Dubin, Krista, Margaret K. Callahan, Boyu Ren, Raya Khanin,\\nAgnes Viale, Lilan Ling, Daniel No, et al. 2016.“Intestinal Mi-\\ncrobiome Analyses Identify Melanoma patients at Risk for\\nCheckpoint‐Blockade‐Induced Colitis.” Nature Communications\\n7: 10391.https://doi.org/10.1038/ncomms10391\\n704. Frankel, Arthur E., Laura A. Coughlin, Jiwoong Kim,\\nThomas W. Froehlich, Yang Xie, Eugene P. Frenkel,\\nAndrew Y. Koh. 2017. “Metagenomic Shotgun Sequencing\\nand Unbiased Metabolomic Profiling Identify Specific\\nHuman Gut Microbiota and Metabolites Associated With\\nImmune Checkpoint Therapy Efficacy in Melanoma Pa-\\ntients.” Neoplasia 19: 848–55. https://doi.org/10.1016/j.neo.\\n2017.08.004\\n705. Huang, Xiaowen, Muni Hu, Tiantian Sun, Jiantao Li,\\nYilu Zhou, Yuqing Yan, Baoqin Xuan, et al. 2023.“Multi‐\\nKingdom Gut Microbiota Analyses Define Bacterial–\\nFungal\\nInterplay and Microbial Markers of Pan‐Cancer Immuno-\\ntherapy Across Cohorts.” Cell host & microbe31: 1930–43.e4.\\nhttps://doi.org/10.1016/j.chom.2023.10.005\\n706. Deng, Weiwei, Zhen Su, Panpan Liang, Yubo Ma,\\nYufang Liu, Kai Zhang, Yi Zhang, et al. 2021.“Single‐Cell\\nImmune Checkpoint Landscape of PBMCs Stimulated With\\nCandida albicans.” Emerging Microbes & Infections10: 1272–\\n83. https://doi.org/10.1080/22221751.2021.1942228\\n707. Prasad, Rishika, Abdur Rehman, Lubna Rehman, Faezeh\\nDarbaniyan, Viktoria Blumenberg, Maria ‐Luisa Schubert,\\nUria Mor, et al. 2025.“Antibiotic‐Induced Loss of Gut Mi-\\ncrobiome Metabolic Output Correlates With Clinical\\n98 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 98, 'page_label': '99'}, page_content='Responses to CAR T ‐Cell Therapy.” Blood 145: 823–39.\\nhttps://doi.org/10.1182/blood.2024025366\\n708. DeFilipp, Zachariah, Marcela V. Maus. 2023. “Linking the\\nMicrobiome to CAR‐T Cell Responses.” Nature Medicine 29:\\n785–6. https://doi.org/10.1038/s41591‐023‐02272‐0\\n709. CD19 CAR T‐Cell Clinical Outcome Is Associated With the\\nGut Microbiome. 2022. Cancer Discovery 12: 1182. https://\\ndoi.org/10.1158/2159‐8290.CD‐RW2022‐052\\n710. Smith, Melody, Anqi Dai, Guido Ghilardi, Kimberly V.\\nAmelsberg, Sean M. Devlin, R aymone Pajarillo, John B.\\nSlingerland, et al. 2022. “Gut Microbiome Correlates of\\nResponse and Toxicity Following Anti‐CD19 CAR T Cell\\nTherapy.” Nature Medicine28: 713–23. https://doi.org/10.1038/\\ns41591‐022‐01702‐9\\n711. Abid, Muhammad Bilal, Nirav N. Shah, Theresa C. Maatman,\\nParameswaran N. Hari. 2019.“Gut Microbiome and CAR‐T\\nTherapy.” Experimental Hematology & Oncology8: 31.https://\\ndoi.org/10.1186/s40164‐019‐0155‐8\\n712. Shen, Junyi, Rong Hu, Anqi Lin, Aimin Jiang, Bufu Tang,\\nZaoqu Liu, Quan Cheng, et al. 2024.“Characterization of?Sec-\\nond Primary Malignancies Post CAR T‐Cell Therapy: Real‐\\nWorld Insights From the Two Global Pharmacovigilance Da-\\ntabases of FAERS and VigiBase.” EClinicalMedicine73: 102684.\\nhttps://doi.org/10.1016/j.eclinm.2024.102684\\n713. Hu, Yongxian, Jingjing Li, Fang Ni, Zhongli Yang,\\nXiaohua Gui, Zhiwei Bao, Houli Zhao, et al. 2022.“CAR‐T\\nCell Therapy ‐Related Cytokine Release Syndrome and\\nTherapeutic Response Is Modulated by the Gut Microbiome\\nin Hematologic Malignancies.” Nature Communications 13:\\n5313. https://doi.org/10.1038/s41467‐022‐32960‐3\\n714. Stergiopoulos, Georgios M., Ianko Iankov, Evanthia Galanis.\\n2024. “Personalizing Oncolytic Immunovirotherapy Ap-\\nproaches.” Molecular Diagnosis & Therapy 28: 153 –68.\\nhttps://doi.org/10.1007/s40291‐023‐00689‐4\\n715. Sivan, Ayelet, Leticia Corrales, Nathaniel Hubert, Jason B.\\nWilliams, Keston Aquino‐Michaels, Zachary M. Earley, Franco\\nW. Benyamin, et al. 2015.“Commensal Bifidobacterium Pro-\\nmotes Antitumor Immunity and Facilitates Anti‐PD‐L1 Effi-\\ncacy.” Science 350: 1084–9. https://doi.org/10.1126/science.\\naac4255\\n716. Vétizou, Marie, Jonathan M. Pitt, Romain Daillère,\\nPatricia Lepage, Nadine Waldschmitt, Caroline Flament,\\nSylvie Rusakiewicz, et al. 2015.“Anticancer Immunotherapy\\nby CTLA‐4 Blockade Relies on the Gut Microbiota.” Science\\n350: 1079–84. https://doi.org/10.1126/science.aad1329\\n717. Mager, Lukas F., Regula Burkhard, Nicola Pett,\\nNoah C. A. Cooke, Kirsty Brown, Hena Ramay, Seungil Paik,\\net al. 2020. “Microbiome‐Derived Inosine Modulates\\nResponse to Checkpoint Inhibitor Immunotherapy.” Science\\n369: 1481–9. https://doi.org/10.1126/science.abc3421\\n718. Ravilla, Rahul, Hannah N. Coleman, Cheryl‐Emiliane Chow,\\nLuisa Chan, Barbara J. Fuhrman, William W. Greenfield,\\nMichael Scott Robeson, et al. 2019.“Cervical Microbiome and\\nResponse to a Human Papillomavirus Therapeutic Vaccine for\\nTreating High ‐Grade Cervical Squamous Intraepithelial\\nLesion.” Integrative Cancer Therapies18: 1534735419893063.\\nhttps://doi.org/10.1177/1534735419893063\\n719. Guardamagna, Mora, Miguel\\n‐Angel Berciano ‐Guerrero,\\nBeatriz Villaescusa ‐González, Elisabeth Perez ‐Ruiz,\\nJavier Oliver, Rocío Lavado ‐Valenzuela, Antonio Rueda ‐\\nDominguez, Isabel Barragán, María Isabel Queipo‐Ortuño.\\n2022. “Gut Microbiota and Therapy in Metastatic Melanoma:\\nFocus on MAPK Pathway Inhibition.” International Journal\\nof Molecular Sciences 23: 11990. https://doi.org/10.3390/\\nijms231911990\\n720. Mu, Wenxin, Yiqun Jia, Xiaobing Chen, Haoyu Li, Zhi Wang,\\nBin Cheng 2020. “Intracellular Porphyromonas gingivalis\\nPromotes the Proliferation of Colorectal Cancer Cells via the\\nMAPK/ERK Signaling Pathway.” Frontiers in Cellular and\\nInfection Microbiology 10: 584798. https://doi.org/10.3389/\\nfcimb.2020.584798\\n721. Shan, Khine S., Tauseef U. Rehman, Stan Ivanov,\\nGelenis Domingo, Luis E. Raez. 2024.“Molecular Targeting\\nof the BRAF Proto ‐Oncogene/Mitogen‐Activated Protein\\nKinase (MAPK) Pathway Across Cancers. ” International\\nJournal of Molecular Sciences 25: 624. https://doi.org/10.\\n3390/ijms25010624\\n722. Kumar, Mukesh, Ramanpreet Kaur, Shruthi Kanthaje,\\nRadha K. Dhiman, Anuradha Chakraborti. 2023.“Bacterial\\nMetabolite Butyrate in Modulating Sorafenib‐Targeted Mi-\\ncroRNAs to Curtail Its Resistance in Hepatocellular Carci-\\nnoma.” Journal of Cancer Research and Clinical Oncology\\n149: 5823–39. https://doi.org/10.1007/s00432‐022‐04544‐7\\n723. Yu, Jingjing, Xiaoping Chen, Xiangliang Yang, Bixiang\\nZhang. 2024. “Understanding Gut Dysbiosis for Hepato-\\ncellular Carcinoma Diagnosis and Treatment. ” Trends in\\nEndocrinology & Metabolism35: 1006–20. https://doi.org/10.\\n1016/j.tem.2024.06.003\\n724. Ahmed, Lamiaa A., Khaled F. Al‐Massri. 2023. “Gut Micro-\\nbiota Modulation for Therapeutic Management of Various\\nDiseases: A New Perspective Using Stem Cell Therapy. ”\\nCurrent Molecular Pharmacology 16: 43–59. https://doi.org/\\n10.2174/1874467215666220222105004\\n725. Hahn, Andrew W., Camryn Froerer, Sidney VanAlstine,\\nNityam Rathi, Erin B. Bailey, David D. Stenehjem,\\nNeeraj Agarwal. 2018. “Targeting Bacteroides in Stool Mi-\\ncrobiome and Response to Treatment With First‐Line VEGF\\nTyrosine Kinase Inhibitors in Metastatic Renal‐Cell Carci-\\nnoma.” Clinical Genitourinary Cancer16: 365–8. https://doi.\\norg/10.1016/j.clgc.2018.05.001\\n726. Ianiro, Gianluca, Ernesto Rossi, Andrew M. Thomas,\\nGiovanni Schinzari, Luca Masucci, Gianluca Quaranta,\\nCarlo Romano Settanni, et al. 2020. “Faecal Microbiota\\nTransplantation for the Treatment of Diarrhoea Induced by\\nTyrosine‐Kinase Inhibitors in Patients With Metastatic Renal\\nCell Carcinoma.” Nature Communications 11: 4333. https://\\ndoi.org/10.1038/s41467‐020‐18127‐y\\n727. Van Praagh, Jasper B., Marcus C. De Goffau, Ilsalien S.\\nBakker, Harry Van Goor, Hermie J. M. Harmsen, Peter\\nOlinga, Klaas Havenga. 2019.“Mucus Microbiome of Anas-\\ntomotic Tissue During Surgery Has Predictive Value for\\nColorectal Anastomotic Leakage.” Annals of Surgery 269:\\n911–6. https://doi.org/10.1097/SLA.0000000000002651\\n728. Rollins, Katie E., Hannah Javanmard‐Emamghissi, Austin G.\\nAcheson, Dileep N. Lobo. 2019.“The Role of Oral Antibiotic\\nPreparation in Elective Colorectal Surgery: A Meta‐analysis.”\\nAnnals of Surgery 270: 43–58. https://doi.org/10.1097/SLA.\\n0000000000003145\\nMICROBIOME IN CANCER | 99 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 99, 'page_label': '100'}, page_content='729. Darbandi, Atieh, Maryam Mirshekar, Aref Shariati,\\nMajid Taati Moghadam, Vahid Lohrasbi, Parisa Asadolahi,\\nMalihe Talebi. 2020.“The Effects of Probiotics on Reducing\\nthe Colorectal Cancer Surgery Complications: A Periodic\\nReview During 2007–2017.” Clinical Nutrition 39: 2358–67.\\nhttps://doi.org/10.1016/j.clnu.2019.11.008\\n730. Kiran, Ravi Pokala, Alice C. A. Murray, Cody Chiuzan,\\nDavid Estrada, Kenneth Forde. 2015. “Combined Pre-\\noperative Mechanical Bowel Preparation With Oral Anti-\\nbiotics Significantly Reduces Surgical Site Infection,\\nAnastomotic Leak, and Ileus After Colorectal Surgery. ”\\nAnnals of Surgery262: 416–25; discussion 423–5. https://doi.\\norg/10.1097/SLA.0000000000001416\\n731. Cong, Jing, Hua Zhu, Dong Liu, Tianjun Li, Chuantao\\nZhang, Jingjuan Zhu, Hongying Lv, et al. 2018. “A Pilot\\nStudy: Changes of Gut Microbiota in Post‐Surgery Colorectal\\nCancer Patients.” Frontiers in Microbiology 9: 2777. https://\\ndoi.org/10.3389/fmicb.2018.02777\\n732. Lapthorne, Susan. 2015. “Changes in the Colon\\nMicrobiota and Intestinal Cytokine Gene Expression\\nFollowing Minimal Intestinal Surgery. ” World Journal of\\nGastroenterology 21: 4150–8. https://doi.org/10.3748/wjg.\\nv21.i14.4150\\n733. Kong, Cheng, Renyuan Gao, Xuebing Yan, Linsheng Huang,\\nJide He, Hao Li, Jie You, Huanlong Qin. 2019.“Alterations in\\nIntestinal Microbiota of Colorectal Cancer Patients Receiving\\nRadical Surgery Combined With Adjuvant CapeOx Therapy.”\\nScience China. Life Sciences62: 1178–93. https://doi.org/10.\\n1007/s11427‐018‐9456‐x\\n734. Peng, Yu ‐Chong, Xin‐Hua Zhao, Chuan‐Fa Zeng, Jing‐Xuan\\nXu, Lu‐Nan Qi, Le‐Qun Li. 2022.“Integrated Omics Analysis:\\nThe Relationship Between Significantly Increased Klebsiella\\nPost‐Hepatectomy and Decreased Hub‐Metabolite 3‐Methyl‐\\n2‐Oxobutanoic Acid Is Associated With Induced Liver Fail-\\nure.”\\nJournal of Gastrointestinal Oncology13: 326–43. https://\\ndoi.org/10.21037/jgo‐21‐906\\n735. Yang, Jie, Yuhua He, Xi Liao, Jiankun Hu, Ka Li. 2023.“Does\\nPostoperative Pulmonary Infection Correlate With Intestinal\\nFlora Following Gastric Cancer Surgery?—A Nested Case‐\\nControl Study. ” Frontiers in Microbiology 14: 1267750.\\nhttps://doi.org/10.3389/fmicb.2023.1267750\\n736. Xu, Fusheng, Zhiming Yu, Yaru Liu, Ting Du, Leilei Yu,\\nFengwei Tian, Wei Chen, Qixiao Zhai. 2023.“A High‐Fat,\\nHigh‐Cholesterol Diet Promotes Intestinal Inflammation by\\nExacerbating Gut Microbiome Dysbiosis and Bile Acid Dis-\\norders in Cholecystectomy.” Nutrients 15: 3829. https://doi.\\norg/10.3390/nu15173829\\n737. Poutahidis, Theofilos, Sean M. Kearney, Tatiana Levkovich,\\nPeimin Qi, Bernard J. Varian, Jessica R. Lakritz,\\nYassin M. Ibrahim, et al. 2013. “Microbial Symbionts\\nAccelerate Wound Healing via the Neuropeptide Hormone\\nOxytocin.” PloS One 8: e78898. https://doi.org/10.1371/\\njournal.pone.0078898\\n738. Obermüller, Beate, Georg Singer, Bernhard Kienesberger,\\nBarbara Mittl, Vanessa Stadlbauer, Angela Horvath,\\nWolfram Miekisch, et al. 2023.“Probiotic OMNi‐BiOTiC® 10\\nAAD Reduces Cyclophosphamide ‐Induced Inflammation\\nand Adipose Tissue Wasting in Mice.” Nutrients 15: 3655.\\nhttps://doi.org/10.3390/nu15163655\\n739. Yazdi, Mohammad Hossein, Mehdi Mahdavi, Neda Setayesh,\\nMohammad Esfandyar, Ahmad Reza Shahverdi. 2013.\\n“Selenium Nanoparticle‐Enriched Lactobacillus brevisCauses\\nMore Efficient Immune Responses In Vivo and Reduces the\\nLiver Metastasis in Metastatic Form of Mouse Breast\\nCancer.” DARU Journal of Pharmaceutical Sciences 21: 33.\\nhttps://doi.org/10.1186/2008‐2231‐21‐33\\n740. Xu, Fuqiang, Qiaoqiao Li, Shuyang Wang, Miaoyin Dong,\\nGuoqing Xiao, Jin Bai, Junkai Wang, Xisi Sun. 2023.“The\\nEfficacy of Prevention for Colon Cancer Based on the Mi-\\ncrobiota Therapy and the Antitumor Mechanisms With\\nIntervention of Dietary Lactobacillus. ” Microbiology\\nSpectrum 11: e0018923. https://doi.org/10.1128/spectrum.\\n00189‐23\\n741. Zeighamy Alamdary, Shabnam, Shahnaz Halimi,\\nAkram Rezaei, Roghayeh Afifirad. 2023. “Association\\nBetween Probiotics and Modulation of Gut Microbial Com-\\nmunity Composition in Colorectal Cancer Animal Models: A\\nSystematic Review (2010–2021).”\\nThe Canadian Journal of\\nInfectious Diseases & Medical Microbiology = Journal\\nCanadien Des Maladies Infectieuses Et De La Microbiologie\\nMedicale 2023: 3571184. https://doi.org/10.1155/2023/\\n3571184\\n742. Wei, Hao, Zhiying Yue, Jialong Han, Ping Chen, Ke Xie,\\nYu Sun, Jiang Zhu. 2024.“Oral Compound Probiotic Sup-\\nplements Can Improve the Quality of Life for Patients With\\nLung Cancer During Chemotherapy: A Randomized Placebo‐\\nControlled Study.” Thoracic Cancer 15: 182–91. https://doi.\\norg/10.1111/1759‐7714.15177\\n743. Shi, Qi, Jia Wang, Mengnan Zhou, Rui Zheng, Xiaoli Zhang,\\nBeixing Liu. 2023. “Gut Lactobacillus Contribute to the\\nProgression of Breast Cancer by Affecting the Anti‐Tumor\\nActivities of Immune Cells in the TME of Tumor‐Bearing\\nMice.” International Immunopharmacology 124: 111039.\\nhttps://doi.org/10.1016/j.intimp.2023.111039\\n744. Jang, Albert, Jake N. Lichterman, Jeffrey Y. Zhong,\\nJonathan E. Shoag, Jorge A. Garcia, Tian Zhang,\\nPedro C. Barata. 2023.“Immune Approaches Beyond Tradi-\\ntional Immune Checkpoint Inhibitors for Advanced Renal\\nCell Carcinoma.” Human Vaccines & Immunotherapeutics19:\\n2276629. https://doi.org/10.1080/21645515.2023.2276629\\n745. Lin, Anqi, Aimin Jiang, Lihaoyun Huang, Yu Li,\\nChunyanx Zhang, Lingxuan Zhu, Weiming Mou, et al. 2025.\\n“From Chaos to Order: Optimizing Fecal Microbiota Trans-\\nplantation for Enhanced Immune Checkpoint Inhibitors\\nEfficacy.” Gut Microbes17: 2452277.https://doi.org/10.1080/\\n19490976.2025.2452277\\n746. Wang, Xinjun, Di Zhao, Dexi Bi, Long Li, Hongliang Tian,\\nFang Yin, Tao Zuo, et al. 2025. “Fecal Microbiota Trans-\\nplantation: Transitioning From Chaos and Controversial\\nRealm to Scientific Precision Era.” Science Bulletin 70: 970‐\\n985. https://doi.org/10.1016/j.scib.2025.01.029\\n747. Kim, Do‐Yeon, So‐Yeon Lee, Jae‐Yun Lee, Tae Woong Whon,\\nJune‐Young Lee, Che Ok Jeon, Jin‐Woo Bae. 2024. “Gut\\nMicrobiome Therapy: Fecal Microbiota Transplantation vs\\nLive Biotherapeutic Products.” Gut Microbes 16: 2412376.\\nhttps://doi.org/10.1080/19490976.2024.2412376\\n748. Yang, Yunwei, Yaping An, Yue Dong, Qiao Chu, Jingge Wei,\\nBangmao Wang, Hailong Cao. 2024. “Fecal Microbiota\\n100 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 100, 'page_label': '101'}, page_content=\"Transplantation: No Longer Cinderella in Tumour Immu-\\nnotherapy.” EBioMedicine 100: 104967. https://doi.org/10.\\n1016/j.ebiom.2024.104967\\n749. Yadegar, Abbas, Haggai Bar‐Yoseph, Tanya Marie Monaghan,\\nSepideh Pakpour, Andrea Severino, Ed J. Kuijper,\\nWiep Klaas Smits, et al. 2024.“Fecal Microbiota Transplanta-\\ntion: Current Challenges and Future Landscapes.” Clinical\\nMicrobiology Reviews37: e0006022.https://doi.org/10.1128/cmr.\\n00060‐22\\n750. Cao, Yanyan, Lijie Zhang, Fu Xiong, Xiaopeng Guo,\\nXuefeng Kan, Songlin Song, Bo Liang, et al. 2024.“Effect of\\nProbiotics and Fecal Microbiota Transplantation in Dirty\\nRats With Established Primary Liver Cancer. ” Future\\nMicrobiology 19: 117 –29. https://doi.org/10.2217/fmb ‐\\n2022‐0234\\n751. Cheng, Xiaoshuo, Xiaozheng Li, Xudong Yang,\\nShaojun Fang, Zhenyu Wang, Tingting Liu, Mengyao Zheng,\\net al. 2023. “Successful Treatment of pMMR MSS IVB\\nColorectal Cancer Using Anti‐VEGF and Anti‐PD‐1 Therapy\\nin Combination of Gut Microbiota Transplantation: A Case\\nReport.” Cureus 15: e42347. https://doi.org/10.7759/cureus.\\n42347\\n752. Cui, Ming, Huiwen Xiao, YuanLi, Lixin Zhou, Shuyi Zhao,\\nDan Luo, Qisheng Zheng, et al. 2017. “Faecal Microbiota\\nTransplantation Protects Against Radiation‐Induced Toxicity.”\\nEMBO Molecular Medicine9: 448–61. https://doi.org/10.15252/\\nemmm.201606932\\n753. Lythgoe, Mark P., Rohma Ghani, Benjamin H. Mullish,\\nJulian R. Marchesi, Jonathan Krell. 2022.“The Potential of\\nFecal Microbiota Transplantation in Oncology.” Trends in\\nMicrobiology 30: 10 –2. https://doi.org/10.1016/j.tim.2021.\\n10.003\\n754. Ninkov, Marina, Crystal L. Schmerk, Manoosh Moradizadeh,\\nSeema N. Parvathy, Rene Figueredo, Jeremy P. Burton,\\nMichael S. Silverman, et al. 2023. “Improved MAIT Cell\\nFunctions Following Fecal Microbiota Transplantation for\\nMetastatic Renal Cell Carcinoma. ” Cancer Immunology,\\nImmunotherapy 72: 1247 –60. https://doi.org/10.1007/\\ns00262‐022\\n‐03329‐8\\n755. Hefazi, Mehrdad, Mrinal M. Patnaik, William J. Hogan,\\nMark R. Litzow, Darrell S. Pardi, Sahil Khanna. 2017.“Safety\\nand Efficacy of Fecal Microbiota Transplant for Recurrent\\nClostridium Difficile Infection in Patients With Cancer\\nTreated With Cytotoxic Chemotherapy: A Single‐Institution\\nRetrospective Case Series. ” Mayo Clinic Proceedings 92:\\n1617–24. https://doi.org/10.1016/j.mayocp.2017.08.016\\n756. Halsey, Taylor M., Anusha S.Thomas, Tomo Hayase, Weijie\\nMa, Hamzah Abu‐Sbeih, Baohua Sun, Edwin Roger Parra, et al.\\n2023. “Microbiome Alteration via Fecal Microbiota Transplan-\\ntation Is Effective for Refractory Immune Checkpoint Inhibitor‐\\nInduced Colitis.” Science Translational Medicine15: eabq4006.\\nhttps://doi.org/10.1126/scitranslmed.abq4006\\n757. Baruch, Erez N., Ilan Youngster, Guy Ben ‐Betzalel,\\nRona Ortenberg, Adi Lahat, Lior Katz, Katerina Adler, et al.\\n2021. “Fecal Microbiota Transplant Promotes Response in\\nImmunotherapy‐Refractory Melanoma Patients. ” Science\\n371: 602–9. https://doi.org/10.1126/science.abb5920\\n758. Davar, Diwakar, Amiran K. Dzutsev, John A. McCulloch,\\nRichard R. Rodrigues, Joe ‐Marc Chauvin, Robert M.\\nMorrison, Richelle N. Deblasio, et al. 2021.“Fecal Microbiota\\nTransplant Overcomes Resistance to Anti‐PD‐1 Therapy in\\nMelanoma Patients.” Science 371: 595–602. https://doi.org/\\n10.1126/science.abf3363\\n759. Routy, Bertrand, John G. Lenehan, Wilson H. Miller,\\nRahima Jamal, Meriem Messaoudene, Brendan A. Daisley,\\nCecilia Hes, et al. 2023.“Fecal Microbiota Transplantation\\nPlus Anti‐PD‐1 Immunotherapy in Advanced Melanoma: A\\nphase I Trial.” Nature Medicine 29: 2121–32. https://doi.org/\\n10.1038/s41591‐023‐02453‐x\\n760. Wang, Yinghong, Diana H. Wiesnoski, Beth A. Helmink,\\nVancheswaran Gopalakrishnan, Kati Choi, Hebert\\nL. DuPont, Zhi‐Dong Jiang, et al. 2018. “Fecal Microbiota\\nTransplantation for Refractory Immune Checkpoint\\nInhibitor‐Associated Colitis.” Nature Medicine 24: 1804–8.\\nhttps://doi.org/10.1038/s41591‐018‐0238‐9\\n761. Simin, Johanna, Rulla M. Tamimi, Lars Engstrand,\\nSteven Callens, Nele Brusselaers. 2020.“Antibiotic Use and\\nthe Risk of Breast Cancer: A Systematic Review and Dose–\\nResponse Meta ‐Analysis.” Pharmacological Research 160:\\n105072. https://doi.org/10.1016/j.phrs.2020.105072\\n762. Lurienne, Lise, Julie Cervesi, Lola Duhalde, Jean De Gunzburg,\\nAntoine Andremont, Gérard Zalcman, Renaud Buffet, Pierre‐\\nAlain Bandinelli. 2020.“NSCLC Immunotherapy Efficacy and\\nAntibiotic Use: A Systematic Review and Meta‐Analysis.”\\nJournal of Thoracic Oncology15: 1147–59. https://doi.org/10.\\n1016/j.jtho.2020.03.002\\n763. Imlay, Hannah, Nicholas C. Laundy, Graeme N. Forrest,\\nMonica A. Slavin. 2023.“Shorter Antibiotic Courses in the\\nImmunocompromised: The Impossible Dream? ” Clinical\\nMicrobiology and Infection 29: 143 –9. https://doi.org/10.\\n1016/j.cmi.2022.08.007\\n764. Singh, Vishal, Beng San Yeoh, Ahmed A. Abokor,\\nRachel M. Golonka, Yuan Tian, Andrew D. Patterson, Bina Joe,\\nMathias Heikenwalder, Matam Vijay‐Kumar. 2020.“Vancomy-\\ncin Prevents Fermentable Fiber‐Induced Liver Cancer in Mice\\nWith Dysbiotic Gut Microbiota.” Gut Microbes 11: 1077–91.\\nhttps://doi.org/10.1080/19490976.2020.1743492\\n765. Pinato, David J., Xiaoxue Li, Pallavi Mishra ‐Kalyani,\\nAntonio D'Alessio, Claudia A. M. Fulgenzi,\\nBernhard Scheiner, Matthias Pinter, et al. 2023.“Association\\nBetween Antibiotics and Adverse Oncological Outcomes in\\nPatients Receiving Targeted or Immune‐Based Therapy for\\nHepatocellular Carcinoma.” JHEP Reports5: 100747.https://\\ndoi.org/10.1016/j.jhepr.2023.100747\\n766. Iida, Noriho, Eishiro Mizukoshi, Tatsuya Yamashita, Takeshi\\nTerashima, Kuniaki Arai, Jun Seishima, Shuichi Kaneko. 2019.\\n“Overuse of Antianaerobic Drug Is Associated With Poor\\nPostchemotherapy Prognosis ofPatients With Hepatocellular\\nCarcinoma.” International Journal of Cancer 145: 2701–11.\\nhttps://doi.org/10.1002/ijc.32339\\n767. López ‐Bucio, José. 2025.“Dietary Auxin May Help Patients\\nto Fight Cancer.” Trends in Plant Science30: 134–6. https://\\ndoi.org/10.1016/j.tplants.2024.10.016\\n768. Arifuzzaman, Mohammad, Nicholas Collins, Chun‐Jun Guo,\\nDavid Artis. 2024. “Nutritional Regulation of Microbiota‐\\nDerived Metabolites: Implications for Immunity and\\nInflammation. ” Immunity 57: 14 –27. https://doi.org/10.\\n1016/j.immuni.2023.12.009\\nMICROBIOME IN CANCER | 101 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 101, 'page_label': '102'}, page_content='769. Ilerhunmwuwa, Nosakhare Paul, Abul Hasan Shadali\\nAbdul Khader, Calvin Smith, Edward R Scheffer Cliff,\\nChristopher M. Booth, Evevanne Hottel, Muhammad Aziz, et al.\\n2024. “Dietary Interventions in Cancer: A Systematic Review of\\nAll Randomized Controlled Trials.” Journal of the National\\nCancer Institute 116: 1026–34. https://doi.org/10.1093/jnci/\\ndjae051\\n770. Xiao, Yu ‐Ling, Yue Gong, Ying‐Jia Qi, Zhi‐Ming Shao, Yi‐\\nZhou Jiang. 2024.“Effects of Dietary Intervention on Human\\nDiseases: Molecular Mechanisms and Therapeutic Potential.”\\nSignal Transduction and Targeted Therapy9: 59.https://doi.\\norg/10.1038/s41392‐024‐01771‐x\\n771. Bolte, Laura A., Karla A. Lee, Johannes R. Björk, Emily R.\\nL e e m i n g ,M a r j oJ .E .C a m p m a n s‐Kuijpers, Jacco J. De Haan,\\nArnau Vich Vila, et al. 2023.“Association of a Mediterranean\\nDiet With Outcomes for Patients Treated With Immune\\nCheckpoint Blockade for Advanced Melanoma. ” JAMA\\nOncology 9: 705–9. https://doi.org/10.1001/jamaoncol.2022.7753\\n772. Li, Ziyuan, Lei Qian, Jianghui Chu, Yuan Liu, Gusonghan\\nMaitiniyazi, Yue Chen, Xinxin Cheng, et al. 2023.“Diet Is\\nAssociated With Frailty in Lung Cancer: A Possible Role of\\nGut Microbiota.” Nutrients 15: 4298.https://doi.org/10.3390/\\nnu15194298\\n773. Li, Lili, Shuling Yan, Shuangjiang Liu, Ping Wang,\\nWenjun Li, Yuetao Yi, Song Qin. 2023. “In‐Depth Insight\\nInto Correlations Between Gut Microbiota and Dietary Fiber\\nElucidates a Dietary Causal Relationship With Host Health.”\\nFood Research International (Ottawa, Ont.) 172: 113133.\\nhttps://doi.org/10.1016/j.foodres.2023.113133\\n774. Wang, Chuhui, Tongtong Lan, Zhao Chen, Xiaowen Wang,\\nYisa Han, Ning Yang, Zhen Xu, et al. 2023.“The Preventive\\nEffects of Inulin, Cellulose, and Their Mixture on Colorectal\\nCancer Liver Metastasis in Mice by Regulating Gut Micro-\\nbiota.” Journal of Food Science88: 4705–17. https://doi.org/\\n10.1111/1750‐3841.16772\\n775. Fang, Gaofeng, Shengquan Wang, Qianyao Chen, Han Luo,\\nXuemei Lian, Dan Shi. 2023.“Time‐Restricted Feeding Af-\\nfects the Fecal Microbiome Metabolome and Its Diurnal\\nOscillations in Lung Cancer Mice.” Neoplasia 45: 100943.\\nhttps://doi.org/10.1016/j.neo.2023.100943\\n776. Ronis, Martin J., Kelly E. Mercer, Kartik Shankar,\\nCasey Pulliam, Kim Pedersen, Magnus Ingelman‐Sundberg,\\nSimonetta Friso, et al. 2020.“Potential Role of Gut Micro-\\nbiota, the Proto‐Oncogene PIKE (Agap2) and Cytochrome\\nP450 CYP2W1 in Promotion of Liver Cancer by Alcoholic\\nand Nonalcoholic Fatty Liver Disease and Protection by\\nDietary Soy Protein.” Chemico‐Biological Interactions 325:\\n109131. https://doi.org/10.1016/j.cbi.2020.109131\\n777. Yu, Yinglan, Xinran Shen, Xin Xiao, Lian Li, Yuan Huang.\\n2023. “Butyrate Modification Promotes Intestinal Absorption\\nand Hepatic Cancer Cells Targeting of Ferroptosis Inducer\\nLoaded Nanoparticle for Enhanced Hepatocellular Carci-\\nnoma Therapy. ” Small (Weinheim an Der Bergstrasse,\\nGermany) 19: e2301149. https://doi.org/10.1002/smll.\\n202301149\\n778. Wang, Hongbin, Mia Borlongan, Howard L. Kaufman,\\nUyen Le, Hans J. Nauwynck, Samuel D. Rabkin,\\nDipongkor Saha. 2024. “Cytokine‐Armed Oncolytic Herpes\\nSimplex Viruses: A Game ‐Changer in Cancer\\nImmunotherapy?” Journal for Immunotherapy of Cancer12:\\ne008025. https://doi.org/10.1136/jitc‐2023‐008025\\n779. Duan, Shijie, Shuhang Wang, Lei Qiao, Xinbo Yu,\\nNan Wang, Liting Chen, Xinyuan Zhang, et al. 2023.“On-\\ncolytic Virus‐Driven Biotherapies From Bench to Bedside.”\\nSmall (Weinheim an Der Bergstrasse, Germany)19: e2206948.\\nhttps://doi.org/10.1002/smll.202206948\\n780. Burchett, Rebecca, Scott Walsh, Yonghong Wan,\\nJonathan L. Bramson. 2020.“A Rational Relationship: On-\\ncolytic Virus Vaccines as Functional Partners for Adoptive T\\nCell Therapy.” Cytokine & Growth Factor Reviews56: 149–59.\\nhttps://doi.org/10.1016/j.cytogfr.2020.07.003\\n781. Nguyen, Hong ‐My, Praveen K. Bommareddy, Ann W. Silk,\\nDipongkor Saha. 2022.“Optimal Timing of PD‐1 Blockade in\\nCombination With Oncolytic Virus Therapy.” Seminars in\\nCancer Biology 86: 971 –80. https://doi.org/10.1016/j.\\nsemcancer.2021.05.019\\n782. DePeaux, Kristin, Greg M. Delgoffe. 2024. “Integrating\\nInnate and Adaptive Immunity in Oncolytic Virus Therapy.”\\nTrends in Cancer10: 135–46. https://doi.org/10.1016/j.trecan.\\n2023.09.012\\n783. Bahreyni, Amirhossein, Yasir Mohamud, Honglin Luo. 2024.\\n“Oncolytic Virus\\n‐Based Combination Therapy in Breast\\nCancer.” Cancer Letters 585: 216634. https://doi.org/10.\\n1016/j.canlet.2024.216634\\n784. Kaufman, Howard L., Frederick J. Kohlhapp, Andrew Zloza.\\n2015. “Oncolytic Viruses: A New Class of Immunotherapy\\nDrugs.” Nature Reviews Drug Discovery 14: 642–62. https://\\ndoi.org/10.1038/nrd4663\\n785. Shalhout, Sophia Z., David M. Miller, Kevin S. Emerick,\\nHoward L. Kaufman. 2023.“Therapy With Oncolytic Viruses:\\nProgress and Challenges.” Nature Reviews Clinical Oncology\\n20: 160–77. https://doi.org/10.1038/s41571‐022‐00719‐w\\n786. Hennessy, Morgan L., Praveen K. Bommareddy,\\nGenevieve Boland, Howard L. Kaufman. 2019. “Oncolytic\\nImmunotherapy.” Surgical Oncology Clinics of North America\\n28: 419–30. https://doi.org/10.1016/j.soc.2019.02.007\\n787. Ma, Rui, Zhenlong Li, E. Antonio Chiocca,\\nMichael A. Caligiuri, Jianhua Yu. 2023.“The Emerging Field\\nof Oncolytic Virus‐Based Cancer Immunotherapy.” Trends in\\nCancer 9: 122 –39. https://doi.org/10.1016/j.trecan.2022.\\n10.003\\n788. Han, Zi ‐Yi, Qi‐Wen Chen, Zhuang‐Jiong Fu, Si‐Xue Cheng,\\nXian‐Zheng Zhang. 2022.“Probiotic Spore‐Based Oral Drug\\nDelivery System for Enhancing Pancreatic Cancer Chemo-\\ntherapy by Gut ‐Pancreas‐Axis‐Guided Delivery. ” Nano\\nLetters 22: 8608 –17. https://doi.org/10.1021/acs.nanolett.\\n2c03131\\n789. Nguyen, Vu H., Hyung‐Seok Kim, Jung‐Min Ha, Yeongjin\\nHong, Hyon E. Choy, Jung‐\\nJoon Min. 2010. “Genetically\\nEngineered Salmonella typhimurium as an Imageable Ther-\\napeutic Probe for Cancer. ” Cancer Research 70: 18 –23.\\nhttps://doi.org/10.1158/0008‐5472.CAN‐09‐3453\\n790. Ballister, Edward R., Alexander Michels, Rosa L. Vincent,\\nLior Kreindler, Sreyan Chowdhury, Samik Upadhaya,\\nAna Rosa Saez‐Ibañez, et al. 2025.“The Emerging Landscape\\nof Engineered Bacteria Cancer Therapies. ” Nature\\nBiotechnology 43: 672–6. https://doi.org/10.1038/s41587‐025‐\\n02623‐x\\n102 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 102, 'page_label': '103'}, page_content=\"791. Schmitz ‐Winnenthal, Friedrich H., Hohmann Nicolas,\\nAndreas G. Niethammer, Tobias Friedrich, Heinz Lubenau,\\nMarco Springer, Klaus M. Breiner, et al. 2015. “Anti‐\\nAngiogenic Activity of VXM01, an Oral T ‐Cell Vaccine\\nAgainst VEGF Receptor 2, in Patients With Advanced Pan-\\ncreatic Cancer: A Randomized, Placebo‐Controlled, Phase 1\\nTrial.” Oncoimmunology 4: e1001217. https://doi.org/10.\\n1080/2162402X.2014.1001217\\n792. Le, Dung T., Dirk G. Brockstedt, Ran Nir ‐Paz, Johannes\\nHampl, Shruti Mathur, John Nemunaitis, Daniel H. Sterman,\\net al. 2012. “A Live‐Attenuated Listeria Vaccine (ANZ‐100)\\nand a Live ‐Attenuated Listeria Vaccine Expressing Me-\\nsothelin (CRS‐207) for Advanced Cancers: Phase I Studies of\\nSafety and Immune Induction.” Clinical Cancer Research: An\\nOfficial Journal of the American Association for Cancer\\nResearch 18: 858 –68. https://doi.org/10.1158/1078 ‐0432.\\nCCR‐11‐2121\\n793. Wang, Heng, Fang Xu, Chao Wang. 2025. “Metabolic Re-\\nprogramming of Tumor Microenviroment by Engineered\\nBacteria.” Seminars in Cancer Biology112: 58–70. https://doi.\\norg/10.1016/j.semcancer.2025.03.003\\n7 9 4 . Z h a n g ,H a o ,L iF u ,X i n w e nL e i l i a n g ,C h u n r u nQ u ,W a n t a oW u ,\\nRong Wen, Ning Huang, et al. 2024.“Beyond the Gut: The\\nIntratumoral Microbiome's Influence on Tumorigenesis and\\nTreatment Response.” Cancer Communications 44: 1130–67.\\nhttps://doi.org/10.1002/cac2.12597\\n795. Sieow, Brendan Fu‐Long, Kwok Soon Wun, Wei Peng Yong,\\nIn Young Hwang, Matthew Wook Chang. 2021.“Tweak to\\nTreat: Reprograming Bacteria for Cancer Treatment.” Trends in\\nCancer 7: 447–64. https://doi.org/10.1016/j.trecan.2020.11.004\\n796. Li, Tong, Wenjing Wang. 2021. “New Strategy for Cancer\\nImmunotherapy: Using Live Engineered Bacteria for Meta-\\nbolic Modulation. ” Signal Transduction and Targeted\\nTherapy 6: 415.https://doi.org/10.1038/s41392‐021\\n‐00829‐4\\n797. Cubillos ‐Ruiz, Juan R., Andres Cubillos ‐Ruiz. 2021. “En-\\ngineered Bacteria Recycle Tumor Metabolic Waste to Boost\\nImmunotherapy.” Cell Host & Microbe 29: 1725–7. https://\\ndoi.org/10.1016/j.chom.2021.11.008\\n798. Wang, Yifan, Anqi Dong, Jianping Man, Hua Chen,\\nWenhao Shen, Lei Wang, Hongli Yang, Lin Hu, Kai Yang.\\n2025. “TREM2 scFv‐Engineering Escherichia coli Displaying\\nModulation of Macrophages to Boost Cancer Radio ‐\\nImmunotherapy.” Advanced Materials (Deerfield Beach, Fla.)\\n37: e2417920.https://doi.org/10.1002/adma.202417920\\n799. Xie, Beibei, Linmiao Don g, Leo Wang, Ruibing Wang,\\nChunlai Li. 2024. “Supramolecularly Engineered Bacteria\\nMediated Calcium Overload and Immunotherapy of Tumors.”\\nTheranostics14: 6560–70. https://doi.org/10.7150/thno.99931\\n800. Liu, Xuemeng, Mengyu Sun, Fang Pu, Jinsong Ren,\\nXiaogang Qu. 2023.“Transforming Intratumor Bacteria Into\\nImmunopotentiators to Reverse Cold Tumors for Enhanced\\nImmuno‐chemodynamic Therapy of Triple‐Negative Breast\\nCancer.” Journal of the American Chemical Society 145:\\n26296–307. https://doi.org/10.1021/jacs.3c09472\\n801. Zhang, Lilong, Dongqi Chai, Chen Chen, Chunlei Li,\\nZhendong Qiu, Tianrui Kuang, Mungur Parveena, et al. 2022.\\n“Mycobiota and C‐Type Lectin Receptors in Cancers: Know\\nThy Neighbors.” Frontiers in Microbiology13: 946995.https://\\ndoi.org/10.3389/fmicb.2022.946995\\n802. De Menezes, Ag ‐Anne P. M., Raí P. S. Aguiar,\\nJosé V. O. Santos, Chandan Sarkar, Muhammad T. Islam,\\nAntonio L. Braga, Mohammad M. Hasan, et al. 2023.\\n“Citrinin as a Potential Anti‐Cancer Therapy: A Compre-\\nhensive Review. ” Chemico‐Biological Interactions 381:\\n110561. https://doi.org/10.1016/j.cbi.2023.110561\\n803. Jin, Jianshi, Reiko Yamamoto, Katsuyuki Shiroguchi. 2024.\\n“High‐Throughput Identification and Quantification of\\nBacterial Cells in the Microbiota Based on 16S rRNA\\nSequencing With Single ‐\\nBase Accuracy Using BarBIQ. ”\\nNature Protocols19: 207–39. https://doi.org/10.1038/s41596 ‐\\n023‐00906‐8\\n804. Galeano Niño, Jorge Luis, Hanrui Wu, Kaitlyn D. LaCourse,\\nAndrew G. Kempchinsky, Alexander Baryiames, Brittany Barber,\\nNeal Futran, et al. 2022.“Effect of the Intratumoral Microbiota\\non Spatial and Cellular Heterogeneity in Cancer.” Nature 611:\\n810–7. https://doi.org/10.1038/s41586‐022‐05435‐0\\n805. Gerber, Georg K. 2024.“AI in Microbiome Research: Where\\nHave We Been, Where Are We Going?” Cell Host & Microbe\\n32: 1230–4. https://doi.org/10.1016/j.chom.2024.07.021\\n806. Ghannam, Ryan B., Stephen M. Techtmann. 2021.“Machine\\nLearning Applications in Microbial Ecology, Human Micro-\\nbiome Studies, and Environmental Monitoring.” Computational\\nand Structural Biotechnology Journal19: 1092–107. https://doi.\\norg/10.1016/j.csbj.2021.01.028\\n807. Pepke, Michael L., Søren B. Hansen, Morten T. Limborg. 2024.\\n“Unraveling Host Regulation of Gut Microbiota Through the\\nEpigenome‐Microbiome Axis.” Trends in Microbiology32: 1229–\\n40. https://doi.org/10.1016/j.tim.2024.05.006\\n808. Ma, Yong ‐Jing, Yuan‐Chen Sun, Lu Wang, Wan‐Xing Xu,\\nXiao‐Dan Fan, Jun Ding, Christopher Heeschen, Wen‐Juan\\nWu, Xiao‐Qi Zheng, Ning‐Ning Liu. 2024. Dissection of In-\\ntratumor Microbiome–Host Interactions at Single‐Cell Level\\nin Lung Cancer.” hLife: in press. https://doi.org/10.1016/j.\\nhlife.2024.09.001\\n809. Ratiner, Karina, Dragos Ciocan, Suhaib K. Abdeen,\\nEran Elinav. 2024. “Utilization of the Microbiome in Per-\\nsonalized Medicine.”\\nNature Reviews Microbiology 22: 291–\\n308. https://doi.org/10.1038/s41579‐023‐00998‐9\\n810. Redenti, Andrew, Jongwon Im, Benjamin Redenti,\\nFangda Li, Mathieu Rouanne, Zeren Sheng, William Sun,\\net al. 2024.“Probiotic Neoantigen Delivery Vectors for Pre-\\ncision Cancer Immunotherapy.” Nature 635: 453–61. https://\\ndoi.org/10.1038/s41586‐024‐08033‐4\\n811. Li, Zhaoting, Yixin Wang, Jun Liu, Piper Rawding,\\nJiyoon Bu, Seungpyo Hong, Quanyin Hu. 2021.“Chemically\\nand Biologically Engineered Bacteria‐Based Delivery Systems\\nfor Emerging Diagnosis and Advanced Therapy.” Advanced\\nMaterials (Deerfield Beach, Fla.) 33: e2102580. https://doi.\\norg/10.1002/adma.202102580\\n812. Shen, Haosheng, Nikhil Aggarwal, Kwok Soon Wun,\\nYung Seng Lee, In Young Hwang, Matthew Wook Chang.\\n2022. “Engineered Microbial Systems for Advanced Drug\\nDelivery.” Advanced Drug Delivery Reviews 187: 114364.\\nhttps://doi.org/10.1016/j.addr.2022.114364\\n813. Krishnamurthy, Malathy, Richard T. Moore, Sathish\\nRajamani, Rekha G. Panchal. 2016.“Bacterial Genome En-\\ngineering and Synthetic Biology: Combating Pathogens.” BMC\\nMicrobiology16: 258.https://doi.org/10.1186/s12866‐016‐0876‐3\\nMICROBIOME IN CANCER | 103 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'producer': 'iText 4.2.0 by 1T3XT', 'creator': 'John Wiley & Sons', 'creationdate': '2025-10-13T16:00:40+05:30', 'keywords': 'cancer; microbiome; precision oncology; treatment; tumor microenvironment', 'moddate': '2025-12-26T02:59:48-08:00', 'subject': 'iMeta 2025.4:e70070', 'wps-proclevel': '3', 'wps-journaldoi': '10.1002/(ISSN)2770-596X', 'author': 'Anqi Lin', 'title': 'The microbiome in cancer', 'wps-articledoi': '10.1002/imt2.70070', 'source': 'data/microbiome_in_cancer.pdf', 'total_pages': 104, 'page': 103, 'page_label': '104'}, page_content='814. Cianci, Rossella, Mario Caldarelli, Paola Brani, Annalisa Bosi,\\nAlessandra Ponti, Cristina Giaroni, Andreina Baj. 2025.“Cyto-\\nkines Meet Phages: A Revolutionary Pathway to Modulating\\nImmunity and Microbial Balance.” Biomedicines 13: 1202.\\nhttps://doi.org/10.3390/biomedicines13051202\\n815. Lei, Lei, Yan, Jiayao, Kai Xin, Lin Li, Qi Sun, Ying Wang,\\nTianran Chen, et al. 2024.“Engineered Bacteriophage‐Based\\nIn?Situ Vaccine Remodels a Tumor Microenvironment and\\nE l i c i t sP o t e n tA n t i t u m o rI m m u n i t y .” ACS Nano18: 12194–209.\\nhttps://doi.org/10.1021/acsnano.4c00413\\n816. Hou, Xiao ‐Lin, Bin Zhang, Kai Cheng, Fang Zhang, Xiao‐\\nTing Xie, Wei Chen, Lin‐Fang Tan, et al. 2024.“Engineering\\nPhage Nanocarriers Integrated With Bio‐Intelligent Plasmids\\nfor Personalized and Tunable Enzyme Delivery to Enhance\\nChemodynamic Therapy. ” Advanced Science (Weinheim,\\nBaden‐Wurttemberg, Germany) 11: e2308349. https://doi.\\norg/10.1002/advs.202308349\\n817. Al ‐Bahrani, Mariam, Paladd Asavarut, Sajee Waramit,\\nKeittisak Suwan, Amin Hajitou. 2023.“Transmorphic Phage‐\\nGuided Systemic Delivery of TNFα Gene for the Treatment of\\nHuman Pediatric Medulloblastoma.” The FASEB Journal37:\\ne23038. https://doi.org/10.1096/fj.202300045R\\n818. Li, Dengxiong, Ruicheng Wu, Qingxin Yu, Zhouting Tuo,\\nJie Wang, Koo Han Yoo, Wuran Wei, et al. 2024.“Micro-\\nbiota and Urinary Tumor Immunity: Mechanisms, Thera-\\npeutic Implications, and Future Perspectives. ” Chinese\\nJournal of Cancer Research = Chung‐Kuo Yen Cheng Yen\\nChiu 36: 596–615. https://doi.org/10.21147/j.issn.1000 ‐9604.\\n2024.06.03\\n819. Zhao, Lihong, Mei Li, Chen Shen, Yurui Luo, Xiaoming Hou,\\nYu Qi, Ziwei Huang, et al. 2024. “Nano‐Assisted Radio-\\ntherapy Strategies: New Opportunities for Treatment of Non‐\\nSmall Cell Lung Cancer.” Research (Washington, D.C.) 7:\\n0429. https://doi.org/10.34133/research.0429\\n820. Fang, Yiran, Yuyun Kong, Guangda Rong, Qingcong Luo,\\nWangjun Liao, Dongqiang Zeng.“Systematic Investigation of\\nTumor Microenvironment and Antitumor Immunity With\\nIOBR.” Med Research\\n1: 136–140. https://doi.org/10.1002/mdr2.\\n70001\\n821. Yu, Li, Ke Huang, Yixiang Liao, Lingzhi Wang,\\nGautam Sethi, Zhaowu Ma. 2024. “Targeting Novel Regu-\\nlated Cell Death: Ferroptosis, Pyroptosis and Necroptosis in\\nAnti‐PD‐1/PD‐L1 Cancer Immunotherapy.” Cell Proliferation\\n57: e13644.https://doi.org/10.1111/cpr.13644\\n822. Jiang, Aimin, Jinxin Li, Ziwei He, Ying Liu, Kun Qiao,\\nYu Fang, Le Qu et al. 2024.“Renal Cancer: Signaling Path-\\nways and Advances in Targeted Therapies.” MedComm 5:\\ne676. https://doi.org/10.1002/mco2.676\\n823. Zetrini, Abdulmottaleb E., HoYin Lip, Azhar Z. Abbasi,\\nIbrahim Alradwan, Taksim Ahmed, Chunsheng He,\\nJeffrey T. Henderson, Andrew M. Rauth, Xiao Yu Wu. 2023.\\n“Remodeling Tumor Immune Microenvironment by Using\\nPolymer‐Lipid‐Manganese Dioxide Nanoparticles With\\nRadiation Therapy to Boost Immune Response of Castration‐\\nResistant Prostate Cancer.” Research (Washington, D.C.) 6:\\n0247. https://doi.org/10.34133/research.0247\\n824. Li, Zizhuo, Anqi Lin, Zhifei Gao, Aimin Jiang,\\nMinying Xiong, Jiapeng Song, Zaoqu Liu, et al. 2024.“B‐Cell\\nPerformance in Chemotherapy: Unravelling the Mystery of\\nB‐Cell Therapeutic Potential. ” Clinical and Translational\\nMedicine 14: e1761.https://doi.org/10.1002/ctm2.1761\\n825. Lin, Anqi, Chang Qi, Ting Wei, Mengyao Li, Quan Cheng,\\nZaoqu Liu, Peng Luo, Jian Zhang. 2022.“CAMOIP: A Web\\nServer for Comprehensive Analysis on Multi ‐Omics of\\nImmunotherapy in Pan‐Cancer.” Briefings in Bioinformatics\\n23: bbac129.https://doi.org/10.1093/bib/bbac129\\n826. Zhang, Haonan, Ge Zhang, Pengyuan Xu, Fengyi Yu,\\nLiwen Li, Runzhi Huang, Pengpeng Zhang, et al.“Optimized\\nDynamic Network Biomarker Deciphers a High‐Resolution\\nHeterogeneity Within Thyroid Cancer Molecular Subtypes.”\\nMed Research 1: 10‐31. https://doi.org/10.1002/mdr2.70004\\n827. Zhu, Xiaoqiang, Muni Hu, Xiaowen Huang, Lingxi Li,\\nXiaolin Lin, Xiaoyan Shao, Jiantao Li, et al. 2025, 37, 806–23.e6.\\n“Interplay Between Gut Microbial Communities and Metabo-\\nlites Modulates Pan‐Cancer Immunotherapy Responses.” Cell\\nMetabolism. https://doi.org/10.1016/j.cmet.2024.12.013\\n828. Battaglia, Thomas W., Iris L. Mimpen, Joleen J. H. Traets,\\nArne Van Hoeck, Laurien J. Zeverijn, Birgit S. Geurts,\\nGijs F. De Wit, et al. 2024.“A Pan‐Cancer Analysis of the\\nMicrobiome in Metastatic Cancer.” Cell 187: 2324–35.e19.\\nhttps://doi.org/10.1016/j.cell.2024.03.021\\n829. Liu, Yali, Chi Chun Wong, Yanqiang Ding, Mengxue Gao,\\nJun Wen, Harry Cheuk‐Hay Lau, Alvin Ho‐Kwan Cheung,\\nDan Huang, He Huang et al. 2024. “Peptostreptococcus\\nanaerobius Mediates Anti‐PD1 Therapy Resistance and Ex-\\nacerbates Colorectal Cancer via Myeloid‐Derived Suppressor\\nCells in Mice.” Nature Microbiology 9: 1467–82. https://doi.\\norg/10.1038/s41564‐024‐01695‐w\\n830. Tian, Jinzhong, Chong Li, Zhixiang Dong, Yunpeng Yang,\\nJing Xing, Peijun Yu, Ying Xin, et al. 2023.“Inactivation of\\nthe Antidiabetic Drug Acarbose by Human Intestinal\\nMicrobial‐Mediated Degradation. ” Nature Metabolism 5:\\n896–909. https://doi.org/10.1038/s42255‐023‐00796‐w\\n831. Schupack, Daniel A., Ruben A. T. Mars, Dayne H. Voelker,\\nJithma P. Abeykoon, Purna C. Kashyap. 2022.“The Promise\\nof the Gut Microbiome as Part of Individualized Treatment\\nStrategies.” Nature Reviews Gastroenterology & Hepatology19:\\n7–25. https://doi.org/10.1038/s41575‐021‐00499‐1\\n832. Liu, Yafeng, Shujun Zhang, Kaijie Liu, Xinjun Hu, Xinyu Gu.\\n2024. “Advances in Drug Discovery Based on Network Phar-\\nmacology and Omics Technology. “Current Pharmaceutical\\nAnalysis 21: 33–43. https://doi.org/10.1016/j.cpan.2024.12.002\\n833. Chen, Jinghong, Anqi Lin, Peng Luo. 2024. “Advancing\\nPharmaceutical Research.\\n” Current Pharmaceutical Analysis\\n21: 1–19. https://doi.org/10.1016/j.cpan.2024.11.001\\n834. Wang, Ziheng, Yang Zhao, Lin Zhang. 2024. “Emerging\\nTrends and Hot Topics in the Application of Multi‐Omics in\\nDrug Discovery.” Current Pharmaceutical Analysis21: 20–32.\\nhttps://doi.org/10.1016/j.cpan.2024.12.001\\nHow to cite this article:Lin, Anqi, Minying\\nXiong, Aimin Jiang, Lihaoyun Huang, Hank Z. H.\\nWong, Suyin Feng, Chunyan Zhang, et al. 2025.\\n“The Microbiome in Cancer.” iMeta 4, e70070.\\nhttps://doi.org/10.1002/imt2.70070\\n104 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 0, 'page_label': '1'}, page_content='Ovarian Cancer\\nA Review\\nGiuseppeCaruso,MD;S.JohnWeroha,MD,PhD;WilliamCliby,MD\\nIMPORTANCE Ovariancanceristheeighthmostcommoncauseofcancerandcancerdeathin\\nwomenworldwide.In2022,ovariancancerwasdiagnosedinapproximately324398\\nindividuals,and206839diedofovariancancerworldwide.In2025,itisestimatedthat\\n20890USwomenwillbediagnosedwithovariancancerand12730patientswilldieof\\novariancancer.\\nOBSERVATIONSApproximately90%ofovariancancersareepithelialmalignancies,ofwhich\\n70%to80%arehigh-gradeserousovariancancers.Lesscommonepithelialsubtypesinclude\\nendometrioid,clearcell,low-gradeserous,mucinous,andcarcinosarcoma.Themedianageat\\ndiagnosisofovariancanceris63years.Riskfactorsincludeolderage,familyhistoryofbreast\\norovariancancer,endometriosis,andnulliparity.Hereditaryfactorsareassociatedwith25%\\nofcases,predominantlylinkedto BRCA1/2 genevariants.Atdiagnosis,approximately95%of\\npatientsexperiencenonspecificsymptoms,suchasabdominalpain,bloating,andurinary\\nurgencyandfrequency,andabout80%haveadvanced-stagedisease(stageIII-IV),including\\nextrapelvicdisease,ascites,andabdominalmasses.Diagnosticandstagingevaluation\\nincludespelvicultrasound;computedtomographyofthechest,abdomen,andpelvis;and\\nserumtumormarkerssuchascarbohydrateantigen125,carbohydrateantigen19-9,and\\ncarcinoembryonicantigen.First-linetreatmentforearly-stageovariancancer,definedas\\nlimitedtotheovaryorfallopiantube(stageI)orconfinedtothepelvis(stageII),issurgery\\n(hysterectomy,bilateralsalpingo-oophorectomy,omentectomy,andlymphadenectomy),\\nfollowedbyadjuvantchemotherapy(carboplatinandpaclitaxel).Withtreatment,early-stage\\novariancancerhasa5-yearoverallsurvivalof70%to95%.Advanced-stageovariancancer\\nmaybetreatedwithprimarycytoreductivesurgery(removalofallvisiblecancerinthe\\nabdominalcavity)andadjuvantchemotherapy(carboplatinandpaclitaxel)orwith\\nneoadjuvantchemotherapyfollowedbycytoreductivesurgeryandadjuvantchemotherapy.\\nMostpatientswithadvanced-stageovariancancerreceivemaintenancetherapywith\\nbevacizumab(amonoclonalantibodythatblocksangiogenesis)and/orpoly–adenosine\\ndiphosphateribosepolymerase(PARP)inhibitors.Withtreatment,the5-yearoverallsurvival\\nrateforadvanced-stageovariancanceris10%to40%.H owever,individualswith\\nBRCA-relatedgenevariantshavea5-yearoverallsurvivalrateofapproximately70%with\\nPARPinhibitortreatment.Despiteaninitialremissionrateof80%,approximately75%of\\npatientswithadvanced-stagediseasehaveovariancancerrelapsewithin2years.\\nCONCLUSIONS AND RELEVANCEApproximately21000womenarediagnosedwithovarian\\ncancerannuallyintheUS,andapproximately80%haveadvanced-stageovariancancerat\\ndiagnosis.First-linetreatmentofearly-stageovariancancerissurgeryandadjuvant\\nplatinum-basedchemotherapy.Treatmentofadvanced-stageovariancancerincludes\\ncytoreductivesurgery,platinum-basedchemotherapy,andtargetedmaintenancetherapies\\nsuchasbevacizumaband/orPARPinhibitors.\\nJAMA.2025;334(14):1278-1291.doi: 10.1001/jama.2025.9495\\nPublishedonlineJuly21,2025.\\nMultimedia\\nSupplemental content\\nCME atjamacmelookup.com\\nAuthor Affiliations:Departmentof\\nObstetricsandGynecology,Division\\nofGynecologicOncology,Mayo\\nClinic,Rochester,Minnesota(Caruso,\\nCliby);DivisionofGynecologic\\nOncology,EuropeanInstituteof\\nOncology(IEO),IRCCS,Milan,Italy\\n(Caruso);Departmentof\\nExperimentalMedicine,Sapienza\\nUniversityofRome,Rome,Italy\\n(Caruso);DepartmentofOncology,\\nMayoClinic,Rochester,Minnesota\\n(Weroha).\\nCorresponding Author:William\\nCliby,MD,DepartmentofObstetrics\\nandGynecology,Divisionof\\nGynecologicSurgery,MayoClinic,\\n200FirstStSW,Rochester,MN\\n55905(cliby.william@mayo.edu).\\nClinical Review & Education\\nJAMA | Review\\n1278 (Reprinted) jama.com\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 1, 'page_label': '2'}, page_content='A\\nmongUSwomen,thelifetimeriskofdevelopingovarian\\ncanceris1in91,anddeathduetoovariancanceroccurs1\\nin143women.\\n1Approximately80%ofovariancancersare\\natanadvancedstageatdiagnosis. 2-4Earlierdetectionisdifficultbe-\\ncause there are no effective screening methods, and 95% of pa-\\ntientshavenonspecificinitialsymptoms,suchasabdominalpain,\\nbloating,andurinaryurgencyorfrequency.\\nEpithelialmalignanciescomprise90%ofovariancancers,with\\n70% to 80% being high-grade serous ovarian cancers.\\n5 Early-\\nstageovariancancerincludesstageI(confinedtotheovaryorfal-\\nlopian tube) and stage II (involving the uterus or other pelvic or-\\ngans) (Figure 1). With surgery and adjuvant platinum-based\\nchemotherapy, the 5-year overall survival of patients with early-\\nstageovariancanceris70%to95%.\\n2,4Advanced-stageovariancan-\\ncer,whichcomprisesstageIII(involvingpelvicandpara-aorticlymph\\nnodesorextrapelvicperitoneum)andstageIV(metastasestothe\\nliverorspleenoroutsidetheabdominalcavity),maybetreatedwith\\na combination of cytoreductive surgery, platinum-based chemo-\\ntherapy, and targeted therapies such as the antiangiogenic drug\\nbevacizumab and/or poly–adenosine diphosphate ribose poly-\\nmerase(PARP)inhibitors.Initialremissionratesofadvanced-stage\\novariancancerare80%,butapproximately75%ofpatientswithad-\\nvanced-stageovariancancerrelapsewithin2years,and5-yearover-\\nallsurvivalis10%to40%.\\n2,4Approximately50%ofpatientswith\\nadvanced-stageovariancancerhaveheritable(germline)ortumor-\\nacquired(somatic)BRCA1/2orotherBRCA-relatedgenevariants,and\\nwhentreatedwithPARPinhibitors,have5-yearoverallsurvivalrates\\napproaching70%.\\n6ThisReviewfocusesondiagnosisandmanage-\\nmentofhigh-gradeserousovariancancer. Box 1providessomecom-\\nmonquestionsandanswersaboutovariancancer.\\nMethods\\nAPubMedsearchofEnglish-languagearticlesdescribingrandom-\\nized clinical trials (RCTs), meta-analyses, systematic reviews, and\\npracticeguidelinespublishedbetweenJanuary1,1995,andMay3,\\n2025,identified1733RCTs,1457meta-analyses,1770systematicre-\\nviews,and295guidelines.Prioritywasgiventotheselectionofhigh-\\nquality RCTs and meta-analyses. A total of 123 articles were in-\\ncluded,comprising65RCTs,9meta-analyses,5systematicreviews,\\n16guidelines,6narrativereviews,12prospectiveobservationalstud-\\nies,and10retrospectiveobservationalstudies.\\nEpidemiologyandRiskFactors\\nOvariancanceristheeighthmostcommoncauseofcancerandcan-\\ncerdeathinwomenworldwide.\\n7Globally,in2022,therewereanes-\\ntimated324398newcasesofovariancancerand206839deaths,\\nwith incidence and mortality rates of 6.6 and 4.2 per 100000\\nwomen-years,respectively.\\n7IntheUS,ovariancancerincidenceand\\nmortalityrateshavedeclinedbyapproximately3%annuallysince\\n2004 due to increased use of oral contraceptives (preventing re-\\npetitive ovulation-related ovarian injury), prophylactic salpingo-\\noophorectomy in women with high-risk genetic variants such as\\nFigure 1. Origin and Stages of Ovarian Cancer\\nOrigin of ovarian cancer Stage I ovarian cancer\\nStage II ovarian cancer Stage III ovarian cancer Stage IV ovarian cancer\\nSerous tubal intraepithelial \\ncarcinoma is most likely \\nthe precursor to high-grade \\nserous ovarian cancer.\\nStage IA\\nCancer within\\n1 ovary or\\nfallopian tube\\nStage IIA\\nCancer spread\\nto the uterus and/or \\nfallopian tubes \\nand/or ovaries\\nStage IIB\\nCancer spread to \\nother pelvic tissues \\n(colon or bladder)\\nStage IIIA\\nIIIA1 Positive retroperitoneal\\n          lymph nodes\\nIIIA2 Microscopic peritoneal \\n          metastasis outside \\n          the pelvisStage IIIB\\nPeritoneal metastasis \\noutside the pelvis ≤2 cm\\nStage IIIC\\nPeritoneal metastasis \\noutside the pelvis >2 cm\\nStage IB\\nCancer within\\nboth ovaries or\\nfallopian tubes\\nStage IC\\nIC1 Surgical spill\\nIC2 Capsule ruptured before surgery \\n        or tumor on surface of ovary \\n        or fallopian tube\\nIC3 Malignant cells in ascites \\n        or peritoneal washings\\nBLADDER\\nUTERUS\\nUTERUT RR USUSUSUSSU\\nVVAGIAGIGIGI NANANNAN A\\nOVARY\\nCOLON\\nOvary\\nOvary\\nFimbriae\\nFallopian tube\\nSerous tubal\\nintraepithelial\\ncarcinoma cells\\nSAGITTAL\\nSECTION\\nOF PELVIS\\nStage IVB\\nMetastases \\nto the liver,\\nspleen, or \\noutside the \\nabdominal cavity\\nStage IVA\\nPleural effusion\\nwith positive\\ncytology\\nOvarianCancer:AReview Review Clinical Review & Education\\njama.com (Reprinted) JAMA October14,2025 Volume334,Number14 1279\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 2, 'page_label': '3'}, page_content='BRCA1/2 (hereditarybreastandovariancancersyndrome),andim-\\nprovedtreatmentoptions. 1 Globally,ovariancancerincidencehas\\ndecreasedinotherhigher-incomecountriessuchastheUK,Austria,\\nthe Netherlands, and Norway but has increased in some lower-\\nincomeregionssuchasAfricaandsomepartsofAsia.\\n7,8\\nOvariancancerispredominantlyofepithelialorigin(90%).The\\nWorldHealthOrganizationin2020recognized6principalepithe-\\nlialsubtypes:high-gradeserous(70%-80%),endometrioid(10%-\\n20%),clearcell(5%-10%),low-gradeserous(5%-10%),mucinous\\n(3%-5%), and carcinosarcoma (1%-3%).\\n5 Less commonly, ovarian\\ncancerinvolvesgermcelltumors(3%-5%)andsexcordstromaltu-\\nmors(2%-5%).\\n5\\nThemedianageatdiagnosisofovariancanceris63years,al-\\nthough it is 10 years earlier in patients with germlineBRCA gene\\nvariants.9Approximately25%ofovariancancersarehereditarydue\\ntogermlinepathogenicvariantsinDNAdamagerepairgenes:90%\\ninthe BRCA1andBRCA2genes\\n10and10%attributabletoothergene\\nvariants, including the DNA mismatch repair genes (Lynch syn-\\ndrome)andgenesinvolvedintheDNAdouble-strandbreaksrepair\\nsystem,suchas CHEK2,RAD51,BRIP1,and PALB2.\\n11 Theestimated\\nlifetime risk of developing ovarian cancer in patients with aBRCA\\ngenevariantis20%to70%andis10%to18%withamismatchre-\\npairgenevariant.\\n12,13\\nEndocrineandreproductiveriskfactorsforovariancancerinclude\\nendometriosis(hazardratio,4.20;95%CI,3.59-4.91),14infertility(stan-\\ndardizedincidenceratio,2.0;95%CI,1.8-4.0),andpostmenopausal\\nestrogentherapy(relativerisk,1.31;95%CI,1.21-1.41).\\n15Possiblepro-\\ntectivefactorsaremultiparity(oddsratioforeachadditionalbirth,0.81;\\n95%CI,0.75-0.85),breastfeeding(oddsratio,0.72;95%CI,0.68-0.76),\\nandoralcontraceptiveuse(oddsratio,0.66;95%CI,0.52-0.83).\\n15\\nPathophysiologyandMolecularBiology\\nThepathogenesisofepithelialovariancancersisnotclear,butgene\\nexpressionprofilingsuggeststhatmosthigh-gradeserousovarian\\ncancerslikelyarisefromprecursorsinthetubalfimbriaandmetas-\\ntasizetotheovaryandperitonealcavity(Figure1).\\n16About50%of\\nhigh-grade serous ovarian cancers have genetic variants (25%\\nheritable/germline and 25% tumor-acquired/somatic) in homolo-\\ngous recombination repair genes resulting in an inability to repair\\nDNAdouble-strandbreaks,knownashomologousrecombination\\ndeficiency(HRD). BRCA1 andBRCA2 genevariantsaccountforap-\\nproximately half of HRD cases, found in 25% of all high-grade se-\\nrousovariancancers.\\n17TheremaininghalfofHRDcasesarecaused\\nby other genes involved in the homologous recombination repair\\npathway,including CHEK2,RAD51,BRIP1,and PALB2.\\nPrevention\\nHigh-Risk Individuals\\nIndividualsathighriskofovariancancerincludethosewithgerm-\\nlineBRCA1/2 genevariantsandthosewithafamilyhistoryofovar-\\niancanceratanyageorearly-onsetbreastcancer(age<40years).\\nTheNationalComprehensiveCancerNetwork(NCCN)guidelinesrec-\\nommendgenetictestingforovariancancersusceptibilitygenes(ATM,\\nBRCA1,BRCA2,BRIP1,MLH1,MSH2,MSH6,EPCAM,PALB2,RAD51C,\\nandRAD51D)inindividualswithafirst-orsecond-degreerelativewith\\nepithelialovariancanceratanyageorwithapersonalorfamilycan-\\ncerhistorysuggestiveofhereditarybreastandovariancancersyn-\\ndromebasedongeneticcounseling.\\n10\\nFor patients with known ovarian cancer susceptibility genes,\\nprophylacticbilateralsalpingo-oophorectomyisthemosteffective\\npreventivemeasureforovariancancer.\\n18Ameta-analysisof3pro-\\nspective studies including 9192 women withBRCA1/2 variants re-\\nported that bilateral salpingo-oophorectomy reduced the risk of\\novarian cancer by 81% over 4 years (hazard ratio, 0.19; 95% CI,\\n0.13-0.27).\\n19 The NCCN guidelines recommend bilateral salpingo-\\noophorectomyatage35to40yearsfor BRCA1 genevariantcarri-\\nerswhohavecompletedchildbearing,age40to45yearsfor BRCA2\\ncarriers,andage45to50yearsforwomenwithotherpathogenic\\ngenevariants,suchasforLynchsyndrome.\\n10 Priortoprophylactic\\nsurgery,cliniciansshoulddiscussthepotentialeffectsofearlymeno-\\npause(eg,increasedriskofvasomotorsymptoms,osteoporosis,car-\\ndiovasculardisease,cognitivedecline,adverseeffectsonsexualfunc-\\ntion),andhormonetherapyshouldbeconsideredforpremenopausal\\nwomenwithnopersonalbreastcancerhistory.\\n10\\nOral contraceptives reduce ovarian cancer risk for high-risk\\nyoungerindividuals(eg, BRCA variantcarriers)whodonotundergo\\nbilateral salpingo-oophorectomy. A meta-analysis of women with\\nBox 1. Commonly Asked Questions About Ovarian Cancer\\nWhat Are Common Presenting Signs and Symptoms\\nof Ovarian Cancer?\\nUpto95%ofwomenwithovariancancerinitiallyexperience\\nnonspecificsymptoms,suchasabdominalpain,bloating,and\\nurinaryurgency.Signsandsymptomsofadvanced-stageovarian\\ncancermayincludeascites,palpableabdominalmasses,and/or\\ndyspnea(causedbyabdominaldistensionorpleuraleffusions).\\nAre There Effective Methods to Decrease Risk of Ovarian Cancer?\\nTheUSPreventiveServicesTaskForcein2018recommended\\nagainstovariancancerscreeningwithtransvaginalultrasound\\nand/orcarbohydrateantigen125inbothasymptomaticindividuals\\nwithoutahigh-riskhereditarycancersyndromeandhigh-risk\\nindividuals(eg, BRCA1/2 genevariantcarriers).Womenwitha\\nfamilyhistoryofa BRCA genevariantorwithafirst-degreerelative\\nwithovariancanceratanyageorearly-onsetbreastcancer(age\\n<40years)shouldundergogenetictesting.Bilateral\\nsalpingo-oophorectomyisrecommendedbetweenages35and40\\nyearsfor BRCA1 carrierswhohavecompletedchildbearing,\\nbetweenages40and45yearsfor BRCA2 carriers,andbetween\\nages45and50yearsforwomenwithLynchsyndrome.\\nWhat Is the Initial Treatment and Prognosis for Patients\\nWith Ovarian Cancer?\\nFirst-linetreatmentforearly-stageovariancancer(stagesI-II)is\\nsurgeryfollowedbyplatinum-basedchemotherapy,whichis\\nassociatedwith5-yearoverallsurvivalratesof70%to95%.\\nAdvanced-stageovariancancer(stagesIII-IV)istreatedwitha\\ncombinationofcytoreductivesurgery,platinum-based\\nchemotherapy,andtargetedtherapies(bevacizumaband/orPARP\\ninhibitors)andhasa5-yearsurvivalrateof10%to40%.H owever,\\nwithPARPinhibitortreatment,patientswithadvanced-stage\\novariancancerand BRCA-relatedgenevariantshavea5-year\\nsurvivalrateofapproximately70%.\\nClinical Review & EducationReview OvarianCancer:AReview\\n1280 JAMA October14,2025 Volume334,Number14 (Reprinted) jama.com\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 3, 'page_label': '4'}, page_content='BRCA1/2 variantsusingoralcontraceptivesreportedahazardratio\\nof0.62(95%CI,0.52-0.74)forovariancancerbasedon2studies\\nof10981womenandanoddsratioof0.49(95%CI,0.38-0.63)from\\n8 studies including 10390 women.\\n20 Similarly, another meta-\\nanalysisof5observationalstudiesincluding1503participantswith\\nBRCA1/2pathogenicvariantsreportedthatoralcontraceptiveuse\\nwas associated with a decreased risk of ovarian cancer (summary\\nrelativerisk,0.50;95%CI,0.33-0.75),andeachadditional10years\\nofusewasassociatedwitha36%reducedrisk(summaryrelative\\nrisk,0.64;95%CI,0.53-0.78).\\n21\\nGeneral Population\\nThe American College of Obstetricians and Gynecologists and In-\\nternationalFederationofGynecologyandObstetricsguidelinesrec-\\nommend considering surgical and hormonal preventive ap-\\nproaches for the general population to reduce ovarian cancer\\nrisk.\\n22,23Forpostmenopausalwomenundergoingpelvicsurgeryfor\\nbenignconditions(eg,hysterectomyforuterinefibroidsorpelvic\\nsurgeryforcolondisease),bilateralsalpingo-oophorectomymaybe\\nconsidered.Forpremenopausalwomennotinterestedinchildbear-\\ningwhoareundergoingpelvicsurgery,bilateralsalpingectomy(re-\\nmovalofthefallopiantubes)whilepreservingtheovariescanavoid\\nearlymenopauseanditspotentialadverseeffectsonhealth.\\n22,23\\nA meta-analysis of 24 case-control and cohort studies re-\\nportedareducedincidenceofovariancancerinoralcontraceptive\\never users compared with never users (odds ratio, 0.73; 95% CI,\\n0.66-0.81).Oralcontraceptiveuseformorethan10yearswasas-\\nsociatedwithlowerrisk(oddsratio,0.43;95%CI,0.37-0.51).\\n24Life-\\ntimeovariancancerreductionattributabletooralcontraceptiveuse\\nwasestimatedat0.54%(numberneededtotreat,185for5years\\noforalcontraceptiveuse).\\n24\\nScreening\\nTheUSPreventiveServicesTaskForcein2018recommendedagainst\\novariancancerscreeninginasymptomaticwomenwithouthigh-risk\\nhereditarysyndromesbasedon2largeRCTs(PLCOandUKCTOCS;\\nN = 271103)thatfoundnosignificantmortalityreductionwithuseof\\nannual transvaginal ultrasound and/or carbohydrate antigen 125\\n(CA125)testingvsnoscreeningamongaverage-risk,asymptomatic\\nwomen(PLCO:relativerisk,1.18;95%CI,0.82-1.71;UKCTOCS:haz-\\nardratio,0.91;95%CI,0.76-1.09).\\n25Similarly,largeprospectivestud-\\niesofhigh-riskindividuals(eg,thosewith BRCA genevariants)have\\nshownthatscreeningwithtransvaginalultrasoundandCA125hasa\\nlowpositivepredictivevalue(4.6%-10.8%)anddoesnotreduceovar-\\niancancermortality.\\n26-29Thus,screeningforovariancancerisnotrec-\\nommended,eveninhigh-riskindividuals.\\nClinicalPresentation\\nPriortoovariancancerdiagnosis,95%ofwomenreportagradualon-\\nsetofnonspecificsymptoms,suchasabdominalpain,bloating,and\\nurinaryurgencyorfrequency.\\n30Aprospectivecase-controlstudyof\\nsymptomsinwomenwithapelvicmass(84benignand44ovarian\\ncancer) compared with women visiting 2 primary care clinics\\n(n=1709)foundthatpatientswithovariancancerreportedmorefre-\\nquentbloating(70%vs38%; P < .001),increasedabdominalsize\\n(64%vs19%; P < .001),urinaryurgency/frequency(55%vs32%;\\nP = .002),abdominalpain(50%vs30%; P = .006),andpelvicpain\\n(41%vs26%; P = .02).\\n31Thecombinationofbloating,increasedab-\\ndominalsize,andurinaryurgency/frequencywasobservedin43%\\nofpatientswithovariancancervs8%inpatientswithoutovariancan-\\ncer(oddsratio,9.4;95%CI,5.0-17.7).\\n31 Inasurveyof1725USand\\nCanadianwomenwithovariancancer,89%ofpatientswithearly-\\nstagediseaseand97%ofpatientswithadvanced-stagediseasere-\\nportedsymptomsbeforediagnosis,withnosignificantdifferencein\\nsymptomtypebasedonstage.\\n30Persistentabdominalpainorbloat-\\ning,ascites,apalpableabdominalorpelvicmass,weightloss,anddys-\\npnea(causedbyabdominaldistensionorpleuraleffusions)mayin-\\ndicateadvanced-stageovariancancer.Rarely(<1%ofpatients),ovarian\\ncancerpresentswithextra-abdominalsymptomscausedbydistant\\nmetastases(bonepain,seizures)orparaneoplasticsyndromes(cer-\\nebellardegeneration,peripheralpolyneuropathy).\\n32\\nDiagnosis\\nDiagnosticworkupincludeselicitingapersonalandfamilyhistory\\nofcancertodetermineifpatientshaveaknowngeneticvariantas-\\nsociated with ovarian cancer, such asBRCA1/2, and a physical ex-\\namination,includingpelvicandabdominalevaluationsforpalpable\\nmassesorascites( Box 2).Pelvicultrasoundishighlysensitive(70%-\\n93%) and specific (80%-98%) for identification of malignant ad-\\nnexal masses.\\n33,34 Computed tomography (CT) of the abdomen/\\npelvis is less sensitive (60%-90%) and specific (85%-94%) for\\ndiagnosisbuthasasensitivityof94%fordetectingperitonealdis-\\neaseandomentalthickening,whichareimportantfindingsforstag-\\ningandplanning.\\n34,35Adetailedstagingsystemforovariancancer\\nisoutlinedineTable1inthe Supplement.36ComparedwithCT,ab-\\ndominalmagneticresonanceimaging(MRI)ismoresensitive(92%-\\n94%)andspecific(85%-98%)forevaluatingperitonealcarcinoma-\\ntosis and mesenteric or bowel serosal involvement.\\n34,35,37 If\\nuncertainty remains about cancer resectability after CT or MRI\\nimaging, diagnostic laparoscopy may be performed prior to lapa-\\nrotomytodetermineresectability.\\n38Total-bodypositronemission\\ntomographyhashighersensitivity(73%-75%)thanCT(43%-55%)\\nfordetectionofextra-abdominallesions,suchasdistantlymphnodes\\n(eg, mediastinal, inguinal, supraclavicular) and pulmonary metas-\\ntases, potentially altering management to systemic therapy in-\\nsteadofabdominaldebulking.\\n39,40\\nSerum Tumor Markers\\nSerumtumormarkers,suchasCA125,humanepididymisprotein4,\\ncarbohydrate antigen 19-9 (CA19-9), and carcinoembryonic anti-\\ngen(CEA),arenotdiagnosticofepithelialovariancancerbutcanhelp\\nmonitor treatment response and posttreatment surveillance.\\nCarbohydrateantigen125isthemostcommonlyelevatedserumtu-\\nmormarker,\\n41 andbasedon17studiesthatincluded2374women\\nwithovariantumors,CA125( /H1135035U/mL)waselevatedatdiagnosis\\ninapproximately50%ofpatientswithearly-stageovariancancer\\nandin75%to92%withadvanced-stageovariancancer.\\n42 Human\\nepididymis protein 4 is another serum tumor marker that can be\\nuseful for early epithelial ovarian cancer detection (sensitivity,\\n65%-83%;specificity,78%-99%).\\n43ElevatedlevelsofCA19-9and\\nOvarianCancer:AReview Review Clinical Review & Education\\njama.com (Reprinted) JAMA October14,2025 Volume334,Number14 1281\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 4, 'page_label': '5'}, page_content='CEA,associatedwithmucinousgastrointestinalcancers,maybede-\\ntectedinpatientswithmucinousovariantumors.Primaryovarian\\nmucinouscancersarerare,soupperandlowerendoscopyshould\\nbeperformedtoevaluateforprimarygastrointestinaltumormeta-\\nstatictotheovary.ACA125/CEAratiogreaterthan25issuggestive\\nofaprimarymucinousovariantumorinsteadofgastrointestinalme-\\ntastasistotheovary.\\n2,4,41\\nBiopsy\\nPreoperativebiopsy(eg,fine-needleaspiration)iscontraindicated\\ninpatientswithpresumedearly-stageovariancancerbecauseitmay\\nleadtotumorruptureandspillageofmalignantcellsintotheperi-\\ntonealcavity.Intraoperativefrozensectionofasurgicalspecimen\\nisusuallyperformedtomakethepathologicaldiagnosisofovarian\\ncancer.\\n4,44Forpatientswithadvanced-stagediseasenotsuitablefor\\nprimary surgery, core biopsy (preferred) or fine-needle aspiration\\npreoperativetissueanalysisisrequiredtoconfirmthediagnosisof\\novariancancerandguidechemotherapyselection.Forpatientswith\\nadvanced-stagediseasewhoareeligibleforinitialsurgerywithout\\npreoperativechemotherapy,diagnosticintraoperativebiopsysuf-\\nfices.Forpatientswithascitesorpleuraleffusion,paracentesisor\\nthoracentesis can provide cytological diagnosis and symptom\\nrelief.\\n2,4\\nGeneticandMolecularTesting\\nTheNCCNandtheAmericanSocietyofClinicalOncologyguidelines\\nrecommend germline blood testing for all patients with epithelial\\novarian cancer to screen for inheritedBRCA1/2 variants.\\n10,13,45\\nNegative germline test findings should prompt surgical specimen\\nevaluationfortumor-specificacquired(somatic) BRCA1/2 variants.\\nGermlineandsomatictestinginformtreatmentandcounselingof\\npatients and at-risk relatives (Box 2).\\n10,13,45 Testing for HRD in\\nadvanced-stage,high-gradeserousorendometrioidovariancancer\\nalso guides treatment,\\n46 with HRD positivity associated with a\\nhigher response to platinum-based chemotherapy and treatment\\nwith DNA repair inhibitors such as PARP inhibitors.\\n42 Mismatch\\nrepairtumortestingisrecommendedbyNCCNguidelinesforendo-\\nmetrioid, clear cell, or mucinous subtypes to screen for Lynch\\nsyndrome.\\n4,13\\nTreatmentofEarly-StageOvarianCancer\\nFirst-linetherapyforpresumedearly-stageovariancancerissurgery\\ntoresectandstagethetumor,followedbyadjuvantplatinum-based\\nchemotherapy(Figure 2).Surgeryformostpatientswithearly-stage\\novariancancerincludes(1)enblocresectionoftheinvolvedovaryand\\nfallopiantubetoavoidspillageofcancercellsintotheperitoneumdur-\\ningsurgery;(2)hysterectomyandsalpingo-oophorectomyofthecon-\\ntralateralovaryandfallopiantube;(3)retroperitoneal(pelvicandpara-\\naortic)lymphadenectomy;(4)omentectomy;(5)randomperitoneal\\nbiopsies;and(6)peritonealfluidcytology.Appendectomymayalso\\nbeperformedforsomepatientswithmucinousovariancancer.\\n4,47\\nIntraoperativestagingisimportantbecauseapproximately30%of\\npatientswithpresumedearly-stageovariancancerbyimaginghave\\noccultmetastaticdiseaseondefinitivepathologicalexaminationof\\nretroperitoneal lymph nodes, omentum, and peritoneal surface\\nbiopsies.\\n4,47\\nPatientswithstageIdiseasewhoareconsideringhavingchil-\\ndreninthefuturemayundergofertility-sparingsurgery,consisting\\nofunilateralsalpingo-oophorectomy(preservingtheuterusandcon-\\ntralateral ovary/fallopian tube without ovarian cancer).\\n2,4 Cryo-\\npreservationofoocytesbeforebilateralsalpingo-oophorectomymay\\nallow future assisted reproductive approaches, such as in vitro\\nfertilization and uterine embryo transfer. Consultation with an\\noncofertilityspecialistpreoperativelyisrecommended.Inamulti-\\ninstitutional retrospective study of 211 women undergoing\\nBox 2. Diagnostic Workup for Epithelial Ovarian Cancer\\n• Cancerhistory(personalandfamily)\\n• Hereditarydisorders:hereditarybreastandovariancancer\\nsyndrome(BRCA genevariants),Lynchsyndrome\\n• Breast,ovarian,endometrial,colon,orpancreaticcancerin\\nfirst-orsecond-degreerelatives\\n• Clinicalexamination(includingpelvicexamination)\\n• Serumtumormarkers\\n• CA125\\n• CA19-9andCEA(forsuspected[basedonimagingfindings]or\\nconfirmedmucinoushistology)\\n• Imaging\\n• Pelvicultrasoundbyexpertexaminer(tocharacterizeadnexal\\nmasses,especiallyinearly-stagedisease)\\n• Computedtomographyofthechest,abdomen,andpelvis\\nwithoralandintravenouscontrast(forclinicalstagingand\\nsurgicalplanning)\\n• Additionaltoolsinselectedcases\\n• Abdominaldiffusion-weightedmagneticresonanceimaging,\\nlaparoscopy,orminilaparotomy(toassessextentof\\nperitonealcarcinomatosisandresectability)\\n• Total-body18F-fluorodeoxyglucose–positronemission\\ntomography(tocharacterizesuspiciousdistantlymphnodes\\nandextra-abdominalsites)\\n• Endoscopy(esophagogastroduodenoscopyandcolonoscopy)if\\nsuspected(basedonelevatedCA19-9,elevatedCEA,CA125/CEA\\nratio/H1134925,and/orimagingfindings)orconfirmedmucinous\\nhistology,toexcludeprimarygastrointestinaltumors\\n• Pathologicalexaminationofadequatetumorsamplefrom\\ndiagnosticcorebiopsyorsurgicalspecimen\\na\\n• Cytologicalassessmentofascitesorpleuraleffusion,ifpresent\\n• Geneticandmoleculartesting\\n• Germlinetestingfor BRCA1/2 andotherovariancancer\\nsusceptibilitygenesforallpatientswithepithelialovarian\\ncancer(toscreenforhereditarybreastandovariancancer\\nsyndromeandinformtreatmentdecisions).Inpatients\\nwithoutagermlinepathogenic BRCA1/2 variant,somatic\\ntumortestingfor BRCA1/2 shouldbeperformedtoguide\\ntreatmentdecisions.\\n• Tumortestingforhomologousrecombinationdeficiencyfor\\nadvancedhigh-gradeserousorendometrioidovariancancers\\n(toinformtreatmentdecisions)\\n• Tumormismatchrepairsystemtesting—specifically,\\nimmunohistochemistryandmicrosatelliteinstabilitytesting—\\nforpatientswithendometrioid,clearcell,ormucinous\\nsubtypes(toscreenforLynchsyndrome)\\nAbbreviations:CA125,carbohydrateantigen125;CA19-9,carbohydrate\\nantigen19-9;CEA,carcinoembryonicantigen.\\na Preoperativebiopsyshouldbeavoidedinpresumedearly-stagediseaseto\\npreventspillingmalignantcellsintotheperitonealcavity.\\nClinical Review & EducationReview OvarianCancer:AReview\\n1282 JAMA October14,2025 Volume334,Number14 (Reprinted) jama.com\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 5, 'page_label': '6'}, page_content='fertility-sparingsurgeryforunilateralepithelialovariancancer(stage\\nIA,n = 126;stageIC,n = 85),5-yearoverallandrecurrence-freesur-\\nvivalvariedbysubtype,withthehighestrates(100%and97.8%)\\nforstageIAgrade1to2diseaseandlowerrates(66.7%and66.0%)\\nforstageICgrade3disease.\\n48\\nThe benefit of adjuvant chemotherapy in early-stage ovarian\\ncancer depends on stage, histotype, and tumor grade, and deci-\\nsionsaboutadjuvantchemotherapyshouldinvolveconsiderations\\nof age, comorbidities, performance status, and possible adverse\\nevents(Table;eTable2inthe Supplement).\\n49-54,65,67,68ACochrane\\nmeta-analysisof5RCTsincluding1277womenwithearly-stageovar-\\niancancerfoundthatadjuvantplatinum-basedchemotherapyim-\\nproved 10-year overall survival compared with no chemotherapy\\n(hazardratio,0.72;95%CI,0.57-0.92;absolutedatanotavailable).\\n97\\nAmongpatientswithhigh-riskdisease(stageIAgrade3,stageIB-IC\\ngrade 2-3, stage II, or any clear cell ovarian cancer), those who\\nFigure 2. Primary Treatment for Early-Stage and Advanced-Stage Ovarian Cancer\\n• Referral to gynecologic oncologist for consultation\\n• Initial assessment performed using serum CA125, pelvis ultrasonography, and computed tomography of the chest, abdomen, and pelvis\\nPatient with newly diagnosed abdominal or pelvic mass suspicious for ovarian cancer \\nAdvanced-stage ovarian cancer (stage III-IV)Presumed early-stage ovarian cancer (stage I-II)\\n• Hysterectomy, bilateral salpingo-oophorectomy, omentectomy, \\npelvic and para-aortic lymphadenectomy, random peritoneal \\nbiopsies, and peritoneal fluid cytology\\n• Fertility-sparing surgery may be considered for patients \\nof childbearing potential with stage I disease\\nConfirmed early-stage disease on definitive pathology\\nComplete surgical staging\\nNeoadjuvant systemic therapy\\n• Intravenous carboplatin AUC 5/6 + \\nintravenous paclitaxel, 175 mg/m2, \\nevery 3 wkb ± intravenous bevacizumab, \\n15 mg/kg, every 3 wkc (3-4 cycles)\\nInterval cytoreductive surgery\\n• For stage III disease, HIPEC \\nmay be considered during surgery\\nSystemic therapy\\n• Additional 2-3 cycles of \\nchemotherapy ± bevacizumabc \\nIndividualized maintenance therapy based on genetic and molecular testing\\nCore biopsy for histologic \\nconfirmation of ovarian cancer\\nReassess disease resectability \\nand patient performance status\\n• Albumin <3.5 g/dL\\n• Age ≥80 y\\n• Age 75-79 y and at least 1 of the following:\\n• ECOG-PS >1 or ASA score 3-4\\n• Stage IV (multiple liver or lung metastases)\\n• Complex surgery likely (more than hysterectomy, \\nbilateral salpingo-oophorectomy, and omentectomy)\\nAdjuvant systemic therapy\\n• Intravenous carboplatin \\nAUC 5/6 + intravenous \\npaclitaxel, 175 mg/m2, \\nevery 3 wkb ± intravenous \\nbevacizumab, 15 mg/kg, \\nevery 3 wk\\nc (6 cycles)\\nPrimary cytoreductive\\nsurgery (removal of all \\nvisible cancer in the \\nabdominal cavity)\\nNot high risk High risk\\nFit for surgery Unfit for surgery\\nRisk-based triage algorithm—High risk for surgical morbidity or \\nmortality if 1 of the following criteria is meta:\\nLow-grade serous \\novarian cancer (stage IC)\\nor\\nLow-grade endometrioid \\novarian cancer (stage IC)\\nor\\nClear cell ovarian \\ncancer (stage IA-IC1)\\nor\\nExpansile mucinous \\novarian cancer (stage IC)\\nor\\nInfiltrative mucinous \\novarian cancer (stage IA)\\nOptional therapy\\nLow-grade serous \\novarian cancer \\n(stage IA-IB)\\nor\\nLow-grade \\nendometrioid ovarian \\ncancer (stage IA-IB)\\nor\\nExpansile mucinous \\novarian cancer \\n(stage IA-IB)\\nObservation\\nHigh-grade serous ovarian cancer \\n(any stage)\\nor\\nHigh-grade endometrioid ovarian \\ncancer (any stage)\\nor\\nClear cell ovarian cancer \\n(stage IC2-IC3)\\nor\\nInfiltrative mucinous ovarian \\ncancer (stage IB-IC)\\nor\\nAny histotype stage II\\nAdjuvant chemotherapy\\n• Intravenous carboplatin AUC 5/6 + \\nintravenous paclitaxel, 175 mg/m\\n2, \\nevery 3 wk (6 cycles; 3 cycles are \\nacceptable unless high-grade serous \\novarian cancer, high-grade \\nendometrioid ovarian cancer, \\nor stage IC-II)\\nor\\n• Intravenous carboplatin AUC 5/6 \\nalone every 3 wk (6 cycles)\\nBRCA gene variant \\nor\\nPositive homologous \\nrecombination deficiency test\\n• PARP inhibitor alone (olaparib \\nor niraparib)\\n• Olaparib + bevacizumab \\nNegative homologous \\nrecombination deficiency test\\nor\\nUnknown\\n• Bevacizumab\\n• Niraparib \\n• Observation\\nPrompt referral to specialized center with multidisciplinary team\\nASAindicatesAmericanSocietyofAnesthesiologists;AUC,areaunderthe\\ncurve;CA125,carbohydrateantigen125;ECOG-PS,EasternCooperative\\nOncologyGroupperformancestatus;HIPEC,hyperthermicintraperitoneal\\nchemotherapy;andPARP,poly–adenosinediphosphateribosepolymerase.\\naConsiderneoadjuvantchemotherapyalsoifrecent(<6months)venous\\nthromboembolism,myocardialinfarction,stent,orlaparotomy.\\nbAnalternativeweeklychemotherapyregimenofcarboplatinAUC2and\\npaclitaxel,60mg/m 2,isareasonableoptionforfrailpatientswhocannot\\ntoleratetheevery-3-weekschedule.\\ncBevacizumabshouldbestopped6to8weeksbeforesurgeryandreinitiated4\\nto8weeksaftersurgeryduetowound-healingissues.\\nOvarianCancer:AReview Review Clinical Review & Education\\njama.com (Reprinted) JAMA October14,2025 Volume334,Number14 1283\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 6, 'page_label': '7'}, page_content='Table. Summary of Evidence on Standard Treatment Strategies for Ovarian Cancer\\nSetting of therapy\\nSelected phase 3 randomized\\nclinical trials\\nNo. of randomized\\npatients Survival data a Common serious adverse eventsa,b Conclusions\\nEarly-stage ovarian cancer\\nAdjuvantsystemicchemotherapy\\naftersurgery\\nACTION,49,50ICON1,51,52\\nGOG-157,53ICON354\\n1765 10-Yearoverallsurvival:73%-82%for\\nadjuvantplatinum-basedchemotherapyvs\\n64%-76%forobservation\\nNeutropenia(86%),thrombocytopenia\\n(21%),anemia(7%),neurotoxicity(grade\\n≥2:18%-29%),nausea/vomiting(9%-10%),\\nallergy(3%),alopecia(grade2:64%-80%)\\nAdjuvantplatinum-basedchemotherapy\\n(carboplatinandpaclitaxel)iswelltolerated\\nandimproveslong-termoverallsurvival\\nAdvanced-stage ovarian cancer\\nCytoreductivesurgery EORTC55971,\\n55CHORUS,56\\nJCOG0602,57SCORPION58\\n1654 Medianoverallsurvival:24-44monthsfor\\nintervalcytoreductivesurgeryvs23-49\\nmonthsforprimarycytoreductivesurgery\\n28-Daymortality(<1%vs1%-6%),\\nhemorrhage(4%-6%vs3%-7%),venous\\nthrombosis(0%-3%vs2%-5%),infection\\n(1%-3%vs1%-8%),gastrointestinalfistula\\n(<1%vs1%-3%)\\nSurvivalwithneoadjuvantchemotherapy\\nfollowedbyintervalcytoreductivesurgeryis\\nnotinferiortoprimarycytoreductivesurgery,\\nwhich,however,carriesahigherriskof\\nperioperativecomplications\\nFirst-linesystemic\\nchemotherapy\\nICON3,\\n54GOG-158,59\\nAGO-OVAR-3,60SCOTROC1,61\\nJGOG3016,62,63MITO-2,64\\nMITO-7,65GOG-262,66\\nICON867,68\\n8847 Medianoverallsurvival:36-57monthsfor\\ncarboplatin-paclitaxelvs35-49monthsfor\\notherplatinum-basedregimens\\nNeutropenia(11%-88%vs3%-94%),\\nthrombocytopenia(2%-39%vs1%-16%),\\nanemia(1%-44%vs1%-11%),neurotoxicity\\n(3%-30%vs1%-13%),nausea/vomiting\\n(3%-10%vs10%-23%),allergy(2%-3%vs\\n3%),alopecia(grade2:60%-80%vs\\n5%-76%)\\nGiventhefavorabletoxicityprofileand\\nefficacy,carboplatin-paclitaxelisthe\\nstandardfirst-linechemotherapy\\nBevacizumabconcurrentwith\\nchemotherapyandas\\nmaintenancetherapy\\nICON7,\\n69,70GOG-021871,72 3401 MedianPFS:14-24monthsfor\\nchemotherapyplusbevacizumabvs10-22\\nmonthsforchemotherapyalone\\nHypertension(18%-23%vs2%-7%),\\nproteinuria(1%-2%vs<1%),gastrointestinal\\nperforation(1%vs<1%),bleeding(1%-2%\\nvs<1%),venousthromboembolism(7%vs\\n6%),wounddehiscence(1%-3%vs<1%-3%)\\nAdditionofbevacizumabtoplatinum-based\\nchemotherapyfollowedbybevacizumab\\nmaintenancetherapyprolongsPFSby2-4\\nmonths\\nPARPinhibitor(olaparibor\\nniraparib)maintenancetherapy\\nafterresponsetoplatinum-based\\nchemotherapy\\nSOLO1,\\n73,74PAOLA-1,75,76\\nPRIMA77,78\\n1930 • BRCA variants:7-yearoverallsurvival:\\n67%forolaparibvs47%forplacebo\\n• HRDpositive:medianoverallsurvival:75\\nmonthsforolaparibplusbevacizumabvs\\n57monthsforbevacizumab\\n• HRDnegative:medianPFS:8.4months\\nforniraparibvs5.4monthsforplacebo\\nNeutropenia(6%-15%),anemia(17%-31%),\\nthrombocytopenia(1%-29%),fatigue\\n(2%-5%),vomiting(1%),diarrhea(2%-3%),\\nabdominalpain(1%-2%),hypertension(7%)\\n• Olaparibimproveslong-termoverall\\nsurvivalinpatientswith BRCA variants\\n• Olaparibplusbevacizumabimproves\\noverallsurvivalinpatientswith\\nHRD-positiveovariancancer\\n• NiraparibimprovesPFSregardlessof BRCA\\norHRDstatus\\nPlatinum-sensitive recurrent ovarian cancer\\nSecondarycytoreductive\\nsurgery\\nGOG-213,\\n79DESKTOPIII,80\\nSOC-181,82\\n1249 Medianoverallsurvival:51-58monthsfor\\nsecondarycytoreductivesurgeryfollowedby\\nchemotherapyvs46-65monthsfor\\nplatinum-basedchemotherapy\\nSurgicalmorbidityat30dayswas5%-9%;\\npatient-reportedqualityoflifedidnotdiffer\\nbetweenthe2groups\\nSecondarycytoreductivesurgerybefore\\nplatinum-basedchemotherapymaybe\\nconsideredifcompleteresectionof\\nmacroscopicdiseaseisachievable\\nSecond-linesystemic\\nchemotherapy\\nICON4,\\n83Pfistereretal, 84\\nCALYPSO,85,86OCEANS,87,88\\nGOG-213,89AGO-OVAR-2.2190\\n3974 Medianoverallsurvival:18-42monthsfor\\nplatinum-baseddoublets(carboplatinplus\\npaclitaxel,gemcitabine,orpegylated\\nliposomaldoxorubicin)withorwithout\\nbevacizumab\\nNeutropenia(70%),anemia(27%),\\nthrombocytopenia(35%),fatigue(2%),\\nvomiting(3%),neurotoxicity(grade≥2:\\n20%)\\nPlatinum-baseddoubletchemotherapywith\\norwithoutbevacizumabisthestandard\\ntreatmentforplatinum-sensitiverecurrent\\novariancancer\\nPARPinhibitor(olaparib,\\nniraparib,orrucaparib)\\nmaintenancetherapy\\nSOLO2,\\n91,92NOVA,93ARIEL-394 1412 BRCA variants:medianoverallsurvival:\\n41-52monthsforPARPinhibitorvs38-39\\nmonthsforplacebo\\nNeutropenia(5%-20%),anemia(19%-25%),\\nthrombocytopenia(1%-34%),hypertension\\n(8%),fatigue(4%-8%),vomiting(2%-4%),\\ndiarrhea(1%),abdominalpain(1%-3%)\\nSecond-linePARPinhibitormaintenance\\ntherapyimprovesoverallsurvivalinpatients\\nwithBRCA variantswhodidnotpreviously\\nreceiveaPARPinhibitor\\n(continued)\\nClinical Review & EducationReview OvarianCancer:AReview\\n1284 JAMA October14,2025 Volume334,Number14 (Reprinted) jama.com\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 7, 'page_label': '8'}, page_content='receivedadjuvantchemotherapyhadimproved10-yearoverallsur-\\nvival compared with those not treated with adjuvant chemo-\\ntherapy (hazard ratio, 0.52; 95% CI, 0.33-0.81; absolute data not\\navailable).\\n97\\nTreatmentofAdvanced-StageOvarianCancer\\nTreatmentofadvanced-stageepithelialovariancancertypicallyin-\\nvolves a combination of cytoreductive surgery, systemic chemo-\\ntherapy,andindividualizedmaintenancetherapy(Figure2).\\n2,4For\\npatientsatlowriskofperioperativecomplicationsandresectable\\nadvanced-stageovariancancerbasedonimaging,primarycytore-\\nductive surgery (hysterectomy with bilateral salpingo-\\noophorectomy and omentectomy) is recommended before adju-\\nvant platinum-based chemotherapy.\\n98-100 Patients with ovarian\\ncancerinvolvingotherorgansmayrequireresectionofenlargedret-\\nroperitonealpelvicandpara-aorticlymphnodes,spleen,bowel,and\\ndiaphragm.\\n98-101\\nPatientswithadvanced-stageovariancancerathighriskofperi-\\noperativecomplicationsorunlikelytoachievecompletecytoreduc-\\ntionatprimarysurgeryshouldreceiveneoadjuvantchemotherapy\\nfollowedbyintervalcytoreductivesurgeryandadjuvantsystemic\\nchemotherapy(Table;eTable2inthe Supplement).\\n55-58,102Ameta-\\nanalysisof4phase3RCTsincluding1692patientswithadvanced-\\nstageovariancancerfoundthatthosewhounderwentintervalcy-\\ntoreductive surgery (n = 847) had similar overall survival (hazard\\nratio,0.96;95%CI,0.86-1.08)andlowerratesofpostoperativecom-\\nplications(relativerisk,0.22;95%CI,0.13-0.38)\\n103comparedwith\\nthosewhounderwentprimarycytoreductivesurgery(n = 845). 103\\nForpatientswithstageIIIovariancancer,hyperthermicintraperito-\\nneal chemotherapy, which involves delivery of heated platinum-\\nbasedchemotherapyintotheabdominalcavityduringsurgery,may\\nbe offered during interval cytoreductive surgery (eTable 2 in the\\nSupplement).\\n102,104-107\\nFirst-line systemic chemotherapy for advanced-stage ovarian\\ncancer consists of carboplatin plus paclitaxel every 3 weeks for 6\\ncycles(Table;eTable2inthe Supplement).\\n4,59-68,108,109Patientsun-\\ndergoingprimarycytoreductivesurgeryreceive6cyclesofchemo-\\ntherapy,startingwithin6weeksaftersurgery.Patientsundergoing\\ninterval cytoreductive surgery typically receive 3 to 4 chemo-\\ntherapycyclespriortosurgeryand2to3aftersurgery.\\n102Aweekly\\nregimenofcarboplatinandpaclitaxeldeliveringthesamecumula-\\ntivedoseastheevery-3-weekscheduleisareasonablealternative\\nforpatientswhoareunabletotoleratethestandardschedule.\\n65,67,68\\nTwolargephase3RCTs(GOG-0218andICON7)demonstratedthat\\nadding bevacizumab to chemotherapy followed by bevacizumab\\nmaintenancetherapyforupto22cycles(15months)inpatientswith\\nadvanced-stage ovarian cancer prolonged mean progression-free\\nsurvival (PFS) by 2 to 4 months vs chemotherapy alone (GOG-\\n0218, 14.1 months vs 10.3 months; median follow-up, 17 months;\\nICON7,24.1monthsvs22.4months;medianfollowup,28months)\\n(Table;eTable2inthe Supplement).\\n69,71Whenthese2trialswere\\ncombined in a meta-analysis of 3401 patients, the bevacizumab\\ngrouphadimprovedPFS(hazardratio,0.72;95%CI,0.65-0.81;ab-\\nsolutedatanotavailable)comparedwithchemotherapyalone,al-\\nthough with no overall survival improvement.\\n110 An exploratory\\nanalysisofapredefinedsubgroupof502high-riskpatients(stageIV,\\nTable. Summary of Evidence on Standard Treatment Strategies for Ovarian Cancer (continued)\\nSetting of therapy\\nSelected phase 3 randomized\\nclinical trials\\nNo. of randomized\\npatients Survival data a Common serious adverse eventsa,b Conclusions\\nPlatinum-resistant recurrent ovarian cancer\\nSecond-linesystemic\\nchemotherapy\\nAURELIA95 361 MedianPFS:6.7monthsforsingle-agent\\nchemotherapy(pegylatedliposomal\\ndoxorubicin,weeklypaclitaxel,ortopotecan)\\nplusbevacizumabvs3.4monthsfor\\nchemotherapyalone\\nHypertension(grade≥2:20%vs7%),\\nproteinuria(2%vs0%),gastrointestinal\\nperforation(2%vs0%),fistula/abscess(2%\\nvs0%),thromboembolism(5%vs4%)\\nNon–platinum-basedsingle-agent\\nchemotherapyplusbevacizumabisthe\\ntraditionalstandardtreatmentfor\\nplatinum-resistantrecurrentovariancancer\\nMirvetuximabsoravtansine\\n(anti–folatereceptorα\\nantibody-drugconjugate)\\nMIRASOL\\n96 453 Medianoverallsurvival:16.5monthsfor\\nmirvetuximabvs12.7monthsfor\\nsingle-agentchemotherapy\\nBlurredvision(7.8%vs0%),keratopathy\\n(9.2%vs0%),dryeye(3.2%vs0%),\\nneutropenia(0.9%vs17.4%),anemia(0.9%\\nvs10.1%),abdominalpain(2.8%vs1.4%)\\nMirvetuximabisthenewstandardtreatment\\nforpatientswithplatinum-resistantrecurrent\\novariancancerwithhighfolatereceptorα\\nexpression\\nAbbreviations:HRD,homologousrecombinationdeficiency;PARP,poly–adenosinediphosphateribose\\npolymerase;PFS,progression-freesurvival.\\na Survivaldataratesandadverseeventratesacrossrandomizedclinicaltrialsarereportedasranges.\\nbGrade3orhigheraccordingtoCommonTerminologyCriteriaforAdverseEventsversion5.0unlessotherwise\\nspecified.\\nOvarianCancer:AReview Review Clinical Review & Education\\njama.com (Reprinted) JAMA October14,2025 Volume334,Number14 1285\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 8, 'page_label': '9'}, page_content='inoperable stage III, or >1-cm residual disease after surgery for\\nstage III ovarian cancer) in ICON7 showed improvement in mean\\noverallsurvivalinthebevacizumabgroupcomparedwithchemo-\\ntherapy alone (39.3 months vs 34.5 months; hazard ratio, 0.78;\\n95%CI,0.63-0.97).\\n70\\nBevacizumaband/orPARPinhibitor(olapariborniraparib)main-\\ntenancetherapyafterprimarytreatmentmaybeconsideredforpa-\\ntientswithadvanced-stageovariancancerdependingontheirre-\\nsponse to chemotherapy, histology, BRCA /HRD status, and\\ncomorbidities(Table;eTable2inthe Supplement).\\n69-78,111-113PARP\\ninhibitorsareguidelinerecommendedformaintenancetherapyin\\npatientswithhigh-gradeserousorendometrioidovariancancerwho\\nrespondtoplatinum-basedchemotherapy,especiallyin BRCA vari-\\nant or HRD-positive tumors.\\n2,4 PARP inhibitors are administered\\norally,witholaparibdosedat300mgtwicedailyfor2yearsandni-\\nraparibdosedat200mgto300mgoncedailyfor3years.\\n6Olaparib\\nprovided a 7-year overall survival benefit compared with placebo\\n(67.0% vs 46.5%; median overall survival, not reached vs 75.2\\nmonths;hazardratio,0.55;95%CI,0.40-0.76)inaphase3RCTof\\n391patientswith BRCA varianthigh-gradeadvanced-stageovarian\\ncancer(eTable2inthe Supplement).\\n73,74Inanotherphase3trialof\\n806 patients with advanced-stage ovarian cancer responding to\\nplatinum-based chemotherapy and bevacizumab, addition of\\nolaparib to bevacizumab improved 5-year overall survival in pa-\\ntients with HRD-positive tumors (includingBRCA variants) com-\\npared with placebo plus bevacizumab (65.5% vs 48.4%; median\\noverall survival, 75.2 months vs 57.3 months; hazard ratio, 0.62;\\n95%CI,0.45-0.85).Five-yearoverallsurvivalrateswere73.2%vs\\n53.8%(medianoverallsurvival,75.2monthsvs66.9months;haz-\\nardratio,0.60;95%CI,0.39-0.93)forthosewith BRCAvariantsand\\n54.7% vs 44.2% (median overall survival, not reached vs 52.0\\nmonths;hazardratio,0.71;95%CI,0.45-1.13)forthosewithout BRCA\\nvariants(eTable2inthe Supplement).\\n75,76Threeothertrialsinves-\\ntigating PARP inhibitor maintenance therapy found that PFS was\\nhighestamongpatientswith BRCAvarianttumorsfollowedbyHRD-\\npositive tumors and lowest for those with HRD-negative tumors\\n(eTable2inthe Supplement).\\n77,111,112\\nPrognosisandSurveillance\\nOverall5-yearsurvivalratesare70%to95%forpatientstreatedfor\\nearly-stageovariancancerand10%to40%forthosewithadvanced-\\nstagedisease.\\n114 Morethan80%ofpatientswithadvanced-stage\\novariancancerhaveinitialcompleteremission(nodiseaseonpost-\\ntreatmentimaging).PARPinhibitormaintenancetherapyafterplati-\\nnum-basedchemotherapyimproves5-yearoverallsurvivalratesto\\napproximately70%inpatientswith BRCA variantorHRD-positive\\nadvanced-stageovariancancer.\\n74,76\\nTherearecurrentlynoevidence-basedtreatmentfollow-uppro-\\ntocols,sosurveillanceshouldbeindividualizedbasedonstageofdis-\\nease,histology,geneticvariants,andradiologicalandserumtumor\\nmarkerresponsetoprimarytreatment.Patientswithcompletere-\\nmissionafterprimarytreatmentshouldreceiveclinicalreevaluation\\nandCA125testing(ifinitiallyelevated)every2to4monthsforthefirst\\n2years,every3to6monthsduringyears3to5,andthenannually.\\nComputedtomographicimagingshouldbeindividualizedbasedon\\nrecurrenceriskorclinicalsuspicionofrecurrence(eg,abdominalpain,\\nbloating,weightloss,increasingCA125level).\\n2,4\\nOvarianCancerRelapse\\nApproximately75%ofpatientswithadvanced-stageovariancan-\\ncerand10%to30%ofpatientswithearly-stageovariancancerre-\\nlapsewithin2years.\\n2,4Decisionsaboutadditionaltherapy,enroll-\\ninginclinicaltrials,orinitiatingpalliativecareshouldbemadeonan\\nindividualbasis(Figure 3).Recurrentdiseasetreatmentdependson\\nFigure 3. Treatment Algorithm for Recurrent Ovarian Cancer\\n• Multidisciplinary assessment of clinical factors (eg, platinum-free interval, prior treatments, age, performance status) \\nand patient goals\\n• Early referral to palliative care\\nPatient with recurrent ovarian cancer \\nStandard treatment Best supportive careEnrollment in clinical trials\\n(strongly encouraged)\\nPlatinum-resistant disease\\n(platinum-free interval <6 mo)\\n• Mirvetuximab soravtansine\\n(anti-FRα antibody-drug conjugate)\\n• Non–platinum-based single-agent\\nchemotherapy ± bevacizumab \\nPlatinum-sensitive disease\\n(platinum-free interval ≥6 mo)\\n• Platinum-based chemotherapy\\n± bevacizumab \\n• Consider eligibility for secondary\\ncytoreductive surgery\\nIndividualized maintenance therapy based on genetic and molecular testing \\nBRCA gene variant \\n• PARP inhibitor (olaparib, \\nniraparib or rucaparib) preferred\\n• Bevacizumab\\nBRCA wild type\\n• Bevacizumab\\n• Observation FRαindicatesfolatereceptorα;\\nPARP,poly–adenosinediphosphate\\nribosepolymerase.\\nClinical Review & EducationReview OvarianCancer:AReview\\n1286 JAMA October14,2025 Volume334,Number14 (Reprinted) jama.com\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 9, 'page_label': '10'}, page_content='theplatinum-freeinterval—thetimefromthelastdoseofplatinum-\\nbasedchemotherapytorecurrence—whichpredictsresponsetosub-\\nsequentplatinumtherapy.Atfirstrelapse,approximately80%of\\npatientshaveplatinum-sensitivedisease(platinum-freeinterval /H113506\\nmonths)and20%haveplatinum-resistantdisease(platinum-free\\ninterval<6months).\\n115 Witheachremissionandrecurrence,plati-\\nnum-freeintervalsshorten,andabout60%to75%ofpatientswith\\nrecurrentdiseasedevelopplatinumresistance.\\n115Patientswithre-\\ncurrentovariancancershouldundergotumortestingforfolatere-\\nceptorα(FRα[ FOLR1])and ERBB2 (previouslyHER2/neu)toiden-\\ntifypotentialtargetsfortherapy.\\nPlatinum-Sensitive Ovarian Cancer\\nForpatientswithplatinum-sensitiverecurrentovariancancer,plati-\\nnum-based chemotherapy (carboplatin plus pegylated liposomal\\ndoxorubicin, gemcitabine, or paclitaxel) with or without bevaci-\\nzumabisrecommended(Table;eTable2inthe Supplement).\\n83-90\\nThemedianPFSwiththistreatmentis8to13months. 115 Second-\\nary cytoreductive surgery prior to chemotherapy may be consid-\\neredforplatinum-sensitivediseaseifimagingandperformancesta-\\ntus indicate potentially resectable disease (Table; eTable 2 in the\\nSupplement).\\n79-82\\nClinical trial data support use of maintenance therapy with a\\nPARPinhibitor(olaparib,niraparib,orrucaparib)forrecurrentdis-\\neaseinPARPinhibitor–naivepatientswithBRCA1/2genevariantswho\\nresponded to platinum-based chemotherapy (Table).\\n91-94,116,117A\\nphase 3 RCT including 295 patients withBRCA gene variants and\\nplatinum-sensitiverecurrentovariancancershowedthatolaparib\\nmaintenance provided a median overall survival benefit of 12.9\\nmonthscomparedwithplacebo(51.7monthsvs38.8months;haz-\\nardratio,0.74;95%CI,0.54-1.00).\\n91,92Niraparibandrucaparibhave\\nsimilarresults(eTable2inthe Supplement).93,94,116,117\\nForpatientswithrecurrentovariancancerwithouta BRCA vari-\\nantorwhoreceivedPARPinhibitorasfirst-linetreatment,bevaci-\\nzumabmaintenancecanbeconsidered.\\n118Aphase3RCTincluding\\n484patientswithfirstplatinum-sensitiverecurrencereportedthat\\naddingbevacizumabtoplatinum-basedchemotherapyfollowedby\\nbevacizumabmaintenanceimprovedmedianPFS(12.4monthsvs\\n8.4months;hazardratio,0.48;95%CI,0.38-0.61)comparedwith\\nchemotherapyalone(eTable2inthe Supplement).\\n87,88\\nPlatinum-Resistant Ovarian Cancer\\nPlatinum-resistant recurrent ovarian cancer may be treated with\\nsingle-agent non–platinum-based chemotherapy (pegylated lipo-\\nsomaldoxorubicin,gemcitabine,weeklypaclitaxel,ortopotecan),\\nassociatedwithamedianPFSof3to9months.\\n95,115Combiningbeva-\\ncizumabwithchemotherapyimprovedPFScomparedwithchemo-\\ntherapy alone in a phase 3 RCT of 361 patients with platinum-\\nresistantrecurrentovariancancer(6.7monthsvs3.4months;hazard\\nratio,0.48;95%CI,0.38-0.61)(Table;eTable2inthe Supplement).\\n95\\nForthe30%to35%ofpatientswithplatinum-resistantovar-\\niancancersexhibitinghighFRαexpressionwhohavereceived1to\\n3roundsofanytypeofanticancertherapy,theUSFoodandDrug\\nAdministration(FDA)hasapprovedanantibody-drugconjugatetar-\\ngetingFRα,mirvetuximabsoravtansine.\\n96,119Inaphase3RCTof453\\npatients,mirvetuximabprovidedamedianoverallsurvivalbenefit\\ncompared with conventional chemotherapy (16.5 months vs 12.7\\nmonths;hazardratio,0.67;95%CI,0.50-0.89)(Table;eTable2in\\nthe Supplement).\\n96 Another recently FDA-approved antibody-\\ndrugconjugateistrastuzumabderuxtecan,whichtargets ERBB2and\\nprovides a treatment option for patients withERBB2-expressing\\novariancancer(eTable2inthe Supplement).120\\nPalliativeCare\\nTheNCCNguidelinesrecommendprovidingsupportivecareforall\\npatientswithovariancancer,includingreferralforpalliativecareas-\\nsessmentforthosewithadvanced-stageorrecurrentdisease.\\n4Pal-\\nliative care clinicians provide comprehensive multimodal symp-\\ntommanagement,addressingpain,mentalhealth,sexualhealth,and\\nmenopausal symptoms, and can facilitate transition to hospice\\ncare.\\n121 Useofpalliativecareoptimizesqualityoflifeandmayim-\\nprovesurvival.121,122Althoughnotspecifictoovariancancer,outpa-\\ntientpalliativecareforadvanced-stagecancershasbeenshownto\\nincrease1-yearsurvivalby14%(56%vs42%; P < .001)andmedian\\noverallsurvivalby4.6months(14.6monthsvs10.0months)com-\\nparedwithusualcare.\\n123\\nLimitations\\nThisReviewhasseverallimitations.First,itisnotasystematicre-\\nview, and the quality of included studies was not formally evalu-\\nated.Second,thisReviewdidnotincludelesscommonepithelialand\\nnonepithelial ovarian cancers. Third, treatments, outcomes, and\\nprognosticestimateswerederivedfromtrialsconductedprimarily\\ninNorthAmericaandEuropeandmaynotbegeneralizabletolower-\\nincomecountries.\\nConclusions\\nApproximately21000womenarediagnosedwithovariancancer\\nannuallyintheUS,andapproximately80%haveadvanced-stage\\novariancanceratdiagnosis.First-linetreatmentofearly-stageovar-\\niancancerissurgeryandadjuvantplatinum-basedchemotherapy.\\nTreatment of advanced-stage ovarian cancer includes cytoreduc-\\ntivesurgery,platinum-basedchemotherapy,andtargetedmainte-\\nnancetherapiessuchasbevacizumaband/orPARPinhibitors.\\nARTICLE INFORMATION\\nAccepted for Publication:May21,2025.\\nPublished Online:July21,2025.\\ndoi:10.1001/jama.2025.9495\\nConflict of Interest Disclosures:DrClibyreported\\nreceiptofaNationalInstitutesofHealthSPORE\\ngrantinovariancancer.Nootherdisclosureswere\\nreported.\\nSubmissions: Weencourageauthorstosubmit\\npapersforconsiderationasaReview.Please\\ncontactKristinWalter,MD,at kristin.walter@\\njamanetwork.org.\\nREFERENCES\\n1. SiegelRL,KratzerTB,GiaquintoAN,SungH,\\nJemalA.Cancerstatistics,2025. CA Cancer J Clin.\\n2025;75(1):10-45.doi:10.3322/caac.21871\\n2. González-MartínA,HarterP,LearyA,etal;\\nESMOGuidelinesCommittee.Newlydiagnosedand\\nrelapsedepithelialovariancancer:ESMOclinical\\nOvarianCancer:AReview Review Clinical Review & Education\\njama.com (Reprinted) JAMA October14,2025 Volume334,Number14 1287\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 10, 'page_label': '11'}, page_content='practiceguidelinefordiagnosis,treatmentand\\nfollow-up.Ann Oncol.2023;34(10):833-848.\\ndoi:10.1016/j.annonc.2023.07.011\\n3. HuepenbeckerSP,SunCC,FuS,etal.Factors\\nimpactingthetimetoovariancancerdiagnosis\\nbasedonclassicsymptompresentationinthe\\nUnitedStates. Cancer.2021;127(22):4151-4160.\\ndoi:10.1002/cncr.33829\\n4. NationalComprehensiveCancerNetwork.\\nOvariancancer(version1.2025).AccessedApril5,\\n2025.https://www.nccn.org/professionals/\\nphysician_gls/pdf/ovarian.pdf\\n5. Femalegenitaltumours.In:WHOClassification\\nofTumoursEditorialBoard,ed. World Health\\nOrganization Classification of Tumours.5thed.IARC\\nPress;2020.\\n6. CarusoG,TomaoF,ParmaG,etal.Poly\\n(ADP-ribose)polymeraseinhibitors(PARPi)in\\novariancancer:lessonslearnedandfuture\\ndirections.Int J Gynecol Cancer.2023;33(4):431-443.\\ndoi:10.1136/ijgc-2022-004149\\n7. BrayF,LaversanneM,SungH,etal.Global\\ncancerstatistics2022:GLOBOCANestimatesof\\nincidenceandmortalityworldwidefor36cancersin\\n185countries. CA Cancer J Clin.2024;74(3):229-263.\\ndoi:10.3322/caac.21834\\n8. CabasagCJ,FaganPJ,FerlayJ,etal.Ovarian\\ncancertodayandtomorrow:aglobalassessmentby\\nworldregionandHumanDevelopmentIndexusing\\nGLOBOCAN 2020.Int J Cancer. 2022;151(9):1535-\\n1541.doi:10.1002/ijc.34002\\n9. KotsopoulosJ,GronwaldJ,KarlanB,etal;\\nHereditaryOvarianCancerClinicalStudyGroup.\\nAge-specificovariancancerrisksamongwomen\\nwitha BRCA1 orBRCA2 mutation.Gynecol Oncol.\\n2018;150(1):85-91.doi:10.1016/j.ygyno.2018.05.011\\n10. DalyMB,PalT,MaxwellKN,etal.NCCN\\nguidelinesinsights:genetic/familialhigh-risk\\nassessment:breast,ovarian,andpancreatic,\\nversion2.2024. J Natl Compr Canc Netw.2023;21\\n(10):1000-1010.doi:10.6004/jnccn.2023.0051\\n11. WalshT,CasadeiS,LeeMK,etal.Mutationsin12\\ngenesforinheritedovarian,fallopiantube,and\\nperitonealcarcinomaidentifiedbymassively\\nparallelsequencing. Proc Natl Acad SciUSA .2011;\\n108(44):18032-18037.doi:10.1073/pnas.1115052108\\n12. BonadonaV,BonaïtiB,OlschwangS,etal;\\nFrenchCancerGeneticsNetwork.Cancerrisks\\nassociatedwithgermlinemutationsin MLH1,MSH2,\\nand MSH6 genes in Lynch syndrome.JAMA. 2011;\\n305(22):2304-2310.doi:10.1001/jama.2011.743\\n13. KonstantinopoulosPA,NorquistB,LacchettiC,\\netal.Germlineandsomatictumortestingin\\nepithelialovariancancer:ASCOguideline. J Clin Oncol.\\n2020;38(11):1222-1245.doi:10.1200/JCO.19.02960\\n14. BarnardME,FarlandLV,YanB,etal.\\nEndometriosistypologyandovariancancerrisk.\\nJAMA.2024;332(6):482-489.doi: 10.1001/jama.\\n2024.9210\\n15. TanhaK,MottaghiA,NojomiM,etal.\\nInvestigationonfactorsassociatedwithovarian\\ncancer:anumbrellareviewofsystematicreview\\nandmeta-analyses. J Ovarian Res.2021;14(1):153.\\ndoi:10.1186/s13048-021-00911-z\\n16. KarnezisAN,ChoKR,GilksCB,PearceCL,\\nHuntsmanDG.Thedisparateoriginsofovarian\\ncancers:pathogenesisandpreventionstrategies.\\nNat Rev Cancer.2017;17(1):65-74.doi:10.1038/nrc.\\n2016.113\\n17. KonstantinopoulosPA,CeccaldiR,ShapiroGI,\\nD’AndreaAD.Homologousrecombination\\ndeficiency:exploitingthefundamentalvulnerability\\nofovariancancer. Cancer Discov.2015;5(11):1137-1154.\\ndoi:10.1158/2159-8290.CD-15-0714\\n18. WalkerJL,PowellCB,ChenLM,etal.Societyof\\nGynecologicOncologyrecommendationsforthe\\npreventionofovariancancer. Cancer.2015;121(13):\\n2108-2120.doi:10.1002/cncr.29321\\n19. MarchettiC,DeFeliceF,PalaiaI,etal.\\nRisk-reducingsalpingo-oophorectomy:\\nameta-analysisonimpactonovariancancerrisk\\nandallcausemortalityin BRCA 1andBRCA 2\\nmutationcarriers.BMC Womens Health.2014;14:150.\\ndoi:10.1186/s12905-014-0150-5\\n20. vanBommelMH,IntHoutJ,VeldmateG,etal.\\nContraceptivesandcancerrisksin BRCA1/2\\npathogenicvariantcarriers:asystematicreviewand\\nmeta-analysis.Hum Reprod Update.2023;29(2):\\n197-217.doi:10.1093/humupd/dmac038\\n21. IodiceS,BarileM,RotmenszN,etal.Oral\\ncontraceptiveuseandbreastorovariancancerrisk\\nin BRCA1/2 carriers: a meta-analysis.Eur J Cancer.\\n2010;46(12):2275-2284.doi:10.1016/j.ejca.2010.04.\\n018\\n22. AmericanCollegeofObstetriciansand\\nGynecologists.ACOGcommitteeopinionNo.774:\\nopportunisticsalpingectomyasastrategyfor\\nepithelialovariancancerprevention. Obstet Gynecol.\\n2019;133(4):e279-e284.doi:10.1097/AOG.\\n0000000000003164\\n23. Mor-HadarD,WilailakS,BerekJ,McNallyOM;\\nFIGOCommitteeonWomen’sCancer.FIGOposition\\nstatementonopportunisticsalpingectomyasan\\novariancancerpreventionstrategy. Int J Gynaecol\\nObstet.2024;167(3):976-980.doi:10.1002/ijgo.15884\\n24. HavrileskyLJ,MoormanPG,LoweryWJ,etal.\\nOralcontraceptivepillsasprimarypreventionfor\\novariancancer:asystematicreviewand\\nmeta-analysis.Obstet Gynecol.2013;122(1):139-147.\\ndoi:10.1097/AOG.0b013e318291c235\\n25. GrossmanDC,CurrySJ,OwensDK,etal;US\\nPreventiveServicesTaskForce.Screeningfor\\novariancancer:USPreventiveServicesTaskForce\\nrecommendationstatement. JAMA.2018;319(6):\\n588-594.doi:10.1001/jama.2017.21926\\n26. SkatesSJ,GreeneMH,BuysSS,etal.Early\\ndetectionofovariancancerusingtheriskofovarian\\ncanceralgorithmwithfrequentCA125testingin\\nwomenatincreasedfamilialrisk—combinedresults\\nfromtwoscreeningtrials. Clin Cancer Res.2017;23\\n(14):3628-3637.doi:10.1158/1078-0432.CCR-15-2750\\n27. RosenthalAN,FraserLS,PhilpottS,etal;\\nUnitedKingdomFamilialOvarianCancerScreening\\nStudyCollaborators.Evidenceofstageshiftin\\nwomendiagnosedwithovariancancerduring\\nphaseIIoftheUnitedKingdomFamilialOvarian\\nCancerScreeningStudy. J Clin Oncol.2017;35(13):\\n1411-1420.doi:10.1200/JCO.2016.69.9330\\n28. LaiT,KesselB,AhnHJ,TeradaKY.Ovarian\\ncancerscreeninginmenopausalfemaleswitha\\nfamilyhistoryofbreastorovariancancer. J Gynecol\\nOncol.2016;27(4):e41.doi: 10.3802/jgo.2016.27.e41\\n29. HermsenBB,OlivierRI,VerheijenRH,etal.\\nNoefficacyofannualgynaecologicalscreeningin\\nBRCA1/2 mutationcarriers:anobservational\\nfollow-upstudy. Br J Cancer.2007;96(9):1335-1342.\\ndoi:10.1038/sj.bjc.6603725\\n30. GoffBA,MandelL,MuntzHG,MelanconCH.\\nOvariancarcinomadiagnosis. Cancer.2000;89\\n(10):2068-2075.doi:10.1002/1097-0142(20001115)\\n89:10<2068::AID-CNCR6>3.0.CO;2-Z\\n31. GoffBA,MandelLS,MelanconCH,MuntzHG.\\nFrequencyofsymptomsofovariancancerin\\nwomenpresentingtoprimarycareclinics. JAMA.\\n2004;291(22):2705-2712.doi:10.1001/jama.291.22.\\n2705\\n32. PeterE,HonnoratJ,DesestretV.\\nParaneoplasticneurologicsyndromeassociated\\nwithgynecologicandbreastmalignancies. Handb\\nClin Neurol.2024;200:409-417.doi:10.1016/B978-\\n0-12-823912-4.00014-1\\n33. TimmermanD,PlanchampF,BourneT,etal.\\nESGO/ISUOG/IOTA/ESGEconsensusstatementon\\npre-operativediagnosisofovariantumors. Int J\\nGynecol Cancer.2021;31(7):961-982.doi: 10.1136/\\nijgc-2021-002565\\n34. FischerovaD,PintoP,BurgetovaA,etal.\\nPreoperativestagingofovariancancer:comparison\\nbetweenultrasound,CTandwhole-body\\ndiffusion-weightedMRI(ISAACstudy). Ultrasound\\nObstet Gynecol.2022;59(2):248-262.doi: 10.1002/\\nuog.23654\\n35. TempanyCM,ZouKH,SilvermanSG,Brown\\nDL,KurtzAB,McNeilBJ.Stagingofadvanced\\novariancancer:comparisonofimaging\\nmodalities—reportfromtheRadiologicalDiagnostic\\nOncologyGroup. Radiology.2000;215(3):761-767.\\ndoi:10.1148/radiology.215.3.r00jn25761\\n36. BerekJS,RenzM,KehoeS,KumarL,\\nFriedlanderM.Canceroftheovary,fallopiantube,\\nandperitoneum:2021update. Int J Gynaecol Obstet.\\n2021;155(suppl1):61-85.doi: 10.1002/ijgo.13878\\n37. MichielsenK,DresenR,VanslembrouckR,etal.\\nDiagnosticvalueofwholebodydiffusion-weighted\\nMRIcomparedtocomputedtomographyfor\\npre-operativeassessmentofpatientssuspectedfor\\novariancancer. Eur J Cancer.2017;83:88-98.doi: 10.\\n1016/j.ejca.2017.06.010\\n38. FlemingND,NickAM,ColemanRL,etal.\\nLaparoscopicsurgicalalgorithmtotriagethetiming\\noftumorreductivesurgeryinadvancedovarian\\ncancer.Obstet Gynecol.2018;132(3):545-554.\\ndoi:10.1097/AOG.0000000000002796\\n39. KangSK,ReinholdC,AtriM,etal;ExpertPanel\\nonWomen’sImaging.ACRappropriatenesscriteria\\nstaging and follow-up of ovarian cancer.JA mC o l l\\nRadiol. 2018;15(5S):S198-S207. doi:10.1016/j.jacr.\\n2018.03.015\\n40. YuanY,GuZX,TaoXF,LiuSY.Computer\\ntomography,magneticresonanceimaging,and\\npositronemissiontomographyorpositronemission\\ntomography/computertomographyfordetection\\nofmetastaticlymphnodesinpatientswithovarian\\ncancer: a meta-analysis.Eur J Radiol. 2012;81(5):\\n1002-1006.doi:10.1016/j.ejrad.2011.01.112\\n41. WrightAA,BohlkeK,ArmstrongDK,etal.\\nNeoadjuvantchemotherapyfornewlydiagnosed,\\nadvancedovariancancer:SocietyofGynecologic\\nOncologyandAmericanSocietyofClinical\\nOncologyclinicalpracticeguideline. J Clin Oncol.\\n2016;34(28):3460-3473.doi:10.1200/JCO.2016.68.\\n6907\\n42. MedeirosLR,RosaDD,daRosaMI,Bozzetti\\nMC.AccuracyofCA125inthediagnosisofovarian\\ntumors:aquantitativesystematicreview. Eur J\\nClinical Review & EducationReview OvarianCancer:AReview\\n1288 JAMA October14,2025 Volume334,Number14 (Reprinted) jama.com\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 11, 'page_label': '12'}, page_content='Obstet Gynecol Reprod Biol.2009;142(2):99-105.\\ndoi:10.1016/j.ejogrb.2008.08.011\\n43. FerraroS,BragaF,LanzoniM,BoracchiP,\\nBiganzoliEM,PanteghiniM.Serumhuman\\nepididymisprotein4vscarbohydrateantigen125\\nforovariancancerdiagnosis:asystematicreview.\\nJ Clin Pathol.2013;66(4):273-281.doi: 10.1136/\\njclinpath-2012-201031\\n44. VergoteI,DeBrabanterJ,FylesA,etal.\\nPrognosticimportanceofdegreeofdifferentiation\\nandcystruptureinstageIinvasiveepithelial\\novariancarcinoma. Lancet.2001;357(9251):176-182.\\ndoi:10.1016/S0140-6736(00)03590-X\\n45. TungN,RickerC,MessersmithH,etal.\\nSelectionofgermlinegenetictestingpanelsin\\npatientswithcancer:ASCOguideline. J Clin Oncol.\\n2024;42(21):2599-2615.doi:10.1200/JCO.24.00662\\n46. MillerRE,LearyA,ScottCL,etal.ESMO\\nrecommendationsonpredictivebiomarkertesting\\nforhomologousrecombinationdeficiencyand\\nPARPinhibitorbenefitinovariancancer. Ann Oncol.\\n2020;31(12):1606-1622.doi:10.1016/j.annonc.2020.\\n08.2102\\n47. ColomboN,SessaC,duBoisA,etal;\\nESMO-ESGOOvarianCancerConsensus\\nConferenceWorkingGroup.ESMO-ESGO\\nconsensusconferencerecommendationson\\novariancancer:pathologyandmolecularbiology,\\nearlyandadvancedstages,borderlinetumoursand\\nrecurrentdisease. Ann Oncol.2019;30(5):672-705.\\ndoi:10.1093/annonc/mdz062\\n48. SatohT,HataeM,WatanabeY,etal.Outcomes\\noffertility-sparingsurgeryforstageIepithelial\\novariancancer:aproposalforpatientselection.\\nJ Clin Oncol.2010;28(10):1727-1732.doi: 10.1200/\\nJCO.2009.24.8617\\n49. TrimbosJB,VergoteI,BolisG,etal;\\nEORTC-ACTIONCollaborators.Impactofadjuvant\\nchemotherapyandsurgicalstaginginearly-stage\\novariancarcinoma:EuropeanOrganisationfor\\nResearchandTreatmentofCancer–Adjuvant\\nChemotherapyinOvarianNeoplasmtrial. J Natl\\nCancer Inst.2003;95(2):113-125.doi: 10.1093/jnci/\\n95.2.113\\n50. TrimbosB,TimmersP,PecorelliS,etal.Surgical\\nstagingandtreatmentofearlyovariancancer:\\nlong-termanalysisfromarandomizedtrial. J Natl\\nCancer Inst.2010;102(13):982-987.doi:10.1093/\\njnci/djq149\\n51. ColomboN,GuthrieD,ChiariS,etal;\\nInternationalCollaborativeOvarianNeoplasm\\n(ICON)Collaborators.InternationalCollaborative\\nOvarianNeoplasmtrial1:arandomizedtrialof\\nadjuvantchemotherapyinwomenwithearly-stage\\novariancancer.J Natl Cancer Inst.2003;95(2):125-132.\\ndoi:10.1093/jnci/95.2.125\\n52. CollinsonF,QianW,FossatiR,etal;ICON1\\nCollaborators.Optimaltreatmentofearly-stage\\novariancancer. Ann Oncol.2014;25(6):1165-1171.\\ndoi:10.1093/annonc/mdu116\\n53. BellJ,BradyMF,YoungRC,etal;Gynecologic\\nOncologyGroup.RandomizedphaseIIItrialofthree\\nversussixcyclesofadjuvantcarboplatinand\\npaclitaxelinearlystageepithelialovarian\\ncarcinoma:aGynecologicOncologyGroupstudy.\\nGynecol Oncol.2006;102(3):432-439.doi:10.1016/j.\\nygyno.2006.06.013\\n54. InternationalCollaborativeOvarianNeoplasm\\nGroup.Paclitaxelpluscarboplatinversusstandard\\nchemotherapywitheithersingle-agentcarboplatin\\norcyclophosphamide,doxorubicin,andcisplatinin\\nwomenwithovariancancer:theICON3randomised\\ntrial.Lancet.2002;360(9332):505-515.doi: 10.\\n1016/S0140-6736(02)09738-6\\n55. VergoteI,CoensC,NankivellM,etal;EORTC;\\nMRCCHORUSStudyInvestigators.Neoadjuvant\\nchemotherapyversusdebulkingsurgeryin\\nadvancedtubo-ovariancancers:pooledanalysisof\\nindividualpatientdatafromtheEORTC55971and\\nCHORUStrials. Lancet Oncol.2018;19(12):1680-1687.\\ndoi:10.1016/S1470-2045(18)30566-7\\n56. KehoeS,HookJ,NankivellM,etal.Primary\\nchemotherapyversusprimarysurgeryfornewly\\ndiagnosedadvancedovariancancer(CHORUS):an\\nopen-label,randomised,controlled,non-inferiority\\ntrial.Lancet\\n.2015;386(9990):249-257.doi:10.\\n1016/S0140-6736(14)62223-6\\n57. OndaT,SatohT,SaitoT,etal;JapanClinical\\nOncologyGroup.Comparisonoftreatment\\ninvasivenessbetweenupfrontdebulkingsurgery\\nversusintervaldebulkingsurgeryfollowing\\nneoadjuvantchemotherapyforstageIII/IVovarian,\\ntubal,andperitonealcancersinaphaseIII\\nrandomisedtrial:JapanClinicalOncologyGroup\\nStudyJCOG0602. Eur J Cancer.2016;64:22-31.\\ndoi:10.1016/j.ejca.2016.05.017\\n58. FagottiA,FerrandinaMG,VizzielliG,etal.\\nRandomizedtrialofprimarydebulkingsurgery\\nversusneoadjuvantchemotherapyforadvanced\\nepithelialovariancancer(SCORPION-\\nNCT01461850).Int J Gynecol Cancer.2020;30(11):\\n1657-1664.doi:10.1136/ijgc-2020-001640\\n59. OzolsRF,BundyBN,GreerBE,etal;\\nGynecologicOncologyGroup.PhaseIIItrialof\\ncarboplatinandpaclitaxelcomparedwithcisplatin\\nandpaclitaxelinpatientswithoptimallyresected\\nstageIIIovariancancer:aGynecologicOncology\\nGroupstudy. J Clin Oncol.2003;21(17):3194-3200.\\ndoi:10.1200/JCO.2003.02.153\\n60. duBoisA,LückHJ,MeierW,etal;\\nArbeitsgemeinschaftGynäkologischeOnkologie\\nOvarianCancerStudyGroup.Arandomizedclinical\\ntrialofcisplatin/paclitaxelversuscarboplatin/\\npaclitaxelasfirst-linetreatmentofovariancancer.\\nJ Natl Cancer Inst. 2003;95(17):1320-1329. doi:10.\\n1093/jnci/djg036\\n61. VaseyPA,JaysonGC,GordonA,etal;Scottish\\nGynaecologicalCancerTrialsGroup.PhaseIII\\nrandomizedtrialofdocetaxel-carboplatinversus\\npaclitaxel-carboplatinasfirst-linechemotherapyfor\\novariancarcinoma. J Natl Cancer Inst.2004;96\\n(22):1682-1691.doi:10.1093/jnci/djh323\\n62. KatsumataN,YasudaM,TakahashiF,etal;\\nJapaneseGynecologicOncologyGroup.\\nDose-densepaclitaxelonceaweekincombination\\nwithcarboplatinevery3weeksforadvanced\\novariancancer:aphase3,open-label,randomised\\ncontrolledtrial. Lancet.2009;374(9698):1331-1338.\\ndoi:10.1016/S0140-6736(09)61157-0\\n63. KatsumataN,YasudaM,IsonishiS,etal;\\nJapaneseGynecologicOncologyGroup.Long-term\\nresultsofdose-densepaclitaxelandcarboplatin\\nversusconventionalpaclitaxelandcarboplatinfor\\ntreatmentofadvancedepithelialovarian,fallopian\\ntube,orprimaryperitonealcancer(JGOG3016):\\narandomised,controlled,open-labeltrial. Lancet\\nOncol.2013;14(10):1020-1026.doi:10.1016/S1470-\\n2045(13)70363-2\\n64. PignataS,ScambiaG,FerrandinaG,etal.\\nCarboplatinpluspaclitaxelversuscarboplatinplus\\npegylatedliposomaldoxorubicinasfirst-line\\ntreatmentforpatientswithovariancancer:the\\nMITO-2randomizedphaseIIItrial. J Clin Oncol.2011;\\n29(27):3628-3635.doi:10.1200/JCO.2010.33.8566\\n65. PignataS,ScambiaG,KatsarosD,etal;\\nMulticentreItalianTrialsinOvariancancer(MITO-7);\\nGrouped’InvestigateursNationauxpourl’Etudedes\\nCancersOvariensetdusein(GINECO);MarioNegri\\nGynecologicOncology(MaNGO);European\\nNetworkofGynaecologicalOncologicalTrialGroups\\n(ENGOT-OV-10);GynecologicCancerInterGroup\\n(GCIG)Investigators.Carboplatinpluspaclitaxel\\nonceaweekversusevery3weeksinpatientswith\\nadvancedovariancancer(MITO-7):arandomised,\\nmulticentre,open-label,phase3trial. Lancet Oncol.\\n2014;15(4):396-405.doi:10.1016/S1470-2045(14)\\n70049-X\\n66. ChanJK,BradyMF,PensonRT,etal.Weeklyvs\\nevery-3-weekpaclitaxelandcarboplatinforovarian\\ncancer.N Engl J Med.2016;374(8):738-748.doi:\\n10.1056/NEJMoa1505067\\n67. ClampAR,JamesEC,McNeishIA,etal.Weekly\\ndose-densechemotherapyinfirst-lineepithelial\\novarian,fallopiantube,orprimaryperitoneal\\ncarcinomatreatment(ICON8):primaryprogression\\nfreesurvivalanalysisresultsfromaGCIGphase3\\nrandomisedcontrolledtrial. Lancet.2019;394\\n(10214):2084-2095.doi:10.1016/S0140-6736(19)\\n32259-7\\n68. ClampAR,JamesEC,McNeishIA,etal.Weekly\\ndose-densechemotherapyinfirst-lineepithelial\\novarian,fallopiantube,orprimaryperitonealcancer\\ntreatment(ICON8):overallsurvivalresultsfroman\\nopen-label,randomised,controlled,phase3trial.\\nLancet Oncol.2022;23(7):919-930.doi: 10.1016/\\nS1470-2045(22)00283-2\\n69. PerrenTJ,SwartAM,PfistererJ,etal;ICON7\\nInvestigators.Aphase3trialofbevacizumabin\\novariancancer. N Engl J Med.2011;365(26):2484-\\n2496.doi:10.1056/NEJMoa1103799\\n70. OzaAM,CookAD,PfistererJ,etal;ICON7Trial\\nInvestigators.Standardchemotherapywithor\\nwithoutbevacizumabforwomenwithnewly\\ndiagnosedovariancancer(ICON7):overallsurvival\\nresultsofaphase3randomisedtrial. Lancet Oncol.\\n2015;16(8):928-936.doi:10.1016/S1470-2045(15)\\n00086-8\\n71. BurgerRA,BradyMF,BookmanMA,etal;\\nGynecologicOncologyGroup.Incorporationof\\nbevacizumabintheprimarytreatmentofovarian\\ncancer.N Engl J Med.2011;365(26):2473-2483.\\ndoi:10.1056/NEJMoa1104390\\n72. TewariKS,BurgerRA,EnserroD,etal.Final\\noverallsurvivalofarandomizedtrialof\\nbevacizumabforprimarytreatmentofovarian\\ncancer.J Clin Oncol.2019;37(26):2317-2328.doi: 10.\\n1200/JCO.19.01009\\n73. MooreK,ColomboN,ScambiaG,etal.\\nMaintenanceolaparibinpatientswithnewly\\ndiagnosedadvancedovariancancer. N Engl J Med.\\n2018;379(26):2495-2505.doi:10.1056/\\nNEJMoa1810858\\n74. DiSilvestroP,BanerjeeS,ColomboN,etal;\\nSOLO1Investigators.Overallsurvivalwith\\nmaintenanceolaparibata7-yearfollow-upin\\npatientswithnewlydiagnosedadvancedovarian\\ncanceranda BRCA mutation:theSOLO1/GOG3004\\nOvarianCancer:AReview Review Clinical Review & Education\\njama.com (Reprinted) JAMA October14,2025 Volume334,Number14 1289\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 12, 'page_label': '13'}, page_content='trial.J Clin Oncol.2023;41(3):609-617.doi:10.1200/\\nJCO.22.01549\\n75. Ray-CoquardI,PautierP,PignataS,etal;\\nPAOLA-1Investigators.Olaparibplusbevacizumab\\nasfirst-linemaintenanceinovariancancer. N Engl J\\nMed.2019;381(25):2416-2428.doi: 10.1056/\\nNEJMoa1911361\\n76. Ray-CoquardI,LearyA,PignataS,etal;\\nPAOLA-1/ENGOT-OV25Investigators.Olaparibplus\\nbevacizumabfirst-linemaintenanceinovarian\\ncancer:finaloverallsurvivalresultsfromthe\\nPAOLA-1/ENGOT-OV25trial.Ann Oncol.2023;34\\n(8):681-692.doi:10.1016/j.annonc.2023.05.005\\n77. González-MartínA,PothuriB,VergoteI,etal;\\nPRIMA/ENGOT-OV26/GOG-3012Investigators.\\nNiraparibinpatientswithnewlydiagnosed\\nadvancedovariancancer. N Engl J Med.2019;381\\n(25):2391-2402.doi:10.1056/NEJMoa1910962\\n78. MonkBJ,Barretina-GinestaMP,PothuriB,etal.\\nNiraparibfirst-linemaintenancetherapyinpatients\\nwithnewlydiagnosedadvancedovariancancer:\\nfinaloverallsurvivalresultsfromthe\\nPRIMA/ENGOT-OV26/GOG-3012trial.Ann Oncol.\\n2024;35(11):981-992.doi:10.1016/j.annonc.2024.08.\\n2241\\n79. ColemanRL,SpirtosNM,EnserroD,etal.\\nSecondarysurgicalcytoreductionforrecurrent\\novarianCancer. N Engl J Med.2019;381(20):1929-\\n1939.doi:10.1056/NEJMoa1902626\\n80. HarterP,SehouliJ,VergoteI,etal;DESKTOPIII\\nInvestigators.Randomizedtrialofcytoreductive\\nsurgeryforrelapsedovariancancer. N Engl J Med.\\n2021;385(23):2123-2131.doi:10.1056/\\nNEJMoa2103294\\n81. ShiT,ZhuJ,FengY,etal.Secondary\\ncytoreductionfollowedbychemotherapyversus\\nchemotherapyaloneinplatinum-sensitiverelapsed\\novariancancer(SOC-1):amulticentre,open-label,\\nrandomised,phase3trial. Lancet Oncol.2021;22\\n(4):439-449.doi:10.1016/S1470-2045(21)00006-1\\n82. JiangR,FengY,ChenY,etal;SOC-1\\nInvestigators.Surgeryversusnosurgeryin\\nplatinum-sensitiverelapsedovariancancer:final\\noverallsurvivalanalysisoftheSOC-1randomized\\nphase3trial. Nat Med.2024;30(8):2181-2188.doi:\\n10.1038/s41591-024-02981-0\\n83. ParmarMKB,LedermannJA,ColomboN,etal;\\nICONandAGOCollaborators.Paclitaxelplus\\nplatinum-basedchemotherapyversusconventional\\nplatinum-basedchemotherapyinwomenwith\\nrelapsedovariancancer:theICON4/AGO-OVAR-2.2\\ntrial. Lancet. 2003;361(9375):2099-2106. doi:10.\\n1016/S0140-6736(03)13718-X\\n84. PfistererJ,PlanteM,VergoteI,etal.\\nGemcitabinepluscarboplatincomparedwith\\ncarboplatininpatientswithplatinum-sensitive\\nrecurrentovariancancer:anintergrouptrialofthe\\nAGO-OVAR,theNCICCTG,andtheEORTCGCG.\\nJ Clin Oncol.2006;24(29):4699-4707.doi:10.1200/\\nJCO.2006.06.0913\\n85. Pujade-LauraineE,WagnerU,Aavall-Lundqvist\\nE,etal.Pegylatedliposomaldoxorubicinand\\ncarboplatincomparedwithpaclitaxeland\\ncarboplatinforpatientswithplatinum-sensitive\\novariancancerinlaterelapse. J Clin Oncol.2010;28\\n(20):3323-3329.doi:10.1200/JCO.2009.25.7519\\n86. WagnerU,MarthC,LargillierR,etal.Final\\noverallsurvivalresultsofphaseIIIGCIGCALYPSO\\ntrialofpegylatedliposomaldoxorubicinand\\ncarboplatinvspaclitaxelandcarboplatinin\\nplatinum-sensitiveovariancancerpatients. Br J\\nCancer.2012;107(4):588-591.doi:10.1038/bjc.2012.\\n307\\n87. AghajanianC,BlankSV,GoffBA,etal.OCEANS:\\narandomized,double-blind,placebo-controlled\\nphaseIIItrialofchemotherapywithorwithout\\nbevacizumabinpatientswithplatinum-sensitive\\nrecurrentepithelialovarian,primaryperitoneal,or\\nfallopiantubecancer. J Clin Oncol.2012;30(17):\\n2039-2045.doi:10.1200/JCO.2012.42.0505\\n88. AghajanianC,GoffB,NycumLR,WangYV,\\nHusainA,BlankSV.Finaloverallsurvivalandsafety\\nanalysisofOCEANS,aphase3trialof\\nchemotherapywithorwithoutbevacizumabin\\npatientswithplatinum-sensitiverecurrentovarian\\ncancer. Gynecol Oncol. 2015;139(1):10-16. doi:10.\\n1016/j.ygyno.2015.08.004\\n89. ColemanRL,BradyMF,HerzogTJ,etal.\\nBevacizumabandpaclitaxel-carboplatin\\nchemotherapyandsecondarycytoreductionin\\nrecurrent,platinum-sensitiveovariancancer(NRG\\nOncology/GynecologicOncologyGroupstudy\\nGOG-0213):amulticentre,open-label,randomised,\\nphase3trial. Lancet Oncol.2017;18(6):779-791.\\ndoi:10.1016/S1470-2045(17)30279-6\\n90. PfistererJ,ShannonCM,BaumannK,etal;\\nAGO-OVAR2.21/ENGOT-OV18Investigators.\\nBevacizumabandplatinum-basedcombinationsfor\\nrecurrentovariancancer:arandomised,open-label,\\nphase3trial. Lancet Oncol.2020;21(5):699-709.\\ndoi:10.1016/S1470-2045(20)30142-X\\n91. Pujade-LauraineE,LedermannJA,SelleF,etal;\\nSOLO2/ENGOT-OV21Investigators.Olaparibtablets\\nasmaintenancetherapyinpatientswith\\nplatinum-sensitive,relapsedovariancanceranda\\nBRCA1/2 mutation(SOLO2/ENGOT-OV21):\\nadouble-blind,randomised,placebo-controlled,\\nphase3trial. Lancet Oncol.2017;18(9):1274-1284.\\ndoi:10.1016/S1470-2045(17)30469-2\\n92. PovedaA,FloquetA,LedermannJA,etal;\\nSOLO2/ENGOT-OV21Investigators.Olaparibtablets\\nasmaintenancetherapyinpatientswith\\nplatinum-sensitiverelapsedovariancanceranda\\nBRCA1/2 mutation(SOLO2/ENGOT-OV21):afinal\\nanalysisofadouble-blind,randomised,\\nplacebo-controlled,phase3trial. Lancet Oncol.\\n2021;22(5):620-631.doi:10.1016/S1470-2045(21)\\n00073-5\\n93. MirzaMR,MonkBJ,HerrstedtJ,etal;\\nENGOT-OV16/NOVAInvestigators.Niraparib\\nmaintenancetherapyinplatinum-sensitive,\\nrecurrentovariancancer. N Engl J Med.2016;375\\n(22):2154-2164.doi:10.1056/NEJMoa1611310\\n94. ColemanRL,OzaAM,LorussoD,etal;ARIEL3\\nInvestigators.Rucaparibmaintenancetreatmentfor\\nrecurrentovariancarcinomaafterresponseto\\nplatinumtherapy(ARIEL3):arandomised,\\ndouble-blind,placebo-controlled,phase3trial.\\nLancet.2017;390(10106):1949-1961.doi: 10.1016/\\nS0140-6736(17)32440-6\\n95. Pujade-LauraineE,HilpertF,WeberB,etal.\\nBevacizumabcombinedwithchemotherapyfor\\nplatinum-resistantrecurrentovariancancer:the\\nAURELIAopen-labelrandomizedphaseIIItrial.\\nJ Clin Oncol.2014;32(13):1302-1308.doi: 10.1200/\\nJCO.2013.51.4489\\n96. MooreKN,AngelerguesA,KonecnyGE,etal;\\nGynecologicOncologyGroupPartners;European\\nNetworkofGynaecologicalOncologicalTrial\\nGroups.Mirvetuximabsoravtansinein\\nFRα-positive,platinum-resistantovariancancer.\\nN Engl J Med.2023;389(23):2162-2174.doi: 10.\\n1056/NEJMoa2309169\\n97. LawrieTA,Winter-RoachBA,HeusP,\\nKitchenerHC.Adjuvant(post-surgery)\\nchemotherapyforearlystageepithelialovarian\\ncancer.Cochrane Database Syst Rev.2015;2015(12):\\nCD004706.doi:10.1002/14651858.CD004706.pub5\\n98. duBoisA,ReussA,Pujade-LauraineE,Harter\\nP,Ray-CoquardI,PfistererJ.Roleofsurgical\\noutcomeasprognosticfactorinadvancedepithelial\\novariancancer:acombinedexploratoryanalysisof\\n3prospectivelyrandomizedphase3multicenter\\ntrials:bytheArbeitsgemeinschaftGynaekologische\\nOnkologieStudiengruppeOvarialkarzinom\\n(AGO-OVAR)andtheGrouped’Investigateurs\\nNationauxPourlesEtudesdesCancersdel’Ovaire\\n(GINECO).Cancer.2009;115(6):1234-1244.doi:\\n10.1002/cncr.24149\\n99. ChiDS,EisenhauerEL,LangJ,etal.Whatisthe\\noptimalgoalofprimarycytoreductivesurgeryfor\\nbulkystageIIICepithelialovariancarcinoma(EOC)?\\nGynecol Oncol.2006;103(2):559-564.doi: 10.1016/j.\\nygyno.2006.03.051\\n100. BristowRE,TomacruzRS,ArmstrongDK,\\nTrimbleEL,MontzFJ.Survivaleffectofmaximal\\ncytoreductivesurgeryforadvancedovarian\\ncarcinomaduringtheplatinumera:ameta-analysis.\\nJ Clin Oncol. 2002;20(5):1248-1259. doi:10.1200/\\nJCO.2002.20.5.1248\\n101. HarterP,SehouliJ,LorussoD,etal.\\nArandomizedtrialoflymphadenectomyinpatients\\nwithadvancedovarianneoplasms. N Engl J Med.\\n2019;380(9):822-832.doi:10.1056/NEJMoa1808424\\n102. GaillardS,LacchettiC,ArmstrongDK,etal.\\nNeoadjuvantchemotherapyfornewlydiagnosed,\\nadvancedovariancancer:ASCOguidelineupdate.\\nJ Clin Oncol.2025;43(7):868-891.doi:10.1200/JCO-\\n24-02589\\n103. ColeridgeSL,BryantA,KehoeS,MorrisonJ.\\nNeoadjuvantchemotherapybeforesurgeryversus\\nsurgeryfollowedbychemotherapyforinitial\\ntreatmentinadvancedovarianepithelialcancer.\\nCochrane Database Syst Rev.2021;7(7):CD005343.\\n104. vanDrielWJ,KooleSN,SonkeGS.\\nHyperthermicintraperitonealchemotherapyin\\novariancancer. N Engl J Med.2018;378(14):1363-\\n1364.\\n105. AronsonSL,Lopez-YurdaM,KooleSN,etal.\\nCytoreductivesurgerywithorwithout\\nhyperthermicintraperitonealchemotherapyin\\npatientswithadvancedovariancancer\\n(OVHIPEC-1):finalsurvivalanalysisofarandomised,\\ncontrolled,phase3trial. Lancet Oncol.2023;24\\n(10):1109-1118.doi:10.1016/S1470-2045(23)00396-\\n0\\n106. LimMC,ChangSJ,ParkB,etal;HIPECfor\\nOvarianCancerCollaborators.Survivalafter\\nhyperthermicintraperitonealchemotherapyand\\nprimaryorintervalcytoreductivesurgeryinovarian\\ncancer:arandomizedclinicaltrial. JAMA Surg.\\n2022;157(5):374-383.doi:10.1001/jamasurg.2022.\\n0143\\n107. AntonioCC,AlidaGG,ElenaGG,etal.\\nCytoreductivesurgerywithorwithoutHIPECafter\\nneoadjuvantchemotherapyinovariancancer:\\naphase3clinicaltrial. Ann Surg Oncol.2022;29(4):\\n2617-2625.doi:10.1245/s10434-021-11087-7\\nClinical Review & EducationReview OvarianCancer:AReview\\n1290 JAMA October14,2025 Volume334,Number14 (Reprinted) jama.com\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025'),\n",
       " Document(metadata={'producer': '; modified using iTextSharp.LGPLv2.Core 3.7.4.0', 'creator': 'XPP', 'creationdate': \"D:20251002093746Z00'00'\", 'subject': '', 'keywords': 'The JAMA Network', 'author': 'American Medical Association', 'moddate': '2025-12-26T10:58:40+00:00', 'title': '', 'source': 'data/jama_cancer_review.pdf', 'total_pages': 14, 'page': 13, 'page_label': '14'}, page_content='108. McGuireWP,HoskinsWJ,BradyMF,etal.\\nCyclophosphamideandcisplatincomparedwith\\npaclitaxelandcisplatininpatientswithstageIIIand\\nstageIVovariancancer. N Engl J Med.1996;334(1):1-6.\\ndoi:10.1056/NEJM199601043340101\\n109. PiccartMJ,BertelsenK,JamesK,etal.\\nRandomizedintergrouptrialofcisplatin-paclitaxel\\nversuscisplatin-cyclophosphamideinwomenwith\\nadvancedepithelialovariancancer:three-year\\nresults.J Natl Cancer Inst.2000;92(9):699-708.\\ndoi:10.1093/jnci/92.9.699\\n110. LiuS,KashermanL,FazelzadR,etal.Theuse\\nofbevacizumabinthemoderneraoftargeted\\ntherapyforovariancancer:asystematicreviewand\\nmeta-analysis.Gynecol Oncol.2021;161(2):601-612.\\ndoi:10.1016/j.ygyno.2021.01.028\\n111. LiN,ZhuJ,YinR,etal.Treatmentwith\\nniraparibmaintenancetherapyinpatientswith\\nnewlydiagnosedadvancedovariancancer:aphase\\n3randomizedclinicaltrial. JAMA Oncol.2023;9(9):\\n1230-1237.doi:10.1001/jamaoncol.2023.2283\\n112. MonkBJ,ParkinsonC,LimMC,etal.\\nArandomized,phaseIIItrialtoevaluaterucaparib\\nmonotherapyasmaintenancetreatmentinpatients\\nwithnewlydiagnosedovariancancer\\n(ATHENA-MONO/GOG-3020/ENGOT-OV45).J Clin\\nOncol.2022;40(34):3952-3964.doi: 10.1200/JCO.\\n22.01003\\n113. ColemanRL,FlemingGF,BradyMF,etal.\\nVeliparibwithfirst-linechemotherapyandas\\nmaintenancetherapyinovariancancer. N Engl J Med.\\n2019;381(25):2403-2415.doi:10.1056/\\nNEJMoa1909707\\n114. TorreLA,TrabertB,DeSantisCE,etal.Ovarian\\ncancer statistics, 2018.CA Cancer J Clin. 2018;68\\n(4):284-296.doi:10.3322/caac.21456\\n115. BaertT,FerreroA,SehouliJ,etal.Thesystemic\\ntreatmentofrecurrentovariancancerrevisited.\\nAnn Oncol.2021;32(6):710-725.doi: 10.1016/j.\\nannonc.2021.02.015\\n116. WuXH,ZhuJQ,YinRT,etal.Niraparib\\nmaintenancetherapyinpatientswith\\nplatinum-sensitiverecurrentovariancancerusing\\nanindividualizedstartingdose(NORA):\\narandomized,double-blind,placebo-controlled\\nphaseIIItrial. Ann Oncol.2021;32(4):512-521.\\ndoi:10.1016/j.annonc.2020.12.018\\n117. WuX,ZhuJ,YinR,etal.Niraparib\\nmaintenancetherapyusinganindividualised\\nstartingdoseinpatientswithplatinum-sensitive\\nrecurrentovariancancer(NORA):finaloverall\\nsurvivalanalysisofaphase3randomised,\\nplacebo-controlledtrial. EClinicalMedicine.2024;72:\\n102629.doi:10.1016/j.eclinm.2024.102629\\n118. PignataS,LorussoD,JolyF,etal;\\nMITO16b/MANGO-OV2/ENGOT-OV17Investigators.\\nCarboplatin-baseddoubletplusbevacizumab\\nbeyondprogressionversuscarboplatin-based\\ndoubletaloneinpatientswithplatinum-sensitive\\novariancancer:arandomised,phase3trial. Lancet\\nOncol.2021;22(2):267-276.doi: 10.1016/S1470-\\n2045(20)30637-9\\n119. MooreKN,OzaAM,ColomboN,etal.PhaseIII,\\nrandomizedtrialofmirvetuximabsoravtansine\\nversuschemotherapyinpatientswith\\nplatinum-resistantovariancancer:primaryanalysis\\nofFORWARDI. Ann Oncol.2021;32(6):757-765.\\ndoi:10.1016/j.annonc.2021.02.017\\n120. Meric-BernstamF,MakkerV,OakninA,etal.\\nEfficacyandsafetyoftrastuzumabderuxtecanin\\npatientswith HER2-expressingsolidtumors:\\nprimaryresultsfromtheDESTINY-PanTumor02\\nphaseIItrial. J Clin Oncol.2024;42(1):47-58.doi: 10.\\n1200/JCO.23.02005\\n121. NationalComprehensiveCancerNetwork.\\nPalliativecare(version2.2025).AccessedApril5,\\n2025.https://www.nccn.org/professionals/\\nphysician_gls/pdf/palliative.pdf\\n122. BakitasMA,TostesonTD,LiZ,etal.Early\\nversusdelayedinitiationofconcurrentpalliative\\noncologycare:patientoutcomesintheENABLEIII\\nrandomized controlled trial.J Clin Oncol. 2015;33\\n(13):1438-1445.doi:10.1200/JCO.2014.58.6362\\n123. HoergerM,WayserGR,SchwingG,SuzukiA,\\nPerryLM.Impactofinterdisciplinaryoutpatient\\nspecialtypalliativecareonsurvivalandqualityof\\nlifeinadultswithadvancedcancer:ameta-analysis\\nofrandomizedcontrolledtrials. Ann Behav Med.\\n2019;53(7):674-685.doi:10.1093/abm/kay077\\nOvarianCancer:AReview Review Clinical Review & Education\\njama.com (Reprinted) JAMA October14,2025 Volume334,Number14 1291\\n© 2025 American Medical Association. All rights reserved, including those for text and data mining, AI training, and similar technologies.\\nDownloaded from jamanetwork.com by Griffith University user on 12/26/2025')]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "extracted_data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "bd8a8478",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "118"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(extracted_data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "8585ea15",
   "metadata": {},
   "outputs": [],
   "source": [
    "from typing import List\n",
    "# from langchain.schema import Document\n",
    "from langchain_core.documents import Document\n",
    "\n",
    "def filter_to_minimal_docs(docs: List[Document]) -> List[Document]:\n",
    "    \"\"\"\n",
    "    Given a list of Document objects, return a new list of Document objects\n",
    "    containing only 'source' in metadata and the original page_content.\n",
    "    \"\"\"\n",
    "    minimal_docs: List[Document] = []\n",
    "    for doc in docs:\n",
    "        src = doc.metadata.get(\"source\")\n",
    "        minimal_docs.append(\n",
    "            Document(\n",
    "                page_content=doc.page_content,\n",
    "                metadata={\"source\": src}\n",
    "            )\n",
    "        )\n",
    "    return minimal_docs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "82a049f9",
   "metadata": {},
   "outputs": [],
   "source": [
    "minimal_docs = filter_to_minimal_docs(extracted_data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "5a59bf5e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content=\"Funding information\\nMacao Science and Technology\\nDevelopment Fund,\\nGrant/Award Numbers: 0009/2022/AKP,\\n0073/2021/A2; Hunan Youth Science and\\nTechnology Talent Project,\\nGrant/Award Number: NO.2023RC3074;\\nthe Science and Technology Program of\\nGuangzhou, Grant/Award Number:\\n2023A04J1257; the Guangdong Basic and\\nApplied Basic Research Foundation,\\nGrant/Award Number:\\n2022A1515111212; the National Natural\\nScience Foundation of China,\\nGrant/Award Numbers: 82172750,\\n82172811, 82260546, 82373129; the\\nNatural Science Foundation of\\nGuangdong Province,\\nGrant/Award Number:\\n2021A1515012593; Multiyear research\\ngrant (MYRG), Grant/Award Number:\\nMYRG‐GRG2023‐00150‐FHS‐UMDF to\\nK. M.\\nmicrobiota transplantation, and engineered microbes offer new avenues for\\nadjunctive cancer therapy. This review provides a roadmap for future investiga-\\ntion and underscores the transformative promise of microbiome modulation in\\ncancer prevention and treatment.\\nKEYWORDS\\ncancer, microbiome, precision oncology, treatment, tumor microenvironment\\nHighlights\\n• Microbiome dysbiosis drives carcinogenesis via multiple mechanisms.\\n• Microbiome‐targeted strategies offer novel therapeutic avenues.\\n• Organ‐specific microbiome signatures serve as non‐invasive biomarkers.\\n• Intratumoral microbiota modulate therapy efficacy and resistance.\\n• Microbial metabolites reprogram the tumor microenvironment.\\nINTRODUCTION\\nMicroorganisms comprise diverse microscopic entities,\\nincluding bacteria, fungi, viruses, archaea, and protists.\\nThese microorganisms, including 1013–1014 bacteria col-\\nonizing human skin, intestines, and respiratory tract,\\nestablish complex symbiotic relationships with the host\\nand are collectively referred to as the body's “second\\ngenome” [1–4]. Symbiotic microbes maintain host health\\nthrough complex metabolic reciprocity [5]. In contrast,\\npathogenic microorganisms pose a threat to the host's\\nhealth. Notably, microorganisms can have dual\\n(beneficial and pathogenic) roles. When host immune\\nfunction is compromised, commensal microbes may\\nconvert to conditionally opportunistic pathogens, poten-\\ntially disrupting host health [6]. Clinical studies have\\nconfirmed that the composition of the gut microbiome\\ncan be modulated through fecal microbiota transplanta-\\ntion (FMT) and the administration of probiotic prepara-\\ntions, influencing immune cell infiltration, immune\\nfactor release, and inflammatory responses in the intes-\\ntine and surrounding tumor tissues, thereby improving\\nthe immune status of the tumor microenvironment\\n(TME) [7–9]. Consequently, the microbiome serves not\\nonly as a biomarker for disease diagnosis but also as a\\npromising target for therapeutic intervention [10–12].\\nWhile research on gut microbes dates back to the\\n18th century, comprehensive investigations into the gut\\nmicrobes expanded exponentially in the 21st century. A\\nstudy in 1972 established that gut microbes play a critical\\nrole in drug transformation [13]. This seminal work laid\\nthe groundwork for understanding the microbiome as a\\nkey modulator of pharmacology, a concept fundamental\\nto modern microbiome–drug interaction studies. Since\\n2006, several pivotal studies have begun to emphasize the\\nsignificant impact of diet on gut microbial composition\\nand host metabolism [ 14–19]. In 2007, the National\\nInstitutes of Health (NIH) launched the Human Micro-\\nbiome Project (HMP), the first large‐scale collaborative\\nprogram to study the human gut microbiome [20]. The\\nMetagenomics of the Human Intestinal Tract (MetaHIT)\\nproject, initiated by the European Union in 2008, repre-\\nsents another landmark effort in the comprehensive and\\nsystematic study of gut microbiota. This initiative sys-\\ntematically characterized the genomic architecture of\\nintestinal microbial communities and established foun-\\ndational frameworks for investigating microbiome –\\ndisease associations, significantly advancing the\\ndevelopment of precision medicine approaches [ 21].\\nSubsequently, Yatsunenko et al. conducted a large‐scale\\ncomparative analysis demonstrating that human\\npopulations across distinct geographic regions have sig-\\nnificantly different compositions and functional char-\\nacteristics of gut microbes [ 22]. The American Gut\\nProject (AGP), launched in 2012, was further expanded\\ninto a global microbiome research program to explore the\\ncomposition and diversity of microbes in geographically\\ndiverse populations [ 23]. Notably, the study of gut\\nmicrobes–host interactions has gradually extended to the\\nfield of disease treatment. A groundbreaking study in\\n2018 demonstrated that gut microbial signatures can\\npredict immunotherapeutic response in patients with\\n2 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content=\"melanoma, non‐small cell lung cancer, and renal cell\\ncarcinoma [24]. This landmark discovery provided the\\nfirst robust clinical evidence establishing a causal rela-\\ntionship between the gut microbiome composition and\\nthe efficacy of cutting ‐edge cancer immunotherapies,\\nthereby creating a paradigm‐shifting theoretical frame-\\nwork for understanding the microbiome's pivotal role in\\ndeveloping therapeutic intervention. These findings es-\\ntablish a crucial theoretical framework for understanding\\nthe pivotal role of tumor‐associated microbes in cancer\\ninitiation, progression, and therapeutic intervention.\\nInvestigations into intratumoral microbes began in\\nthe 19th century, but significant advances remained\\nlimited until recent decades [25]. In 2020, through a\\nlarge‐scale multi‐omic analysis of diverse primary human\\nmalignancies, researchers established the presence of\\nhighly specific microbiome profiles and functional char-\\nacteristics across different tumor types [26]. This study\\nprovides comprehensive evidence for the existence of\\ndistinct tumor‐type‐specific intratumoral microbiomes.\\nIn recent years, the prevalence of microbes within\\ntumors has been further confirmed by increasing\\namounts of reliable evidence obtained through various\\nadvanced technological approaches [27, 28]. Studies have\\ndemonstrated that the species composition and abun-\\ndance distribution of intratumoral microbes in different\\ncancer types show marked heterogeneity [ 29–31]. As\\ninvestigations into tumor‐associated microbiota intensify,\\nthe precise origins and molecular mechanisms governing\\nmicrobial colonization remain fundamental unresolved\\nquestions in the field. Recent investigations have char-\\nacterized three potential routes of microbial access to\\ntumor tissues: disruption of epithelial barrier integrity,\\nmigration from adjacent microenvironmental niches, and\\nsystemic dissemination [ 29, 32]. Substantial evidence\\nindicates that intratumoral microbes are involved in the\\nregulation of tumorigenesis, progression, and therapeutic\\nresponse through multiple molecular mechanisms,\\nincluding DNA damage induction, oncogenic signaling\\npathway activation, and alteration of epigenetic\\nregulation [ 33]. Additionally, intratumoral microbes\\nsignificantly affect the clinical efficacy of immuno-\\ntherapy and chemotherapy by remodeling the tumor\\nimmune microenvironment (TIME). As intratumoral\\nmicrobiome research advances, microbiome ‐targeted\\ntumor precision diagnosis and individualized treatment\\nstrategies have emerged as an international research\\nfrontier. This recognition underscores the immense\\ntranslational potential of the intratumoral microbiome\\nand its associated metabolites, positioning them as criti-\\ncal targets for the next generation of diagnostics and\\npersonalized therapeutic interventions, particularly\\nwithin the TME and immunomodulation approaches.\\nHowever, methodological challenges still exist, including\\nsample acquisition, microbial detection, and functional\\nvalidation [34–37]. The study of intratumoral microbiota\\nholds profound clinical translational potential and is\\npositioned to constitute a significant breakthrough and\\npromising research direction in anti‐tumor therapy.\\nThe microbiome plays a critical role in multiple\\naspects of tumor growth, proliferation, diagnosis, treat-\\nment, and prognosis. Microorganisms and malignant\\ncells engage in complex bidirectional interactions [38].\\nMicroorganisms can promote tumorigenesis and devel-\\nopment through direct genotoxic mechanisms or indirect\\nimmunomodulatory effects [39, 40]. Certain beneficial\\nbacteria are also associated with anti ‐cancer effects.\\nProbiotics maintain intestinal barrier homeostasis, regu-\\nlate anti‐\\ninflammatory cytokine levels with anti‐cancer\\nproperties, and enhance immune surveillance through\\nactivation of phagocytic cells. Germ‐free murine models\\ndemonstrate compromised immunosurveillance and\\nincreased susceptibility to chemical carcinogenesis [41].\\nLactobacillus and Bifidobacterium have also demon-\\nstrated potential to bind and degrade carcinogenic com-\\npounds [ 42]. Importantly, the microbial community\\nserves as a diagnostic and prognostic biomarker [43–45].\\nFurthermore, the microbiome influences therapeutic ef-\\nfects by metabolizing drugs and dynamically modulating\\nthe immune microenvironment [46]. In immunotherapy,\\ndysbiosis of the intestinal microbes (such as reduction of\\nBifidobacteria) exacerbates anti‐CTLA‐4 therapy‐induced\\ncolitis, whereas probiotic supplementation both mitigates\\ntoxicity and enhances therapeutic efficacy [ 47]. As\\nresearch exploring microbiome–tumor interactions in-\\ntensifies, microbe‐targeted therapies are gradually being\\nrecognized for their anti‐tumor effects. For example, pro-\\nbiotics help maintain intestinal homeostasis and inhibit\\nproliferation of malignant cells [48]. FMT therapy con-\\ntributes to immunomodulation of the TME, resulting in a\\ntumor‐preventive effect at an early stage [8, 49]. The\\nmicrobiome, as an emerging biomarker and potential\\ntherapeutic target, has broad applications in tumor diag-\\nnosis and treatment [ 50–53]. Consequently, microbial\\ndysbiosis and intervention emerge as critical factors\\nthroughout carcinogenesis and disease progression. Tar-\\ngeted microbial therapy is emerging as a promising\\napproach for integrative oncological strategies [54–59].\\nThis review comprehensively synthesizes recent ad-\\nvances in understanding the complex interactions\\nbetween the microbiome and cancer biology. Initially, we\\nexamine the intrinsic relationships between the intestinal\\nmicrobiota (encompassing bacteria, viruses, fungi, and\\ntheir metabolites) and the intestinal barrier in carcino-\\ngenesis, characterize microbial signatures within tumor\\ntissues, and assess their organ ‐specific manifestations\\nMICROBIOME IN CANCER | 3 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content='and diagnostic implications across diverse anatomical\\nsites including the oral cavity, integumentary system,\\nurogenital tract, and respiratory tract. We subsequently\\nfocus on the diagnostic and prognostic implications of\\nmicrobiome profiles, emphasizing the potential applica-\\ntions of site‐specific and kingdom‐specific microbial sig-\\nnatures as clinically actionable biomarkers for early\\ncancer detection, therapeutic response prediction, and\\noutcome assessment. Building upon the above studies,\\nwe further elucidate the synergistic effects of microbiome\\nregulation in chemotherapy, radiotherapy, immuno-\\ntherapy, targeted therapy, and surgery. Additionally,\\nwe comprehensively summarize various innovative\\nmicrobiome‐based therapeutic strategies, including pro-\\nbiotic/prebiotic interventions, FMT, engineered bacteria\\ntherapy, and targeted antimicrobial approaches. Studies\\nhave shown that the microbiome not only influences\\ntumor initiation and progression but also offers promis-\\ning new targets for tumor prevention and treatment.\\nLooking forward, continued technological innovation in\\nmicrobiome research methodologies, elucidation of\\nmolecular mechanisms underlying microbiome –host\\ninteractions, development of precision microbiome ‐\\ntargeted therapeutic strategies, and advancement of\\nrobust clinical translational studies will collectively\\ndeliver more personalized, effective, and safer therapeu-\\ntic interventions for cancer patients.\\nGUT MICROBIOME AND TUMOR\\nDEVELOPMENT\\nAssociation of bacteria with tumorigenesis\\nGut microbes contribute to tumor development through\\na variety of complex mechanisms, predominantly pro-\\nmoting neoplastic progression via induction of chronic\\ninflammatory responses, disruption of epithelial barrier\\nintegrity, and comprehensive remodeling of the TIME\\n[60–63]. The relationship between microbiota composi-\\ntion and carcinogenesis demonstrates pronounced\\nspecies‐specificity, with pro‐tumorigenic effects predom-\\ninantly characterized by depletion of beneficial com-\\nmensal taxa, enrichment of pathogenic bacteria, and\\ndysregulated expansion of opportunistic bacteria [64–66].\\nHowever, some investigations have reported heteroge-\\nneous findings [67, 68]. Table 1 summarizes the asso-\\nciations between different types of bacteria (probiotics\\nand pathogenic bacteria) and various tumors, delineating\\ntheir proposed mechanistic contributions to carcinogen-\\nesis or tumor suppression, thereby providing a compre-\\nhensive framework for understanding the diverse roles of\\nthe microbiome in cancer pathogenesis.\\nProbiotics and tumorigenesis\\nIntestinal probiotics, beneficial bacteria colonizing the\\nhuman gastrointestinal tract, play a crucial role in pro-\\nmoting nutrient absorption and maintaining gastro-\\nintestinal health by orchestrating gut microbial balance\\nand calibrating intestinal mucosa and systemic immune\\nfunction through multiple mechanisms [69–71]. Com-\\nmon intestinal probiotics primarily includeBifidobacter-\\nium, Lactobacillus, and Bacillus [72, 73, 83, 84 ].\\nLactobacillus and Bifidobacterium demonstrate signifi-\\ncant anti‐tumor activity, inhibiting not only digestive\\nmalignancies, such as esophageal, gastric, colon, and li-\\nver cancer, but also bladder cancer (BCa), through\\nmechanisms including regulation of the pH of the TME,\\nmodulation of bile acid metabolism, and degradation of\\npotential carcinogens and their metabolites [66, 85–88].\\nAdditionally, tumor metastasis in patients with gastric\\ncardia adenocarcinoma considerably correlates with\\ndecreased Lactobacillus counts in the stomach [89]. Pa-\\ntients with hepatocellular carcinoma (HCC) exhibit sig-\\nnificantly altered gut microbial composition, with\\nhepatitis B virus (HBV)‐associated HCC displaying sig-\\nnificant enrichment of Bacteroides and Lachnospiraceae\\nincertae sedis, while non ‐HBV‐associated HCC shows\\ndecreased abundance ofFaecalibacterium, Ruminococcus,\\nand Ruminoclostridium [90, 91]. Notably, patients with\\nintrahepatic cholangiocarcinoma had a significantly\\nhigher abundance of Lactobacillus in fecal samples\\ncompared to healthy controls [ 67]. Comprehensive\\nanalyses have established significantly reduced duodenal\\nmicrobial diversity in patients with pancreatic cancer\\n(PC), with enrichment ofBifidobacterium spp. andRothia\\nbacteria [67]. Furthermore, multiple microorganisms,\\nincluding seven Bacteroidales species, significantly en-\\nhanced anti‐tumor immune responses mediated by CD8\\n+\\nT cells in mice through synergistic effects [92]. Although\\nBifidobacterium and Lactobacillus are widely recognized\\nas probiotics within the intestinal ecosystems, their\\nfunctions in a specific TME may diverge substantially\\nfrom their canonical probiotic activities. This effect is\\ninfluenced by multiple factors, including tumor histol-\\nogy, metabolic state, and the dominant bacterial strain.\\nTherefore, “probiotics” cannot be simplistically equated\\nwith “anti‐tumor,” necessitating that future microbiome‐\\ntargeted therapies carefully consider strain ‐specific\\nfunctions within particular TME.\\nPathogenic bacteria and tumorigenesis\\nPathogenic bacteria are disease‐causing microorganisms.\\nThese bacteria can damage host cells either directly or\\n4 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content=\"TABLE 1 Associations and potential mechanisms between different bacterial types and various tumors.\\nBacterial type Representative strains\\nAssociated\\ncancer types Mechanism Specificity References\\nProbiotics Lactobacillus GC\\nCRC\\nLiver cancer\\nBCa\\n• Regulate the TME pH and bile acid\\nmetabolism;\\n• Degrade carcinogens.\\nParticular case: elevated\\nLactobacillus abundance in\\nintrahepatic\\ncholangiocarcinoma\\n[58, 67, 69–73]\\nBifidobacterium GC\\nCRC\\nLiver cancer\\n• Competitively inhibit pathogen colonization;\\n• Regulate the TME pH and bile acid\\nmetabolism;\\n• Maintain intestinal barrier integrity.\\nEnriched in PC [ 58, 67, 73]\\nPathogens Helicobacter pylori GC\\nCRC\\nPC\\nHCC\\n• Induce DNA damage;\\n• Disrupt DNA repair;\\n• Activate STAT3 signaling;\\n• Decrease Treg cells;\\n• Increase CD3 cells.\\nReduces Barrett's esophagus,\\nEAC and IBD risk\\n[64, 68, 74–78]\\nFusobacterium nucleatum CRC\\nPC\\nOral cancer\\nGliomas\\n• Activate pro‐cancer pathways;\\n• Induce DNA methylation;\\n• Suppress immune cell anti‐tumor function.\\nPromotes tumorigenesis in PC [ 40, 79–81]\\nPseudomonas PC • Pseudoxanthomonas: recruits CD8\\n+ T cells\\n(anti‐tumor).\\nImprove the OS of PC [ 81]\\nEnterotoxigenic Bacteroides fragilis CRC • Highly inducible to inflammatory stimuli. [ 82]\\nMicrobial\\nsynergy\\nHelicobacter pylori + phage CRC • Altering the enterovirus composition. Synergism promotes CRC\\nprogression\\n[75]\\nRuminococcaceae + Bacteroides HCC • Induce hepatic steatosis;\\n• Accumulate toxic metabolites (such as\\nammonia).\\nAlters liver metabolic\\nmicroenvironment\\n[66]\\nAbbreviations: BCa, bladder cancer; CRC, colorectal cancer; EAC, esophageal adenocarcinoma; GC, gastric cancer; HCC, hepatocellular carcinoma;IBD, inflammatory bowel disease; OS, overall survival; PC,\\npancreatic cancer; pH, potential of hydrogen; STAT3, signal transducer and activator of transcription 3; TME, tumor microenvironment.\\nMICROBIOME IN CANCER | 5 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content=\"indirectly by releasing toxins or inducing immune\\nresponses. Intestinal pathogenic bacteria can induce\\nacute or chronic inflammatory responses in the host,\\nseveral of which directly contribute to tumor develop-\\nment and progression [ 93–98]. Common intestinal\\npathogenic bacteria include Salmonella, Shigella, en-\\nterohemorrhagic E.coli, Staphylococcus aureus(S. aureus),\\nand Clostridium difficile[99–104]. Gastrointestinal tumor\\ndevelopment closely relates to the dynamics of various\\nintestinal microbiota, including enterotoxigenic Bacter-\\noides fragilis, Porphyromonas, Flavonifractor plauti[105],\\nCampylobacter spp.[ 106, 107], and Fusobacterium nu-\\ncleatum (F. nucleatum)[ 66]. Bacteroides and Rumino-\\ncoccaceae can contribute to HCC development through\\npromoting inflammatory responses, facilitating toxic\\nsubstances, and inducing hepatic steatosis [66]. Signifi-\\ncant enrichment ofActinomyces spp. has been identified\\nin extrahepatic cholangiocarcinoma tissues [ 108],\\nwhile chronic Salmonella typhi infection significantly\\ncorrelates with gallbladder cancer development [109]. PC\\ntissues harbor significant Elizabethkingia enrichment\\n[110], and Hungatella hathewayi can promote cancer by\\nreprogramming host DNA methylation patterns [ 79].\\nBeyond digestive system tumors, gut microbiome dysre-\\ngulation exhibits associations with malignancies in\\nother organ systems. Breast cancer (BC) patients' fecal\\nsamples show significantly lower relative abundance\\nof Bacteroidetes, Firmicutes, and Faecalibacterium\\nprausnitzii, with higher levels ofProteobacteria, Actino-\\nbacteria, Verrucomicrobia, and Firmicutes/Bacteroidetes\\nratio [111, 112]. Specific\\nClostridium species and Rumi-\\nnococcaceae family members increase BC risk by altering\\nestrogen metabolic pathways [ 113, 114]. Lung cancer\\npatients show significantly upregulated Ruminococci\\n(R. gnavus ) abundance in fecal microbiome, with\\nsquamous cell carcinoma (SCC) patients having higher\\nProteobacteria, Gammaproteobacteria, Bacteroides,a n d\\nEnterobacteriaceae abundance, while adenocarcinoma\\npatients' feces contain higherFusicatenibacter and Rose-\\nburia abundance [115]. Prostate cancer (PCa) studies\\ndemonstrate a significant correlation between cancer\\ndevelopment and relative abundance of Alphaproteo-\\nbacteria and Bacteroides massiliensis[116].\\nAmong microorganisms closely associated with gas-\\ntric tumors, Helicobacter pylori (H. pylori) is the most\\nextensively documented. H. pylori is the most common\\npathogenic bacterium in gastric tissues, promoting\\ntumorigenesis and progression by inducing DNA dam-\\nage, interfering with DNA repair mechanisms, and acti-\\nvating oncogenic pathways [ 64]. Numerous studies\\nconfirm H. pylori's significant positive association with\\ngastric, colorectal, and PC risks, leading to its classifica-\\ntion as a class I carcinogen by the World Health\\nOrganization [117–119]. Cross‐sectional studies demon-\\nstrate thatH. pyloriinfection significantly correlates with\\ncolorectal cancer (CRC) risk (OR = 1.9), surpassing the\\nassociations observed with established risk factors\\nincluding body mass index (BMI), smoking, and alcohol\\nconsumption [ 120]. Research indicates that both H.\\npylori‐infected CRC patients and mouse models exhibit\\nactivation of pro ‐carcinogenic signal transducer and\\nactivator of transcription 3 (STAT3) signaling, loss of\\nintestinal epithelial goblet cells, reduction in Treg cells,\\nand significant increases in CD3 cells (pro‐inflammatory\\nT cells) [74]. These findings suggest thatH. pylori func-\\ntions not merely as a risk indicator for CRC but also\\nestablishes a pro‐carcinogenic microenvironment within\\nthe colon. Additionally, animal experiments demonstrate\\nthat H. pylori can increase the abundance of CRC ‐\\nassociated intestinal bacteriophages, thereby altering the\\nintestinal virome composition, which is associated with\\nCRC development [75]. Cohort studies reveal that sero-\\nlogical responses to the\\nH. pylori virulence factor VacA\\nexhibit positive correlations with CRC risk [76]. A recent\\ncohort study demonstrated thatH. pylori has significant\\npositive associations with both CRC incidence and\\nmortality. Individuals with untreatedH. pylori infection\\nexhibited significantly higher incidence and mortality\\nrates compared to those who received treatment [121].\\nHelicobacter hepaticus has been identified in hepatitis C\\nvirus (HCV)‐associated HCC tissue specimens [122, 123],\\nalthough subsequent research challenged its pathogenic\\nrole in HBV‐associated HCC [122]. Mouse model studies\\nhave shown that Helicobacter hepaticus can inhibit\\nHCC progression by suppressing intrinsic immunity's\\nrecognition and clearance functions of tumor cells [124].\\nThe detection ofHelicobacter bilis, Helicobacter hepaticus,\\nor H. pylori in biliary tract cancer tissues demonstrates\\nsignificant correlations with increased cancer risk [109,\\n125–127]. Various intestinal microorganisms, including\\nBacteroidetes, Proteobacteria,a n dStreptococcaceae,s h o w\\nsynergistic effects with H. pylori in promoting gastro-\\nintestinal tumorigenesis [128]. Interestingly,H. pylori ex-\\nhibits protective effects against Barrett's esophagus,\\nesophageal adenocarcinoma (EAC), and inflammatory\\nbowel disease (IBD) [69, 77, 78]. Multiple studies have\\ndemonstrated a negative correlation between H. pylori\\ninfection and EAC and Barrett's esophagus [129]. A case‐\\ncontrol study established thatH. pylori infection exhibits\\nstrong negative associations with both erosive esophagitis\\nand Barrett's esophagus [68]. A previous community‐\\nbased study reached the same conclusion, showing thatH.\\npylori infection and CagA\\n+ status are negatively associated\\nwith newly diagnosed Barrett's esophagus [130]. A sub-\\nsequent meta‐analysis including 10 studies revealed that\\namong Asian populations, IBD patients had significantly\\n6 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content=\"lower H. pylori infection rates compared to non‐IBD pa-\\ntients [78]. The negative association between H. pylori\\ninfection and IBD is independent of ethnicity, age, H.\\npylori detection methods, and previous medication use,\\nwhile antibiotic use influenced this association [131, 132].\\nResearch suggests that the H. pylori genome contains\\nimmunomodulatory elements that may explain its pro-\\ntective effect against IBD through suppression of inflam-\\nmatory responses via downregulation of dendritic\\ncell (DC)‐produced IL‐12 and type I interferon levels\\n[133, 134]. Case‐control studies show that both gastric\\natrophy and H. pylori seropositivity are associated with\\nreduced risk of EAC, Barrett's esophagus, and reflux eso-\\nphagitis, with this negative correlation persisting in pa-\\ntients without gastric atrophy [135, 136]. A recent study\\npostulates that H. pylori eradication may disrupt the\\ninflammatory microenvironment, thereby inhibiting the\\nproliferation of other bacteria and reducing EAC risk\\n[137]. However, some cohort studies have not observed\\nincreased EAC risk followingH. pylorieradication [138].\\nConditional pathogens are bacteria that may cause\\ndisease when there is an imbalance in the host micro-\\nbiota homeostasis [139, 140], and they are associated\\nwith impaired immune systems. These bacteria can ele-\\nvate cancer risk and facilitate tumorigenesis and pro-\\ngression through direct DNA damage or by inducing\\nmicrobiota dysbiosis [6, 141, 142]. Veillonella and Pre-\\nvotella are commensal in the gut most of the time, but\\ncan also cause disease in the context of the dysregulated\\nimmune microenvironment.Veillonella spp. and specific\\nspecies of Prevotella are associated with gastric cancer\\n(GC) risk [143]. Elevated levels of Bacteroides fragilis\\n[144–146] and Streptococcus gallolyticus (S. gallolyticus)\\n[147, 148] were detected in tumor tissues of patients with\\nCRC, and fecal specimens from these patients demon-\\nstrated significantly higher prevalence of Enterococcus\\nfaecalis [149]. Additionally, enterotoxigenic Bacteroides\\nfragili (ETBF) is highly responsive to inflammatory\\nstimuli and considered a risk factor for CRC [82]. En-\\nterococcus faecalis potentially promotes chromosomal\\ninstability associated with both sporadic adenomatous\\npolyps and CRC [150\\n].\\nHepatobiliary tumors have also been associated with\\nvarious conditionally pathogenic bacteria. Enrichment of\\nFusobacterium, Prevotella, and Novosphingobium was\\nobserved in extrahepatic cholangiocarcinoma tissues\\n[108]. Fusobacteria enrichment was observed in duode-\\nnal fluids from PC patients, especially those with\\nshort‐term survival [ 151]. The presence of Pseudox-\\nanthomonas in PC tissues correlated with increased\\ndensity of CD8\\n+ T cells and improved overall survival\\n(OS) [152]. In contrast, animal experiments showedB.\\npseudolongum migrating from intestine to pancreas and\\nenriched in PC tissues, obstructing immunity mediated\\nby T cells, and suppressing immune responses to\\npancreatic tumors [110]. Additionally, large amounts of\\nFusobacterium in PC tumor tissues often indicate poor\\nprognosis [153].\\nF. nucleatum is a common opportunistic pathogen\\nresiding in the gastrointestinal tract and oral cavity,\\nassociated with multiple cancers, including CRC, PC,\\noral cancer, and gliomas [40, 80, 110, 154]. This micro-\\norganism is significantly enriched in neoplastic tissues\\ncompared to healthy mucosa across these diverse cancer\\ntypes. F. nucleatum contributes to tumor development\\nand progression through diverse mechanisms: it repro-\\ngrams host DNA methylation patterns, potentially facil-\\nitating the initiation of PC [79]; enhances CRC metastasis\\nby suppressing anti ‐tumor immunity, promoting\\nimmune evasion, and modulating the E ‐cadherin/β‐\\ncatenin signaling pathway via FadA adhesin [ 81,\\n154–156]; recruits tumor ‐associated macrophages\\n(TAMs) to the TME via CCL20 upregulation, thereby\\naccelerating the progression of CRC [157]; promotes M2\\nmacrophage polarization and activation [ 158]; and\\nstimulates cancer cell invasion in PC through autocrine/\\nparacrine pathways [159]. Within the TME, intratumoral\\nF. nucleatum modulates immune responses by inducing\\nT helper 17 (Th17) cell enrichment and promoting\\nsecretion of IL‐17 family cytokines via the metabolite‐\\nsensing receptor FFAR2, thereby establishing pro ‐\\ninflammatory microenvironmental conditions [ 160].\\nClinically, F. nucleatum serves as a specific diagnostic\\nand prognostic biomarker in CRC, GC, oral cancer, PC,\\nand lung cancer, with its intratumoral abundance dem-\\nonstrating a significant negative correlation with OS in\\nCRC patients [40, 161–164]. Furthermore, F. nucleatum\\ninduces chemoresistance in CRC, particularly to ox-\\naliplatin, by inhibiting caspase ‐mediated cascades via\\nautophagy modulation and BIRC3 upregulation\\n[165–167]. As previously described, F. nucleatum can\\nregulate the expression ofβ‐catenin [156]. Studies have\\nshown that theβ‐\\ncatenin signaling pathway, which can\\nbe activated through multiple pathways, is associated\\nwith the development of resistance to lenvatinib in HCC\\n[168–170]. Therefore, the modulation of β‐catenin ex-\\npression byF. nucleatummay influence the sensitivity of\\ntumor cells to lenvatinib, although the precision molec-\\nular mechanisms and clinical relevance of this associa-\\ntion require further rigorous investigation. Conversely,\\nthe development of targeted therapeutic approaches,\\nsuch as employing butyrate derivatives to inhibit bacte-\\nrial proliferation and adhesion, utilizing engineered\\nbacterial outer membrane vesicles to specifically target\\nF. nucleatum, or leveraging its presence to implement\\nimmunostimulatory strategies, collectively demonstrates\\nMICROBIOME IN CANCER | 7 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License\"),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content='promising therapeutic potential, particularly when inte-\\ngrated with established treatment modalities [165, 171].\\nAssociation of viruses with tumorigenesis\\nViruses contribute to tumor development through both\\ndirect oncogenic mechanisms and modulation of the host\\nimmune microenvironment [ 172–174]. The molecular\\nmechanisms span multiple pathways, including onco-\\ngenic virus‐induced genetic material damage, microbiota\\ndysregulation, and phage‐mediated metabolic network\\nregulation [175–177]. Among these, oncogenic viruses\\nfunction as primary drivers of tumorigenesis, exerting\\ncritical regulatory influences on cancer development.\\nTable 2 summarizes the oncogenic mechanisms of major\\nviral types in different tumors and their clinical signifi-\\ncance, providing a systematic reference for under-\\nstanding virus‐mediated tumorigenesis.\\nCarcinogenic viruses and tumorigenesis\\nOncogenic viruses represent a specialized category of\\nviruses that infect host cells and interfere with cellular\\ngrowth regulatory mechanisms, thereby inducing aber-\\nrant cell proliferation and ultimately promoting tumor\\nformation [191–195]. These viruses can promote tumor-\\nigenesis through multiple mechanisms, including\\ninduction of DNA damage, disruption of DNA damage\\nresponse (DDR) system, expression of oncogenic pro-\\nteins, activating cancer‐related signaling pathways, acti-\\nvation of cell cycle control, and dysregulation of\\napoptosis [196, 197]. Within the spectrum of digestive\\nsystem oncogenic viruses, HBV and HCV represent the\\npredominant pathogens responsible for HCC. These\\nviruses could promote HCC development and progres-\\nsion through multiple mechanisms, including inducing\\nepithelial–mesenchymal transition (EMT), regulating the\\ncell cycle, and inducing chronic inflammatory responses\\nand tissue fibrosis [175, 178, 198]. Epstein–Barr virus\\n(EBV) demonstrates strong associations with B ‐cell\\nlymphoproliferative disorders and nasopharyngeal car-\\ncinoma (NPC) [ 179], while additionally influencing\\ntumor development through modulation of intestinal\\nmicrobiota and their metabolites [199], as well as con-\\ntributing to GC progression [180]. Human papilloma-\\nvirus (HPV) may contribute to cervical cancer (CC)\\npathogenesis by suppressing immune surveillance and\\ninhibiting cytotoxic responses in lymphocytes, thereby\\nsignificantly influencing the TIME [181]. Additionally,\\nhuman immunodeficiency virus (HIV) may contribute\\nto the development and progression of oral and\\nanal cancerous lesions through alteration of regional\\nmicrobiota composition via mechanisms linked to\\ncarcinogenesis [182].\\nPhages and tumorigenesis\\nIn recent years, the role of phages in tumor development\\nhas garnered increasing attention, with their regulatory\\nmechanisms exhibiting distinct characteristics across\\ndifferent tumor types [200–206]. Several studies in CRC\\nhave revealed specific alterations in phage communities.\\nPhages of the familiesSiphoviridae and Myoviridae were\\nsignificantly enriched in the feces of CRC patients, while\\nstreptococcal phages and Vibrio‐inhabiting phage popu-\\nlations were also abnormally increased [ 176, 183 ].\\nAlthough the biological significance of temperate phages\\nremains controversial [176], experiments have shown\\nthat they may indirectly influence CRC progression\\nthrough modulation of gut microbial composition and\\nmetabolic functions [184].\\nDysbiosis in phage communities induces lysis of host\\nbacteria and subsequent alterations in gut microbial abun-\\ndance, resulting in the release of antigenic substances\\nincluding proteins, lipids, and nucleic acids from bacteria.\\nThis process can induce host inflammatory responses and\\ntissue damage, promoting CRC [185]. Notably,Caudovirales\\nphages exhibit dual roles inthe intestinal inflammatory\\nenvironment, both enhancing survival in CRC‐prone ani-\\nmals through inhibition of oncogenic bacterial colonization,\\nwhile paradoxically exacerbating IBD pathology through\\ntheir over‐amplification [186]. Certain phages directly inter-\\nact with cancer cells and modulate the expression levels of\\nproteins involved in carcinogenesis and metastasis, particu-\\nlarly integrins [207]. These findings indicate that phages\\nfunction as both potential drivers of tumorigenesis and novel\\nbioregulatory targets, though their specific mechanisms\\nwarrant comprehensive investigation within the context of\\ncancer‐specific microbial‐immune microenvironments.\\nOther viruses and tumorigenesis\\nBeyond established oncogenic viruses, diverse viral\\nagents contribute to tumorigenesis and progression\\nthrough both direct and indirect mechanisms. In patients\\nwith persistent enterovirus 71 (EV71) infection, chronic\\npresence of EV71 viral antigen in intestinal tissue sig-\\nnificantly correlates with advanced CRC, potentially\\npromoting disease progression through recruitment and\\nstimulation of Th17 cells in the TME [177]. In BC studies,\\n8 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content='TABLE 2 Oncogenic mechanisms of different virus types in different tumors and their clinical significance.\\nVirus type\\nRepresentative\\nviruses\\nAssociated\\ncancer types Primary carcinogenic mechanisms Clinical significance References\\nCarcinogenic viruses HBV\\nHCV\\nHCC • Induce chronic inflammation and fibrosis;\\n• Regulate cell cycle;\\n• Promote EMT.\\nHBV/HCV vaccination and antiviral therapy\\nsignificantly reduce HCC risk.\\n[175, 178]\\nEBV NPC\\nGC\\nB‐cell lymphoma\\n• Encode oncoproteins (such as LMP1 and\\nEBNA2);\\n• Modulate gut microbiota metabolites.\\nEBV DNA detection for early screening of NPC. [179, 180]\\nHPV CC • Suppress immune surveillance;\\n• Inhibit lymphocyte cytotoxicity;\\n• Regulate TIME.\\nThe HPV vaccine can prevent CC and other\\nrelated cancers.\\n[181]\\nHIV Oral cancer\\nAnal cancer\\n• Alter local microbiota composition;\\n• Chronic immunosuppression promotes\\ncarcinogenesis.\\nHIV‐infected individuals require intensified\\ncancer screening.\\n[182]\\nPhages Siphoviridae\\nMyoviridae\\nCRC • Regulate gut microbiota composition;\\n• Release bacterial antigens to induce\\ninflammation;\\n• Indirectly promote carcinogenic bacterial\\nproliferation.\\nPhage community analysis as a potential early\\ndiagnostic marker for CRC.\\n[176, 183, 184]\\nCaudovirales CRC Bidirectional role:\\n• Inhibit carcinogen colonization (anti‐cancer);\\n• Exacerbate inflammation (pro‐cancer) upon\\novergrowth.\\nPhage therapy requires precise regulation to avoid\\ninflammatory aggravation.\\n[185, 186]\\nOther viruses EV71 Advanced CRC • Recruit Th17 cells to promote TME\\ninflammation;\\n• Persistent viral antigen stimulation drives cell\\nproliferation.\\nEV71‐persistent infection requires enhanced CRC\\nmonitoring.\\n[\\n177]\\nMMTV\\nBLV\\nBC • Genomic integration activates host oncogenes;\\n• Evade immune clearance.\\nMMTV/BLV antibody detection as a potential BC\\nrisk prediction tool.\\n[187]\\nHCMV BC\\nGlioma\\n• Encode oncogenes;\\n• Suppress tumor suppressor proteins.\\nHCMV‐targeted therapy may enhance chemo/\\nradiotherapy sensitivity.\\n[188, 189]\\nHERV‐KO C\\nPC\\nHCC\\n• Retrotransposition induces genomic instability;\\n• Activate pro‐cancer signaling pathways.\\nHERV‐K expression correlates with tumor\\nprognosis.\\n[190]\\nAbbreviations: BLV, bovine leukemia virus; CC, cervical cancer; EBNA2, Epstein–Barr virus nuclear antigen‐2; EBV, Epstein–Barr virus; HBV, Hepatitis B virus; HCV, Hepatitis C virus; HIV, human\\nimmunodeficiency virus; HPV, human papilloma virus; EMT, epithelial‐mesenchymal transition; EV71, enterovirus 71; HCMV, human cytomegalovirus; HERV‐K, human endogenous retrovirus‐K; LMP1, latent\\nmembrane protein1; MMTV, mouse mammary tumor virus; NPC, nasopharyngeal carcinoma; OC, ovarian cancer; Th17, T helper 17; TIME, tumor immune microenvironment.\\nMICROBIOME IN CANCER | 9 of 104\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License'),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content='both mouse mammary tumor virus (MMTV) and Bovine\\nleukemia virus (BLV) have been associated with BC\\ndevelopment, with studies suggesting a likely causal rela-\\ntionship [ 187]. Additionally, human cytomegalovirus\\n(HCMV) has been associated with various malignant\\ntumors, including BC, possibly related to potent oncogenes\\nin its genome [188, 189]. Human endogenous retrovirus K\\n(HERV‐K) is highly expressed in various reproductive\\ntumors, including ovarian cancer, and has also been asso-\\nciated with the progression of PC and HCC [190].\\nAssociation of fungi with tumorigenesis\\nFungal communities represent a key component of the\\nhuman microbiome, with their homeostatic imbalances\\nstrongly associated with tumordevelopment. Specific fungal\\nspecies induce inflammatory responses and disrupt immune\\nhomeostasis, consequently promoting tumorigenesis and\\ntumor progression [208–211]. This process manifests pri-\\nmarily through aberrant proliferation of commensal fungi or\\nexcessive colonization by pathogenic fungi [ 212, 213].\\nA ss h o w ni nT a b l e3, distinct tumors exhibit characteristic\\nfungal signatures that correlate significantly with tumori-\\ngenesis, progression, and prognosis, providing critical in-\\nsights into fungal ‐mediated tumor pathogenesis and\\npotential therapeutic interventions.\\nSymbiotic fungi and tumorigenesis\\nThe human body harbors numerous fungi that establish\\nsymbiotic relationships with host tissues, predomi-\\nnantly colonizing mucosal surfaces (including the oral\\ncavity, intestines, and vagina) and skin [223, 224]. These\\nsymbiotic fungi serve critical functions in maintaining\\nmicrobial community equilibrium, regulating metabolic\\nprocesses, and preserving immune system homeostasis\\n[225]. However, ecological dysbiosis within the fungal\\nmicrobiome contributes to both initiation and progres-\\nsion of digestive system malignancies through diverse\\nmechanisms, including modulation of the host immune\\nmicroenvironment and fungal –bacterial interactions\\n[212]. Studies demonstrate that fungal community\\nTABLE 3 Characteristic fungal alteration patterns in tumors and their mechanisms.\\nCancer type\\nCharacteristic fungal\\nalterations Mechanism Clinical relevance References\\nGC Symbiotic fungal imbalance:\\nMalassezia globosa ↑\\nSaccharomyces cerevisiae ↓\\n• Fungal dysbiosis induces chronic\\ninflammation;\\n• Disrupts mucosal barrier function.\\nSalivary/tongue coating fungal\\ncommunities as early screening\\nmarkers for GC.\\n[214, 215]\\nCRC Pathogenic fungal\\nenrichment:\\nCandida albicans ↑\\nSchizosaccharomyces\\npombe ↑\\nMalassezia spp. ↑\\n• Activate Wnt/β‐catenin pathway to\\npromote proliferation;\\n• Secrete pro‐cancer proteins;\\n• Induce cell adhesion gene\\ndysregulation.\\n• Candida abundance correlates\\nwith advanced CRC;\\n• Fungal translocation to blood\\nindicates metastasis risk.\\n[216–218]\\nPC 3000 ‐ fold fungal\\nconcentration ↑:\\n• Complement C3 pathway activation\\npromotes pro‐inflammatory\\nmicroenvironment.\\n• Fungal load correlates with poor\\nprognosis;\\n• Potential biomarker for\\ntherapeutic resistance.\\n[219, 220]\\nMalassezia ↑\\nHCC Opportunistic fungal\\ncolonization:\\nCandida albicans ↑\\nMalassezia furfur ↑\\n• Induce oxidative stress;\\n• Synergize with HBV/HCV to\\npromote liver fibrosis.\\nFungal‐viral co‐infection\\naccelerates HCC progression.\\n[217]\\nBC Tumor microenvironment\\ndysbiosis:\\nCandida albicans infection ↑\\n• Increase Treg cells to suppress\\nimmunity;\\n• Promote angiogenesis.\\nAntifungal therapy may enhance\\nICIs efficacy.\\n[213]\\nMelanoma Saccharomycetales ↑ • T cells recognize melanoma\\nantigens.\\nFungal diversity may correlate\\nwith immunotherapy response.\\n[221]\\nPCa Plasma fungal dysbiosis:\\nSordariomycetes ↑\\n• Promote inflammatory\\nmicroenvironment.\\n• Specific fungi may assist in\\nidentifying PCa\\n[222]\\nAbbreviations: ↑, Increased; ↓, Decreased; ICIs, immune checkpoint inhibitors; IL‐6, Interleukin‐6; PCa, prostate cancer; Treg, regulatory T cells.\\n10 of 104 | LIN ET AL.\\n 2770596x, 2025, 5, Downloaded from https://onlinelibrary.wiley.com/doi/10.1002/imt2.70070 by National Health And Medical Research Council, Wiley Online Library on [26/12/2025]. See the Terms and Conditions (https://onlinelibrary.wiley.com/terms-and-conditions) on Wiley Online Library for rules of use; OA articles are governed by the applicable Creative Commons License')]"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "minimal_docs[1:10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "71a8ff78",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Split the documents into smaller chunks\n",
    "def text_split(minimal_docs):\n",
    "    text_splitter = RecursiveCharacterTextSplitter(\n",
    "        chunk_size=500,\n",
    "        chunk_overlap=20,\n",
    "    )\n",
    "    texts_chunk = text_splitter.split_documents(minimal_docs)\n",
    "    return texts_chunk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "efc7c869",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of chunks: 1454\n"
     ]
    }
   ],
   "source": [
    "texts_chunk = text_split(minimal_docs)\n",
    "print(f\"Number of chunks: {len(texts_chunk)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "1ddd5ae9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content='Jindong Xie15 | Xinpei Deng16 | Qi Wang17 | Jianying Xu18 |\\nWenjie Shi19 | Chang Qi20 | Chunrun Qu21 | Xufeng Huang22,23 |\\nAndrás Hajdu22 | Chaoqun Li24 | Changmin Peng25 | Xuanye Cao26 |\\nGuangsheng Pei27 | Lin Zhang28 | Yujia Huo28 | Jiabao Xu29 |\\nAntonino Glaviano30 | Attila Gábor Szöllősi31 | Sicheng Bian32 |\\nZhengrui Li33 | Hailin Tang15 | Bufu Tang34 | Zaoqu Liu35 |\\nJian Zhang2 | Kai Miao5,36 | Quan Cheng37,38 | Ting Wei2 |\\nShuofeng Yuan39,40 | Peng Luo1,2,40\\nCorrespondence'),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content=\"Correspondence\\nQuan Cheng, Department of Neurosurgery,\\nXiangya Hospital, Central South University,\\nChangsha 410008, China.\\nEmail: chengquan@csu.edu.cn\\nTing Wei, Department of Oncology,\\nZhujiang Hospital, Southern Medical\\nUniversity, Guangzhou 510282, China.\\nEmail: weitingyouyou@qq.com\\nShuofeng Yuan, Department of Infectious\\nDisease and Microbiology, The University of\\nHong Kong‐Shenzhen Hospital, Shenzhen\\n518053, China.\\nEmail: yuansf@hku.hk\\nPeng Luo, Donghai County People's\"),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content='Hospital (Affiliated Kangda College of\\nNanjing Medical University), Lianyungang,\\nChina.\\nEmail: luopeng@smu.edu.cn\\nAbstract\\nThe human microbiome is now recognized as a central regulator of cancer\\nbiology, intricately shaping tumor development, immune dynamics, and thera-\\npeutic response. This comprehensive review delineates the multifaceted roles of\\nb a c t e r i a ,v i r u s e s ,a n df u n g ii nm o d u lating the tumor microenvironment and'),\n",
       " Document(metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content='systemic immunity across diverse cancer ty p e s .W es y n t h e s i z ec u r r e n te v i d e n c e\\non how microbial dysbiosis promotes carcinogenesis via chronic inflammation,\\nmetabolic reprogramming, genotoxic stress, immune evasion, and epigenetic\\nremodeling. This review emphasizes organ ‐specific microbiome signatures\\nand highlights their potential as non‐invasive biomarkers for early detection,\\ntreatment stratification, and prognosis. Furthermore, we explore the impact of')]"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "texts_chunk[1:5]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "22ed5d27",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/tmp/ipykernel_886298/1593868590.py:10: LangChainDeprecationWarning: The class `HuggingFaceEmbeddings` was deprecated in LangChain 0.2.2 and will be removed in 1.0. An updated version of the class exists in the `langchain-huggingface package and should be used instead. To use it run `pip install -U `langchain-huggingface` and import as `from `langchain_huggingface import HuggingFaceEmbeddings``.\n",
      "  embeddings = HuggingFaceEmbeddings(\n"
     ]
    }
   ],
   "source": [
    "# from langchain.embeddings import HuggingFaceEmbeddings\n",
    "from langchain_community.embeddings import HuggingFaceEmbeddings\n",
    "\n",
    "\n",
    "def download_embeddings():\n",
    "    \"\"\"\n",
    "    Download and return the HuggingFace embeddings model.\n",
    "    \"\"\"\n",
    "    model_name = \"sentence-transformers/all-MiniLM-L6-v2\"\n",
    "    embeddings = HuggingFaceEmbeddings(\n",
    "        model_name=model_name,\n",
    "        model_kwargs={\"device\": \"cpu\"},                 # for CPU\n",
    "        # encode_kwargs={\"normalize_embeddings\": True}, # for CPU\n",
    "    )\n",
    "    return embeddings\n",
    "\n",
    "embedding = download_embeddings()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "671e0570",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "HuggingFaceEmbeddings(client=SentenceTransformer(\n",
       "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False, 'architecture': 'BertModel'})\n",
       "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
       "  (2): Normalize()\n",
       "), model_name='sentence-transformers/all-MiniLM-L6-v2', cache_folder=None, model_kwargs={'device': 'cpu'}, encode_kwargs={}, multi_process=False, show_progress=False)"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "embedding"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "09d50251",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[-0.03447726368904114,\n",
       " 0.031023193150758743,\n",
       " 0.0067349947057664394,\n",
       " 0.02610894665122032,\n",
       " -0.03936201333999634,\n",
       " -0.16030244529247284,\n",
       " 0.06692396849393845,\n",
       " -0.006441481877118349,\n",
       " -0.04745052754878998,\n",
       " 0.014758824370801449,\n",
       " 0.07087526470422745,\n",
       " 0.05552756413817406,\n",
       " 0.019193341955542564,\n",
       " -0.026251332834362984,\n",
       " -0.010109550319612026,\n",
       " -0.026940450072288513,\n",
       " 0.022307418286800385,\n",
       " -0.022226607427001,\n",
       " -0.1496926248073578,\n",
       " -0.01749303564429283,\n",
       " 0.007676276378333569,\n",
       " 0.0543522946536541,\n",
       " 0.00325443921610713,\n",
       " 0.03172590211033821,\n",
       " -0.08462145179510117,\n",
       " -0.0294059868901968,\n",
       " 0.051595594733953476,\n",
       " 0.048124048858881,\n",
       " -0.003314807778224349,\n",
       " -0.05827918276190758,\n",
       " 0.04196925461292267,\n",
       " 0.022210685536265373,\n",
       " 0.1281888484954834,\n",
       " -0.02233896590769291,\n",
       " -0.01165627222508192,\n",
       " 0.06292841583490372,\n",
       " -0.03287633880972862,\n",
       " -0.09122608602046967,\n",
       " -0.031175406649708748,\n",
       " 0.052699532359838486,\n",
       " 0.04703480750322342,\n",
       " -0.08420310169458389,\n",
       " -0.030056176707148552,\n",
       " -0.02074483036994934,\n",
       " 0.009517806582152843,\n",
       " -0.003721789922565222,\n",
       " 0.0073432838544249535,\n",
       " 0.0393243208527565,\n",
       " 0.09327404201030731,\n",
       " -0.003788574365898967,\n",
       " -0.052742086350917816,\n",
       " -0.05805818736553192,\n",
       " -0.006864399183541536,\n",
       " 0.005283276084810495,\n",
       " 0.08289305120706558,\n",
       " 0.019362781196832657,\n",
       " 0.006284540984779596,\n",
       " -0.010330780409276485,\n",
       " 0.009032380767166615,\n",
       " -0.037683747708797455,\n",
       " -0.045206114649772644,\n",
       " 0.024016333743929863,\n",
       " -0.006944159045815468,\n",
       " 0.013491624966263771,\n",
       " 0.10005496442317963,\n",
       " -0.0716838613152504,\n",
       " -0.02169506810605526,\n",
       " 0.03161845728754997,\n",
       " -0.051634665578603745,\n",
       " -0.08224772661924362,\n",
       " -0.06569330394268036,\n",
       " -0.009895351715385914,\n",
       " 0.005816372577100992,\n",
       " 0.07355459779500961,\n",
       " -0.034050341695547104,\n",
       " 0.024886108934879303,\n",
       " 0.014488051645457745,\n",
       " 0.026457343250513077,\n",
       " 0.009656710550189018,\n",
       " 0.030217260122299194,\n",
       " 0.05280395597219467,\n",
       " -0.07535987347364426,\n",
       " 0.00989716686308384,\n",
       " 0.02983682230114937,\n",
       " 0.017555590718984604,\n",
       " 0.02309199795126915,\n",
       " 0.0019338536076247692,\n",
       " 0.001400224631652236,\n",
       " -0.047175969928503036,\n",
       " -0.011194318532943726,\n",
       " -0.11420144885778427,\n",
       " -0.019811972975730896,\n",
       " 0.0402662418782711,\n",
       " 0.002192958490923047,\n",
       " -0.07979219406843185,\n",
       " -0.025382284075021744,\n",
       " 0.09448301047086716,\n",
       " -0.028981119394302368,\n",
       " -0.14500251412391663,\n",
       " 0.23097746074199677,\n",
       " 0.027731135487556458,\n",
       " 0.032111458480358124,\n",
       " 0.03106500208377838,\n",
       " 0.04283282160758972,\n",
       " 0.06423778831958771,\n",
       " 0.03216312825679779,\n",
       " -0.0048766965046525,\n",
       " 0.055699385702610016,\n",
       " -0.03753235936164856,\n",
       " -0.021505502983927727,\n",
       " -0.02834261767566204,\n",
       " -0.02884693816304207,\n",
       " 0.03835303708910942,\n",
       " -0.017468685284256935,\n",
       " 0.052485305815935135,\n",
       " -0.07487601041793823,\n",
       " -0.031259723007678986,\n",
       " 0.021841557696461678,\n",
       " -0.0398956760764122,\n",
       " -0.00858711265027523,\n",
       " 0.026956627145409584,\n",
       " -0.04849550873041153,\n",
       " 0.011469880118966103,\n",
       " 0.029618194326758385,\n",
       " -0.020572183653712273,\n",
       " 0.013103886507451534,\n",
       " 0.028833463788032532,\n",
       " -3.19419761545428e-33,\n",
       " 0.06478208303451538,\n",
       " -0.018130170181393623,\n",
       " 0.051789961755275726,\n",
       " 0.12198273837566376,\n",
       " 0.028780192136764526,\n",
       " 0.008721995167434216,\n",
       " -0.07052117586135864,\n",
       " -0.016907300800085068,\n",
       " 0.040739696472883224,\n",
       " 0.0421161986887455,\n",
       " 0.025447171181440353,\n",
       " 0.03574628010392189,\n",
       " -0.049144767224788666,\n",
       " 0.002129028318449855,\n",
       " -0.015546568669378757,\n",
       " 0.05073055997490883,\n",
       " -0.04818534106016159,\n",
       " 0.035880621522665024,\n",
       " -0.004067053087055683,\n",
       " 0.10172472149133682,\n",
       " -0.05597001686692238,\n",
       " -0.010681030340492725,\n",
       " 0.011235754936933517,\n",
       " 0.09068653732538223,\n",
       " 0.004234452731907368,\n",
       " 0.035138681530952454,\n",
       " -0.009702864103019238,\n",
       " -0.09386512637138367,\n",
       " 0.0928555279970169,\n",
       " 0.008004914969205856,\n",
       " -0.00770534435287118,\n",
       " -0.052086688578128815,\n",
       " -0.01258802693337202,\n",
       " 0.0032669261563569307,\n",
       " 0.006013570353388786,\n",
       " 0.00758156506344676,\n",
       " 0.010517144575715065,\n",
       " -0.08634556829929352,\n",
       " -0.06987875699996948,\n",
       " -0.002533882623538375,\n",
       " -0.09097660332918167,\n",
       " 0.04688732326030731,\n",
       " 0.05207652226090431,\n",
       " 0.007193798664957285,\n",
       " 0.010903642512857914,\n",
       " -0.005229541566222906,\n",
       " 0.01393731590360403,\n",
       " 0.021968325600028038,\n",
       " 0.03420865163207054,\n",
       " 0.06022467091679573,\n",
       " 0.00011664949852274731,\n",
       " 0.01473192311823368,\n",
       " -0.07008923590183258,\n",
       " 0.02849900908768177,\n",
       " -0.027601731941103935,\n",
       " 0.01076840702444315,\n",
       " 0.034830935299396515,\n",
       " -0.022487888112664223,\n",
       " 0.00976902712136507,\n",
       " 0.07722784578800201,\n",
       " 0.02158835344016552,\n",
       " 0.11495622247457504,\n",
       " -0.0680011510848999,\n",
       " 0.023760972544550896,\n",
       " -0.01598387211561203,\n",
       " -0.01782696135342121,\n",
       " 0.06439491361379623,\n",
       " 0.03202570229768753,\n",
       " 0.05027029663324356,\n",
       " -0.005913723725825548,\n",
       " -0.03370802849531174,\n",
       " 0.01784025877714157,\n",
       " 0.016573328524827957,\n",
       " 0.06329656392335892,\n",
       " 0.03467714414000511,\n",
       " 0.046473532915115356,\n",
       " 0.09790614247322083,\n",
       " -0.006635506171733141,\n",
       " 0.025207100436091423,\n",
       " -0.07798831909894943,\n",
       " 0.01692643202841282,\n",
       " -0.0009458272834308445,\n",
       " 0.022471915930509567,\n",
       " -0.038253214210271835,\n",
       " 0.09570483863353729,\n",
       " -0.005350810941308737,\n",
       " 0.010469065047800541,\n",
       " -0.11524052172899246,\n",
       " -0.013262507505714893,\n",
       " -0.010709457099437714,\n",
       " -0.08311727643013,\n",
       " 0.07327357679605484,\n",
       " 0.04939224570989609,\n",
       " -0.008994328789412975,\n",
       " -0.09584560245275497,\n",
       " 3.366147459724626e-33,\n",
       " 0.12493187934160233,\n",
       " 0.019349711015820503,\n",
       " -0.058225732296705246,\n",
       " -0.03598819673061371,\n",
       " -0.050746724009513855,\n",
       " -0.04566234350204468,\n",
       " -0.08260343968868256,\n",
       " 0.14819477498531342,\n",
       " -0.08842121064662933,\n",
       " 0.06027441844344139,\n",
       " 0.051030173897743225,\n",
       " 0.010303194634616375,\n",
       " 0.14121423661708832,\n",
       " 0.030813779681921005,\n",
       " 0.06103309988975525,\n",
       " -0.05285125970840454,\n",
       " 0.13664893805980682,\n",
       " 0.009189874865114689,\n",
       " -0.017325179651379585,\n",
       " -0.012848635204136372,\n",
       " -0.007995332591235638,\n",
       " -0.0509801059961319,\n",
       " -0.05235063657164574,\n",
       " 0.007593054790049791,\n",
       " -0.015166284516453743,\n",
       " 0.016960343345999718,\n",
       " 0.021270545199513435,\n",
       " 0.020558038726449013,\n",
       " -0.12002808600664139,\n",
       " 0.014461803250014782,\n",
       " 0.026759866625070572,\n",
       " 0.02533065713942051,\n",
       " -0.04275466501712799,\n",
       " 0.006768485996872187,\n",
       " -0.01445857435464859,\n",
       " 0.04526198282837868,\n",
       " -0.09147650748491287,\n",
       " -0.019439077004790306,\n",
       " -0.01783348247408867,\n",
       " -0.05491011589765549,\n",
       " -0.0526411235332489,\n",
       " -0.01045901793986559,\n",
       " -0.05201609060168266,\n",
       " 0.020892055705189705,\n",
       " -0.0799703299999237,\n",
       " -0.012111252173781395,\n",
       " -0.05773142725229263,\n",
       " 0.023178258910775185,\n",
       " -0.008031770586967468,\n",
       " -0.025989318266510963,\n",
       " -0.07995671033859253,\n",
       " -0.020728854462504387,\n",
       " 0.048817697912454605,\n",
       " -0.02038910798728466,\n",
       " -0.04917658120393753,\n",
       " 0.014159663580358028,\n",
       " -0.06362202763557434,\n",
       " -0.007807412650436163,\n",
       " 0.016431495547294617,\n",
       " -0.025682462379336357,\n",
       " 0.01338114868849516,\n",
       " 0.026248749345541,\n",
       " 0.009978404268622398,\n",
       " 0.06322886794805527,\n",
       " 0.002672194968909025,\n",
       " -0.006582746747881174,\n",
       " 0.016631951555609703,\n",
       " 0.03236645832657814,\n",
       " 0.03794249892234802,\n",
       " -0.036376092582941055,\n",
       " -0.006910891272127628,\n",
       " 0.00015962125326041132,\n",
       " -0.0016335457330569625,\n",
       " -0.027278156951069832,\n",
       " -0.02803807333111763,\n",
       " 0.04968145489692688,\n",
       " -0.02886720560491085,\n",
       " -0.0024180745240300894,\n",
       " 0.014774860814213753,\n",
       " 0.00976458378136158,\n",
       " 0.005797586403787136,\n",
       " 0.013486133888363838,\n",
       " 0.005567891523241997,\n",
       " 0.0372270904481411,\n",
       " 0.007232534699141979,\n",
       " 0.04015626385807991,\n",
       " 0.08150327205657959,\n",
       " 0.07199167460203171,\n",
       " -0.013056175783276558,\n",
       " -0.04288202524185181,\n",
       " -0.011011245660483837,\n",
       " 0.00489779794588685,\n",
       " -0.00922971498221159,\n",
       " 0.03519147261977196,\n",
       " -0.05103498697280884,\n",
       " -1.571437557856825e-08,\n",
       " -0.08862442523241043,\n",
       " 0.023909350857138634,\n",
       " -0.01623876765370369,\n",
       " 0.03170047700405121,\n",
       " 0.02728419564664364,\n",
       " 0.052468810230493546,\n",
       " -0.047070957720279694,\n",
       " -0.05884748324751854,\n",
       " -0.0632082149386406,\n",
       " 0.04088851809501648,\n",
       " 0.04982799291610718,\n",
       " 0.10655169934034348,\n",
       " -0.07450232654809952,\n",
       " -0.012495421804487705,\n",
       " 0.018370645120739937,\n",
       " 0.03947414830327034,\n",
       " -0.024797946214675903,\n",
       " 0.01451632846146822,\n",
       " -0.03706920146942139,\n",
       " 0.020015766844153404,\n",
       " -4.8584795877104625e-05,\n",
       " 0.009866534732282162,\n",
       " 0.024838751181960106,\n",
       " -0.052458129823207855,\n",
       " 0.029314173385500908,\n",
       " -0.08719193935394287,\n",
       " -0.014499752782285213,\n",
       " 0.026019064709544182,\n",
       " -0.018746372312307358,\n",
       " -0.07620514929294586,\n",
       " 0.035043273121118546,\n",
       " 0.10363952070474625,\n",
       " -0.02805054932832718,\n",
       " 0.012718182988464832,\n",
       " -0.07632551342248917,\n",
       " -0.018652359023690224,\n",
       " 0.024976730346679688,\n",
       " 0.08144530653953552,\n",
       " 0.06875889003276825,\n",
       " -0.0640566423535347,\n",
       " -0.08389384299516678,\n",
       " 0.06136242300271988,\n",
       " -0.033545512706041336,\n",
       " -0.10615336149930954,\n",
       " -0.04008055105805397,\n",
       " 0.03253021091222763,\n",
       " 0.07662486284971237,\n",
       " -0.0730162188410759,\n",
       " 0.0003375500382389873,\n",
       " -0.04087163507938385,\n",
       " -0.07578852027654648,\n",
       " 0.02752763219177723,\n",
       " 0.07462543994188309,\n",
       " 0.017717270180583,\n",
       " 0.09121846407651901,\n",
       " 0.11022017151117325,\n",
       " 0.0005697978194803,\n",
       " 0.051463350653648376,\n",
       " -0.014551280066370964,\n",
       " 0.03323202207684517,\n",
       " 0.02379225380718708,\n",
       " -0.022889815270900726,\n",
       " 0.0389375202357769,\n",
       " 0.030206842347979546]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vector = embedding.embed_query(\"Hello world\")\n",
    "vector"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "4fde710e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Vector length: 384\n"
     ]
    }
   ],
   "source": [
    "print( \"Vector length:\", len(vector))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "0df2a4f9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 57,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from dotenv import load_dotenv\n",
    "import os\n",
    "load_dotenv()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "241584b9",
   "metadata": {},
   "outputs": [],
   "source": [
    "PINECONE_API_KEY = os.getenv(\"PINECONE_API_KEY\")\n",
    "OPENAI_API_KEY = os.getenv(\"OPENAI_API_KEY\")\n",
    "os.environ[\"PINECONE_API_KEY\"] = PINECONE_API_KEY\n",
    "os.environ[\"OPENAI_API_KEY\"] = OPENAI_API_KEY\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "ff7276bb",
   "metadata": {},
   "outputs": [],
   "source": [
    "from pinecone import Pinecone \n",
    "pinecone_api_key = PINECONE_API_KEY\n",
    "\n",
    "pc = Pinecone(api_key=pinecone_api_key)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "bd8f108f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<pinecone.pinecone.Pinecone at 0x74f0651b0af0>"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "f97626b5",
   "metadata": {},
   "outputs": [],
   "source": [
    "from pinecone import ServerlessSpec \n",
    "\n",
    "index_name = \"rag-guided-chatbot\" #lowercase and hyphens\n",
    "\n",
    "if not pc.has_index(index_name):\n",
    "    pc.create_index(\n",
    "        name = index_name,\n",
    "        dimension=384,  # Dimension of the embeddings\n",
    "        metric= \"cosine\",  # Cosine similarity\n",
    "        spec=ServerlessSpec(cloud=\"aws\", region=\"us-east-1\")\n",
    "    )\n",
    "\n",
    "\n",
    "index = pc.Index(index_name)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "dae49b02",
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_pinecone import PineconeVectorStore\n",
    "\n",
    "docsearch = PineconeVectorStore.from_documents(\n",
    "    documents=texts_chunk,\n",
    "    embedding=embedding,\n",
    "    index_name=index_name\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "62b28080",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load Existing index \n",
    "# Embed each chunk and upsert the embeddings into your Pinecone index.\n",
    "docsearch = PineconeVectorStore.from_existing_index(\n",
    "    index_name=index_name,\n",
    "    embedding=embedding\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b2fdc76e",
   "metadata": {},
   "source": [
    "## Add sample data to the existing Pinecone index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "cc313313",
   "metadata": {},
   "outputs": [],
   "source": [
    "ukw_info = Document(\n",
    "    page_content=\"Mr. UKW has a youtube channel on learning Machine learning technologies.\",\n",
    "    metadata={\"source\": \"Youtube\"}\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "1046d323",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['8b0b79d3-bcbe-4051-917a-f3d011812c71']"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "docsearch.add_documents(documents=[ukw_info])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "f0f56976",
   "metadata": {},
   "outputs": [],
   "source": [
    "retriever = docsearch.as_retriever(search_type=\"similarity\", search_kwargs={\"k\":2})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "89c7c64d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[Document(id='8b0b79d3-bcbe-4051-917a-f3d011812c71', metadata={'source': 'Youtube'}, page_content='Mr. UKW has a youtube channel on learning Machine learning technologies.'),\n",
       " Document(id='ab9437fa-62d1-4389-ac2b-736af56802ad', metadata={'source': 'data/microbiome_in_cancer.pdf'}, page_content='This is an open access article under the terms of theCreative Commons AttributionLicense, which permits use, distribution and reproduction in any medium, provided\\nthe original work is properly cited.\\nFor affiliations refer to page 65.\\nAnqi Lin, Minying Xiong, Aimin Jiang, Lihaoyun Huang, Hank Z. H. Wong, and Suyin Feng contributed equally.\\nZhengrui Li, Hailin Tang, Bufu Tang, Zaoqu Liu, Jian Zhang, Kai Miao were senior authors.')]"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "retrieved_docs = retriever.invoke(\"What does gut microbiomes contribute to?\")\n",
    "retrieved_docs\n",
    "retrieved_docs2 = retriever.invoke(\"What does Mr UKW have?\")\n",
    "retrieved_docs2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "c053668f",
   "metadata": {},
   "outputs": [],
   "source": [
    "from langchain_openai import ChatOpenAI\n",
    "\n",
    "chatModel = ChatOpenAI(model=\"gpt-4o-mini\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "fd20257d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# from langchain.chains import create_retrieval_chain\n",
    "# from langchain.chains.combine_documents import create_stuff_documents_chain\n",
    "# from langchain_core.prompts import ChatPromptTemplate\n",
    "\n",
    "\n",
    "from langchain_classic.chains.retrieval import create_retrieval_chain\n",
    "from langchain_classic.chains.combine_documents import create_stuff_documents_chain\n",
    "from langchain_core.prompts import ChatPromptTemplate"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "67d5b331",
   "metadata": {},
   "outputs": [],
   "source": [
    "system_prompt = (\n",
    "    \"You are an Medical assistant for question-answering tasks. \"\n",
    "    \"Use the following pieces of retrieved context to answer \"\n",
    "    \"the question. If you don't know the answer, say that you \"\n",
    "    \"don't know. Use three sentences maximum and keep the \"\n",
    "    \"answer concise.\"\n",
    "    \"\\n\\n\"\n",
    "    \"{context}\"\n",
    ")\n",
    "\n",
    "\n",
    "prompt = ChatPromptTemplate.from_messages(\n",
    "    [\n",
    "        (\"system\", system_prompt),\n",
    "        (\"human\", \"{input}\"),\n",
    "    ]\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "aec49f22",
   "metadata": {},
   "outputs": [],
   "source": [
    "question_answer_chain = create_stuff_documents_chain(chatModel, prompt)\n",
    "rag_chain = create_retrieval_chain(retriever, question_answer_chain)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "75ddec90",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Gut microbiomes contribute to nutrient absorption, maintaining gastrointestinal health, and regulating immune function. They help orchestrate microbial balance and can influence cancer pathogenesis. Additionally, they are involved in conditions like inflammatory bowel disease and colorectal cancers.\n"
     ]
    }
   ],
   "source": [
    "response = rag_chain.invoke({\"input\": \"What does gut microbiomes contribute to?\"})   #what is Acromegaly and gigantism?\"})\n",
    "print(response[\"answer\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "2d073e2a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Yes, Mr. UKW has a YouTube channel focused on learning machine learning technologies.\n"
     ]
    }
   ],
   "source": [
    "response = rag_chain.invoke({\"input\": \"does ukw has a youtube channel?\"})\n",
    "print(response[\"answer\"])"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "py_rag",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.19"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
